

<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>

</head>

<body>

    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637127159786238750%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJFYZ201909010%26RESULT%3d1%26SIGN%3dubiMp0lwax0araeuMrJwqnEG%252fTE%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JFYZ201909010&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JFYZ201909010&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>


    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201909010&amp;v=MDA1MzZXNzNCTHl2U2RMRzRIOWpNcG85RVpJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSc0Z5L24=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#76" data-title="&lt;b&gt;1 算  法&lt;/b&gt; "><b>1 算  法</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#77" data-title="&lt;b&gt;1.1 异质信息网络构建&lt;/b&gt;"><b>1.1 异质信息网络构建</b></a></li>
                                                <li><a href="#83" data-title="&lt;b&gt;1.2 元路径选择&lt;/b&gt;"><b>1.2 元路径选择</b></a></li>
                                                <li><a href="#86" data-title="&lt;b&gt;1.3 模型描述&lt;/b&gt;"><b>1.3 模型描述</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#106" data-title="&lt;b&gt;2 实验结果与分析&lt;/b&gt; "><b>2 实验结果与分析</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#107" data-title="&lt;b&gt;2.1 实验数据&lt;/b&gt;"><b>2.1 实验数据</b></a></li>
                                                <li><a href="#114" data-title="&lt;b&gt;2.2 性能分析&lt;/b&gt;"><b>2.2 性能分析</b></a></li>
                                                <li><a href="#124" data-title="&lt;b&gt;2.3 案例分析&lt;/b&gt;"><b>2.3 案例分析</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#130" data-title="&lt;b&gt;3 结  论&lt;/b&gt; "><b>3 结  论</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#80" data-title="图1 异质信息网络实例和元路径">图1 异质信息网络实例和元路径</a></li>
                                                <li><a href="#85" data-title="图2 LncRNA-疾病异质信息网络和元路径&lt;i&gt;LDLD&lt;/i&gt;">图2 LncRNA-疾病异质信息网络和元路径<i>LDLD</i></a></li>
                                                <li><a href="#90" data-title="图3 插入中间类型&lt;i&gt;M&lt;/i&gt;前后">图3 插入中间类型<i>M</i>前后</a></li>
                                                <li><a href="#113" data-title="&lt;b&gt;表1 lncRNA/基因-疾病异质信息网络中的具体信息&lt;/b&gt;"><b>表1 lncRNA/基因-疾病异质信息网络中的具体信息</b></a></li>
                                                <li><a href="#121" data-title="图4 留一交叉验证ROC曲线图">图4 留一交叉验证ROC曲线图</a></li>
                                                <li><a href="#127" data-title="&lt;b&gt;表2 Top10与卵巢癌有关的基因&lt;/b&gt;"><b>表2 Top10与卵巢癌有关的基因</b></a></li>
                                                <li><a href="#129" data-title="&lt;b&gt;表3 Top10与胃癌有关的基因&lt;/b&gt;"><b>表3 Top10与胃癌有关的基因</b></a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="3">


                                    <a id="bibliography_1" title="Bertone P,Stolc V,Royce T E,et al.Global identification of human transcribed sequences with genome tiling arrays[J].Science,2004,306(5705):2242- 2246" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Global Identification of Human Transcribed Sequences with Genome Tiling Arrays">
                                        <b>[1]</b>
                                        Bertone P,Stolc V,Royce T E,et al.Global identification of human transcribed sequences with genome tiling arrays[J].Science,2004,306(5705):2242- 2246
                                    </a>
                                </li>
                                <li id="5">


                                    <a id="bibliography_2" title="The ENCODE Project Consortium.Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project[J].Nature,2007,447(7146):799- 816" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project">
                                        <b>[2]</b>
                                        The ENCODE Project Consortium.Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project[J].Nature,2007,447(7146):799- 816
                                    </a>
                                </li>
                                <li id="7">


                                    <a id="bibliography_3" title="Kapranov P,Cheng J,Dike S,et al.RNA maps reveal new RNA classes and a possible function for pervasive transcription[J].Science,2007,316(5830):1484- 1488" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=RNA maps reveal new RNA classes and a possible function for pervasive transcription">
                                        <b>[3]</b>
                                        Kapranov P,Cheng J,Dike S,et al.RNA maps reveal new RNA classes and a possible function for pervasive transcription[J].Science,2007,316(5830):1484- 1488
                                    </a>
                                </li>
                                <li id="9">


                                    <a id="bibliography_4" title="Carninci P,Kasukawa T,Katayama S,et al.The transcriptional landscape of the mammalian genome[J].Science,2005,309(5740):1559- 1563" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The transcriptional landscape of the mammalian genome">
                                        <b>[4]</b>
                                        Carninci P,Kasukawa T,Katayama S,et al.The transcriptional landscape of the mammalian genome[J].Science,2005,309(5740):1559- 1563
                                    </a>
                                </li>
                                <li id="11">


                                    <a id="bibliography_5" title="Taft R J,Pang K C,Mercer T R,et al.Non-coding RNAs:Regulators of disease[J].The Journal of Pathology,2010,220(2):126- 139" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD00001142422&amp;v=MDU5NjJITnJvdEhZT2tOWTNrNXpCZGg0ajk5U1hxUnJ4b3hjTUg3UjdxZWJ1ZHRGQ3JsVkwzQUlsYz1OaWZjYXJPNEh0&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[5]</b>
                                        Taft R J,Pang K C,Mercer T R,et al.Non-coding RNAs:Regulators of disease[J].The Journal of Pathology,2010,220(2):126- 139
                                    </a>
                                </li>
                                <li id="13">


                                    <a id="bibliography_6" title="Derrien T,Johnson R,Bussotti G,et al.The GENCODE v7 catalog of human long noncoding RNAs:Analysis of their gene structure,evolution,and expression[J].Genome Research,2012,22(9):1775- 1789" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJRS&amp;filename=SJRS120925009235&amp;v=MTcxMDlSTTh6eFVTbURkOVNIN24zeEU5ZmJ2bktyaWZadTl1RkN2blU3aktLRndkTmlmWmZiSzZIdGpPcW85RmJla01D&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[6]</b>
                                        Derrien T,Johnson R,Bussotti G,et al.The GENCODE v7 catalog of human long noncoding RNAs:Analysis of their gene structure,evolution,and expression[J].Genome Research,2012,22(9):1775- 1789
                                    </a>
                                </li>
                                <li id="15">


                                    <a id="bibliography_7" title="Quek X C,Thomson D W,Maag J L,et al.lncRNAdb v2.0:Expanding the reference database for functional long noncoding RNAs[J].Nucleic Acids Research,2015,43(D1):168- 173" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=lnc RNAdb v2.0:expanding the reference database for functional long noncoding RNAs">
                                        <b>[7]</b>
                                        Quek X C,Thomson D W,Maag J L,et al.lncRNAdb v2.0:Expanding the reference database for functional long noncoding RNAs[J].Nucleic Acids Research,2015,43(D1):168- 173
                                    </a>
                                </li>
                                <li id="17">


                                    <a id="bibliography_8" title="Chakraborty S,Deb A,Maji R K,et al.LncRBase:An enriched resource for lncRNA information[J].PloS One,2014,9(9):e108010" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=LncRBase:an enriched resource for lncRNA information">
                                        <b>[8]</b>
                                        Chakraborty S,Deb A,Maji R K,et al.LncRBase:An enriched resource for lncRNA information[J].PloS One,2014,9(9):e108010
                                    </a>
                                </li>
                                <li id="19">


                                    <a id="bibliography_9" title="Jiang Qinghua,Ma Rui,Wang Jixuan,et al.LncRNA2-Function:A comprehensive resource for functional investigation of human lncRNAs based on RNA-seq data[J].BMC Genomics,2015,16(Suppl 3):S2" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=LncRNA2Function:a comprehensive resource for functional investigation of human lncRNAs based on RNA-seq data">
                                        <b>[9]</b>
                                        Jiang Qinghua,Ma Rui,Wang Jixuan,et al.LncRNA2-Function:A comprehensive resource for functional investigation of human lncRNAs based on RNA-seq data[J].BMC Genomics,2015,16(Suppl 3):S2
                                    </a>
                                </li>
                                <li id="21">


                                    <a id="bibliography_10" title="Jiang Qinghua,Wang Jixuan,Wu Xiaoliang,et al.LncRNA2Target:A database for differentially expressed genes after lncRNA knockdown or overexpression[J].Nucleic Acids Research,2014,43(D1):D193- D196" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=LncRNA Target:A database for differentially expressed genes after lncRNA knockdown or overexpression">
                                        <b>[10]</b>
                                        Jiang Qinghua,Wang Jixuan,Wu Xiaoliang,et al.LncRNA2Target:A database for differentially expressed genes after lncRNA knockdown or overexpression[J].Nucleic Acids Research,2014,43(D1):D193- D196
                                    </a>
                                </li>
                                <li id="23">


                                    <a id="bibliography_11" title="Guo Xingli,Gao Lin,Liao Qi,et al.Long non-coding RNAs function annotation:A global prediction method based on bi-colored networks[J].Nucleic Acids Research,2013,41(2):e35" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Long non-coding RNAs function annotation: a global prediction method based on bi-colored networks">
                                        <b>[11]</b>
                                        Guo Xingli,Gao Lin,Liao Qi,et al.Long non-coding RNAs function annotation:A global prediction method based on bi-colored networks[J].Nucleic Acids Research,2013,41(2):e35
                                    </a>
                                </li>
                                <li id="25">


                                    <a id="bibliography_12" title="Guo Xingli,Gao Lin,Wang Yu,et al.Advances in long noncoding RNAs:Identification,structure prediction and function annotation[J].Briefings in Functional Genomics,2015,15(1):38- 46" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Advances in long noncod ing RNAs:Identification,structure prediction and func tion annotation">
                                        <b>[12]</b>
                                        Guo Xingli,Gao Lin,Wang Yu,et al.Advances in long noncoding RNAs:Identification,structure prediction and function annotation[J].Briefings in Functional Genomics,2015,15(1):38- 46
                                    </a>
                                </li>
                                <li id="27">


                                    <a id="bibliography_13" >
                                        <b>[13]</b>
                                    Chen Geng,Wang Ziyun,Wang Dongqing,et al.LncRNADisease:A database for long-non-coding RNA-associated diseases[J].Nucleic Acids Research,2012,41(D1):D983- D986</a>
                                </li>
                                <li id="29">


                                    <a id="bibliography_14" title="Ning Shangwei,Zhang Jizhou,Wang Peng,et al.Lnc2Cancer:A manually curated database of experimentally supported lncRNAs associated with various human cancers[J].Nucleic Acids Research,2015,44(D1):D980- D985" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Lnc2Cancer:a manually curated database of experimentally supported lncRNAs associated with various human cancers">
                                        <b>[14]</b>
                                        Ning Shangwei,Zhang Jizhou,Wang Peng,et al.Lnc2Cancer:A manually curated database of experimentally supported lncRNAs associated with various human cancers[J].Nucleic Acids Research,2015,44(D1):D980- D985
                                    </a>
                                </li>
                                <li id="31">


                                    <a id="bibliography_15" title="Zhao Tingting,Xu Jinyuan,Liu Ling,et al.Identification of cancer-related lncRNAs through integrating genome,regulome and transcriptome features[J].Molecular BioSystems,2014,11(1):126- 136" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Identification of cancer-related lnc RNAs through integrating genome,regulome and transcriptome features">
                                        <b>[15]</b>
                                        Zhao Tingting,Xu Jinyuan,Liu Ling,et al.Identification of cancer-related lncRNAs through integrating genome,regulome and transcriptome features[J].Molecular BioSystems,2014,11(1):126- 136
                                    </a>
                                </li>
                                <li id="33">


                                    <a id="bibliography_16" title="Chen Xing,Yan Guiying.Novel human lncRNA-disease association inference based on lncRNA expression profiles[J].Bioinformatics,2013,29(20):2617- 2624" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJRS&amp;filename=SJRS13102500001233&amp;v=MjQ0NDlSZEdlcnFRVE1ud1plWnVIeWptVUwzSUoxd2NhQnM9TmlmWmZiSzdIOUhPcW85RlpPc09Ebjg2b0JNVDZUNFBRSC9pcg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[16]</b>
                                        Chen Xing,Yan Guiying.Novel human lncRNA-disease association inference based on lncRNA expression profiles[J].Bioinformatics,2013,29(20):2617- 2624
                                    </a>
                                </li>
                                <li id="35">


                                    <a id="bibliography_17" title="Sun Jie,Shi Hongbo,Wang Zhenzhen,et al.Inferring novel lncRNA-disease associations based on a random walk model of a lncRNA functional similarity network[J].Molecular BioSystems,2014,10(8):2074- 2081" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Inferring novel lncRNA–disease associations based on a random walk model of a lncRNA functional similarity network">
                                        <b>[17]</b>
                                        Sun Jie,Shi Hongbo,Wang Zhenzhen,et al.Inferring novel lncRNA-disease associations based on a random walk model of a lncRNA functional similarity network[J].Molecular BioSystems,2014,10(8):2074- 2081
                                    </a>
                                </li>
                                <li id="37">


                                    <a id="bibliography_18" title="Liu Yongjing,Zhang Rui,Qiu Fujun,et al.Construction of a lncRNA-PCG bipartite network and identification of cancer-related lncRNAs:A case study in prostate cancer[J].Molecular BioSystems,2015,11(2):384- 393" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Construction of a lncRNA-PCG bipartite network and identification of cancer-related lncRNAs:a case study in prostate cancer">
                                        <b>[18]</b>
                                        Liu Yongjing,Zhang Rui,Qiu Fujun,et al.Construction of a lncRNA-PCG bipartite network and identification of cancer-related lncRNAs:A case study in prostate cancer[J].Molecular BioSystems,2015,11(2):384- 393
                                    </a>
                                </li>
                                <li id="39">


                                    <a id="bibliography_19" title="Zhou Meng,Wang Xiaojun,Li Jiawei,et al.Prioritizing candidate disease-related long non-coding RNAs by walking on the heterogeneous lncRNA and disease network[J].Molecular BioSystems,2015,11(3):760- 769" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prioritizing candidate disease-related long non-coding RNAs by walking on the heterogeneous lncRNA and disease network">
                                        <b>[19]</b>
                                        Zhou Meng,Wang Xiaojun,Li Jiawei,et al.Prioritizing candidate disease-related long non-coding RNAs by walking on the heterogeneous lncRNA and disease network[J].Molecular BioSystems,2015,11(3):760- 769
                                    </a>
                                </li>
                                <li id="41">


                                    <a id="bibliography_20" title="Ganegoda G U,Li Min,Wang Weiping,et al.Heterogeneous network model to infer human disease-long intergenic non-coding RNA associations[J].IEEE Transactions on Nanobioscience,2015,14(2):175- 183" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Heterogeneous Network Model to Infer Human Disease-Long Intergenic Non Coding RNA Associations.">
                                        <b>[20]</b>
                                        Ganegoda G U,Li Min,Wang Weiping,et al.Heterogeneous network model to infer human disease-long intergenic non-coding RNA associations[J].IEEE Transactions on Nanobioscience,2015,14(2):175- 183
                                    </a>
                                </li>
                                <li id="43">


                                    <a id="bibliography_21" title="Chen Xing,You Zhuhong,Yan Guiying,et al.IRWRLDA:Improved random walk with restart for lncRNA-disease association prediction[J].Oncotarget,2016,7(36):57919" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=IRWRLDA:improved random walk with restart for lnc RNA-disease association prediction">
                                        <b>[21]</b>
                                        Chen Xing,You Zhuhong,Yan Guiying,et al.IRWRLDA:Improved random walk with restart for lncRNA-disease association prediction[J].Oncotarget,2016,7(36):57919
                                    </a>
                                </li>
                                <li id="45">


                                    <a id="bibliography_22" title="Chen Xing.KATZLDA:KATZ measure for the lncRNA-disease association prediction[J].Scientific Reports,2015,5(1):No.16840" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=KATZLDA:KATZ measure for the IncRNA-disease association prediction">
                                        <b>[22]</b>
                                        Chen Xing.KATZLDA:KATZ measure for the lncRNA-disease association prediction[J].Scientific Reports,2015,5(1):No.16840
                                    </a>
                                </li>
                                <li id="47">


                                    <a id="bibliography_23" title="Zeng Xiangxiang,Liao Yuanlu,Liu Yuansheng,et al.Prediction and validation of disease genes using HeteSim Scores[J].IEEE/ACM Transactions on Computational Biology and Bioinformatics,2017,14(3):687- 695" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCMB8EE5F078DFD48021E9B79C66BCA8C3D&amp;v=MTg4MThOdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TnBodzdxM3dxRT1OaWZJWThHd2E2VEoyWTlDYko5NWVIZ3h6eFFTbnpZUFQzYVJxaFJIQ3NPY05ybnJDTw==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[23]</b>
                                        Zeng Xiangxiang,Liao Yuanlu,Liu Yuansheng,et al.Prediction and validation of disease genes using HeteSim Scores[J].IEEE/ACM Transactions on Computational Biology and Bioinformatics,2017,14(3):687- 695
                                    </a>
                                </li>
                                <li id="49">


                                    <a id="bibliography_24" title="Shi Chuan,Kong Xiangnan,Huang Yue,et al.HeteSim:A general framework for relevance measure in heterogeneous networks[J].IEEE Transactions on Knowledge and Data Engineering,2014,26(10):2479- 2492" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Hete Sim:A General Framework for Relevance Measure in Heterogeneous Networks">
                                        <b>[24]</b>
                                        Shi Chuan,Kong Xiangnan,Huang Yue,et al.HeteSim:A general framework for relevance measure in heterogeneous networks[J].IEEE Transactions on Knowledge and Data Engineering,2014,26(10):2479- 2492
                                    </a>
                                </li>
                                <li id="51">


                                    <a id="bibliography_25" title="Yang Xiaofei,Gao Lin,Guo Xingli,et al.A network based method for analysis of lncRNA-disease associations and prediction of lncRNAs implicated in diseases[J].PloS One,2014,9(1):e87797" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A network based method for analysis of lncRNA-disease associations and prediction of lncRNAs implicated in diseases">
                                        <b>[25]</b>
                                        Yang Xiaofei,Gao Lin,Guo Xingli,et al.A network based method for analysis of lncRNA-disease associations and prediction of lncRNAs implicated in diseases[J].PloS One,2014,9(1):e87797
                                    </a>
                                </li>
                                <li id="53">


                                    <a id="bibliography_26" title="Hamosh A,Scott A F,Amberger J S,et al.Online mendelian inheritance in man (OMIM),a knowledgebase of human genes and genetic disorders[J].Nucleic Acids Research,2005,33(Suppl 1):D514- D517" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic disorders">
                                        <b>[26]</b>
                                        Hamosh A,Scott A F,Amberger J S,et al.Online mendelian inheritance in man (OMIM),a knowledgebase of human genes and genetic disorders[J].Nucleic Acids Research,2005,33(Suppl 1):D514- D517
                                    </a>
                                </li>
                                <li id="55">


                                    <a id="bibliography_27" title="Zhou Yanqing,Xu Xiaying,L&#252; Huabing,et al.The long noncoding RNA MALAT-1 is highly expressed in ovarian cancer and induces cell growth and migration[J].PLoS One,2016,11(5):e0155250" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The Long Noncoding RNA MALAT-1 Is Highly Expressed in Ovarian Cancer and Induces Cell Growth and Migration">
                                        <b>[27]</b>
                                        Zhou Yanqing,Xu Xiaying,L&#252; Huabing,et al.The long noncoding RNA MALAT-1 is highly expressed in ovarian cancer and induces cell growth and migration[J].PLoS One,2016,11(5):e0155250
                                    </a>
                                </li>
                                <li id="57">


                                    <a id="bibliography_28" title="Yang Yijun,Jiang Yi,Wan Yicong,et al.UCA1 functions as a competing endogenous RNA to suppress epithelial ovarian cancer metastasis[J].Tumor Biology,2016,37(8):10633- 10641" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=UCA1 functions as a competing endogenous RNA to suppress epithelial ovarian cancer metastasis">
                                        <b>[28]</b>
                                        Yang Yijun,Jiang Yi,Wan Yicong,et al.UCA1 functions as a competing endogenous RNA to suppress epithelial ovarian cancer metastasis[J].Tumor Biology,2016,37(8):10633- 10641
                                    </a>
                                </li>
                                <li id="59">


                                    <a id="bibliography_29" title="Xiu Yinling,Sun Kaixuan,Chen Xi,et al.Upregulation of the lncRNA Meg3 induces autophagy to inhibit tumorigenesis and progression of epithelial ovarian carcinoma by regulating activity of ATG3[J].Oncotarget,2017,8(19):31714- 31725" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Upregulation of the lncRNA Meg3 induces autophagy to inhibit tumorigenesis and progression of epithelial ovarian carcinoma by regulating activity of ATG3">
                                        <b>[29]</b>
                                        Xiu Yinling,Sun Kaixuan,Chen Xi,et al.Upregulation of the lncRNA Meg3 induces autophagy to inhibit tumorigenesis and progression of epithelial ovarian carcinoma by regulating activity of ATG3[J].Oncotarget,2017,8(19):31714- 31725
                                    </a>
                                </li>
                                <li id="61">


                                    <a id="bibliography_30" title="Zhang Zhongbao,Cheng Jiajing,Wu Yi,et al.LncRNA HOTAIR controls the expression of Rab22a by sponging miR-373 in ovarian cancer[J].Molecular Medicine Reports,2016,14(3):2465- 2472" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJSD&amp;filename=SJSDD01273F3EA10AD5C1AB20211C3512816&amp;v=MTI3MDhyUGxHRVpvT0RBMU55bVVTbTAxL1NIM2pyV0UyZkxPV1RidVpDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOcGh3N3Ezd3FFPU5pZllhc2U0SDlQTA==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[30]</b>
                                        Zhang Zhongbao,Cheng Jiajing,Wu Yi,et al.LncRNA HOTAIR controls the expression of Rab22a by sponging miR-373 in ovarian cancer[J].Molecular Medicine Reports,2016,14(3):2465- 2472
                                    </a>
                                </li>
                                <li id="63">


                                    <a id="bibliography_31" title="Okugawa Y,Toiyama Y,Hur K,et al.Metastasis-associated long non-coding RNA drives gastric cancer development and promotes peritoneal metastasis[J].Carcinogenesis,2014,35(12):2731- 2739" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Metastasis-associated long non-coding RNA drives gastric cancer development and promotes peritoneal metastasis">
                                        <b>[31]</b>
                                        Okugawa Y,Toiyama Y,Hur K,et al.Metastasis-associated long non-coding RNA drives gastric cancer development and promotes peritoneal metastasis[J].Carcinogenesis,2014,35(12):2731- 2739
                                    </a>
                                </li>
                                <li id="65">


                                    <a id="bibliography_32" title="Chen Di,Liu Lili,Wang Kai,et al.The role of MALAT-1 in the invasion and metastasis of gastric cancer[J].Scandinavian Journal of Gastroenterology,2017,52(6/7):790- 796" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=STJD&amp;filename=STJDFD2FE98DE637125905C7C06BD76C2462&amp;v=MDUxODM2TzMvazNtWXlmOEdXUWJ5ZENPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU5waHc3cTN3cUU9TmpuQmFzWE1IS2U1cG9jeEVlME1DMzA3eWg4VDcweA==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[32]</b>
                                        Chen Di,Liu Lili,Wang Kai,et al.The role of MALAT-1 in the invasion and metastasis of gastric cancer[J].Scandinavian Journal of Gastroenterology,2017,52(6/7):790- 796
                                    </a>
                                </li>
                                <li id="67">


                                    <a id="bibliography_33" title="Xu Gang,Meng Lei,Yuan Dawei,et al.MEG3/miR 21 axis affects cell mobility by suppressing epithelial mesenchymal transition in gastric cancer[J].Oncology Reports,2018,40(1):39- 48" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJSD&amp;filename=SJSD5A1E05D608A859BDA16A4DE533310D97&amp;v=MTMwODFPTitCSGt3dldKaTZ6a01UQXVYcVJFMmVyT1VNYk9ZQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TnBodzdxM3dxRT1OaWZZYXJiSkg2VE1xdnREWg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[33]</b>
                                        Xu Gang,Meng Lei,Yuan Dawei,et al.MEG3/miR 21 axis affects cell mobility by suppressing epithelial mesenchymal transition in gastric cancer[J].Oncology Reports,2018,40(1):39- 48
                                    </a>
                                </li>
                                <li id="69">


                                    <a id="bibliography_34" title="Li Min,Meng Xiangmao.Progress in the construction,analysis and application of dynamic protein networks[J].Journal of Computer Research and Development,2017:54(6):1281- 1299 (in Chinese)(李敏,孟祥茂.动态蛋白质网络的构建,分析及应用研究进展[J].计算机研究与发展,2017,54(6):1281- 1299)" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201706012&amp;v=MzE2MTN5L25XNzNCTHl2U2RMRzRIOWJNcVk5RVpvUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSc0Y=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[34]</b>
                                        Li Min,Meng Xiangmao.Progress in the construction,analysis and application of dynamic protein networks[J].Journal of Computer Research and Development,2017:54(6):1281- 1299 (in Chinese)(李敏,孟祥茂.动态蛋白质网络的构建,分析及应用研究进展[J].计算机研究与发展,2017,54(6):1281- 1299)
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JFYZ" target="_blank">计算机研究与发展</a>
                2019,56(09),1889-1896 DOI:10.7544/issn1000-1239.2019.20180834            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>基于HeteSim的疾病关联长非编码RNA预测</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E9%A9%AC%E6%AF%85&amp;code=42840455&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">马毅</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E9%83%AD%E6%9D%8F%E8%8E%89&amp;code=09039105&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">郭杏莉</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%AD%99%E5%AE%87%E5%BD%A4&amp;code=42840456&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">孙宇彤</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E8%8B%91%E5%80%A9%E5%80%A9&amp;code=42840457&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">苑倩倩</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E4%BB%BB%E9%98%B3&amp;code=42840458&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">任阳</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%AE%B5%E7%84%B6&amp;code=42840459&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">段然</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E9%AB%98%E7%90%B3&amp;code=09086853&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">高琳</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E8%A5%BF%E5%AE%89%E7%94%B5%E5%AD%90%E7%A7%91%E6%8A%80%E5%A4%A7%E5%AD%A6%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%A7%91%E5%AD%A6%E4%B8%8E%E6%8A%80%E6%9C%AF%E5%AD%A6%E9%99%A2&amp;code=0008505&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">西安电子科技大学计算机科学与技术学院</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>越来越多的研究表明,长非编码RNA(long non-coding RNA, lncRNA)在许多生物过程中具有重要的功能,而这些长非编码RNA的变异或功能失调会导致一些复杂疾病的发生.通过生物信息学方法预测潜在的长非编码RNA-疾病关联关系,对于致病机理的探索以及疾病诊断、治疗、预后和预防都具有重要的意义.基于疾病基因关联关系的异质信息网络,研究者使用了一种相关性计算法方法——HeteSim来计算疾病基因之间的相关性,进而预测致病基因.使用的方法基于路径约束,具有可扩展性,算法效率高,留一交叉验证实验表明该方法的预测结果优于其他方法.将其应用在卵巢癌和胃癌的预测分析中,相关文献表明,所提方法的预测结果已被生物实验等验证,再次表明该方法的有效性.</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%87%B4%E7%97%85%E5%9F%BA%E5%9B%A0%E9%A2%84%E6%B5%8B&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">致病基因预测;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%9B%B8%E5%85%B3%E6%80%A7%E8%AE%A1%E7%AE%97&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">相关性计算;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%BC%82%E8%B4%A8%E4%BF%A1%E6%81%AF%E7%BD%91%E7%BB%9C&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">异质信息网络;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=HeteSim%E6%96%B9%E6%B3%95&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">HeteSim方法;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%85%83%E8%B7%AF%E5%BE%84&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">元路径;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    *郭杏莉,xlguo@mail.xidian.edu.cn;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-12-18</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金面上项目(61672407,61672406);国家自然科学基金重点项目(61432010,61532014);</span>
                    </p>
            </div>
                    <h1><b>Prediction of Disease Associated Long Non-Coding RNA Based on HeteSim</b></h1>
                    <h2>
                    <span>Ma Yi</span>
                    <span>Guo Xingli</span>
                    <span>Sun Yutong</span>
                    <span>Yuan Qianqian</span>
                    <span>Ren Yang</span>
                    <span>Duan Ran</span>
                    <span>Gao Lin</span>
            </h2>
                    <h2>
                    <span>School of Computer Science and Technology, Xidian University</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>A growing number of evidences indicate that long non-coding RNAs(lncRNAs) play important roles in many biological processes, and mutations or dysfunction in these long non-coding RNAs can cause serious diseases in human bodies, such as various cancers. Biological methods have been exploited to predict potential associations between diseases and long non-coding RNAs, which are of great significance for the exploration of pathogenesis, diagnosis, treatment, prognosis and prevention of complex diseases. Heterogeneous information network is constructed based on the known disease-gene associations. The association strength between lncRNAs and diseases can be measured by an association score in the heterogeneous network. A simple method called HeteSim is applied to calculate the association scores between lncRNAs and diseases. The method used in this paper is based on all paths existing between a given disease and a given lncRNA. The experiments show that our method can achieve superior performance than state-of-art methods. Our predictions for ovarian cancer and gastric cancer have been verified by biological experiments, indicating the effectiveness of this method. The case studies indicate that our method can give informative clues for further investigation. In conclusion, the only paths based on known disease-gene associations are exploited, and it is can be expected that other disease associated information can also be integrated into our method, and better performance can be available.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=disease-gene%20prediction&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">disease-gene prediction;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=correlation%20calculation&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">correlation calculation;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=heterogeneous%20information%20networks&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">heterogeneous information networks;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=HeteSim&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">HeteSim;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=meta-path&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">meta-path;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                            <b>Author：</b>
                                                        <span>
                                    Ma Yi,born in 1996.Master candidate. His main research interests include bioinformatics,data mining.&lt;image id="182" type="formula" href="images/JFYZ201909010_18200.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                                <span>
                                    Guo Xingli,born in 1979.PhD,associate professor.Member of CCF.Her main research interests include data mining algorithm,complex network model and its application in bioinformatics research. (xlguo@mail.xidian.edu.cn)&lt;image id="184" type="formula" href="images/JFYZ201909010_18400.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                                <span>
                                    Sun Yutong, born in 1995. Master candidate.Her main research interests include data mining,bioinformatics,long non-coding RNA analysis.&lt;image id="186" type="formula" href="images/JFYZ201909010_18600.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                                <span>
                                    Yuan Qianqian,born in 1996.Master candidate.Her main research interests include data mining,bioinformatics,long non-coding RNA analysis.&lt;image id="188" type="formula" href="images/JFYZ201909010_18800.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                                <span>
                                    Ren Yang,born in 1993.Master candidate. His main research interests include data mining,bioinformatics.&lt;image id="190" type="formula" href="images/JFYZ201909010_19000.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                                <span>
                                    Duan Ran,born in 1990.PhD candidate. His main research interests include multi-omics data integration,cancer integrative analysis,and machine learning.&lt;image id="192" type="formula" href="images/JFYZ201909010_19200.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                                <span>
                                    Gao Lin,born in 1964.PhD,professor and PhD supervisor.Her main research interests include bioinformatics, data mining,graph theory and combinatorial optimization algorithm and applications. (lgao@mail.xidian.edu.cn)&lt;image id="194" type="formula" href="images/JFYZ201909010_19400.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                    </p>
                                    <p><b>Received：</b> 2018-12-18</p>
                                    <p>
                            <b>Fund：</b>
                                                        <span>supported by the General Program of the National Natural Science Foundation of China(61672407,61672406);the Key Program of the National Natural Science Foundation of China(61432010,61532014);</span>
                    </p>
            </div>


        <!--brief start-->
                        <div class="p1">
                    <p id="71">全基因组研究表明,2/3的基因组能够被转录为RNA,但其中只有一小部分可以翻译为蛋白质<citation id="144" type="reference"><link href="3" rel="bibliography" /><link href="5" rel="bibliography" /><link href="7" rel="bibliography" /><link href="9" rel="bibliography" /><sup>[<a class="sup">1</a>,<a class="sup">2</a>,<a class="sup">3</a>,<a class="sup">4</a>]</sup></citation>,非编码RNA大量存在于生物体内.通常,为了区别于其他短非编码RNA,长非编码RNA(long non-coding RNA, lncRNA)简单地定义为长度大于200nt且不编码蛋白质的一类RNA分子.lncRNA在许多重要的生物过程中扮演关键角色,例如染色质修饰、转录和转录后调节<citation id="143" type="reference"><link href="11" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>.由于lncRNA在生命过程中发挥了重要作用,因此很大一部分人类疾病与lncRNA的变异以及功能失调息息相关.</p>
                </div>
                <div class="p1">
                    <p id="72">随着已确定的lncRNA的数量持续增长,许多相关的数据库、计算方法被提出来,其中包括通用的数据库GENCODE<citation id="145" type="reference"><link href="13" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>,针对lncRNA的专用数据库lncRNAdb<citation id="146" type="reference"><link href="15" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>,LncRbase<citation id="147" type="reference"><link href="17" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>,LncRNA2Function<citation id="148" type="reference"><link href="19" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>,LncRNA2Target<citation id="149" type="reference"><link href="21" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>,同时包括基于网络的大规模lncRNA功能预测方法lncGFP<citation id="150" type="reference"><link href="23" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>,以及通用的计算模型和框架<citation id="151" type="reference"><link href="25" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>.关于lncRNA在普通疾病和癌症中的作用,分别有LncRNADisease<citation id="152" type="reference"><link href="27" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>和Lnc2Cancer<citation id="153" type="reference"><link href="29" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>数据库.即使有一定数量的lncRNA-疾病关联关系已经得到实验验证,不可忽略的是,绝大多数lncRNA-疾病关联关系仍然是未知的.因此,分析lncRNA与疾病关联关系并预测潜在的关联关系具有重要的研究价值和社会意义.这些研究不仅可以帮助我们加深对复杂疾病在分子层面的致病机理的理解,而且可以利用lncRNA作为疾病诊断、预测的生物靶标以及治疗和预防的药物靶标.</p>
                </div>
                <div class="p1">
                    <p id="73">预测潜在的疾病与lncRNA关联关系的计算方法可分为2大类:基于机器学习和基于网络的方法.基于机器学习的方法通常使用疾病与lncRNA关联关系来训练学习模型,然后用学习得到的模型来预测新的关联关系.这类方法整合了各种生物信息来注释lncRNA.例如,Zhao等人<citation id="154" type="reference"><link href="31" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation>使用朴素贝叶斯模型来整合基因组、调节子和转录组特征,进而识别与癌症相关的潜在lncRNA.这个方法需要阴性的训练样本(即与疾病无关的lncRNA)来训练模型,考虑到并没有这种实验验证的阴性样本,在这项研究中,所有未知的lncRNA-疾病关联关系被认为是阴性样本用于训练.最近,一个半监督模型——正则化最小二乘(RLS)<citation id="155" type="reference"><link href="33" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation>克服了这一限制,该模型不需要阴性的训练样本.</p>
                </div>
                <div class="p1">
                    <p id="74">相对于比较少的基于机器学习方法的研究,许多基于网络的方法被提出来预测与疾病相关的潜在lncRNA.基于网络的方法通常根据lncRNA与疾病的关联得分大小对候选的lncRNA进行排序,进而预测致病基因.最常用的算法是标签传播算法,比如随机漫步(RWR)<citation id="161" type="reference"><link href="35" rel="bibliography" /><link href="37" rel="bibliography" /><link href="39" rel="bibliography" /><link href="41" rel="bibliography" /><link href="43" rel="bibliography" /><sup>[<a class="sup">17</a>,<a class="sup">18</a>,<a class="sup">19</a>,<a class="sup">20</a>,<a class="sup">21</a>]</sup></citation>和KATZ<citation id="156" type="reference"><link href="45" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>.这些研究的主要区别在于传播算法所应用的底层网络不同.例如:Sun等人<citation id="157" type="reference"><link href="35" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>将RWR应用于lncRNA功能相似网络(lncRNA FSN);Liu等人<citation id="158" type="reference"><link href="37" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>基于lncRNA和蛋白质编码基因表达谱构建了蛋白质编码基因-lncRNA二部网络,然后利用RWR算法来预测癌症相关的lncRNA;与此同时,Zhou等人<citation id="159" type="reference"><link href="39" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>和Ganegoda等人<citation id="160" type="reference"><link href="41" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>结合lncRNA相似网络建立了lncRNA-疾病异质信息网络,然后在该网络上应用RWR算法预测潜在疾病lncRNA关联关系.这些基于网络的方法是基于一种观察结果提出的,即在功能上类似的lncRNA通常与相同或相似的疾病联系在一起,即疾病模块原理.以上方法都是通过构建网络提出基于网络的计算模型,有的方法结合基因表达谱数据等构建网络,所构建网络结合了多种信息的逻辑关联网络,构建方法相对复杂.</p>
                </div>
                <div class="p1">
                    <p id="75">本文使用了一种异质信息网络中节点相关性计算方法——HeteSim,该方法用来预测基因和疾病的关联关系,得到了很好的实验验证<citation id="162" type="reference"><link href="47" rel="bibliography" /><sup>[<a class="sup">23</a>]</sup></citation>.因此,我们将这种方法应用到lncRNA-疾病异质信息网络中,通过挖掘网络中疾病与lncRNA之间的关联关系,计算疾病与lncRNA关联得分,预测潜在疾病关联lncRNA,预测结果优于其他方法.</p>
                </div>
                <h3 id="76" name="76" class="anchor-tag"><b>1 算  法</b></h3>
                <h4 class="anchor-tag" id="77" name="77"><b>1.1 异质信息网络构建</b></h4>
                <div class="p1">
                    <p id="78">预测lncRNA与疾病之间的关联关系可以理解为lncRNA-疾病异质信息网络上的一个相关性搜索任务.异质信息网络是一种特殊的信息网络,下面是信息网络的定义,在此基础上可以定义得到同质信息网络和异质信息网络.</p>
                </div>
                <div class="p1">
                    <p id="79"><b>定义1</b>. 信息网络.给定一个模式<i>S</i> =(<i>A</i>, <i>R</i>),它由对象类型集合<i>A</i>和关系集合<i>R</i>构成.信息网络被抽象定义为一个有向图<i>G</i> = (<i>V</i>,<i>E</i>),其中,<i>V</i>是所有实体节点的集合,<i>E</i>是所有关系边的集合.并且存在一个节点类型的映射函数<i>φ</i>:<i>V</i>→<i>A</i>和一个边类型的映射函数<i>θ</i>: <i>E</i>→<i>R</i>,对于每个对象<i>v</i>∈<i>V</i>属于一种特殊的对象类型<i>φ</i>(<i>v</i>)∈<i>A</i>,每个链接<i>e</i>∈<i>E</i>属于一种特殊的关系类型<i>θ</i>(<i>e</i>)∈<i>R</i>,那么这种网络类型就是信息网络.当对象类型的种类|<i>A</i>|&gt;1或者关系类型的种类|<i>R</i>|&gt;1时,这种信息网络是异质信息网络.例如图1(a)就是由电影数据构建成电影异质信息网络.</p>
                </div>
                <div class="area_img" id="80">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909010_080.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 异质信息网络实例和元路径[24]" src="Detail/GetImg?filename=images/JFYZ201909010_080.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 异质信息网络实例和元路径<citation id="163" type="reference"><link href="49" rel="bibliography" /><sup>[<a class="sup">24</a>]</sup></citation>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909010_080.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 1 Heterogeneous information network instance and meta-path<citation id="164" type="reference"><link href="49" rel="bibliography" /><sup>[24]</sup></citation></p>

                </div>
                <div class="p1">
                    <p id="81">在信息网络中,我们将对象的类型和关系的类型明确区分开,不同类型对象之间存在的关系可以用网络模式清晰地描述.我们把类型<i>A</i>和类型<i>B</i>之间的关系<i>R</i>表示为<mathml id="134"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>A</mi><mover><mo stretchy="true" symmetric="false">→</mo><mi>R</mi></mover><mi>B</mi></mrow></math></mathml>,其中<i>A</i>和<i>B</i>分别是关系<i>R</i>的源类型和目标类型,逆关系<i>R</i><sup>-1</sup>可以表示为<mathml id="135"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>B</mi><mover><mo stretchy="true" symmetric="false">→</mo><mrow><mi>R</mi><msup><mrow></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup></mrow></mover><mi>A</mi></mrow></math></mathml>.一般情况下关系<i>R</i>不等于关系<i>R</i><sup>-1</sup>,除非<i>R</i>是对称的并且关系两端的对象类型是相同的.此外,元路径是基于网络模式定义的,表示对象类型之间的关系,如图1(b)就表示电影异质信息网络里的一种元路径<i>AMA</i>,表示演员之间的合作关系.</p>
                </div>
                <div class="p1">
                    <p id="82">基于已知的lncRNA与疾病关联关系,构建lncRNA-疾病异质信息网络,如图2(a)所示.网络中包含2种类型节点,分别为lncRNA和疾病,包含1种类型的边,即lncRNA-疾病关联关系.为了集成更多的疾病相关的基因信息,类似地,我们集成了OMIM(online mendelian inheritance in man)数据库中已知的编码基因与疾病的关联关系,将上面所构建的异质信息网络进行了扩展.扩展后的网络中包含2种类型节点,分别为基因和疾病,其中基因包括lncRNA和从OMIM中集成的编码基因.相应的边扩展为基因-疾病关联关系.lncRNA与疾病的关联预测在基因-疾病关联异质信息网络上进行.</p>
                </div>
                <h4 class="anchor-tag" id="83" name="83"><b>1.2 元路径选择</b></h4>
                <div class="p1">
                    <p id="84">由于HeteSim是一种路径约束的相关性计算方法,所以选择相关路径是非常重要的.构建了异质信息网络之后,我们的目的是要研究lncRNA和疾病的相关关系,即通过现有的异质信息网络预测出lncRNA是否和其他疾病相关联,因此我们选择lncRNA-疾病-lncRNA-疾病(<i>LDLD</i>)作为元路径,如图2所示.在此路径下使用HeteSim算法计算lncRNA和疾病之间的相关性,就能根据已有的关系预测出潜在的lncRNA-疾病关联关系.</p>
                </div>
                <div class="area_img" id="85">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909010_085.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 LncRNA-疾病异质信息网络和元路径LDLD" src="Detail/GetImg?filename=images/JFYZ201909010_085.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 LncRNA-疾病异质信息网络和元路径<i>LDLD</i>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909010_085.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 2 LncRNA-Disease heterogeneous information 
 network and meta-path <i>LDLD</i></p>

                </div>
                <h4 class="anchor-tag" id="86" name="86"><b>1.3 模型描述</b></h4>
                <div class="p1">
                    <p id="87">石川等人<citation id="165" type="reference"><link href="49" rel="bibliography" /><sup>[<a class="sup">24</a>]</sup></citation>提出了HeteSim算法来计算异质信息网络中任意节点对的相关性,该方法具有对称特性而且可以计算相同或不同类型对象之间的相关性,从而适用于很多的应用.HeteSim是一种基于双向随机游走(pair-wise random walk)的相关性计算方法,它将元路径<i>P</i>分割成2条相等长度的元路径<i>P</i><sub>L</sub>和<i>P</i><sub>R</sub>,之后将对象<i>s</i>和<i>t</i>分别沿着元路径<i>P</i><sub>L</sub>和<i>P</i><sub>R</sub>进行随机游走,最后将2个对象走到相同中间节点的概率作为<i>s</i>和<i>t</i>的相关性.</p>
                </div>
                <div class="p1">
                    <p id="88">给定一个相关路径<i>P</i>=<i>A</i><sub>1</sub><i>A</i><sub>2</sub>…<i>A</i><sub><i>l</i></sub><sub>+1</sub>,该路径可以被分解为2条相等长度的路径<i>P</i><sub>L</sub>和<i>P</i><sub>R</sub>.<i>P</i>=<i>P</i><sub>L</sub><i>P</i><sub>R</sub>,<i>P</i><sub>L</sub>=<i>A</i><sub>1</sub><i>A</i><sub>2</sub>…<i>A</i><sub><i>mid</i></sub><sub>-1</sub><i>M</i>,<i>P</i><sub>R</sub>=<i>MA</i><sub><i>mid</i></sub><sub>+1</sub>…<i>A</i><sub><i>l</i></sub><sub>+1</sub>.<i>M</i>为路径中的中间类型对象,当路径长度为偶数时<mathml id="136"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>m</mi><mi>i</mi><mi>d</mi><mo>=</mo><mfrac><mi>l</mi><mn>2</mn></mfrac><mo>+</mo><mn>1</mn></mrow></math></mathml>,当路径长度为奇数时<mathml id="137"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>m</mi><mi>i</mi><mi>d</mi><mo>=</mo><mfrac><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow><mn>2</mn></mfrac><mo>+</mo><mn>1</mn><mo>.</mo></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="89">对于我们选择的元路径lncRNA-疾病-lncRNA-疾病(<i>LDLD</i>),由于路径长度是奇数,元路径两端的2个节点始终都不会在1个点相遇,因此我们需要插入中间类型<i>M</i>从而使路径可以等分成路径<i>P</i><sub>L</sub>=<i>LDM</i>和<i>P</i><sub>R</sub>=<i>MLD</i>,如图3所示:</p>
                </div>
                <div class="area_img" id="90">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909010_090.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 插入中间类型M前后" src="Detail/GetImg?filename=images/JFYZ201909010_090.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 插入中间类型<i>M</i>前后  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909010_090.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 3 Before and after insertion of the intermediate type <i>M</i></p>

                </div>
                <div class="p1">
                    <p id="91">下面介绍如何利用矩阵乘法计算lncRNA和疾病之间的关联得分.首先,我们定义2类矩阵:转移概率矩阵和可达概率矩阵.</p>
                </div>
                <div class="p1">
                    <p id="92"><b>定义2</b>. 转移概率矩阵.定义有向元路径<mathml id="138"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>A</mi><mover><mo stretchy="true" symmetric="false">→</mo><mi>R</mi></mover><mi>B</mi><mo>,</mo></mrow></math></mathml>对象<i>A</i>和对象<i>B</i>之间的连接关系为<i>R</i>(<i>A</i>和<i>B</i>表示同一类型对象构成的集合),<i>A</i>和<i>B</i>之间的关系可以用0/1邻接矩阵<i><b>W</b></i><sub><i>AB</i></sub>表示,元素1表示2节点连通,元素0表示2节点不连通.将0/1邻接矩阵<i><b>W</b></i><sub><i>AB</i></sub>分别按照行向量和列向量进行标准化操作,得到矩阵<i><b>X</b></i><sub><i>AB</i></sub>和<i><b>Y</b></i><sub><i>AB</i></sub>.<i><b>X</b></i><sub><i>AB</i></sub>和<i><b>Y</b></i><sub><i>AB</i></sub>就是转移概率矩阵,分别表示<mathml id="139"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mi>A</mi><mover><mo stretchy="true" symmetric="false">→</mo><mi>R</mi></mover><mi>B</mi><mtext>和</mtext></mrow><mi>B</mi><mover><mo stretchy="true" symmetric="false">→</mo><mrow><mi>R</mi><msup><mrow></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup></mrow></mover><mi>A</mi></mrow></math></mathml>这2种有向关系.根据矩阵的性质,可以得到:</p>
                </div>
                <div class="p1">
                    <p id="93"><i><b>X</b></i><sub><i>AB</i></sub>=<i><b>Y</b></i><sup>T</sup><sub><i>BA</i></sub>,<i><b>Y</b></i><sub><i>AB</i></sub>=<i><b>X</b></i><sup>T</sup><sub><i>BA</i></sub>.</p>
                </div>
                <div class="p1">
                    <p id="94"><b>定义3</b>. 可达概率矩阵.转移概率矩阵是可达概率矩阵的特例.转移概率矩阵用来描述长度为1的元路径节点间的关系,而可达概率矩阵则用来衡量在元路径长度大于1(复合关系<i>R</i>=<i>R</i><sub>1</sub>。<i>R</i><sub>2</sub>…。<i>R</i><sub><i>l</i></sub>)的情况下节点间的关系.基于复合关系<i>R</i>给定元路径<mathml id="140"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ρ</mi><mo>=</mo><mrow><mi>A</mi><msub><mrow></mrow><mn>1</mn></msub><mover><mo stretchy="true" symmetric="false">→</mo><mrow><mi>R</mi><msub><mrow></mrow><mn>1</mn></msub></mrow></mover></mrow><mi>A</mi><msub><mrow></mrow><mn>2</mn></msub><mover><mo stretchy="true" symmetric="false">→</mo><mrow><mi>R</mi><msub><mrow></mrow><mn>2</mn></msub></mrow></mover><mo>⋯</mo><mover><mo stretchy="true" symmetric="false">→</mo><mrow><mi>R</mi><msub><mrow></mrow><mi>l</mi></msub></mrow></mover><mi>A</mi><msub><mrow></mrow><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow></math></mathml>,元路径<i>P</i>上的对象<i>A</i><sub>1</sub>与<i>A</i><sub><i>l</i></sub><sub>+1</sub>之间的可达概率矩阵定义为<i><b>Q</b></i><sub><i>P</i></sub>=<i><b>X</b></i><sub><i>A</i></sub><sub>1</sub><sub><i>A</i></sub><sub>2</sub><i><b>X</b></i><sub><i>A</i></sub><sub>2</sub><sub><i>A</i></sub><sub>3</sub>…<i><b>X</b></i><sub><i>AlAl</i></sub><sub>+1</sub>,它表示<i>A</i><sub>1</sub>沿着路径<i>P</i>随机游走到<i>A</i><sub><i>l</i></sub><sub>+1</sub>的概率.</p>
                </div>
                <div class="p1">
                    <p id="95">根据HeteSim的定义,类型<i>L</i>中的节点基于元路径<i>P</i>=<i>LDLD</i>到类型<i>D</i>中的节点之间的相似度为类型<i>L</i>的节点和类型<i>D</i>中的节点随机游走恰好在元路径中间类型<i>M</i>相遇的概率,计算公式为</p>
                </div>
                <div class="p1">
                    <p id="96"><i>HeteSim</i>(<i>L</i>,<i>D</i>|<i>P</i>)=<i>HeteSim</i>(<i>L</i>,<i>D</i>|<i>P</i><sub>L</sub><i>P</i><sub>R</sub>)=</p>
                </div>
                <div class="p1">
                    <p id="97"><i><b>X</b></i><sub><i>LD</i></sub><i><b>X</b></i><sub><i>DM</i></sub><i><b>Y</b></i><sub><i>ML</i></sub><i><b>Y</b></i><sub><i>LD</i></sub>=<i><b>X</b></i><sub><i>LD</i></sub><i><b>X</b></i><sub><i>DM</i></sub><i><b>X</b></i><sup>T</sup><sub><i>LM</i></sub><i><b>X</b></i><sup>T</sup><sub><i>DL</i></sub>=</p>
                </div>
                <div class="p1">
                    <p id="98"><i><b>X</b></i><sub><i>LD</i></sub><i><b>X</b></i><sub><i>DM</i></sub>(<i><b>X</b></i><sub><i>DL</i></sub><i><b>X</b></i><sub><i>LM</i></sub>)<sup>T</sup>=<i><b>Q</b></i><sub><i>P</i></sub><sub>L</sub><i><b>Q</b></i><sup>T</sup><sub><i>PR</i></sub><sub><sup>-1</sup></sub>,(1)</p>
                </div>
                <div class="p1">
                    <p id="99">式(1)表明<i>L</i>和<i>D</i>之间基于路径<i>P</i>的相关性是2个概率分布的内积.</p>
                </div>
                <div class="p1">
                    <p id="100">对于lncRNA和疾病类型中具体的对象<i>l</i>,<i>d</i>,基于路径<i>P</i>的关联得分计算为</p>
                </div>
                <div class="p1">
                    <p id="101"><i>HeteSim</i>(<i>l</i>,<i>d</i>|<i>P</i>)=<i><b>Q</b></i><sub><i>P</i></sub><sub>L</sub>(<i>l</i>,:)<i><b>Q</b></i><sup>T</sup><sub><i>PR</i></sub><sub><sup>-1</sup></sub>(<i>d</i>,:),(2)</p>
                </div>
                <div class="p1">
                    <p id="102">其中<i><b>Q</b></i><sub><i>P</i></sub>(<i>l</i>,:)为矩阵<i><b>Q</b></i><sub><i>P</i></sub>中对象<i>l</i>所对应的行向量.</p>
                </div>
                <div class="p1">
                    <p id="103">为了使得HeteSim得分取值位于区间[0,1],还需要对计算出的关联得分进行标准化处理:</p>
                </div>
                <div class="p1">
                    <p id="104" class="code-formula">
                        <mathml id="104"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>Η</mi><mi>e</mi><mi>t</mi><mi>e</mi><mi>S</mi><mi>i</mi><mi>m</mi><mo stretchy="false">(</mo><mi>l</mi><mo>,</mo><mi>d</mi><mo stretchy="false">|</mo><mi>Ρ</mi><mo stretchy="false">)</mo><mo>=</mo></mtd></mtr><mtr><mtd><mfrac><mrow><mi mathvariant="bold-italic">Q</mi><msub><mrow></mrow><mrow><mi>Ρ</mi><msub><mrow></mrow><mtext>L</mtext></msub></mrow></msub><mo stretchy="false">(</mo><mi>l</mi><mo>,</mo><mo>:</mo><mo stretchy="false">)</mo><mi mathvariant="bold-italic">Q</mi><msubsup><mrow></mrow><mrow><mi>Ρ</mi><msub><mrow></mrow><mrow><mi>R</mi><msup><mrow></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup></mrow></msub></mrow><mtext>Τ</mtext></msubsup><mo stretchy="false">(</mo><mi>d</mi><mo>,</mo><mo>:</mo><mo stretchy="false">)</mo></mrow><mrow><msqrt><mrow><mrow><mo>|</mo><mrow><mi mathvariant="bold-italic">Q</mi><msub><mrow></mrow><mrow><mi>Ρ</mi><msub><mrow></mrow><mtext>L</mtext></msub></mrow></msub><mo stretchy="false">(</mo><mi>l</mi><mo>,</mo><mo>:</mo><mo stretchy="false">)</mo></mrow><mo>|</mo></mrow><mrow><mo>|</mo><mrow><mi mathvariant="bold-italic">Q</mi><msubsup><mrow></mrow><mrow><mi>Ρ</mi><msub><mrow></mrow><mrow><mi>R</mi><msup><mrow></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup></mrow></msub></mrow><mtext>Τ</mtext></msubsup><mo stretchy="false">(</mo><mi>d</mi><mo>,</mo><mo>:</mo><mo stretchy="false">)</mo></mrow><mo>|</mo></mrow></mrow></msqrt></mrow></mfrac><mo>.</mo><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>3</mn><mo stretchy="false">)</mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="105">由式(1)～(3)我们就可以计算出lncRNA和疾病之间的关联得分.可以看到,计算HeteSim得分的过程主要包括3个部分:邻接矩阵标准化运算、矩阵连乘运算、相似度标准化运算.</p>
                </div>
                <h3 id="106" name="106" class="anchor-tag"><b>2 实验结果与分析</b></h3>
                <h4 class="anchor-tag" id="107" name="107"><b>2.1 实验数据</b></h4>
                <div class="p1">
                    <p id="108">实验中所使用疾病与基因关联数据均来自文献<citation id="166" type="reference">[<a class="sup">25</a>]</citation>,包括lncRNA与疾病关联数据以及已知的编码基因与疾病关联数据.lncRNA与疾病关联数据包括2个部分:1)来自LncRNADisease数据库<citation id="167" type="reference"><link href="27" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>的数据,其中包含480条实验验证的lncRNA与疾病关联关系,涉及到166种疾病和118种lncRNA;2)在PubMed上进行文本挖掘得到的lncRNA与疾病关联数据,其中包含380条lncRNA-疾病关联的数据,包括226种lncRNA和145种疾病.</p>
                </div>
                <div class="p1">
                    <p id="109">整合上述2种数据集,最终得到了578条lncRNA-疾病关联关系,其中包括295种lncRNA和214种疾病,构成了lncRNA-疾病异质信息网络.</p>
                </div>
                <div class="p1">
                    <p id="110">编码基因与疾病关联数据来自OMIM数据库<citation id="168" type="reference"><link href="53" rel="bibliography" /><sup>[<a class="sup">26</a>]</sup></citation>.针对上述lncRNA-疾病关联数据中涉及到的214种疾病,其中160种疾病可通过MIM编号在OMIM数据库中找到该疾病的致病基因,Yang等人<citation id="169" type="reference"><link href="51" rel="bibliography" /><sup>[<a class="sup">25</a>]</sup></citation>提取了OMIM数据库中这160种疾病与编码基因的关联关系,得到980条编码基因与疾病关联的数据条目,包括801个编码基因和160种疾病.</p>
                </div>
                <div class="p1">
                    <p id="111">通过整合上述lncRNA与疾病关联数据、编码基因与疾病关联数据,得到1 558条编码-长非编码基因与疾病的关联关系,其中包括214种疾病和1 096种基因(编码基因或lncRNA),根据以上数据构建基因-疾病异质信息网络.</p>
                </div>
                <div class="p1">
                    <p id="112">上述2个网络中的具体信息如表1所示:</p>
                </div>
                <div class="area_img" id="113">
                    <p class="img_tit"><b>表1 lncRNA/基因-疾病异质信息网络中的具体信息</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 1 Specific Information in the LncRNA/Gene-Disease Heterogeneous Information Network</b></p>
                    <p class="img_note"></p>
                    <table id="113" border="1"><tr><td><br />Heterogeneous<br />Information<br />Networks</td><td>Number of<br />LncRNA/Coding<br />Gene</td><td>Number <br />of Disease</td><td>Number<br />of Edges</td></tr><tr><td>LncRNA-Disease</td><td>295/0</td><td>214</td><td>578</td></tr><tr><td><br />Gene-Disease</td><td>295/801</td><td>214</td><td>1 558</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h4 class="anchor-tag" id="114" name="114"><b>2.2 性能分析</b></h4>
                <div class="p1">
                    <p id="115">对基因-疾病异质信息网络中不存在连边的基因与疾病对,采用HeteSim算法计算疾病与基因之间的关联得分,预测潜在的lncRNA和疾病关联关系.对每一个疾病,选取关联得分在top10的基因认为是其潜在的致病基因.</p>
                </div>
                <div class="p1">
                    <p id="116">HeteSim在lncRNA-疾病异质信息网络中的性能通过留一交叉验证(leave-one-out cross valida-tion, LOOCV)实验来评估.由于二部网络中度为1的节点所关联边被移除后会成为孤立节点,不能通过网络方法和计算模型得到任何信息,因此本文的预测方法无法计算这些边的得分值.所以,在进行留一交叉验证之前应过滤这类边.最后,我们保留了532条边,其中包括103个疾病和163个基因(包括44个lncRNA和119个编码基因).对于保留的每一条关联关系中的疾病,我们在没有边相连的lncRNA中随机选取1个lncRNA与该疾病相连,构造本文实验的负样本.</p>
                </div>
                <div class="p1">
                    <p id="117">在每次留一交叉验证运行过程中,我们删除1个已知的lncRNA-疾病关联边,然后在剩下的网络中应用HeteSim算法计算出删除边的HeteSim关联得分.这个被删除的边被认为是测试样本,剩下的网络结构被认为是训练样本.通过设定不同的阈值(top <i>k</i>%,1≤<i>k</i>≤100),我们使用ROC曲线和ROC曲线下的区域(<i>AUC</i>)来评估HeteSim在网络上的表现.ROC曲线的横轴是“假阳性率”(<i>FPR</i>),它是实际负样本中错误地识别为正样本的比例;纵轴是“真阳性率”(<i>TPR</i>),它是所有实际正样本中正确识别的正样本的比例.二者的计算公式为</p>
                </div>
                <div class="p1">
                    <p id="118"><mathml id="141"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>F</mi><mi>Ρ</mi><mi>R</mi><mo>=</mo><mfrac><mrow><mi>F</mi><mi>Ρ</mi></mrow><mrow><mi>F</mi><mi>Ρ</mi><mo>+</mo><mi>Τ</mi><mi>Ν</mi></mrow></mfrac></mrow></math></mathml>,(4)</p>
                </div>
                <div class="p1">
                    <p id="119"><mathml id="142"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Τ</mi><mi>Ρ</mi><mi>R</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac></mrow></math></mathml>.(5)</p>
                </div>
                <div class="p1">
                    <p id="120"><i>TPR</i>表示的是移除的关联边排名在<i>k</i>%以内的比率;<i>FPR</i>表示的是不存在的关联边排名在<i>k</i>%以内的比率.当阈值<i>k</i>在1～100之间变化时可以得到相应的<i>TPR</i>和<i>FPR</i>.通过这种方式,可以绘制ROC曲线,从而计算<i>AUC</i>.按照以上步骤,我们在lncRNA-疾病异质信息网络上进行了留一交叉验证,并取得了0.682 8的<i>AUC</i>.相应的ROC曲线如图4所示:</p>
                </div>
                <div class="area_img" id="121">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909010_121.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 留一交叉验证ROC曲线图" src="Detail/GetImg?filename=images/JFYZ201909010_121.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 留一交叉验证ROC曲线图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909010_121.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 4 Leave-one-out cross validation ROC curve</p>

                </div>
                <div class="p1">
                    <p id="122">为了提高方法的性能,我们将编码基因和疾病关联关系整合到lncRNA-疾病网络中得到基因-疾病异质信息网络.我们在基因-疾病异质信息网络上进行了留一交叉验证,负样本的构造方法与之前类似,得到的<i>AUC</i>值为0.783 5,如图4所示.很明显,编码基因-疾病关联关系的整合可以提高我们方法的性能,分析原因主要是通过集成编码基因-疾病关联数据增加了网络中边的数量,使网络结构变得更紧密,潜在的基因可以从其他基因和疾病中获得更多信息传播,从而可以更好地进行预测.因此,在我们做链路预测相关方面研究时,通过整合多种数据,结合更有意义的语义信息,可以有效地提升预测的准确性.</p>
                </div>
                <div class="p1">
                    <p id="123">在这里我们与Yang等人<citation id="170" type="reference"><link href="51" rel="bibliography" /><sup>[<a class="sup">25</a>]</sup></citation>提出的方法在相同的数据集上进行比较,这2种方法都是基于已知的基因与疾病之间的关联,不借助其他的信息进行疾病与基因的关联预测,图4给出了本文方法与Yang等人的方法预测结果比较,本文方法优于Yang等人的方法.此外.我们又与IRWRLDA<citation id="171" type="reference"><link href="43" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>和KATZLDA<citation id="172" type="reference"><link href="45" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>这2种方法进行比较,这2种方法除了已知的lncRNA-疾病关联数据,还加入了lncRNA相似性和疾病相似性的数据来进行预测,本文的方法优于这2种方法,比较结果如图4所示.</p>
                </div>
                <h4 class="anchor-tag" id="124" name="124"><b>2.3 案例分析</b></h4>
                <div class="p1">
                    <p id="125">为进一步验证本文方法的可靠性和实用性,分别对卵巢癌和胃癌2种疾病做案例分析.对每一种疾病,所有未与该疾病有关联连边的基因按照其与该疾病的关联得分从大到小进行排序,排名top10的基因被认为是与该疾病潜在关联的基因.</p>
                </div>
                <div class="p1">
                    <p id="126">卵巢恶性肿瘤是女性常见的恶性肿瘤之一,发病率仅次于子宫颈癌和子宫体癌.而卵巢上皮癌死亡率占各类妇科肿瘤的首位,对妇女的生命造成非常严重的威胁.表2显示了卵巢癌中排名top10的基因,包括4个lncRNA,目前这4个已有文献通过生物实验等证实确实与该疾病有关,对应的PubMed唯一标识码(PubMed unique identifier, PMID)也在表2中给出,通过PMID可以在PubMed搜索引擎中查阅对应的文献.例如:Zhou等人<citation id="173" type="reference"><link href="55" rel="bibliography" /><sup>[<a class="sup">27</a>]</sup></citation>通过研究发现MALAT-1在卵巢肿瘤中高表达,会促进卵巢癌细胞的生长和迁移,表明MALAT-1可能是卵巢癌发展的重要因素;Yang等人<citation id="174" type="reference"><link href="57" rel="bibliography" /><sup>[<a class="sup">28</a>]</sup></citation>通过实验发现UCA1在上皮性卵巢癌组织和细胞中异常上调,研究表明UCA1是上皮性卵巢癌的新预后生物标志物;Xiu等人<citation id="175" type="reference"><link href="59" rel="bibliography" /><sup>[<a class="sup">29</a>]</sup></citation>发现MEG3的表达在上皮性卵巢癌中较低,通过调节ATG3活性和诱导自噬在上皮性卵巢癌中充当肿瘤抑制剂,并可能被认为是卵巢癌的生物标志物;Zhang等人<citation id="176" type="reference"><link href="61" rel="bibliography" /><sup>[<a class="sup">30</a>]</sup></citation>研究发现在患有卵巢癌的患者中,HOTAIR显著上调.此外,HOTAIR的上调增加了卵巢癌细胞的增殖、迁移和侵袭,从而促成了卵巢癌细胞的恶性进展.</p>
                </div>
                <div class="area_img" id="127">
                    <p class="img_tit"><b>表2 Top10与卵巢癌有关的基因</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 2 Top10 Genes Linked to Ovarian Cancer</b></p>
                    <p class="img_note"></p>
                    <table id="127" border="1"><tr><td>Gene</td><td>Rank</td><td>PMID</td><td>Gene</td><td>Rank</td><td>PMID</td></tr><tr><td>MALAT-1</td><td>1</td><td>27 227 769</td><td>HOTAIR</td><td>6</td><td>27 484 896</td></tr><tr><td><br />UCA1</td><td>2</td><td>26 867 765</td><td>CASP8</td><td>7</td><td></td></tr><tr><td><br />KRAS</td><td>3</td><td></td><td>KCNQ1OT1</td><td>8</td><td></td></tr><tr><td><br />MEG3</td><td>4</td><td>28 423 647</td><td>KLF6</td><td>9</td><td></td></tr><tr><td><br />TP53</td><td>5</td><td></td><td>CHEK2</td><td>10</td><td></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="128">胃癌是起源于胃黏膜上皮的恶性肿瘤,在我国各种恶性肿瘤中发病率居首位,对人类的健康造成巨大威胁.表3显示了胃癌中排名top10的基因,包括5个lncRNA,其中有3个目前已有文献证实确实与该疾病有关.例如:Okugawa等人<citation id="177" type="reference"><link href="63" rel="bibliography" /><sup>[<a class="sup">31</a>]</sup></citation>通过实验发现在腹膜播散的胃癌细胞中,HOTAIR的SiRNA抑制细胞增殖、迁移和侵袭,为HOTAIR表达作为鉴定腹膜转移患者的潜在生物标志物的生物学和临床意义提供了新的证据,并且作为胃肿瘤患者的新治疗靶点;Chen等人<citation id="178" type="reference"><link href="65" rel="bibliography" /><sup>[<a class="sup">32</a>]</sup></citation>通过实验发现MALAT-1在胃癌细胞系和组织中上调;此外,MALAT-1在高转移潜能胃癌细胞系SGC7901M中的表达高于在低转移潜能胃癌细胞系SGC7901NM中的表达,结果表明MALAT-1可能部分通过调节上皮间质转化(EMT)促进胃癌细胞的迁移和侵袭;Xu等人<citation id="179" type="reference"><link href="67" rel="bibliography" /><sup>[<a class="sup">33</a>]</sup></citation>通过实验证明MEG3/miR21通过调节EMT参与胃癌的肿瘤进展和转移.</p>
                </div>
                <div class="area_img" id="129">
                    <p class="img_tit"><b>表3 Top10与胃癌有关的基因</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 3 Top10 Genes Linked to Gastric Cancer</b></p>
                    <p class="img_note"></p>
                    <table id="129" border="1"><tr><td>Gene</td><td>Rank</td><td>PMID</td><td>Gene</td><td>Rank</td><td>PMID</td></tr><tr><td>MALAT-1</td><td>1</td><td>28 276 823</td><td>SLC22A1L</td><td>6</td><td></td></tr><tr><td><br />FGFR3</td><td>2</td><td></td><td>HOTAIR</td><td>7</td><td>25 280 565</td></tr><tr><td><br />BRCA2</td><td>3</td><td></td><td>PRKN</td><td>8</td><td></td></tr><tr><td><br />MEG3</td><td>4</td><td>29 749 532</td><td>BC043430</td><td>9</td><td></td></tr><tr><td><br />CASP8</td><td>5</td><td></td><td>BC017743</td><td>10</td><td></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h3 id="130" name="130" class="anchor-tag"><b>3 结  论</b></h3>
                <div class="p1">
                    <p id="131">长非编码 RNA在许多生物过程中具有重要的功能,这些长非编码 RNA 的变异或功能失调会导致一些复杂疾病的发生.因此,通过生物信息学方法预测潜在的长非编码 RNA-疾病关联关系,这对于致病机理的探索以及疾病诊断、治疗、预后和预防都具有重要的意义.</p>
                </div>
                <div class="p1">
                    <p id="132">近年来,针对这一问题,很多研究者已提出了其他基于网络的预测方法,并且在网络模型的基础上集成基因表达数据或者基因与miRNA之间的调控关系数据,实现lncRNA与疾病关联的预测.</p>
                </div>
                <div class="p1">
                    <p id="133">本文使用了一种异质信息网络中的相关性计算方法——HeteSim,用来预测lncRNA与疾病之间的关联.该方法基于路径约束,通过元路径两端节点随机游走到中间节点相遇的概率作为疾病与lncRNA之间的关联得分,发掘潜在的疾病与lncRNA关联关系.实验结果表明该计算方法有较高的预测准确性和鲁棒性,并且该方法可以很好地集成其他类型的关联数据,例如基因间的蛋白质相互作用<citation id="180" type="reference"><link href="69" rel="bibliography" /><sup>[<a class="sup">34</a>]</sup></citation>、lncRNA和编码基因的共表达、miRNA对lncRNA和编码基因的调控、疾病之间的相似性信息等.集成这些关联数据,从而对元路径进行扩展,可以使更多与lncRNA疾病相关的语义信息被用来预测,有利于预测的准确性,这也是本文工作进一步深入研究的方向.</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="3">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Global Identification of Human Transcribed Sequences with Genome Tiling Arrays">

                                <b>[1]</b>Bertone P,Stolc V,Royce T E,et al.Global identification of human transcribed sequences with genome tiling arrays[J].Science,2004,306(5705):2242- 2246
                            </a>
                        </p>
                        <p id="5">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project">

                                <b>[2]</b>The ENCODE Project Consortium.Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project[J].Nature,2007,447(7146):799- 816
                            </a>
                        </p>
                        <p id="7">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=RNA maps reveal new RNA classes and a possible function for pervasive transcription">

                                <b>[3]</b>Kapranov P,Cheng J,Dike S,et al.RNA maps reveal new RNA classes and a possible function for pervasive transcription[J].Science,2007,316(5830):1484- 1488
                            </a>
                        </p>
                        <p id="9">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The transcriptional landscape of the mammalian genome">

                                <b>[4]</b>Carninci P,Kasukawa T,Katayama S,et al.The transcriptional landscape of the mammalian genome[J].Science,2005,309(5740):1559- 1563
                            </a>
                        </p>
                        <p id="11">
                            <a id="bibliography_5" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD00001142422&amp;v=MDIyMTZjTUg3UjdxZWJ1ZHRGQ3JsVkwzQUlsYz1OaWZjYXJPNEh0SE5yb3RIWU9rTlkzazV6QmRoNGo5OVNYcVJyeG94&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[5]</b>Taft R J,Pang K C,Mercer T R,et al.Non-coding RNAs:Regulators of disease[J].The Journal of Pathology,2010,220(2):126- 139
                            </a>
                        </p>
                        <p id="13">
                            <a id="bibliography_6" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJRS&amp;filename=SJRS120925009235&amp;v=MjgzMTE4enhVU21EZDlTSDduM3hFOWZidm5LcmlmWnU5dUZDdm5VN2pLS0Z3ZE5pZlpmYks2SHRqT3FvOUZiZWtNQ1JN&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[6]</b>Derrien T,Johnson R,Bussotti G,et al.The GENCODE v7 catalog of human long noncoding RNAs:Analysis of their gene structure,evolution,and expression[J].Genome Research,2012,22(9):1775- 1789
                            </a>
                        </p>
                        <p id="15">
                            <a id="bibliography_7" target="_blank" href="http://scholar.cnki.net/result.aspx?q=lnc RNAdb v2.0:expanding the reference database for functional long noncoding RNAs">

                                <b>[7]</b>Quek X C,Thomson D W,Maag J L,et al.lncRNAdb v2.0:Expanding the reference database for functional long noncoding RNAs[J].Nucleic Acids Research,2015,43(D1):168- 173
                            </a>
                        </p>
                        <p id="17">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=LncRBase:an enriched resource for lncRNA information">

                                <b>[8]</b>Chakraborty S,Deb A,Maji R K,et al.LncRBase:An enriched resource for lncRNA information[J].PloS One,2014,9(9):e108010
                            </a>
                        </p>
                        <p id="19">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=LncRNA2Function:a comprehensive resource for functional investigation of human lncRNAs based on RNA-seq data">

                                <b>[9]</b>Jiang Qinghua,Ma Rui,Wang Jixuan,et al.LncRNA2-Function:A comprehensive resource for functional investigation of human lncRNAs based on RNA-seq data[J].BMC Genomics,2015,16(Suppl 3):S2
                            </a>
                        </p>
                        <p id="21">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=LncRNA Target:A database for differentially expressed genes after lncRNA knockdown or overexpression">

                                <b>[10]</b>Jiang Qinghua,Wang Jixuan,Wu Xiaoliang,et al.LncRNA2Target:A database for differentially expressed genes after lncRNA knockdown or overexpression[J].Nucleic Acids Research,2014,43(D1):D193- D196
                            </a>
                        </p>
                        <p id="23">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Long non-coding RNAs function annotation: a global prediction method based on bi-colored networks">

                                <b>[11]</b>Guo Xingli,Gao Lin,Liao Qi,et al.Long non-coding RNAs function annotation:A global prediction method based on bi-colored networks[J].Nucleic Acids Research,2013,41(2):e35
                            </a>
                        </p>
                        <p id="25">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Advances in long noncod ing RNAs:Identification,structure prediction and func tion annotation">

                                <b>[12]</b>Guo Xingli,Gao Lin,Wang Yu,et al.Advances in long noncoding RNAs:Identification,structure prediction and function annotation[J].Briefings in Functional Genomics,2015,15(1):38- 46
                            </a>
                        </p>
                        <p id="27">
                            <a id="bibliography_13" >
                                    <b>[13]</b>
                                Chen Geng,Wang Ziyun,Wang Dongqing,et al.LncRNADisease:A database for long-non-coding RNA-associated diseases[J].Nucleic Acids Research,2012,41(D1):D983- D986
                            </a>
                        </p>
                        <p id="29">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Lnc2Cancer:a manually curated database of experimentally supported lncRNAs associated with various human cancers">

                                <b>[14]</b>Ning Shangwei,Zhang Jizhou,Wang Peng,et al.Lnc2Cancer:A manually curated database of experimentally supported lncRNAs associated with various human cancers[J].Nucleic Acids Research,2015,44(D1):D980- D985
                            </a>
                        </p>
                        <p id="31">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Identification of cancer-related lnc RNAs through integrating genome,regulome and transcriptome features">

                                <b>[15]</b>Zhao Tingting,Xu Jinyuan,Liu Ling,et al.Identification of cancer-related lncRNAs through integrating genome,regulome and transcriptome features[J].Molecular BioSystems,2014,11(1):126- 136
                            </a>
                        </p>
                        <p id="33">
                            <a id="bibliography_16" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJRS&amp;filename=SJRS13102500001233&amp;v=MDQ5MDU5SE9xbzlGWk9zT0RuODZvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMM0lKMXdjYUJzPU5pZlpmYks3SA==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[16]</b>Chen Xing,Yan Guiying.Novel human lncRNA-disease association inference based on lncRNA expression profiles[J].Bioinformatics,2013,29(20):2617- 2624
                            </a>
                        </p>
                        <p id="35">
                            <a id="bibliography_17" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Inferring novel lncRNA–disease associations based on a random walk model of a lncRNA functional similarity network">

                                <b>[17]</b>Sun Jie,Shi Hongbo,Wang Zhenzhen,et al.Inferring novel lncRNA-disease associations based on a random walk model of a lncRNA functional similarity network[J].Molecular BioSystems,2014,10(8):2074- 2081
                            </a>
                        </p>
                        <p id="37">
                            <a id="bibliography_18" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Construction of a lncRNA-PCG bipartite network and identification of cancer-related lncRNAs:a case study in prostate cancer">

                                <b>[18]</b>Liu Yongjing,Zhang Rui,Qiu Fujun,et al.Construction of a lncRNA-PCG bipartite network and identification of cancer-related lncRNAs:A case study in prostate cancer[J].Molecular BioSystems,2015,11(2):384- 393
                            </a>
                        </p>
                        <p id="39">
                            <a id="bibliography_19" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prioritizing candidate disease-related long non-coding RNAs by walking on the heterogeneous lncRNA and disease network">

                                <b>[19]</b>Zhou Meng,Wang Xiaojun,Li Jiawei,et al.Prioritizing candidate disease-related long non-coding RNAs by walking on the heterogeneous lncRNA and disease network[J].Molecular BioSystems,2015,11(3):760- 769
                            </a>
                        </p>
                        <p id="41">
                            <a id="bibliography_20" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Heterogeneous Network Model to Infer Human Disease-Long Intergenic Non Coding RNA Associations.">

                                <b>[20]</b>Ganegoda G U,Li Min,Wang Weiping,et al.Heterogeneous network model to infer human disease-long intergenic non-coding RNA associations[J].IEEE Transactions on Nanobioscience,2015,14(2):175- 183
                            </a>
                        </p>
                        <p id="43">
                            <a id="bibliography_21" target="_blank" href="http://scholar.cnki.net/result.aspx?q=IRWRLDA:improved random walk with restart for lnc RNA-disease association prediction">

                                <b>[21]</b>Chen Xing,You Zhuhong,Yan Guiying,et al.IRWRLDA:Improved random walk with restart for lncRNA-disease association prediction[J].Oncotarget,2016,7(36):57919
                            </a>
                        </p>
                        <p id="45">
                            <a id="bibliography_22" target="_blank" href="http://scholar.cnki.net/result.aspx?q=KATZLDA:KATZ measure for the IncRNA-disease association prediction">

                                <b>[22]</b>Chen Xing.KATZLDA:KATZ measure for the lncRNA-disease association prediction[J].Scientific Reports,2015,5(1):No.16840
                            </a>
                        </p>
                        <p id="47">
                            <a id="bibliography_23" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCMB8EE5F078DFD48021E9B79C66BCA8C3D&amp;v=MjYxNzdPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU5waHc3cTN3cUU9TmlmSVk4R3dhNlRKMlk5Q2JKOTVlSGd4enhRU256WVBUM2FScWhSSENzT2NOcm5yQw==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[23]</b>Zeng Xiangxiang,Liao Yuanlu,Liu Yuansheng,et al.Prediction and validation of disease genes using HeteSim Scores[J].IEEE/ACM Transactions on Computational Biology and Bioinformatics,2017,14(3):687- 695
                            </a>
                        </p>
                        <p id="49">
                            <a id="bibliography_24" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Hete Sim:A General Framework for Relevance Measure in Heterogeneous Networks">

                                <b>[24]</b>Shi Chuan,Kong Xiangnan,Huang Yue,et al.HeteSim:A general framework for relevance measure in heterogeneous networks[J].IEEE Transactions on Knowledge and Data Engineering,2014,26(10):2479- 2492
                            </a>
                        </p>
                        <p id="51">
                            <a id="bibliography_25" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A network based method for analysis of lncRNA-disease associations and prediction of lncRNAs implicated in diseases">

                                <b>[25]</b>Yang Xiaofei,Gao Lin,Guo Xingli,et al.A network based method for analysis of lncRNA-disease associations and prediction of lncRNAs implicated in diseases[J].PloS One,2014,9(1):e87797
                            </a>
                        </p>
                        <p id="53">
                            <a id="bibliography_26" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic disorders">

                                <b>[26]</b>Hamosh A,Scott A F,Amberger J S,et al.Online mendelian inheritance in man (OMIM),a knowledgebase of human genes and genetic disorders[J].Nucleic Acids Research,2005,33(Suppl 1):D514- D517
                            </a>
                        </p>
                        <p id="55">
                            <a id="bibliography_27" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The Long Noncoding RNA MALAT-1 Is Highly Expressed in Ovarian Cancer and Induces Cell Growth and Migration">

                                <b>[27]</b>Zhou Yanqing,Xu Xiaying,Lü Huabing,et al.The long noncoding RNA MALAT-1 is highly expressed in ovarian cancer and induces cell growth and migration[J].PLoS One,2016,11(5):e0155250
                            </a>
                        </p>
                        <p id="57">
                            <a id="bibliography_28" target="_blank" href="http://scholar.cnki.net/result.aspx?q=UCA1 functions as a competing endogenous RNA to suppress epithelial ovarian cancer metastasis">

                                <b>[28]</b>Yang Yijun,Jiang Yi,Wan Yicong,et al.UCA1 functions as a competing endogenous RNA to suppress epithelial ovarian cancer metastasis[J].Tumor Biology,2016,37(8):10633- 10641
                            </a>
                        </p>
                        <p id="59">
                            <a id="bibliography_29" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Upregulation of the lncRNA Meg3 induces autophagy to inhibit tumorigenesis and progression of epithelial ovarian carcinoma by regulating activity of ATG3">

                                <b>[29]</b>Xiu Yinling,Sun Kaixuan,Chen Xi,et al.Upregulation of the lncRNA Meg3 induces autophagy to inhibit tumorigenesis and progression of epithelial ovarian carcinoma by regulating activity of ATG3[J].Oncotarget,2017,8(19):31714- 31725
                            </a>
                        </p>
                        <p id="61">
                            <a id="bibliography_30" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJSD&amp;filename=SJSDD01273F3EA10AD5C1AB20211C3512816&amp;v=MDc4NjhDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOcGh3N3Ezd3FFPU5pZllhc2U0SDlQTHJQbEdFWm9PREExTnltVVNtMDEvU0gzanJXRTJmTE9XVGJ1Wg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[30]</b>Zhang Zhongbao,Cheng Jiajing,Wu Yi,et al.LncRNA HOTAIR controls the expression of Rab22a by sponging miR-373 in ovarian cancer[J].Molecular Medicine Reports,2016,14(3):2465- 2472
                            </a>
                        </p>
                        <p id="63">
                            <a id="bibliography_31" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Metastasis-associated long non-coding RNA drives gastric cancer development and promotes peritoneal metastasis">

                                <b>[31]</b>Okugawa Y,Toiyama Y,Hur K,et al.Metastasis-associated long non-coding RNA drives gastric cancer development and promotes peritoneal metastasis[J].Carcinogenesis,2014,35(12):2731- 2739
                            </a>
                        </p>
                        <p id="65">
                            <a id="bibliography_32" target="_blank" href="/kcms/detail/detail.aspx?dbcode=STJD&amp;filename=STJDFD2FE98DE637125905C7C06BD76C2462&amp;v=MTcyNzB1SFlmT0dRbGZDcGJRMzVOcGh3N3Ezd3FFPU5qbkJhc1hNSEtlNXBvY3hFZTBNQzMwN3loOFQ3MHg2TzMvazNtWXlmOEdXUWJ5ZENPTnZGU2lXV3I3SklGcG1hQg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[32]</b>Chen Di,Liu Lili,Wang Kai,et al.The role of MALAT-1 in the invasion and metastasis of gastric cancer[J].Scandinavian Journal of Gastroenterology,2017,52(6/7):790- 796
                            </a>
                        </p>
                        <p id="67">
                            <a id="bibliography_33" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJSD&amp;filename=SJSD5A1E05D608A859BDA16A4DE533310D97&amp;v=MDAxMjF6a01UQXVYcVJFMmVyT1VNYk9ZQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TnBodzdxM3dxRT1OaWZZYXJiSkg2VE1xdnREWk9OK0JIa3d2V0ppNg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[33]</b>Xu Gang,Meng Lei,Yuan Dawei,et al.MEG3/miR 21 axis affects cell mobility by suppressing epithelial mesenchymal transition in gastric cancer[J].Oncology Reports,2018,40(1):39- 48
                            </a>
                        </p>
                        <p id="69">
                            <a id="bibliography_34" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201706012&amp;v=MTgxMjF5dlNkTEc0SDliTXFZOUVab1FLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUnNGeS9uVzczQkw=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[34]</b>Li Min,Meng Xiangmao.Progress in the construction,analysis and application of dynamic protein networks[J].Journal of Computer Research and Development,2017:54(6):1281- 1299 (in Chinese)(李敏,孟祥茂.动态蛋白质网络的构建,分析及应用研究进展[J].计算机研究与发展,2017,54(6):1281- 1299)
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JFYZ201909010" />
        <input id="dpi" type="hidden" value="600" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>


    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201909010&amp;v=MDA1MzZXNzNCTHl2U2RMRzRIOWpNcG85RVpJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSc0Z5L24=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QeitJK1dUenpTMFV5a3FOUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>


    <link href="/kxreader/Content/css/LeftDetail?v=NLcKG8I1SJUaVFrQ0iGpF2klAT0OsmHRaVSZ1rKb5xg1" rel="stylesheet"/>

</body>
</html>

