

<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>

</head>

<body>

    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637127160502176250%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJFYZ201909012%26RESULT%3d1%26SIGN%3dWrUk0vuDKvOREKBW8oH%252b3bvyMuA%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JFYZ201909012&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JFYZ201909012&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>


    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201909012&amp;v=MDUwNjg2ajU0TzN6cXFCdEdGckNVUkxPZVplUnNGeS9nVXJ2Skx5dlNkTEc0SDlqTXBvOUVab1FLREg4NHZSNFQ=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#72" data-title="&lt;b&gt;1 比对算法的建模过程&lt;/b&gt; "><b>1 比对算法的建模过程</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#73" data-title="&lt;b&gt;1.1 领域分析&lt;/b&gt;"><b>1.1 领域分析</b></a></li>
                                                <li><a href="#78" data-title="&lt;b&gt;1.2 特征建模流程&lt;/b&gt;"><b>1.2 特征建模流程</b></a></li>
                                                <li><a href="#82" data-title="&lt;b&gt;1.3 DPPSAA领域特征建模&lt;/b&gt;"><b>1.3 DPPSAA领域特征建模</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#89" data-title="&lt;b&gt;2 比对算法构件的设计与实现&lt;/b&gt; "><b>2 比对算法构件的设计与实现</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#90" data-title="&lt;b&gt;2.1 DPPSAA领域算法构件交互设计&lt;/b&gt;"><b>2.1 DPPSAA领域算法构件交互设计</b></a></li>
                                                <li><a href="#95" data-title="&lt;b&gt;2.2 Apla形式化实现&lt;/b&gt;"><b>2.2 Apla形式化实现</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#151" data-title="&lt;b&gt;3 NW算法的装配实现&lt;/b&gt; "><b>3 NW算法的装配实现</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#208" data-title="&lt;b&gt;4 实验及其结果分析&lt;/b&gt; "><b>4 实验及其结果分析</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#231" data-title="&lt;b&gt;5 相关研究比较&lt;/b&gt; "><b>5 相关研究比较</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#236" data-title="&lt;b&gt;6 结束语&lt;/b&gt; "><b>6 结束语</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#76" data-title="图1 特征图">图1 特征图</a></li>
                                                <li><a href="#81" data-title="图2 领域分析流程图">图2 领域分析流程图</a></li>
                                                <li><a href="#87" data-title="图3 DPPSAA特征模型">图3 DPPSAA特征模型</a></li>
                                                <li><a href="#93" data-title="图4 算法构件交互模型">图4 算法构件交互模型</a></li>
                                                <li><a href="#108" data-title="图5 得分矩阵元素的得分来源">图5 得分矩阵元素的得分来源</a></li>
                                                <li><a href="#225" data-title="图6 ADT转换结果">图6 ADT转换结果</a></li>
                                                <li><a href="#226" data-title="图7 泛型子程序转换结果">图7 泛型子程序转换结果</a></li>
                                                <li><a href="#227" data-title="图8 NW的C++装配过程">图8 NW的C++装配过程</a></li>
                                                <li><a href="#228" data-title="图9 NW比对结果">图9 NW比对结果</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="3">


                                    <a id="bibliography_1" title="Illumina.HiSeqX instrument performance parameters[EB/OL].2016[2018-09-26].https://www.illumina.com/systems/sequencing-platforms/hiseq-x/specifications.html" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=HiSeqX instrument performance parameters">
                                        <b>[1]</b>
                                        Illumina.HiSeqX instrument performance parameters[EB/OL].2016[2018-09-26].https://www.illumina.com/systems/sequencing-platforms/hiseq-x/specifications.html
                                    </a>
                                </li>
                                <li id="5">


                                    <a id="bibliography_2" title="Needleman S B,Wunsch C D.A general method applicable to the search for similarities in the amino acid sequence of two proteins[J].Journal of Molecular Biology,1970,48(3):443- 453" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A general method applicable to the search for similarities in the amino acid sequence of two proteins">
                                        <b>[2]</b>
                                        Needleman S B,Wunsch C D.A general method applicable to the search for similarities in the amino acid sequence of two proteins[J].Journal of Molecular Biology,1970,48(3):443- 453
                                    </a>
                                </li>
                                <li id="7">


                                    <a id="bibliography_3" title="Smith T F,Waterman M S.Identification of common molecular subsequences[J].Journal of Molecular Biology,1981,147(1):195- 197" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Identification of common molecular subsequences">
                                        <b>[3]</b>
                                        Smith T F,Waterman M S.Identification of common molecular subsequences[J].Journal of Molecular Biology,1981,147(1):195- 197
                                    </a>
                                </li>
                                <li id="9">


                                    <a id="bibliography_4" title="Hirschberg D S.A linear space algorithm for computing maximal common subsequences[J].Communications of the ACM,1975,18(18):341- 343" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM13091000024362&amp;v=MTcyMzlpclJkR2VycVFUTW53WmVadUh5am1VTDNJSjFzVmJoTT1OaWZJWTdLN0h0ak5yNDlGWk9rTEQzbzdvQk1UNlQ0UFFILw==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[4]</b>
                                        Hirschberg D S.A linear space algorithm for computing maximal common subsequences[J].Communications of the ACM,1975,18(18):341- 343
                                    </a>
                                </li>
                                <li id="11">


                                    <a id="bibliography_5" title="Ukkonen E.Algorithms for approximate string matching[J].Information &amp;amp; Control,1985,64(1/2/3):100- 118" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Algorithms for approximate string matching">
                                        <b>[5]</b>
                                        Ukkonen E.Algorithms for approximate string matching[J].Information &amp;amp; Control,1985,64(1/2/3):100- 118
                                    </a>
                                </li>
                                <li id="13">


                                    <a id="bibliography_6" title="Abbasi M,Paquete L,Liefooghe A,et al.Improvements on bicriteria pairwise sequence alignment:Algorithms and applications[J].Bioinformatics,2013,29(8):996- 1003" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Improvements on bicriteria pairwise sequence alignment:Algorithms and applications">
                                        <b>[6]</b>
                                        Abbasi M,Paquete L,Liefooghe A,et al.Improvements on bicriteria pairwise sequence alignment:Algorithms and applications[J].Bioinformatics,2013,29(8):996- 1003
                                    </a>
                                </li>
                                <li id="15">


                                    <a id="bibliography_7" title="Train C M,Glover N M,Gonnet G H,et al.Orthologous matrix (OMA) algorithm 2.0:More robust to asymmetric evolutionary rates and more scalable hierarchical orthologous group inference[J].Bioinformatics,2017,33(14):i75- i82" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Orthologous Matrix(OMA)algorithm 2.0,more robust to asymmetric evolutionary rates and more scalable hierarchical orthologous group inference">
                                        <b>[7]</b>
                                        Train C M,Glover N M,Gonnet G H,et al.Orthologous matrix (OMA) algorithm 2.0:More robust to asymmetric evolutionary rates and more scalable hierarchical orthologous group inference[J].Bioinformatics,2017,33(14):i75- i82
                                    </a>
                                </li>
                                <li id="17">


                                    <a id="bibliography_8" title="Altenhoff A M,Škunca N,Glover N,et al.The OMA orthology database in 2015:Function predictions,better plant support,synteny view and other improvements[J].Nucleic Acids Research,2015,43(D1):240- 249" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The OMA orthology database in 2015:function predictions,better plant support,synteny view and other improvements">
                                        <b>[8]</b>
                                        Altenhoff A M,Škunca N,Glover N,et al.The OMA orthology database in 2015:Function predictions,better plant support,synteny view and other improvements[J].Nucleic Acids Research,2015,43(D1):240- 249
                                    </a>
                                </li>
                                <li id="19">


                                    <a id="bibliography_9" title="Šoši M,Šiki M.Edlib:A C/C++ library for fast,exact sequence alignment using edit distance[J].Bioinformatics,2017,33(9):1394- 1395" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Edlib:A C/C++ library for fast,exact sequence alignment using edit distance">
                                        <b>[9]</b>
                                        Šoši M,Šiki M.Edlib:A C/C++ library for fast,exact sequence alignment using edit distance[J].Bioinformatics,2017,33(9):1394- 1395
                                    </a>
                                </li>
                                <li id="21">


                                    <a id="bibliography_10" title="Chattopadhyay A K,Nasiev D,Flower D R.A statistical physics perspective on alignment-independent protein sequence comparison[J].Bioinformatics,2015,31(15):2469- 2674" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A statistical physics perspective on alignment-independent protein sequence comparison">
                                        <b>[10]</b>
                                        Chattopadhyay A K,Nasiev D,Flower D R.A statistical physics perspective on alignment-independent protein sequence comparison[J].Bioinformatics,2015,31(15):2469- 2674
                                    </a>
                                </li>
                                <li id="23">


                                    <a id="bibliography_11" title="Cattaneo G,Petrillo U F,Giancarlo R,et al.Alignment-free sequence comparison over Hadoop for computational biology[C] //Proc of the 44th Int Conf on Parallel Processing Workshops.Piscataway,NJ:IEEE,2015:184- 192" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Alignment-free sequence comparison over Hadoop for computational biology">
                                        <b>[11]</b>
                                        Cattaneo G,Petrillo U F,Giancarlo R,et al.Alignment-free sequence comparison over Hadoop for computational biology[C] //Proc of the 44th Int Conf on Parallel Processing Workshops.Piscataway,NJ:IEEE,2015:184- 192
                                    </a>
                                </li>
                                <li id="25">


                                    <a id="bibliography_12" title="Huo Hongwei,Sun Zhigang,Li Shuangjiang,et al.CS2A:A compressed suffix array-based method for short read alignment[C] //Proc of the 2016 IEEE Data Compression Conf.Piscataway,NJ:IEEE,2016:271- 278" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=CS2A A compressed suffix array-based method for short read alignment">
                                        <b>[12]</b>
                                        Huo Hongwei,Sun Zhigang,Li Shuangjiang,et al.CS2A:A compressed suffix array-based method for short read alignment[C] //Proc of the 2016 IEEE Data Compression Conf.Piscataway,NJ:IEEE,2016:271- 278
                                    </a>
                                </li>
                                <li id="27">


                                    <a id="bibliography_13" title="Isa M N,Murad S A Z,Ismail R C,et al.An efficient processing element architecture for pairwise sequence alignment[C] // Proc of the 33rd Int Conf on Electronic Design.Piscataway,NJ:IEEE,2015:461- 464" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=An efficient processing element architecture for pairwise sequence alignment">
                                        <b>[13]</b>
                                        Isa M N,Murad S A Z,Ismail R C,et al.An efficient processing element architecture for pairwise sequence alignment[C] // Proc of the 33rd Int Conf on Electronic Design.Piscataway,NJ:IEEE,2015:461- 464
                                    </a>
                                </li>
                                <li id="29">


                                    <a id="bibliography_14" title="Farrar M.Striped Smith-Waterman speeds database searches six times over other SIMD implementations[J].Bioinformatics,2007,23(2):156- 161" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Striped Smith--Waterman speeds database searches six times over other SIMD implementations">
                                        <b>[14]</b>
                                        Farrar M.Striped Smith-Waterman speeds database searches six times over other SIMD implementations[J].Bioinformatics,2007,23(2):156- 161
                                    </a>
                                </li>
                                <li id="31">


                                    <a id="bibliography_15" title="Houtgast E,Sima V M,Al-Ars Z.High performance streaming Smith-Waterman implementation with implicit synchronization on Intel FPGA using OpenCL[C] //Proc of the 2018 IEEE Int Conf on Bioinformatics and Bioengineering.Piscataway,NJ:IEEE,2018:492- 496" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=High performance streaming Smith-Waterman implementation with implicit synchronization on Intel FPGA using OpenCL">
                                        <b>[15]</b>
                                        Houtgast E,Sima V M,Al-Ars Z.High performance streaming Smith-Waterman implementation with implicit synchronization on Intel FPGA using OpenCL[C] //Proc of the 2018 IEEE Int Conf on Bioinformatics and Bioengineering.Piscataway,NJ:IEEE,2018:492- 496
                                    </a>
                                </li>
                                <li id="33">


                                    <a id="bibliography_16" title="Junid S A M A,Idros M F M,Razak A H A,et al.Parallel processing cell score design of linear gap penalty Smith-Waterman algorithm[C] //Proc of the 13th IEEE Int Colloquium on Signal Processing &amp;amp; ITS Applications.Piscataway,NJ:IEEE,2017:299- 302" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Parallel processing cell score design of linear gap penalty Smith-Waterman algorithm">
                                        <b>[16]</b>
                                        Junid S A M A,Idros M F M,Razak A H A,et al.Parallel processing cell score design of linear gap penalty Smith-Waterman algorithm[C] //Proc of the 13th IEEE Int Colloquium on Signal Processing &amp;amp; ITS Applications.Piscataway,NJ:IEEE,2017:299- 302
                                    </a>
                                </li>
                                <li id="35">


                                    <a id="bibliography_17" title="Xue Jinyun.A unified approach for developing efficient algorithmic programs[J].Journal of Computer Science &amp;amp; Technology,1997,12(4):314- 329" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00003315832&amp;v=MDQ0OTZiT2dOWTNrNXpCZGg0ajk5U1hxUnJ4b3hjTUg3UjdxZWJ1ZHRGQ3JsVkxySkpGOD1OajdCYXJPNEh0SFBySTVB&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[17]</b>
                                        Xue Jinyun.A unified approach for developing efficient algorithmic programs[J].Journal of Computer Science &amp;amp; Technology,1997,12(4):314- 329
                                    </a>
                                </li>
                                <li id="37">


                                    <a id="bibliography_18" title="Wang Changjing,Xue Jinyun.Formal derivation of a generic algorithmic program for solving a class of extremum problems[C] //Proc of the 10th ACIS Int Conf on Software Engineering,Artificial Intelligences,Networking and Parallel/Distributed Computing.Piscataway,NJ:IEEE,2009:100- 105" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Formal derivation of a generic algorithmic program for solving a class of extremum problems">
                                        <b>[18]</b>
                                        Wang Changjing,Xue Jinyun.Formal derivation of a generic algorithmic program for solving a class of extremum problems[C] //Proc of the 10th ACIS Int Conf on Software Engineering,Artificial Intelligences,Networking and Parallel/Distributed Computing.Piscataway,NJ:IEEE,2009:100- 105
                                    </a>
                                </li>
                                <li id="39">


                                    <a id="bibliography_19" title="Xue Jinyun.Genericity in PAR platform[C] //Proc of the 5th Int Workshop on Structured Object-Oriented Formal Language and Method.Berlin:Springer,2015:3- 14" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Genericity in PAR Platform">
                                        <b>[19]</b>
                                        Xue Jinyun.Genericity in PAR platform[C] //Proc of the 5th Int Workshop on Structured Object-Oriented Formal Language and Method.Berlin:Springer,2015:3- 14
                                    </a>
                                </li>
                                <li id="41">


                                    <a id="bibliography_20" title="Kang K C,Cohen S G,Hess J A,et al.Feature-oriented domain analysis (FODA) feasibility study,CMU/SEI-90-TR-21[R].Pittsburgh:Carnegie-Mellon University,1990" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Feature-Oriented Domain Analysis (FODA) Feasibility Study">
                                        <b>[20]</b>
                                        Kang K C,Cohen S G,Hess J A,et al.Feature-oriented domain analysis (FODA) feasibility study,CMU/SEI-90-TR-21[R].Pittsburgh:Carnegie-Mellon University,1990
                                    </a>
                                </li>
                                <li id="43">


                                    <a id="bibliography_21" title="Czarnecki K,Eisenecker U W.Generative Programming:Methods,Tools,and Applications[M].New York:ACM,2000" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Generative Programming: Methods, Tools, And Applications">
                                        <b>[21]</b>
                                        Czarnecki K,Eisenecker U W.Generative Programming:Methods,Tools,and Applications[M].New York:ACM,2000
                                    </a>
                                </li>
                                <li id="45">


                                    <a id="bibliography_22" title="Zhang Wei,Mei Hong.A feature-oriented domain model and its modeling process[J].Journal of Software,2003,14(8):1345- 1356 (in Chinese)(张伟,梅宏.一种面向特征的领域模型及其建模过程[J].软件学报,2003,14(8):1345- 1356)" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=RJXB200308000&amp;v=MDYwMjlqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSc0Z5L2dVcnZKTnlmVGJMRzRIdExNcDQ5RlpJUUtESDg0dlI0VDY=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[22]</b>
                                        Zhang Wei,Mei Hong.A feature-oriented domain model and its modeling process[J].Journal of Software,2003,14(8):1345- 1356 (in Chinese)(张伟,梅宏.一种面向特征的领域模型及其建模过程[J].软件学报,2003,14(8):1345- 1356)
                                    </a>
                                </li>
                                <li id="47">


                                    <a id="bibliography_23" title="Giegerich R.A systematic approach to dynamic programming in bioinformatics[J].Bioinformatics,2000,16(8):665- 677" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJOX&amp;filename=SJOXC9B2CA4A344D863D2ACF5166E3AA45F6&amp;v=MjQwNzh1SFlmT0dRbGZDcGJRMzVOcGh3NzIreEtrPU5pZkVkc0N4Yk5PLzNvczBaKzhMZUhRL3pHSVJtMHdMVFg3a3FtYzJDTU9RUU15WkNPTnZGU2lXV3I3SklGcG1hQg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[23]</b>
                                        Giegerich R.A systematic approach to dynamic programming in bioinformatics[J].Bioinformatics,2000,16(8):665- 677
                                    </a>
                                </li>
                                <li id="49">


                                    <a id="bibliography_24" title="Steffen P,Giegerich R,Giraud M.GPU parallelization of algebraic dynamic programming[C] //Proc of the 8th Int Conf on Parallel Processing and Applied Mathematics.Berlin:Springer,2009:290- 299" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=GPU Parallelization of Algebraic Dynamic Programming">
                                        <b>[24]</b>
                                        Steffen P,Giegerich R,Giraud M.GPU parallelization of algebraic dynamic programming[C] //Proc of the 8th Int Conf on Parallel Processing and Applied Mathematics.Berlin:Springer,2009:290- 299
                                    </a>
                                </li>
                                <li id="51">


                                    <a id="bibliography_25" title="Saule C,Giegerich R.Pareto optimization in algebraic dynamic programming[J].Algorithms for Molecular Biology,2015,10(1):22" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Pareto optimization in algebraic dynamic programming">
                                        <b>[25]</b>
                                        Saule C,Giegerich R.Pareto optimization in algebraic dynamic programming[J].Algorithms for Molecular Biology,2015,10(1):22
                                    </a>
                                </li>
                                <li id="53">


                                    <a id="bibliography_26" title="Searls D B,Murphy K P.Automata-theoretic models of mutation and alignment[C] //Proc of the 3rd Intelligent Systems for Molecular Biology.Menlo Park,CA:AAAI Press,1995:341- 349" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Automata-theoretic models of mutation and alignment">
                                        <b>[26]</b>
                                        Searls D B,Murphy K P.Automata-theoretic models of mutation and alignment[C] //Proc of the 3rd Intelligent Systems for Molecular Biology.Menlo Park,CA:AAAI Press,1995:341- 349
                                    </a>
                                </li>
                                <li id="55">


                                    <a id="bibliography_27" title="Wang Wendi,Tang Wen,Duan Bo,et al.Parallel accelerator design for high-throughput DNA sequence alignment with Hash-index[J].Journal of Computer Research and Development,2013,50(11):2463- 2471 (in Chinese)(王文迪,汤文,段勃,等.基于Hash索引的高通量基因序列比对并行加速技术研究[J].计算机研究与发展,2013,50(11):2463- 2471)" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201311028&amp;v=MTY4NDllUnNGeS9nVXJ2Skx5dlNkTEc0SDlMTnJvOUhiSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVo=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[27]</b>
                                        Wang Wendi,Tang Wen,Duan Bo,et al.Parallel accelerator design for high-throughput DNA sequence alignment with Hash-index[J].Journal of Computer Research and Development,2013,50(11):2463- 2471 (in Chinese)(王文迪,汤文,段勃,等.基于Hash索引的高通量基因序列比对并行加速技术研究[J].计算机研究与发展,2013,50(11):2463- 2471)
                                    </a>
                                </li>
                                <li id="57">


                                    <a id="bibliography_28" title="Ge Hongwei,Liang Yanchun.A multiple sequence alignment algorithm based on a hidden Markov model and immune particle swarm optimization[J].Journal of Computer Research and Development,2006,43(8):756- 765 (in Chinese)(葛宏伟,梁艳春.基于隐马尔可夫模型和免疫粒子群优化的多序列比对算法[J].计算机研究与发展,2006,43(8):756- 765)" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ200608005&amp;v=MjgzMjRkTEc0SHRmTXA0OUZZWVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUnNGeS9nVXJ2Skx5dlM=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[28]</b>
                                        Ge Hongwei,Liang Yanchun.A multiple sequence alignment algorithm based on a hidden Markov model and immune particle swarm optimization[J].Journal of Computer Research and Development,2006,43(8):756- 765 (in Chinese)(葛宏伟,梁艳春.基于隐马尔可夫模型和免疫粒子群优化的多序列比对算法[J].计算机研究与发展,2006,43(8):756- 765)
                                    </a>
                                </li>
                                <li id="59">


                                    <a id="bibliography_29" title="Orlando G,Raimondi D,Khan T,et al.SVM-dependent pairwise HMM:An application to protein pairwise alignments[J].Bioinformatics,2017,33(24):3902- 3908" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=SVM-dependent pairwise HMM An application to protein pairwise alignments">
                                        <b>[29]</b>
                                        Orlando G,Raimondi D,Khan T,et al.SVM-dependent pairwise HMM:An application to protein pairwise alignments[J].Bioinformatics,2017,33(24):3902- 3908
                                    </a>
                                </li>
                                <li id="61">


                                    <a id="bibliography_30" title="Li Ruiqiang,Zhu Hongmei,Ruan Jue,et al.De novo assembly of human genomes with massively parallel short read sequencing[J].Genome Research,2010,20(2):265- 272" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=De novo assembly of human genomes with massively parallel short read sequencing">
                                        <b>[30]</b>
                                        Li Ruiqiang,Zhu Hongmei,Ruan Jue,et al.De novo assembly of human genomes with massively parallel short read sequencing[J].Genome Research,2010,20(2):265- 272
                                    </a>
                                </li>
                                <li id="63">


                                    <a id="bibliography_31" title="Peng Yu,Leung H C M,Yiu S M,et al.IDBA-UD:A de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth[J].Bioinformatics,2012,28(11):1420- 1428" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth">
                                        <b>[31]</b>
                                        Peng Yu,Leung H C M,Yiu S M,et al.IDBA-UD:A de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth[J].Bioinformatics,2012,28(11):1420- 1428
                                    </a>
                                </li>
                                <li id="65">


                                    <a id="bibliography_32" title="Li Min,Liao Zhongxiang,He Yiming,et al.ISEA:Iterative seed-extension algorithm for de novo assembly using paired-end information and insert size distribution[J].IEEE/ACM Transactions on Computational Biology &amp;amp; Bioinformatics,2017,14(4):916- 925" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=ISEA:Iterative Seed-Extension Algorithm for De Novo Assembly Using Paired-End Information and Insert Size Distribution">
                                        <b>[32]</b>
                                        Li Min,Liao Zhongxiang,He Yiming,et al.ISEA:Iterative seed-extension algorithm for de novo assembly using paired-end information and insert size distribution[J].IEEE/ACM Transactions on Computational Biology &amp;amp; Bioinformatics,2017,14(4):916- 925
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JFYZ" target="_blank">计算机研究与发展</a>
                2019,56(09),1907-1917 DOI:10.7544/issn1000-1239.2019.20180835            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>基于动态规划的双序列比对算法构件设计与实现</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E7%9F%B3%E6%B5%B7%E9%B9%A4&amp;code=07872949&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">石海鹤</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%91%A8%E5%8D%AB%E6%98%9F&amp;code=39307077&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">周卫星</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E6%B1%9F%E8%A5%BF%E5%B8%88%E8%8C%83%E5%A4%A7%E5%AD%A6%E8%AE%A1%E7%AE%97%E6%9C%BA%E4%BF%A1%E6%81%AF%E5%B7%A5%E7%A8%8B%E5%AD%A6%E9%99%A2&amp;code=0240878&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">江西师范大学计算机信息工程学院</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>双序列比对算法是生物信息学中的一个关键算法,广泛应用于序列相似性分析以及基因组序列数据库搜索.现有研究主要针对特定应用问题优化和使用相对应比对算法,缺乏高抽象层算法框架的细致研究,在一定程度上导致了序列比对算法的冗余性以及人为选择算法可能造成的误差等问题,也使得人们难以有效地了解算法结构.通过深入分析基于动态规划的双序列比对算法(dynamic programming-based pairwise sequence alignment algorithm, DPPSAA)领域,在建立该算法领域的特征模型以及对应算法构件交互模型基础上,利用PAR平台形式化实现双序列比对算法构件库,并装配生成具体算法,保证了形式化装配算法的可靠性,为序列相似性分析算法应用提供了一条有价值的参考途径.最后,利用PAR平台C++程序生成系统将组装的比对算法转换为C++程序,运行结果表明DPPSAA算法构件库具有一定的实用性.</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8F%8C%E5%BA%8F%E5%88%97%E6%AF%94%E5%AF%B9%E7%AE%97%E6%B3%95&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">双序列比对算法;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8A%A8%E6%80%81%E8%A7%84%E5%88%92&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">动态规划;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%89%B9%E5%BE%81%E6%A8%A1%E5%9E%8B&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">特征模型;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%9E%84%E4%BB%B6%E4%BA%A4%E4%BA%92%E6%A8%A1%E5%9E%8B&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">构件交互模型;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=PAR%E5%B9%B3%E5%8F%B0&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">PAR平台;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    *石海鹤,haiheshi@jxnu.edu.cn;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-12-18</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金项目(61662035,61762049,61862033);</span>
                    </p>
            </div>
                    <h1><b>Design and Implementation of Pairwise Sequence Alignment Algorithm Components Based on Dynamic Programming</b></h1>
                    <h2>
                    <span>Shi Haihe</span>
                    <span>Zhou Weixing</span>
            </h2>
                    <h2>
                    <span>School of Computer and Information Engineering, Jiangxi Normal University</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>Pairwise sequence alignment algorithm is a key algorithm in bioinformatics, and it is widely used in sequence similarity analysis and genomic sequence database searching. The existing study mainly focuses on the optimization and use of relative alignment algorithms for specific application problems. To some extent, those studies lack a high-level algorithm framework that not only has led to the redundancy of the sequence alignment algorithms and the possible errors caused by the artificial selection algorithm, but also made the structure of algorithm difficult to be understood effectively. Through in-depth analysis of the dynamic programming-based pairwise sequence alignment algorithms domain(DPPSAA), a domain feature model and the corresponding algorithm component interactive model have been established, a DPPSAA component library has been formally implemented by the PAR platform, and a concrete algorithm has been assembled, thus the reliability of the algorithm for formal assembly is guaranteed, moreover a valuable reference for the application of sequence similarity analysis algorithms is provided. Finally, the C++ program generation system of PAR platform is used to transform the assembly alignment algorithm into C++ program and the running results show that the dynamic programming-based pairwise sequence alignment algorithm component library has certain practicability.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=pairwise%20sequence%20alignment%20algorithm&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">pairwise sequence alignment algorithm;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=dynamic%20programming&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">dynamic programming;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=feature%20model&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">feature model;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=component%20interactive%20model&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">component interactive model;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=PAR%20platform&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">PAR platform;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                            <b>Author：</b>
                                                        <span>
                                    Shi Haihe,born in 1979.PhD,professor. Member of CCF. Her main research interests include bioinformatics,formal method and software engineering.&lt;image id="259" type="formula" href="images/JFYZ201909012_25900.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                                <span>
                                    Zhou Weixing, born in 1994. Master. Student member of CCF. His main research interests include bioinformatics, formal method.&lt;image id="261" type="formula" href="images/JFYZ201909012_26100.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                    </p>
                                    <p><b>Received：</b> 2018-12-18</p>
                                    <p>
                            <b>Fund：</b>
                                                        <span>supported by the National Natural Science Foundation of China(61662035,61762049,61862033);</span>
                    </p>
            </div>


        <!--brief start-->
                        <div class="p1">
                    <p id="67">序列比对是一种通过排列基因组序列来识别序列相似性区域,从而获得待比对序列之间的功能、结构或进化关系的技术.随着人类基因组计划的实施,测序技术的发展产生了大量的有关生物分子的原始序列数据,例如,Illumina HiSeqX Ten在3天之内可以产生大约30亿个2×150 bp的双端测序数据<citation id="241" type="reference"><link href="3" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>.面对如此丰富的基因组序列数据,如何高效处理和分析这些丰富的基因组序列数据,如比较两序列之间的相似区域和保守性位点,寻求序列同源结构,揭示生物遗传、变异和进化关系等,成为了序列比对算法研究的主要动力之一.</p>
                </div>
                <div class="p1">
                    <p id="68">研究表明,计算机的微处理性能和存储设备容量平均每18～24个月增长1倍,而基因组测序数据则平均4～5月就增长1倍,这成为了高性能计算发展史中一个前所未有的挑战,因此序列比对算法的时空开销和精准度成为生物序列比对过程中的关键因素之一.由于序列比对属于NP-hard问题,而运筹学中动态规划策略多用于解决多阶段决策过程最优化问题,因此,动态规划策略被广泛应用于序列比对算法之中.这里我们主要针对基于动态规划的双序列比对算法(dynamic programming-based pairwise sequence alignment algorithm, DPPSAA)领域开展研究.</p>
                </div>
                <div class="p1">
                    <p id="69">在双序列比对算法中最为经典的确定性双序列比对算法有基于全局比对的NW算法<citation id="242" type="reference"><link href="5" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>(Needleman-Wunsch algorithm)、基于局部比对的SW算法<citation id="243" type="reference"><link href="7" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>(Smith-Waterman algorithm)和准全局比对算法.在后续的研究中,发展出了基于以上算法进行优化<citation id="244" type="reference"><link href="9" rel="bibliography" /><link href="11" rel="bibliography" /><link href="13" rel="bibliography" /><sup>[<a class="sup">4</a>,<a class="sup">5</a>,<a class="sup">6</a>]</sup></citation>或作为主要比对策略<citation id="245" type="reference"><link href="15" rel="bibliography" /><link href="17" rel="bibliography" /><link href="19" rel="bibliography" /><sup>[<a class="sup">7</a>,<a class="sup">8</a>,<a class="sup">9</a>]</sup></citation>的系列算法.</p>
                </div>
                <div class="p1">
                    <p id="70">目前,比对算法的研究大部分集中于序列相似性分析领域中的特定问题<citation id="246" type="reference"><link href="21" rel="bibliography" /><link href="23" rel="bibliography" /><link href="25" rel="bibliography" /><link href="27" rel="bibliography" /><sup>[<a class="sup">10</a>,<a class="sup">11</a>,<a class="sup">12</a>,<a class="sup">13</a>]</sup></citation>或者特定算法优化<citation id="247" type="reference"><link href="29" rel="bibliography" /><link href="31" rel="bibliography" /><link href="33" rel="bibliography" /><sup>[<a class="sup">14</a>,<a class="sup">15</a>,<a class="sup">16</a>]</sup></citation>,而较少面向于整个问题域,难以得到一个具有更高抽象层次且适用于整个序列相似性分析领域的算法构件库,在一定程度上导致了序列比对算法的冗余以及人为选择算法可能造成的误差等问题,也使得人们难以有效地了解算法结构,无法保证算法的正确使用,在一定程度上降低了序列相似性分析结果的准确性.由于现有算法的专用性和低抽象性,不仅导致研究人员需要花费大量时间去学习和使用该类算法,降低了算法的可维护性和复用性,而且难以定位和解决算法产生的错误,加重了序列相似性分析的负担.</p>
                </div>
                <div class="p1">
                    <p id="71">通过深入分析DPPSAA领域,本文设计和实现了该领域抽象泛型算法构件库,提高了序列相似性分析领域算法可靠性和开发效率.首先对DPPSAA领域进行特征分析,提取出其中的通用和可变特征以及它们之间的约束与依赖关系,建立了一致的领域构件模型,基于此设计了算法构件交互模型,进一步利用新型高可靠软件开发平台PAR<citation id="248" type="reference"><link href="35" rel="bibliography" /><link href="37" rel="bibliography" /><link href="39" rel="bibliography" /><sup>[<a class="sup">17</a>,<a class="sup">18</a>,<a class="sup">19</a>]</sup></citation>中高抽象程序设计语言Apla(abstract programming language)进行形式化实现,形成了一个高抽象DPPSAA构件库,以期自动或半自动装配构件产生特定领域序列比对算法,甚至于装配出更为高效的新型序列比对算法.</p>
                </div>
                <h3 id="72" name="72" class="anchor-tag"><b>1 比对算法的建模过程</b></h3>
                <h4 class="anchor-tag" id="73" name="73"><b>1.1 领域分析</b></h4>
                <div class="p1">
                    <p id="74">面向特征的领域分析<sup></sup><citation id="249" type="reference"><link href="41" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>(feature oriented domain analysis, FODA)是由软件工程研究所(Software Engineering Institute, SEI)于1990年首次提出的一种领域分析方法,主要包含上下文分析以及特征建模2个阶段<citation id="250" type="reference"><link href="43" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>.其中上下文分析主要由界定待研究领域需求范围以及领域的输出与输入构成,该阶段的成功实施能够保证特征建模过程中被选取特征的有效性和可靠性.特征建模阶段则包含了特征选取与建立、特征关系描述以及特征模型建立等过程,在该阶段需要对被研究领域进行分析,充分了解其功能性特征和非功能性特征,并找出特征之间的约束和依赖关系以及优先级等附加信息,进而获得该领域的主次要特征,建立一种具有更高抽象层次的特征模型或者构件框架.</p>
                </div>
                <div class="p1">
                    <p id="75">特征建模阶段主要包含以下4种特征:强制特征(mandatory)、可变特征(optional)、OR特征、XOR特征.强制特征表示领域内的所有实例都必须包含的特征,可变特征表示领域中实例的一个可选特征,XOR特征表示领域中实例有且只能选择一组XOR特征中的一个特征,OR特征表示领域中实例至少包含了一组OR特征中的一个特征,特征表示如图1所示.特征建模清晰地刻画了特征模型构建过程中所需的各类特征,是识别和捕捉差异性和可变性的关键技术.</p>
                </div>
                <div class="area_img" id="76">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909012_076.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 特征图" src="Detail/GetImg?filename=images/JFYZ201909012_076.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 特征图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909012_076.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 1 Feature graph</p>

                </div>
                <div class="p1">
                    <p id="77">图1中强制特征被末端为实心圆的边所指向,可变特征被末端为空心圆的边所指向,XOR特征被一组用弧连接的边所指向,OR特征被一组用实心弧连接的边所指向.这4种特征是特征建模中的主要属性,也是特征模型的重要组成部分,代表了领域中的通用和可变属性,同时它们之间的约束以及依赖关系则需通过特征内在的层次结构关系、领域业务逻辑设计的约束关系以及运行时依赖关系来表示.</p>
                </div>
                <h4 class="anchor-tag" id="78" name="78"><b>1.2 特征建模流程</b></h4>
                <div class="p1">
                    <p id="79">以FODA为基础,我们在其后增添了领域实现阶段,即将建立的特征作为构件进行构件交互模式设计和形式化实现,建立抽象构件库.需要注意的是,由于建立的构件之间的交互性,程序人员使用C,C++或Java等低抽象层次高级编程语言进行代码编程时,不仅增加了特征之间的关联性与复杂性,而且难以让使用者对整个构件库有具体和全面的了解.因此需要使用一种具有高抽象性的编程语言来实现这些构件,以至不用在意算法构件的具体实现细节,而能够清晰地了解这些算法构件的功能以及它们之间的交互关系.例如在本文中主要利用Apla语言来实现DPPSAA领域构件.</p>
                </div>
                <div class="p1">
                    <p id="80">基于以上介绍,对具体领域进行分析时,领域构件库建立流程图如图2所示:</p>
                </div>
                <div class="area_img" id="81">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909012_081.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 领域分析流程图" src="Detail/GetImg?filename=images/JFYZ201909012_081.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 领域分析流程图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909012_081.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 2 Flow chart of domain analysis</p>

                </div>
                <h4 class="anchor-tag" id="82" name="82"><b>1.3 DPPSAA领域特征建模</b></h4>
                <div class="p1">
                    <p id="83">特征模型描述了领域内实例的通用和可变特征以及它们之间的约束与依赖关系,是在特征建模过程中构建的,一般由描述特征层次关系的特征图和相关的文本描述组成.其中特征图通常使用具有不同层次的树状结构来表示,在该结构中包含了1组节点、1组节点之间的定向边以及两边之间的关系描述.其中1个节点代表一个特征,具有唯一的标识符;节点之间的定向边将特征连结成树结构;两边之间的关系描述则代表特征之间的关系,是对特征节点的划分.同时特征模型中的文本描述则表示了特征的语义描述、原理以及约束和默认依赖规则等信息.</p>
                </div>
                <div class="p1">
                    <p id="84">因此,针对基于动态规划的双序列比对算法领域,建模过程有3方面:</p>
                </div>
                <div class="p1">
                    <p id="85">1) 上下文分析.该算法领域的范围被限制为一种在生物序列相似性分析领域中以动态规划为主要罚分策略、双序列比对为主要比对方式的算法形式.</p>
                </div>
                <div class="p1">
                    <p id="86">2) 特征建模.我们利用特征建模方法<citation id="251" type="reference"><link href="45" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>对该领域进行特征建模,即从分析领域中的服务(service)、功能(function)以及行为(behavior)特点入手构建特征模型.比对操作服务是该领域中的核心服务,通过控制序列比对过程中的比对方式和各算法特征之间的执行优先级以及组合方式来实现使用者定义的序列比对算法.通过分析双序列比对算法中的一些主要执行步骤,可以将该比对操作服务进一步划分为检查序列合法性、得分矩阵操作、动态规划算法方式选择、记住得分来源、回溯和比对结果输出等功能,同时得分矩阵初始化、动态规划方式选择以及检查序列合法性作为3个必选的功能.在上述分析的基础上,输出方式是比对结果输出的显著行为特点,包括比对序列输出和比对得分输出两种主要行为特点,且二者属于OR特征.动态规划方式可以分为全局动态规划方式、准全局动态规划方式、局部动态规划方式3种,三者为XOR特征.我们根据上述分析对该领域建立了特征模型,如图3所示:</p>
                </div>
                <div class="area_img" id="87">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909012_087.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 DPPSAA特征模型" src="Detail/GetImg?filename=images/JFYZ201909012_087.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 DPPSAA特征模型  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909012_087.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 3 DPPSAA feature model</p>

                </div>
                <div class="p1">
                    <p id="88">3) 领域实现.该过程针对已建立的特征模型进行DPPSAA领域算法构件交互设计并使用Apla语言形式化实现.</p>
                </div>
                <h3 id="89" name="89" class="anchor-tag"><b>2 比对算法构件的设计与实现</b></h3>
                <h4 class="anchor-tag" id="90" name="90"><b>2.1 DPPSAA领域算法构件交互设计</b></h4>
                <div class="p1">
                    <p id="91">不同构件通过交互实现完整的算法构件库,而算法构件的交互则需要由其包含的特征之间的约束以及依赖关系来体现.因此针对1.3节建立的特征模型,我们对DPPSAA领域的算法构件交互模型进行了设计.</p>
                </div>
                <div class="p1">
                    <p id="92">通过分析整个DPPSAA领域得知,算法主要包括3个主要变化过程特征:得分矩阵操作、动态规划算法方式以及比对结果输出.因此,我们将特征模型中的这些特征以及检查序列合法性作为主要构件,其他特征以及相关数据结构作为辅助构件,并根据其优先级建立了构件间的交互模型,如图4所示:</p>
                </div>
                <div class="area_img" id="93">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909012_093.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 算法构件交互模型" src="Detail/GetImg?filename=images/JFYZ201909012_093.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 算法构件交互模型  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909012_093.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 4 Algorithm components interaction model</p>

                </div>
                <div class="p1">
                    <p id="94">图4中实线所连接的节点表示为DPPSAA领域中所必须含有的基本构件,即对应于特征模型中的4个强制特征,实线箭头所代表的方向表示4个构件的执行优先级为由高到低;点连线箭头表明在算法执行过程中2个构件之间的交互,如在图4中,当比对结果输出构件选择得分输出时,则需要使用得分矩阵操作构件中的获取元素得分操作;虚线箭头则代表在算法组装过程中所需的数据、结构以及关联操作等,如在得分矩阵操作构件中需要使用2个抽象数据类型(abstract data type, ADT):罚分模型ADT和得分矩阵元素ADT,且罚分模型ADT同时作用于动态规划算法方式构件等.</p>
                </div>
                <h4 class="anchor-tag" id="95" name="95"><b>2.2 Apla形式化实现</b></h4>
                <div class="p1">
                    <p id="96">Apla语言可以直接使用抽象数据类型和抽象过程编写程序,因此能够更抽象地描述算法问题,并易于对其进行正确性验证,保证了程序正确性和可靠性.</p>
                </div>
                <div class="p1">
                    <p id="97">PAR方法和PAR平台包含循环不变式的新定义和新的开发策略、统一的算法程序设计方法、新的算法表示方法、自定义算法设计语言和抽象程序设计语言等关键技术.它集成涵盖了泛型、生成式、模型驱动和构件组装等新型软件开发技术,其系列程序自动生成系统可将一个正确的Apla程序自动转换成C++,Java,Delphi等高级语言程序.因此,本文利用该语言形式化实现并建立了DPPSAA领域算法构件库,不仅可高抽象表示算法程序的功能特性与非功能特性,而且能够直观地展示各算法构件之间的约束以及依赖关系.这样既减少了各构件之间的干扰,降低了算法程序的复杂性,提高了算法构件安全性,又消除了传统算法设计方式中算法与数据难以分离的问题,提高了构件装配产生算法的可复用性和可维护性,同时使用该算法构件库时,我们只需关注算法功能即可,而不用在意具体构件实现细节,从而提高了算法设计效率.</p>
                </div>
                <div class="p1">
                    <p id="98">下面展示了对DPPSAA领域构件的Apla程序实现.</p>
                </div>
                <h4 class="anchor-tag" id="99" name="99">1) 罚分模型以及得分矩阵元素结构设计</h4>
                <div class="p1">
                    <p id="100">首先我们将罚分模型设置为一个ADT,这里我们为解释方便,只使用固定空位罚分策略,与扩展罚分时操作类似.其中罚分模型的ADT定义为</p>
                </div>
                <div class="area_img" id="262">
                                <img alt="" src="Detail/GetImg?filename=images/JFYZ201909012_26200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="106">其中sometype为Apla语言中的关键字,用来定义类型变量.其中的<i>match</i>,<i>mismatch</i>,<i>space</i>分别表示罚分模型中的匹配罚分值、错配罚分值以及空位罚分值.</p>
                </div>
                <div class="p1">
                    <p id="107">在得分矩阵中,由于序列比对结果输出时需要进行回溯操作,因此我们将得分矩阵中的元素定义为一个ADT,命名为<i>H</i>_<i>Struct</i>,该ADT中包含了整型变量<i>value</i>和boolean型的数组<i>dp</i>_<i>direct</i>,前者表示2个字符比对得分值,后者表示该得分值的来源(其中true代表对应得分来源为真,false代表为假).并且数组内的元素含义如图5所示,4个方框分别代表得分矩阵中的4个元素,3个箭头分别表示(<i>i</i>,<i>j</i>)中的得分来源为上、左和对角元素,分别用数组<i>dp</i>_<i>direct</i>下标0,1,2对应的数组元素来表示.</p>
                </div>
                <div class="area_img" id="108">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909012_108.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图5 得分矩阵元素的得分来源" src="Detail/GetImg?filename=images/JFYZ201909012_108.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图5 得分矩阵元素的得分来源  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909012_108.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 5 Score source of score matrix element</p>

                </div>
                <div class="p1">
                    <p id="109">该ADT定义为</p>
                </div>
                <div class="area_img" id="263">
                                <img alt="" src="Detail/GetImg?filename=images/JFYZ201909012_26300.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="263">
                                <img alt="" src="Detail/GetImg?filename=images/JFYZ201909012_26301.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="114" name="114">2) 得分矩阵操作</h4>
                <div class="p1">
                    <p id="115">该构件被定义为一个ADT类型,因为在不同的DPPSAA中得分矩阵初始化方式不同,因此在该ADT内部包含了一个泛型子程序<i>Memory</i>_<i>Score</i>_<i>of</i>_<i>Matrix</i>,并将<i>Init</i>_<i>score</i>_<i>matrix</i>方法作为它的泛型参数,使得泛型子程序可以支持实例化具有不同得分矩阵初始化方式的比对算法,即当使用不同的方法参数实例化<i>Init</i>_<i>score</i>_<i>matrix</i>时,将返回不同的得分矩阵.同时在该ADT中还包含了一些常用的得分矩阵操作,如求矩阵最大得分、矩阵元素取值操作和赋值操作等.</p>
                </div>
                <div class="area_img" id="264">
                                <img alt="" src="Detail/GetImg?filename=images/JFYZ201909012_26400.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="124">其中该ADT类型名为<i>score</i>_<i>matrix</i>_<i>mani</i>,且带有一个类型参数<i>elem</i>;<i>apply</i>_<i>memory</i>泛型子程序的作用是根据整型变量<i>length</i>_<i>s</i>和<i>length</i>_<i>t</i>值为<i>score</i>_<i>matrix</i>_<i>mani</i>动态分配内存空间;函数<i>Max</i>_<i>score</i>_<i>of</i>_<i>Matrix</i>和<i>the</i>_<i>Last</i>_<i>element</i>_<i>score</i>分别表示获取<i>score</i>_<i>matrix</i>_<i>mani</i>中得分最大值以及最后一个元素的得分值;<i>get</i>_<i>value</i>以及<i>set</i>_<i>value</i>分别为获取和设置<i>score</i>_<i>matrix</i>_<i>mani</i>中元素得分值,(<i>i</i>,<i>j</i>)(0≤<i>i</i>≤<i>length</i>_<i>s</i>,0≤<i>j</i>≤<i>length</i>_<i>t</i>表示对应元素的下标,<i>length</i>_<i>s</i>和<i>length</i>_<i>t</i>分别表示两比对序列的长度.</p>
                </div>
                <h4 class="anchor-tag" id="125" name="125">3) 动态规划算法方式选择</h4>
                <div class="p1">
                    <p id="126">该构件也被定义为泛型ADT,可以支持不同的序列比对算法所使用的动态规划得分模式,得分模式的变化主要依靠泛型子程序<i>dp</i>_<i>align</i>_<i>score</i>来实现,同时该ADT类型中还包括求2字符的最大比对得分操作<i>max</i>_<i>score</i>_<i>of</i>_<i>char</i>,函数中的3个参数分别表示不同来源的罚分值.</p>
                </div>
                <div class="area_img" id="265">
                                <img alt="" src="Detail/GetImg?filename=images/JFYZ201909012_26500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="131">这里<i>elemMatrix</i>是一个类型参数;函数 <i>set</i>_<i>and</i>_<i>remember</i>是泛型子程序<i>dp</i>_<i>align</i>_<i>score</i>的泛型参数,该函数的功能为记录得分矩阵中各元素的得分值以及得分来源,可以被用于各种动态规划罚分模型的实例化.</p>
                </div>
                <h4 class="anchor-tag" id="132" name="132">4) 检查序列合法性</h4>
                <div class="p1">
                    <p id="133">检查序列是否属于字符集{A,T,C,G}(默认为DNA序列,如果为其他序列,加入对应表示字符即可,如果为RNA序列,则将G换为U即可).</p>
                </div>
                <div class="p1">
                    <p id="134">procedure <i>check</i>(<i>s</i>,<i>t</i>:String).</p>
                </div>
                <div class="p1">
                    <p id="135">其中<i>s</i>和<i>t</i>都为字符串类型.</p>
                </div>
                <h4 class="anchor-tag" id="136" name="136">5) 记住得分来源</h4>
                <div class="p1">
                    <p id="137">该构件表示记住(<i>i</i>,<i>j</i>)处得分的来源,即将(<i>i</i>,<i>j</i>)处中对应的方向标志赋值为true.其为回溯阶段输出序列比对结果提供支持.</p>
                </div>
                <div class="p1">
                    <p id="138">procedure <i>rmb</i>_<i>source</i>(<i>i</i>:integer;<i>j</i>:integer).</p>
                </div>
                <h4 class="anchor-tag" id="139" name="139">6) 回溯</h4>
                <div class="p1">
                    <p id="140">该构件的Apla语言定义为</p>
                </div>
                <div class="p1">
                    <p id="141">procedure <i>traceback</i>(proc <i>print</i>_<i>align</i>();proc <i>print</i>_<i>extrude</i>()=NULL).</p>
                </div>
                <div class="p1">
                    <p id="142">在回溯过程中,一般由短序列比对区间回溯和两端突出回溯组成,其中短序列比对区间回溯表示从2序列匹配的第1个序列字符开始,到最后匹配的字符结束位置之间的字符区间序列输出,两端突出回溯则表示从头开始到第1个匹配字符的前一个字符区间序列输出或者最后一个匹配字符的下一个字符到最后所有字符序列对应输出.在<i>trackback</i>泛型子程序中分别由<i>print</i>_<i>align</i>和<i>print</i>_<i>extrude</i>表示,并且在默认配置下<i>print</i>_<i>extrude</i>为空,即为全局比对.</p>
                </div>
                <h4 class="anchor-tag" id="143" name="143">7) 比对结果输出</h4>
                <div class="p1">
                    <p id="144">这里将该构件定义成一个泛型子程序.</p>
                </div>
                <div class="p1">
                    <p id="145">procedure <i>op</i>_<i>mode</i>(func <i>finally</i>_<i>score</i>():integer;proc <i>traceback</i>(proc <i>print</i>_<i>align</i>();proc <i>print</i>_<i>extrude</i>()=NULL)).</p>
                </div>
                <div class="p1">
                    <p id="146">在此泛型子程序定义中,函数<i>finally</i>_<i>score</i>功能为输出最终比对得分.</p>
                </div>
                <h4 class="anchor-tag" id="147" name="147">8) 比对操作</h4>
                <div class="p1">
                    <p id="148">为了能够实现现有的序列比对算法,需要对上述算法构件进行人工装配,因此将比对操作服务定义为一个泛型子程序,并将各功能作为其参数,使之能够支持装配产生DPPSAA.</p>
                </div>
                <div class="p1">
                    <p id="149">procedure <i>align</i>_<i>manipulation</i>(<i>op</i>_<i>mode</i>(func <i>finally</i>_<i>score</i>():integer;proc <i>traceback</i>(proc <i>print</i>_<i>align</i>();proc <i>print</i>_<i>extrude</i>()=NULL));ADT <i>dp</i>_<i>mode</i>(<i>eM</i>:<i>elemMatrix</i>);sometype <i>elemMatrix</i>;<i>result</i>:boolean;<i>eM</i>:<i>elemMatrix</i>;<i>s</i>:String;<i>t</i>:String)).</p>
                </div>
                <div class="p1">
                    <p id="150">该算法构件<i>align</i>_<i>manipulation</i>主要包含4个参数:比对输出构件<i>op</i>_<i>mode</i>、回溯构件<i>traceback</i>、自定义泛型ADT <i>dp</i>_<i>mode</i>以及待比对序列<i>s</i>和<i>t</i>.其中<i>elemMatrix</i>表示为一个类型参数,并且后面4个变量参数为在主程序中实例化参数所需代入的.这样我们就可以通过手工装配该子程序,以达到实现相应比对算法的目的.</p>
                </div>
                <h3 id="151" name="151" class="anchor-tag"><b>3 NW算法的装配实现</b></h3>
                <div class="p1">
                    <p id="152">通过第2节的介绍,可以较清晰地了解到整个DPPSAA领域构件库的建立过程,下面我们利用上述构件库来装配实现基于全局的双序列比对算法NW,程序为</p>
                </div>
                <div class="area_img" id="266">
                                <img alt="" src="Detail/GetImg?filename=images/JFYZ201909012_26600.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="266">
                                <img alt="" src="Detail/GetImg?filename=images/JFYZ201909012_26601.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="266">
                                <img alt="" src="Detail/GetImg?filename=images/JFYZ201909012_26602.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="207">在上面程序中,过程①为得分矩阵初始化的不同实例化方式;代码块②和④由于得分矩阵初始化过程占用篇幅过大,因此用…表示;过程③则为记录比对得分值以及得分来源的实例化;⑤则表示NW算法的动态规划算法的实例化;泛型子程序⑥则表示在步骤⑦实例化NW算法对象时内部所要执行的算法构件间的关联操作;⑧以下的代码块为主程序.</p>
                </div>
                <h3 id="208" name="208" class="anchor-tag"><b>4 实验及其结果分析</b></h3>
                <div class="p1">
                    <p id="209">目前,由于Apla语言无法在PAR平台上直接运行,本节利用PAR平台C++程序生成系统,将组装NW算法过程中所需的Apla算法构件代码转换为相对应的C++代码.</p>
                </div>
                <div class="p1">
                    <p id="210">Apla代码中只包含数据成员的ADT算法构件被转换为C++中的结构体,如<i>penaltyMatrix</i>以及<i>H</i>_<i>Struct</i>等,其结果为</p>
                </div>
                <div class="area_img" id="267">
                                <img alt="" src="Detail/GetImg?filename=images/JFYZ201909012_26700.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="222">含有数据成员和成员函数的ADT则被转换为C++函数中的类,如<i>score</i>_<i>matrix</i>_<i>mani</i>,<i>dp</i>_<i>mode</i>等,其中大括号内的省略号表示函数体,且<i>Sequence</i>类将2个待比对序列作为其数据成员.其转换的部分结果如图6所示.</p>
                </div>
                <div class="p1">
                    <p id="223">同时,Apla代码中的泛型子程序和泛型函数等被转换为C++中独立的类成员函数,降低构件间的耦合性.其中,主调函数与主调函数泛型参数之间的关系被转换为C++中的函数指针,即将泛型参数转换为主调函数的指针函数参数,从而具有与Apla程序同样的多态性特征,如<i>traceback</i>泛型子程序的C++转换结果如图7所示.</p>
                </div>
                <div class="p1">
                    <p id="224">可将Apla算法构件代码转换成可执行的C++代码,最后利用转换后的算法构件进行手工装配NW算法(即将服务<i>align</i>_<i>manipulation</i>转换为主函数)如图8所示,并输出代码结果如图9所示.</p>
                </div>
                <div class="area_img" id="225">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909012_225.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图6 ADT转换结果" src="Detail/GetImg?filename=images/JFYZ201909012_225.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图6 ADT转换结果  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909012_225.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 6 Result of ADT transformation</p>

                </div>
                <div class="area_img" id="226">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909012_226.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图7 泛型子程序转换结果" src="Detail/GetImg?filename=images/JFYZ201909012_226.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图7 泛型子程序转换结果  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909012_226.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 7 Result of generic program transformation</p>

                </div>
                <div class="area_img" id="227">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909012_227.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图8 NW的C++装配过程" src="Detail/GetImg?filename=images/JFYZ201909012_227.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图8 NW的C++装配过程  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909012_227.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 8 C++ assembly process of NW</p>

                </div>
                <div class="area_img" id="228">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201909012_228.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图9 NW比对结果" src="Detail/GetImg?filename=images/JFYZ201909012_228.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图9 NW比对结果  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201909012_228.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 9 NW alignment result</p>

                </div>
                <div class="p1">
                    <p id="229">在PAR平台的支撑下,我们以半自动的方式将上述Apla语言编写的DPPSAA算法构件程序转换成C++代码,并装配形成NW算法.经实际运行NW算法检测,算法的运行结果与原本NW算法结果一致.使用DPPSAA领域构件装配形成的具体双序列比对算法,不但提高了装配算法程序的可靠性、执行效率以及可维护性,而且可以根据客户需求进行手工装配形成指定算法,增强了DPPSAA算法构件的通用性.</p>
                </div>
                <div class="p1">
                    <p id="230">在下一步研究工作中,为实现自动化装配DPPSAA领域算法,我们将使用产生式编程的相关方法学来开展工作,为进一步装配生成奠定了基础,可望装配形成一种相较于现有算法具有更高执行效率和更低内存消耗的新型双序列比对算法.</p>
                </div>
                <h3 id="231" name="231" class="anchor-tag"><b>5 相关研究比较</b></h3>
                <div class="p1">
                    <p id="232">针对序列比对算法研究的不同方面,国内外学者通过使用并融合相关方法学技术开展了较多研究工作,可分为3个方面:</p>
                </div>
                <div class="p1">
                    <p id="233">1) 基于代数方法的研究.比勒费尔德大学的Giegerich等人<citation id="253" type="reference"><link href="47" rel="bibliography" /><link href="49" rel="bibliography" /><link href="51" rel="bibliography" /><sup>[<a class="sup">23</a>,<a class="sup">24</a>,<a class="sup">25</a>]</sup></citation>提出了一种面向系统族的基于代数结构的新型动态规划构造算法,通过利用代数数据类型概念将动态规划算法过程形式化定义为2个阶段:识别阶段和评价阶段.在具体应用过程中,算法通过迭代进行这2个阶段,可为序列比对算法领域内待生成算法提供一种更具一般性的生成范式,简化了算法开发过程,提高了生成算法的正确性和可靠性.</p>
                </div>
                <div class="p1">
                    <p id="234">2) 基于有限状态机的研究.宾夕法尼亚大学的Searls等人<citation id="254" type="reference"><link href="53" rel="bibliography" /><sup>[<a class="sup">26</a>]</sup></citation>提出了一种基于有限状态机和具有自适应加权的序列比对算法生成方法.该算法利用有限状态机模型形式化表示序列比对过程中的罚分、比对以及动态规划过程,并对比对过程中的不同碱基字符比对结果(匹配、错配以及空位)进行自适应加权,能够快速地生成新型序列比对算法.算法能够支持可视化编程.</p>
                </div>
                <div class="p1">
                    <p id="235">3) 基于算法优化的研究.算法优化是指为了提高已有算法处理问题能力而进行研究的过程.文献<citation id="255" type="reference">[<a class="sup">27</a>]</citation>采用经典的脉动阵列结构的并行加速器体系结构来提高序列加载、比对和结果收集效率的数据通路,并且利用Hash索引优化技术过滤比对过程中的非相似行序列,增强了序列比对算法的性能.为了提高序列比对算法的准确度,文献<citation id="256" type="reference">[<a class="sup">28</a>,<a class="sup">29</a>]</citation>等利用改进的隐马尔可夫模型对序列比对算法进行优化.</p>
                </div>
                <h3 id="236" name="236" class="anchor-tag"><b>6 结束语</b></h3>
                <div class="p1">
                    <p id="237">序列比对作为生物序列分析中的关键问题,其算法及应用研究受到广泛关注,然而,尚未有工作将其视为一个专门领域从高抽象层次开展研究,从而提高算法可靠性和开发效率、降低算法次优解及误差等问题出现的概率.我们采用特征建模,分析和提取出基于动态规划双序列比对算法领域的通用以及可变特征,并利用高抽象语言进行形式化实现,以期能够以自动或者半自动方式进行形式化组装生成特定问题的求解算法,从而降低人工选择算法进行序列相似性分析的错误发生率以及时间开销,提高算法执行效率,甚至于装配出一种更为高效的基于动态规划的新型序列比对算法.</p>
                </div>
                <div class="p1">
                    <p id="238">本文首先提出了一种领域建模过程,包括上下文分析、特征建模以及领域实现3个主要阶段;其次通过分析DPPSAA领域的一般执行过程,抽取出算法的通用以及可变特征,并建立领域特征模型;然后对建立的构件特征模型进行构件交互设计,同时利用Apla语言形式化实现这些算法构件,并通过手工装配生成了NW算法实例;最后利用PAR平台C++程序生成系统将Apla程序转换为C++可运行程序,转换结果以及算法运行结果展示了其具有一定的实用性.由于Apla的高抽象性,建立的一系列泛型构件,如<i>dp</i>_<i>mode</i>,<i>result</i>_<i>op</i>等,保证了构件装配后的算法多样性,也能够较好地展示出算法特征之间的联系.</p>
                </div>
                <div class="p1">
                    <p id="239">我们对整个DPPSAA领域进行了精确分析,对其各功能特性与非功能特性有充分的了解,同时在算法设计过程中建立了特征模型以及构件交互设计,从而有利于算法构件库的学习和使用;另外,使用高抽象语言Apla形式化实现算法构件,不仅易于对Apla算法程序进行正确性验证,保证算法构件库的可靠性,而且在使用过程中可利用泛型编程机制快速发现和定位算法中的错误,提高算法构件库的鲁棒性.</p>
                </div>
                <div class="p1">
                    <p id="240">本文研究DPPSAA领域的主要方法学思想和成果,不仅适用于DNA序列比对算法,理论上对于一些其他的生物序列分析算法领域也具有参考价值和实用价值,例如基因组装过程中的基于de bruijn graph结构的组装算法<citation id="257" type="reference"><link href="61" rel="bibliography" /><link href="63" rel="bibliography" /><link href="65" rel="bibliography" /><sup>[<a class="sup">30</a>,<a class="sup">31</a>,<a class="sup">32</a>]</sup></citation>.下一步的研究尚需扩充本文结果在生物序列分析算法领域的应用范围,同时基于PAR平台实现算法构件库的自动或半自动装配.</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="3">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=HiSeqX instrument performance parameters">

                                <b>[1]</b>Illumina.HiSeqX instrument performance parameters[EB/OL].2016[2018-09-26].https://www.illumina.com/systems/sequencing-platforms/hiseq-x/specifications.html
                            </a>
                        </p>
                        <p id="5">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A general method applicable to the search for similarities in the amino acid sequence of two proteins">

                                <b>[2]</b>Needleman S B,Wunsch C D.A general method applicable to the search for similarities in the amino acid sequence of two proteins[J].Journal of Molecular Biology,1970,48(3):443- 453
                            </a>
                        </p>
                        <p id="7">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Identification of common molecular subsequences">

                                <b>[3]</b>Smith T F,Waterman M S.Identification of common molecular subsequences[J].Journal of Molecular Biology,1981,147(1):195- 197
                            </a>
                        </p>
                        <p id="9">
                            <a id="bibliography_4" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM13091000024362&amp;v=MDk3MzVOcjQ5RlpPa0xEM283b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadUh5am1VTDNJSjFzVmJoTT1OaWZJWTdLN0h0ag==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[4]</b>Hirschberg D S.A linear space algorithm for computing maximal common subsequences[J].Communications of the ACM,1975,18(18):341- 343
                            </a>
                        </p>
                        <p id="11">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Algorithms for approximate string matching">

                                <b>[5]</b>Ukkonen E.Algorithms for approximate string matching[J].Information &amp; Control,1985,64(1/2/3):100- 118
                            </a>
                        </p>
                        <p id="13">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Improvements on bicriteria pairwise sequence alignment:Algorithms and applications">

                                <b>[6]</b>Abbasi M,Paquete L,Liefooghe A,et al.Improvements on bicriteria pairwise sequence alignment:Algorithms and applications[J].Bioinformatics,2013,29(8):996- 1003
                            </a>
                        </p>
                        <p id="15">
                            <a id="bibliography_7" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Orthologous Matrix(OMA)algorithm 2.0,more robust to asymmetric evolutionary rates and more scalable hierarchical orthologous group inference">

                                <b>[7]</b>Train C M,Glover N M,Gonnet G H,et al.Orthologous matrix (OMA) algorithm 2.0:More robust to asymmetric evolutionary rates and more scalable hierarchical orthologous group inference[J].Bioinformatics,2017,33(14):i75- i82
                            </a>
                        </p>
                        <p id="17">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The OMA orthology database in 2015:function predictions,better plant support,synteny view and other improvements">

                                <b>[8]</b>Altenhoff A M,Škunca N,Glover N,et al.The OMA orthology database in 2015:Function predictions,better plant support,synteny view and other improvements[J].Nucleic Acids Research,2015,43(D1):240- 249
                            </a>
                        </p>
                        <p id="19">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Edlib:A C/C++ library for fast,exact sequence alignment using edit distance">

                                <b>[9]</b>Šoši M,Šiki M.Edlib:A C/C++ library for fast,exact sequence alignment using edit distance[J].Bioinformatics,2017,33(9):1394- 1395
                            </a>
                        </p>
                        <p id="21">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A statistical physics perspective on alignment-independent protein sequence comparison">

                                <b>[10]</b>Chattopadhyay A K,Nasiev D,Flower D R.A statistical physics perspective on alignment-independent protein sequence comparison[J].Bioinformatics,2015,31(15):2469- 2674
                            </a>
                        </p>
                        <p id="23">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Alignment-free sequence comparison over Hadoop for computational biology">

                                <b>[11]</b>Cattaneo G,Petrillo U F,Giancarlo R,et al.Alignment-free sequence comparison over Hadoop for computational biology[C] //Proc of the 44th Int Conf on Parallel Processing Workshops.Piscataway,NJ:IEEE,2015:184- 192
                            </a>
                        </p>
                        <p id="25">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=CS2A A compressed suffix array-based method for short read alignment">

                                <b>[12]</b>Huo Hongwei,Sun Zhigang,Li Shuangjiang,et al.CS2A:A compressed suffix array-based method for short read alignment[C] //Proc of the 2016 IEEE Data Compression Conf.Piscataway,NJ:IEEE,2016:271- 278
                            </a>
                        </p>
                        <p id="27">
                            <a id="bibliography_13" target="_blank" href="http://scholar.cnki.net/result.aspx?q=An efficient processing element architecture for pairwise sequence alignment">

                                <b>[13]</b>Isa M N,Murad S A Z,Ismail R C,et al.An efficient processing element architecture for pairwise sequence alignment[C] // Proc of the 33rd Int Conf on Electronic Design.Piscataway,NJ:IEEE,2015:461- 464
                            </a>
                        </p>
                        <p id="29">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Striped Smith--Waterman speeds database searches six times over other SIMD implementations">

                                <b>[14]</b>Farrar M.Striped Smith-Waterman speeds database searches six times over other SIMD implementations[J].Bioinformatics,2007,23(2):156- 161
                            </a>
                        </p>
                        <p id="31">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=High performance streaming Smith-Waterman implementation with implicit synchronization on Intel FPGA using OpenCL">

                                <b>[15]</b>Houtgast E,Sima V M,Al-Ars Z.High performance streaming Smith-Waterman implementation with implicit synchronization on Intel FPGA using OpenCL[C] //Proc of the 2018 IEEE Int Conf on Bioinformatics and Bioengineering.Piscataway,NJ:IEEE,2018:492- 496
                            </a>
                        </p>
                        <p id="33">
                            <a id="bibliography_16" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Parallel processing cell score design of linear gap penalty Smith-Waterman algorithm">

                                <b>[16]</b>Junid S A M A,Idros M F M,Razak A H A,et al.Parallel processing cell score design of linear gap penalty Smith-Waterman algorithm[C] //Proc of the 13th IEEE Int Colloquium on Signal Processing &amp; ITS Applications.Piscataway,NJ:IEEE,2017:299- 302
                            </a>
                        </p>
                        <p id="35">
                            <a id="bibliography_17" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00003315832&amp;v=MTMyNDBWTHJKSkY4PU5qN0Jhck80SHRIUHJJNUFiT2dOWTNrNXpCZGg0ajk5U1hxUnJ4b3hjTUg3UjdxZWJ1ZHRGQ3Js&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[17]</b>Xue Jinyun.A unified approach for developing efficient algorithmic programs[J].Journal of Computer Science &amp; Technology,1997,12(4):314- 329
                            </a>
                        </p>
                        <p id="37">
                            <a id="bibliography_18" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Formal derivation of a generic algorithmic program for solving a class of extremum problems">

                                <b>[18]</b>Wang Changjing,Xue Jinyun.Formal derivation of a generic algorithmic program for solving a class of extremum problems[C] //Proc of the 10th ACIS Int Conf on Software Engineering,Artificial Intelligences,Networking and Parallel/Distributed Computing.Piscataway,NJ:IEEE,2009:100- 105
                            </a>
                        </p>
                        <p id="39">
                            <a id="bibliography_19" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Genericity in PAR Platform">

                                <b>[19]</b>Xue Jinyun.Genericity in PAR platform[C] //Proc of the 5th Int Workshop on Structured Object-Oriented Formal Language and Method.Berlin:Springer,2015:3- 14
                            </a>
                        </p>
                        <p id="41">
                            <a id="bibliography_20" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Feature-Oriented Domain Analysis (FODA) Feasibility Study">

                                <b>[20]</b>Kang K C,Cohen S G,Hess J A,et al.Feature-oriented domain analysis (FODA) feasibility study,CMU/SEI-90-TR-21[R].Pittsburgh:Carnegie-Mellon University,1990
                            </a>
                        </p>
                        <p id="43">
                            <a id="bibliography_21" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Generative Programming: Methods, Tools, And Applications">

                                <b>[21]</b>Czarnecki K,Eisenecker U W.Generative Programming:Methods,Tools,and Applications[M].New York:ACM,2000
                            </a>
                        </p>
                        <p id="45">
                            <a id="bibliography_22" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=RJXB200308000&amp;v=MjYzMTA1NE8zenFxQnRHRnJDVVJMT2VaZVJzRnkvZ1VydkpOeWZUYkxHNEh0TE1wNDlGWklRS0RIODR2UjRUNmo=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[22]</b>Zhang Wei,Mei Hong.A feature-oriented domain model and its modeling process[J].Journal of Software,2003,14(8):1345- 1356 (in Chinese)(张伟,梅宏.一种面向特征的领域模型及其建模过程[J].软件学报,2003,14(8):1345- 1356)
                            </a>
                        </p>
                        <p id="47">
                            <a id="bibliography_23" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJOX&amp;filename=SJOXC9B2CA4A344D863D2ACF5166E3AA45F6&amp;v=MDQ2NDhDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOcGh3NzIreEtrPU5pZkVkc0N4Yk5PLzNvczBaKzhMZUhRL3pHSVJtMHdMVFg3a3FtYzJDTU9RUU15Wg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[23]</b>Giegerich R.A systematic approach to dynamic programming in bioinformatics[J].Bioinformatics,2000,16(8):665- 677
                            </a>
                        </p>
                        <p id="49">
                            <a id="bibliography_24" target="_blank" href="http://scholar.cnki.net/result.aspx?q=GPU Parallelization of Algebraic Dynamic Programming">

                                <b>[24]</b>Steffen P,Giegerich R,Giraud M.GPU parallelization of algebraic dynamic programming[C] //Proc of the 8th Int Conf on Parallel Processing and Applied Mathematics.Berlin:Springer,2009:290- 299
                            </a>
                        </p>
                        <p id="51">
                            <a id="bibliography_25" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Pareto optimization in algebraic dynamic programming">

                                <b>[25]</b>Saule C,Giegerich R.Pareto optimization in algebraic dynamic programming[J].Algorithms for Molecular Biology,2015,10(1):22
                            </a>
                        </p>
                        <p id="53">
                            <a id="bibliography_26" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Automata-theoretic models of mutation and alignment">

                                <b>[26]</b>Searls D B,Murphy K P.Automata-theoretic models of mutation and alignment[C] //Proc of the 3rd Intelligent Systems for Molecular Biology.Menlo Park,CA:AAAI Press,1995:341- 349
                            </a>
                        </p>
                        <p id="55">
                            <a id="bibliography_27" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201311028&amp;v=MzA2MDc0SDlMTnJvOUhiSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUnNGeS9nVXJ2Skx5dlNkTEc=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[27]</b>Wang Wendi,Tang Wen,Duan Bo,et al.Parallel accelerator design for high-throughput DNA sequence alignment with Hash-index[J].Journal of Computer Research and Development,2013,50(11):2463- 2471 (in Chinese)(王文迪,汤文,段勃,等.基于Hash索引的高通量基因序列比对并行加速技术研究[J].计算机研究与发展,2013,50(11):2463- 2471)
                            </a>
                        </p>
                        <p id="57">
                            <a id="bibliography_28" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ200608005&amp;v=MTI1ODBGckNVUkxPZVplUnNGeS9nVXJ2Skx5dlNkTEc0SHRmTXA0OUZZWVFLREg4NHZSNFQ2ajU0TzN6cXFCdEc=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[28]</b>Ge Hongwei,Liang Yanchun.A multiple sequence alignment algorithm based on a hidden Markov model and immune particle swarm optimization[J].Journal of Computer Research and Development,2006,43(8):756- 765 (in Chinese)(葛宏伟,梁艳春.基于隐马尔可夫模型和免疫粒子群优化的多序列比对算法[J].计算机研究与发展,2006,43(8):756- 765)
                            </a>
                        </p>
                        <p id="59">
                            <a id="bibliography_29" target="_blank" href="http://scholar.cnki.net/result.aspx?q=SVM-dependent pairwise HMM An application to protein pairwise alignments">

                                <b>[29]</b>Orlando G,Raimondi D,Khan T,et al.SVM-dependent pairwise HMM:An application to protein pairwise alignments[J].Bioinformatics,2017,33(24):3902- 3908
                            </a>
                        </p>
                        <p id="61">
                            <a id="bibliography_30" target="_blank" href="http://scholar.cnki.net/result.aspx?q=De novo assembly of human genomes with massively parallel short read sequencing">

                                <b>[30]</b>Li Ruiqiang,Zhu Hongmei,Ruan Jue,et al.De novo assembly of human genomes with massively parallel short read sequencing[J].Genome Research,2010,20(2):265- 272
                            </a>
                        </p>
                        <p id="63">
                            <a id="bibliography_31" target="_blank" href="http://scholar.cnki.net/result.aspx?q=IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth">

                                <b>[31]</b>Peng Yu,Leung H C M,Yiu S M,et al.IDBA-UD:A de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth[J].Bioinformatics,2012,28(11):1420- 1428
                            </a>
                        </p>
                        <p id="65">
                            <a id="bibliography_32" target="_blank" href="http://scholar.cnki.net/result.aspx?q=ISEA:Iterative Seed-Extension Algorithm for De Novo Assembly Using Paired-End Information and Insert Size Distribution">

                                <b>[32]</b>Li Min,Liao Zhongxiang,He Yiming,et al.ISEA:Iterative seed-extension algorithm for de novo assembly using paired-end information and insert size distribution[J].IEEE/ACM Transactions on Computational Biology &amp; Bioinformatics,2017,14(4):916- 925
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JFYZ201909012" />
        <input id="dpi" type="hidden" value="600" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>


    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201909012&amp;v=MDUwNjg2ajU0TzN6cXFCdEdGckNVUkxPZVplUnNGeS9nVXJ2Skx5dlNkTEc0SDlqTXBvOUVab1FLREg4NHZSNFQ=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDZ3NE9QMDZGYWtIa1dUZEVqTkxlVT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>


    <link href="/kxreader/Content/css/LeftDetail?v=NLcKG8I1SJUaVFrQ0iGpF2klAT0OsmHRaVSZ1rKb5xg1" rel="stylesheet"/>

</body>
</html>

