

<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>

</head>

<body>

    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637127883375900000%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJFYZ201908008%26RESULT%3d1%26SIGN%3daU6Yx5SejJIV93092RYz6nnYkPE%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JFYZ201908008&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JFYZ201908008&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>


    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201908008&amp;v=MTk4NzZiSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUnJGeXZnVjczUEx5dlNkTEc0SDlqTXA0OUY=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#74" data-title="&lt;b&gt;1 相关工作&lt;/b&gt; "><b>1 相关工作</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#80" data-title="&lt;b&gt;2 数据预处理&lt;/b&gt; "><b>2 数据预处理</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#82" data-title="&lt;b&gt;2.1 数据集构建&lt;/b&gt;"><b>2.1 数据集构建</b></a></li>
                                                <li><a href="#90" data-title="&lt;b&gt;2.2 序列编码&lt;/b&gt;"><b>2.2 序列编码</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#94" data-title="&lt;b&gt;3 CNN和Bi-LSTM融合模型&lt;/b&gt; "><b>3 CNN和Bi-LSTM融合模型</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#96" data-title="&lt;b&gt;3.1 嵌入阶段&lt;/b&gt;"><b>3.1 嵌入阶段</b></a></li>
                                                <li><a href="#101" data-title="&lt;b&gt;3.2 卷积阶段&lt;/b&gt;"><b>3.2 卷积阶段</b></a></li>
                                                <li><a href="#113" data-title="&lt;b&gt;3.3 Bi-LSTM阶段&lt;/b&gt;"><b>3.3 Bi-LSTM阶段</b></a></li>
                                                <li><a href="#128" data-title="&lt;b&gt;3.4 模型实现&lt;/b&gt;"><b>3.4 模型实现</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#131" data-title="&lt;b&gt;4 实验与结果&lt;/b&gt; "><b>4 实验与结果</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#133" data-title="&lt;b&gt;4.1 验证方法与评价标准&lt;/b&gt;"><b>4.1 验证方法与评价标准</b></a></li>
                                                <li><a href="#145" data-title="&lt;b&gt;4.2 数据集&lt;/b&gt;"><b>4.2 数据集</b></a></li>
                                                <li><a href="#148" data-title="&lt;b&gt;4.3 特征提取&lt;/b&gt;"><b>4.3 特征提取</b></a></li>
                                                <li><a href="#168" data-title="&lt;b&gt;4.4 基于不同方法的分类结果&lt;/b&gt;"><b>4.4 基于不同方法的分类结果</b></a></li>
                                                <li><a href="#174" data-title="&lt;b&gt;4.5 基于不同物种的分类结果&lt;/b&gt;"><b>4.5 基于不同物种的分类结果</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#178" data-title="&lt;b&gt;5 结束语&lt;/b&gt; "><b>5 结束语</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#87" data-title="图1 序列连接方式">图1 序列连接方式</a></li>
                                                <li><a href="#92" data-title="图2 序列编码方式">图2 序列编码方式</a></li>
                                                <li><a href="#111" data-title="图3 卷积操作流程">图3 卷积操作流程</a></li>
                                                <li><a href="#115" data-title="图5 模型整体结构">图5 模型整体结构</a></li>
                                                <li><a href="#116" data-title="图4 双向循环神经网络">图4 双向循环神经网络</a></li>
                                                <li><a href="#144" data-title="&lt;b&gt;表1 分类结果含义&lt;/b&gt;"><b>表1 分类结果含义</b></a></li>
                                                <li><a href="#147" data-title="&lt;b&gt;表2 各物种数据集&lt;/b&gt;"><b>表2 各物种数据集</b></a></li>
                                                <li><a href="#170" data-title="&lt;b&gt;表3 基于不同方法的分类结果&lt;/b&gt;"><b>表3 基于不同方法的分类结果</b></a></li>
                                                <li><a href="#173" data-title="图6 不同方法在测试集上的ROC曲线">图6 不同方法在测试集上的ROC曲线</a></li>
                                                <li><a href="#176" data-title="&lt;b&gt;表4 基于不同物种的分类结果&lt;/b&gt; %"><b>表4 基于不同物种的分类结果</b> %</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="212">


                                    <a id="bibliography_1" title="Schmitz S U.Mechanisms of long noncoding RNA function in development and disease[J].Cellular and Molecular Life Sciences, 2016, 73 (13) :2491- 2509" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Mechanisms of long noncoding RNA function in development and disease">
                                        <b>[1]</b>
                                        Schmitz S U.Mechanisms of long noncoding RNA function in development and disease[J].Cellular and Molecular Life Sciences, 2016, 73 (13) :2491- 2509
                                    </a>
                                </li>
                                <li id="214">


                                    <a id="bibliography_2" title="Hu Mingyang, Ming Jia.Research progress of the regulatory mechanisms between lncRNAs and miRNAs[J].E-Journal of Translational Medicine, 2017, 4 (2) :71- 76 (in Chinese) (胡铭阳, 明佳.lncRNA与miRNA相互调控机制的研究进展[J].转化医学电子杂志, 2017, 4 (2) :71- 76) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZHDZ201702020&amp;v=MTQzNjg2ajU0TzN6cXFCdEdGckNVUkxPZVplUnJGeXZnVjczUFB5WFBkTEc0SDliTXJZOUhaSVFLREg4NHZSNFQ=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[2]</b>
                                        Hu Mingyang, Ming Jia.Research progress of the regulatory mechanisms between lncRNAs and miRNAs[J].E-Journal of Translational Medicine, 2017, 4 (2) :71- 76 (in Chinese) (胡铭阳, 明佳.lncRNA与miRNA相互调控机制的研究进展[J].转化医学电子杂志, 2017, 4 (2) :71- 76) 
                                    </a>
                                </li>
                                <li id="216">


                                    <a id="bibliography_3" title="Cesana M, Cacchihiarelli D, Legnini I, et al.A long noncoding RNA controls muscle differentiation by function-ing as a competing endogenous RNA[J].Cell, 2011, 147 (2) :358- 369" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES14121800006404&amp;v=MDE4MzZOaWZPZmJLOEg5UE5wNDlGWk9zSkNIdzlvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMcklJMXNRYUJVPQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[3]</b>
                                        Cesana M, Cacchihiarelli D, Legnini I, et al.A long noncoding RNA controls muscle differentiation by function-ing as a competing endogenous RNA[J].Cell, 2011, 147 (2) :358- 369
                                    </a>
                                </li>
                                <li id="218">


                                    <a id="bibliography_4" title="Jiang Hui, Ma Rong, Zou Shubiao, et al.Reconstruction and analysis of the lncRNA-miRNA-mRNA network based on competitive endogenous RNA reveal functional lncRNAs in rheumatoid arthritis[J].Molecular Biosystems, 2017, 13 (6) :1182- 1192" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Reconstruction and analysis of the lncRNA-miRNA-mRNA network based on competitive endogenous RNA reveal functional lncRNAs in rheumatoid arthritis">
                                        <b>[4]</b>
                                        Jiang Hui, Ma Rong, Zou Shubiao, et al.Reconstruction and analysis of the lncRNA-miRNA-mRNA network based on competitive endogenous RNA reveal functional lncRNAs in rheumatoid arthritis[J].Molecular Biosystems, 2017, 13 (6) :1182- 1192
                                    </a>
                                </li>
                                <li id="220">


                                    <a id="bibliography_5" title="Kim S K, Nam J W, Rhee J K, et al.miTarget:MicroRNA target gene prediction using a support vector machine[J].BMC Bioinformatics, 2006, 7 (1) :411- 423" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=miTarget: microRNA target gene prediction using a support vector machine">
                                        <b>[5]</b>
                                        Kim S K, Nam J W, Rhee J K, et al.miTarget:MicroRNA target gene prediction using a support vector machine[J].BMC Bioinformatics, 2006, 7 (1) :411- 423
                                    </a>
                                </li>
                                <li id="222">


                                    <a id="bibliography_6" title="Asgari E, Mofrad M R K.Continuous distributed representation of biological sequences for deep proteomics and genomics[J].PLOS ONE, 2015, 10 (11) :e0141287" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Continuous Distributed Representation of Biological Sequences for Deep Proteomics and Genomics">
                                        <b>[6]</b>
                                        Asgari E, Mofrad M R K.Continuous distributed representation of biological sequences for deep proteomics and genomics[J].PLOS ONE, 2015, 10 (11) :e0141287
                                    </a>
                                </li>
                                <li id="224">


                                    <a id="bibliography_7" title="Cheng Jian, Wang Peisong, Li Gang, et al.Recent advances in efficient computation of deep convolutional neural networks[J].Frontiers of Information Technology &amp;amp; Electronic Engineering, 2018, 19 (1) :64- 77" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JZUS201801007&amp;v=MTg5MzZxQnRHRnJDVVJMT2VaZVJyRnl2Z1Y3M1BMemZlZmJHNEg5bk1ybzlGWTRRS0RIODR2UjRUNmo1NE8zenE=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[7]</b>
                                        Cheng Jian, Wang Peisong, Li Gang, et al.Recent advances in efficient computation of deep convolutional neural networks[J].Frontiers of Information Technology &amp;amp; Electronic Engineering, 2018, 19 (1) :64- 77
                                    </a>
                                </li>
                                <li id="226">


                                    <a id="bibliography_8" title="Hochreiter S, Schmidhuber J.Long short-term memory[J].Neural Computation, 1997, 9 (8) :1735- 1780" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJBK&amp;filename=SJBK15090500014198&amp;v=MDgwNTQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMcklJMXNRYUJVPU5pZkpaYks5SHRqTXFvOUZaT29MRFhVeG9CTVQ2VA==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[8]</b>
                                        Hochreiter S, Schmidhuber J.Long short-term memory[J].Neural Computation, 1997, 9 (8) :1735- 1780
                                    </a>
                                </li>
                                <li id="228">


                                    <a id="bibliography_9" title="Hou Min, Jiang Lin, Zhan Hongmei, et al.Prediction of breast cancer related lncRNA-miRNA-mRNA co-expression and key genes based on network construction method[J].Chinese Journal of Cancer Prevention and Treatment, 2018, 25 (1) :26- 33 (in Chinese) (侯敏, 蒋琳, 詹红梅, 等.乳腺癌相关lncRNA-miRNA-mRNA共表达及关键基因网络构建预测[J].中华肿瘤防治杂志, 2018, 25 (1) :26- 33) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=QLZL201801005&amp;v=MzE4MTdZUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSckZ5dmdWNzNQTkNIUllyRzRIOW5Ncm85Rlk=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[9]</b>
                                        Hou Min, Jiang Lin, Zhan Hongmei, et al.Prediction of breast cancer related lncRNA-miRNA-mRNA co-expression and key genes based on network construction method[J].Chinese Journal of Cancer Prevention and Treatment, 2018, 25 (1) :26- 33 (in Chinese) (侯敏, 蒋琳, 詹红梅, 等.乳腺癌相关lncRNA-miRNA-mRNA共表达及关键基因网络构建预测[J].中华肿瘤防治杂志, 2018, 25 (1) :26- 33) 
                                    </a>
                                </li>
                                <li id="230">


                                    <a id="bibliography_10" title="Han Siyu, Liang Yanchun, Li Ying, et al.Long noncoding RNA identification:Comparing machine learning based tools for long noncoding transcripts discrimination[J].BioMed Research International, 2016, (2016) :1- 14" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJHD&amp;filename=SJHD347D5FAA785F565F26433BF67FAE1B34&amp;v=MjY5MDVNZVZON21iQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TjFoeDcyN3dxOD1OaWZEYXJDOEdhWEoyZjQwWStNS2Vuay95bUFSN0R0K1N3MlVxaFZEQw==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[10]</b>
                                        Han Siyu, Liang Yanchun, Li Ying, et al.Long noncoding RNA identification:Comparing machine learning based tools for long noncoding transcripts discrimination[J].BioMed Research International, 2016, (2016) :1- 14
                                    </a>
                                </li>
                                <li id="232">


                                    <a id="bibliography_11" title="Hinton G E, Salakhutdinov R R.Reducing the dimensionality of data with neural networks[J].Science, 2006, 313 (5786) :504- 507" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Reducing the dimensionality of data with neural networks">
                                        <b>[11]</b>
                                        Hinton G E, Salakhutdinov R R.Reducing the dimensionality of data with neural networks[J].Science, 2006, 313 (5786) :504- 507
                                    </a>
                                </li>
                                <li id="234">


                                    <a id="bibliography_12" title="Williams R J, Zipser D.A learning algorithm for continually running fully recurrent neural networks[J].Neural Computation, 1989, 1 (2) :270- 280" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJBK&amp;filename=SJBK15090500012713&amp;v=MDE4NTZDMzA2b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadUh5am1VTHJJSTFzUWFCVT1OaWZKWmJLOUh0ak1xbzlGWk9vTg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[12]</b>
                                        Williams R J, Zipser D.A learning algorithm for continually running fully recurrent neural networks[J].Neural Computation, 1989, 1 (2) :270- 280
                                    </a>
                                </li>
                                <li id="236">


                                    <a id="bibliography_13" title="Tripathi R, Patel S, Kumari V, et al.DeepLNC, a long non-coding RNA prediction tool using deep neural network[J].Network Modeling Analysis in Health Informatics and Bioinformatics, 2016, 5 (1) :No.21" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=DeepLNC,a long non-coding RNA prediction tool using deep neural network">
                                        <b>[13]</b>
                                        Tripathi R, Patel S, Kumari V, et al.DeepLNC, a long non-coding RNA prediction tool using deep neural network[J].Network Modeling Analysis in Health Informatics and Bioinformatics, 2016, 5 (1) :No.21
                                    </a>
                                </li>
                                <li id="238">


                                    <a id="bibliography_14" title="Junghwan B, Byunghan L, Sunyoung K, et al.lncRNAnet:Long non-coding RNA identification using deep learning[J].Bioinformatics, 2018, 34 (22) :3889- 3897" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=lncRNAnet:Long non-coding RNA identification using deep learning">
                                        <b>[14]</b>
                                        Junghwan B, Byunghan L, Sunyoung K, et al.lncRNAnet:Long non-coding RNA identification using deep learning[J].Bioinformatics, 2018, 34 (22) :3889- 3897
                                    </a>
                                </li>
                                <li id="240">


                                    <a id="bibliography_15" title="Andreu Paytuv&#237; Gallart, Pulido A H, Irantzu Anzar Mart&#237;nez De Lagr&#225;n, et al.GREENC:A wiki-based database of plant lncRNAs[J].Nucleic Acids Research, 2016, 44 (D1) :D1161- D1166" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=GREENC:a Wikibased database of plant lnc RNAs">
                                        <b>[15]</b>
                                        Andreu Paytuv&#237; Gallart, Pulido A H, Irantzu Anzar Mart&#237;nez De Lagr&#225;n, et al.GREENC:A wiki-based database of plant lncRNAs[J].Nucleic Acids Research, 2016, 44 (D1) :D1161- D1166
                                    </a>
                                </li>
                                <li id="242">


                                    <a id="bibliography_16" title="Griffithsjones S.miRBase:The microRNA sequence database[J].Methods in Molecular Biology, 2006, 342:129- 138" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=miRBase: the microRNA sequence data- base">
                                        <b>[16]</b>
                                        Griffithsjones S.miRBase:The microRNA sequence database[J].Methods in Molecular Biology, 2006, 342:129- 138
                                    </a>
                                </li>
                                <li id="244">


                                    <a id="bibliography_17" title="Dai Xinbin, Zhuang Zhaohong, Zhao P X.psRNATarget:A plant small RNA target analysis server (2017 release) [J].Nucleic Acids Research, 2018, 46:W49- W54" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=psRNATarget:A plant small RNA target analysis server (2017 release)">
                                        <b>[17]</b>
                                        Dai Xinbin, Zhuang Zhaohong, Zhao P X.psRNATarget:A plant small RNA target analysis server (2017 release) [J].Nucleic Acids Research, 2018, 46:W49- W54
                                    </a>
                                </li>
                                <li id="246">


                                    <a id="bibliography_18" title="Negri T, Alves W, Bugatti P H, et al.Pattern recognition analysis on long noncoding RNAs:A tool for prediction in plants[J].Briefings in Bioinformatics, 2019, 20 (2) :682- 689" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Pattern recognition analysis on long noncoding RNAs:A tool for prediction in plants">
                                        <b>[18]</b>
                                        Negri T, Alves W, Bugatti P H, et al.Pattern recognition analysis on long noncoding RNAs:A tool for prediction in plants[J].Briefings in Bioinformatics, 2019, 20 (2) :682- 689
                                    </a>
                                </li>
                                <li id="248">


                                    <a id="bibliography_19" title="Jin Chen, Li Weihua, Ji Chen, et al.Bidirectional long shortterm memory neural networks for Chinese word segmentation[J].Journal of Chinese Information Processing, 2018, 32 (2) :29- 37 (in Chinese) (金宸, 李维华, 姬晨, 等.基于双向LSTM神经网络模型的中文分词[J].中文信息学报, 2018, 32 (2) :29- 37) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=MESS201802004&amp;v=Mjk2MjZPZVplUnJGeXZnVjczUEtDallmYkc0SDluTXJZOUZZSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkw=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[19]</b>
                                        Jin Chen, Li Weihua, Ji Chen, et al.Bidirectional long shortterm memory neural networks for Chinese word segmentation[J].Journal of Chinese Information Processing, 2018, 32 (2) :29- 37 (in Chinese) (金宸, 李维华, 姬晨, 等.基于双向LSTM神经网络模型的中文分词[J].中文信息学报, 2018, 32 (2) :29- 37) 
                                    </a>
                                </li>
                                <li id="250">


                                    <a id="bibliography_20" title="Zhang Hao, Wu Jianxin.A survey on unsupervised image retrieval using deep features[J].Journal of Computer Research and Development, 2018, 55 (9) :1829- 1842 (in Chinese) (张皓, 吴建鑫.基于深度特征的无监督图像检索研究综述[J].计算机研究与发展, 2018, 55 (9) :1829- 1842) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201809002&amp;v=MTUyMTJPM3pxcUJ0R0ZyQ1VSTE9lWmVSckZ5dmdWNzNQTHl2U2RMRzRIOW5NcG85RlpvUUtESDg0dlI0VDZqNTQ=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[20]</b>
                                        Zhang Hao, Wu Jianxin.A survey on unsupervised image retrieval using deep features[J].Journal of Computer Research and Development, 2018, 55 (9) :1829- 1842 (in Chinese) (张皓, 吴建鑫.基于深度特征的无监督图像检索研究综述[J].计算机研究与发展, 2018, 55 (9) :1829- 1842) 
                                    </a>
                                </li>
                                <li id="252">


                                    <a id="bibliography_21" title="Li Hongshun, Yu Hua, Gong Xiujun.A deep learning model for predicting RNA-binding proteins only from primary sequences[J].Journal of Computer Research and Development, 2018, 55 (1) :93- 101 (in Chinese) (李洪顺, 于华, 宫秀军.一种只利用序列信息预测RNA结合蛋白的深度学习模型[J].计算机研究与发展, 2018, 55 (1) :93- 101) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201801006&amp;v=MjIzMDhaZVJyRnl2Z1Y3M1BMeXZTZExHNEg5bk1ybzlGWW9RS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2U=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[21]</b>
                                        Li Hongshun, Yu Hua, Gong Xiujun.A deep learning model for predicting RNA-binding proteins only from primary sequences[J].Journal of Computer Research and Development, 2018, 55 (1) :93- 101 (in Chinese) (李洪顺, 于华, 宫秀军.一种只利用序列信息预测RNA结合蛋白的深度学习模型[J].计算机研究与发展, 2018, 55 (1) :93- 101) 
                                    </a>
                                </li>
                                <li id="254">


                                    <a id="bibliography_22" title="Schuster M, Paliwal K K.Bidirectional recurrent neural network[J].IEEE Transactions on Signal Processing, 1997, 45 (11) :2673- 2681" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Bidirectional recurrent neural networks">
                                        <b>[22]</b>
                                        Schuster M, Paliwal K K.Bidirectional recurrent neural network[J].IEEE Transactions on Signal Processing, 1997, 45 (11) :2673- 2681
                                    </a>
                                </li>
                                <li id="256">


                                    <a id="bibliography_23" title="Wang Xiaowo, Zhang Jing, Li Fei, et al.MicroRNA identification based on sequence and structure alignment[J].Bioinfor-matics, 2005, 21 (18) :3610- 3614" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=MicroRNA identification based on sequence and structure alignment">
                                        <b>[23]</b>
                                        Wang Xiaowo, Zhang Jing, Li Fei, et al.MicroRNA identification based on sequence and structure alignment[J].Bioinfor-matics, 2005, 21 (18) :3610- 3614
                                    </a>
                                </li>
                                <li id="258">


                                    <a id="bibliography_24" title="Lorenz R, Bernhart S H, Siederdissen C H Z, et al.ViennaRNA package 2.0[J].Algorithms for Molecular Biology, 2011, 6:No.26" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=ViennaRNA Package 2.0">
                                        <b>[24]</b>
                                        Lorenz R, Bernhart S H, Siederdissen C H Z, et al.ViennaRNA package 2.0[J].Algorithms for Molecular Biology, 2011, 6:No.26
                                    </a>
                                </li>
                                <li id="260">


                                    <a id="bibliography_25" title="Chang Zheng, Meng Jun, Shi Yunsheng, et al.LncRNA recognition by fusing multiple features and its function prection[J].CAAI Transactions on Intelligent Systems, 2018, 13 (6) :928- 934 (in Chinese) (常征, 孟军, 施云生, 等.多特征融合的lncRNA识别与其功能预测[J].智能系统学报, 2018, 13 (6) :928- 934) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZNXT201806009&amp;v=MDc0MjZPZVplUnJGeXZnVjczUFB5UFRlckc0SDluTXFZOUZiWVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkw=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[25]</b>
                                        Chang Zheng, Meng Jun, Shi Yunsheng, et al.LncRNA recognition by fusing multiple features and its function prection[J].CAAI Transactions on Intelligent Systems, 2018, 13 (6) :928- 934 (in Chinese) (常征, 孟军, 施云生, 等.多特征融合的lncRNA识别与其功能预测[J].智能系统学报, 2018, 13 (6) :928- 934) 
                                    </a>
                                </li>
                                <li id="262">


                                    <a id="bibliography_26" title="Gao Chongzhi, Cheng Qiong, Pei He, et al.Privacy-preserving Naive Bayes classifiers secure against the substitution-then-comparison attack[J].Information Sciences, 2018, 444:72- 88" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES295D525CA48899AA0EFBFCA91EE6B470&amp;v=Mjc2NTF2RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOMWh4NzI3d3E4PU5pZk9mYkd4RzZYSnJZbzJGZThIQkhVd3ZtY1RuMGtQUGd5VHBSTkFETFRtUWIyZkNPTg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[26]</b>
                                        Gao Chongzhi, Cheng Qiong, Pei He, et al.Privacy-preserving Naive Bayes classifiers secure against the substitution-then-comparison attack[J].Information Sciences, 2018, 444:72- 88
                                    </a>
                                </li>
                                <li id="264">


                                    <a id="bibliography_27" title="Ma Xiaolei, Ding Chuan, Luan Sen, et al.Prioritizing influential factors for freeway incident clearance time prediction using the gradient boosting decision trees method[J].IEEE Transactions on Intelligent Transportation Systems, 2017, 18 (9) :1- 8" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prioritizing Influential Factors for Freeway Incident Clearance Time Prediction Using the Gradient Boosting Decision Trees Method">
                                        <b>[27]</b>
                                        Ma Xiaolei, Ding Chuan, Luan Sen, et al.Prioritizing influential factors for freeway incident clearance time prediction using the gradient boosting decision trees method[J].IEEE Transactions on Intelligent Transportation Systems, 2017, 18 (9) :1- 8
                                    </a>
                                </li>
                                <li id="266">


                                    <a id="bibliography_28" title="Rodriguez-Galiano V F, Ghimire B, Rogan J, et al.An assessment of the effectiveness of a random forest classifier for land-cover classification[J].Isprs Journal of Photo-grammetry &amp;amp; Remote Sensing, 2012, 67 (1) :93- 104" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011501968360&amp;v=MDY2OTBNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadUh5am1VTHJJSTFzUWFCVT1OaWZPZmJLN0h0RE5xbzlFYmUwSEQzbzVvQg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[28]</b>
                                        Rodriguez-Galiano V F, Ghimire B, Rogan J, et al.An assessment of the effectiveness of a random forest classifier for land-cover classification[J].Isprs Journal of Photo-grammetry &amp;amp; Remote Sensing, 2012, 67 (1) :93- 104
                                    </a>
                                </li>
                                <li id="268">


                                    <a id="bibliography_29" title="Landgrebe D.A survey of decision tree classifier methodology[J].IEEE Transactions on Systems Man &amp;amp; Cybernetics, 2002, 21 (3) :660- 674" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A Survey of Decision Tree Classifier Methodology">
                                        <b>[29]</b>
                                        Landgrebe D.A survey of decision tree classifier methodology[J].IEEE Transactions on Systems Man &amp;amp; Cybernetics, 2002, 21 (3) :660- 674
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JFYZ" target="_blank">计算机研究与发展</a>
                2019,56(08),1652-1660 DOI:10.7544/issn1000-1239.2019.20190128            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>融合CNN和Bi-LSTM的miRNA-lncRNA互作关系预测模型</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E7%9F%B3%E6%96%87%E6%B5%A9&amp;code=42588555&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">石文浩</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%AD%9F%E5%86%9B&amp;code=06529665&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">孟军</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BC%A0%E6%9C%8B&amp;code=10571420&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">张朋</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%88%98%E5%A9%B5%E5%A8%9F&amp;code=42588556&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">刘婵娟</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%A4%A7%E8%BF%9E%E7%90%86%E5%B7%A5%E5%A4%A7%E5%AD%A6%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%A7%91%E5%AD%A6%E4%B8%8E%E6%8A%80%E6%9C%AF%E5%AD%A6%E9%99%A2&amp;code=0222286&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">大连理工大学计算机科学与技术学院</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>非编码RNA (ncRNA) 在很多动植物生命活动方面起着重要的调节作用, 而微小RNA (miRNA) 与长非编码RNA (lncRNA) 的相互作用更为重要, 其互作关系的研究不仅有助于深入分析基因间生物学功能, 也可为疾病的诊治和植物的遗传育种方面提供新思路.目前, miRNA-lncRNA互作关系的预测大多使用生物实验和传统机器学习方法.由于生物鉴定代价高耗时长和机器学习涉及过多人工干预且特征提取过程复杂, 在此提出一种融合卷积神经网络 (convolutional neural network, CNN) 和双向长短期记忆网络 (bidirectional long short-term memory network, Bi-LSTM) 的深度学习模型, 兼备两者优势, 既考虑序列间信息相关性和结合上下文信息, 又能充分提取序列数据的特征.采用交叉检验评估模型性能, 在玉米数据集上与传统机器学习方法和单一模型比对, 取得较优的分类效果.另外, 采用马铃薯和小麦数据集进行模型测试, 准确率分别达到95%和93%以上, 验证了模型具有良好的泛化能力.</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8D%B7%E7%A7%AF%E7%A5%9E%E7%BB%8F%E7%BD%91%E7%BB%9C&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">卷积神经网络;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8F%8C%E5%90%91%E9%95%BF%E7%9F%AD%E6%9C%9F%E8%AE%B0%E5%BF%86%E7%BD%91%E7%BB%9C&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">双向长短期记忆网络;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=miRNA-lncRNA&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">miRNA-lncRNA;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E9%A2%84%E6%B5%8B&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">预测;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">深度学习;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    *孟军, mengjun@dlut.edu.cn;
                                </span>
                                <span>
                                    石文浩, swh31809184@mail.dlut.edu.cn;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2019-03-06</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金项目 (61872055, 61702075);</span>
                    </p>
            </div>
                    <h1><b>Prediction of miRNA-lncRNA Interaction by Combining CNN and Bi-LSTM</b></h1>
                    <h2>
                    <span>Shi Wenhao</span>
                    <span>Meng Jun</span>
                    <span>Zhang Peng</span>
                    <span>Liu Chanjuan</span>
            </h2>
                    <h2>
                    <span>School of Computer Science and Technology, Dalian University of Technology</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>Non-coding RNA (ncRNA) plays an important regulatory role in many animal and plant life activities, and the interaction of microRNA (miRNA) and long non-coding RNA (lncRNA) is more important. The study of their interaction not only helps to analyze the biological functions of genes, but also provides new ideas for disease diagnosis and treatment and plant genetic breeding. At present, biological experiments and machine learning methods are mostly used to predict miRNA-lncRNA interaction. Due to high cost and time consuming of biological identification and the excessive manual intervention of machine learning and the complex feature extraction process, a deep learning model combining convolutional neural network (CNN) and bidirectional long short-term memory network (Bi-LSTM) is proposed. It combines the advantages of two models, considering the information correlation between sequences and combining context information, and fully extracting features between sequence data. In the experiment, the performance of model is evaluated by cross-validation, compared with the traditional machine learning methods and single model on <i>zea mays</i> dataset, and the superior classification effect is obtained. In addition, the model tests of <i>solanum tuberosum</i> and <i>triticum aestivum</i> species are carried out, and the accuracy rates are up to 95% and 93%, respectively, which verifies good generalization ability of the model.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=convolutional%20neural%20network%20(CNN)%20&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">convolutional neural network (CNN) ;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=bidirectional%20long%20short-term%20memory%20network%20(Bi-LSTM)%20&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">bidirectional long short-term memory network (Bi-LSTM) ;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=miRNA-lncRNA&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">miRNA-lncRNA;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=prediction&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">prediction;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=deep%20learning&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">deep learning;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                            <b>Author：</b>
                                                        <span>
                                    Shi Wenhao, born in 1995. Master candidate. His main research interests include deep learning and data mining. <image id="302" type="" href="images/JFYZ201908008_30200.jpg" display="inline" placement="inline"><alt></alt></image>;
                                </span>
                                <span>
                                    Meng Jun, born in 1964. PhD and professor.Member of CCF.Her main research interests include data mining, machine learning, and bioinformatics. <image id="304" type="" href="images/JFYZ201908008_30400.jpg" display="inline" placement="inline"><alt></alt></image>;
                                </span>
                                <span>
                                    Zhang Peng, born in 1994. Master candidate. His main research interests include gene selection and machine learning. <image id="309" type="" href="images/JFYZ201908008_30900.jpg" display="inline" placement="inline"><alt></alt></image>;
                                </span>
                                <span>
                                    Liu Chanjuan, born in 1986.PhD and assistant professor.Her main research interests include logic, multiagent systems, and artificial intelligence.<image id="308" type="" href="images/JFYZ201908008_30800.jpg" display="inline" placement="inline"><alt></alt></image>;
                                </span>
                    </p>
                                    <p><b>Received：</b> 2019-03-06</p>
                                    <p>
                            <b>Fund：</b>
                                                        <span>supported by the National Natural Science Foundation of China (61872055, 61702075);</span>
                    </p>
            </div>


        <!--brief start-->
                        <div class="p1">
                    <p id="66">随着对非编码RNA研究的深入, 人们发现长非编码RNA (long non-coding RNA, lncRNA) 和微小RNA (microRNA, miRNA) 具有调节生物生命活动的重要作用, 它们在细胞生长、分化和增殖等方面起着重要的调节作用<citation id="270" type="reference"><link href="212" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>.研究表明:lncRNA可与miRNA竞争性结合mRNA或以诱饵方式吸附miRNA来调控miRNA<citation id="271" type="reference"><link href="214" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>.反之, miRNA通过与lncRNA的3’UTR不完全匹配进行负性调节, 从而直接作用于lncRNA<citation id="272" type="reference"><link href="216" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>.另外, 由于两者调节网络的重叠或位置的关系影响其相互作用, miRNA也可间接作用于lncRNA.</p>
                </div>
                <div class="p1">
                    <p id="67">目前, 研究lncRNA-miRNA-mRNA的相互调控网络是新热点<citation id="273" type="reference"><link href="218" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>.由于lncRNA通过与mRNA竞争miRNA的靶基因结合位点来实现对mRNA的调控, 故研究miRNA是否靶向lncRNA是研究miRNA调控功能的突破口.现有鉴定miRNA靶基因的方法主要分为生物实验和计算预测方法2类.生物实验一方面鉴定代价高耗时长, 另一方面不适合大批量的鉴定.传统计算预测方法则是利用机器学习算法建立预测模型, 通过提取miRNA靶基因的序列特征和结构特征作为输入数据, 构建分类器模型, 但机器学习方法涉及了过多的人工干预且特征提取过程复杂.为克服两者的弊端, 利用深度学习方法可自动学习特征的特点来实现分类预测是一个突破点.</p>
                </div>
                <div class="p1">
                    <p id="68">对于miRNA与lncRNA相互调控机制的研究大多集中在动物和人类癌症方面, 植物方面的研究相对较少.为深入探索植物miRNA与lncRNA的互作关系.本文借鉴miTarget<citation id="274" type="reference"><link href="220" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>方法, 利用“LLLLLL”将有互作关系的miRNA与lncRNA序列连接成单链序列, 使用基因组学中生物序列的连续表示方式<citation id="275" type="reference"><link href="222" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>, 对单链序列进行编码后作为输入数据, 并提出了一种融合卷积神经网络 (convolutional neural network, CNN) <citation id="276" type="reference"><link href="224" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>和双向长短期记忆网络 (bidirectional long short-term memory network, Bi-LSTM) <citation id="277" type="reference"><link href="226" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>的深度学习模型.该模型结合CNN充分提取特征和Bi-LSTM兼顾上下文信息的特点, 充分地学习了序列数据的特征, 实现对miRNA-lncRNA互作关系的分类预测.</p>
                </div>
                <div class="p1">
                    <p id="69">本文利用5折交叉检验方法, 通过与传统的机器学习方法、单一模型的比较和对多个物种数据集进行独立测试, 对在玉米、马铃薯和小麦数据集上的实验结果进行了分析.结果表明:本文提出的模型具有良好的分类效果和泛化能力.</p>
                </div>
                <div class="p1">
                    <p id="70">本文的贡献主要有3个方面:</p>
                </div>
                <div class="p1">
                    <p id="71">1) 借鉴miTarget方法, 将miRNA与lncRNA利用“LLLLLL”连接成单链序列, 从而方便使用深度学习模型;</p>
                </div>
                <div class="p1">
                    <p id="72">2) 借鉴自然语言处理中分词思想, 利用基因组学中生物序列的连续表示方式, 对生物序列进行编码, 使每条序列映射成一个<i>n</i>维的数字向量, 适用于LSTM的输入格式;</p>
                </div>
                <div class="p1">
                    <p id="73">3) 提出了一种融合CNN和Bi-LSTM的深度学习模型实现对miRNA-lncRNA互作关系的分类预测.</p>
                </div>
                <h3 id="74" name="74" class="anchor-tag"><b>1 相关工作</b></h3>
                <div class="p1">
                    <p id="75">目前, 对miRNA, lncRNA和mRNA之间调控机制方面的研究大多使用生物鉴定和计算预测方法<citation id="280" type="reference"><link href="228" rel="bibliography" /><link href="230" rel="bibliography" /><sup>[<a class="sup">9</a>,<a class="sup">10</a>]</sup></citation>.如使用高通量RNA-seq测序技术构建lncRNA-miRNA-mRNA共表达网络来研究乳腺癌中关键基因, 以期达到治疗癌症的目的<citation id="278" type="reference"><link href="228" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>;通过提取lncRNA的序列特征、二级结构等特征, 使用传统机器学习方法来识别lncRNA, 进而预测其功能<citation id="279" type="reference"><link href="230" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>.机器学习方法较之生物鉴定代价低耗时少, 但涉及过多人工干预和特征提取过程繁杂.</p>
                </div>
                <div class="p1">
                    <p id="76">自2006年Hinton等人<citation id="281" type="reference"><link href="232" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>在《Science》上提出深度学习, 自动学习特征、学习能力好等优点使之被各个领域广泛应用.CNN、循环神经网络 (recurrent neural network, RNN) <citation id="282" type="reference"><link href="234" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>和LSTM等模型也已很好解决了生物信息方面的问题.</p>
                </div>
                <div class="p1">
                    <p id="77">2016年Tripathi等人<citation id="283" type="reference"><link href="236" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>提出使用深度神经网络 (deep neural network, DNN) 模型, 利用多层神经网络层反馈调节, 逐层学习lncRNA的特征, 达到更好识别lncRNA的目的.2018年, Junghwan等人<citation id="284" type="reference"><link href="238" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>提出一种基于深度学习方法lncRNANet, 该方法将用于RNA序列建模的RNNs和检测密码子的CNNs进行结合, 从而更好地学习了lncRNA特征, 实现对lncRNA的鉴别.</p>
                </div>
                <div class="p1">
                    <p id="78">CNN是一种通过卷积操作提取特征, 再利用池化层学习数据的局部特征的前馈神经网络, 它无需对输入数据进行大量预处理, 且可以学习大量的特征信息.RNN具有内部记忆特点, 且在处理元间既有内部反馈连接又可前馈调节, 故对于处理序列信息具有良好的效果.但对于序列数据, CNN只考虑连续序列之间的相关性而忽略非连续序列之间的相关性.RNN虽适合处理序列数据, 但很难处理信息长期依赖的问题, 且存在梯度下降和梯度爆炸问题.LSTM是RNN的扩展, 专门用来处理无法长期依赖信息的问题, 它虽考虑长距离词的相关性, 但提取特征不够充分, 且单向LSTM不能处理下文单词信息.而双向LSTM (Bi-LSTM) 有正反LSTM, 正向LSTM捕获上文特征信息, 反向LSTN捕获下文特征信息, 故相较于单向LSTM, 能够更有效地处理句子中单词间的长距离影响.结合CNN与Bi-LSTM的优势, 既可以充分地提取特征, 又考虑了序列间信息长期依赖和上下信息间的关系, 故能充分地学习序列特征信息实现更好地分类预测.</p>
                </div>
                <div class="p1">
                    <p id="79">本文提出了一种融合CNN和Bi-LSTM的深度学习模型, 不仅避免机器学习特征提取时的人工干预, 而且利用了两者的优势, 充分考虑了miRNA-lncRNA序列间连续数据和非连续数据的相关性, 克服了无法长期依赖信息和充分提取特征的弊端, 从而更好地实现对miRNA-lncRNA互作关系的预测.</p>
                </div>
                <h3 id="80" name="80" class="anchor-tag"><b>2 数据预处理</b></h3>
                <div class="p1">
                    <p id="81">在本节中, 主要介绍对生物序列的数据预处理过程和对序列进行分词编码的步骤.</p>
                </div>
                <h4 class="anchor-tag" id="82" name="82"><b>2.1 数据集构建</b></h4>
                <div class="p1">
                    <p id="83">文中所用的玉米、马铃薯和小麦3个物种的lncRNA和miRNA数据分别是下载于GreeNC (http://greenc.sciencedesigners.com/wiki/) <citation id="285" type="reference"><link href="240" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation>和miRBase (http://mirbase.org/) <citation id="286" type="reference"><link href="242" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation>数据库.首先, 将每个物种去重后的lncRNA和miRNA数据分别上传到在线软件psRNATarget (https://plantgrn.noble.org/psRNATarget/analysis) <citation id="287" type="reference"><link href="244" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>上对应位置, 得出miRNA-lncRNA互作关系对中对应的miRNA和lncRNA名称, 根据名称从原始的miRNA和lncRNA序列中提取序列.对于互作关系对的序列, 如图1所示, 其处理步骤为:</p>
                </div>
                <div class="p1">
                    <p id="84">1) 为方便序列编码, 先将miRNA序列中U替换为T;</p>
                </div>
                <div class="p1">
                    <p id="85">2) 借鉴miTarget方法, 为区分miRNA和lncRNA连接处, 利用“LLLLLL”将对应的miRNA与lncRNA序列连接成单链序列;</p>
                </div>
                <div class="p1">
                    <p id="86">3) 对每个互作关系对重复以上步骤.</p>
                </div>
                <div class="area_img" id="87">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201908008_087.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 序列连接方式" src="Detail/GetImg?filename=images/JFYZ201908008_087.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 序列连接方式  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201908008_087.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 1 Sequence connection mode</p>

                </div>
                <div class="p1">
                    <p id="88">对psRNATarget软件得出的所有互作关系对经上述处理并去重后, 将其作为正样本.</p>
                </div>
                <div class="p1">
                    <p id="89">由于lncRNA序列长度远远大于miRNA, 在整合序列中lncRNA占比重大, 故将总lncRNA分为参与互作关系和未参与互作关系的lncRNA, 并采用Needleman-Wunsch算法将未参与互作关系的lncRNA与正样本中lncRNA进行相似性对比, 去除相似性高于80%的lncRNA样本<citation id="288" type="reference"><link href="246" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>, 最后, 将去相似性后的未参与互作关系的lncRNA与所有miRNA进行随机组合, 并进行如图1所示的处理, 从而得到负集样本库.为保证正负样本均衡, 采用随机抽样方法, 从中抽取与正样本数量相当的样本作为负集.</p>
                </div>
                <h4 class="anchor-tag" id="90" name="90"><b>2.2 序列编码</b></h4>
                <div class="p1">
                    <p id="91">对于整合后的miRNA-lncRNA序列, 利用基因组学中的生物序列的连续表示方式, 类似于自然语言处理中的分词<citation id="289" type="reference"><link href="248" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>, 将每条序列分成多个子序列 (生物单词) , 即每3个连续的碱基作为一个子序列, 子序列间不重叠.对正负样本中所有序列进行分词处理后, 统计得出一个大小为4×4×4=64的生物单词表, 按照单词在生物序列中出现的概率, 从大到小进行编码, 则可将每条序列样本嵌入到一个<i>n</i>维向量中, 即为模型的输入格式, 具体编码方式如图2所示:</p>
                </div>
                <div class="area_img" id="92">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201908008_092.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 序列编码方式" src="Detail/GetImg?filename=images/JFYZ201908008_092.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 序列编码方式  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201908008_092.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 2 Sequence coding mode</p>

                </div>
                <div class="p1">
                    <p id="93">如图2所示, 将输入序列<i>S</i>= (TATACGTGT…TGCAACCAG) , 按上述方案, 每3个连续碱基为一个词, 进行分词处理, 再按词频大小进行编码, 最终经程序运行后, 即可将<i>S</i>编码为一个固定长度的向量<b><i>SC</i></b>= (59, 55, 41, …, 18, 52, 16) , 即编码向量<b><i>SC</i></b>为模型的最终输入格式.</p>
                </div>
                <h3 id="94" name="94" class="anchor-tag"><b>3 CNN和Bi-LSTM融合模型</b></h3>
                <div class="p1">
                    <p id="95">提出的模型主要由嵌入阶段、卷积阶段和双向LSTM阶段组成.</p>
                </div>
                <h4 class="anchor-tag" id="96" name="96"><b>3.1 嵌入阶段</b></h4>
                <div class="p1">
                    <p id="97">嵌入阶段主要是将输入序列映射成矩阵向量的形式, 每一列对应一个字.即将输入序列中的每一个数字映射成一个具有固定长度的向量, 则输入序列被映射成<i>m</i>×<i>n</i>的矩阵形式, 其中, <i>m</i>为嵌入向量维度, <i>n</i>为序列长度.嵌入层作用是将一些关键的特征放大或把一些笼统的特征分开, 把数字序列映射成卷积层易处理的矩阵向量形式, 便于之后的卷积层充分提取特征.例如设编码向量为<b><i>SC</i></b>= (19, 2, 30, 42, 28) , 经嵌入层作用后, 即可映射为一个矩阵</p>
                </div>
                <div class="area_img" id="98">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JFYZ201908008_09800.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="100">本实验中嵌入层参数为输入维度是66, 输出维度是128, 输出长度是2 840.即经嵌入层后每条序列可映射成一个128×2 840向量, 作为卷积层的输入.</p>
                </div>
                <h4 class="anchor-tag" id="101" name="101"><b>3.2 卷积阶段</b></h4>
                <div class="p1">
                    <p id="102">由于1维卷积 (Convolution1D) 主要用于自然语言处理上, 2维卷积 (Convolution2D) 常用于计算机视觉上<citation id="290" type="reference"><link href="250" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>, 故实验模型卷积层选用Convolution1D函数.实验卷积阶段主要由2个卷积层构成.此外, 为防止过拟合, 在嵌入层和卷积层之间添加一个Dropout层, 其参数为0.5.</p>
                </div>
                <div class="p1">
                    <p id="103">第1层卷积层是利用64个长度为10的过滤器进行卷积, 即相当于利用64个10×128卷积核去探测经嵌入层映射的矩阵.即利用卷积核<b><i>W</i></b>对矩阵进行卷积操作:</p>
                </div>
                <div class="p1">
                    <p id="104"><i>x</i><sup><b><i>h</i></b></sup><sub><i>j</i></sub>=<i>f</i> (<i>x</i><mathml id="105"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>i</mi><mrow><mi mathvariant="bold-italic">h</mi><mo>-</mo><mn>1</mn></mrow></msubsup></mrow></math></mathml>*<b><i>W</i></b><sub><i>j</i></sub>+<i>b</i><sup><b><i>h</i></b></sup><sub><i>j</i></sub>) , (1) </p>
                </div>
                <div class="p1">
                    <p id="106">其中, <i>x</i><sup><b><i>h</i></b></sup><sub><i>j</i></sub>是第<i>j</i>个特征映像, <i>h</i>表示卷积层数;<b><i>W</i></b><sub><i>j</i></sub>表示第<i>j</i>个卷积核;<i>b</i><sup><b><i>h</i></b></sup><sub><i>j</i></sub>表示第<i>h</i>层中第<i>j</i>个特征映射的偏置数;*代表卷积操作.</p>
                </div>
                <div class="p1">
                    <p id="107">由于RELU函数比sigmoid函数具有便于稀疏化及有效减少梯度似然值的优势<citation id="291" type="reference"><link href="252" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>, 故卷积层的激活函数选用RELU函数:</p>
                </div>
                <div class="p1">
                    <p id="108"><i>RELU</i> (<i>x</i>) =max (0, <i>x</i>) . (2) </p>
                </div>
                <div class="p1">
                    <p id="109">经卷积操作后, 可提取尺寸为64×2831特征映射.再选用<i>pool</i>_<i>length</i>为2的MaxPooling对卷积出来的特征进行采样处理, 即对卷积出的特征的局部区域取最大值, 提取最重要的特征信息.故第1次卷积后输出维度为64×1415, 作为下一卷积层的输入.</p>
                </div>
                <div class="p1">
                    <p id="110">例如使用3个5×5的卷积核对64×64的矩阵进行卷积操作, 得到3个60×60的特征映射, 再使用2×2的池化窗口进行下采样, 即得到3个30×30的特征映射矩阵, 具体卷积阶段流程如图3所示:</p>
                </div>
                <div class="area_img" id="111">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201908008_111.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 卷积操作流程" src="Detail/GetImg?filename=images/JFYZ201908008_111.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 卷积操作流程  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201908008_111.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 3 Flow of convolution operation</p>

                </div>
                <div class="p1">
                    <p id="112">模型的第2个卷积层利用64个长度为5的过滤器卷积, 即等同于用5×64卷积核对上层提取的特征再次卷积, 则提取出特征映射尺寸为64×1 411, 再经最大池化采样后, 即可得尺寸为64×705的特征映射.将其作为双向LSTM层的输入.</p>
                </div>
                <h4 class="anchor-tag" id="113" name="113"><b>3.3 Bi-LSTM阶段</b></h4>
                <div class="p1">
                    <p id="114">LSTM是RNN的变体, 它通过设置输入门、遗忘门、输入门和记忆细胞来解决RNN梯度消失和梯度爆炸以及长期依赖缺失的问题, 但单向LSTM只能处理序列上一个方向的信息, 而无法处理另一个方向的信息.双向RNN (bidirecrional recurrent neural network) <citation id="292" type="reference"><link href="254" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>能同时捕获序列正反方向信息, 从而更好学习序列信息特征.Bi-LSTM则是为解决LSTM只能处理单一方向信息而做的进一步扩展, 其借鉴双向RNN的方法, 将双向RNN中循环单元替换为LSTM单元, 则Bi-LSTM等同于在每个训练序列的前后各连着一个单向LSTM, 且这2个单向LSTM都连接同一个层, 从正、反2个方向提取特征信息, 可以充分地学习更多的特征.图4是个双向循环神经网络<citation id="293" type="reference"><link href="248" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>.</p>
                </div>
                <div class="area_img" id="115">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201908008_115.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图5 模型整体结构" src="Detail/GetImg?filename=images/JFYZ201908008_115.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图5 模型整体结构  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201908008_115.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 5 The overall structure of model</p>

                </div>
                <div class="area_img" id="116">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201908008_116.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 双向循环神经网络" src="Detail/GetImg?filename=images/JFYZ201908008_116.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 双向循环神经网络  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201908008_116.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 4 Bidirectional recurrence neural network</p>

                </div>
                <div class="p1">
                    <p id="117">其中, 自左向右循环神经网络层的更新公式为</p>
                </div>
                <div class="p1">
                    <p id="118"><mathml id="119"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">h</mi><msub><mrow></mrow><mi>t</mi></msub></mrow><mo stretchy="true">→</mo></mover><mo>=</mo><mi>Η</mi><mo stretchy="false"> (</mo><mi mathvariant="bold-italic">W</mi><msub><mrow></mrow><mrow><mi>x</mi><mrow><mover><mstyle mathsize="140%" displaystyle="true"><mi mathvariant="bold-italic">h</mi></mstyle><mrow><mspace width="0.25em" /><mo>→</mo></mrow></mover><msub><mrow></mrow><mi>t</mi></msub></mrow></mrow></msub><mi mathvariant="bold-italic">x</mi><msub><mrow></mrow><mi>t</mi></msub><mo>+</mo><mi mathvariant="bold-italic">W</mi><msub><mrow></mrow><mrow><mover><mstyle mathsize="140%" displaystyle="true"><mi mathvariant="bold-italic">h</mi></mstyle><mrow><mspace width="0.25em" /><mo>→</mo></mrow></mover><mspace width="0.25em" /><mover><mstyle mathsize="140%" displaystyle="true"><mi mathvariant="bold-italic">h</mi></mstyle><mrow><mspace width="0.25em" /><mo>→</mo></mrow></mover></mrow></msub><mover accent="true"><mrow><mi mathvariant="bold-italic">h</mi><msub><mrow></mrow><mrow><mi>t</mi><mo>-</mo><mn>1</mn></mrow></msub></mrow><mo stretchy="true">→</mo></mover><mo>+</mo><mi mathvariant="bold-italic">b</mi><msub><mrow></mrow><mrow><mover><mstyle mathsize="140%" displaystyle="true"><mi mathvariant="bold-italic">h</mi></mstyle><mrow><mspace width="0.25em" /><mo>→</mo></mrow></mover></mrow></msub><mo stretchy="false">) </mo></mrow></math></mathml>. (3) </p>
                </div>
                <div class="p1">
                    <p id="120">自右向左循环神经网络层的更新公式为</p>
                </div>
                <div class="p1">
                    <p id="121"><mathml id="122"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">h</mi><msub><mrow></mrow><mi>t</mi></msub></mrow><mo stretchy="true">←</mo></mover><mo>=</mo><mi>Η</mi><mo stretchy="false"> (</mo><mi mathvariant="bold-italic">W</mi><msub><mrow></mrow><mrow><mi>x</mi><mrow><mover><mstyle mathsize="140%" displaystyle="true"><mi mathvariant="bold-italic">h</mi></mstyle><mrow><mspace width="0.25em" /><mo>←</mo></mrow></mover><msub><mrow></mrow><mi>t</mi></msub></mrow></mrow></msub><mi mathvariant="bold-italic">x</mi><msub><mrow></mrow><mi>t</mi></msub><mo>+</mo><mi mathvariant="bold-italic">W</mi><msub><mrow></mrow><mrow><mover><mstyle mathsize="140%" displaystyle="true"><mi mathvariant="bold-italic">h</mi></mstyle><mrow><mspace width="0.25em" /><mo>←</mo></mrow></mover><mspace width="0.25em" /><mover><mstyle mathsize="140%" displaystyle="true"><mi mathvariant="bold-italic">h</mi></mstyle><mrow><mspace width="0.25em" /><mo>←</mo></mrow></mover></mrow></msub><mover accent="true"><mrow><mi mathvariant="bold-italic">h</mi><msub><mrow></mrow><mrow><mi>t</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo stretchy="true">←</mo></mover><mo>+</mo><mi mathvariant="bold-italic">b</mi><msub><mrow></mrow><mrow><mover><mstyle mathsize="140%" displaystyle="true"><mi mathvariant="bold-italic">h</mi></mstyle><mrow><mspace width="0.25em" /><mo>←</mo></mrow></mover></mrow></msub><mo stretchy="false">) </mo></mrow></math></mathml>. (4) </p>
                </div>
                <div class="p1">
                    <p id="123">前后2层循环神经网络层叠加后输出为</p>
                </div>
                <div class="p1">
                    <p id="124"><mathml id="125"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi mathvariant="bold-italic">y</mi><msub><mrow></mrow><mi>t</mi></msub><mo>=</mo><mi mathvariant="bold-italic">W</mi><msub><mrow></mrow><mrow><mover><mstyle mathsize="140%" displaystyle="true"><mi mathvariant="bold-italic">h</mi></mstyle><mrow><mspace width="0.25em" /><mo>→</mo></mrow></mover><mi mathvariant="bold-italic">y</mi></mrow></msub><mover accent="true"><mrow><mi mathvariant="bold-italic">h</mi><msub><mrow></mrow><mi>t</mi></msub></mrow><mo stretchy="true">→</mo></mover><mo>+</mo><mi mathvariant="bold-italic">W</mi><msub><mrow></mrow><mrow><mover><mstyle mathsize="140%" displaystyle="true"><mi mathvariant="bold-italic">h</mi></mstyle><mrow><mspace width="0.25em" /><mo>←</mo></mrow></mover><mi mathvariant="bold-italic">y</mi></mrow></msub><mover accent="true"><mrow><mi mathvariant="bold-italic">h</mi><msub><mrow></mrow><mi>t</mi></msub></mrow><mo stretchy="true">←</mo></mover><mo>+</mo><mi mathvariant="bold-italic">b</mi><msub><mrow></mrow><mi mathvariant="bold-italic">y</mi></msub></mrow></math></mathml>, (5) </p>
                </div>
                <div class="p1">
                    <p id="126">其中, <i>t</i>表示时间序列;<b><i>h</i></b><sub><i>t</i></sub>表示时刻<i>t</i>时隐层向量, 其上箭头表示方向;<b><i>x</i></b><sub><i>t</i></sub>表示时刻<i>t</i>的输入;<b><i>y</i></b><sub><i>t</i></sub>表示时刻<i>t</i>的输出;<b><i>W</i></b><sub><b><i>xh</i></b></sub>表示输入-隐层的权重矩阵;<b><i>W</i></b><sub><b><i>hh</i></b></sub>表示隐层-隐层的权重矩阵;<b><i>W</i></b><sub><b><i>hy</i></b></sub>表示隐层-输出层的权重矩阵;<b><i>b</i></b><sub><b><i>h</i></b></sub>为隐层偏置向量;<b><i>b</i></b><sub><b><i>y</i></b></sub>为输出层偏置向量;<i>H</i>为隐层激活函数, 这里选取sigmoid函数.</p>
                </div>
                <div class="p1">
                    <p id="127">Bi-LSTM模型则是将图4中信息处理单元变换为LSTM模型单元, 利用LSTM的记忆细胞来处理长期依赖缺失, 并结合正反方向互补的信息, 更充分地学习序列数据的特征.其中, 本实验Bi-LSTM的隐层神经元个数为64, dropout参数设为0.3.</p>
                </div>
                <h4 class="anchor-tag" id="128" name="128"><b>3.4 模型实现</b></h4>
                <div class="p1">
                    <p id="129">实验模型是在Windows10系统下, 利用Python 3.6.5语言编写的一个基于TensorFlow 1.12.0的Keras框架, 模型主要由7层组成.</p>
                </div>
                <div class="p1">
                    <p id="130">模型首先利用嵌入层将输入序列映射成一个128×2 840的矩阵向量, 方便进行卷积操作;紧跟着使用参数为0.5的Dropout层来防止过拟合;通过2个卷积层进行卷积操作, 并使用最大池化操作过滤出重要的局部特征信息, 经RELU函数激励后, 则将矩形向量转变成一个64×705维特征映射作为Bi-LSTM层输入;利用Bi-LSTM结合上下文信息的优势, 充分学习特征之间的依赖关系, 将卷积阶段输出的特征映射向量变为一个128维向量;最后, 使用参数为1的密集层Dense层将Bi-LSTM输出的特征向量映射成一个具体数字, 并使用sigmoid函数将该数字映射在[0, 1]之间, 即得出预测结果.根据真实值和预测值间的损失, 通过BP算法逐层计算, 更新参数, 完成一轮训练.其模型的整体结构如图5所示.</p>
                </div>
                <h3 id="131" name="131" class="anchor-tag"><b>4 实验与结果</b></h3>
                <div class="p1">
                    <p id="132">基于玉米 (<i>zea mays</i>) 、马铃薯 (<i>solanum tuberosum</i>) 和小麦 (<i>triticum aestivum</i>) 数据集, 通过对传统机器学习方法和不同物种进行测试来验证提出的模型对miRNA-lncRNA互作关系的预测能力和泛化能力.</p>
                </div>
                <h4 class="anchor-tag" id="133" name="133"><b>4.1 验证方法与评价标准</b></h4>
                <div class="p1">
                    <p id="134">实验采用5折交叉验证法来验证模型的性能.5折交叉验证思想是将数据集均分为5份, 轮流取其中一份作为验证集, 其余4份作为训练集, 5次结果的均值作为最终评估值.实验选取准确值 (accuracy, <i>Acc</i>) 、精确率 (precision, <i>P</i>) 、召回率 (recall, <i>R</i>) 和<i>F</i>1分数 (<i>F</i>1_score, <i>F</i>1) 作为评价指标:</p>
                </div>
                <div class="p1">
                    <p id="135"><mathml id="136"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>A</mi><mi>c</mi><mi>c</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>Τ</mi><mi>Ν</mi></mrow><mrow><mi>Τ</mi><mi>Ν</mi><mo>+</mo><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac></mrow></math></mathml>, (6) </p>
                </div>
                <div class="p1">
                    <p id="137"><mathml id="138"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ρ</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi></mrow></mfrac></mrow></math></mathml>, (7) </p>
                </div>
                <div class="p1">
                    <p id="139"><mathml id="140"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>R</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac></mrow></math></mathml>, (8) </p>
                </div>
                <div class="p1">
                    <p id="141"><mathml id="142"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>F</mi><mn>1</mn><mo>=</mo><mfrac><mrow><mn>2</mn><mo>×</mo><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mn>2</mn><mo>×</mo><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi></mrow></mfrac></mrow></math></mathml>, (9) </p>
                </div>
                <div class="p1">
                    <p id="143">其中, <i>TP</i>, <i>FP</i>, <i>TN</i>, <i>FN</i>的含义如表1所示:</p>
                </div>
                <div class="area_img" id="144">
                    <p class="img_tit"><b>表1 分类结果含义</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 1 Meaning of Classification Results</b></p>
                    <p class="img_note"></p>
                    <table id="144" border="1"><tr><td rowspan="2"><br />Real Result</td><td colspan="2"><br />Predicted Result</td></tr><tr><td><br />Positive Class</td><td>Negative Class</td></tr><tr><td><br />Positive Class</td><td><i>TP</i></td><td><i>FN</i></td></tr><tr><td><br />Negative Class</td><td><i>FP</i></td><td><i>TN</i></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h4 class="anchor-tag" id="145" name="145"><b>4.2 数据集</b></h4>
                <div class="p1">
                    <p id="146">按照第1节介绍方法从GreeNC和miRBase数据库中下载了玉米、马铃薯和小麦的相关数据, 并进行第2节的数据预处理过程.首先采用玉米数据集, 用传统机器学习方法、单一模型与提出的模型进行对比实验, 从而验证提出方法的有效性;另外, 采用马铃薯和小麦为数据集, 用提出模型分别对两者进行独立测试来验证提出模型的泛化能力.为保证正负样本均衡, 从负集样本库中随机抽取与正集相同数目的样本作为负集.各物种数据集具体数据如表2所示:</p>
                </div>
                <div class="area_img" id="147">
                    <p class="img_tit"><b>表2 各物种数据集</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 2 Datasets of Each Species</b></p>
                    <p class="img_note"></p>
                    <table id="147" border="1"><tr><td><br />Dataset</td><td><i>Zea</i><br /><i>Mays</i></td><td><i>Solanum</i><br /><i>Tuberosum</i></td><td><i>Triticum</i><br /><i>Aestivum</i></td></tr><tr><td><br />lncRNA</td><td>17 684</td><td>6 565</td><td>38 051</td></tr><tr><td><br />miRNA</td><td>207</td><td>244</td><td>121</td></tr><tr><td><br />miRNA-lncRNA</td><td>18 488</td><td>17 542</td><td>15 787</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h4 class="anchor-tag" id="148" name="148"><b>4.3 特征提取</b></h4>
                <div class="p1">
                    <p id="149">基于传统的miRNA与lncRNA的提取方法<citation id="294" type="reference"><link href="256" rel="bibliography" /><sup>[<a class="sup">23</a>]</sup></citation>, 分别提取玉米miRNA, lncRNA相关特征, 并将两者特征组成多维特征集作为机器学习的特征向量.</p>
                </div>
                <div class="p1">
                    <p id="150">首先, 利用ViennaRNA<citation id="295" type="reference"><link href="258" rel="bibliography" /><sup>[<a class="sup">24</a>]</sup></citation>中RNAfold软件得到lncRNA序列形成二级结构时释放的自由能<i>MFE</i>和其二级结构的点括号形式<citation id="296" type="reference"><link href="260" rel="bibliography" /><sup>[<a class="sup">25</a>]</sup></citation>, 并从中提取配对碱基数、 (C+G) 碱基含量和G, C的比例, 即可得到最小自由能<i>MFE</i>、配对碱基数<i>n</i>_<i>pairs</i>、 (C+G) 含量<i>CG</i>_<i>content</i>和<i>GC</i>_<i>ratio</i>四个特征, 融合后特征记为<i>Feature</i><sub>1</sub>:</p>
                </div>
                <div class="p1">
                    <p id="151"><i>CG</i>_<i>content</i>= (<i>C</i>_<i>num</i>+<i>G</i>_<i>num</i>) /<i>L</i>, (10) </p>
                </div>
                <div class="p1">
                    <p id="152"><i>GC</i>_<i>ratio</i>=<i>G</i>_<i>num</i>/<i>C</i>_<i>num</i>, (11) </p>
                </div>
                <div class="p1">
                    <p id="153"><i>Feature</i><sub>1</sub>=[<i>MFE</i>, <i>n</i>_<i>pairs</i>, <i>CG</i>_<i>content</i>, </p>
                </div>
                <div class="p1">
                    <p id="154"><i>GC</i>_<i>ratio</i>], (12) </p>
                </div>
                <div class="p1">
                    <p id="155">其中, <i>C</i>_<i>num</i>为序列中碱基C的数目;<i>G</i>_<i>num</i>为序列中碱基G的数目;<i>L</i>为序列的长度.</p>
                </div>
                <div class="p1">
                    <p id="156">此外, 还提取lncRNA的<i>k</i>-mers特征.一个<i>k</i>-mers由<i>k</i>个碱基组成, 则1-mer={A, T, C, G}有4种, 2-mer={AA, AT, AC, AG, …}, 每个碱基可以是A, T, C或G, 故有4×4=16种, 实验中<i>k</i>=1, 2.<i>k</i>-mers提取方法为:沿lncRNA序列使用长度为<i>k</i>的滑动窗口以步长为1碱基进行滑动匹配, 则有:</p>
                </div>
                <div class="p1">
                    <p id="157"><mathml id="158"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>h</mi><msub><mrow></mrow><mi>i</mi></msub><mo>=</mo><mi>w</mi><msub><mrow></mrow><mi>k</mi></msub><mfrac><mrow><mi>m</mi><msub><mrow></mrow><mi>i</mi></msub></mrow><mrow><mi>s</mi><msub><mrow></mrow><mi>k</mi></msub></mrow></mfrac><mo>, </mo><mi>k</mi><mo>=</mo><mn>1</mn><mo>, </mo><mn>2</mn><mo>, </mo><mi>i</mi><mo>=</mo><mn>1</mn><mo>, </mo><mn>2</mn><mo>, </mo><mo>⋯</mo><mo>, </mo><mn>2</mn><mn>0</mn></mrow></math></mathml>; (13) </p>
                </div>
                <div class="p1">
                    <p id="159"><i>s</i><sub><i>k</i></sub>=<i>l</i>-<i>k</i>+1, <i>k</i>=1, 2; (14) </p>
                </div>
                <div class="p1">
                    <p id="160"><mathml id="161"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>w</mi><msub><mrow></mrow><mi>k</mi></msub><mo>=</mo><mfrac><mn>1</mn><mrow><mn>4</mn><msup><mrow></mrow><mrow><mn>3</mn><mo>-</mo><mi>k</mi></mrow></msup></mrow></mfrac><mo>, </mo><mi>k</mi><mo>=</mo><mn>1</mn><mo>, </mo><mn>2</mn></mrow></math></mathml>; (15) </p>
                </div>
                <div class="p1">
                    <p id="162">其中, <i>w</i><sub><i>k</i></sub>为权重;<i>s</i><sub><i>k</i></sub>为匹配总数;<i>l</i>为序列长度;<i>m</i><sub><i>i</i></sub>为每个<i>k</i>-mers的匹配数;<i>h</i><sub><i>i</i></sub>为每个<i>k</i>-mers的频率, 则可得到lncRNA的4+16=20个<i>k</i>-mers特征, 记为<i>Feature</i><sub>2</sub>:</p>
                </div>
                <div class="p1">
                    <p id="163"><i>Feature</i><sub>2</sub>=[<i>h</i><sub>1</sub>, <i>h</i><sub>2</sub>, <i>h</i><sub>3</sub>, …, <i>h</i><sub>20</sub>]. (16) </p>
                </div>
                <div class="p1">
                    <p id="164">对miRNA序列提取特征为序列长度<i>m</i>_<i>l</i>和miRNA的<i>k</i>-mers特征, 其中<i>k</i>=1, 2, 则可得1+4+16=21个miRNA的特征, 记为<i>Feature</i><sub>3</sub>:</p>
                </div>
                <div class="p1">
                    <p id="165"><i>Feature</i><sub>3</sub>=[<i>m</i>_<i>l</i>, <i>m</i>_<i>h</i><sub>1</sub>, <i>m</i>_<i>h</i><sub>2</sub>, <i>m</i>_<i>h</i><sub>3</sub>, …, <i>m</i>_<i>h</i><sub>20</sub>]. (17) </p>
                </div>
                <div class="p1">
                    <p id="166">最终将lncRNA的特征<i>Feature</i><sub>1</sub>, <i>Feature</i><sub>2</sub>和miRNA的特征<i>Feature</i><sub>3</sub>组成一个4+20+21=45维的特征集, 将其作为传统机器学习的特征向量<b><i>Feature</i></b>:</p>
                </div>
                <div class="p1">
                    <p id="167"><b><i>Feature</i></b>= (<i>Feature</i><sub>1</sub>, <i>Feature</i><sub>2</sub>, <i>Feature</i><sub>3</sub>) . (18) </p>
                </div>
                <h4 class="anchor-tag" id="168" name="168"><b>4.4 基于不同方法的分类结果</b></h4>
                <div class="p1">
                    <p id="169">为验证提出方法的有效性和优势, 实验以玉米为数据集, 按照3.3节方法对miRNA和lncRNA的特征进行提取后融合, 并使用朴素贝叶斯 (Naive Bayes, NB) <citation id="297" type="reference"><link href="262" rel="bibliography" /><sup>[<a class="sup">26</a>]</sup></citation>、梯度提升决策树 (gradient boosting decision tree, GBDT) <citation id="298" type="reference"><link href="264" rel="bibliography" /><sup>[<a class="sup">27</a>]</sup></citation>、随机森林 (random forest, RF) <citation id="299" type="reference"><link href="266" rel="bibliography" /><sup>[<a class="sup">28</a>]</sup></citation>和决策树 (decision tree, DT) <citation id="300" type="reference"><link href="268" rel="bibliography" /><sup>[<a class="sup">29</a>]</sup></citation>方法进行分类预测, 进行对比实验, 5折交叉检验的实验结果如表3所示:</p>
                </div>
                <div class="area_img" id="170">
                    <p class="img_tit"><b>表3 基于不同方法的分类结果</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 3 Classification Results Based on Different Methods</b></p>
                    <p class="img_note"> %</p>
                    <table id="170" border="1"><tr><td><br />Model</td><td><i>Acc</i>±<i>SD</i><sup>②</sup></td><td><i>P</i></td><td><i>R</i></td><td><i>F</i>1</td></tr><tr><td><br />NB</td><td>80.35±0.78<sup>①d</sup></td><td>81.55</td><td>78.68</td><td>80.09</td></tr><tr><td><br />GBDT</td><td>90.55±0.74<sup>①c</sup></td><td>94.43</td><td>86.29</td><td>90.16</td></tr><tr><td><br />RF</td><td>91.14±0.85<sup>c</sup></td><td>98.66</td><td>83.49</td><td>90.45</td></tr><tr><td><br />DT</td><td>94.51±0.64<sup>①b</sup></td><td>95.46</td><td>93.51</td><td>94.48</td></tr><tr><td><br />CNN</td><td>95.40±0.76<sup>b</sup></td><td>99.64</td><td>91.12</td><td>95.19</td></tr><tr><td><br />BiLSTM</td><td>92.36±0.67<sup>c</sup></td><td>96.97</td><td>87.43</td><td>91.95</td></tr><tr><td><br />Ours</td><td>97.70±0.60<sup>①a</sup></td><td>99.86</td><td>95.53</td><td>97.65</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note">Note: ① a, b, c, d, e:same letter means difference in <i>Acc</i> is not significant, otherwise, it is significant ② <i>SD</i>: Standard deviation</p>
                </div>
                <div class="p1">
                    <p id="171">从表3可看出, 较之于传统机器学习方法, 提出方法在准确率、精确率、召回率和<i>F</i>1值4个指标上性能都具有明显的优势;其中, 在准确率上比NB, GBDT, RF和DT方法分别高出17.35%, 7.15%, 6.56%和3.19%, 说明提出方法在预测miRNA-lncRNA有无互作关系上具有良好的分类能力.同时, 与单一模型CNN和Bi-LSTM相比, 融合模型兼顾两者优势, 既能提取丰富的特征, 又可解决长距离信息依赖缺失问题, 比单一模型性能略有提升.此外, 从最小显著性差异法 (LSD) 分析结果看, 提出方法显著优于其他方法且准确率的标准差 (<i>SD</i>) 仅为0.60%, 表明模型的稳定性较好.</p>
                </div>
                <div class="p1">
                    <p id="172">图6描绘出在玉米测试集上, 不同方法下的ROC曲线.从结果可看出, 较之于机器学习模型与单一模型, 融合模型的ROC曲线下面积最大, 其面积即<i>AUC</i>值高达0.99以上几乎接近于1, 十分接近真实情况, 表明模型的分类效果十分显著.</p>
                </div>
                <div class="area_img" id="173">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JFYZ201908008_173.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图6 不同方法在测试集上的ROC曲线" src="Detail/GetImg?filename=images/JFYZ201908008_173.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图6 不同方法在测试集上的ROC曲线  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JFYZ201908008_173.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 6 ROC curves of different methods on test sets</p>

                </div>
                <h4 class="anchor-tag" id="174" name="174"><b>4.5 基于不同物种的分类结果</b></h4>
                <div class="p1">
                    <p id="175">为证明提出方法的泛化能力, 选用马铃薯和小麦数据集作为独立测试集, 进行模型测试, 通过2个不同物种的实验结果, 证明提出方法泛化能力好且适合于大多数物种.5折交叉检验的实验结果如表4所示:</p>
                </div>
                <div class="area_img" id="176">
                    <p class="img_tit"><b>表4 基于不同物种的分类结果</b> % <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 4 Classification Results Based on Different Species</b></p>
                    <p class="img_note"></p>
                    <table id="176" border="1"><tr><td><br />Species</td><td><i>Acc</i>±<i>SD</i></td><td><i>P</i></td><td><i>R</i></td><td><i>F</i>1</td></tr><tr><td><br /><i>Solanum Tuberosum</i></td><td>95.57±0.57</td><td>98.33</td><td>92.80</td><td>95.49</td></tr><tr><td><br /><i>Triticum Aestivum</i></td><td>93.81±0.81</td><td>97.57</td><td>89.91</td><td>93.58</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="177">从表4结果可看出, 提出方法在预测马铃薯和小麦的miRNA-lncRNA有无互作关系上各方面性能指标都较好, 表明模型具有良好泛化能力, 适合大多数物种.此外, 两者的方差都较小, 说明不同物种数据下, 模型的稳定性也较好.</p>
                </div>
                <h3 id="178" name="178" class="anchor-tag"><b>5 结束语</b></h3>
                <div class="p1">
                    <p id="179">文中提出一种融合CNN和Bi-LSTM的深度学习模型, 兼顾CNN和Bi-LSTM的优势, 充分考虑了序列数据间的相关性和较好地结合了上下文信息, 从而充分地提取特征.实验结果表明:模型与传统机器学习、单一模型对比, 取得了优于传统机器学习和单一模型的分类效果;此外, 在马铃薯和小麦数据集上进行独立测试, 也取得良好的分类效果, 验证了提出模型具有不错的泛化能力, 适合于大多数物种测试.</p>
                </div>
                <div class="p1">
                    <p id="180">未来将会尝试使用更多的模型, 如胶囊网络、深度信念网络等来进一步改善对miRNA-lncRNA互作关系的预测;另外, 结合机器学习和深度学习方法, 提高预测性能也是将来的研究方向.</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="212">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Mechanisms of long noncoding RNA function in development and disease">

                                <b>[1]</b>Schmitz S U.Mechanisms of long noncoding RNA function in development and disease[J].Cellular and Molecular Life Sciences, 2016, 73 (13) :2491- 2509
                            </a>
                        </p>
                        <p id="214">
                            <a id="bibliography_2" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZHDZ201702020&amp;v=MDE5MzhRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJyRnl2Z1Y3M1BQeVhQZExHNEg5Yk1yWTlIWkk=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[2]</b>Hu Mingyang, Ming Jia.Research progress of the regulatory mechanisms between lncRNAs and miRNAs[J].E-Journal of Translational Medicine, 2017, 4 (2) :71- 76 (in Chinese) (胡铭阳, 明佳.lncRNA与miRNA相互调控机制的研究进展[J].转化医学电子杂志, 2017, 4 (2) :71- 76) 
                            </a>
                        </p>
                        <p id="216">
                            <a id="bibliography_3" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES14121800006404&amp;v=MTU0MzJhQlU9TmlmT2ZiSzhIOVBOcDQ5RlpPc0pDSHc5b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadUh5am1VTHJJSTFzUQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[3]</b>Cesana M, Cacchihiarelli D, Legnini I, et al.A long noncoding RNA controls muscle differentiation by function-ing as a competing endogenous RNA[J].Cell, 2011, 147 (2) :358- 369
                            </a>
                        </p>
                        <p id="218">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Reconstruction and analysis of the lncRNA-miRNA-mRNA network based on competitive endogenous RNA reveal functional lncRNAs in rheumatoid arthritis">

                                <b>[4]</b>Jiang Hui, Ma Rong, Zou Shubiao, et al.Reconstruction and analysis of the lncRNA-miRNA-mRNA network based on competitive endogenous RNA reveal functional lncRNAs in rheumatoid arthritis[J].Molecular Biosystems, 2017, 13 (6) :1182- 1192
                            </a>
                        </p>
                        <p id="220">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=miTarget: microRNA target gene prediction using a support vector machine">

                                <b>[5]</b>Kim S K, Nam J W, Rhee J K, et al.miTarget:MicroRNA target gene prediction using a support vector machine[J].BMC Bioinformatics, 2006, 7 (1) :411- 423
                            </a>
                        </p>
                        <p id="222">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Continuous Distributed Representation of Biological Sequences for Deep Proteomics and Genomics">

                                <b>[6]</b>Asgari E, Mofrad M R K.Continuous distributed representation of biological sequences for deep proteomics and genomics[J].PLOS ONE, 2015, 10 (11) :e0141287
                            </a>
                        </p>
                        <p id="224">
                            <a id="bibliography_7" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JZUS201801007&amp;v=MTQ3OTJPM3pxcUJ0R0ZyQ1VSTE9lWmVSckZ5dmdWNzNQTHpmZWZiRzRIOW5Ncm85Rlk0UUtESDg0dlI0VDZqNTQ=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[7]</b>Cheng Jian, Wang Peisong, Li Gang, et al.Recent advances in efficient computation of deep convolutional neural networks[J].Frontiers of Information Technology &amp; Electronic Engineering, 2018, 19 (1) :64- 77
                            </a>
                        </p>
                        <p id="226">
                            <a id="bibliography_8" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJBK&amp;filename=SJBK15090500014198&amp;v=MzAyOTFQUUgvaXJSZEdlcnFRVE1ud1plWnVIeWptVUxySUkxc1FhQlU9TmlmSlpiSzlIdGpNcW85RlpPb0xEWFV4b0JNVDZUNA==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[8]</b>Hochreiter S, Schmidhuber J.Long short-term memory[J].Neural Computation, 1997, 9 (8) :1735- 1780
                            </a>
                        </p>
                        <p id="228">
                            <a id="bibliography_9" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=QLZL201801005&amp;v=MTg4NzdCdEdGckNVUkxPZVplUnJGeXZnVjczUE5DSFJZckc0SDluTXJvOUZZWVFLREg4NHZSNFQ2ajU0TzN6cXE=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[9]</b>Hou Min, Jiang Lin, Zhan Hongmei, et al.Prediction of breast cancer related lncRNA-miRNA-mRNA co-expression and key genes based on network construction method[J].Chinese Journal of Cancer Prevention and Treatment, 2018, 25 (1) :26- 33 (in Chinese) (侯敏, 蒋琳, 詹红梅, 等.乳腺癌相关lncRNA-miRNA-mRNA共表达及关键基因网络构建预测[J].中华肿瘤防治杂志, 2018, 25 (1) :26- 33) 
                            </a>
                        </p>
                        <p id="230">
                            <a id="bibliography_10" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJHD&amp;filename=SJHD347D5FAA785F565F26433BF67FAE1B34&amp;v=MzE2OTViQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TjFoeDcyN3dxOD1OaWZEYXJDOEdhWEoyZjQwWStNS2Vuay95bUFSN0R0K1N3MlVxaFZEQ01lVk43bQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[10]</b>Han Siyu, Liang Yanchun, Li Ying, et al.Long noncoding RNA identification:Comparing machine learning based tools for long noncoding transcripts discrimination[J].BioMed Research International, 2016, (2016) :1- 14
                            </a>
                        </p>
                        <p id="232">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Reducing the dimensionality of data with neural networks">

                                <b>[11]</b>Hinton G E, Salakhutdinov R R.Reducing the dimensionality of data with neural networks[J].Science, 2006, 313 (5786) :504- 507
                            </a>
                        </p>
                        <p id="234">
                            <a id="bibliography_12" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJBK&amp;filename=SJBK15090500012713&amp;v=MTkwMzdCVT1OaWZKWmJLOUh0ak1xbzlGWk9vTkMzMDZvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMcklJMXNRYQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[12]</b>Williams R J, Zipser D.A learning algorithm for continually running fully recurrent neural networks[J].Neural Computation, 1989, 1 (2) :270- 280
                            </a>
                        </p>
                        <p id="236">
                            <a id="bibliography_13" target="_blank" href="http://scholar.cnki.net/result.aspx?q=DeepLNC,a long non-coding RNA prediction tool using deep neural network">

                                <b>[13]</b>Tripathi R, Patel S, Kumari V, et al.DeepLNC, a long non-coding RNA prediction tool using deep neural network[J].Network Modeling Analysis in Health Informatics and Bioinformatics, 2016, 5 (1) :No.21
                            </a>
                        </p>
                        <p id="238">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=lncRNAnet:Long non-coding RNA identification using deep learning">

                                <b>[14]</b>Junghwan B, Byunghan L, Sunyoung K, et al.lncRNAnet:Long non-coding RNA identification using deep learning[J].Bioinformatics, 2018, 34 (22) :3889- 3897
                            </a>
                        </p>
                        <p id="240">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=GREENC:a Wikibased database of plant lnc RNAs">

                                <b>[15]</b>Andreu Paytuví Gallart, Pulido A H, Irantzu Anzar Martínez De Lagrán, et al.GREENC:A wiki-based database of plant lncRNAs[J].Nucleic Acids Research, 2016, 44 (D1) :D1161- D1166
                            </a>
                        </p>
                        <p id="242">
                            <a id="bibliography_16" target="_blank" href="http://scholar.cnki.net/result.aspx?q=miRBase: the microRNA sequence data- base">

                                <b>[16]</b>Griffithsjones S.miRBase:The microRNA sequence database[J].Methods in Molecular Biology, 2006, 342:129- 138
                            </a>
                        </p>
                        <p id="244">
                            <a id="bibliography_17" target="_blank" href="http://scholar.cnki.net/result.aspx?q=psRNATarget:A plant small RNA target analysis server (2017 release)">

                                <b>[17]</b>Dai Xinbin, Zhuang Zhaohong, Zhao P X.psRNATarget:A plant small RNA target analysis server (2017 release) [J].Nucleic Acids Research, 2018, 46:W49- W54
                            </a>
                        </p>
                        <p id="246">
                            <a id="bibliography_18" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Pattern recognition analysis on long noncoding RNAs:A tool for prediction in plants">

                                <b>[18]</b>Negri T, Alves W, Bugatti P H, et al.Pattern recognition analysis on long noncoding RNAs:A tool for prediction in plants[J].Briefings in Bioinformatics, 2019, 20 (2) :682- 689
                            </a>
                        </p>
                        <p id="248">
                            <a id="bibliography_19" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=MESS201802004&amp;v=MTMxNDNZZmJHNEg5bk1yWTlGWUlRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJyRnl2Z1Y3M1BLQ2o=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[19]</b>Jin Chen, Li Weihua, Ji Chen, et al.Bidirectional long shortterm memory neural networks for Chinese word segmentation[J].Journal of Chinese Information Processing, 2018, 32 (2) :29- 37 (in Chinese) (金宸, 李维华, 姬晨, 等.基于双向LSTM神经网络模型的中文分词[J].中文信息学报, 2018, 32 (2) :29- 37) 
                            </a>
                        </p>
                        <p id="250">
                            <a id="bibliography_20" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201809002&amp;v=MTI3MTZWNzNQTHl2U2RMRzRIOW5NcG85RlpvUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSckZ5dmc=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[20]</b>Zhang Hao, Wu Jianxin.A survey on unsupervised image retrieval using deep features[J].Journal of Computer Research and Development, 2018, 55 (9) :1829- 1842 (in Chinese) (张皓, 吴建鑫.基于深度特征的无监督图像检索研究综述[J].计算机研究与发展, 2018, 55 (9) :1829- 1842) 
                            </a>
                        </p>
                        <p id="252">
                            <a id="bibliography_21" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201801006&amp;v=MDk1NTQ5RllvUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSckZ5dmdWNzNQTHl2U2RMRzRIOW5Ncm8=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[21]</b>Li Hongshun, Yu Hua, Gong Xiujun.A deep learning model for predicting RNA-binding proteins only from primary sequences[J].Journal of Computer Research and Development, 2018, 55 (1) :93- 101 (in Chinese) (李洪顺, 于华, 宫秀军.一种只利用序列信息预测RNA结合蛋白的深度学习模型[J].计算机研究与发展, 2018, 55 (1) :93- 101) 
                            </a>
                        </p>
                        <p id="254">
                            <a id="bibliography_22" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Bidirectional recurrent neural networks">

                                <b>[22]</b>Schuster M, Paliwal K K.Bidirectional recurrent neural network[J].IEEE Transactions on Signal Processing, 1997, 45 (11) :2673- 2681
                            </a>
                        </p>
                        <p id="256">
                            <a id="bibliography_23" target="_blank" href="http://scholar.cnki.net/result.aspx?q=MicroRNA identification based on sequence and structure alignment">

                                <b>[23]</b>Wang Xiaowo, Zhang Jing, Li Fei, et al.MicroRNA identification based on sequence and structure alignment[J].Bioinfor-matics, 2005, 21 (18) :3610- 3614
                            </a>
                        </p>
                        <p id="258">
                            <a id="bibliography_24" target="_blank" href="http://scholar.cnki.net/result.aspx?q=ViennaRNA Package 2.0">

                                <b>[24]</b>Lorenz R, Bernhart S H, Siederdissen C H Z, et al.ViennaRNA package 2.0[J].Algorithms for Molecular Biology, 2011, 6:No.26
                            </a>
                        </p>
                        <p id="260">
                            <a id="bibliography_25" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZNXT201806009&amp;v=Mjg0NDk5bk1xWTlGYllRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJyRnl2Z1Y3M1BQeVBUZXJHNEg=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[25]</b>Chang Zheng, Meng Jun, Shi Yunsheng, et al.LncRNA recognition by fusing multiple features and its function prection[J].CAAI Transactions on Intelligent Systems, 2018, 13 (6) :928- 934 (in Chinese) (常征, 孟军, 施云生, 等.多特征融合的lncRNA识别与其功能预测[J].智能系统学报, 2018, 13 (6) :928- 934) 
                            </a>
                        </p>
                        <p id="262">
                            <a id="bibliography_26" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES295D525CA48899AA0EFBFCA91EE6B470&amp;v=MDQ0NDNUbjBrUFBneVRwUk5BRExUbVFiMmZDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOMWh4NzI3d3E4PU5pZk9mYkd4RzZYSnJZbzJGZThIQkhVd3ZtYw==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[26]</b>Gao Chongzhi, Cheng Qiong, Pei He, et al.Privacy-preserving Naive Bayes classifiers secure against the substitution-then-comparison attack[J].Information Sciences, 2018, 444:72- 88
                            </a>
                        </p>
                        <p id="264">
                            <a id="bibliography_27" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prioritizing Influential Factors for Freeway Incident Clearance Time Prediction Using the Gradient Boosting Decision Trees Method">

                                <b>[27]</b>Ma Xiaolei, Ding Chuan, Luan Sen, et al.Prioritizing influential factors for freeway incident clearance time prediction using the gradient boosting decision trees method[J].IEEE Transactions on Intelligent Transportation Systems, 2017, 18 (9) :1- 8
                            </a>
                        </p>
                        <p id="266">
                            <a id="bibliography_28" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011501968360&amp;v=MDg2NzhxUVRNbndaZVp1SHlqbVVMcklJMXNRYUJVPU5pZk9mYks3SHRETnFvOUViZTBIRDNvNW9CTVQ2VDRQUUgvaXJSZEdlcg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[28]</b>Rodriguez-Galiano V F, Ghimire B, Rogan J, et al.An assessment of the effectiveness of a random forest classifier for land-cover classification[J].Isprs Journal of Photo-grammetry &amp; Remote Sensing, 2012, 67 (1) :93- 104
                            </a>
                        </p>
                        <p id="268">
                            <a id="bibliography_29" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A Survey of Decision Tree Classifier Methodology">

                                <b>[29]</b>Landgrebe D.A survey of decision tree classifier methodology[J].IEEE Transactions on Systems Man &amp; Cybernetics, 2002, 21 (3) :660- 674
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JFYZ201908008" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>


    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFYZ201908008&amp;v=MTk4NzZiSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUnJGeXZnVjczUEx5dlNkTEc0SDlqTXA0OUY=&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVG5hVDYzUnVIR29xNjE2VHo1YnlSMVNQcz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>


    <link href="/kxreader/Content/css/LeftDetail?v=NLcKG8I1SJUaVFrQ0iGpF2klAT0OsmHRaVSZ1rKb5xg1" rel="stylesheet"/>

</body>
</html>

