<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=NOOK-mvTMnxl11s6oPYV0boPXN3iP4N7Rc-A56nk4KI1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2fKXReader%2fDetail%3fTIMESTAMP%3d637139250879975000%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJSJZ201910073%26RESULT%3d1%26SIGN%3dZQWjORWBs5cW8WzlMYmvyKIR18w%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJZ201910073&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJZ201910073&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJZ201910073&amp;v=MDc4MTJyNDlDWjRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpwRnkvblU3ekJMejdCZExHNEg5ak4=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#43" data-title="&lt;b&gt;1 引言&lt;/b&gt; "><b>1 引言</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#50" data-title="&lt;b&gt;2 数据与方法&lt;/b&gt; "><b>2 数据与方法</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#51" data-title="&lt;b&gt;2.1 数据集&lt;/b&gt;"><b>2.1 数据集</b></a></li>
                                                <li><a href="#54" data-title="&lt;b&gt;2.2 肽序列的长度分布&lt;/b&gt;"><b>2.2 肽序列的长度分布</b></a></li>
                                                <li><a href="#57" data-title="&lt;b&gt;2.3 肽序列编码&lt;/b&gt;"><b>2.3 肽序列编码</b></a></li>
                                                <li><a href="#61" data-title="&lt;b&gt;2.4 CNN-LSTM框架&lt;/b&gt;"><b>2.4 CNN-LSTM框架</b></a></li>
                                                <li><a href="#64" data-title="&lt;b&gt;2.5 深度卷积神经网络的结构&lt;/b&gt;"><b>2.5 深度卷积神经网络的结构</b></a></li>
                                                <li><a href="#67" data-title="&lt;b&gt;2.6 评价指标&lt;/b&gt;"><b>2.6 评价指标</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#69" data-title="&lt;b&gt;3 结果与分析&lt;/b&gt; "><b>3 结果与分析</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#70" data-title="&lt;b&gt;3.1 模型参数的优化&lt;/b&gt;"><b>3.1 模型参数的优化</b></a></li>
                                                <li><a href="#78" data-title="&lt;b&gt;3.2 与其它两种深度学习方法的比较&lt;/b&gt;"><b>3.2 与其它两种深度学习方法的比较</b></a></li>
                                                <li><a href="#82" data-title="&lt;b&gt;3.3 在训练集和测试集上的准确率和损失函数分布&lt;/b&gt;"><b>3.3 在训练集和测试集上的准确率和损失函数分布</b></a></li>
                                                <li><a href="#85" data-title="&lt;b&gt;3.4 与已有的预测方法的比较&lt;/b&gt;"><b>3.4 与已有的预测方法的比较</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#88" data-title="&lt;b&gt;4 结束语&lt;/b&gt; "><b>4 结束语</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#53" data-title="&lt;b&gt;表1 数据集的统计&lt;/b&gt;"><b>表1 数据集的统计</b></a></li>
                                                <li><a href="#56" data-title="&lt;b&gt;图1 CPP序列的长度分布&lt;/b&gt;"><b>图1 CPP序列的长度分布</b></a></li>
                                                <li><a href="#59" data-title="&lt;b&gt;图2 字符嵌入步骤的示例图&lt;/b&gt;"><b>图2 字符嵌入步骤的示例图</b></a></li>
                                                <li><a href="#66" data-title="&lt;b&gt;图3 CPPDeep方法的流程图&lt;/b&gt;"><b>图3 CPPDeep方法的流程图</b></a></li>
                                                <li><a href="#73" data-title="&lt;b&gt;图4 采用不同字符嵌入维数的模型在训练集上的5交叉验证精确度分布&lt;/b&gt;"><b>图4 采用不同字符嵌入维数的模型在训练集上的5交叉验证精确度分布</b></a></li>
                                                <li><a href="#75" data-title="&lt;b&gt;图5 采用不同数量LSTM存储单元的模型在训练集上的5交叉验证精确度分布&lt;/b&gt;"><b>图5 采用不同数量LSTM存储单元的模型在训练集上的5交叉验证精确度分布</b></a></li>
                                                <li><a href="#76" data-title="&lt;b&gt;图6 CPPDeep预测模型中每层的输入和输出参数情况&lt;/b&gt;"><b>图6 CPPDeep预测模型中每层的输入和输出参数情况</b></a></li>
                                                <li><a href="#80" data-title="&lt;b&gt;图7 三种不同方法在测试集CPP_test上的ROC曲线图&lt;/b&gt;"><b>图7 三种不同方法在测试集CPP_test上的ROC曲线图</b></a></li>
                                                <li><a href="#81" data-title="&lt;b&gt;图8 CPPDeep方法在测试集上的ROC曲线图&lt;/b&gt;"><b>图8 CPPDeep方法在测试集上的ROC曲线图</b></a></li>
                                                <li><a href="#84" data-title="&lt;b&gt;图9 训练250次epoch时,CPPDeep模型在训练集和测试集上的结果&lt;/b&gt;"><b>图9 训练250次epoch时,CPPDeep模型在训练集和测试集上的结果</b></a></li>
                                                <li><a href="#87" data-title="&lt;b&gt;表2 与已有方法在测试集上的性能比较&lt;/b&gt;"><b>表2 与已有方法在测试集上的性能比较</b></a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="3">


                                    <a id="bibliography_1" title=" 范博,金明姬,黄伟,王启明,高钟镐,等.细胞穿膜肽在药物递送系统中的研究进展[J].药学学报,2016,(2):264-271." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=YXXB201602017&amp;v=MTYzMjJYVGJMRzRIOWZNclk5RVk0UUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVacEZ5L25VN3pCUEQ=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[1]</b>
                                         范博,金明姬,黄伟,王启明,高钟镐,等.细胞穿膜肽在药物递送系统中的研究进展[J].药学学报,2016,(2):264-271.
                                    </a>
                                </li>
                                <li id="5">


                                    <a id="bibliography_2" title=" 孙春萌,沈雁,涂家生.细胞穿膜肽的研究进展[J].中国药学杂志,2013,48(14):1143-1146." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGYX201314003&amp;v=MDY0MjJyU2RyRzRIOUxOcTQ5Rlo0UUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVacEZ5L25VN3pCUHk=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[2]</b>
                                         孙春萌,沈雁,涂家生.细胞穿膜肽的研究进展[J].中国药学杂志,2013,48(14):1143-1146.
                                    </a>
                                </li>
                                <li id="7">


                                    <a id="bibliography_3" title=" 周华华,董坚.细胞穿膜肽的研究进展与前景展望[J].西部医学,2012,24(7):1410-1411." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=XIBU201207086&amp;v=MDQ4OTh0R0ZyQ1VSN3FmWnVacEZ5L25VN3pCUFNUSmU3RzRIOVBNcUk5TllvUUtESDg0dlI0VDZqNTRPM3pxcUI=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[3]</b>
                                         周华华,董坚.细胞穿膜肽的研究进展与前景展望[J].西部医学,2012,24(7):1410-1411.
                                    </a>
                                </li>
                                <li id="9">


                                    <a id="bibliography_4" title=" S Lata,N K Mishra,and G P S Raghava.AntiBP2:improved version of antibacterial peptide prediction[C].BMC Bioinformatics.2010-11:S19." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=AntiBP2: improved version of antibacterial peptide prediction">
                                        <b>[4]</b>
                                         S Lata,N K Mishra,and G P S Raghava.AntiBP2:improved version of antibacterial peptide prediction[C].BMC Bioinformatics.2010-11:S19.
                                    </a>
                                </li>
                                <li id="11">


                                    <a id="bibliography_5" title=" B Manavalan,S Subramaniyam,T H Shin,M O Kim,G Lee.Machine-learning-based prediction of cell-penetrating peptides and their uptake efficiency with improved accuracy[C].J Proteome Res.2018,17:2715-2726." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Machine-learning-based prediction of cell-penetrating peptides and their uptake efficiency with improved accuracy">
                                        <b>[5]</b>
                                         B Manavalan,S Subramaniyam,T H Shin,M O Kim,G Lee.Machine-learning-based prediction of cell-penetrating peptides and their uptake efficiency with improved accuracy[C].J Proteome Res.2018,17:2715-2726.
                                    </a>
                                </li>
                                <li id="13">


                                    <a id="bibliography_6" title=" Vinod Kumar,Piyush Agrawal,P S Gajendra.Raghava,et al.Prediction of Cell-Penetrating Potential of Modified Peptides Containing Natural and Chemically Modified Residues[C].Front.Microbiol.,2018-9:725." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prediction of cellpenetrating potential of modified peptides containing natural and chemically modified residues">
                                        <b>[6]</b>
                                         Vinod Kumar,Piyush Agrawal,P S Gajendra.Raghava,et al.Prediction of Cell-Penetrating Potential of Modified Peptides Containing Natural and Chemically Modified Residues[C].Front.Microbiol.,2018-9:725.
                                    </a>
                                </li>
                                <li id="15">


                                    <a id="bibliography_7" title=" P Pandey,V Patel,N V George,Mallajosyula SS.KELM-CPPpred:kernel extreme learning machine based prediction model for cell-penetrating peptides[C].J Proteome Res.2018." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=KELM-CPPpred:kernel extreme learning machine based prediction model for cell-penetrating peptides">
                                        <b>[7]</b>
                                         P Pandey,V Patel,N V George,Mallajosyula SS.KELM-CPPpred:kernel extreme learning machine based prediction model for cell-penetrating peptides[C].J Proteome Res.2018.
                                    </a>
                                </li>
                                <li id="17">


                                    <a id="bibliography_8" title=" T A Holton,G Pollastri,D C Shields,C Mooney.CPPpred:prediction of cell penetrating peptides[C].Bioinformatics.2013,29:3094-3096." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=CPPpred:prediction of cell penetrating peptides">
                                        <b>[8]</b>
                                         T A Holton,G Pollastri,D C Shields,C Mooney.CPPpred:prediction of cell penetrating peptides[C].Bioinformatics.2013,29:3094-3096.
                                    </a>
                                </li>
                                <li id="19">


                                    <a id="bibliography_9" title=" P Agrawal,S Bhalla,G P S Raghava,et al.In silico approach for prediction of antifungal peptides[C].Front Microbiol.2018,26(9):323." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=In silico approach for prediction of antifungal peptides">
                                        <b>[9]</b>
                                         P Agrawal,S Bhalla,G P S Raghava,et al.In silico approach for prediction of antifungal peptides[C].Front Microbiol.2018,26(9):323.
                                    </a>
                                </li>
                                <li id="21">


                                    <a id="bibliography_10" title=" S Thomas,S Karnik,R S Barai,V K Jayaraman,S Idicula-Thomas.CAMP:a useful resource for research on antimicrobial peptides[C].Nucleic Acids Res.2010,38:D774–D780." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=CAMP: a useful resource for research on antimicrobial peptides">
                                        <b>[10]</b>
                                         S Thomas,S Karnik,R S Barai,V K Jayaraman,S Idicula-Thomas.CAMP:a useful resource for research on antimicrobial peptides[C].Nucleic Acids Res.2010,38:D774–D780.
                                    </a>
                                </li>
                                <li id="23">


                                    <a id="bibliography_11" title=" B Manavalan,et al.MLACP:machine learning based prediction of anticancer peptides[C].Oncotarget.2017,8:77121–77136." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=MLACP:machine-learning-based prediction of anticancer peptides">
                                        <b>[11]</b>
                                         B Manavalan,et al.MLACP:machine learning based prediction of anticancer peptides[C].Oncotarget.2017,8:77121–77136.
                                    </a>
                                </li>
                                <li id="25">


                                    <a id="bibliography_12" title=" B Manavalan,T H Shin,M O Kim,et al,AIPpred:sequence-based prediction of anti-inflammatory peptides using random forest[C].Front Pharmacol,9,2018,276:1-12." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=AIPpred:sequence-based prediction of anti-inflammatory peptides using random forest">
                                        <b>[12]</b>
                                         B Manavalan,T H Shin,M O Kim,et al,AIPpred:sequence-based prediction of anti-inflammatory peptides using random forest[C].Front Pharmacol,9,2018,276:1-12.
                                    </a>
                                </li>
                                <li id="27">


                                    <a id="bibliography_13" title=" Jason Brownlee.Deep Learning for Natural Language Processing[DB].https://machinelearningmastery.com/deep-learning-for-nlp/]" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Deep Learning for Natural Language Processing[DB]">
                                        <b>[13]</b>
                                         Jason Brownlee.Deep Learning for Natural Language Processing[DB].https://machinelearningmastery.com/deep-learning-for-nlp/]
                                    </a>
                                </li>
                                <li id="29">


                                    <a id="bibliography_14" title=" Zuckerkandl E Paulin.Molecules as documents of evolutionary history[C].J Theor Biol.1965,8:357–366." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Molecules as documents of evolutionary history">
                                        <b>[14]</b>
                                         Zuckerkandl E Paulin.Molecules as documents of evolutionary history[C].J Theor Biol.1965,8:357–366.
                                    </a>
                                </li>
                                <li id="31">


                                    <a id="bibliography_15" title=" Y LeCun,Y Bengio,G Hinton.Deep learning[C].Nature.2015,521:436–444." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Deep Learning">
                                        <b>[15]</b>
                                         Y LeCun,Y Bengio,G Hinton.Deep learning[C].Nature.2015,521:436–444.
                                    </a>
                                </li>
                                <li id="33">


                                    <a id="bibliography_16" title=" Yoav Goldberg.Neural Network Methods for Natural Language Processing[DB].2017,(https://doi.org/10.2200/S00762ED1V01Y201703HLT037)" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Neural Network Methods for Natural Language Processing[DB]">
                                        <b>[16]</b>
                                         Yoav Goldberg.Neural Network Methods for Natural Language Processing[DB].2017,(https://doi.org/10.2200/S00762ED1V01Y201703HLT037)
                                    </a>
                                </li>
                                <li id="35">


                                    <a id="bibliography_17" title=" Jason Brownlee.Long Short-Term Memory Networks With Python[DB].https:// machinelearningmastery.com/lstms-with-python/]" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Long Short-Term Memory Networks With Python[DB]">
                                        <b>[17]</b>
                                         Jason Brownlee.Long Short-Term Memory Networks With Python[DB].https:// machinelearningmastery.com/lstms-with-python/]
                                    </a>
                                </li>
                                <li id="37">


                                    <a id="bibliography_18" title=" P Agrawal,S Bhalla,G P Raghava,et al.Gautam A.CPPsite 2.0:a repository of experimentally validated cell-penetrating peptides[J].Nucleic Acids Res.2016,44(D1):D1098-103." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=CPPsite 2.0:a repository of experimentally validated cell-penetrating peptides">
                                        <b>[18]</b>
                                         P Agrawal,S Bhalla,G P Raghava,et al.Gautam A.CPPsite 2.0:a repository of experimentally validated cell-penetrating peptides[J].Nucleic Acids Res.2016,44(D1):D1098-103.
                                    </a>
                                </li>
                                <li id="39">


                                    <a id="bibliography_19" title=" S Singh,K Chaudhary,G P Raghava,et al.SATPdb:a database of structurally annotated therapeutic peptides[J].Nucleic Acids Res.2016,44(D1):D1119-26." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=SATPdb:a database of structurally annotated therapeutic peptides">
                                        <b>[19]</b>
                                         S Singh,K Chaudhary,G P Raghava,et al.SATPdb:a database of structurally annotated therapeutic peptides[J].Nucleic Acids Res.2016,44(D1):D1119-26.
                                    </a>
                                </li>
                                <li id="41">


                                    <a id="bibliography_20" title=" Keras:[https://keras.io/]" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Keras">
                                        <b>[20]</b>
                                         Keras:[https://keras.io/]
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JSJZ" target="_blank">计算机仿真</a>
                2019,36(10),353-358             </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>基于深度学习和字符嵌入的细胞穿透肽预测</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%96%B9%E6%98%A5&amp;code=32912429&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">方春</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%AD%99%E7%A6%8F%E6%8C%AF&amp;code=08877286&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">孙福振</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%9D%8E%E5%BD%A9%E8%99%B9&amp;code=08288170&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">李彩虹</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E9%82%A2%E6%9E%97%E6%9E%97&amp;code=43106142&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">邢林林</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%B1%B1%E4%B8%9C%E7%90%86%E5%B7%A5%E5%A4%A7%E5%AD%A6%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%A7%91%E5%AD%A6%E4%B8%8E%E6%8A%80%E6%9C%AF%E5%AD%A6%E9%99%A2&amp;code=0245633&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">山东理工大学计算机科学与技术学院</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>针对传统细胞穿透肽的预测方法严重依赖于繁琐的特征抽取和特征重建步骤、算法复杂且准确度不高等问题,提出了一种利用自然语言处理中的字符嵌入方法结合CNN-LSTM组合机器学习框架来预测细胞穿透肽的方法。方法采用字符嵌入将氨基酸的代表字符通过网络学习映射到紧凑表示的向量空间中,每种氨基酸字符对应一个紧凑表示的向量,然后将肽序列通过由训练得到的嵌入向量转化为数值矩阵作为CNN-LSTM模型的输入,模型自行抽取特征后自动对输入序列的细胞穿透性进行预测。实验结果显示,在相同数据集进行实验时,研究的方法在测试集上的AUC (the area under ROC curve)值达到0.97,正确指数达到0.846,优于其它方法,说明上述方法能够简单、高效地进行细胞穿透肽的预测。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">深度学习;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%AD%97%E7%AC%A6%E5%B5%8C%E5%85%A5&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">字符嵌入;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%BB%86%E8%83%9E%E7%A9%BF%E9%80%8F%E8%82%BD&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">细胞穿透肽;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E9%A2%84%E6%B5%8B&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">预测;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    方春(1981-),女(汉族),山东淄博人,讲师,主要研究领域为智能计算、模式识别。;
                                </span>
                                <span>
                                    孙福振(1978-),男(汉族),山东淄博人,博士生,主要研究方向为数据挖掘、模式识别。;
                                </span>
                                <span>
                                    邢林林(1987-),男(汉族),山东淄博人,讲师,主要研究领域为生物信息学。;
                                </span>
                                <span>
                                    李彩虹(1970-),女(汉族),山东烟台市人,教授,硕士研究生导师,主要研究领域为智能算法、数据挖掘等。;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-12-17</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金项目(61602280,61473179);</span>
                                <span>山东省自然科学基金项目(ZR2014FQ028);</span>
                    </p>
            </div>
                    <h1><b>Prediction of Cell Penetrating Peptides Based on Deep Learning and Character Embedding</b></h1>
                    <h2>
                    <span>FANG Chun</span>
                    <span>SUN Fun-zhen</span>
                    <span>LI Cai-hong</span>
                    <span>XING Lin-lin</span>
            </h2>
                    <h2>
                    <span>School of Computer Science and Technology, Shandong University of Technology</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>Traditional methods of cell-penetrating peptide prediction heavily rely on cumbersome feature extraction and reconstruction steps. These algorithms are complex and their accuracy still needs improving. To overcome the above shortcomings, this study proposes a novel method which uses character embedding method combined with CNN-LSTM machine learning framework for predicting cell-penetrating peptides. First, each amino acid character was mapped into a compact character embedding vector space through network learning on the training dataset. Second, peptides sequences were transformed into numerical feature matrix by these character embedding vectors. Finally, a CNN-LSTM model was used to automatically extract features for training, and gave the final prediction result of the input sequence. Experimental results show that, when comparing with other methods on the same datasets, our method performed best, achieving an AUC value of 0.97 and a correct index of 0.846, indicating that the proposed method is effective for improving the prediction of cell penetrating peptides.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Deep%20learning&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Deep learning;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Character%20embedding&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Character embedding;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Cell%20penetrating%20peptide&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Cell penetrating peptide;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Prediction&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Prediction;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                                            </p>
                                    <p><b>Received：</b> 2018-12-17</p>
                                    <p>
                                            </p>
            </div>


        <!--brief start-->
                        <h3 id="43" name="43" class="anchor-tag"><b>1 引言</b></h3>
                <div class="p1">
                    <p id="44">细胞穿透肽(cell penetrating peptides,CPPs)是一类长度约3到40个氨基酸的短肽,能够携带蛋白质、多肽、核酸、纳米颗粒及病毒颗粒等多种活性物质穿过细胞膜进入细胞内,并且具有不破坏细胞膜、高效和低毒等优点,是靶向药物的良好载体<citation id="91" type="reference"><link href="3" rel="bibliography" /><link href="5" rel="bibliography" /><sup>[<a class="sup">1</a>,<a class="sup">2</a>]</sup></citation>。细胞穿透肽在荧光成像、肿瘤治疗、抗炎治疗及药物靶向治疗等领域发挥了重要作用,具有广阔的应用前景<citation id="90" type="reference"><link href="7" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>。鉴定细胞穿透肽对于医药、农业等领域变得越来越重要。</p>
                </div>
                <div class="p1">
                    <p id="45">通过湿实验鉴定细胞穿透肽昂贵且耗时,计算方法能够快速地分析海量生物数据,因此,预测细胞穿透肽必不可少的方法。目前,人们已经开发了许多基于序列的方法来预测各种生物肽,如抗菌肽<citation id="92" type="reference"><link href="9" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>、细胞穿透肽<citation id="97" type="reference"><link href="11" rel="bibliography" /><link href="13" rel="bibliography" /><link href="15" rel="bibliography" /><link href="17" rel="bibliography" /><sup>[<a class="sup">5</a>,<a class="sup">6</a>,<a class="sup">7</a>,<a class="sup">8</a>]</sup></citation>、抗真菌肽<citation id="93" type="reference"><link href="19" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>、抗微生物肽<citation id="94" type="reference"><link href="21" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>、抗癌肽<citation id="95" type="reference"><link href="23" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>、抗炎症肽<citation id="96" type="reference"><link href="25" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>等。这些方法主要使用了传统的机器学习方法,包括马尔可夫模型、随机森林、支持向量机、判别分析等,并且预测性能严重依赖于基于特定领域专业知识的计算出来的大量特征及其组合,比如,氨基酸组成、二肽组成、氨基酸指数、理化性质、二元谱和位置特异性打分矩阵等。这些特征的提取过程繁琐,且容易导致特征不完整和维数灾难。此外,这些特征提取过程是独立于机器学习过程的,没有考虑这两步之间的潜在的不匹配。因此,肽序列的分类算法存在很大的改进空间。</p>
                </div>
                <div class="p1">
                    <p id="46">近年来,词嵌入方法(word embedding)给自然语言处理领域的诸多问题带来了突破性进展,其优点是可以进行特征维数约简,即通过神经网络训练学习,将词汇表中的单词映射到一个高密度连续的向量空间。向量空间中的每个单词的位置叫做“词嵌入”<citation id="98" type="reference"><link href="27" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>,是从大量文本集中训练学习得到的,并且是基于各个词的上下文信息而学习得到的,向量空间中具有“相近语义”的词在空间中的位置也靠近。</p>
                </div>
                <div class="p1">
                    <p id="47">生物序列与自然语言非常相似,它们都由按特定顺序排列的字符(单词)组成,生物序列中的每个残基可以被视为句子中的单词。已有研究表明<citation id="99" type="reference"><link href="29" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>,在蛋白质的进化过程中,序列中给定位置的残基的替换主要取决于残基的理化性质,即氨基酸更容易被具有相似性质的氨基酸所取代。可以理解为具有相似“语义”的氨基酸容易出现在序列中的特定位置。因此,word embedding方法也可用于表示蛋白质序列,即为蛋白质序列创建一个词汇表(通常为20或21个氨基酸),将氨基酸字符看作是构成序列的“单词”,然后为每个氨基酸训练出一个词向量。在本研究中,把由氨基酸字符对应的嵌入空间中的向量叫做“字符嵌入”(character embedding)。</p>
                </div>
                <div class="p1">
                    <p id="48">长期以来,传统的机器学习方法是在高维稀疏的特征向量上进行训练的。随着深度学习的出现,序列分类算法研究已经逐渐转变到了注重紧凑的特征表示和选择最佳的机器学习方法上。深度学习中的卷积神经网络 (Convolutional Neural Network, CNN)和长短期记忆(Long Short Term Memory networks, LSTM)等在许多领域取得了巨大的成功,包括图像处理,语音识别和视频处理<citation id="100" type="reference"><link href="31" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation>。它们也非常擅长文本处理,已在一系列自然语言处理问题上取得了先进的成果。 CNN和LSTM本质上是特征提取的架构。先前研究<citation id="101" type="reference"><link href="33" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation>认为,在处理自然语言分类问题时候, CNN可以直接从原始信号中通过卷积和池化操作提取位于序列中不同位置的有意义的子结构(不连续的语义信息),而不需要关注具体语言的句法或语义结构知识。而LSTM擅长学习序列中对整体预测任务有用的长期相关性信息(连续语义信息)<citation id="102" type="reference"><link href="35" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>。然而,CNN和LSTM本身并不构成独立的、高效的网络,必须把它们集成到更大的网络中相互协作以发挥其优势。因此,词嵌入、CNN与LSTM(word embedding)具有天然的结合基础。</p>
                </div>
                <div class="p1">
                    <p id="49">综合上述分析,提出了一种基于字符嵌入和混合深度学习框架的细胞穿透肽的预测方法。该方法仅需将原始肽序列作为输入,并使用字符嵌入方法来表示序列,然后采用CNN-LSTM组合架构来进行自动的特征提取,并训练网络以产生细胞穿透肽的预测结果。</p>
                </div>
                <h3 id="50" name="50" class="anchor-tag"><b>2 数据与方法</b></h3>
                <h4 class="anchor-tag" id="51" name="51"><b>2.1 数据集</b></h4>
                <div class="p1">
                    <p id="52">为了方便比较,本研究采用了与文献<citation id="103" type="reference">[<a class="sup">6</a>]</citation>相同的数据集来进行实验。该数据集中的正样本(细胞穿透肽)均来自于细胞穿透肽数据库CPPsite 2.0<citation id="104" type="reference"><link href="37" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>,均已被物理实验证实为细胞穿透肽,并测定了三级结构。负样本数据non-CPPs选自肽数据库SATPdb <citation id="105" type="reference"><link href="39" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>(Singh et al.,2016)中未标注有细胞穿透性性质的肽。训练集由1,164条序列构成,其中包含582条细胞穿透肽和582条非细胞穿透肽;测试集由300条序列构成,其中包含150条细胞穿透肽和150条非细胞穿透肽。以上数据集的详细情况如表1所示。</p>
                </div>
                <div class="area_img" id="53">
                    <p class="img_tit"><b>表1 数据集的统计</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="53" border="1"><tr><td><br />数据集</td><td>序列条数</td><td>正样本数</td><td>负样本数</td></tr><tr><td><br />训练集CPP_train</td><td>1164</td><td>582</td><td>582</td></tr><tr><td><br />测试集CPP_test</td><td>300</td><td>150</td><td>150</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h4 class="anchor-tag" id="54" name="54"><b>2.2 肽序列的长度分布</b></h4>
                <div class="p1">
                    <p id="55">首先分析了CPP_train数据集中序列的肽序列的长度分布。 图1显示CPP_train数据集中CPP序列的长度主要分布在3到41个残基之间,并且大多数序列集中16个残基左右的长度。 因此,在使用Keras<citation id="106" type="reference"><link href="41" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>提供的Tokenizer API训练字符嵌入的向量空间时,每个肽序列被填充到固定长度41,并将嵌入维度的数量初始值设置为50。</p>
                </div>
                <div class="area_img" id="56">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJZ201910073_056.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 CPP序列的长度分布" src="Detail/GetImg?filename=images/JSJZ201910073_056.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图1 CPP序列的长度分布</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJZ201910073_056.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="57" name="57"><b>2.3 肽序列编码</b></h4>
                <div class="p1">
                    <p id="58">如自然语言处理中的词嵌入方法一样,本研究中字符嵌入方法为每种氨基酸训练一个紧凑表示的实值向量。<b>Keras</b><citation id="107" type="reference"><link href="41" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup><sup><b>20</b></sup><sup>]</sup></citation>提供了一个嵌入层接口可以用来定义嵌入层,使得该层作为神经网络模型的一部分,与模型本身一起学习。它要求对输入数据进行整数编码,这样每种字符都由唯一的整数表示。嵌入层使用随机权重进行初始化,并进行学习训练,使得训练数据集中的所有字符都对应到一个嵌入向量。该数据准备步骤可以由<b>Keras</b>提供的<b>Tokenizer API</b>来执行完成<citation id="108" type="reference"><link href="41" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup><sup><b>20</b></sup><sup>]</sup></citation>。首先,训练数据集中的每个序列被编码成<b>41</b>维整数向量,不够长度的序列右端用<b>0</b>填充。然后,将每个整数映射到指定维度的向量空间,该空间维度可以被指定为任意维,在这里,暂时将其设定为<b>50</b>维。这样,每种氨基酸被训练为<b>50</b>维的嵌入向量,每条长度为 <b>L</b>的肽序列被表示为<b>L×50</b>特征矩阵。图<b>2</b>显示了字符嵌入步骤的示例图。</p>
                </div>
                <div class="area_img" id="59">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJZ201910073_059.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 字符嵌入步骤的示例图" src="Detail/GetImg?filename=images/JSJZ201910073_059.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图2 字符嵌入步骤的示例图</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJZ201910073_059.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="60">。例如,假设数据集中序列的最大长度为<b>4</b>,并且想要创建<b>5</b>维的嵌入向量空间,则可以将序列“<b>BTS</b>”转换为:<b>BTS→ [2, 4, 1, 0] → [[0.55, 0.10,-0.48,-0.31,-0.41], [-0.40, 0.01,-0.20,-0.18, 0.01], [0.62, 0.54,-0.17,-0.78,-0.32], [-0.80, 0.30, 0.61, 0.34, 0.09</b>]]。</p>
                </div>
                <h4 class="anchor-tag" id="61" name="61"><b>2.4 CNN-LSTM框架</b></h4>
                <div class="p1">
                    <p id="62">在字符嵌入步骤后,<b>CPP</b>序列被映射为特征矩阵,作为神经网络的输入。<b>CNN</b>已被证明擅长于捕获密集表示的特征矩阵中数据的空间局部相关性,已经在图像、音频、视频等多个领域的应用取得了巨大成功。它能够通过卷积和池化操作自动抽取的特征矩阵中分散的子结构信息,而不依赖于它们在原始数据中的位置信息。</p>
                </div>
                <div class="p1">
                    <p id="63">尽管<b>CNN</b>层可以捕获出现在序列中不同位置的局部线索,但它不擅长提取序列中符号之间的长期相关性,而这些正是生物序列的重要特性。庆幸的是,<b>LSTM (Long Short Term Memory</b>)架构是正好可以弥补这一短处。<b>LSTM</b>模型由一组循环连接的带记忆功能的网络单元组成,每个网络单元包含三个门控,即输入门、忘记门和输出门,用于控制信号的输入、调整和输出<sup>[</sup><sup><b>21</b></sup><sup>]</sup>。这些门控运算使得模型能够自动学习和利用输入序列中的有序特性来处理序列相关的任务。因此,<b>CNN</b>和<b>LSTM</b>结合,既能抽取字符嵌入表示后的序列中的离散子结构特征,又能捕获对整体预测有用的长程相关性信息,使得<b>CNN-LSTM</b>架构非常适合于处理列序列分类问题。并且,这种混合架构将特征提取步骤和模型学习步骤交织在一起,避免了分别执行这两个步骤所带来的潜在的不匹配效应。</p>
                </div>
                <h4 class="anchor-tag" id="64" name="64"><b>2.5 深度卷积神经网络的结构</b></h4>
                <div class="p1">
                    <p id="65">图<b>3</b>显示了<b>CPPDeep</b>模型的整体架构。该模型包括四个主要层:<b>1</b>)嵌入层;<b>2)CNN</b>层;<b>3)LSTM</b>层;以及<b>4</b>)全连接输出层。 首先,嵌入层用来学习字符嵌入,为每个字符训练出一个紧凑的向量表示,并将肽序列编码为特征矩阵,作为<b>CNN</b>网络的输入;接着,<b>CNN</b>层用来提取的特征矩阵中有用的子结构信息;然后,<b>LSTM</b>层来用来提取对整体预测任务有意义的数据中的长期相关性和上下文信息;最后,全连接层采用激活函数运算来产生模型的输出结果。以上所有层都在<b>Keras</b>中共同训练。</p>
                </div>
                <div class="area_img" id="66">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJZ201910073_066.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 CPPDeep方法的流程图" src="Detail/GetImg?filename=images/JSJZ201910073_066.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图3 CPPDeep方法的流程图</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJZ201910073_066.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="67" name="67"><b>2.6 评价指标</b></h4>
                <div class="p1">
                    <p id="68">为了评估相关方法的性能,本研究中使用了如下评估指标:<b>1)Receiver Operating Characteristic Curve (ROC</b>)曲线及曲线下的面积(<b>area under ROC curve, AUC);2</b>)真阳性率(<b>True Positive Rate, TPR</b>),其值等于灵敏度(<b>sensitivity</b>),是被正确预测的<b>CPP</b>的比率; <b>3</b>)假阳性率(<b>False Positive Rate, FPR</b>), 是被误分类为正样本的负样本的比率, 其值等于<b>1</b>-特异性(<b>specificity); 4</b>)正确指数,又称约登指数(<b>Youden's index</b>),为灵敏度和特异性之和减去<b>1</b>, 表示分类器对样本预测正确的总能力,其值越大,表示性能越好.</p>
                </div>
                <h3 id="69" name="69" class="anchor-tag"><b>3 结果与分析</b></h3>
                <h4 class="anchor-tag" id="70" name="70"><b>3.1 模型参数的优化</b></h4>
                <div class="p1">
                    <p id="71">深度学习模型中需要设置各种参数,为了寻找最合适的参数,采用各种不同的参数设置进行了大量实验。</p>
                </div>
                <div class="p1">
                    <p id="72">首先,按如下方式初始化了部分网络参数:①<b>CNN</b>层含有<b>32</b>个大小为<b>3</b>的卷积核,池化层核大小为<b>2</b>,丢失率为<b>0.5;②LSTM</b>层具有<b>50</b>个存储单元,丢失率为<b>0.2</b>。③编译模型时,损失函数为“<b>binary_crossentropy</b>”,优化函数为“<b>rmsprop</b>”。然后,在训练集<b>CPP_train</b>上分别测试了嵌入维度为<b>10、30、50、70</b>和<b>90</b>的模型的性能。相关的五交叉验证精确度如图<b>4</b>所示,在嵌入维度大于<b>50</b>时,精度趋于稳定,因此,本研究的预测模型字符嵌入维度设置为<b>50</b>。</p>
                </div>
                <div class="area_img" id="73">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJZ201910073_073.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 采用不同字符嵌入维数的模型在训练集上的5交叉验证精确度分布" src="Detail/GetImg?filename=images/JSJZ201910073_073.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图4 采用不同字符嵌入维数的模型在训练集上的5交叉验证精确度分布</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJZ201910073_073.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="74">为了获得<b>LSTM</b>层的最佳存储单元数,在<b>CPP_train</b>上分别测试了在<b>LSTM</b>层中包含不同个数的存储单元的模型性能。 图<b>5</b>显示了<b>LSTM</b>层中包含<b>10、30、50、70</b>和<b>90</b>个存储器单元时,相关模型的五交叉验证精确度。可以看出,存储器单元数在<b>50</b>到<b>70</b>时,模型取得最佳性能。因此,本研究设置模型的<b>LSTM</b>层包含<b>50</b>个存储单元。最后,<b>CPPDeep</b>预测模型中每层的输入和输出参数情况如图<b>6</b>所示。</p>
                </div>
                <div class="area_img" id="75">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJZ201910073_075.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图5 采用不同数量LSTM存储单元的模型在训练集上的5交叉验证精确度分布" src="Detail/GetImg?filename=images/JSJZ201910073_075.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图5 采用不同数量LSTM存储单元的模型在训练集上的5交叉验证精确度分布</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJZ201910073_075.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="76">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJZ201910073_076.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图6 CPPDeep预测模型中每层的输入和输出参数情况" src="Detail/GetImg?filename=images/JSJZ201910073_076.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图6 CPPDeep预测模型中每层的输入和输出参数情况</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJZ201910073_076.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="77">其中,“(<b>None,100,50</b>)”表示嵌入层的输出是一个<b>100×50</b>的矩阵,“<b>None</b>”表示此处不包括处理数据的批量维度,这些向量被<b>LSTM</b>层压缩为包含<b>50</b>个元素向量(<b>None,50</b>)。</p>
                </div>
                <h4 class="anchor-tag" id="78" name="78"><b>3.2 与其它两种深度学习方法的比较</b></h4>
                <div class="p1">
                    <p id="79">为了证明在<b>CPPDeep</b>预测器中使用的<b>CNN-LSTM</b>架构的有效性,将其与其它两种方法进行了比较:(<b>1) “embedding_CNN</b>”方法,它将字符嵌入与<b>CNN</b>相结合,但不包括<b>LSTM</b>层;(<b>2)“embedding_LSTM</b>”方法,包括字符嵌入和<b>LSTM</b>层,不包括<b>CNN</b>层。这三种方法都采用相同的参数,图<b>7</b>中显示了三种方法在测试集上的<b>ROC</b>曲线图。从图<b>7</b>可以看出,<b>embedding_CNN_LSTM</b>方法(<b>CPPDeep</b>)性能最好,其 <b>AUC</b>值达到<b>0.9533</b>(图<b>8</b>)。</p>
                </div>
                <div class="area_img" id="80">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJZ201910073_080.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图7 三种不同方法在测试集CPP_test上的ROC曲线图" src="Detail/GetImg?filename=images/JSJZ201910073_080.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图7 三种不同方法在测试集CPP_test上的ROC曲线图</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJZ201910073_080.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="81">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJZ201910073_081.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图8 CPPDeep方法在测试集上的ROC曲线图" src="Detail/GetImg?filename=images/JSJZ201910073_081.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图8 CPPDeep方法在测试集上的ROC曲线图</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJZ201910073_081.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="82" name="82"><b>3.3 在训练集和测试集上的准确率和损失函数分布</b></h4>
                <div class="p1">
                    <p id="83">为了观察提出的预测模型在整个数据集合上的匹配情况,测试了在训练<b>250</b>次<b>epoch</b>时,<b>CPPDeep</b>方法在训练集和测试集上的准确率和损失函数变化情况。如图<b>9</b>所示,在整个训练过程中,准确率和损失函数在训练集和测试集上的曲线变化趋势一致且相互靠近,说明本<b>CPPDeep</b>没有发生过拟合现象,即<b>CPPDeep</b>模型不仅能很好地拟合训练数据,而且对未知样本也能够进行准确分类。</p>
                </div>
                <div class="area_img" id="84">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJZ201910073_084.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图9 训练250次epoch时,CPPDeep模型在训练集和测试集上的结果" src="Detail/GetImg?filename=images/JSJZ201910073_084.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图9 训练250次epoch时,CPPDeep模型在训练集和测试集上的结果</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJZ201910073_084.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="85" name="85"><b>3.4 与已有的预测方法的比较</b></h4>
                <div class="p1">
                    <p id="86">由于使用了论文<citation id="110" type="reference">[<a class="sup">6</a>]</citation>中的数据集进行实验,也将“<b>CPPDeep</b>”方法与论文<citation id="111" type="reference">[<a class="sup">6</a>]</citation>中提出的两种单模型方法进行了比较:(<b>1</b>)基于<b>144</b>种二维结构特征和随机森林的“<b>CellPPDMod_2D</b>”方法;(<b>2</b>)基于<b>47</b>种三维结构特征和随机森林的“<b>CellPPDMod_3D</b>”方法。这些特征都是使用第三方软件<b>PaDEL</b>计算得到。 “<b>CPPDeep</b>”方法与这两种方法在同一数据集上的比较结果如表<b>2</b>所示,其中“<b>CellPPDMod_2D</b>”与“<b>CellPPDMod_3D</b>”方法的结果来自论文<citation id="112" type="reference">[<a class="sup">6</a>]</citation>。从表<b>2</b>可以看出,本文所提出的<b>CPPDeep</b>方法在正确指数和<b>AUC</b>上都有显著高于<b>CellPPDMod_3D</b>方法;相比于<b>CellPPDMod_2D,CPPDeep</b>达到与其同样的<b>AUC</b>值,但在正确指数方面优于<b>CellPPDMod_2D</b>。值得注意的是,<b>CellPPDMod_2D</b>借助于第三方软件进行了大量特征筛选工作,而本研究提出<b>CPPDeep</b>方法不需要繁琐的特征抽取和特征重建步骤,仅使用了粗略调整的模型参数即可达到与其相当的性能。如果通过网格搜索进一步优化参数,则<b>CPPDeep</b>预测准确度能进一步提高,但这同时也意味着需要更大的计算成本。</p>
                </div>
                <div class="area_img" id="87">
                    <p class="img_tit"><b>表2 与已有方法在测试集上的性能比较</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="87" border="1"><tr><td><br />方法名称</td><td>正确指数</td><td>真阳性率</td><td>假阳性率</td><td>AUC</td></tr><tr><td><br />文献[6] CellPPDMod_2D</td><td>0.833</td><td>0.947</td><td>0.114</td><td>0.97</td></tr><tr><td><br />文献[6] CellPPDMod_3D</td><td>0.469</td><td>0.798</td><td>0.329</td><td>0.83</td></tr><tr><td><br />本文提出的CPPDeep</td><td>0.846</td><td>0.946</td><td>0.100</td><td>0.97</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h3 id="88" name="88" class="anchor-tag"><b>4 结束语</b></h3>
                <div class="p1">
                    <p id="89">在本研究中,将CPP识别任务作为一种自然语言处理问题,并提出了一种将字符嵌入与CNN-LSTM的混合架构结合的方法来预测肽序列的细胞穿透性。该方法直接将序列作为输入,不需要手动计算任何特征。字符嵌入技术自动将CPP序列映射到紧凑向量表示的特征矩阵上,作为CNN-LSTM模型的输入。接着用CNN-LSTM组合模型对输入数据进行训练、预测并输出最终结果。尽管只使用了粗略调整的模型参数,在相同数据集上的实验结果证明,本研究所提出的方法优于传统方法的性能。该结果证明了字符嵌入方法可以高效地表征生物序列,避免传统算法中繁重的特征提取和重建工作,同时,也证明了字符嵌入与CNN、LSTM相结合能构成细胞穿透肽预测的有效模型。本研究方法也为其它生物序列分类问题提供了新的思路。</p>
                </div>
                <div class="area_img" id="113">
                                <img alt="" src="Detail/GetImg?filename=images/JSJZ201910073_11300.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="3">
                            <a id="bibliography_1" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=YXXB201602017&amp;v=MDcwMDBGckNVUjdxZlp1WnBGeS9uVTd6QlBEWFRiTEc0SDlmTXJZOUVZNFFLREg4NHZSNFQ2ajU0TzN6cXFCdEc=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[1]</b> 范博,金明姬,黄伟,王启明,高钟镐,等.细胞穿膜肽在药物递送系统中的研究进展[J].药学学报,2016,(2):264-271.
                            </a>
                        </p>
                        <p id="5">
                            <a id="bibliography_2" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGYX201314003&amp;v=MTgwMDk5TE5xNDlGWjRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpwRnkvblU3ekJQeXJTZHJHNEg=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[2]</b> 孙春萌,沈雁,涂家生.细胞穿膜肽的研究进展[J].中国药学杂志,2013,48(14):1143-1146.
                            </a>
                        </p>
                        <p id="7">
                            <a id="bibliography_3" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=XIBU201207086&amp;v=MTgwMTQ5TllvUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVacEZ5L25VN3pCUFNUSmU3RzRIOVBNcUk=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[3]</b> 周华华,董坚.细胞穿膜肽的研究进展与前景展望[J].西部医学,2012,24(7):1410-1411.
                            </a>
                        </p>
                        <p id="9">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=AntiBP2: improved version of antibacterial peptide prediction">

                                <b>[4]</b> S Lata,N K Mishra,and G P S Raghava.AntiBP2:improved version of antibacterial peptide prediction[C].BMC Bioinformatics.2010-11:S19.
                            </a>
                        </p>
                        <p id="11">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Machine-learning-based prediction of cell-penetrating peptides and their uptake efficiency with improved accuracy">

                                <b>[5]</b> B Manavalan,S Subramaniyam,T H Shin,M O Kim,G Lee.Machine-learning-based prediction of cell-penetrating peptides and their uptake efficiency with improved accuracy[C].J Proteome Res.2018,17:2715-2726.
                            </a>
                        </p>
                        <p id="13">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prediction of cellpenetrating potential of modified peptides containing natural and chemically modified residues">

                                <b>[6]</b> Vinod Kumar,Piyush Agrawal,P S Gajendra.Raghava,et al.Prediction of Cell-Penetrating Potential of Modified Peptides Containing Natural and Chemically Modified Residues[C].Front.Microbiol.,2018-9:725.
                            </a>
                        </p>
                        <p id="15">
                            <a id="bibliography_7" target="_blank" href="http://scholar.cnki.net/result.aspx?q=KELM-CPPpred:kernel extreme learning machine based prediction model for cell-penetrating peptides">

                                <b>[7]</b> P Pandey,V Patel,N V George,Mallajosyula SS.KELM-CPPpred:kernel extreme learning machine based prediction model for cell-penetrating peptides[C].J Proteome Res.2018.
                            </a>
                        </p>
                        <p id="17">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=CPPpred:prediction of cell penetrating peptides">

                                <b>[8]</b> T A Holton,G Pollastri,D C Shields,C Mooney.CPPpred:prediction of cell penetrating peptides[C].Bioinformatics.2013,29:3094-3096.
                            </a>
                        </p>
                        <p id="19">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=In silico approach for prediction of antifungal peptides">

                                <b>[9]</b> P Agrawal,S Bhalla,G P S Raghava,et al.In silico approach for prediction of antifungal peptides[C].Front Microbiol.2018,26(9):323.
                            </a>
                        </p>
                        <p id="21">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=CAMP: a useful resource for research on antimicrobial peptides">

                                <b>[10]</b> S Thomas,S Karnik,R S Barai,V K Jayaraman,S Idicula-Thomas.CAMP:a useful resource for research on antimicrobial peptides[C].Nucleic Acids Res.2010,38:D774–D780.
                            </a>
                        </p>
                        <p id="23">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=MLACP:machine-learning-based prediction of anticancer peptides">

                                <b>[11]</b> B Manavalan,et al.MLACP:machine learning based prediction of anticancer peptides[C].Oncotarget.2017,8:77121–77136.
                            </a>
                        </p>
                        <p id="25">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=AIPpred:sequence-based prediction of anti-inflammatory peptides using random forest">

                                <b>[12]</b> B Manavalan,T H Shin,M O Kim,et al,AIPpred:sequence-based prediction of anti-inflammatory peptides using random forest[C].Front Pharmacol,9,2018,276:1-12.
                            </a>
                        </p>
                        <p id="27">
                            <a id="bibliography_13" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Deep Learning for Natural Language Processing[DB]">

                                <b>[13]</b> Jason Brownlee.Deep Learning for Natural Language Processing[DB].https://machinelearningmastery.com/deep-learning-for-nlp/]
                            </a>
                        </p>
                        <p id="29">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Molecules as documents of evolutionary history">

                                <b>[14]</b> Zuckerkandl E Paulin.Molecules as documents of evolutionary history[C].J Theor Biol.1965,8:357–366.
                            </a>
                        </p>
                        <p id="31">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Deep Learning">

                                <b>[15]</b> Y LeCun,Y Bengio,G Hinton.Deep learning[C].Nature.2015,521:436–444.
                            </a>
                        </p>
                        <p id="33">
                            <a id="bibliography_16" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Neural Network Methods for Natural Language Processing[DB]">

                                <b>[16]</b> Yoav Goldberg.Neural Network Methods for Natural Language Processing[DB].2017,(https://doi.org/10.2200/S00762ED1V01Y201703HLT037)
                            </a>
                        </p>
                        <p id="35">
                            <a id="bibliography_17" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Long Short-Term Memory Networks With Python[DB]">

                                <b>[17]</b> Jason Brownlee.Long Short-Term Memory Networks With Python[DB].https:// machinelearningmastery.com/lstms-with-python/]
                            </a>
                        </p>
                        <p id="37">
                            <a id="bibliography_18" target="_blank" href="http://scholar.cnki.net/result.aspx?q=CPPsite 2.0:a repository of experimentally validated cell-penetrating peptides">

                                <b>[18]</b> P Agrawal,S Bhalla,G P Raghava,et al.Gautam A.CPPsite 2.0:a repository of experimentally validated cell-penetrating peptides[J].Nucleic Acids Res.2016,44(D1):D1098-103.
                            </a>
                        </p>
                        <p id="39">
                            <a id="bibliography_19" target="_blank" href="http://scholar.cnki.net/result.aspx?q=SATPdb:a database of structurally annotated therapeutic peptides">

                                <b>[19]</b> S Singh,K Chaudhary,G P Raghava,et al.SATPdb:a database of structurally annotated therapeutic peptides[J].Nucleic Acids Res.2016,44(D1):D1119-26.
                            </a>
                        </p>
                        <p id="41">
                            <a id="bibliography_20" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Keras">

                                <b>[20]</b> Keras:[https://keras.io/]
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JSJZ201910073" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJZ201910073&amp;v=MDc4MTJyNDlDWjRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpwRnkvblU3ekJMejdCZExHNEg5ak4=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRyNFB2YmhVNFRrV0FxZi9KeHFTZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="0" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=P0ea_aWnQfx5aXr8bS4u3GGmKX_ORbqAf_1HJ_4Trvg1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
