<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637132348218780000%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJSJK201910005%26RESULT%3d1%26SIGN%3doBrHOZkBLGHY9JcPOPTxaLq3oa4%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJK201910005&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJK201910005&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJK201910005&amp;v=MjcyMDFIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJtRnkvZ1ZiN0tMejdCWmJHNEg5ak5yNDlGWVlRS0Q=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#81" data-title="&lt;b&gt;1 引言&lt;/b&gt; "><b>1 引言</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#91" data-title="&lt;b&gt;2 PEC算法&lt;/b&gt; "><b>2 PEC算法</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#92" data-title="&lt;b&gt;2.1 加权蛋白质网络&lt;/b&gt;"><b>2.1 加权蛋白质网络</b></a></li>
                                                <li><a href="#105" data-title="&lt;b&gt;2.2 复合物的挖掘&lt;/b&gt;"><b>2.2 复合物的挖掘</b></a></li>
                                                <li><a href="#115" data-title="&lt;b&gt;2.3 复合物参与度计算&lt;/b&gt;"><b>2.3 复合物参与度计算</b></a></li>
                                                <li><a href="#119" data-title="&lt;b&gt;2.4 子图稠密度&lt;/b&gt;"><b>2.4 子图稠密度</b></a></li>
                                                <li><a href="#127" data-title="&lt;b&gt;2.5 关键蛋白质识别&lt;/b&gt;"><b>2.5 关键蛋白质识别</b></a></li>
                                                <li><a href="#132" data-title="&lt;b&gt;2.6 PEC算法描述&lt;/b&gt;"><b>2.6 PEC算法描述</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#149" data-title="&lt;b&gt;3 实验结果与分析&lt;/b&gt; "><b>3 实验结果与分析</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#150" data-title="&lt;b&gt;3.1 实验环境&lt;/b&gt;"><b>3.1 实验环境</b></a></li>
                                                <li><a href="#152" data-title="&lt;b&gt;3.2 实验数据集&lt;/b&gt;"><b>3.2 实验数据集</b></a></li>
                                                <li><a href="#158" data-title="&lt;b&gt;3.3 参数影响分析&lt;/b&gt;"><b>3.3 参数影响分析</b></a></li>
                                                <li><a href="#162" data-title="&lt;b&gt;3.4 性能比较分析&lt;/b&gt;"><b>3.4 性能比较分析</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#181" data-title="&lt;b&gt;4 结束语&lt;/b&gt; "><b>4 结束语</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#160" data-title="&lt;b&gt;表1 在不同&lt;/b&gt;&lt;i&gt;&lt;b&gt;θ&lt;/b&gt;&lt;/i&gt;&lt;b&gt;取值下的 PEC算法识别正确的关键蛋白质数目&lt;/b&gt;"><b>表1 在不同</b><i><b>θ</b></i><b>取值下的 PEC算法识别正确的关键蛋白质数目</b></a></li>
                                                <li><a href="#168" data-title="图1 DIP数据集上3种算法在不同网络中正确识别的关键蛋白质数目">图1 DIP数据集上3种算法在不同网络中正确识别的关键蛋白质数目</a></li>
                                                <li><a href="#169" data-title="图2 Krogan数据集上3种算法在不同网络中正确识别的关键蛋白质数目">图2 Krogan数据集上3种算法在不同网络中正确识别的关键蛋白质数目</a></li>
                                                <li><a href="#170" data-title="图3 DIP数据集上PEC和其他算法正确识别的关键蛋白质数目">图3 DIP数据集上PEC和其他算法正确识别的关键蛋白质数目</a></li>
                                                <li><a href="#171" data-title="图4 Krogan数据集上PEC和其他算法正确识别的关键蛋白质数目">图4 Krogan数据集上PEC和其他算法正确识别的关键蛋白质数目</a></li>
                                                <li><a href="#180" data-title="图5 PEC算法与其他10种算法在不同蛋白质中
 正确识别的关键蛋白质所占比例的比较">图5 PEC算法与其他10种算法在不同蛋白质中
 正确识别的关键蛋白质所占比例的比较</a></li>
                                                <li><a href="#183" data-title="&lt;b&gt;表2 PEC与其他10种算法在DIP数据集上的统计指标比较&lt;/b&gt;"><b>表2 PEC与其他10种算法在DIP数据集上的统计指标比较</b></a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="5">


                                    <a id="bibliography_1" title=" Lei X J,Yang X Q.A new method for predicting essential proteins based on participation degree in protein complex and subgraph density [J].PLoS ONE,2018,13(6):0198998-0199016." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A new method for predicting essential proteins based on participation degree in protein complex and subgraph density">
                                        <b>[1]</b>
                                         Lei X J,Yang X Q.A new method for predicting essential proteins based on participation degree in protein complex and subgraph density [J].PLoS ONE,2018,13(6):0198998-0199016.
                                    </a>
                                </li>
                                <li id="7">


                                    <a id="bibliography_2" title=" Li M,Zheng R Q,Zhang H H,et al.Effective identification of essential proteins based on priori knowledge,network topology and gene expressions[J].Methods,2014,67:325-333." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES14032300121250&amp;v=MDM0MDBOaWZPZmJLOEh0TE9ySTlGWmVrT0RuazVvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMZklKMXNTYXhBPQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[2]</b>
                                         Li M,Zheng R Q,Zhang H H,et al.Effective identification of essential proteins based on priori knowledge,network topology and gene expressions[J].Methods,2014,67:325-333.
                                    </a>
                                </li>
                                <li id="9">


                                    <a id="bibliography_3" title=" Zhao B H,Wang J X,Li X Y,et al.Essential protein discovery based on a combination of modularity and conservatism[J].Methods,2016,110:54-63." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES668C12C11EEF2383BDC4AD36DF5EC5F0&amp;v=MDE2OTFocW1aRGZNZm5RTXlmQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TkJodzcyNXdhbz1OaWZPZmJXK0ZxTE5yZnhFWlo1NmVuNDZ4eFZobmt4NU9Rdg==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[3]</b>
                                         Zhao B H,Wang J X,Li X Y,et al.Essential protein discovery based on a combination of modularity and conservatism[J].Methods,2016,110:54-63.
                                    </a>
                                </li>
                                <li id="11">


                                    <a id="bibliography_4" title=" Li M,Lu Y,Wang J X,et al.A topology potential-based method for identifying essential proteins from PPI networks [J].IEEE/ACM Transactions on Computational Biology and Bioinformatics,2015,12(2):372-383." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM138E56080BE6A893ABFD005D8209D1AA&amp;v=MjAzMjA1TkJodzcyNXdhbz1OaWZJWTdLN0ZxVEpxWTlOWkpsNkNnMHh4aFZpbUVrSlNIL24yQm8zZWJ2Z1JNdnVDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[4]</b>
                                         Li M,Lu Y,Wang J X,et al.A topology potential-based method for identifying essential proteins from PPI networks [J].IEEE/ACM Transactions on Computational Biology and Bioinformatics,2015,12(2):372-383.
                                    </a>
                                </li>
                                <li id="13">


                                    <a id="bibliography_5" title=" Jeong H,Mason S P,Barabasi A L,et al.Lethality and centrality in protein networks [J].Nature,2001,411 (6833):41-42." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Lethality and centrality in protein networks">
                                        <b>[5]</b>
                                         Jeong H,Mason S P,Barabasi A L,et al.Lethality and centrality in protein networks [J].Nature,2001,411 (6833):41-42.
                                    </a>
                                </li>
                                <li id="15">


                                    <a id="bibliography_6" title=" Vallabhajosyula R R,Chakravarti D,Lutfeali S,et al.Identifying hubs in protein interaction networks [J].PLoS ONE,2009,4(4):5344-5353." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Identifying hubs in protein interaction networks">
                                        <b>[6]</b>
                                         Vallabhajosyula R R,Chakravarti D,Lutfeali S,et al.Identifying hubs in protein interaction networks [J].PLoS ONE,2009,4(4):5344-5353.
                                    </a>
                                </li>
                                <li id="17">


                                    <a id="bibliography_7" title=" Newman M E J.A measure of betweenness centrality based on random walks [J].Social Networks,2005,27(1):39-54." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13012100484943&amp;v=MDA3NzY2VDRQUUgvaXJSZEdlcnFRVE1ud1plWnVIeWptVUxmSUoxc1NheEE9TmlmT2ZiSzdIdERPcm85RllPTUxCWGc2b0JNVA==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[7]</b>
                                         Newman M E J.A measure of betweenness centrality based on random walks [J].Social Networks,2005,27(1):39-54.
                                    </a>
                                </li>
                                <li id="19">


                                    <a id="bibliography_8" title=" Wuchty S,Stadler P F.Centers of complex networks [J].Journal of Theoretical Biology,2003,223(1):45-53." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011600976673&amp;v=MzExMjl5am1VTGZJSjFzU2F4QT1OaWZPZmJLN0h0RE5xWTlGYmV3SkNuczZvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SA==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[8]</b>
                                         Wuchty S,Stadler P F.Centers of complex networks [J].Journal of Theoretical Biology,2003,223(1):45-53.
                                    </a>
                                </li>
                                <li id="21">


                                    <a id="bibliography_9" title=" Estrada E,Rodriguez-Velazquez J A.Subgraph centrality in complex networks [J].Physical Review E,Statistical,Nonlinear,&amp;amp; Soft Matter Physics,2005,71(2):056103-056111." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Subgraph centrality in complex networks">
                                        <b>[9]</b>
                                         Estrada E,Rodriguez-Velazquez J A.Subgraph centrality in complex networks [J].Physical Review E,Statistical,Nonlinear,&amp;amp; Soft Matter Physics,2005,71(2):056103-056111.
                                    </a>
                                </li>
                                <li id="23">


                                    <a id="bibliography_10" title=" Bonacich P.Power and centrality:A family of measures [J].American Journal of Sociology,1987,92(5):1170-1182." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJST&amp;filename=SJST14112600395588&amp;v=Mjg0MTRmSUoxc1NheEE9TmlmWWVySzhIOURPcVk5RlorSUtDWFF4b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadUh5am1VTA==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[10]</b>
                                         Bonacich P.Power and centrality:A family of measures [J].American Journal of Sociology,1987,92(5):1170-1182.
                                    </a>
                                </li>
                                <li id="25">


                                    <a id="bibliography_11" title=" Stephenson K,Zelen M.Rethinking centrality:Methods and examples [J].Social Networks,1989,11(1):1-37." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Rethinking centrality: Methods and examples">
                                        <b>[11]</b>
                                         Stephenson K,Zelen M.Rethinking centrality:Methods and examples [J].Social Networks,1989,11(1):1-37.
                                    </a>
                                </li>
                                <li id="27">


                                    <a id="bibliography_12" title=" Li M,Wang J X,Chen X,et al.A local average connectivity-based method for identifying essential proteins from the network level [J].Computational Biology &amp;amp; Chemistry,2011,35(3):143-150." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011300467914&amp;v=MDY3NzBkR2VycVFUTW53WmVadUh5am1VTGZJSjFzU2F4QT1OaWZPZmJLN0h0RE5ySTlGWU8wSUJYMDlvQk1UNlQ0UFFIL2lyUg==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[12]</b>
                                         Li M,Wang J X,Chen X,et al.A local average connectivity-based method for identifying essential proteins from the network level [J].Computational Biology &amp;amp; Chemistry,2011,35(3):143-150.
                                    </a>
                                </li>
                                <li id="29">


                                    <a id="bibliography_13" title=" Wang J X,Li M,Wang H,et al.Identification of essential proteins based on edge clustering coefficient [J].IEEE/ACM Transactions on Computational Biology &amp;amp; Bioinformatics,2012,9(4):1070-1080." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM50DD78CECE49B7D30F4E0BCE52366242&amp;v=Mjg3NTlXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU5CaHc3MjV3YW89TmlmSVk3YTRhcVhMcC93d0Y1NExCUTQrdXhVVG5Ec0lTQTJSMlJjM2VyU1NSNzZkQ09OdkZTaQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[13]</b>
                                         Wang J X,Li M,Wang H,et al.Identification of essential proteins based on edge clustering coefficient [J].IEEE/ACM Transactions on Computational Biology &amp;amp; Bioinformatics,2012,9(4):1070-1080.
                                    </a>
                                </li>
                                <li id="31">


                                    <a id="bibliography_14" title=" Zhao B H,Wang J X,Li M,et al.Prediction of essential proteins based on overlapping essential modules[J].IEEE Transactions on Nano Bioscience,2014,13(4):415-438." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prediction of essential proteins based on overlapping essential modules">
                                        <b>[14]</b>
                                         Zhao B H,Wang J X,Li M,et al.Prediction of essential proteins based on overlapping essential modules[J].IEEE Transactions on Nano Bioscience,2014,13(4):415-438.
                                    </a>
                                </li>
                                <li id="33">


                                    <a id="bibliography_15" title=" Li Min,Zhang Han-hui,Fei Yao-ping.Essential protein discovery method based on integration of PPI and gene expression data [J].Journal of Central South University(Science and Technology),2013,44(3):1024-1029.(in Chinese)" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZNGD201303027&amp;v=MTMzOThRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJtRnkvZ1ZiN0tQeVBNYXJHNEg5TE1ySTlIWTQ=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[15]</b>
                                         Li Min,Zhang Han-hui,Fei Yao-ping.Essential protein discovery method based on integration of PPI and gene expression data [J].Journal of Central South University(Science and Technology),2013,44(3):1024-1029.(in Chinese)
                                    </a>
                                </li>
                                <li id="35">


                                    <a id="bibliography_16" title=" Tang X W,Wang J X,Zhong J C,et al.Predicting essential proteins based on weighted degree centrality[J].IEEE/ACM Transactions on Computational Biology &amp;amp; Bioinformatics,2014,11(2):407-418." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM605B69FA435C552FC3A1ACF53A55DF11&amp;v=Mjc2NDlPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU5CaHc3MjV3YW89TmlmSVk3VzRHNlBLcHZrMFlPZ0tmM2s4eldCZzZVNThPUXlVcVJGRWZMZmdNN3VlQw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[16]</b>
                                         Tang X W,Wang J X,Zhong J C,et al.Predicting essential proteins based on weighted degree centrality[J].IEEE/ACM Transactions on Computational Biology &amp;amp; Bioinformatics,2014,11(2):407-418.
                                    </a>
                                </li>
                                <li id="37">


                                    <a id="bibliography_17" title=" Luo J W,Qi Y.Identification of essential proteins based on a new combination of local interaction density and protein complexes [J].PLoS ONE,2015,10(6):0131418-0131444." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Identification of Essential Proteins Based on a New Combination of Local Interaction Density and Protein Complexes">
                                        <b>[17]</b>
                                         Luo J W,Qi Y.Identification of essential proteins based on a new combination of local interaction density and protein complexes [J].PLoS ONE,2015,10(6):0131418-0131444.
                                    </a>
                                </li>
                                <li id="39">


                                    <a id="bibliography_18" title=" Qin C,Sun Y Q,Dong Y D.A new method for identifying essential proteins based on network topology properties and protein complexes [J].PLoS ONE,2016,11(8):0161042-0161071." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A new method for identifying essential proteins based on network topology properties and protein complexes">
                                        <b>[18]</b>
                                         Qin C,Sun Y Q,Dong Y D.A new method for identifying essential proteins based on network topology properties and protein complexes [J].PLoS ONE,2016,11(8):0161042-0161071.
                                    </a>
                                </li>
                                <li id="41">


                                    <a id="bibliography_19" title=" Li M,Lu Y,Niu Z B,et al.United complex centrality for identification of essential proteins from PPI networks [J].IEEE/ACM Transactions on Computational Biology &amp;amp; Bioinformatics,2017,14(2):370-380." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCMEBA518DA150F1129D89F1968FD2633A9&amp;v=MzE3Mzdhbz1OaWZJWThiS2I5VE5wL3MwWmU0UGVuMDR6UjluNGpZTFNYYmtwR1JCZTdTWFJzdVdDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOQmh3NzI1dw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[19]</b>
                                         Li M,Lu Y,Niu Z B,et al.United complex centrality for identification of essential proteins from PPI networks [J].IEEE/ACM Transactions on Computational Biology &amp;amp; Bioinformatics,2017,14(2):370-380.
                                    </a>
                                </li>
                                <li id="43">


                                    <a id="bibliography_20" title=" Jiang Y X,Wang Y,Pang W,et al.Essential protein identification based on essential protein-protein interaction prediction by integrated edge weights[J].Methods,2015,83(13):51-62." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES9F6BD82D006F6CA7604219A2AF08ABAD&amp;v=MDg1MzVHUWxmQ3BiUTM1TkJodzcyNXdhbz1OaWZPZmJyT0dLTzRwNDB4Wk9zSmVucEt2aEVWNmp0L1NYYVRybU5EZWJybE44dnJDT052RlNpV1dyN0pJRnBtYUJ1SFlmTw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[20]</b>
                                         Jiang Y X,Wang Y,Pang W,et al.Essential protein identification based on essential protein-protein interaction prediction by integrated edge weights[J].Methods,2015,83(13):51-62.
                                    </a>
                                </li>
                                <li id="45">


                                    <a id="bibliography_21" >
                                        <b>[21]</b>
                                     Hong Hai-yan,Liu Wei.Efficient prediction method of essential proteins based on PPI network [J].Computer Science,2016,43(11):16-20.(in Chinese)</a>
                                </li>
                                <li id="47">


                                    <a id="bibliography_22" title=" Zhong J C,Wang J X,Peng W,et al.A feature selection method for prediction essential protein [J].Tsinghua Science and Technology,2015,20(5):491-499." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=QHDY201505007&amp;v=MDA3MzNvOUZZNFFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUm1GeS9nVmI3S05DWFBkN0c0SDlUTXE=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[22]</b>
                                         Zhong J C,Wang J X,Peng W,et al.A feature selection method for prediction essential protein [J].Tsinghua Science and Technology,2015,20(5):491-499.
                                    </a>
                                </li>
                                <li id="49">


                                    <a id="bibliography_23" >
                                        <b>[23]</b>
                                     Fan Zi-jing,Luo Ze,Ma Yong-zheng.A spectral clustering algorithm based on fuzzy kernel clustering [J].Computer Engineering,2017,43(11):161-165.(in Chinese)</a>
                                </li>
                                <li id="51">


                                    <a id="bibliography_24" title=" Zhao Xue-wu,Cheng Xin-dang,L&#252; Jia-wei,et al.Identify protein complexes by integrating temporal function continue feature and ant colony clustering on dynamic PPI networks[J].Journal of Chinese Computer Systems,2017,38(6):1311-1316.(in Chinese)" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=XXWX201706026&amp;v=MTM1OTdCdEdGckNVUkxPZVplUm1GeS9nVmI3S1BUWGNkckc0SDliTXFZOUhZb1FLREg4NHZSNFQ2ajU0TzN6cXE=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[24]</b>
                                         Zhao Xue-wu,Cheng Xin-dang,L&#252; Jia-wei,et al.Identify protein complexes by integrating temporal function continue feature and ant colony clustering on dynamic PPI networks[J].Journal of Chinese Computer Systems,2017,38(6):1311-1316.(in Chinese)
                                    </a>
                                </li>
                                <li id="53">


                                    <a id="bibliography_25" title=" Shang X,Wang Y,Chen B.Identifying essential proteins based on dynamic protein-protein interaction networks and RNA-Seq datasets [J].Science China Information Sciences,2016,59(7):1-11." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFXG201607007&amp;v=MzA0MjJ2VGFiRzRIOWZNcUk5Rlk0UUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSbUZ5L2dWYjdLTHk=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[25]</b>
                                         Shang X,Wang Y,Chen B.Identifying essential proteins based on dynamic protein-protein interaction networks and RNA-Seq datasets [J].Science China Information Sciences,2016,59(7):1-11.
                                    </a>
                                </li>
                                <li id="55">


                                    <a id="bibliography_26" title=" Xenarios I,Salwinski L,Duan X J,et al.DIP,the database of interacting proteins:A research tool for studying cellular networks of protein interactions [J].Nucleic Acids Research,2002,30(1):303-305." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=DIP, the Database of Interacting Proteins: a research tool for studying cellular networks of protein interactions">
                                        <b>[26]</b>
                                         Xenarios I,Salwinski L,Duan X J,et al.DIP,the database of interacting proteins:A research tool for studying cellular networks of protein interactions [J].Nucleic Acids Research,2002,30(1):303-305.
                                    </a>
                                </li>
                                <li id="57">


                                    <a id="bibliography_27" title=" Krogan N J,Cagney G,Yu H Y,et al.Global landscape of protein complexes in the yeast saccharomyces cerevisiae [J].Nature,2006,440(7084):637-643." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Global landscape of protein complexes in the yeast Saccharomyces cerevisiae">
                                        <b>[27]</b>
                                         Krogan N J,Cagney G,Yu H Y,et al.Global landscape of protein complexes in the yeast saccharomyces cerevisiae [J].Nature,2006,440(7084):637-643.
                                    </a>
                                </li>
                                <li id="59">


                                    <a id="bibliography_28" title=" Blake J A,Christie K R,Done J,et al.Gene ontology consortium:Going forward [J].Nucleic Acids Research,2015,43(Database issue):1049-1056." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Gene Ontology Consortium:going forward">
                                        <b>[28]</b>
                                         Blake J A,Christie K R,Done J,et al.Gene ontology consortium:Going forward [J].Nucleic Acids Research,2015,43(Database issue):1049-1056.
                                    </a>
                                </li>
                                <li id="61">


                                    <a id="bibliography_29" title=" Mewes H W,Frishman D,Mayer K F X,et al.MIPS:Analysis and annotation of proteins from whole genomes [J].Nucleic Acids Research,2006,34(Database issue):169-172." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=MIPS: analysis and annotation of proteins from whole genomes in 2005">
                                        <b>[29]</b>
                                         Mewes H W,Frishman D,Mayer K F X,et al.MIPS:Analysis and annotation of proteins from whole genomes [J].Nucleic Acids Research,2006,34(Database issue):169-172.
                                    </a>
                                </li>
                                <li id="63">


                                    <a id="bibliography_30" title=" Cherry J M,Adler C,Ball C,et al.SGD:Saccharomyces genome database [J].Nucleic Acids Research,1998,26 (1):73-79." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=SGD: Saccharomyces Genome Database">
                                        <b>[30]</b>
                                         Cherry J M,Adler C,Ball C,et al.SGD:Saccharomyces genome database [J].Nucleic Acids Research,1998,26 (1):73-79.
                                    </a>
                                </li>
                                <li id="65">


                                    <a id="bibliography_31" title=" Zhang R,Lin Y.DEG 5.0,a database of essential genes in both prokaryotes and eukaryotes [J].Nucleic Acids Research,2009,37 (Database issue):455-458." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=DEG 5.0, a database of essential genes in both prokaryotes and eukaryotes">
                                        <b>[31]</b>
                                         Zhang R,Lin Y.DEG 5.0,a database of essential genes in both prokaryotes and eukaryotes [J].Nucleic Acids Research,2009,37 (Database issue):455-458.
                                    </a>
                                </li>
                                <li id="67">


                                    <a id="bibliography_32" title=" Antoniotti M,Bader G D,Caravagna G,et al.Gestodifferent:A cytoscape plugin for the generation and the identification of gene regulatory networks describing a stochastic cell differentiation process [J].Bioinformatics,2013,29(4):513-514." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=GeStoDifferent: a Cytoscape plugin for the generation and the identification of gene regulatory networks describing a stochastic cell differentiation process">
                                        <b>[32]</b>
                                         Antoniotti M,Bader G D,Caravagna G,et al.Gestodifferent:A cytoscape plugin for the generation and the identification of gene regulatory networks describing a stochastic cell differentiation process [J].Bioinformatics,2013,29(4):513-514.
                                    </a>
                                </li>
                                <li id="69">


                                    <a id="bibliography_33" title=" Hong Hai-yan,Liu Wei.Research on essential protein identification method based on improved PSP algorithm [J].Computer Science,2017,44(10):38-44.(in Chinese)" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJA201710007&amp;v=MjA4NDlqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSbUZ5L2dWYjdLTHo3QmI3RzRIOWJOcjQ5Rlk0UUtESDg0dlI0VDY=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[33]</b>
                                         Hong Hai-yan,Liu Wei.Research on essential protein identification method based on improved PSP algorithm [J].Computer Science,2017,44(10):38-44.(in Chinese)
                                    </a>
                                </li>
                                <li id="71">


                                    <a id="bibliography_15" >
                                        <b>[15]</b>
                                     李敏,张含会,费耀平.融合PPI和基因表达数据的关键蛋白质识别方法[J].中南大学学报(自然科学版),2013,44(3):1024-1029.</a>
                                </li>
                                <li id="73">


                                    <a id="bibliography_21" title=" 洪海燕,刘维.基于PPI网络的关键蛋白质的高效预测算法[J].计算机科学,2016,43(11):16-20." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJA2016S2004&amp;v=MDk5MTdJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSbUZ5L2dWYjdLTHo3QmI3RzRIOWV2clk5Rlk=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[21]</b>
                                         洪海燕,刘维.基于PPI网络的关键蛋白质的高效预测算法[J].计算机科学,2016,43(11):16-20.
                                    </a>
                                </li>
                                <li id="75">


                                    <a id="bibliography_23" title=" 范子静,罗泽,马永征.一种基于模糊核聚类的谱聚类算法[J].计算机工程,2017,43(11):161-165." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201711027&amp;v=MjcxODZHNEg5Yk5ybzlIWTRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJtRnkvZ1ZiN0tMejdCYmI=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[23]</b>
                                         范子静,罗泽,马永征.一种基于模糊核聚类的谱聚类算法[J].计算机工程,2017,43(11):161-165.
                                    </a>
                                </li>
                                <li id="77">


                                    <a id="bibliography_24" >
                                        <b>[24]</b>
                                     赵学武,程新党,吕嘉伟,等.融合时序保持特征和蚁群聚类的动态PPI网络复合物识别[J].小型微型计算机系统,2017,38(6):1311-1316.</a>
                                </li>
                                <li id="79">


                                    <a id="bibliography_33" >
                                        <b>[33]</b>
                                     洪海燕,刘维.基于改进PSO算法的关键蛋白质识别方法研究[J].计算机科学,2017,44(10):38-44.</a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JSJK" target="_blank">计算机工程与科学</a>
                2019,41(10),1738-1748 DOI:10.3969/j.issn.1007-130X.2019.10.004            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>基于复合物参与度和密度的关键蛋白质预测</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%AF%9B%E4%BC%8A%E6%95%8F&amp;code=10557422&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">毛伊敏</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%88%98%E9%93%B6%E8%90%8D&amp;code=41172130&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">刘银萍</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E6%B1%9F%E8%A5%BF%E7%90%86%E5%B7%A5%E5%A4%A7%E5%AD%A6%E4%BF%A1%E6%81%AF%E5%B7%A5%E7%A8%8B%E5%AD%A6%E9%99%A2&amp;code=0217990&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">江西理工大学信息工程学院</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>针对在蛋白质相互作用网络上的关键蛋白质识别只关注拓扑特性,蛋白质相互作用数据中存在较高比例的假阳性数据以及基于复合物信息的关键蛋白质识别算法对节点的邻域信息和复合物的挖掘对关键蛋白质的识别影响效果考虑不够全面等导致的识别准确率和特异性不高的问题,提出一种基于复合物参与度和密度的关键蛋白质预测算法PEC。首先融合GO注释信息和边聚集系数构造加权PPI网络,克服假阳性对实验结果的影响;基于蛋白质相互作用的边权重,构造相似度矩阵,设计特征值间的最大本征差值来自动确定划分数目<i>K</i>,同时根据加权网络中的蛋白质节点度来选取<i>K</i>个初始聚类中心,进而利用谱聚类结合模糊C-means聚类算法实现复合物的挖掘,提高聚类的准确率,降低数据的维数;其次基于蛋白质节点的复合物参与度以及节点邻域子图密度,设计出关键节点的关键性得分。在DIP和Krogan 2个数据集上,将PEC与DC、BC、CC、SC、IC、PeC、WDC、LIDC、LBCC和UC 10种经典算法相比,实验结果表明,PEC算法能够识别出更多的关键蛋白质,且聚类结果的准确率和特异性较高。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%9B%B8%E4%BA%92%E4%BD%9C%E7%94%A8%E7%BD%91%E7%BB%9C&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蛋白质相互作用网络;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%B0%B1%E8%81%9A%E7%B1%BB%E7%AE%97%E6%B3%95&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">谱聚类算法;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%9B%8B%E7%99%BD%E8%B4%A8%E5%A4%8D%E5%90%88%E7%89%A9&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蛋白质复合物;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%AF%86%E5%BA%A6&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">密度;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%85%B3%E9%94%AE%E8%9B%8B%E7%99%BD%E8%B4%A8&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">关键蛋白质;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    毛伊敏（1970-），女，江西赣州人，博士，教授，研究方向为数据挖掘和地理信息系统。E-mail:mymlyc@163.com，通信地址：341000江西省赣州市黄金校区章贡区客家大道156号江西理工大学信息工程学院&lt;image id="244" type="formula" href="images/JSJK201910005_24400.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                                <span>
                                    刘银萍（1993-），女，山西朔州人，硕士生，研究方向为数据挖掘和地理信息系统。E-mail:2304676691@qq.com，通信地址：341000江西省赣州市黄金校区章贡区客家大道156号江西理工大学信息工程学院&lt;image id="246" type="formula" href="images/JSJK201910005_24600.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-12-21</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金(41562019);</span>
                                <span>江西省自然科学基金(GJJ161566);</span>
                    </p>
            </div>
                    <h1><b>An essential proteins prediction algorithm based on participation degree in protein complex and density</b></h1>
                    <h2>
                    <span>MAO Yi-min</span>
                    <span>LIU Yin-ping</span>
            </h2>
                    <h2>
                    <span>School of Information Engineering,Jiangxi University of Science and Technology</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>The identification of essential proteins in the protein-protein interaction(PPI) network tends to only focus on the topological characteristics of the nodes, and the PPI data contains high false positive, the neighborhood information of nodes and the influence of complex mining on the recognition of essential proteins are not considered comprehensively by the essential proteins recognition algorithm based on complex information, so the accuracy and specificity of the recognition results are not high. In order to deal with these problems, an essential proteins prediction algorithm based on participation degree in protein complex and density(PEC) is proposed. Firstly, the GO annotation information and the edge aggregation coefficient are used to construct the weighted PPI network to overcome the influence of false positives on the experimental results. Based on the edge weight of protein interaction, the similarity matrix is constructed. The maximum difference between eigenvectors is designed to automatically determine the partition number <i>K</i>. Meanwhile, <i>K</i> initial clustering centers are selected according to the degree of protein nodes in the weighted network. Furthermore, the spectral clustering and the fuzzy C-means(FCM) clustering algorithm are combined to excavate the protein complex, thus improving the clustering accuracy and reduces the data dimension. Secondly, based on the degree of participation in protein complex and the neighborhood subgraph density, the scores of the essential proteins are proposed. The experiment results on DIP and Krogan datasets show that, compared with 10 classic algorithms such as DC, BC, CC, SC, IC, PeC, WDC, LIDC, LBCC and UC, PEC can correctly identify more essential proteins with higher accuracy and specificity.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=protein-protein%20interaction%20network&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">protein-protein interaction network;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=spectral%20clustering%20algorithm&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">spectral clustering algorithm;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=protein%20complexes&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">protein complexes;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=density&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">density;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=essential%20proteins&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">essential proteins;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                            <b>Author：</b>
                                                        <span>
                                    MAO Yi-min,born in 1970,PhD,professor,her research interests include data mining,and geographic information system.;
                                </span>
                                <span>
                                    LIU Yin-ping,born in 1993,MS candidate,her research interests include data mining,and geographic information system.;
                                </span>
                    </p>
                                    <p><b>Received：</b> 2018-12-21</p>
                                    <p>
                                            </p>
            </div>


        <!--brief start-->
                        <h3 id="81" name="81" class="anchor-tag"><b>1 引言</b></h3>
                <div class="p1">
                    <p id="82">关键蛋白质是指将其通过基因剔除式突变移除后,会造成有关蛋白质复合体功能丧失,并导致生物体无法生存的蛋白质<citation id="196" type="reference"><link href="5" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>。关键蛋白质的预测识别对研究组织细胞结构、执行生理功能以及药物研究设计具有重要的意义。因此,正确地识别出关键蛋白质是生物信息领域中的一项重要研究内容。</p>
                </div>
                <div class="p1">
                    <p id="83">随着生物信息学的发展,已经出现了大量关键蛋白质识别的算法。早期的识别算法是生物实验方法<sup></sup><citation id="197" type="reference"><link href="7" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>。生物实验方法识别关键蛋白质虽然准确率高,但是这些算法不仅费时费力,而且代价极高,不同的实验条件也影响着关键蛋白质的识别效果<citation id="198" type="reference"><link href="9" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>。随着生物学信息领域高通量技术的发展,尤其是蛋白质相互作用数据日益完善,研究者们提出了大量基于计算方法的识别关键蛋白质的新算法。在计算方法上,大多数研究者都致力于研究蛋白质相互作用PPI(Protein-Protein Interaction)网络的拓扑结构和关键蛋白质之间的联系<sup></sup><citation id="199" type="reference"><link href="11" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>。Jeong等<citation id="200" type="reference"><link href="13" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>指出在蛋白质相互作用网络中,参与较多相互作用的蛋白质即高度中心连接的蛋白质具有很高的概率成为关键蛋白质,并且也证明了关键蛋白质与拓扑结构之间存在紧密关系。因此,基于蛋白质的关键性和网络的拓扑中心性之间的相关性,大量识别关键蛋白质的中心性算法被提出。典型的中心性识别算法有度中心DC(Degree Centrality)<sup></sup><citation id="201" type="reference"><link href="15" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>、介数中心性BC(Betweeness Centrality)<sup></sup><citation id="202" type="reference"><link href="17" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>、接近度中心性CC(Closeness Centrality)<sup></sup><citation id="203" type="reference"><link href="19" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>、子图中心性SC(Subgraph Centrality)<sup></sup><citation id="204" type="reference"><link href="21" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>、特征向量中心性EC(Eigenvector Centrality)<sup></sup><citation id="205" type="reference"><link href="23" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>、信息中心性IC(Information Centrality)<sup></sup><citation id="206" type="reference"><link href="25" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>、局部平均连通性LAC(Local Average Connectivity)<sup></sup><citation id="207" type="reference"><link href="27" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>和邻居中心性NC(Neighbor Centrality)<sup></sup><citation id="208" type="reference"><link href="29" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>等。但是,仅仅依靠PPI网络的拓扑特性来预测关键蛋白质是不足的。首先蛋白质相互作用数据存在着较高比例的假阳性和假阴性;其次蛋白质相互作用数据包含不稳定或者发生在不同的时间点的相互作用,不能代表真实的相互作用网络;并且已有的算法仅仅考虑了蛋白质网络的拓扑结构,没有考虑关键蛋白质的内在属性以及生物特性<citation id="209" type="reference"><link href="31" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>。</p>
                </div>
                <div class="p1">
                    <p id="84">因此,为了进一步提高预测准确率,研究者将生物信息融入到拓扑蛋白质网络中,提出了一些新的识别关键蛋白质的算法。基于基因表达数据和PPI数据,李敏等<citation id="210" type="reference"><link href="33" rel="bibliography" /><link href="71" rel="bibliography" /><sup>[<a class="sup">15</a>,<a class="sup">15</a>]</sup></citation>提出一种新识别关键蛋白质的中心性测度PeC(essential protein discovery method based on integration of PPI and gene expression data)。Tang等<citation id="211" type="reference"><link href="35" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation>利用基因表达数据来计算2个蛋白质节点之间的皮尔逊相关系数,以衡量蛋白质之间的可信度,并且也将边聚集系数ECC (Edge Clustering Coefficient)作为计算的参数,计算相互作用的边权值,进而提出了识别关键蛋白质的加权度中心算法WDC(Weighted Degree Centrality),降低了假阳性数据对实验结果的影响。实验结果表明,融合了生物信息的算法比仅仅考虑拓扑结构的算法能够识别出更多的关键蛋白质。除此之外,研究者还提出基于蛋白质复合物信息来识别关键蛋白质。例如,Luo等<citation id="212" type="reference"><link href="37" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>在对蛋白质和蛋白质复合物进行统计分析的基础上,提出了一种预测蛋白质相互作用网络中关键蛋白质的新算法,即局部相互作用联合蛋白质复合物LIDC(Local Interaction Density Combined with protein complexes)算法。Qin等<citation id="213" type="reference"><link href="39" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>提出了基于局部密度、介数中心性和复合物度中心性IDC (In-Degree Centrality of complex)的关键蛋白质识别算法LBCC(Local density,Betweenness Centrality and in-degree Centrality of complex)。Li等<citation id="214" type="reference"><link href="41" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>通过整合蛋白复合物与PPI网络拓扑特征,提出识别关键蛋白质的联合度中心性UC(United complex Centrality)算法。</p>
                </div>
                <div class="p1">
                    <p id="85">虽然基于复合物信息的关键蛋白质识别取得了一定的成效,但是只考虑了复合物节点的内度,没有考虑复合物外部节点的复合物参与度,即没有考虑节点的邻域信息,对蛋白质节点的复合物参与程度考虑不全面;而且基于复合物信息的关键蛋白质识别算法仅仅是基于已知的复合物来进行关键蛋白质挖掘,而复合物的挖掘过程对关键蛋白质识别影响效果的相关研究较少;此外,对于蛋白质网络中参与度小的蛋白质节点,算法的预测性能比较差。虽然基于拓扑特性的关键蛋白质识别算法取得了一定的成效,但是高通量生物技术获得的大规模蛋白质相互作用网络数据存在一定比例的假阳性数据、不完备数据和噪声数据,因此仅仅依靠网络的拓扑特性来识别关键蛋白质较依赖于网络的可靠性;以及基于复合物信息的识别关键蛋白质的算法对节点的邻域信息和复合物的挖掘对于关键蛋白质的识别效果的影响研究考虑不够全面等导致识别的准确性和特异性不高等缺陷,仍是亟待解决的问题。</p>
                </div>
                <div class="p1">
                    <p id="86">针对上述问题,本文提出了一种基于复合物参与度和密度的关键蛋白质预测算法PEC(Prediction algorithm of Essential proteins based on participation degree in protein Complex and density)。主要工作为:</p>
                </div>
                <div class="p1">
                    <p id="87">(1)融合GO(Gene Ontology)注释信息以及边聚集系数构建加权蛋白质网络,其中GO注释信息用来计算功能相似性,边聚集系数用来计算节点间的拓扑相似性。</p>
                </div>
                <div class="p1">
                    <p id="88">(2)设计谱聚类算法结合模糊C-means FCM(Fuzzy C-means)算法挖掘蛋白质复合物。</p>
                </div>
                <div class="p1">
                    <p id="89">(3)基于蛋白质节点的复合物参与度以及节点邻域子图密度,设计关键节点的关键性得分。</p>
                </div>
                <div class="p1">
                    <p id="90">(4)设计实验对比分析本文算法与其他经典算法。</p>
                </div>
                <h3 id="91" name="91" class="anchor-tag"><b>2 PEC算法</b></h3>
                <h4 class="anchor-tag" id="92" name="92"><b>2.1 加权蛋白质网络</b></h4>
                <div class="p1">
                    <p id="93">针对蛋白质相互作用数据中存在着一定数量的假阳性数据,以及通过高通量算法获得的蛋白质相互作用数据具有不完整性,仅仅依靠网络的拓扑特性来识别关键蛋白质较依赖于网络可靠性的问题,限制了关键蛋白质识别算法的性能。考虑到在蛋白质相互作用网络中,蛋白质的关键性也存在着与网络节点同样的性质,即关键蛋白质之间的联系是非常密切的,相互之间会有很大的概率产生相互作用,而且联系较密切的蛋白质成为关键蛋白质的可能性较大<citation id="215" type="reference"><link href="43" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>。因此,本文基于关键蛋白质倾向于成簇出现,且蛋白质的关键性与相互作用的可靠程度之间关系密切,综合考虑蛋白质网络的拓扑特性和节点的生物信息,设计出GO语义相似性和边聚集系数衡量相互作用的可靠程度,进而构建加权PPI网络。相互作用的可靠程度由2方面来衡量,一方面设计GO注释信息来衡量蛋白质之间的功能相似性,另一方面设计边聚集系数来衡量相互作用的可信度。</p>
                </div>
                <div class="p1">
                    <p id="94">相互作用的可靠程度可通过蛋白质之间的功能相似性衡量,若2个蛋白质之间的功能相似性越高,其相互作用越可靠。GO术语是对基因的功能进行注释的结构化自然语言,研究证实,2个蛋白质之间共有的GO术语越多,它们的生物功能越相似<citation id="216" type="reference"><link href="45" rel="bibliography" /><link href="73" rel="bibliography" /><sup>[<a class="sup">21</a>,<a class="sup">21</a>]</sup></citation>,故蛋白质之间的可靠性可以通过GO术语上的功能相似性进行评价。</p>
                </div>
                <div class="p1">
                    <p id="95">令<i>GO</i>(<i>v</i>)表示蛋白质节点<i>v</i>的GO注释信息组成的集合,<i>GO</i>-<i>sum</i>表示2个蛋白质节点之间的GO功能相似性。则2个蛋白质节点<i>v</i>和<i>u</i>之间功能相似性的计算如式(1)所示:</p>
                </div>
                <div class="p1">
                    <p id="96" class="code-formula">
                        <mathml id="96"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>G</mi><mi>Ο</mi><mo>-</mo><mi>s</mi><mi>u</mi><mi>m</mi><mrow><mo>(</mo><mrow><mi>v</mi><mo>,</mo><mi>u</mi></mrow><mo>)</mo></mrow><mo>=</mo><mrow><mo>{</mo><mtable columnalign="left"><mtr><mtd><mfrac><mrow><mo stretchy="false">|</mo><mi>G</mi><mi>Ο</mi><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo><mstyle displaystyle="true"><mo>∩</mo><mi>G</mi></mstyle><mi>Ο</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo><msup><mrow></mrow><mn>2</mn></msup></mrow><mrow><mo stretchy="false">|</mo><mi>G</mi><mi>Ο</mi><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo><mo>×</mo><mo stretchy="false">|</mo><mi>G</mi><mi>Ο</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo></mrow></mfrac><mo>,</mo></mtd></mtr><mtr><mtd><mtext> </mtext><mo stretchy="false">|</mo><mi>G</mi><mi>Ο</mi><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo><mo>&gt;</mo><mn>0</mn><mo>,</mo><mo stretchy="false">|</mo><mi>G</mi><mi>Ο</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo><mo>&gt;</mo><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn><mo>,</mo><mtext>其</mtext><mtext>他</mtext></mtd></mtr></mtable></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="97">其中,<i>GO</i>(<i>v</i>)∩<i>GO</i>(<i>u</i>)表示<i>v</i>和<i>u</i>共同的GO术语集合,|<i>GO</i>(<i>v</i>)|和|<i>GO</i>(<i>u</i>)|分别表示<i>v</i>和<i>u</i>的GO注释数目。</p>
                </div>
                <div class="p1">
                    <p id="98">边聚集系数作为PPI网络的一个重要拓扑特性,它不仅可以描述蛋白质之间相互作用的重要程度以及2个蛋白质之间的亲疏程度,还可以评估蛋白质之间属于同一簇的概率。为了降低假阳性对识别关键蛋白质实验结果的影响,基于关键蛋白质往往是成簇出现,本文使用边聚集系数来衡量一对蛋白质之间相互作用的概率。边聚集系数的定义为<citation id="217" type="reference"><link href="47" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>:</p>
                </div>
                <div class="p1">
                    <p id="99" class="code-formula">
                        <mathml id="99"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>E</mi><mi>C</mi><mi>C</mi><mo>=</mo><mrow><mo>{</mo><mtable columnalign="left"><mtr><mtd><mfrac><mrow><mi>z</mi><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>min</mi><mo stretchy="false">(</mo><mi>d</mi><msub><mrow></mrow><mi>u</mi></msub><mo>-</mo><mn>1</mn><mo>,</mo><mi>d</mi><msub><mrow></mrow><mi>v</mi></msub><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></mfrac><mo>,</mo><mi>d</mi><msub><mrow></mrow><mi>u</mi></msub><mo>&gt;</mo><mn>1</mn><mo>,</mo><mi>d</mi><msub><mrow></mrow><mi>v</mi></msub><mo>&gt;</mo><mn>1</mn></mtd></mtr><mtr><mtd><mn>0</mn><mo>,</mo><mtext>其</mtext><mtext>他</mtext></mtd></mtr></mtable></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>2</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="100">其中,<mathml id="185"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>z</mi><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mo>)</mo></mrow></mrow></math></mathml>表示蛋白质节点<i>v</i>和蛋白质节点<i>u</i>共同的邻居节点数,<i>d</i><sub><i>u</i></sub>和<i>d</i><sub><i>v</i></sub>分别表示蛋白质节点<i>u</i>和蛋白质节点<i>v</i>的度。PPI网络中的相互作用权值的计算如式(3)所示:</p>
                </div>
                <div class="p1">
                    <p id="101" class="code-formula">
                        <mathml id="101"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>W</mi><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mo>)</mo></mrow><mo>=</mo><mi>G</mi><mi>Ο</mi><mo>-</mo><mi>s</mi><mi>u</mi><mi>m</mi><mo stretchy="false">(</mo><mi>u</mi><mo>,</mo><mi>v</mi><mo stretchy="false">)</mo><mo>+</mo><mi>E</mi><mi>C</mi><mi>C</mi><mo stretchy="false">(</mo><mi>u</mi><mo>,</mo><mi>v</mi><mo stretchy="false">)</mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>3</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="102">蛋白质节点<i>u</i>的加权度的计算如式(4)所示:</p>
                </div>
                <div class="p1">
                    <p id="103" class="code-formula">
                        <mathml id="103"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>S</mi><mi>W</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo>=</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>v</mi><mo>∈</mo><mi>Μ</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo></mrow></munder><mi>W</mi></mstyle><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mo>)</mo></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>4</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="104">其中,<i>M</i>(<i>u</i>)表示蛋白质节点<i>u</i>的邻域节点集合。</p>
                </div>
                <h4 class="anchor-tag" id="105" name="105"><b>2.2 复合物的挖掘</b></h4>
                <div class="p1">
                    <p id="106">针对基于复合物信息的关键蛋白质识别算法中复合物的挖掘对关键蛋白质的识别影响效果考虑不够全面导致的识别准确率和特异性不高的问题,由于基于图论的谱聚类算法实现简单,不局限于原始数据的分布形状,可以收敛于全局最优解<citation id="218" type="reference"><link href="49" rel="bibliography" /><link href="75" rel="bibliography" /><sup>[<a class="sup">23</a>,<a class="sup">23</a>]</sup></citation>,因此,本文采用改进的谱聚类结合FCM算法来实现蛋白质复合物的挖掘。首先,基于蛋白质相互作用的边权重,构造相似度矩阵,设计特征值间的最大本征差值来自动确定划分数目<i>K</i>;同时根据加权网络中的蛋白质节点加权度来选取<i>K</i>个初始聚类中心,进而利用谱聚类结合FCM聚类算法实现复合物的挖掘。基本实现步骤如下:</p>
                </div>
                <div class="p1">
                    <p id="107"><b>输入</b>:<i>n</i>个蛋白质数据。</p>
                </div>
                <div class="p1">
                    <p id="108"><b>输出</b>:节点集合<i>C</i><sub>1</sub>,<i>C</i><sub>2</sub>,…,<i>C</i><sub><i>K</i></sub>。</p>
                </div>
                <div class="p1">
                    <p id="109"><b>步骤1</b> 利用式(3)构造蛋白质节点之间的相似度矩阵<i><b>W</b></i>,其中<i>W</i>(<i>i</i>,<i>j</i>)=<i>Go</i>-<i>sum</i>(<i>i</i>,<i>j</i>)+<i>ECC</i>(<i>i</i>,<i>j</i>);</p>
                </div>
                <div class="p1">
                    <p id="110"><b>步骤2</b> 构造Laplacian矩阵<i><b>P</b></i>=<i><b>D</b></i><sup>-1/22</sup><i><b>WD</b></i><sup>-1/2</sup>,其中<i><b>D</b></i>是对角矩阵,<mathml id="186"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>D</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>i</mi></mrow></msub><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mi>W</mi></mstyle><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>;</mo></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="111"><b>步骤3</b> 计算规范化矩阵<i><b>P</b></i>的特征值,按照从大到小的顺序排列,计算特征值之间的差值<i>d</i><sub><i>i</i></sub>,同时根据<mathml id="187"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Κ</mi><mo>=</mo><munder><mstyle mathsize="140%" displaystyle="true"><mrow><mi>arg</mi><mspace width="0.25em" /><mi>max</mi></mrow></mstyle><mi>i</mi></munder><mo stretchy="false">{</mo><mi>d</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">}</mo></mrow></math></mathml>来确定网络划分数量;</p>
                </div>
                <div class="p1">
                    <p id="112"><b>步骤3</b> 根据<i><b>P</b></i>的<i>K</i>个最大特征值对应的特征向量,构造矩阵<i><b>V</b></i>=[<i><b>v</b></i><sub>1</sub>,<i><b>v</b></i><sub>2</sub>,…,<i><b>v</b></i><sub><i>k</i></sub>];</p>
                </div>
                <div class="p1">
                    <p id="113"><b>步骤4</b> 规范化<i><b>V</b></i>的列向量,得到矩阵<i><b>Y</b></i>,其中,矩阵<i><b>Y</b></i>的元素<mathml id="188"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Y</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>V</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mrow><mo stretchy="false">(</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mi>j</mi></munder><mi>V</mi></mstyle><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo><msup><mrow></mrow><mrow><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup></mrow></mfrac></mrow></math></mathml>,<i>V</i><sub><i>ij</i></sub>是矩阵<i><b>V</b></i>的元素;</p>
                </div>
                <div class="p1">
                    <p id="114"><b>步骤5</b> 根据蛋白质节点的加权<mathml id="189"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>S</mi><mi>W</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo>=</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>v</mi><mo>∈</mo><mi>Μ</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo></mrow></munder><mi>W</mi></mstyle><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mo>)</mo></mrow></mrow></math></mathml>,选择出前<i>K</i>个加权度较高的数据初始化FCM,进而采用改进的谱聚类和FCM聚成<i>K</i>类。</p>
                </div>
                <h4 class="anchor-tag" id="115" name="115"><b>2.3 复合物参与度计算</b></h4>
                <div class="p1">
                    <p id="116">蛋白质复合物是指在相同的空间和时间通过相互作用组成1个多分子机制的1组蛋白质,而关键蛋白质和蛋白质复合物之间存在密切的关系<citation id="219" type="reference"><link href="51" rel="bibliography" /><link href="77" rel="bibliography" /><sup>[<a class="sup">24</a>,<a class="sup">24</a>]</sup></citation>。因此,针对复合物参与度的关键蛋白质识别算法对节点的邻域信息考虑不全面,即忽略复合物外部节点的参与程度导致的识别准确率不高的问题,基于2.2节模糊谱聚类算法识别的蛋白质复合物,考虑到蛋白质相互作用网络的节点可以出现在复合物内部,也可以出现在复合物的外部,故本文在计算复合物参与度时,分为复合物内部节点的参与度和复合物外部节点的参与度,充分考虑节点的邻域信息,则蛋白质节点<i>u</i>的复合物参与度的计算如式(5)所示:</p>
                </div>
                <div class="p1">
                    <p id="117" class="code-formula">
                        <mathml id="117"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ρ</mi><mi>C</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo>=</mo><mrow><mo>{</mo><mtable columnalign="left"><mtr><mtd><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>u</mi><mo>∈</mo><mi>C</mi><msub><mrow></mrow><mi>u</mi></msub></mrow></munder><mi>S</mi></mstyle><mi>W</mi><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>C</mi><msub><mrow></mrow><mi>u</mi></msub></mrow><mo>)</mo></mrow><mo>,</mo><mi>u</mi><mo>∈</mo><mi>V</mi><mrow><mo>(</mo><mrow><mo stretchy="false">|</mo><mi>C</mi><mo stretchy="false">|</mo></mrow><mo>)</mo></mrow></mtd></mtr><mtr><mtd><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>v</mi><mo>∈</mo><mi>V</mi><mo stretchy="false">(</mo><mo stretchy="false">|</mo><mi>C</mi><mo stretchy="false">|</mo><mo stretchy="false">)</mo></mrow></munder><mi>W</mi></mstyle><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mo>)</mo></mrow><mo>,</mo><mi>u</mi><mo>∉</mo><mi>V</mi><mrow><mo>(</mo><mrow><mo stretchy="false">|</mo><mi>C</mi><mo stretchy="false">|</mo></mrow><mo>)</mo></mrow></mtd></mtr></mtable></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>5</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="118">其中,<mathml id="190"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>V</mi><mrow><mo>(</mo><mrow><mo stretchy="false">|</mo><mi>C</mi><mo stretchy="false">|</mo></mrow><mo>)</mo></mrow></mrow></math></mathml>表示在蛋白质相互作用网络中,经模糊谱聚类算法挖掘出的构成复合物的所有节点集合;<i>C</i><sub><i>u</i></sub>表示基于模糊谱聚类算法挖掘出的包含节点<i>u</i>的所有复合物。复合物内部节点的参与度是指所有经模糊谱聚类算法挖掘出的复合物内部包含节点<i>u</i>的加权度之和;复合物外部节点的参与度是指该节点的邻域节点中属于模糊谱聚类算法挖掘出的复合物内部节点的加权度之和。</p>
                </div>
                <h4 class="anchor-tag" id="119" name="119"><b>2.4 子图稠密度</b></h4>
                <div class="p1">
                    <p id="120">复合物中包含的蛋白质比复合物中不包含的蛋白质更有可能是关键蛋白质,并且出现在多个复合物中的蛋白质比出现在单个复合物中的蛋白质更有可能是关键蛋白质,如果相互作用的2个蛋白质包含在同1个复合物中,那这2个蛋白质之间的相互作用就更加可靠<citation id="220" type="reference"><link href="41" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>。因此,基于2.1节构建的加权蛋白质网络,本文将每个蛋白质在二阶邻域内构造子图,设计PPI网络中的局部子图密度来评估蛋白质的关键性。而选择蛋白质的二阶邻域构造子图是因为大多数生物网络具有小世界特性,单个蛋白质的高阶邻域相关索引可能涉及到太多的节点,这对于检测节点的重要性是无效的<citation id="221" type="reference"><link href="53" rel="bibliography" /><sup>[<a class="sup">25</a>]</sup></citation>,故本文选择蛋白质节点的二阶邻域来构建子图,通过这种算法,可以在更大范围内测量拓扑信息。</p>
                </div>
                <div class="p1">
                    <p id="121">首先计算相互作用的2个蛋白质节点所共有的蛋白质复合物数目,如果相互作用的2个蛋白质包含在同1个复合物中,那么这2个蛋白质之间的相互作用就更加可靠,则基于复合物的2个蛋白质节点<i>u</i>和<i>v</i>相互作用可靠程度的计算如式(6)所示:</p>
                </div>
                <div class="p1">
                    <p id="122" class="code-formula">
                        <mathml id="122"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>S</mi><mi>R</mi><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mo>)</mo></mrow><mo>=</mo><mrow><mo>{</mo><mtable columnalign="left"><mtr><mtd><mfrac><mrow><mo stretchy="false">|</mo><mi>C</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mstyle displaystyle="true"><mo>∩</mo><mi>C</mi></mstyle><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo></mrow><mrow><mo stretchy="false">|</mo><mi>C</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mstyle displaystyle="true"><mo>∪</mo><mi>C</mi></mstyle><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo></mrow></mfrac><mo>,</mo></mtd></mtr><mtr><mtd><mtext> </mtext><mo stretchy="false">|</mo><mi>C</mi><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo><mo>&gt;</mo><mn>0</mn><mo>,</mo><mo stretchy="false">|</mo><mi>C</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo><mo>&gt;</mo><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn><mo>,</mo><mtext>其</mtext><mtext>他</mtext></mtd></mtr></mtable></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>6</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="123">其中,|<i>C</i>(<i>u</i>)∩<i>C</i>(<i>v</i>)|表示同时包含蛋白质节点<i>u</i>和<i>v</i>的复合物数目,|<i>C</i>(<i>u</i>)|和|<i>C</i>(<i>v</i>)|分别表示包含蛋白质节点<i>u</i>和<i>v</i>的复合物数目。</p>
                </div>
                <div class="p1">
                    <p id="124">基于复合物的相互作用的可靠程度度量,则每1个节点以及其二阶邻域节点构成的子图的稠密度计算如式(7)所示:</p>
                </div>
                <div class="p1">
                    <p id="125" class="code-formula">
                        <mathml id="125"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>S</mi><mi>D</mi><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><mn>2</mn><mo>×</mo><mstyle displaystyle="true"><mo>∑</mo><mi>S</mi></mstyle><mi>R</mi><mrow><mo>(</mo><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mo>)</mo></mrow></mrow><mrow><mo stretchy="false">|</mo><mi>Ν</mi><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo><mo>×</mo><mrow><mo>(</mo><mrow><mo stretchy="false">|</mo><mi>Ν</mi><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo><mo stretchy="false">|</mo><mo>-</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>7</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="126">其中,<i>N</i>(<i>v</i>)表示蛋白质节点<i>v</i>的二阶邻域节点集合,<mathml id="191"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mstyle displaystyle="true"><mo>∑</mo><mi>S</mi></mstyle><mi>R</mi><mrow><mo>(</mo><mrow><mi>v</mi><mo>,</mo><mi>u</mi></mrow><mo>)</mo></mrow></mrow></math></mathml>表示蛋白质节点<i>v</i>的所有二阶邻域节点包括<i>v</i>之间形成的边的可靠性度量之和。</p>
                </div>
                <h4 class="anchor-tag" id="127" name="127"><b>2.5 关键蛋白质识别</b></h4>
                <div class="p1">
                    <p id="128">基于2.1节构建的加权PPI网络以及2.2节模糊谱聚类算法的蛋白质复合物挖掘过程,最终的蛋白质关键性得分由2部分决定:一部分是2.3节得到的蛋白质的复合物参与度,另一部分是2.4节得到的子图密度。采用线性组合模型对蛋白质的复合物参与度和子图密度进行组合,则蛋白质<i>u</i>的关键性可以通过式(8)计算得到:</p>
                </div>
                <div class="p1">
                    <p id="129" class="code-formula">
                        <mathml id="129"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ρ</mi><mi>S</mi><mi>D</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo>=</mo><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><mi>θ</mi><mo stretchy="false">)</mo><mi>Ρ</mi><mi>C</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo>+</mo><mi>θ</mi><mi>S</mi><mi>D</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>8</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="130">其中,<i>θ</i>是一个参数,用于调整蛋白质的复合物参与度和子图密度在关键性度量中所占的比重。当<i>θ</i>=0时,只有复合物参与度被考虑到;当<i>θ</i>=1,只有子图密度被考虑。在实验和结果部分我们将详细讨论<i>θ</i>的取值。</p>
                </div>
                <div class="p1">
                    <p id="131">由于基于图论的谱聚类算法实现简单,不局限于原始数据的分布形状,可以收敛于全局最优解,故本文选择谱聚类结合FCM算法来挖掘蛋白质复合物。然后根据每个蛋白质的关键性值,将所有的蛋白质按照关键性值进行降序排列,关键性值越高的蛋白质,排名越靠前,越可能是关键蛋白质。将排序后的前<i>H</i>个蛋白质输出作为PEC算法识别出的关键蛋白质,<i>H</i>是加权PPI网络中包含关键蛋白质的个数。</p>
                </div>
                <h4 class="anchor-tag" id="132" name="132"><b>2.6 PEC算法描述</b></h4>
                <div class="p1">
                    <p id="133">PEC算法首先基于GO术语相似性和边聚集系数网络拓扑特性来构建加权蛋白质网络;然后利用谱聚类结合FCM算法挖掘出蛋白质复合物;接着基于蛋白质节点在复合物中的参与程度以及与二阶邻域节点形成的子图的密度来衡量一个蛋白质的关键性;最后根据关键性值进行降序排序,将排序后的前<i>H</i>个蛋白质输出作为PEC算法识别出的关键蛋白质。具体的算法实现如下所示:</p>
                </div>
                <div class="p1">
                    <p id="134"><b>输入</b>:PPI network;GO annotation;<i>θ</i>。</p>
                </div>
                <div class="p1">
                    <p id="135"><b>输出</b>:The top <i>K</i> ranked proteins by PSD value in descent order 。</p>
                </div>
                <div class="area_img" id="247">
                                <img alt="" src="Detail/GetImg?filename=images/JSJK201910005_24700.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="247">
                                <img alt="" src="Detail/GetImg?filename=images/JSJK201910005_24701.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <h3 id="149" name="149" class="anchor-tag"><b>3 实验结果与分析</b></h3>
                <h4 class="anchor-tag" id="150" name="150"><b>3.1 实验环境</b></h4>
                <div class="p1">
                    <p id="151">PEC算法实验仿真的编程环境为Python 3.5.2;操作系统为Windows 10家庭中文版;内存12 GB;处理器为Intel(R)Core(TM)i5-4200H CPU @ 2.8 GHz。</p>
                </div>
                <h4 class="anchor-tag" id="152" name="152"><b>3.2 实验数据集</b></h4>
                <div class="p1">
                    <p id="153">为验证本文提出算法的性能,选用蛋白质相互作用数据相对完整和可靠的酵母蛋白质相互作用网络数据作为实验数据。具体实验数据如下所示:</p>
                </div>
                <div class="p1">
                    <p id="154">(1) 酵母PPI网络数据来源于DIP数据库<citation id="222" type="reference"><link href="55" rel="bibliography" /><sup>[<a class="sup">26</a>]</sup></citation>,去除重复和自相互作用,该数据库包含5 093个蛋白质和24 743对相互作用。</p>
                </div>
                <div class="p1">
                    <p id="155">(2) Krogan数据<citation id="223" type="reference"><link href="57" rel="bibliography" /><sup>[<a class="sup">27</a>]</sup></citation>是用串联亲和纯化来处理4 562个不同标签酵母蛋白质,去除自相互作用和重复相互作用后,该网络中包含3 672个蛋白质和14 317条可靠的相互作用。</p>
                </div>
                <div class="p1">
                    <p id="156">(3) 酵母蛋白质GO注释信息<citation id="224" type="reference"><link href="59" rel="bibliography" /><sup>[<a class="sup">28</a>]</sup></citation>下载自基因本体数据库。</p>
                </div>
                <div class="p1">
                    <p id="157">(4) 标准关键蛋白质数据可整合MIPS(Munich Information center for Protein Sequences)<citation id="225" type="reference"><link href="61" rel="bibliography" /><sup>[<a class="sup">29</a>]</sup></citation>、SGD(Sacharomyces Genome Database)<citation id="226" type="reference"><link href="63" rel="bibliography" /><sup>[<a class="sup">30</a>]</sup></citation>、DEG(Diethylene Glycol)<citation id="227" type="reference"><link href="65" rel="bibliography" /><sup>[<a class="sup">31</a>]</sup></citation>和SGDP(Simons Genome Diversity Project)<citation id="228" type="reference"><link href="67" rel="bibliography" /><sup>[<a class="sup">32</a>]</sup></citation>,总共有1 285个关键蛋白质,其中出现在酵母DIP网络中有1 167个,Krogan网络中有929个。</p>
                </div>
                <h4 class="anchor-tag" id="158" name="158"><b>3.3 参数影响分析</b></h4>
                <div class="p1">
                    <p id="159">在PEC算法中,<i>θ</i>参数是用来调整蛋白质的复合物参与度和子图密度在关键性度量中所占的比重。为了评估参数<i>θ</i>对实验结果的影响,本文设置<i>θ</i>的取值为0～1,步长为0.1,在DIP和Krogan 2个数据集上运行20次,取实验的平均值进行比较分析,研究参数<i>θ</i>的不同取值对PEC识别关键蛋白质准确率的影响。基于本文提出的蛋白质关键性,分别取排名前1%,5%,10%,15%,20%和25%的蛋白质作为候选关键蛋白质集,分别将候选出来的关键蛋白质与实验的标准关键蛋白质进行对比,分析候选关键蛋白质集中预测正确的关键蛋白质数目,在DIP和Krogan 2个数据集上具体的实验结果如表1所示。</p>
                </div>
                <div class="area_img" id="160">
                    <p class="img_tit"><b>表1 在不同</b><i><b>θ</b></i><b>取值下的 PEC算法识别正确的关键蛋白质数目</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 1 Number of true essential proteins correctly identified by PEC with different</b><i>θ</i></p>
                    <p class="img_note"></p>
                    <table id="160" border="1"><tr><td><br />dataset</td><td><i>θ</i></td><td>Top <br />1%</td><td>Top <br />5%</td><td>Top <br />10%</td><td>Top <br />15%</td><td>Top <br />20%</td><td>Top <br />25%</td></tr><tr><td><br /></td><td>0</td><td>16</td><td>73</td><td>127</td><td>189</td><td>284</td><td>296</td></tr><tr><td><br /></td><td>0.1</td><td>46</td><td>197</td><td>324</td><td>428</td><td>400</td><td>553</td></tr><tr><td><br /></td><td>0.2</td><td>48</td><td>198</td><td>326</td><td>429</td><td>531</td><td>567</td></tr><tr><td><br /></td><td>0.3</td><td>48</td><td>198</td><td>327</td><td>430</td><td>534</td><td>571</td></tr><tr><td><br /></td><td>0.4</td><td>48</td><td>198</td><td>328</td><td>431</td><td>535</td><td>577</td></tr><tr><td><br />DIP</td><td>0.5</td><td>48</td><td>198</td><td>329</td><td>433</td><td>536</td><td>609</td></tr><tr><td><br /></td><td>0.6</td><td>48</td><td>197</td><td>329</td><td>433</td><td>534</td><td>609</td></tr><tr><td><br /></td><td>0.7</td><td>48</td><td>196</td><td>329</td><td>433</td><td>532</td><td>609</td></tr><tr><td><br /></td><td>0.8</td><td>48</td><td>196</td><td>329</td><td>433</td><td>532</td><td>609</td></tr><tr><td><br /></td><td>0.9</td><td>48</td><td>196</td><td>329</td><td>433</td><td>530</td><td>575</td></tr><tr><td><br /></td><td>1</td><td>48</td><td>196</td><td>329</td><td>433</td><td>530</td><td>575</td></tr><tr><td><br /></td><td>0</td><td>9</td><td>57</td><td>98</td><td>143</td><td>174</td><td>214</td></tr><tr><td><br /></td><td>0.1</td><td>25</td><td>94</td><td>179</td><td>256</td><td>309</td><td>356</td></tr><tr><td><br /></td><td>0.2</td><td>25</td><td>116</td><td>192</td><td>254</td><td>311</td><td>357</td></tr><tr><td><br /></td><td>0.3</td><td>26</td><td>117</td><td>192</td><td>253</td><td>312</td><td>358</td></tr><tr><td><br /></td><td>0.4</td><td>26</td><td>117</td><td>192</td><td>252</td><td>313</td><td>359</td></tr><tr><td><br />Krogan</td><td>0.5</td><td>26</td><td>117</td><td>192</td><td>258</td><td>314</td><td>360</td></tr><tr><td><br /></td><td>0.6</td><td>24</td><td>117</td><td>191</td><td>257</td><td>310</td><td>355</td></tr><tr><td><br /></td><td>0.7</td><td>24</td><td>117</td><td>191</td><td>257</td><td>308</td><td>354</td></tr><tr><td><br /></td><td>0.8</td><td>24</td><td>117</td><td>191</td><td>256</td><td>308</td><td>354</td></tr><tr><td><br /></td><td>0.9</td><td>24</td><td>117</td><td>191</td><td>256</td><td>309</td><td>353</td></tr><tr><td><br /></td><td>1</td><td>24</td><td>117</td><td>191</td><td>256</td><td>309</td><td>352</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="161">从表1可知,当<i>θ</i>=0时,只考虑了复合物的参与度得分;当<i>θ</i>=1时,只考虑子图密度。并且当<i>θ</i>=0或者<i>θ</i>=1时,PEC预测的关键蛋白质中被正确识别的数量明显比<i>θ</i>值介于0～1的预测数量少,这说明结合蛋白质复合物的参与度以及邻居子图密度节点信息会比只考虑其中任何一种属性更能准确地识别关键蛋白质。实验结果表明,当<i>θ</i>=0.5时,PEC算法正确识别出的关键蛋白质数目较多,算法的识别准确率较高。</p>
                </div>
                <h4 class="anchor-tag" id="162" name="162"><b>3.4 性能比较分析</b></h4>
                <h4 class="anchor-tag" id="163" name="163">3.4.1 PPI网络对检测结果的影响</h4>
                <div class="p1">
                    <p id="164">为了验证PEC算法构建的加权蛋白质网络的性能,由于DC<citation id="229" type="reference"><link href="15" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>、LAC<citation id="230" type="reference"><link href="27" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>和NC<citation id="231" type="reference"><link href="29" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>算法是从节点的邻域节点数、邻域节点的重要性和边的重要性出发来衡量蛋白质节点的重要性,故本文分别将3个算法基于未使用和使用本文的加权蛋白质网络,在DIP和Krogan 2个数据集上运行20次,取实验的平均值进行比较分析。基于各个算法蛋白质的关键性排序,分别取前1%,5%,10%,15%,20%和25%的蛋白质作为候选关键蛋白质集。在不同排序比例之下,将候选关键蛋白质集与实验标准的关键蛋白质进行对比,分析候选关键蛋白质集中的预测正确的关键蛋白质数目,进而比较分析使用和未使用本文构建的加权蛋白质网络对于正确识别出的关键蛋白质数目的影响,具体的实验结果分别如图1和图2所示。</p>
                </div>
                <div class="p1">
                    <p id="165">从图1和图2可以看出,基于本文的加权网络,3种算法可以正确识别出更多关键蛋白质。在选取排序前25%的候选蛋白质时,在DIP数据集上,DC算法在本文的加权网络上识别关键蛋白质的准确率提高了0.40%,NC算法准确率提高0.37%;在Krogan数据集上,DC算法在本文的加权网络上识别关键蛋白质的准确率提高了0.94%,LAC算法的准确率提高了0.31%,NC算法的准确率提高了0.31%。实验结果表明,本文构建的加权蛋白质网络能够提高关键蛋白质预测的准确率。</p>
                </div>
                <h4 class="anchor-tag" id="166" name="166">3.4.2 不同比例的关键蛋白质预测准确率的比较</h4>
                <div class="p1">
                    <p id="167">为了评估PEC算法的性能,将PEC算法与DC<citation id="232" type="reference"><link href="15" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>、BC<citation id="233" type="reference"><link href="17" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>、CC<citation id="234" type="reference"><link href="19" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>、SC<citation id="235" type="reference"><link href="21" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>、IC<citation id="236" type="reference"><link href="25" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>、PeC<citation id="237" type="reference"><link href="33" rel="bibliography" /><link href="71" rel="bibliography" /><sup>[<a class="sup">15</a>,<a class="sup">15</a>]</sup></citation>、WDC<citation id="238" type="reference"><link href="35" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation>、LIDC<citation id="239" type="reference"><link href="37" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>、LBCC<citation id="240" type="reference"><link href="39" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>和UC<citation id="241" type="reference"><link href="41" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>算法在DIP和Krogan 2个数据集上运行20次,取实验的平均值进行比较分析。蛋白质分别根据各自算法计算排名,分别取排名前1%,5%,10%,15%,20%和25%的蛋白质作为候选关键蛋白质集。基于每一比例的候选集,将关键蛋白质与实验标准的关键蛋白质进行对比,分析候选关键蛋白质集中预测正确的关键蛋白质数目来衡量每种算法的性能。在DIP和Krogan 2个数据集上的实验结果分别如图3和图4所示。</p>
                </div>
                <div class="area_img" id="168">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJK201910005_168.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 DIP数据集上3种算法在不同网络中正确识别的关键蛋白质数目" src="Detail/GetImg?filename=images/JSJK201910005_168.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 DIP数据集上3种算法在不同网络中正确识别的关键蛋白质数目  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJK201910005_168.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Figure 1 Number of true essential proteins predicted in different networks by the three algorithms on DIP dataset</p>

                </div>
                <div class="area_img" id="169">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJK201910005_169.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 Krogan数据集上3种算法在不同网络中正确识别的关键蛋白质数目" src="Detail/GetImg?filename=images/JSJK201910005_169.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 Krogan数据集上3种算法在不同网络中正确识别的关键蛋白质数目  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJK201910005_169.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Figure 2 Number of true essential proteins predicted in different networks by the three algorithms on Krogan dataset</p>

                </div>
                <div class="area_img" id="170">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJK201910005_170.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 DIP数据集上PEC和其他算法正确识别的关键蛋白质数目" src="Detail/GetImg?filename=images/JSJK201910005_170.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 DIP数据集上PEC和其他算法正确识别的关键蛋白质数目  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJK201910005_170.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Figure 3 Number of true essential proteins predicted by PEC and other algorithms on DIP dataset</p>

                </div>
                <div class="area_img" id="171">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJK201910005_171.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 Krogan数据集上PEC和其他算法正确识别的关键蛋白质数目" src="Detail/GetImg?filename=images/JSJK201910005_171.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 Krogan数据集上PEC和其他算法正确识别的关键蛋白质数目  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJK201910005_171.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Figure 4 Number of true essential proteins predicted by PEC and other algorithms on Krogan dataset</p>

                </div>
                <div class="p1">
                    <p id="172">从图3的DIP数据集上的实验结果可以看出,PEC算法在前1%～25%的排序候选蛋白质中,正确识别出最多的关键蛋白质数目。当选取51个蛋白质作为候选关键蛋白质(Top 1%)时,PEC算法正确预测48个关键蛋白质,准确率达到了94.12%。而DC、BC、CC、SC、IC、PeC、WDC、LIDC、LBCC和UC算法分别正确预测出22,24,24,24,24,39,36,37,37和35种关键蛋白质,准确率分别为43.14%,47.06%,47.06%,47.06%,47.06%,76.47%,70.59%,72.55%,72.55%和68.63%。在所有算法中,BC算法的预测准确率相对较低,在前1%,5%,10%,15%,20%和25%情况下,PEC算法的准确率比BC算法分别提高了100%,108.42%,80.77%,59.78%,48.48%和40.65%。从图4的Krogan数据集上的实验结果可以看出,PEC算法在前1%～25%的排序候选蛋白质中,正确识别出最多的关键蛋白质数目。以Top 25%为例,PEC算法正确预测360个关键蛋白质,准确率达到了53.81%。而DC、BC、CC、SC、IC、PeC、WDC、LIDC、LBCC和UC算法分别正确预测出318,319,324,272,314,317,332,339,343和334种关键蛋白质,准确率分别为47.60%,47.75%,48.50%,40.72%,47.01%,47.46%,49.70%,50.75%,51.35%和50.00%。实验结果表明,本文提出的算法能够识别出较多关键蛋白质。</p>
                </div>
                <h4 class="anchor-tag" id="173" name="173">3.4.3 统计指标对比分析</h4>
                <div class="p1">
                    <p id="174">为进一步验证PEC算法预测关键蛋白质的性能,本文使用文献<citation id="242" type="reference">[<a class="sup">33</a>,<a class="sup">33</a>]</citation>中的敏感度<i>SN</i>(Sensitivity)、特异性<i>SP</i>(Specificity)、阳性预测值<i>PPV</i>(Positive Predictive Value)、阴性预测值<i>NPV</i>(Negative Predictive Value)、<i>F</i>-测度(F-measure)和正确率<i>ACC</i>(Accuracy)这6个统计指标对各个算法的性能进行对比分析。这些统计指标的定义如下所示:</p>
                </div>
                <div class="p1">
                    <p id="175" class="code-formula">
                        <mathml id="175"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>S</mi><mi>Ν</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac></mtd></mtr><mtr><mtd><mi>S</mi><mi>Ρ</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ν</mi></mrow><mrow><mi>Τ</mi><mi>Ν</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi></mrow></mfrac></mtd></mtr><mtr><mtd><mi>Ρ</mi><mi>Ρ</mi><mi>V</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi></mrow></mfrac></mtd></mtr><mtr><mtd><mi>Ν</mi><mi>Ρ</mi><mi>V</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ν</mi></mrow><mrow><mi>Τ</mi><mi>Ν</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac></mtd></mtr><mtr><mtd><mi>F</mi><mo>=</mo><mfrac><mrow><mn>2</mn><mo>×</mo><mi>S</mi><mi>Ν</mi><mo>×</mo><mi>Ρ</mi><mi>Ρ</mi><mi>V</mi></mrow><mrow><mi>S</mi><mi>Ν</mi><mo>+</mo><mi>Ρ</mi><mi>Ρ</mi><mi>V</mi></mrow></mfrac></mtd></mtr><mtr><mtd><mi>A</mi><mi>C</mi><mi>C</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>Τ</mi><mi>Ν</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>Τ</mi><mi>Ν</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="176">其中,<i>TP</i>表示关键蛋白质被正确预测的数目,<i>FN</i>表示关键蛋白质被错误地预测为非关键蛋白质的数目,<i>TN</i>表示非关键蛋白质被正确预测的数目,<i>FP</i>为非关键蛋白质被错误地预测为关键蛋白质的数目。根据已知关键蛋白质的数量,在DIP数据集中,本文选取排序后的前1 167个蛋白质作为候选关键蛋白质,具体6项实验统计指标值如表2所示。</p>
                </div>
                <div class="p1">
                    <p id="177">从表2可以看出,在DIP数据集上,PEC算法的各项指标均高于其他10种算法的。在DIP数据集上,PEC算法与10种算法中相对最好LBCC算法相比,各项指标分别提高了0.56%,0.36%,0.56%,0.36%,0.56%和0.51%。实验结果表明,PEC算法在识别关键蛋白质上具有更好的效果。</p>
                </div>
                <h4 class="anchor-tag" id="178" name="178">3.4.4 识别关键蛋白质的差异性比较分析</h4>
                <div class="p1">
                    <p id="179">为了进一步分析PEC算法识别关键蛋白质的性能,在识别出相同的关键蛋白质,本文比较了PEC算法和其他10种算法预测正确的关键蛋白的百分比。图5显示了PEC算法和其他10种算法在识别相同关键蛋白质中预测正确的关键蛋白质的百分比。从图5中可以看出,PEC算法对不同蛋白质预测正确的关键蛋白质的百分比始终高于其他10种算法的。本文选取CC算法和LIDC算法来进行分析,与CC算法相比,在前175位的蛋白质中,PEC算法预测正确的关键蛋白质比例是83.6%,而CC算法检测到的蛋白中有64.9%不是关键蛋白质。与LIDC算法相比,在前55位蛋白质中,PEC算法预测正确的关键蛋白质比例是82.78%,而LIDC算法检测到的蛋白中有54.7%是识别正确的关键蛋白质。实验结果表明,本文提出的PEC算法识别关键蛋白质的性能较好。</p>
                </div>
                <div class="area_img" id="180">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJK201910005_180.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图5 PEC算法与其他10种算法在不同蛋白质中
 正确识别的关键蛋白质所占比例的比较" src="Detail/GetImg?filename=images/JSJK201910005_180.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图5 PEC算法与其他10种算法在不同蛋白质中
 正确识别的关键蛋白质所占比例的比较  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJK201910005_180.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Figure 5 Comparison of the percentage of essential 
 proteins correctly identified by PEC algorithm 
 and the other ten algorithms in different proteins</p>

                </div>
                <h3 id="181" name="181" class="anchor-tag"><b>4 结束语</b></h3>
                <div class="p1">
                    <p id="182">本文基于加权蛋白质相互作用网络,提出一种基于复合物参与度和密度的关键蛋白质预测算法PEC。该算法首先融合GO注释信息和边聚集系数构造加权PPI网络;设计模糊谱聚类算法实现复合物的挖掘,降低数据的维数;其次基于蛋白质节点的复合物参与度以及节点邻域子图密度,设计出关键节点的关键性得分。为了评估算法的性能,本文将PEC算法与DC、BC、CC、SC、IC、PeC、WDC、LIDC、LBCC和UC算法在DIP和Krogan 2个数据集上进行了对比,实验结果表明,PEC算法具有更高的准确率、特异性和敏感度等。对关键蛋白质挖掘今后的研究可以从以下几个方面入手:(1) 深入研究PPI网络拓扑结构,综合考虑PPI数据构建动态蛋白质网络,以降低数据噪声的影响。(2) 结合多元生物数据的算法以提升挖掘结果。</p>
                </div>
                <div class="area_img" id="183">
                    <p class="img_tit"><b>表2 PEC与其他10种算法在DIP数据集上的统计指标比较</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 2 Comparison of statistical indicators of PEC and the other ten algorithms on DIP dataset</b></p>
                    <p class="img_note"></p>
                    <table id="183" border="1"><tr><td><br />algorithm</td><td><i>SN</i></td><td><i>SP</i></td><td><i>PPV</i></td><td><i>NPV</i></td><td><i>F</i></td><td><i>ACC</i></td></tr><tr><td><br />DC</td><td>0.400 2</td><td>0.821 7</td><td>0.400 2</td><td>0.821 7</td><td>0.400 2</td><td>0.725 114 150</td></tr><tr><td><br />BC</td><td>0.350 5</td><td>0.806 9</td><td>0.350 5</td><td>0.806 9</td><td>0.350 5</td><td>0.702 306 076</td></tr><tr><td><br />CC</td><td>0.354 8</td><td>0.808 2</td><td>0.354 8</td><td>0.808 2</td><td>0.354 8</td><td>0.704 302 605</td></tr><tr><td><br />SC</td><td>0.367 6</td><td>0.812 0</td><td>0.367 6</td><td>0.812 0</td><td>0.367 6</td><td>0.710 162 847</td></tr><tr><td><br />IC</td><td>0.350 6</td><td>0.806 8</td><td>0.350 6</td><td>0.806 8</td><td>0.350 6</td><td>0.702 197 769</td></tr><tr><td><br />PeC</td><td>0.436 2</td><td>0.832 4</td><td>0.436 2</td><td>0.832 4</td><td>0.436 2</td><td>0.741 610 938</td></tr><tr><td><br />WDC</td><td>0.461 9</td><td>0.840 0</td><td>0.461 9</td><td>0.840 0</td><td>0.461 9</td><td>0.753 341 928</td></tr><tr><td><br />LIDC</td><td>0.465 8</td><td>0.841 1</td><td>0.465 8</td><td>0.841 1</td><td>0.465 8</td><td>0.755 058 779</td></tr><tr><td><br />LBCC</td><td>0.466 1</td><td>0.841 2</td><td>0.466 1</td><td>0.841 2</td><td>0.466 1</td><td>0.755 209 124</td></tr><tr><td><br />UC</td><td>0.462 3</td><td>0.840 3</td><td>0.462 3</td><td>0.840 3</td><td>0.462 3</td><td>0.753 740 493</td></tr><tr><td><br />PEC</td><td>0.468 7</td><td>0.844 2</td><td>0.468 7</td><td>0.844 2</td><td>0.468 7</td><td>0.759 055 334</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="5">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A new method for predicting essential proteins based on participation degree in protein complex and subgraph density">

                                <b>[1]</b> Lei X J,Yang X Q.A new method for predicting essential proteins based on participation degree in protein complex and subgraph density [J].PLoS ONE,2018,13(6):0198998-0199016.
                            </a>
                        </p>
                        <p id="7">
                            <a id="bibliography_2" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES14032300121250&amp;v=MTU4MDBvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMZklKMXNTYXhBPU5pZk9mYks4SHRMT3JJOUZaZWtPRG5rNQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[2]</b> Li M,Zheng R Q,Zhang H H,et al.Effective identification of essential proteins based on priori knowledge,network topology and gene expressions[J].Methods,2014,67:325-333.
                            </a>
                        </p>
                        <p id="9">
                            <a id="bibliography_3" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES668C12C11EEF2383BDC4AD36DF5EC5F0&amp;v=MjkwOTZ4eFZobmt4NU9RdmhxbVpEZk1mblFNeWZDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOQmh3NzI1d2FvPU5pZk9mYlcrRnFMTnJmeEVaWjU2ZW40Ng==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[3]</b> Zhao B H,Wang J X,Li X Y,et al.Essential protein discovery based on a combination of modularity and conservatism[J].Methods,2016,110:54-63.
                            </a>
                        </p>
                        <p id="11">
                            <a id="bibliography_4" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM138E56080BE6A893ABFD005D8209D1AA&amp;v=MjkxMTdNdnVDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOQmh3NzI1d2FvPU5pZklZN0s3RnFUSnFZOU5aSmw2Q2cweHhoVmltRWtKU0gvbjJCbzNlYnZnUg==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[4]</b> Li M,Lu Y,Wang J X,et al.A topology potential-based method for identifying essential proteins from PPI networks [J].IEEE/ACM Transactions on Computational Biology and Bioinformatics,2015,12(2):372-383.
                            </a>
                        </p>
                        <p id="13">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Lethality and centrality in protein networks">

                                <b>[5]</b> Jeong H,Mason S P,Barabasi A L,et al.Lethality and centrality in protein networks [J].Nature,2001,411 (6833):41-42.
                            </a>
                        </p>
                        <p id="15">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Identifying hubs in protein interaction networks">

                                <b>[6]</b> Vallabhajosyula R R,Chakravarti D,Lutfeali S,et al.Identifying hubs in protein interaction networks [J].PLoS ONE,2009,4(4):5344-5353.
                            </a>
                        </p>
                        <p id="17">
                            <a id="bibliography_7" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13012100484943&amp;v=MjY0OTFHZXJxUVRNbndaZVp1SHlqbVVMZklKMXNTYXhBPU5pZk9mYks3SHRET3JvOUZZT01MQlhnNm9CTVQ2VDRQUUgvaXJSZA==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[7]</b> Newman M E J.A measure of betweenness centrality based on random walks [J].Social Networks,2005,27(1):39-54.
                            </a>
                        </p>
                        <p id="19">
                            <a id="bibliography_8" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011600976673&amp;v=MDE4NTVKQ25zNm9CTVQ2VDRQUUgvaXJSZEdlcnFRVE1ud1plWnVIeWptVUxmSUoxc1NheEE9TmlmT2ZiSzdIdEROcVk5RmJldw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[8]</b> Wuchty S,Stadler P F.Centers of complex networks [J].Journal of Theoretical Biology,2003,223(1):45-53.
                            </a>
                        </p>
                        <p id="21">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Subgraph centrality in complex networks">

                                <b>[9]</b> Estrada E,Rodriguez-Velazquez J A.Subgraph centrality in complex networks [J].Physical Review E,Statistical,Nonlinear,&amp; Soft Matter Physics,2005,71(2):056103-056111.
                            </a>
                        </p>
                        <p id="23">
                            <a id="bibliography_10" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJST&amp;filename=SJST14112600395588&amp;v=MTU0MDVSZEdlcnFRVE1ud1plWnVIeWptVUxmSUoxc1NheEE9TmlmWWVySzhIOURPcVk5RlorSUtDWFF4b0JNVDZUNFBRSC9pcg==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[10]</b> Bonacich P.Power and centrality:A family of measures [J].American Journal of Sociology,1987,92(5):1170-1182.
                            </a>
                        </p>
                        <p id="25">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Rethinking centrality: Methods and examples">

                                <b>[11]</b> Stephenson K,Zelen M.Rethinking centrality:Methods and examples [J].Social Networks,1989,11(1):1-37.
                            </a>
                        </p>
                        <p id="27">
                            <a id="bibliography_12" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011300467914&amp;v=MjMzMjliSzdIdEROckk5RllPMElCWDA5b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadUh5am1VTGZJSjFzU2F4QT1OaWZPZg==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[12]</b> Li M,Wang J X,Chen X,et al.A local average connectivity-based method for identifying essential proteins from the network level [J].Computational Biology &amp; Chemistry,2011,35(3):143-150.
                            </a>
                        </p>
                        <p id="29">
                            <a id="bibliography_13" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM50DD78CECE49B7D30F4E0BCE52366242&amp;v=MDU1NjhDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOQmh3NzI1d2FvPU5pZklZN2E0YXFYTHAvd3dGNTRMQlE0K3V4VVRuRHNJU0EyUjJSYzNlclNTUjc2ZA==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[13]</b> Wang J X,Li M,Wang H,et al.Identification of essential proteins based on edge clustering coefficient [J].IEEE/ACM Transactions on Computational Biology &amp; Bioinformatics,2012,9(4):1070-1080.
                            </a>
                        </p>
                        <p id="31">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prediction of essential proteins based on overlapping essential modules">

                                <b>[14]</b> Zhao B H,Wang J X,Li M,et al.Prediction of essential proteins based on overlapping essential modules[J].IEEE Transactions on Nano Bioscience,2014,13(4):415-438.
                            </a>
                        </p>
                        <p id="33">
                            <a id="bibliography_15" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZNGD201303027&amp;v=MjgwMDJDVVJMT2VaZVJtRnkvZ1ZiN0tQeVBNYXJHNEg5TE1ySTlIWTRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnI=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[15]</b> Li Min,Zhang Han-hui,Fei Yao-ping.Essential protein discovery method based on integration of PPI and gene expression data [J].Journal of Central South University(Science and Technology),2013,44(3):1024-1029.(in Chinese)
                            </a>
                        </p>
                        <p id="35">
                            <a id="bibliography_16" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM605B69FA435C552FC3A1ACF53A55DF11&amp;v=MDk3NzMzazh6V0JnNlU1OE9ReVVxUkZFZkxmZ003dWVDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOQmh3NzI1d2FvPU5pZklZN1c0RzZQS3B2azBZT2dLZg==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[16]</b> Tang X W,Wang J X,Zhong J C,et al.Predicting essential proteins based on weighted degree centrality[J].IEEE/ACM Transactions on Computational Biology &amp; Bioinformatics,2014,11(2):407-418.
                            </a>
                        </p>
                        <p id="37">
                            <a id="bibliography_17" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Identification of Essential Proteins Based on a New Combination of Local Interaction Density and Protein Complexes">

                                <b>[17]</b> Luo J W,Qi Y.Identification of essential proteins based on a new combination of local interaction density and protein complexes [J].PLoS ONE,2015,10(6):0131418-0131444.
                            </a>
                        </p>
                        <p id="39">
                            <a id="bibliography_18" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A new method for identifying essential proteins based on network topology properties and protein complexes">

                                <b>[18]</b> Qin C,Sun Y Q,Dong Y D.A new method for identifying essential proteins based on network topology properties and protein complexes [J].PLoS ONE,2016,11(8):0161042-0161071.
                            </a>
                        </p>
                        <p id="41">
                            <a id="bibliography_19" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCMEBA518DA150F1129D89F1968FD2633A9&amp;v=MzEyNzJ3NzI1d2FvPU5pZklZOGJLYjlUTnAvczBaZTRQZW4wNHpSOW40allMU1hia3BHUkJlN1NYUnN1V0NPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU5CaA==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[19]</b> Li M,Lu Y,Niu Z B,et al.United complex centrality for identification of essential proteins from PPI networks [J].IEEE/ACM Transactions on Computational Biology &amp; Bioinformatics,2017,14(2):370-380.
                            </a>
                        </p>
                        <p id="43">
                            <a id="bibliography_20" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES9F6BD82D006F6CA7604219A2AF08ABAD&amp;v=MDI3NjVick9HS080cDQweFpPc0plbnBLdmhFVjZqdC9TWGFUcm1ORGVicmxOOHZyQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TkJodzcyNXdhbz1OaWZPZg==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[20]</b> Jiang Y X,Wang Y,Pang W,et al.Essential protein identification based on essential protein-protein interaction prediction by integrated edge weights[J].Methods,2015,83(13):51-62.
                            </a>
                        </p>
                        <p id="45">
                            <a id="bibliography_21" >
                                    <b>[21]</b>
                                 Hong Hai-yan,Liu Wei.Efficient prediction method of essential proteins based on PPI network [J].Computer Science,2016,43(11):16-20.(in Chinese)
                            </a>
                        </p>
                        <p id="47">
                            <a id="bibliography_22" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=QHDY201505007&amp;v=MjM1NjZPZVplUm1GeS9nVmI3S05DWFBkN0c0SDlUTXFvOUZZNFFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkw=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[22]</b> Zhong J C,Wang J X,Peng W,et al.A feature selection method for prediction essential protein [J].Tsinghua Science and Technology,2015,20(5):491-499.
                            </a>
                        </p>
                        <p id="49">
                            <a id="bibliography_23" >
                                    <b>[23]</b>
                                 Fan Zi-jing,Luo Ze,Ma Yong-zheng.A spectral clustering algorithm based on fuzzy kernel clustering [J].Computer Engineering,2017,43(11):161-165.(in Chinese)
                            </a>
                        </p>
                        <p id="51">
                            <a id="bibliography_24" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=XXWX201706026&amp;v=MDkxOThRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJtRnkvZ1ZiN0tQVFhjZHJHNEg5Yk1xWTlIWW8=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[24]</b> Zhao Xue-wu,Cheng Xin-dang,Lü Jia-wei,et al.Identify protein complexes by integrating temporal function continue feature and ant colony clustering on dynamic PPI networks[J].Journal of Chinese Computer Systems,2017,38(6):1311-1316.(in Chinese)
                            </a>
                        </p>
                        <p id="53">
                            <a id="bibliography_25" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JFXG201607007&amp;v=MDI1MjF5dlRhYkc0SDlmTXFJOUZZNFFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUm1GeS9nVmI3S0w=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[25]</b> Shang X,Wang Y,Chen B.Identifying essential proteins based on dynamic protein-protein interaction networks and RNA-Seq datasets [J].Science China Information Sciences,2016,59(7):1-11.
                            </a>
                        </p>
                        <p id="55">
                            <a id="bibliography_26" target="_blank" href="http://scholar.cnki.net/result.aspx?q=DIP, the Database of Interacting Proteins: a research tool for studying cellular networks of protein interactions">

                                <b>[26]</b> Xenarios I,Salwinski L,Duan X J,et al.DIP,the database of interacting proteins:A research tool for studying cellular networks of protein interactions [J].Nucleic Acids Research,2002,30(1):303-305.
                            </a>
                        </p>
                        <p id="57">
                            <a id="bibliography_27" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Global landscape of protein complexes in the yeast Saccharomyces cerevisiae">

                                <b>[27]</b> Krogan N J,Cagney G,Yu H Y,et al.Global landscape of protein complexes in the yeast saccharomyces cerevisiae [J].Nature,2006,440(7084):637-643.
                            </a>
                        </p>
                        <p id="59">
                            <a id="bibliography_28" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Gene Ontology Consortium:going forward">

                                <b>[28]</b> Blake J A,Christie K R,Done J,et al.Gene ontology consortium:Going forward [J].Nucleic Acids Research,2015,43(Database issue):1049-1056.
                            </a>
                        </p>
                        <p id="61">
                            <a id="bibliography_29" target="_blank" href="http://scholar.cnki.net/result.aspx?q=MIPS: analysis and annotation of proteins from whole genomes in 2005">

                                <b>[29]</b> Mewes H W,Frishman D,Mayer K F X,et al.MIPS:Analysis and annotation of proteins from whole genomes [J].Nucleic Acids Research,2006,34(Database issue):169-172.
                            </a>
                        </p>
                        <p id="63">
                            <a id="bibliography_30" target="_blank" href="http://scholar.cnki.net/result.aspx?q=SGD: Saccharomyces Genome Database">

                                <b>[30]</b> Cherry J M,Adler C,Ball C,et al.SGD:Saccharomyces genome database [J].Nucleic Acids Research,1998,26 (1):73-79.
                            </a>
                        </p>
                        <p id="65">
                            <a id="bibliography_31" target="_blank" href="http://scholar.cnki.net/result.aspx?q=DEG 5.0, a database of essential genes in both prokaryotes and eukaryotes">

                                <b>[31]</b> Zhang R,Lin Y.DEG 5.0,a database of essential genes in both prokaryotes and eukaryotes [J].Nucleic Acids Research,2009,37 (Database issue):455-458.
                            </a>
                        </p>
                        <p id="67">
                            <a id="bibliography_32" target="_blank" href="http://scholar.cnki.net/result.aspx?q=GeStoDifferent: a Cytoscape plugin for the generation and the identification of gene regulatory networks describing a stochastic cell differentiation process">

                                <b>[32]</b> Antoniotti M,Bader G D,Caravagna G,et al.Gestodifferent:A cytoscape plugin for the generation and the identification of gene regulatory networks describing a stochastic cell differentiation process [J].Bioinformatics,2013,29(4):513-514.
                            </a>
                        </p>
                        <p id="69">
                            <a id="bibliography_33" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJA201710007&amp;v=MDM2MjBESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSbUZ5L2dWYjdLTHo3QmI3RzRIOWJOcjQ5Rlk0UUs=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[33]</b> Hong Hai-yan,Liu Wei.Research on essential protein identification method based on improved PSP algorithm [J].Computer Science,2017,44(10):38-44.(in Chinese)
                            </a>
                        </p>
                        <p id="71">
                            <a id="bibliography_15" >
                                    <b>[15]</b>
                                 李敏,张含会,费耀平.融合PPI和基因表达数据的关键蛋白质识别方法[J].中南大学学报(自然科学版),2013,44(3):1024-1029.
                            </a>
                        </p>
                        <p id="73">
                            <a id="bibliography_21" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJA2016S2004&amp;v=MjEyNjM0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSbUZ5L2dWYjdLTHo3QmI3RzRIOWV2clk5RllJUUtESDg=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[21]</b> 洪海燕,刘维.基于PPI网络的关键蛋白质的高效预测算法[J].计算机科学,2016,43(11):16-20.
                            </a>
                        </p>
                        <p id="75">
                            <a id="bibliography_23" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201711027&amp;v=Mjk0NjZHNEg5Yk5ybzlIWTRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJtRnkvZ1ZiN0tMejdCYmI=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[23]</b> 范子静,罗泽,马永征.一种基于模糊核聚类的谱聚类算法[J].计算机工程,2017,43(11):161-165.
                            </a>
                        </p>
                        <p id="77">
                            <a id="bibliography_24" >
                                    <b>[24]</b>
                                 赵学武,程新党,吕嘉伟,等.融合时序保持特征和蚁群聚类的动态PPI网络复合物识别[J].小型微型计算机系统,2017,38(6):1311-1316.
                            </a>
                        </p>
                        <p id="79">
                            <a id="bibliography_33" >
                                    <b>[33]</b>
                                 洪海燕,刘维.基于改进PSO算法的关键蛋白质识别方法研究[J].计算机科学,2017,44(10):38-44.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JSJK201910005" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJK201910005&amp;v=MjcyMDFIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJtRnkvZ1ZiN0tMejdCWmJHNEg5ak5yNDlGWVlRS0Q=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
