<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637132356212373750%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJSJK201909004%26RESULT%3d1%26SIGN%3dd7cGM9Rhdpequv9UkAP3yaEzt9k%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJK201909004&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJK201909004&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJK201909004&amp;v=MTgzNzdJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSbUZ5N2tVcnpJTHo3QlpiRzRIOWpNcG85Rlk=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#35" data-title="&lt;b&gt;1 引言&lt;/b&gt; "><b>1 引言</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#39" data-title="&lt;b&gt;2 相关研究&lt;/b&gt; "><b>2 相关研究</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#41" data-title="&lt;b&gt;2.1  种子-扩展法&lt;/b&gt;"><b>2.1  种子-扩展法</b></a></li>
                                                <li><a href="#48" data-title="&lt;b&gt;2.2 现有三代测序序列比对方法&lt;/b&gt;"><b>2.2 现有三代测序序列比对方法</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#55" data-title="&lt;b&gt;3 基于低频种子的序列比对方法设计&lt;/b&gt; "><b>3 基于低频种子的序列比对方法设计</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#56" data-title="&lt;b&gt;3.1 问题定义&lt;/b&gt;"><b>3.1 问题定义</b></a></li>
                                                <li><a href="#59" data-title="&lt;b&gt;3.2 方法描述&lt;/b&gt;"><b>3.2 方法描述</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#77" data-title="&lt;b&gt;4 实验结果与分析&lt;/b&gt; "><b>4 实验结果与分析</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#80" data-title="&lt;b&gt;4.1 模拟数据实验结果&lt;/b&gt;"><b>4.1 模拟数据实验结果</b></a></li>
                                                <li><a href="#84" data-title="&lt;b&gt;4.2 真实数据实验结果&lt;/b&gt;"><b>4.2 真实数据实验结果</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#88" data-title="&lt;b&gt;5 结束语&lt;/b&gt; "><b>5 结束语</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#47" data-title="图1 种子-扩展方法流程">图1 种子-扩展方法流程</a></li>
                                                <li><a href="#53" data-title="图2 种子分布图">图2 种子分布图</a></li>
                                                <li><a href="#63" data-title="图3 索引结构示意图">图3 索引结构示意图</a></li>
                                                <li><a href="#67" data-title="图4 种子生成和投票">图4 种子生成和投票</a></li>
                                                <li><a href="#74" data-title="图5 相邻窗口候选区域合并">图5 相邻窗口候选区域合并</a></li>
                                                <li><a href="#82" data-title="&lt;b&gt;表1 模拟数据集上各方法比对结果&lt;/b&gt;"><b>表1 模拟数据集上各方法比对结果</b></a></li>
                                                <li><a href="#86" data-title="&lt;b&gt;表2 真实数据集上各方法比对结果&lt;/b&gt;"><b>表2 真实数据集上各方法比对结果</b></a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="111">


                                    <a id="bibliography_1" title=" Bleidorn C.Third generation sequencing:Technology and its potential impact on evolutionary biodiversity research[J].Systematics and Biodiversity,2016,14(1):1-8." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=STJD&amp;filename=STJD375A00B75C11EA87085D3D74B8951CA7&amp;v=MjA2NThOdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TkJod3JtK3c2Zz1Oam5CYXJDL0c2RE1yLzFDWVpnT0RRbEl4eEVUNGpvSlN3dmxxR0E5Y0xlVk5zdVlDTw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[1]</b>
                                         Bleidorn C.Third generation sequencing:Technology and its potential impact on evolutionary biodiversity research[J].Systematics and Biodiversity,2016,14(1):1-8.
                                    </a>
                                </li>
                                <li id="113">


                                    <a id="bibliography_2" title=" Roberts R J,Carneiro M O,Schatz M C.Erratum to:The advantages of SMRT sequencing[J].Genome Biology,2017,18(1):156." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDBE64D9295424559E4BAA114DD81BD106&amp;v=MjY0MjFlOE5DSGs4eG1NWG1FNE1TWDdtMkdZOWVNRGdSTHFaQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TkJod3JtK3c2Zz1OajdCYXNITkdOVzRwbzFNWQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[2]</b>
                                         Roberts R J,Carneiro M O,Schatz M C.Erratum to:The advantages of SMRT sequencing[J].Genome Biology,2017,18(1):156.
                                    </a>
                                </li>
                                <li id="115">


                                    <a id="bibliography_3" title=" Jain M,Olsen H E,Paten B,et al.The Oxford nanopore minion:Delivery of nanopore sequencing to the genomics community[J].Genome Biology,2016,17(1):239." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The Oxford Nanopore MinION:delivery of nanopore sequencing to the genomics community">
                                        <b>[3]</b>
                                         Jain M,Olsen H E,Paten B,et al.The Oxford nanopore minion:Delivery of nanopore sequencing to the genomics community[J].Genome Biology,2016,17(1):239.
                                    </a>
                                </li>
                                <li id="117">


                                    <a id="bibliography_4" title=" Chin C S,Alexander D H,Marks P,et al.Nonhybrid,finished microbial genome assemblies from long-read SMRT sequencing data[J].Nature Methods,2013,10(6):563-569." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data">
                                        <b>[4]</b>
                                         Chin C S,Alexander D H,Marks P,et al.Nonhybrid,finished microbial genome assemblies from long-read SMRT sequencing data[J].Nature Methods,2013,10(6):563-569.
                                    </a>
                                </li>
                                <li id="119">


                                    <a id="bibliography_5" title=" Cao H,Hastie A R,Cao D,et al.Rapid detection of structural variation in a human genome using nano channel-based genome mapping technology[J].Giga Science,2014,3(1):34." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Rapid detection of structural variation in a human genome using nanochannel-based genome mapping technology">
                                        <b>[5]</b>
                                         Cao H,Hastie A R,Cao D,et al.Rapid detection of structural variation in a human genome using nano channel-based genome mapping technology[J].Giga Science,2014,3(1):34.
                                    </a>
                                </li>
                                <li id="121">


                                    <a id="bibliography_6" title=" Akalin P K.Introduction to bioinformatics[J].Molecular Nutrition &amp;amp; Food Research,2006,50(7):610-619." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Introduction to bioinformatics">
                                        <b>[6]</b>
                                         Akalin P K.Introduction to bioinformatics[J].Molecular Nutrition &amp;amp; Food Research,2006,50(7):610-619.
                                    </a>
                                </li>
                                <li id="123">


                                    <a id="bibliography_7" title=" Altshuler D M,Durbin R M,Abecasis G R.An integrated map of genetic variation from 1,092 human genomes[J].Nature,2012,491(7422):56-65." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=An integrated map of genetic variation from 1,092 human genomes">
                                        <b>[7]</b>
                                         Altshuler D M,Durbin R M,Abecasis G R.An integrated map of genetic variation from 1,092 human genomes[J].Nature,2012,491(7422):56-65.
                                    </a>
                                </li>
                                <li id="125">


                                    <a id="bibliography_8" title=" Flannick J,Thorleifsson G,Beer N L,et al.Loss-of-function mutations in SLC30A8 protect against type 2 diabetes[J].Nature Genetics,2014,46(4):357-363." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Loss-of-function Mutations in SLC30A8Protect against Type 2 Diabetes">
                                        <b>[8]</b>
                                         Flannick J,Thorleifsson G,Beer N L,et al.Loss-of-function mutations in SLC30A8 protect against type 2 diabetes[J].Nature Genetics,2014,46(4):357-363.
                                    </a>
                                </li>
                                <li id="127">


                                    <a id="bibliography_9" title=" Scally A,Dutheil J Y,Hillier L D W,et al.Insights into hominid evolution from the gorilla genome sequence[J].Nature,2012,483(7388):169-175." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Insights into hominid evolution from the gorilla genome sequence">
                                        <b>[9]</b>
                                         Scally A,Dutheil J Y,Hillier L D W,et al.Insights into hominid evolution from the gorilla genome sequence[J].Nature,2012,483(7388):169-175.
                                    </a>
                                </li>
                                <li id="129">


                                    <a id="bibliography_10" title=" Alkan C,Kidd J M,Marques-Bonet T,et al.Personalized copy number and segmental duplication maps using next-generation sequencing[J].Nature Genetics,2009,41(10):1061-1067." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Personalized copy number and segmental duplication maps using next-generation sequencing">
                                        <b>[10]</b>
                                         Alkan C,Kidd J M,Marques-Bonet T,et al.Personalized copy number and segmental duplication maps using next-generation sequencing[J].Nature Genetics,2009,41(10):1061-1067.
                                    </a>
                                </li>
                                <li id="131">


                                    <a id="bibliography_11" >
                                        <b>[11]</b>
                                     Ni Gui-qiang,Li Bin,Luo Jian-xin,et al.Research on BWT and classical compression algorithm[J].Computer and Digital Engineering,2010,38(11):26-41.(in Chinese)</a>
                                </li>
                                <li id="133">


                                    <a id="bibliography_12" title=" Chaisson M J,Tesler G.Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR):Application and theory[J].BMC Bioinformatics,2012,13(1):238." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Mapping single molecule sequencing reads using Basic Local Alignment with Successive Refinement (BLASR): Theory and Application">
                                        <b>[12]</b>
                                         Chaisson M J,Tesler G.Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR):Application and theory[J].BMC Bioinformatics,2012,13(1):238.
                                    </a>
                                </li>
                                <li id="135">


                                    <a id="bibliography_13" title=" Houtgast E J,Sima V M,Bertels K,et al.An FPGA-based systolic array to accelerate the BWA-MEM genomic mapping algorithm[C]//Proc of 2015 IEEE International Conference on Embedded Computer Systems:Architectures,Modeling,and Simulation (SAMOS),2015:221-227." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=An FPGA-based systolic array to accelerate the BWA-MEM genomic mapping algorithm">
                                        <b>[13]</b>
                                         Houtgast E J,Sima V M,Bertels K,et al.An FPGA-based systolic array to accelerate the BWA-MEM genomic mapping algorithm[C]//Proc of 2015 IEEE International Conference on Embedded Computer Systems:Architectures,Modeling,and Simulation (SAMOS),2015:221-227.
                                    </a>
                                </li>
                                <li id="137">


                                    <a id="bibliography_14" title=" Liu B,Guan D,Teng M,et al.rHAT:Fast alignment of noisy long reads with regional hashing[J].Bioinformatics,2015,32(11):1625-1631." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=rHAT Fast alignment of noisy long reads with regional hashing">
                                        <b>[14]</b>
                                         Liu B,Guan D,Teng M,et al.rHAT:Fast alignment of noisy long reads with regional hashing[J].Bioinformatics,2015,32(11):1625-1631.
                                    </a>
                                </li>
                                <li id="139">


                                    <a id="bibliography_15" title=" Ono Y,Asai K,Hamada M.PBSIM:PacBio reads simulator—toward accurate genome assembly[J].Bioinformatics,2012,29(1):119-121.附中文参考文献:" target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=PBSIM:PacBio reads simulator—toward accurate genome assembly">
                                        <b>[15]</b>
                                         Ono Y,Asai K,Hamada M.PBSIM:PacBio reads simulator—toward accurate genome assembly[J].Bioinformatics,2012,29(1):119-121.附中文参考文献:
                                    </a>
                                </li>
                                <li id="141">


                                    <a id="bibliography_11" title=" 倪桂强,李彬,罗健欣,等.BWT与经典压缩算法研究[J].计算机与数字工程,2010,38(11):26-41." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSSG201011009&amp;v=Mjg3NzR6cXFCdEdGckNVUkxPZVplUm1GeTdrVXJ6SUx6N1lhYkc0SDlITnJvOUZiWVFLREg4NHZSNFQ2ajU0TzM=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[11]</b>
                                         倪桂强,李彬,罗健欣,等.BWT与经典压缩算法研究[J].计算机与数字工程,2010,38(11):26-41.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JSJK" target="_blank">计算机工程与科学</a>
                2019,41(09),1551-1556 DOI:10.3969/j.issn.1007-130X.2019.09.004            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>一种基于低频种子的三代测序序列比对方法</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%AE%8B%E6%80%9D%E6%80%A1&amp;code=42294779&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">宋思怡</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E7%A8%8B%E6%98%8A%E5%AE%87&amp;code=34278874&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">程昊宇</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BE%90%E4%BA%91&amp;code=10336456&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">徐云</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E4%B8%AD%E5%9B%BD%E7%A7%91%E5%AD%A6%E6%8A%80%E6%9C%AF%E5%A4%A7%E5%AD%A6%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%A7%91%E5%AD%A6%E4%B8%8E%E6%8A%80%E6%9C%AF%E5%AD%A6%E9%99%A2&amp;code=0002522&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">中国科学技术大学计算机科学与技术学院</a>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%AE%89%E5%BE%BD%E7%9C%81%E9%AB%98%E6%80%A7%E8%83%BD%E8%AE%A1%E7%AE%97%E9%87%8D%E7%82%B9%E5%AE%9E%E9%AA%8C%E5%AE%A4&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">安徽省高性能计算重点实验室</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>随着测序技术的发展,三代测序已广泛应用于基因研究中。但是,由于三代测序序列具有平均长度长、错误率高的特性,如何快速、准确地将测序片段比对到参考基因组上成为严峻挑战。现有方法使用种子(从测序片段中挑选的短序列)来加速比对过程,但在挑选时未考虑频率特性,导致定位候选区域阶段时间消耗较大。因此,提出了一种基于低频种子的三代测序序列比对方法,该方法采用种子投票策略,使用低频种子进行投票,减少投票计数的时间消耗,并根据位置及票数关系对候选区域进行再过滤,进一步提高比对速度。实验结果表明,在确保敏感性和准确率的同时,本文方法比现有方法快3倍左右。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E4%B8%89%E4%BB%A3%E6%B5%8B%E5%BA%8F&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">三代测序;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8D%95%E5%88%86%E5%AD%90%E5%AE%9E%E6%97%B6%E6%B5%8B%E5%BA%8F&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">单分子实时测序;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%BA%8F%E5%88%97%E6%AF%94%E5%AF%B9&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">序列比对;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%A7%8D%E5%AD%90-%E6%89%A9%E5%B1%95%E6%B3%95&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">种子-扩展法;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    宋思怡（1994-），女，安徽淮南人，硕士生，研究方向为生物信息学。E-mail:songsiyi@mail.ustc.edu.cn,通信地址：230027安徽省合肥市中国科学技术大学计算机科学与技术学院&lt;image id="106" type="formula" href="images/JSJK201909004_10600.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                                <span>
                                    程昊宇（1992-），男，安徽合肥人，博士生，研究方向为生物信息学。E-mail:chhy@mail.ustc.edu.cn,通信地址：230027安徽省合肥市中国科学技术大学计算机科学与技术学院&lt;image id="108" type="formula" href="images/JSJK201909004_10800.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                                <span>
                                    徐云（1960-），男，安徽合肥人，博士，教授，研究方向为生物信息学。E-mail:xuyun@mail.ustc.edu.cn,通信地址：230027安徽省合肥市中国科学技术大学计算机科学与技术学院&lt;image id="110" type="formula" href="images/JSJK201909004_11000.jpg" display="inline" placement="inline"&gt;&lt;alt&gt;&lt;/alt&gt;&lt;/image&gt;;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-11-27</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金(61672480);</span>
                    </p>
            </div>
                    <h1><b>A novel sequence alignment method for third-generation sequencing based on low frequency seeds</b></h1>
                    <h2>
                    <span>SONG Si-yi</span>
                    <span>CHEN Hao-yu</span>
                    <span>XU Yun</span>
            </h2>
                    <h2>
                    <span>School of Computer Science and Technology,University of Science and Technology of China</span>
                    <span>Key Laboratory of High Performance Computing of Anhui Province</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>With the development of sequencing technology, the third-generation sequencing has been widely used in genetic research. It can generate longer sequences but has a higher error rate. It is difficult to align sequences to the reference genome quickly and accurately. Existing methods utilizes seeds which are subsequences selected from test sequences to speed up the alignment process. However, the seed frequency is not fully considered, which results in a large time consumption in the stage of finding candidate regions. We therefore propose a sequence alignment method for third-generation sequencing based on low frequency seeds. Its key idea is a modified seed-voting strategy, which adopts frequency seeds for voting to reduce the time consumption for counting the votes. Moreover, the alignment method re-filters the candidate regions based on the position and the number of votes, further increasing the speed of alignment. Experimental results show that the method is about 3 times faster than existing methods while ensuring sensitivity and accuracy.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=third-generation%20sequencing&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">third-generation sequencing;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=single%20molecule%20real-time%20sequencing&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">single molecule real-time sequencing;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=sequence%20alignment&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">sequence alignment;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=seed-and-extend%20method&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">seed-and-extend method;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                            <b>Author：</b>
                                                        <span>
                                    SONG Si-yi,born in 1994,MS candi-date,his research interest includes bioinformatics.Address:School of Computer Science and Technology,University of Science and Technology of China,Hefei 230027,Anhui,P.R.China;
                                </span>
                                <span>
                                    CHEN Hao-yu,born in 1992,PhD can-didate,his research interest includes bioinformatics.Address:School of Computer Science and Technology,University of Science and Technology of China,Hefei 230027,Anhui,P.R.China;
                                </span>
                                <span>
                                    XU Yun,born in 1960,PhD,professor, his research interest includes bioinformatics.Address:School of Computer Science and Technology,University of Science and Technology of China,Hefei 230027,Anhui,P.R.China;
                                </span>
                    </p>
                                    <p><b>Received：</b> 2018-11-27</p>
                                    <p>
                                            </p>
            </div>


        <!--brief start-->
                        <h3 id="35" name="35" class="anchor-tag"><b>1 引言</b></h3>
                <div class="p1">
                    <p id="36">近年来,高通量测序技术的不断发展为基因组学带来了极大的变革,使得生物学家能够在碱基层面进行研究。目前,三代测序已经广泛地应用于基因研究中<citation id="143" type="reference"><link href="111" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>。常见的三代测序技术有单分子实时测序技术SMRT(Single Molecule Real-Time)<citation id="144" type="reference"><link href="113" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>和纳米孔测序技术<citation id="145" type="reference"><link href="115" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>等。第三代测序技术采用了基于纳米孔的单分子读取技术,在核酸外切酶的作用下,将DNA分子以1次1个碱基的速度穿过小孔,从而将DNA的4个碱基A、C、G、T清楚地分离出来。这种技术不需要荧光标记物并且不需要进行聚合酶链式反应PCR(Polymerase Chain Reaction)扩增,能直接并快速地读出DNA,使大量重复实验成为可能。同时,与二代测序NGS(Next-Generation Sequencing)产生较短的测序片段不同,三代测序产生平均长度为3 k～10 kbp的DNA序列片段,又称为reads,有利于解决de novo基因组装<citation id="146" type="reference"><link href="117" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>和结构变异分析<citation id="147" type="reference"><link href="119" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>中长期存在的问题。然而,由于测序技术的局限性,三代测序产生的reads具有很高的错误率(通常为15%～30%),这会影响测序数据分析的各个关键步骤,其中一个重要问题就是序列比对。</p>
                </div>
                <div class="p1">
                    <p id="37">序列比对是生物信息学中重要的基础性问题之一<citation id="148" type="reference"><link href="121" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>。已有研究表明,序列比对能够解释人类种群之间的多样性<citation id="149" type="reference"><link href="123" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation><sup></sup>,也可以解析导致疾病的基因组变异<citation id="150" type="reference"><link href="125" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>,并可以研究猿类<citation id="151" type="reference"><link href="127" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>和古人类基因的演化过程<citation id="152" type="reference"><link href="129" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>。基因序列比对的具体定义是利用相关的算法,将测序仪测出的reads比对到已知生物序列的参考库中,找出reads在基因组上最可能的起源位置,并根据比对结果判断序列间相似性以及相关的生物学特征。从计算的角度来看,由于三代测序reads错误率高,为NGS所创建的序列比对算法不能很好地直接应用到三代测序数据中。</p>
                </div>
                <div class="p1">
                    <p id="38">三代测序序列比对在基因数据分析中占据了大量的时间。例如,现有方法仅处理4个细胞的测序数据(290 992条reads)就需要7～20 h。因此,设计高效的序列比对算法,加速序列比对的过程,减少比对过程的时间消耗是三代测序序列比对面临的一大挑战。基于此,本文提出了一种基于低频种子投票的三代测序序列比对方法,在加速三代测序序列比对的同时确保一定程度的敏感性和准确率。</p>
                </div>
                <h3 id="39" name="39" class="anchor-tag"><b>2 相关研究</b></h3>
                <div class="p1">
                    <p id="40">现有的三代测序序列比对方法沿用了二代测序序列比对算法中的种子-扩展法,在种子选取及候选位置定位策略和扩展验证阶段上各有侧重。</p>
                </div>
                <h4 class="anchor-tag" id="41" name="41"><b>2.1  种子-扩展法</b></h4>
                <div class="p1">
                    <p id="42">种子-扩展法的目的是通过种子定位,将比对范围由整个参考基因组缩小至部分候选区域,大幅缩小需要进行复杂度较高的精确比对的候选空间,从而减少比对过程的时间消耗。</p>
                </div>
                <div class="p1">
                    <p id="43">种子扩展法先从reads 中挑选一些相对较短的片段,这些分割后的片段又称作种子(seed)。种子的优点是在查找时,能够缩小搜索空间,并能够加速查找的过程。</p>
                </div>
                <div class="p1">
                    <p id="44">然后,对种子进行快速查找,种子的位置存储在索引中,根据索引的分类,查找方法可分为基于哈希的方法和基于Burrows-Wheeler 变换<citation id="153" type="reference"><link href="131" rel="bibliography" /><link href="141" rel="bibliography" /><sup>[<a class="sup">11</a>,<a class="sup">11</a>]</sup></citation>的方法。根据查找出来的位置间的相互关系,通过一定策略定位最有可能的区域,即为候选区域(或位置),这一步骤称之为过滤。</p>
                </div>
                <div class="p1">
                    <p id="45">最后,需要将候选区域(或位置)与测序片段进行碱基级别的精确比对,通过扩展验证检查 reads 在基因组中匹配的质量,得出最终的位置,即最匹配的位置,并用于后续的生物学意义分析。</p>
                </div>
                <div class="p1">
                    <p id="46">种子-扩展方法流程如图1所示。</p>
                </div>
                <div class="area_img" id="47">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJK201909004_047.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 种子-扩展方法流程" src="Detail/GetImg?filename=images/JSJK201909004_047.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 种子-扩展方法流程  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJK201909004_047.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Figure 1 Work flow of the seed-and-extend method</p>

                </div>
                <h4 class="anchor-tag" id="48" name="48"><b>2.2 现有三代测序序列比对方法</b></h4>
                <div class="p1">
                    <p id="49">目前主流的三代测序序列比对方法,如BLASR(Basic Local Alignment with Successive Refinement)<citation id="154" type="reference"><link href="133" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>、BWA-MEM(Burrows-Wheeler Alignment-Maximal Exact Match)<citation id="155" type="reference"><link href="135" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>、rHAT(regional Hashing-based Alignment Tool)<citation id="156" type="reference"><link href="137" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>都采用了种子-扩展法。BLASR是第1个专门为三代测序而设计的序列比对方法,从reads的起始位置开始,重叠地在FM-index或后缀数组中查找reads在参考基因组上的最长公共前缀LCP(Longest Common Subsequence);取LCP-1的片段作为种子,使用全局串链(Global Chaining)算法将匹配的种子链起来,将链中种子数目大于一定阈值的链所在位置定为候选位置;对候选位置采用稀疏动态规划算法SDP(Sparse Dynamic Programming)进行验证。</p>
                </div>
                <div class="p1">
                    <p id="50">BWA-MEM的过滤方法与BLASR类似,在每个查询位置上找到最长的精确匹配MEM(Max Exact Match),将MEM作为种子,使用贪心算法将共线的、接近匹配的种子链起来,并过滤掉一些短链,旨在减少后期不成功的种子扩展。在扩展阶段,BWA-MEM引入一种特殊的分数截断方式得到最后的比对结果。</p>
                </div>
                <div class="p1">
                    <p id="51">rHAT使用了一种基于区域哈希的过滤方法,采用种子投票的方法定位候选区域,其选取reads中段1000 bp作为种子区域,将种子区域中所有重叠的长度为<i>k</i>的连续子序列视为种子,通过种子计数确定候选区域,然后使用基于SDP的启发式方法验证候选区域。</p>
                </div>
                <div class="p1">
                    <p id="52">上述方法中,rHAT方法效果相对较好,但rHAT选取种子时是无差别、顺序截取的,没有考虑频率特性,在参考基因组规模较大时,种子频率高,导致时间消耗大。对人类基因组进行实验,选择一万条reads,统计对候选位置产生影响的种子的个数和频率,得到如图2所示的种子分布图。</p>
                </div>
                <div class="area_img" id="53">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJK201909004_053.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 种子分布图" src="Detail/GetImg?filename=images/JSJK201909004_053.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 种子分布图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJK201909004_053.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Figure 2 Seed distribution</p>

                </div>
                <div class="p1">
                    <p id="54">由图2可发现,对候选区域产生影响的种子中,有72%的种子频率低于200,这些种子的频率总和仅占全部种子频率总和的34%。低频种子往往具有更高的区分度,这种特性在小规模基因组上更加明显。基于上述发现,本文提出一种基于低频种子的三代测序序列比对方法,采用低频种子进行投票,减少投票计数的时间消耗,并根据位置及票数关系对前<i>m</i>个票数最高的候选位置进行再过滤,进一步减少候选位置数目,提高比对速度。</p>
                </div>
                <h3 id="55" name="55" class="anchor-tag"><b>3 基于低频种子的序列比对方法设计</b></h3>
                <h4 class="anchor-tag" id="56" name="56"><b>3.1 问题定义</b></h4>
                <div class="p1">
                    <p id="57">三代测序序列比对就是将测序仪测出的错误率较高(15%～30%)的序列片段,即reads,比对到参考基因组中,目的是为了找到reads在基因组上的近似起源位置。错误包括插入、删除和替换。DNA序列包括A、T、C和G 4个碱基。对一条DNA序列而言,其本质上是一个由有限字符组成的字符串。因此,序列比对问题可以转化为字符串近似匹配问题。</p>
                </div>
                <div class="p1">
                    <p id="58">序列比对问题可用四元组表示为<i>MSA</i> =(<i>Σ</i>,<i>S</i>,<i>A</i>,<i>F</i>)。其中,<i>Σ</i>表示字符串组成元素的集合,对于DNA序列,<i>Σ</i>={A,C,G,T},分别代表4种脱氧核糖核酸。<i>S</i> ={<i>S</i><sub>1</sub>,<i>S</i><sub>2</sub>,…,<i>S</i><sub><i>n</i></sub>}表示由测序平台测出的<i>n</i>条reads,其中<i>S</i><sub><i>i</i></sub> =(<i>b</i><sub><i>i</i></sub><sub>1</sub>,<i>b</i><sub><i>i</i></sub><sub>2</sub>,…,<i>b</i><sub><i>iLi</i></sub>),<i>b</i><sub><i>ij</i></sub>∈<i>Σ</i>,<i>L</i><sub><i>i</i></sub>为第<i>i</i>个测序片段的长度。<i>A</i>(<i>ij</i>)表示第<i>i</i>个测序片段的第<i>j</i>个碱基验证后最终的比对结果。<i>F</i>为打分函数,<i>F</i>(<i>A</i>)的值代表最终比对结果的分值。在此问题中,目的是找到分值最高的比对结果。</p>
                </div>
                <h4 class="anchor-tag" id="59" name="59"><b>3.2 方法描述</b></h4>
                <div class="p1">
                    <p id="60">基于低频种子的序列比对方法采用种子-投票策略,从reads中抽取一定频率范围的种子,使用种子对候选区域进行投票,对得票数较高的区域进行再过滤,最终对过滤后的候选区域进行扩展验证,得到最终比对结果。该方法主要包括4个步骤:(1)建造索引;(2)生成种子投票;(3)过滤候选区域;(4)扩展验证。</p>
                </div>
                <h4 class="anchor-tag" id="61" name="61">3.2.1 建造索引</h4>
                <div class="p1">
                    <p id="62">对参考基因组进行窗口(区域)划分,对每一个窗口进行标序,得到参考基因组窗口列表,并据此建立Hash索引。<i>key</i>值为参考基因组中长度为<i>k</i>的子序列转换的32位无符号整数,代表基因组中长度为<i>k</i>的任意子序列。<i><b>value</b></i>值为32位无符号整数向量,指向包含该子序列的窗口列表。索引结构如图3所示。例如,<i>key</i>值为2的子串出现在参考基因组中的第28号和第97号窗口中。</p>
                </div>
                <div class="area_img" id="63">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJK201909004_063.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 索引结构示意图" src="Detail/GetImg?filename=images/JSJK201909004_063.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 索引结构示意图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJK201909004_063.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Figure 3 Diagram of index structure</p>

                </div>
                <h4 class="anchor-tag" id="64" name="64">3.2.2 生成种子投票</h4>
                <div class="p1">
                    <p id="65">首先,对reads上所有长度为<i>k</i>的连续子序列(简称为<i>k</i>-mer)进行统计,可以得到所有<i>k</i>-mer的频率信息,进而可以得到不同频率的<i>k</i>-mer个数,如图4a所示。当频率范围设置为<i>f</i> %时(<i>f</i>值可由用户定义),从低频开始遍历选择种子。例如,当频率范围设置为10%时,种子区域中频率为1～50的<i>k</i>-mer数目占全部<i>k</i>-mer数目的10%,因此选择频率为1～50的<i>k</i>-mer作为种子。</p>
                </div>
                <div class="p1">
                    <p id="66">在reads上选择一段长为<i>L</i>的区域作为种子区域。在种子区域中,对所有的重叠<i>k</i>-mer进行判断,当频率符合范围时,视为种子。然后,在索引中查找种子出现的位置,并对窗口列表进行多路合并排序并统计票数,选择得票数前<i>m</i>的区域作为候选区域进行后续的过滤和扩展,如图4b所示。</p>
                </div>
                <div class="area_img" id="67">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJK201909004_067.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 种子生成和投票" src="Detail/GetImg?filename=images/JSJK201909004_067.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 种子生成和投票  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJK201909004_067.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Figure 4 Seed generation and voting</p>

                </div>
                <h4 class="anchor-tag" id="68" name="68">3.2.3 过滤候选区域</h4>
                <div class="p1">
                    <p id="69">当<i>m</i>设置为5时,每条reads需要验证5个候选区域,这5个候选区域均需进行碱基间的精确比对,而碱基间精确比对的时间消耗很大。因此,本文采用2个启发式策略对候选区域进行再过滤。在几乎不影响精度的前提下,进一步减少候选区域的数目,以加速比对过程。</p>
                </div>
                <div class="p1">
                    <p id="70">(1) 由于定位候选窗口时使用的是种子区域而不是全部的reads,因此每一个候选窗口验证时都需要根据种子区域位置和reads长度进行扩展。通过实验发现,大部分得票数较高的窗口是相邻的,因此可以将这些窗口进行合并扩展,从而减少需要扩展验证的窗口数目及区域长度。如图5所示,若有2个候选窗口相邻,合并后可将需要进行动态规划比对的序列长度从2(<i>L</i><sub>R</sub>+2<i>L</i>)降至(<i>L</i><sub>R</sub>+3<i>L</i>),其中<i>L</i><sub>R</sub>代表reads长度,<i>L</i>代表种子区域长度。</p>
                </div>
                <div class="p1">
                    <p id="71">(2) 根据得票数以及得票数之间的差距可以进一步过滤,减少需要扩展验证的候选区域数目。默认对得票数最高的窗口进行验证,后<i>m</i>-1个窗口需根据下述公式进行验证:</p>
                </div>
                <div class="p1">
                    <p id="72" class="code-formula">
                        <mathml id="72"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mi>Μ</mi><mi>Ν</mi></mfrac><mo>≥</mo><mn>0</mn><mo>.</mo><mn>2</mn><mn>5</mn><mspace width="0.25em" /><mo>&amp;</mo><mo>&amp;</mo><mspace width="0.25em" /><mfrac><mi>Μ</mi><mrow><mi>Μ</mi><msub><mrow></mrow><mrow><mi>max</mi></mrow></msub></mrow></mfrac><mo>≥</mo><mn>0</mn><mo>.</mo><mn>2</mn><mn>5</mn></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="73">其中,<i>M</i>为该窗口的得票数,<i>N</i>为全部种子数,<i>M</i><sub>max</sub>为得票数最高的窗口的得票数。仅对满足条件的候选窗口进行验证,可灵活调整候选区域数目,减少验证阶段的时间消耗,进而加速序列比对过程。</p>
                </div>
                <div class="area_img" id="74">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJK201909004_074.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图5 相邻窗口候选区域合并" src="Detail/GetImg?filename=images/JSJK201909004_074.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图5 相邻窗口候选区域合并  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJK201909004_074.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Figure 5 Merging of adjacent window candidate regions</p>

                </div>
                <h4 class="anchor-tag" id="75" name="75">3.2.4 扩展验证</h4>
                <div class="p1">
                    <p id="76">确定候选区域后,对每个候选区域使用基于SDP的动态规划方法进行扩展验证。先对候选区域进行哈希索引,从起始位置对reads进行扫描,获得重叠的<i>k</i>-mer,在索引中查找获得锚点;然后对共线的锚点使用动态规划进行串链;最后对得分最高的链进行碱基间的精确比对,得到最终比对结果。</p>
                </div>
                <h3 id="77" name="77" class="anchor-tag"><b>4 实验结果与分析</b></h3>
                <div class="p1">
                    <p id="78">为了研究本文方法的性能,将本文方法与BLASR、BWA-MEM和rHAT进行比较。当运行BLASR、BWA-MEM和rHAT时,所有参数都设置为默认值。虽然本文方法可以在多个线程上运行,但为了简化时间开销的比较,只在单个线程上运行测试。</p>
                </div>
                <div class="p1">
                    <p id="79">本文实验的测试平台为:Intel Core i7-4770 CPU(4核心8线程,3.40 GHz),24.00 GB DDR3 RAM,Ubuntu16.04 64位操作系统。</p>
                </div>
                <h4 class="anchor-tag" id="80" name="80"><b>4.1 模拟数据实验结果</b></h4>
                <div class="p1">
                    <p id="81">首先使用reads模拟工具PBSIM(PacBio reads SIMulator)<citation id="157" type="reference"><link href="139" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation>生成10万条平均长度为8 000 bp的reads。由于模拟reads比对到参考基因组上的真实位置是已知的,因此将报告的位置与真实位置进行比较可知上述方法的准确性。如果报告的位置与真实位置间的距离小于100 bp,认为该结果是正确的。在拟南芥和人类基因组上分别进行模拟reads的比对,两个不同规模的基因组能较为全面地反映方法的效果。实验结果如表1所示。</p>
                </div>
                <div class="area_img" id="82">
                    <p class="img_tit"><b>表1 模拟数据集上各方法比对结果</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 1 Results of different methods on simulated dataset</b></p>
                    <p class="img_note"></p>
                    <table id="82" border="1"><tr><td><br />方法</td><td colspan="2"><math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mrow><mtext>拟</mtext><mtext>南</mtext><mtext>芥</mtext><mtext>基</mtext><mtext>因</mtext><mtext>组</mtext></mrow><mrow><mtext>运</mtext><mtext>行</mtext><mtext>时</mtext><mtext>间</mtext><mo>/</mo><mtext>s</mtext><mtext> </mtext><mtext>准</mtext><mtext>确</mtext><mtext>率</mtext><mi>%</mi></mrow></mfrac></mrow></math></td><td colspan="2"><math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mrow><mtext>人</mtext><mtext>类</mtext><mtext>基</mtext><mtext>因</mtext><mtext>组</mtext></mrow><mrow><mtext>运</mtext><mtext>行</mtext><mtext>时</mtext><mtext>间</mtext><mo>/</mo><mtext>s</mtext><mtext> </mtext><mtext>准</mtext><mtext>确</mtext><mtext>率</mtext><mi>%</mi></mrow></mfrac></mrow></math></td></tr><tr><td><br />BLASR</td><td>5 143.68</td><td>93.25</td><td>6 569.63</td><td>91.93</td></tr><tr><td><br />BWA-MEM</td><td>5 181.58</td><td>94.66</td><td>6 644.65</td><td>92.77</td></tr><tr><td><br />rHAT</td><td>584.76</td><td>99.96</td><td>2 642.88</td><td>99.89</td></tr><tr><td><br />本文方法</td><td>259.49</td><td>99.78</td><td>494.96</td><td>99.60</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="83">由表1可知,本文方法的准确度与rHAT相当,运行速度最快,在拟南芥基因组上运行时间比BLASR、BWA-MEM快约20倍,比rHAT快约2倍;在人类基因组上比BLASR、BWA-MEM快约14倍,比rHAT快约5倍。rHAT在大基因组上性能提升更为明显,这是由于基于种子-投票策略的方法对种子和基因组的变化更加敏感,大基因组具有更高的重复率,在定位候选位置阶段的耗时会随着基因组的增大而大幅增加,使用低频的种子加速效果更为显著。</p>
                </div>
                <h4 class="anchor-tag" id="84" name="84"><b>4.2 真实数据实验结果</b></h4>
                <div class="p1">
                    <p id="85">为了验证本文方法在真实数据集上的性能,随机选取10万条真实reads在人类基因组上进行比对,实验结果如表2所示。</p>
                </div>
                <div class="area_img" id="86">
                    <p class="img_tit"><b>表2 真实数据集上各方法比对结果</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><b>Table 2 Results of different methods on real dataset</b></p>
                    <p class="img_note">s</p>
                    <table id="86" border="1"><tr><td><br />方法</td><td>BLASR</td><td>BWA-MEM</td><td>rHAT</td><td>本文方法</td></tr><tr><td><br />运行时间</td><td>7 798.75</td><td>7 943.78</td><td>4 456.96</td><td>1 484.22</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="87">从表2可以看出,由于真实数据集测序错误及变异情况更为复杂,相较模拟数据,各类方法速度均有所下降,本文方法在真实数据集上仍保持最短的运行时间,运行速度比BLASR、BWA-MEM快约6倍,比rHAT快约3倍。</p>
                </div>
                <h3 id="88" name="88" class="anchor-tag"><b>5 结束语</b></h3>
                <div class="p1">
                    <p id="89">本文提出了一种基于低频种子的三代测序序列比对方法,利用种子的频率特性快速地过滤候选区域,采用了2种启发式策略对候选区域进行再过滤,以减少需要验证的候选区域的数目,从而大幅提高比对速度。实验结果表明,本文方法使用低频种子可以减少过滤阶段的计算量,同时由于低频种子具有一定的代表性和区分度,在加速序列比对的同时仍能保持较高的敏感性和准确度,在大基因组数据集上效果更为显著。下一步工作的重点是改进索引结构,提高其扩展性,以便减少硬盘及内存开销。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="111">
                            <a id="bibliography_1" target="_blank" href="/kcms/detail/detail.aspx?dbcode=STJD&amp;filename=STJD375A00B75C11EA87085D3D74B8951CA7&amp;v=MDM0MTcvMUNZWmdPRFFsSXh4RVQ0am9KU3d2bHFHQTljTGVWTnN1WUNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU5CaHdybSt3Nmc9TmpuQmFyQy9HNkRNcg==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[1]</b> Bleidorn C.Third generation sequencing:Technology and its potential impact on evolutionary biodiversity research[J].Systematics and Biodiversity,2016,14(1):1-8.
                            </a>
                        </p>
                        <p id="113">
                            <a id="bibliography_2" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDBE64D9295424559E4BAA114DD81BD106&amp;v=Mjg4ODc9Tmo3QmFzSE5HTlc0cG8xTVllOE5DSGs4eG1NWG1FNE1TWDdtMkdZOWVNRGdSTHFaQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TkJod3JtK3c2Zw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[2]</b> Roberts R J,Carneiro M O,Schatz M C.Erratum to:The advantages of SMRT sequencing[J].Genome Biology,2017,18(1):156.
                            </a>
                        </p>
                        <p id="115">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The Oxford Nanopore MinION:delivery of nanopore sequencing to the genomics community">

                                <b>[3]</b> Jain M,Olsen H E,Paten B,et al.The Oxford nanopore minion:Delivery of nanopore sequencing to the genomics community[J].Genome Biology,2016,17(1):239.
                            </a>
                        </p>
                        <p id="117">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data">

                                <b>[4]</b> Chin C S,Alexander D H,Marks P,et al.Nonhybrid,finished microbial genome assemblies from long-read SMRT sequencing data[J].Nature Methods,2013,10(6):563-569.
                            </a>
                        </p>
                        <p id="119">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Rapid detection of structural variation in a human genome using nanochannel-based genome mapping technology">

                                <b>[5]</b> Cao H,Hastie A R,Cao D,et al.Rapid detection of structural variation in a human genome using nano channel-based genome mapping technology[J].Giga Science,2014,3(1):34.
                            </a>
                        </p>
                        <p id="121">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Introduction to bioinformatics">

                                <b>[6]</b> Akalin P K.Introduction to bioinformatics[J].Molecular Nutrition &amp; Food Research,2006,50(7):610-619.
                            </a>
                        </p>
                        <p id="123">
                            <a id="bibliography_7" target="_blank" href="http://scholar.cnki.net/result.aspx?q=An integrated map of genetic variation from 1,092 human genomes">

                                <b>[7]</b> Altshuler D M,Durbin R M,Abecasis G R.An integrated map of genetic variation from 1,092 human genomes[J].Nature,2012,491(7422):56-65.
                            </a>
                        </p>
                        <p id="125">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Loss-of-function Mutations in SLC30A8Protect against Type 2 Diabetes">

                                <b>[8]</b> Flannick J,Thorleifsson G,Beer N L,et al.Loss-of-function mutations in SLC30A8 protect against type 2 diabetes[J].Nature Genetics,2014,46(4):357-363.
                            </a>
                        </p>
                        <p id="127">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Insights into hominid evolution from the gorilla genome sequence">

                                <b>[9]</b> Scally A,Dutheil J Y,Hillier L D W,et al.Insights into hominid evolution from the gorilla genome sequence[J].Nature,2012,483(7388):169-175.
                            </a>
                        </p>
                        <p id="129">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Personalized copy number and segmental duplication maps using next-generation sequencing">

                                <b>[10]</b> Alkan C,Kidd J M,Marques-Bonet T,et al.Personalized copy number and segmental duplication maps using next-generation sequencing[J].Nature Genetics,2009,41(10):1061-1067.
                            </a>
                        </p>
                        <p id="131">
                            <a id="bibliography_11" >
                                    <b>[11]</b>
                                 Ni Gui-qiang,Li Bin,Luo Jian-xin,et al.Research on BWT and classical compression algorithm[J].Computer and Digital Engineering,2010,38(11):26-41.(in Chinese)
                            </a>
                        </p>
                        <p id="133">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Mapping single molecule sequencing reads using Basic Local Alignment with Successive Refinement (BLASR): Theory and Application">

                                <b>[12]</b> Chaisson M J,Tesler G.Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR):Application and theory[J].BMC Bioinformatics,2012,13(1):238.
                            </a>
                        </p>
                        <p id="135">
                            <a id="bibliography_13" target="_blank" href="http://scholar.cnki.net/result.aspx?q=An FPGA-based systolic array to accelerate the BWA-MEM genomic mapping algorithm">

                                <b>[13]</b> Houtgast E J,Sima V M,Bertels K,et al.An FPGA-based systolic array to accelerate the BWA-MEM genomic mapping algorithm[C]//Proc of 2015 IEEE International Conference on Embedded Computer Systems:Architectures,Modeling,and Simulation (SAMOS),2015:221-227.
                            </a>
                        </p>
                        <p id="137">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=rHAT Fast alignment of noisy long reads with regional hashing">

                                <b>[14]</b> Liu B,Guan D,Teng M,et al.rHAT:Fast alignment of noisy long reads with regional hashing[J].Bioinformatics,2015,32(11):1625-1631.
                            </a>
                        </p>
                        <p id="139">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=PBSIM:PacBio reads simulator—toward accurate genome assembly">

                                <b>[15]</b> Ono Y,Asai K,Hamada M.PBSIM:PacBio reads simulator—toward accurate genome assembly[J].Bioinformatics,2012,29(1):119-121.附中文参考文献:
                            </a>
                        </p>
                        <p id="141">
                            <a id="bibliography_11" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSSG201011009&amp;v=MTExNDRSTE9lWmVSbUZ5N2tVcnpJTHo3WWFiRzRIOUhOcm85RmJZUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1U=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[11]</b> 倪桂强,李彬,罗健欣,等.BWT与经典压缩算法研究[J].计算机与数字工程,2010,38(11):26-41.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JSJK201909004" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJK201909004&amp;v=MTgzNzdJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSbUZ5N2tVcnpJTHo3QlpiRzRIOWpNcG85Rlk=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQ3cmEvcXIxT0wrTHhvaVdCNFNyZz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
