<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637131277384493750%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJSJC201902026%26RESULT%3d1%26SIGN%3d9Bm50z%252bArx7B84QVzt75e3Z%252b92I%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJC201902026&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJC201902026&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201902026&amp;v=MDkxMTJGeWprVUwvTEx6N0JiYkc0SDlqTXJZOUhZb1FLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUm4=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#95" data-title="0概述 ">0概述</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#100" data-title="1 miREmb方法描述 ">1 miREmb方法描述</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#102" data-title="1.1 多源miRNA相关网络">1.1 多源miRNA相关网络</a></li>
                                                <li><a href="#106" data-title="1.2 多源网络表示学习">1.2 多源网络表示学习</a></li>
                                                <li><a href="#117" data-title="1.3 节点的相关性定义">1.3 节点的相关性定义</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#142" data-title="2 实验结果与分析 ">2 实验结果与分析</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#143" data-title="2.1 实验数据集">2.1 实验数据集</a></li>
                                                <li><a href="#156" data-title="2.2 对比算法">2.2 对比算法</a></li>
                                                <li><a href="#162" data-title="2.3 miRNA家族的功能相似性">2.3 miRNA家族的功能相似性</a></li>
                                                <li><a href="#165" data-title="2.4 miRNA与疾病关联">2.4 miRNA与疾病关联</a></li>
                                                <li><a href="#173" data-title="2.5 未知miRNA预测与疾病关联">2.5 未知miRNA预测与疾病关联</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#177" data-title="3 结束语 ">3 结束语</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#105" data-title="图1 miRNA相关网络结构">图1 miRNA相关网络结构</a></li>
                                                <li><a href="#108" data-title="图2 多源miRNA相关网络的关系矩阵表示及学习到的miRNA特征向量">图2 多源miRNA相关网络的关系矩阵表示及学习到的miRNA特征向量</a></li>
                                                <li><a href="#120" data-title="表1 miRNA的关联路径及其含义">表1 miRNA的关联路径及其含义</a></li>
                                                <li><a href="#138" data-title="表2 各计算方式的采样候选集大小及计算次数">表2 各计算方式的采样候选集大小及计算次数</a></li>
                                                <li><a href="#164" data-title="图3 各方法相似度对比结果">图3 各方法相似度对比结果</a></li>
                                                <li><a href="#172" data-title="表3 4种方法对miRNA相关疾病的NDCG对比结果">表3 4种方法对miRNA相关疾病的NDCG对比结果</a></li>
                                                <li><a href="#176" data-title="表4 miREmb算法疾病候选miRNA预测以及在miRNet数据库中的确认记录数">表4 miREmb算法疾病候选miRNA预测以及在miRNet数据库中的确认记录数</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="49">


                                    <a id="bibliography_1" title="GUZZI P H, DI M M, TAGLIAFERRI P, et al.Analysis of miRNA, mRNA, and TF interactions through networkbased methods[J].Eurasip Journal on Bioinformatics and Systems Biology, 2015 (1) :4." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD15070900002085&amp;v=MDQzNjdGWk9zTkRIUThvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMYklJRjhYYWhFPU5qN0Jhcks5SHRiTXBvOQ==&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[1]</b>
                                        GUZZI P H, DI M M, TAGLIAFERRI P, et al.Analysis of miRNA, mRNA, and TF interactions through networkbased methods[J].Eurasip Journal on Bioinformatics and Systems Biology, 2015 (1) :4.
                                    </a>
                                </li>
                                <li id="51">


                                    <a id="bibliography_2" title="彭可可, 欧阳伶宣, 邓朝谦, 等.MicroRNA研究概述[J].动物医学进展, 2016, 37 (4) :110-114." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=DYJZ201604024&amp;v=MTk4MjdUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJuRnlqa1VML0xJVFRCZExHNEg5Zk1xNDlIWUlRS0RIODR2UjQ=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[2]</b>
                                        彭可可, 欧阳伶宣, 邓朝谦, 等.MicroRNA研究概述[J].动物医学进展, 2016, 37 (4) :110-114.
                                    </a>
                                </li>
                                <li id="53">


                                    <a id="bibliography_3" title="WANG D, WANG J, LU M, et al.Inferring the human microRNA functional similarity and functional network based on microRNA-associated diseases[J].Bioinformatics, 2010, 26 (13) :1644-1650." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Inferring the human microRNA functional similarity and functional network based on microRNA-associated diseases">
                                        <b>[3]</b>
                                        WANG D, WANG J, LU M, et al.Inferring the human microRNA functional similarity and functional network based on microRNA-associated diseases[J].Bioinformatics, 2010, 26 (13) :1644-1650.
                                    </a>
                                </li>
                                <li id="55">


                                    <a id="bibliography_4" title="CHEN X, YAN C C, ZHANG X, et al.WBSMDA:within and between score for miRNA-disease association prediction[EB/OL].[2017-12-01].https://www.nature.com/articles/srep21106." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=WBSMDA:within and between score for miRNA-disease association prediction">
                                        <b>[4]</b>
                                        CHEN X, YAN C C, ZHANG X, et al.WBSMDA:within and between score for miRNA-disease association prediction[EB/OL].[2017-12-01].https://www.nature.com/articles/srep21106.
                                    </a>
                                </li>
                                <li id="57">


                                    <a id="bibliography_5" title="LE D H.Network-based ranking methods for prediction of novel disease associated microRNAs[J].Computational Biology and Chemistry, 2015, 58:139-148." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES5EDB327039C952A748A713E2088A99D4&amp;v=MDgwOTJ3S3M9TmlmT2ZiYk5hcVBQclloRlorSjhCWGs3dmhFWDRrNTZTWHlYcmhJOWNjT2RUTTZiQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TkZoeExtOA==&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[5]</b>
                                        LE D H.Network-based ranking methods for prediction of novel disease associated microRNAs[J].Computational Biology and Chemistry, 2015, 58:139-148.
                                    </a>
                                </li>
                                <li id="59">


                                    <a id="bibliography_6" title="DING P, LUO J, LIANG C, et al.A novel group wisebased method for calculating human miRNA functional similarity[J].IEEE Access, 2017, 5 (99) :2364-2372." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A novel group wisebased method for calculating human miRNA functional similarity">
                                        <b>[6]</b>
                                        DING P, LUO J, LIANG C, et al.A novel group wisebased method for calculating human miRNA functional similarity[J].IEEE Access, 2017, 5 (99) :2364-2372.
                                    </a>
                                </li>
                                <li id="61">


                                    <a id="bibliography_7" title="CUMMINS J M, VELCULESCU V E.Implications of microRNA profiling for cancer diagnosis[J].Oncogene, 2006, 25 (46) :6220-6227." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Implications of micro-RNA profiling for cancer diagnosis">
                                        <b>[7]</b>
                                        CUMMINS J M, VELCULESCU V E.Implications of microRNA profiling for cancer diagnosis[J].Oncogene, 2006, 25 (46) :6220-6227.
                                    </a>
                                </li>
                                <li id="63">


                                    <a id="bibliography_8" title="LU Ming, ZHANG Qipeng, DENG Min, et al.An analysis of human microRNA and disease associations[J].PloS One, 2008, 3 (10) ." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=An analysis of human microRNA and disease associations">
                                        <b>[8]</b>
                                        LU Ming, ZHANG Qipeng, DENG Min, et al.An analysis of human microRNA and disease associations[J].PloS One, 2008, 3 (10) .
                                    </a>
                                </li>
                                <li id="65">


                                    <a id="bibliography_9" title="侯妍妍, 应晓敏, 李伍举.microRNA计算发现方法的研究进展[J].遗传, 2008, 30 (6) :687-696." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=YCZZ200806005&amp;v=MDQ2MzJxWTlGWVlRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJuRnlqa1VML0xQQzdSZExHNEh0bk0=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[9]</b>
                                        侯妍妍, 应晓敏, 李伍举.microRNA计算发现方法的研究进展[J].遗传, 2008, 30 (6) :687-696.
                                    </a>
                                </li>
                                <li id="67">


                                    <a id="bibliography_10" title="涂存超, 杨成, 刘知远, 等.网络表示学习综述[J].中国科学:信息科学, 2017 (8) ." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=PZKX201708003&amp;v=MTcxMzR6cXFCdEdGckNVUkxPZVplUm5GeWprVUwvTE5UZkFkckc0SDliTXA0OUZaNFFLREg4NHZSNFQ2ajU0TzM=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[10]</b>
                                        涂存超, 杨成, 刘知远, 等.网络表示学习综述[J].中国科学:信息科学, 2017 (8) .
                                    </a>
                                </li>
                                <li id="69">


                                    <a id="bibliography_11" title="PEROZZI B, AL-RFOU R, SKIENA S.DeepWalk:online learning of social representations[C]//Proceedings of the20th ACM SIGKDD International Conference on Knowledge Discovery and Data Mining.New York, USA:ACM Press, 2014:701-710." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Deepwalk:online learning of social representations">
                                        <b>[11]</b>
                                        PEROZZI B, AL-RFOU R, SKIENA S.DeepWalk:online learning of social representations[C]//Proceedings of the20th ACM SIGKDD International Conference on Knowledge Discovery and Data Mining.New York, USA:ACM Press, 2014:701-710.
                                    </a>
                                </li>
                                <li id="71">


                                    <a id="bibliography_12" title="MIKOLOV T, SUTSKEVER I, CHEN K, et al.Distributed representations of words and phrases and their compositionality[C]//Proceedings of the 26th International Conference on Neural Information Processing Systems.[S.l.]:Curran Associates Inc., 2013:3111-3119." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Distributed Representations of Words and Phrases and their Compositionality">
                                        <b>[12]</b>
                                        MIKOLOV T, SUTSKEVER I, CHEN K, et al.Distributed representations of words and phrases and their compositionality[C]//Proceedings of the 26th International Conference on Neural Information Processing Systems.[S.l.]:Curran Associates Inc., 2013:3111-3119.
                                    </a>
                                </li>
                                <li id="73">


                                    <a id="bibliography_13" title="段旭磊, 张仰森, 孙祎卓.微博文本的句向量表示及相似度计算方法研究[J].计算机工程, 2017, 43 (5) :143-148." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201705024&amp;v=MDM1ODFyQ1VSTE9lWmVSbkZ5amtVTC9MTHo3QmJiRzRIOWJNcW85SFlJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0Y=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[13]</b>
                                        段旭磊, 张仰森, 孙祎卓.微博文本的句向量表示及相似度计算方法研究[J].计算机工程, 2017, 43 (5) :143-148.
                                    </a>
                                </li>
                                <li id="75">


                                    <a id="bibliography_14" title="HSU S D, LIN F M, WU W Y, et al.miRTarBase:a database curates experimentally validated microRNAtarget interactions[J].Nucleic Acids Research, 2011, 39:163-169." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=miRTarBase:a database curates experimentally validated microRNA-target interactions">
                                        <b>[14]</b>
                                        HSU S D, LIN F M, WU W Y, et al.miRTarBase:a database curates experimentally validated microRNAtarget interactions[J].Nucleic Acids Research, 2011, 39:163-169.
                                    </a>
                                </li>
                                <li id="77">


                                    <a id="bibliography_15" title="CHEN H, ZHANG Z.Similarity-based methods for potential human microRNA-disease association prediction[J].BMCMedical Genomics, 2013, 6 (1) :12." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Similarity-based methods for potential human microRNA-disease association prediction">
                                        <b>[15]</b>
                                        CHEN H, ZHANG Z.Similarity-based methods for potential human microRNA-disease association prediction[J].BMCMedical Genomics, 2013, 6 (1) :12.
                                    </a>
                                </li>
                                <li id="79">


                                    <a id="bibliography_16" title="PIERO J, BRAVO, QUERALTROSINACH N, et al.DisGeNET:a comprehensive platform integrating information on human disease-associated genes and variants[J].Nucleic Acids Research, 2017, 45 (D1) :833-839." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=DisGeNET:a comprehensive platform integrating information on human disease-associated genes and variants">
                                        <b>[16]</b>
                                        PIERO J, BRAVO, QUERALTROSINACH N, et al.DisGeNET:a comprehensive platform integrating information on human disease-associated genes and variants[J].Nucleic Acids Research, 2017, 45 (D1) :833-839.
                                    </a>
                                </li>
                                <li id="81">


                                    <a id="bibliography_17" title="KESHAVA PRASAD T S, GOEL R, KANDASAMY K, et al.Human protein reference database[J].Nucleic Acids Research, 2009, 37:767-772." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Human Protein Reference Database--2009 update">
                                        <b>[17]</b>
                                        KESHAVA PRASAD T S, GOEL R, KANDASAMY K, et al.Human protein reference database[J].Nucleic Acids Research, 2009, 37:767-772.
                                    </a>
                                </li>
                                <li id="83">


                                    <a id="bibliography_18" title="VAN D M A, BRUGGEMAN J, VRIEND G, et al.Atext-mining analysis of the human phenome[J].European Journal of Human Genetics, 2006, 14 (5) :535-542." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A text-mining analysis of the human phenome">
                                        <b>[18]</b>
                                        VAN D M A, BRUGGEMAN J, VRIEND G, et al.Atext-mining analysis of the human phenome[J].European Journal of Human Genetics, 2006, 14 (5) :535-542.
                                    </a>
                                </li>
                                <li id="85">


                                    <a id="bibliography_19" title="HAMOSH A, SCOTT A F, AMBERGER J S, et al.Online mendelian inheritance in man (OMIM) , a knowledgebase of human genes and genetic disorders[J].Nucleic Acids Research, 2005, 33 (1) :514-517." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic disorders">
                                        <b>[19]</b>
                                        HAMOSH A, SCOTT A F, AMBERGER J S, et al.Online mendelian inheritance in man (OMIM) , a knowledgebase of human genes and genetic disorders[J].Nucleic Acids Research, 2005, 33 (1) :514-517.
                                    </a>
                                </li>
                                <li id="87">


                                    <a id="bibliography_20" title="SUN J, ZHOU M, YANG H, et al.Inferring potential microRNA-microRNA associations based on targeting propensity and connectivity in the context of protein interaction network.[J].PloS One, 2013, 8 (7) ." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Inferring Potential microRNA-microRNA Associations Based on Targeting Propensity and Connectivity in the Context of Protein Interaction Network">
                                        <b>[20]</b>
                                        SUN J, ZHOU M, YANG H, et al.Inferring potential microRNA-microRNA associations based on targeting propensity and connectivity in the context of protein interaction network.[J].PloS One, 2013, 8 (7) .
                                    </a>
                                </li>
                                <li id="89">


                                    <a id="bibliography_21" title="GRIFFITHS J S.miRBase:the microRNA sequence database.[J].Methods in Molecular Biology, 2006, 342:129-138." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=miRBase:the microRNA sequence database">
                                        <b>[21]</b>
                                        GRIFFITHS J S.miRBase:the microRNA sequence database.[J].Methods in Molecular Biology, 2006, 342:129-138.
                                    </a>
                                </li>
                                <li id="91">


                                    <a id="bibliography_22" title="FAN Y, SIKLENKA K, ARORA S K, et al.miRNetdissecting miRNA-target interactions and functional associations through network-based visual analysis[J].Nucleic Acids Research, 2016, 44:135-141." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=miRNet-dissecting miRNA-target interactions and functional associations through network-based visual analysis">
                                        <b>[22]</b>
                                        FAN Y, SIKLENKA K, ARORA S K, et al.miRNetdissecting miRNA-target interactions and functional associations through network-based visual analysis[J].Nucleic Acids Research, 2016, 44:135-141.
                                    </a>
                                </li>
                                <li id="93">


                                    <a id="bibliography_23" title="XUAN P, HAN K, GUO Y, et al.Prediction of potential disease-associated microRNAs based on random walk[J].Bioinformatics, 2015, 31 (11) :1805-1815." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prediction of potential disease-associated micro RNAs based on random walk">
                                        <b>[23]</b>
                                        XUAN P, HAN K, GUO Y, et al.Prediction of potential disease-associated microRNAs based on random walk[J].Bioinformatics, 2015, 31 (11) :1805-1815.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JSJC" target="_blank">计算机工程</a>
                2019,45(02),154-159 DOI:10.19678/j.issn.1000-3428.0049931            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm">基于网络表示学习的miRNA功能相似性研究</span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E9%98%AE%E7%92%90&amp;code=38677790&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">阮璐</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E7%86%8A%E8%B5%9F&amp;code=11508862&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">熊赟</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%A4%8D%E6%97%A6%E5%A4%A7%E5%AD%A6%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%A7%91%E5%AD%A6%E6%8A%80%E6%9C%AF%E5%AD%A6%E9%99%A2&amp;code=0075855&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">复旦大学计算机科学技术学院</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>miRNA是一类重要的非编码小RNA分子, 与癌症等疾病有密切的关系。目前研究者已经识别大量miRNA, 但是多数miRNA的功能仍然未知。为此, 提出一种网络表示学习的miRNA功能相似性计算方法。通过miRNA的相关数据集如目标基因和关联疾病可以有效地计算miRNA的功能相似性, 从而预测疾病相关的候选miRNA。利用不同类型生物数据集构建miRNA相关多源网络, 采用网络表示学习的方式为网络中的每一个miRNA节点学习一个特征向量, 进而使用特征向量来衡量miRNA的相似性。实验结果表明, 与DeepWalk方法相比, 该方法在同一家族的miRNA中能够取得较高的得分, 并且可以在已有的数据库中找到疾病候选miRNA验证记录。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8A%9F%E8%83%BD%E7%9B%B8%E4%BC%BC%E6%80%A7&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">功能相似性;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%BD%91%E7%BB%9C%E8%A1%A8%E7%A4%BA%E5%AD%A6%E4%B9%A0&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">网络表示学习;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%BD%91%E7%BB%9C%E5%B5%8C%E5%85%A5&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">网络嵌入;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%A4%9A%E6%BA%90%E7%BD%91%E7%BB%9C&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">多源网络;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%89%B9%E5%BE%81%E5%90%91%E9%87%8F&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">特征向量;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    阮璐 (1992—) , 女, 硕士研究生, 主研方向为数据挖掘、数据科学;E-mail: lruan15@ fudan. edu. cn
;
                                </span>
                                <span>
                                    熊赟, 教授。;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-01-02</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家高技术研究发展计划 (2015AA020105-10);</span>
                                <span>上海市科委基金 (16JC1400800, 17511105502);</span>
                    </p>
            </div>
                    <h1>Research on Functional Similarity of miRNA Based on Network Representation Learning</h1>
                    <h2>
                    <span>RUAN Lu</span>
                    <span>XIONG Yun</span>
            </h2>
                    <h2>
                    <span>School of Computer Science, Fudan University</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>miRNA is an important class of non-coding small RNA molecules that are closely related to diseases such as cancer. Researchers have identified a large number of miRNA, but the function of most miRNA remains unknown.Therefore, a method for calculating the similarity of miRNA functions based on network representation learning is proposed.The functional similarity of miRNA can be efficiently calculated by relevant data sets of miRNA such as target genes and associated diseases, thereby predicting disease-related candidate miRNA. miRNA-related multi-source network is constructed by utilizing different types of biological data sets. The network representation learning method is used to learn a feature vector for each miRNA node in the network, and the similarity of the miRNA is measured by the learned feature vector.Experimental results show that compared with the DeepWalk method, the method can obtain higher scores in the same family of miRNA, and the disease candidate miRNA verification records can be found in the existing database.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=functional%20similarity&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">functional similarity;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=network%20representation%20learning&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">network representation learning;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=network%20embedding&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">network embedding;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=multi-source%20network&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">multi-source network;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=feature%20vector&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">feature vector;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                                            </p>
                                    <p><b>Received：</b> 2018-01-02</p>
                                    <p>
                                            </p>
            </div>


        <!--brief start-->
                        <h3 id="95" name="95" class="anchor-tag">0概述</h3>
                <div class="p1">
                    <p id="96">miRNA是一种内源性非编码单链RNA分子, 对基因的表达具有重要的调控作用<citation id="182" type="reference"><link href="49" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>。miRNA参与动植物的细胞分化、增殖、凋亡、组织和器官形成等生命过程, miRNA的功能失调会导致各种疾病, 包括免疫系统的失调以及多种癌症<citation id="183" type="reference"><link href="51" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>。</p>
                </div>
                <div class="p1">
                    <p id="97">国内外学者已经识别出数以万计的miRNA, 并通过生物医学实验方法确定一部分miRNA的功能, 但大部分miRNA的功能仍然未知<citation id="185" type="reference"><link href="53" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>。研究者需要有效的方法推测可靠的候选miRNA集, 以提高实验效率。随着近年来生物数据的大量积累和开放共享, 通过生物信息学的方法预测miRNA功能的方法成为一种高效的方法<citation id="184" type="reference"><link href="55" rel="bibliography" /><link href="57" rel="bibliography" /><sup>[<a class="sup">4</a>,<a class="sup">5</a>]</sup></citation>。</p>
                </div>
                <div class="p1">
                    <p id="98">研究miRNA主要的任务是确定miRNA之间的功能相似性, 即通过计算功能明确的miRNA和功能未知的miRNA之间的功能相似性推测功能未知miRNA的功能。尽管可以通过miRNA序列和结构来度量miRNA的功能相似性, 但研究表明有些具有相似序列和结构的miRNA却有不同的功能<citation id="186" type="reference"><link href="59" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>。因此, 寻找一个有效的方法度量miRNA之间的功能相似性是研究的一个重点。通过生物信息学来计算miRNA功能相似性的方法主要分为2类, 一类方法基于miRNA的目标基因, miRNA的功能由其调控的下游目标基因决定<citation id="187" type="reference"><link href="61" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>, 2个miRNA调控的目标基因群越相似, 它们有相似功能的可能性就越高。另一类方法基于与miRNA关联的疾病。研究表明功能相似的miRNA经常涉及表型相似的疾病, 而与相似表型疾病相关的miRNA具有相似的功能<citation id="188" type="reference"><link href="63" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>。相关基因和相关疾病对miRNA相似性研究有较大作用, 然而多数研究主要考虑其中一类信息, 并且这些方法针对特定的数据集, 无法简单地扩展到其他类型的数据集或者融合不同类型的数据来源<citation id="189" type="reference"><link href="65" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>。</p>
                </div>
                <div class="p1">
                    <p id="99">本文基于miRNA-基因关系、基因-基因相似性、miRNA-疾病关系和疾病-疾病相似性等多个数据集构建多源miRNA相关网络, 通过网络表示学习 (或称为网络嵌入) 方法学习miRNA的特征向量, 根据特征向量的相似性衡量miRNA的功能相似性, 从而提供可靠的miRNA功能相似性度量方式。</p>
                </div>
                <h3 id="100" name="100" class="anchor-tag">1 miREmb方法描述</h3>
                <div class="p1">
                    <p id="101">为充分利用现有的生物实验数据, 本文提出从多个公开数据集构建miRNA相关网络, 并将网络表示学习方法应用到构建的多源网络, 为网络中的每一个miRNA节点学习一个特征向量。学习到的网络表示向量不仅能直接根据余弦相似性计算miRNA功能相似分, 而且可以作为特征向量运用到其他任务中。</p>
                </div>
                <h4 class="anchor-tag" id="102" name="102">1.1 多源miRNA相关网络</h4>
                <div class="p1">
                    <p id="103">在生物信息学中, 网络结构描述生物网络的特性, 节点表示实际的生物物质, 边表示它们之间的相互作用关系。比如在基因相互作用网络中, 节点代表单个基因, 节点之间的边表示基因之间存在着相互作用关系, 边的权重表示节点间的相关性或相似性程度。随着生物实验技术的进步, 各种生物数据的种类越来越多, 规模也越来越大, 将多个数据集进行整合, 构建miRNA相关网络, 进一步挖掘数据的深层信息。</p>
                </div>
                <div class="p1">
                    <p id="104">现有研究主要通过miRNA的目标基因和关联疾病计算miRNA的功能相似性, 这些方法主要用到miRNA-基因关系网络和miRNA-疾病关系网络。此外, 生物学家对一些基因和疾病之间的关系做了研究, 提供基因-疾病关系网络。除了不同生物物质之间的相互作用关系网络, 同一种物质也存在相互作用和相似关系, 其中可以利用的数据包括基因相互作用网络和疾病相似关系网络, 根据上述列举的相关数据构建多源miRNA相关网络, 网络结构如图1所示。</p>
                </div>
                <div class="area_img" id="105">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902026_10500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 miRNA相关网络结构" src="Detail/GetImg?filename=images/JSJC201902026_10500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 miRNA相关网络结构  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902026_10500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="106" name="106">1.2 多源网络表示学习</h4>
                <div class="p1">
                    <p id="107">图2给出多个关系矩阵表示多源网络, 用网络表示学习算法从网络数据中学习得到每个节点的低维向量表示, 然后利用空间中向量的距离衡量miRNA节点的相似性。</p>
                </div>
                <div class="area_img" id="108">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902026_10800.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 多源miRNA相关网络的关系矩阵表示及学习到的miRNA特征向量" src="Detail/GetImg?filename=images/JSJC201902026_10800.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 多源miRNA相关网络的关系矩阵表示及学习到的miRNA特征向量  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902026_10800.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="109">定义1多源网络表示为G= (V, E) =∪G<sub>xy</sub>, 其中, 每个单源二分网络为G<sub>xy</sub>= (V<sub>x</sub>, V<sub>y</sub>, E<sub>xy</sub>) , V<sub>x</sub>和V<sub>y</sub>分别为类型x、y的节点集合, E<sub>xy</sub>是边的集合。每个G<sub>xy</sub>可以用关系矩阵M<sub>xy</sub>表示, M<sub>xy</sub><sup>ij</sup>=1表示 (V<sub>x</sub><sup>i</sup>, V<sub>y</sub><sup>j</sup>) ∈E<sub>xy</sub>, 否则为0。</p>
                </div>
                <div class="p1">
                    <p id="179">定义2网络表示学习<citation id="205" type="reference"><link href="67" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>, 也叫网络嵌入, 是网络中节点学习低维稠密的向量表示。网络表示学习的目标是对每个节点v∈V学习一个实数向量<image id="180" type="formula" href="images/JSJC201902026_18000.jpg" display="inline" placement="inline"><alt></alt></image>, 其中, 向量的维度d远远小于节点的总个数|V|。多源miRNA相关网络表示学习的目的是为每个miRNA节点学习一个低维向量, 使得调控基因相似或关联疾病相似miRNA学习到相似的特征向量, 并通过向量相似度来度量miRNA的功能相似性。</p>
                </div>
                <div class="p1">
                    <p id="112">基于神经网络的网络表示学习是一种快速而有效的方法<citation id="190" type="reference"><link href="69" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>, 这种方法通过最大化相关节点的共现概率使得学习到的节点表示向量保留原始网络结构, 节点v和节点u具有相关性或是相似性的概率可以通过Softmax函数建模。</p>
                </div>
                <div class="area_img" id="113">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJC201902026_11300.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="114">其中, f (u, v) 定义为节点u和节点v特征向量的点积, 即f (u, v) =X<sub>u</sub><sup>T</sup>·X<sub>v</sub>, V<sub>v</sub>是和节点v类型相同的节点集合, 表示学习的目标是根据网络结构来最大化所有相关节点对的共现概率, 即:</p>
                </div>
                <div class="area_img" id="115">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJC201902026_11500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="116">其中, S (u) 表示节点u在网络中所有相关节点的集合, X是待学习的网络表示向量。</p>
                </div>
                <h4 class="anchor-tag" id="117" name="117">1.3 节点的相关性定义</h4>
                <div class="p1">
                    <p id="118">为得到最优的网络表示向量X, 在miRNA相关网络中, 定义具有相关性的节点对代入式 (2) 进行优化。调控目标基因群相似的miRNA, 或相似表型疾病相关的2个miRNA功能相似。通过基因相似性和疾病相似性, 以及已知的基因-疾病关联, 在构建的miRNA相关网络中定义多种miRNA相关性。</p>
                </div>
                <div class="p1">
                    <p id="119">定义3关联路径代表网络中具有关联含义的节点路径, 表示为P=[R<sub>1</sub>→R<sub>2</sub>→…→R<sub>L</sub>], R<sub>i</sub>代表不同节点类型。在多源miRNA相关网络表示学习问题中, R<sub>i</sub>∈{miRNA, 基因, 疾病}, R<sub>1</sub>=R<sub>L</sub>=miRNA。表1列举了长度小于4的关联路径及其含义。</p>
                </div>
                <div class="area_img" id="120">
                                            <p class="img_tit">
                                                表1 miRNA的关联路径及其含义
                                                
                                                &nbsp;&nbsp;
                                                <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902026_12000.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a>
                                                <a class="table downimg" data-tablename="Detail/GetImg?filename=images/JSJC201902026_12000.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">下载原表</a>
                                            </p>
                                    <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902026_12000.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <img alt="表1 miRNA的关联路径及其含义" src="Detail/GetImg?filename=images/JSJC201902026_12000.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                    </a>

                </div>
                <div class="p1">
                    <p id="121">根据表1中定义的多种miRNA相关性, 可以找出网络中存在的所有相关节点, 并根据式 (2) 训练节点向量, 但存在以下2个问题:</p>
                </div>
                <div class="p1">
                    <p id="122">1) 如何根据多种miRNA相关性定义网络中的相关节点对。</p>
                </div>
                <div class="p1">
                    <p id="123">2) 如何高效地找出所有相关节点对, 并代入式 (2) 进行训练。</p>
                </div>
                <div class="p1">
                    <p id="124">对第1个问题, 存在2种方式:1) 只关注miRNA节点, 即只将关联路径连接的miRNA节点代入公式训练;2) 表1关联路径中的两两节点都具有相关性, 因此将所有每一条路径中的所有节点对都代入训练。前者只关注miRNA这一种类型的节点, 关联路径用于确定miRNA之间的相关性, 而后者考虑基因、疾病、miRNA3种节点之间多种类型的相关性, 相较于前者会产生较多相关节点对, 捕捉到更多信息。</p>
                </div>
                <div class="p1">
                    <p id="125">对第2个问题, 计算每个节点在每种关联路径中各个位置出现的概率, 根据概率采样得到节点对集合, 然后将这些节点对代入式 (2) , 并利用随机梯度下降方式训练每个节点的表示向量。计算节点出现概率的过程如算法1所示。</p>
                </div>
                <div class="p1">
                    <p id="126">算法1计算节点u出现在关联路径P中第i个位置的概率Pr (u, i|P) </p>
                </div>
                <div class="area_img" id="181">
                                <img alt="" src="Detail/GetImg?filename=images/JSJC201902026_18100.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="135">计算出Pr (u, i|P) 之后, 对网络中的相关节点对进行采样。表1中的关联路径同样重要, 因此对每个路径采样相同的次数, 实验中对每个路径采样10 000次。采样得到训练样本后, 需要将每组节点对代入式 (2) 进行优化。由于Softmax函数的计算代价较高, 为提高计算效率, 采用负采样技术 (Negative Sampling) <citation id="191" type="reference"><link href="71" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>, 并针对多源网络中的多个节点类型进行改进。负采样技术优化了式 (2) , 其思想是最大化正样本的概率而最小化随机样本的概率, 避免对所有节点都计算一次概率。其目标函数如下:</p>
                </div>
                <div class="area_img" id="136">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJC201902026_13600.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="137">对每一对相关节点对 (u, v) , 随机采样K (K=5) 个节点, 传统随机负采样会等概率对网络中的所有节点采样作为负样本, 而在多源网络中, 每次选择与节点u类型一致的节点作为负样本, 减小采样的候选集。例如, 已知 (疾病, miRNA<sub>1</sub>) 是相关节点, 则可以认为随机采样的 (疾病, miRNA<sub>2</sub>) 不存在关联。表2对比各个计算方式的计算复杂度, 包括负采样候选集的大小以及向量相乘的计算次数。</p>
                </div>
                <div class="area_img" id="138">
                                            <p class="img_tit">
                                                表2 各计算方式的采样候选集大小及计算次数
                                                
                                                &nbsp;&nbsp;
                                                <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902026_13800.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a>
                                                <a class="table downimg" data-tablename="Detail/GetImg?filename=images/JSJC201902026_13800.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">下载原表</a>
                                            </p>
                                    <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902026_13800.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <img alt="表2 各计算方式的采样候选集大小及计算次数" src="Detail/GetImg?filename=images/JSJC201902026_13800.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                    </a>

                </div>
                <div class="p1">
                    <p id="139">将所有的正负样本代入式 (3) 训练后, 得到所有miRNA的表示向量。这些表示向量可以直接用向量的余弦相似度来计算两两之间的相似性<citation id="192" type="reference"><link href="73" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>, 也可以作为特征向量应用到相关的miRNA预测任务中。通过余弦相似度计算miRNA功能相似性的公式为:</p>
                </div>
                <div class="area_img" id="140">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJC201902026_14000.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="141">其中, X<sub>u</sub>和X<sub>v</sub>分别表示miREmb学习到节点u和节点v的表示向量。式 (4) 计算2个节点在向量空间中的相似度, 以此衡量2个miRNA的功能相似性。</p>
                </div>
                <h3 id="142" name="142" class="anchor-tag">2 实验结果与分析</h3>
                <h4 class="anchor-tag" id="143" name="143">2.1 实验数据集</h4>
                <div class="p1">
                    <p id="144">本文构建多源miRNA相关网络用到的数据包括以下5个方面:</p>
                </div>
                <div class="p1">
                    <p id="145">1) miRNA-基因关系网络</p>
                </div>
                <div class="p1">
                    <p id="146">miRTarBase<citation id="193" type="reference"><link href="75" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>是一个专门收集有实验证据支持miRNA及其靶基因间关系的数据库, 并按来源分为文献、免疫印迹实验、新一代测序技术等, 共有超过360 000条miRNA-基因关系记录。本文选择其中和人类miRNA相关的18 519条miRNA-目标基因关系记录, 其中5 504条由报告基因分析和蛋白质印迹法得到的关系可信度较高, 因此在网络中边的权重设为1, 其余13 015条关系的边权重设为0.3。</p>
                </div>
                <div class="p1">
                    <p id="147">2) miRNA-表型疾病关系网络</p>
                </div>
                <div class="p1">
                    <p id="148">文献<citation id="194" type="reference">[<a class="sup">15</a>]</citation>提供99个miRNA和51种疾病间共计242条miRNA-表型疾病关系, 其源自于文献中实验验证的记录, 且在网络中的边权重均为1。</p>
                </div>
                <div class="p1">
                    <p id="149">3) 基因-疾病关系网络</p>
                </div>
                <div class="p1">
                    <p id="150">本文使用的DisGeNET<citation id="195" type="reference"><link href="79" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation>是目前最大的公开数据集, 本文提取19 714条基因-疾病关联, 每一条关系都根据来源的可靠性对应一个0～1的可信度分数, 将此分数作为构建网络中边的权重。</p>
                </div>
                <div class="p1">
                    <p id="151">4) 基因相互作用网络</p>
                </div>
                <div class="p1">
                    <p id="152">基因相互作用网络数据来源于数据库HPRD (Human Protein Reference Database) <citation id="196" type="reference"><link href="81" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>。该数据库提供目前最为广泛使用的蛋白质相互作用数据, 由于蛋白质可链接其编码基因, 因此可以看作基因-基因关系网络。HRPD包含39 122条相互作用数据, 边权重均设为1。</p>
                </div>
                <div class="p1">
                    <p id="153">5) 疾病表型相似网络</p>
                </div>
                <div class="p1">
                    <p id="154">疾病表型是指一组描述该疾病的临床症状的词汇, 目前针对人类疾病之间的研究还不是很完善, 疾病相似性可以通过对2个疾病之间症状的重叠进行量化。本文疾病相似性网络来源于MimMiner数据库<citation id="197" type="reference"><link href="83" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>, 根据文献<citation id="198" type="reference">[<a class="sup">19</a>]</citation>描述的疾病进行文本挖掘处理后得到5 081个疾病两两之间相似度的量化关系。本文选择其中相似度大于0.5的9 716条记录, 并将相似值作为网络中边的权重。</p>
                </div>
                <div class="p1">
                    <p id="155">利用上述数据集构建如图1所示的多源miRNA相关网络, 其中基因和miRNA由各自的符号表示, 疾病由OMIM数据库中的编号表示。在多源网络上运行miREmb算法, 得到该网络的网络表示向量, 并根据式 (4) 计算miRNA两两之间的余弦相似度, 以此作为miRNA的功能相似性得分。</p>
                </div>
                <h4 class="anchor-tag" id="156" name="156">2.2 对比算法</h4>
                <div class="p1">
                    <p id="157">本文实验对比算法如下:</p>
                </div>
                <div class="p1">
                    <p id="158">1) miRFunSim方法<citation id="199" type="reference"><link href="87" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>是用miRNA目标基因在整合的蛋白质相互作用网络中基因连通性来定量分析miRNA之间的功能相似性。该方法从数据库中获取miRNA目标基因数据, 对每对miRNA基于整合的蛋白质相互作用网络构建目标基因子图, 在子图上计算这2个miRNA目标基因对之间最短路径长度的平均值, 取倒数作为miRNA的功能相似性得分。</p>
                </div>
                <div class="p1">
                    <p id="159">2) DeepWalk是一种网络表示学习方法, 在网络上采样生成大量的随机游走序列, 对随机游走序列中每个局部窗口内的节点对进行概率建模, 通过最大化随机游走序列的似然概率学习节点的向量表示。该方法同等地对待网络中的所有类型节点, 并且不考虑关联路径的信息。</p>
                </div>
                <div class="p1">
                    <p id="160">3) miREmb＿single方法是本文提出的第1种方法, 在学习网络表示时只关注miRNA节点, 即只将关联路径中具有相关性的miRNA节点进行训练, 而忽略路径中的其他类型节点。</p>
                </div>
                <div class="p1">
                    <p id="161">4) miREmb方法是本文提出的第2种方法, 在采样得到关联路径之后, 将路径中所有类型的节点两两组对进行训练。</p>
                </div>
                <h4 class="anchor-tag" id="162" name="162">2.3 miRNA家族的功能相似性</h4>
                <div class="p1">
                    <p id="163">属于同一个家族的miRNA通常具有相似的miRNA序列和完全相同的miRNA目标种子区域, 因此, 同一个家族的miRNA具有相似的功能<citation id="201" type="reference"><link href="87" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>。为了评估方法计算得到的miRNA之间功能相似性得分的可靠性, 从miRBase数据库<citation id="200" type="reference"><link href="89" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>中下载miRNA的家族信息。网络中的miRNA分为3组:家族间miRNA组, 跨家族miRNA组和随机miRNA组, 分别计算3个组内的miRNA相似度, 结果如图3所示。从图3可以看出, 属于同一个家族的miRNA之间的功能相似性得分比属于随机组和跨家族组的功能相似性得分都要高, 而跨家族组的得分比随机组低。miREmb方法计算出的家族内得分要比其他方法高, 而跨家族得分则要更低。DeepWalk方法由于无法区分多源网络中不同类型节点的差异, 学习的miRNA特征向量对家族内和家族外的差异较小, 效果较差。miREmb＿single方法尽管考虑关联路径的信息, 但在训练时只利用相关的miRNA节点对而忽略其他类型节点之间的关系, miRFunSim方法只利用基因相互作用网络的信息。这2种方法和miREmb方法相比都只利用部分信息, 效果都略差于miREmb方法。miREmb方法充分利用miRNA和相关基因以及相关疾病之间的关系, 计算出的miRNA家族内得分要比其他方法高, 而跨家族得分则要更低, 效果最好。</p>
                </div>
                <div class="area_img" id="164">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902026_16400.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 各方法相似度对比结果" src="Detail/GetImg?filename=images/JSJC201902026_16400.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 各方法相似度对比结果  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902026_16400.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="165" name="165">2.4 miRNA与疾病关联</h4>
                <div class="p1">
                    <p id="166">研究miRNA之间的功能相似性的基本目的是利用已知miRNA的实验数据推测未知miRNA的功能, 而探索miRNA功能最主要的作用是预测新疾病相关的候选miRNA, 为生物学家进行生物实验准确识别疾病相关的miRNA指明方向。为对比各方法在预测疾病这一任务上的表现, 本文进行如下实验。</p>
                </div>
                <div class="p1">
                    <p id="167">本文检索了miRNet数据库<citation id="202" type="reference"><link href="91" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>, 该数据库收录19 342条文献报告的疾病-miRNA关系记录。由于该数据库中的疾病没有根据OMIM编号记录, 无法用于构建实验数据, 因此将该数据库用于实验验证。从这些疾病-miRNA关系记录中过滤实验数据集中用到的99条记录以及只和一种miRNA相关的疾病。对每一个miRNA, 从剩余的记录找出与之有共同关联疾病的miRNA, 将数量作为真实相似性评分, 同时通过各方法计算的miRNA相似分排序后作为预测排序, 采用排序常用的评价指标NDCG作为衡量方法, 其计算公式为:</p>
                </div>
                <div class="area_img" id="168">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJC201902026_16800.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="area_img" id="169">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJC201902026_16900.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="170">其中, |REL|表示与该miRNA有共同关联疾病的候选miRNA总数量, rel<sub>i</sub>表示排在i个位置的miRNA的相似性分数 (共同关联疾病的数量) 。</p>
                </div>
                <div class="p1">
                    <p id="171">对每一个miRNA能得到NDCG分数, 各方法对所有miRNA取均值后的结果如表3所示。从表3可以看出, miREmb方法的分数比miRFunSim方法和DeepWalk方法的分数都要高。结果表明, 在反映miRNA和疾病之间关系的问题上miREmb方法的效果要优于其他方法。</p>
                </div>
                <div class="area_img" id="172">
                                            <p class="img_tit">
                                                表3 4种方法对miRNA相关疾病的NDCG对比结果
                                                
                                                &nbsp;&nbsp;
                                                <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902026_17200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a>
                                                <a class="table downimg" data-tablename="Detail/GetImg?filename=images/JSJC201902026_17200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">下载原表</a>
                                            </p>
                                    <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902026_17200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <img alt="表3 4种方法对miRNA相关疾病的NDCG对比结果" src="Detail/GetImg?filename=images/JSJC201902026_17200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                    </a>

                </div>
                <h4 class="anchor-tag" id="173" name="173">2.5 未知miRNA预测与疾病关联</h4>
                <div class="p1">
                    <p id="174">多数预测疾病候选miRNA方法是基于miRNA功能相似性进行预测<citation id="203" type="reference"><link href="93" rel="bibliography" /><sup>[<a class="sup">23</a>]</sup></citation>。在实验中, 本文选择4种常见疾病, 通过miRNA功能相似性来预测这几种疾病的候选miRNA, 从而说明miREmb方法的实际应用场景。</p>
                </div>
                <div class="p1">
                    <p id="175">根据计算得到的miRNA相似性分数构建m iRNA网络 (保留相似分数大于0.5的边) , 然后用文献<citation id="204" type="reference">[<a class="sup">23</a>]</citation>提出的基于网络随机游走的预测方法来得到白血病、老年痴呆症、乳腺癌和胃癌这4种疾病各自排名前10的候选m iRNA, 如表4所示。同时, 本文分别将每种疾病以及预测得到的候选miRNA在miRNet数据库中进行检索, 将检索到的记录数列举在表4中。从表4可以看出, m iREm b方法预测出的疾病候选m iRNA大部分可以在数据库中查到相关记录, 并且排名靠前的m iRNA能查到的验证记录更多。因此, 本文验证了miREmb方法的有效性。</p>
                </div>
                <div class="area_img" id="176">
                                            <p class="img_tit">
                                                表4 miREmb算法疾病候选miRNA预测以及在miRNet数据库中的确认记录数
                                                
                                                &nbsp;&nbsp;
                                                <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902026_17600.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a>
                                                <a class="table downimg" data-tablename="Detail/GetImg?filename=images/JSJC201902026_17600.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">下载原表</a>
                                            </p>
                                    <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902026_17600.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <img alt="表4 miREmb算法疾病候选miRNA预测以及在miRNet数据库中的确认记录数" src="Detail/GetImg?filename=images/JSJC201902026_17600.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                    </a>

                </div>
                <h3 id="177" name="177" class="anchor-tag">3 结束语</h3>
                <div class="p1">
                    <p id="178">本文提出基于多源网络表示学习的miRNA功能相似性计算方法。该方法将多个来源的数据集进行融合, 包括miRNA相关的基因和表型疾病, 以及基因和疾病的相似关系, 构造miRNA相关多源网络。通过多源网络表示学习的方法将构造的网络映射到向量空间, 为网络中的每一个miRNA节点学习对应的特征向量。学习到的特征向量可以通过向量余弦相似度度量miRNA功能相似性, 也可以作为特征向量应用到miRNA相关疾病预测等任务中。实验结果表明, 该方法在区分家族内和家族外miRNA时优于DeepWalk方法。此外, 通过计算miRNA相似值, 可以准确地预测疾病的候选miRNA。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="49">
                            <a id="bibliography_1" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD15070900002085&amp;v=Mjg4ODFlWnVIeWptVUxiSUlGOFhhaEU9Tmo3QmFySzlIdGJNcG85RlpPc05ESFE4b0JNVDZUNFBRSC9pclJkR2VycVFUTW53Wg==&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[1]</b>GUZZI P H, DI M M, TAGLIAFERRI P, et al.Analysis of miRNA, mRNA, and TF interactions through networkbased methods[J].Eurasip Journal on Bioinformatics and Systems Biology, 2015 (1) :4.
                            </a>
                        </p>
                        <p id="51">
                            <a id="bibliography_2" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=DYJZ201604024&amp;v=MDk3OTJxNDlIWUlRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJuRnlqa1VML0xJVFRCZExHNEg5Zk0=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[2]</b>彭可可, 欧阳伶宣, 邓朝谦, 等.MicroRNA研究概述[J].动物医学进展, 2016, 37 (4) :110-114.
                            </a>
                        </p>
                        <p id="53">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Inferring the human microRNA functional similarity and functional network based on microRNA-associated diseases">

                                <b>[3]</b>WANG D, WANG J, LU M, et al.Inferring the human microRNA functional similarity and functional network based on microRNA-associated diseases[J].Bioinformatics, 2010, 26 (13) :1644-1650.
                            </a>
                        </p>
                        <p id="55">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=WBSMDA:within and between score for miRNA-disease association prediction">

                                <b>[4]</b>CHEN X, YAN C C, ZHANG X, et al.WBSMDA:within and between score for miRNA-disease association prediction[EB/OL].[2017-12-01].https://www.nature.com/articles/srep21106.
                            </a>
                        </p>
                        <p id="57">
                            <a id="bibliography_5" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES5EDB327039C952A748A713E2088A99D4&amp;v=MTE4NzVOYXFQUHJZaEZaK0o4QlhrN3ZoRVg0azU2U1h5WHJoSTljY09kVE02YkNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU5GaHhMbTh3S3M9TmlmT2ZiYg==&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[5]</b>LE D H.Network-based ranking methods for prediction of novel disease associated microRNAs[J].Computational Biology and Chemistry, 2015, 58:139-148.
                            </a>
                        </p>
                        <p id="59">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A novel group wisebased method for calculating human miRNA functional similarity">

                                <b>[6]</b>DING P, LUO J, LIANG C, et al.A novel group wisebased method for calculating human miRNA functional similarity[J].IEEE Access, 2017, 5 (99) :2364-2372.
                            </a>
                        </p>
                        <p id="61">
                            <a id="bibliography_7" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Implications of micro-RNA profiling for cancer diagnosis">

                                <b>[7]</b>CUMMINS J M, VELCULESCU V E.Implications of microRNA profiling for cancer diagnosis[J].Oncogene, 2006, 25 (46) :6220-6227.
                            </a>
                        </p>
                        <p id="63">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=An analysis of human microRNA and disease associations">

                                <b>[8]</b>LU Ming, ZHANG Qipeng, DENG Min, et al.An analysis of human microRNA and disease associations[J].PloS One, 2008, 3 (10) .
                            </a>
                        </p>
                        <p id="65">
                            <a id="bibliography_9" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=YCZZ200806005&amp;v=MDU2NThRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJuRnlqa1VML0xQQzdSZExHNEh0bk1xWTlGWVk=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[9]</b>侯妍妍, 应晓敏, 李伍举.microRNA计算发现方法的研究进展[J].遗传, 2008, 30 (6) :687-696.
                            </a>
                        </p>
                        <p id="67">
                            <a id="bibliography_10" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=PZKX201708003&amp;v=MTI4NjVSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUm5GeWprVUwvTE5UZkFkckc0SDliTXA0OUZaNFFLREg4NHY=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[10]</b>涂存超, 杨成, 刘知远, 等.网络表示学习综述[J].中国科学:信息科学, 2017 (8) .
                            </a>
                        </p>
                        <p id="69">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Deepwalk:online learning of social representations">

                                <b>[11]</b>PEROZZI B, AL-RFOU R, SKIENA S.DeepWalk:online learning of social representations[C]//Proceedings of the20th ACM SIGKDD International Conference on Knowledge Discovery and Data Mining.New York, USA:ACM Press, 2014:701-710.
                            </a>
                        </p>
                        <p id="71">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Distributed Representations of Words and Phrases and their Compositionality">

                                <b>[12]</b>MIKOLOV T, SUTSKEVER I, CHEN K, et al.Distributed representations of words and phrases and their compositionality[C]//Proceedings of the 26th International Conference on Neural Information Processing Systems.[S.l.]:Curran Associates Inc., 2013:3111-3119.
                            </a>
                        </p>
                        <p id="73">
                            <a id="bibliography_13" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201705024&amp;v=MTA0NTQ5SFlJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVSbkZ5amtVTC9MTHo3QmJiRzRIOWJNcW8=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[13]</b>段旭磊, 张仰森, 孙祎卓.微博文本的句向量表示及相似度计算方法研究[J].计算机工程, 2017, 43 (5) :143-148.
                            </a>
                        </p>
                        <p id="75">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=miRTarBase:a database curates experimentally validated microRNA-target interactions">

                                <b>[14]</b>HSU S D, LIN F M, WU W Y, et al.miRTarBase:a database curates experimentally validated microRNAtarget interactions[J].Nucleic Acids Research, 2011, 39:163-169.
                            </a>
                        </p>
                        <p id="77">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Similarity-based methods for potential human microRNA-disease association prediction">

                                <b>[15]</b>CHEN H, ZHANG Z.Similarity-based methods for potential human microRNA-disease association prediction[J].BMCMedical Genomics, 2013, 6 (1) :12.
                            </a>
                        </p>
                        <p id="79">
                            <a id="bibliography_16" target="_blank" href="http://scholar.cnki.net/result.aspx?q=DisGeNET:a comprehensive platform integrating information on human disease-associated genes and variants">

                                <b>[16]</b>PIERO J, BRAVO, QUERALTROSINACH N, et al.DisGeNET:a comprehensive platform integrating information on human disease-associated genes and variants[J].Nucleic Acids Research, 2017, 45 (D1) :833-839.
                            </a>
                        </p>
                        <p id="81">
                            <a id="bibliography_17" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Human Protein Reference Database--2009 update">

                                <b>[17]</b>KESHAVA PRASAD T S, GOEL R, KANDASAMY K, et al.Human protein reference database[J].Nucleic Acids Research, 2009, 37:767-772.
                            </a>
                        </p>
                        <p id="83">
                            <a id="bibliography_18" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A text-mining analysis of the human phenome">

                                <b>[18]</b>VAN D M A, BRUGGEMAN J, VRIEND G, et al.Atext-mining analysis of the human phenome[J].European Journal of Human Genetics, 2006, 14 (5) :535-542.
                            </a>
                        </p>
                        <p id="85">
                            <a id="bibliography_19" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic disorders">

                                <b>[19]</b>HAMOSH A, SCOTT A F, AMBERGER J S, et al.Online mendelian inheritance in man (OMIM) , a knowledgebase of human genes and genetic disorders[J].Nucleic Acids Research, 2005, 33 (1) :514-517.
                            </a>
                        </p>
                        <p id="87">
                            <a id="bibliography_20" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Inferring Potential microRNA-microRNA Associations Based on Targeting Propensity and Connectivity in the Context of Protein Interaction Network">

                                <b>[20]</b>SUN J, ZHOU M, YANG H, et al.Inferring potential microRNA-microRNA associations based on targeting propensity and connectivity in the context of protein interaction network.[J].PloS One, 2013, 8 (7) .
                            </a>
                        </p>
                        <p id="89">
                            <a id="bibliography_21" target="_blank" href="http://scholar.cnki.net/result.aspx?q=miRBase:the microRNA sequence database">

                                <b>[21]</b>GRIFFITHS J S.miRBase:the microRNA sequence database.[J].Methods in Molecular Biology, 2006, 342:129-138.
                            </a>
                        </p>
                        <p id="91">
                            <a id="bibliography_22" target="_blank" href="http://scholar.cnki.net/result.aspx?q=miRNet-dissecting miRNA-target interactions and functional associations through network-based visual analysis">

                                <b>[22]</b>FAN Y, SIKLENKA K, ARORA S K, et al.miRNetdissecting miRNA-target interactions and functional associations through network-based visual analysis[J].Nucleic Acids Research, 2016, 44:135-141.
                            </a>
                        </p>
                        <p id="93">
                            <a id="bibliography_23" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prediction of potential disease-associated micro RNAs based on random walk">

                                <b>[23]</b>XUAN P, HAN K, GUO Y, et al.Prediction of potential disease-associated microRNAs based on random walk[J].Bioinformatics, 2015, 31 (11) :1805-1815.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JSJC201902026" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201902026&amp;v=MDkxMTJGeWprVUwvTEx6N0JiYkc0SDlqTXJZOUhZb1FLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUm4=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
