<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637131281103868750%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJSJC201902049%26RESULT%3d1%26SIGN%3d0ns8ZZMpC7SpkeUYALph6tAjFFU%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJC201902049&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJC201902049&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201902049&amp;v=MDM4NzlHRnJDVVJMT2VaZVJuRnlqa1dyek5MejdCYmJHNEg5ak1yWTlCYllRS0RIODR2UjRUNmo1NE8zenFxQnQ=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#67" data-title="0概述 ">0概述</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#71" data-title="1 背景知识 ">1 背景知识</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#72" data-title="1.1 贝叶斯系统发育推断">1.1 贝叶斯系统发育推断</a></li>
                                                <li><a href="#77" data-title="1.2 似然估计算法">1.2 似然估计算法</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#87" data-title="2 相关工作 ">2 相关工作</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#93" data-title="3 算法设计与实现 ">3 算法设计与实现</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#94" data-title="3.1 输入数据结构重建">3.1 输入数据结构重建</a></li>
                                                <li><a href="#97" data-title="3.2 任务间并行优化方法">3.2 任务间并行优化方法</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#105" data-title="4 实验 ">4 实验</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#106" data-title="4.1 实验环境">4.1 实验环境</a></li>
                                                <li><a href="#108" data-title="4.2 实验数据">4.2 实验数据</a></li>
                                                <li><a href="#110" data-title="4.3 性能对比">4.3 性能对比</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#121" data-title="5 结束语 ">5 结束语</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#85" data-title="图1 有根树拓扑结构和树节点之间的依赖关系">图1 有根树拓扑结构和树节点之间的依赖关系</a></li>
                                                <li><a href="#89" data-title="图2 不同存储方式和并行模式建立情况">图2 不同存储方式和并行模式建立情况</a></li>
                                                <li><a href="#100" data-title="图3 匹配似然估计的GPU线程和线程块访问的层级划分">图3 匹配似然估计的GPU线程和线程块访问的层级划分</a></li>
                                                <li><a href="#104" data-title="图4 半任务间并行策略的线程分配方式">图4 半任务间并行策略的线程分配方式</a></li>
                                                <li><a href="#112" data-title="表1 不同方法分析模拟数据集的运行时间比较">表1 不同方法分析模拟数据集的运行时间比较</a></li>
                                                <li><a href="#119" data-title="图5 ta MC和tgp MC加速比的比较结果">图5 ta MC和tgp MC加速比的比较结果</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="35">


                                    <a id="bibliography_1" title="孙啸, 陆祖宏, 谢建明.生物信息学基础[M].北京:清华大学出版社, 2006." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CBBD&amp;filename=0007302102708999&amp;v=MjU4MDlPSUdCUk04enhVU21EZDlTSDduM3hFOWZidm5LcmlmWnU5dUZDdnNVNy9KS1YwUlZWMjdHYkM0SE5ETXJZaEZi&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[1]</b>
                                        孙啸, 陆祖宏, 谢建明.生物信息学基础[M].北京:清华大学出版社, 2006.
                                    </a>
                                </li>
                                <li id="37">


                                    <a id="bibliography_2" title="FITCH W M.Toward defining the course of evolution:minimum change for a specific tree topology[J].Systematic Zoology, 1971, 20 (4) :406-416." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Toward defining the course of evolution: minimum change for a specific tree topology">
                                        <b>[2]</b>
                                        FITCH W M.Toward defining the course of evolution:minimum change for a specific tree topology[J].Systematic Zoology, 1971, 20 (4) :406-416.
                                    </a>
                                </li>
                                <li id="39">


                                    <a id="bibliography_3" title="FELSENSTEIN J.Evolutionary trees from DNAsequences:a maximum likelihood approach[J].Journal of Molecular Evolution, 1981, 17 (6) :368-376." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00000361350&amp;v=MDAxMDByeG94Y01IN1I3cWVidWR0RkNIbFU3N0JJMXM9Tmo3QmFyTzRIdEhNcklsRVorNFBZM2s1ekJkaDRqOTlTWHFS&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[3]</b>
                                        FELSENSTEIN J.Evolutionary trees from DNAsequences:a maximum likelihood approach[J].Journal of Molecular Evolution, 1981, 17 (6) :368-376.
                                    </a>
                                </li>
                                <li id="41">


                                    <a id="bibliography_4" title="HASTINGS W K.Monte carlo sampling methods usingMarkov chains and their applications[J].Biometrika, 1970, 57 (1) :97-109." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Monte Carlo sampling methods using Markov chains and their applications">
                                        <b>[4]</b>
                                        HASTINGS W K.Monte carlo sampling methods usingMarkov chains and their applications[J].Biometrika, 1970, 57 (1) :97-109.
                                    </a>
                                </li>
                                <li id="43">


                                    <a id="bibliography_5" title="张舒, 褚艳利.GPU高性能运算之CUDA[M].北京:中国水利水电出版社, 2009." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CBBD&amp;filename=9787508465432001&amp;v=MTAxNTR2bktyaWZadTl1RkN2c1U3L0pLVjBSWEZxekdiYTRGdFhLcW90R1p1c1BEUk04enhVU21EZDlTSDduM3hFOWZi&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[5]</b>
                                        张舒, 褚艳利.GPU高性能运算之CUDA[M].北京:中国水利水电出版社, 2009.
                                    </a>
                                </li>
                                <li id="45">


                                    <a id="bibliography_6" title="FENG X, CAMERON K W, BUELL D A.PBPI:a high performance implementation of Bayesian phylogenetic inference[C]//Proceedings of 2006 ACM/IEEEConference on Supercomputing.Washington D.C., USA:IEEE Press, 2006:40." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=PBPI:a high performance implementation of Bayesian phylogenetic inference">
                                        <b>[6]</b>
                                        FENG X, CAMERON K W, BUELL D A.PBPI:a high performance implementation of Bayesian phylogenetic inference[C]//Proceedings of 2006 ACM/IEEEConference on Supercomputing.Washington D.C., USA:IEEE Press, 2006:40.
                                    </a>
                                </li>
                                <li id="47">


                                    <a id="bibliography_7" title="WATH R C V D, WATH E V D, CARAPELLI A, et al.Bayesian phylogeny on grid[C]//Proceedings of the 2nd International Conference on Bioinformatics Research and Development.[S.l.]:DBLP, 2008:404-416." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Bayesian phylogeny on grid">
                                        <b>[7]</b>
                                        WATH R C V D, WATH E V D, CARAPELLI A, et al.Bayesian phylogeny on grid[C]//Proceedings of the 2nd International Conference on Bioinformatics Research and Development.[S.l.]:DBLP, 2008:404-416.
                                    </a>
                                </li>
                                <li id="49">


                                    <a id="bibliography_8" title="PRATAS F, TRANCOSO P, SOUSA L, et al.Fine-grain parallelism using multi-core, Cell/BE, and GPU systems[C]//Proceedings of 2009 International Conference on Parallel Processing.Washington D.C., USA:IEEE Computer Society, 2009:9-17." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Fine-grain parallelism using multi-core,Cell/BE,and GPU systems">
                                        <b>[8]</b>
                                        PRATAS F, TRANCOSO P, SOUSA L, et al.Fine-grain parallelism using multi-core, Cell/BE, and GPU systems[C]//Proceedings of 2009 International Conference on Parallel Processing.Washington D.C., USA:IEEE Computer Society, 2009:9-17.
                                    </a>
                                </li>
                                <li id="51">


                                    <a id="bibliography_9" title="PRATAS F, SOUSA L.Applying the stream-based computing model to design hardware accelerators:a case study[C]//Proceedings of the 9th International Workshop on Embedded Computer Systems:Architectures, Modeling, and Simulation.Berlin, Germany:Springer, 2009:237-246." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Applying the Stream-Based Computing Model to Design Hardware Accelerators: A Case Study">
                                        <b>[9]</b>
                                        PRATAS F, SOUSA L.Applying the stream-based computing model to design hardware accelerators:a case study[C]//Proceedings of the 9th International Workshop on Embedded Computer Systems:Architectures, Modeling, and Simulation.Berlin, Germany:Springer, 2009:237-246.
                                    </a>
                                </li>
                                <li id="53">


                                    <a id="bibliography_10" title="ZHOU J, LIU X, STONES D S, et al.MrBayes on a graphics processing unit[J].Bioinformatics, 2011, 27 (9) :1255-1261." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=MrBayes on a Graphics Processing Unit">
                                        <b>[10]</b>
                                        ZHOU J, LIU X, STONES D S, et al.MrBayes on a graphics processing unit[J].Bioinformatics, 2011, 27 (9) :1255-1261.
                                    </a>
                                </li>
                                <li id="55">


                                    <a id="bibliography_11" title="BAO J, XIA H, ZHOU J, et al.Efficient implementation of MrBayes on multi-GPU[J].Molecular Biologyand Evolution, 2013, 30 (6) :1471-1479." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Efficient Implementation of MrBayes on Multi-GPU">
                                        <b>[11]</b>
                                        BAO J, XIA H, ZHOU J, et al.Efficient implementation of MrBayes on multi-GPU[J].Molecular Biologyand Evolution, 2013, 30 (6) :1471-1479.
                                    </a>
                                </li>
                                <li id="57">


                                    <a id="bibliography_12" title="CHAI J, SU H, WEN M, et al.Resource-efficient utilization of CPU/GPU-based heterogeneous supercomputers for Bayesian phylogenetic inference[J].Journal of Supercomputing, 2013, 66 (1) :364-380." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD13101000018526&amp;v=MDUzMjFPb0hDWDQvb0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadUh5am1VTGJJSUY4ZGFSYz1OajdCYXJLN0g5SE5yNDlGWg==&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[12]</b>
                                        CHAI J, SU H, WEN M, et al.Resource-efficient utilization of CPU/GPU-based heterogeneous supercomputers for Bayesian phylogenetic inference[J].Journal of Supercomputing, 2013, 66 (1) :364-380.
                                    </a>
                                </li>
                                <li id="59">


                                    <a id="bibliography_13" title="LING C, HAMADA T, BAI J, et al.MrBayes tgMC3:a tight GPU implementation of MrBayes[J].Plos One, 2013, 8 (4) :1-9." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Mrbayes tgMC3:A tight gpu implementation of mrbayes">
                                        <b>[13]</b>
                                        LING C, HAMADA T, BAI J, et al.MrBayes tgMC3:a tight GPU implementation of MrBayes[J].Plos One, 2013, 8 (4) :1-9.
                                    </a>
                                </li>
                                <li id="61">


                                    <a id="bibliography_14" title="LING C, HAMADA T, GAO J, et al.MrBayes tgMC3++:a high performance and resource-efficient GPU-oriented phylogenetic analysis method[J].IEEE/ACM Transactions on Computational Biology and Bioinformatics, 2016, 13 (5) :845-854." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=MrBayes tgMC3++:a high performance and resource-efficient GPU-oriented phylogenetic analysis method">
                                        <b>[14]</b>
                                        LING C, HAMADA T, GAO J, et al.MrBayes tgMC3++:a high performance and resource-efficient GPU-oriented phylogenetic analysis method[J].IEEE/ACM Transactions on Computational Biology and Bioinformatics, 2016, 13 (5) :845-854.
                                    </a>
                                </li>
                                <li id="63">


                                    <a id="bibliography_15" title="AYRES D L, DARLING A E, ZWICKL D J, et al.Beagle:an application programming interface and highperformance computing library for statistical phylogenetics[J].Systematic Biology, 2012, 61 (1) :170-173." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=BEAGLE: An Application Programming Interface and High-Performance Computing Library for Statistical Phylogenetics">
                                        <b>[15]</b>
                                        AYRES D L, DARLING A E, ZWICKL D J, et al.Beagle:an application programming interface and highperformance computing library for statistical phylogenetics[J].Systematic Biology, 2012, 61 (1) :170-173.
                                    </a>
                                </li>
                                <li id="65">


                                    <a id="bibliography_16" title="PANG S, STONES R J, REN M, et al.GPU MrBayes V3.1:MrBayes on graphics processing units for protein sequence data[J].Molecular Biology and Evolution, 2015, 32 (9) :2496-2497." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=GPU MrBayes V3.1:MrBayes on graphics processing units for protein sequence data">
                                        <b>[16]</b>
                                        PANG S, STONES R J, REN M, et al.GPU MrBayes V3.1:MrBayes on graphics processing units for protein sequence data[J].Molecular Biology and Evolution, 2015, 32 (9) :2496-2497.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JSJC" target="_blank">计算机工程</a>
                2019,45(02),296-302 DOI:10.19678/j.issn.1000-3428.0049690            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm">蛋白质系统发育分析并行计算方法研究</span>
 <span class="shoufa"></span>                                     </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%9D%8E%E6%98%93%E7%A6%85&amp;code=38477645&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">李易禅</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%87%8C%E8%AF%9A&amp;code=36391427&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">凌诚</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%8C%97%E4%BA%AC%E5%8C%96%E5%B7%A5%E5%A4%A7%E5%AD%A6%E4%BF%A1%E6%81%AF%E7%A7%91%E5%AD%A6%E4%B8%8E%E6%8A%80%E6%9C%AF%E5%AD%A6%E9%99%A2&amp;code=0211331&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">北京化工大学信息科学与技术学院</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>在目前系统发育学研究中, 多数系统发育分析工具不能在GPU架构上分析蛋白质序列。为此, 提出一种大规模系统发育分析方法 tgpMC<sup>3</sup>。以添加虚字符的形式重新构造条件似然概率矩阵, 降低由于多线程分支发散导致的时间消耗。设计粒度适中的半任务间并行策略, 增加流多处理器上活跃的线程块数量。通过简单的键值对应方法传输含有模糊状态的转移概率矩阵, 实现数据访问速度的提升。实验结果表明, 与MrBayes v3. 1. 2串行版本方法相比, 该方法最高可实现117的加速比, 与taMC<sup>3</sup>方法相比, 该方法的并行分析性能更好。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%B3%BB%E7%BB%9F%E5%8F%91%E8%82%B2%E5%88%86%E6%9E%90&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">系统发育分析;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%9D%A1%E4%BB%B6%E4%BC%BC%E7%84%B6%E6%A6%82%E7%8E%87&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">条件似然概率;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=CUDA%E7%BC%96%E7%A8%8B&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">CUDA编程;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%B9%B6%E8%A1%8C%E8%AE%A1%E7%AE%97&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">并行计算;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=MC3%E7%AE%97%E6%B3%95&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">MC3算法;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    李易禅 (1993—) , 女, 硕士研究生, 主研方向为生物信息学、并行计算;
;
                                </span>
                                <span>
                                    凌诚, 讲师、博士。E-mail: lingcheng@ mail. buct. edu. cn;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2017-12-13</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金 (61602026);</span>
                    </p>
            </div>
                    <h1>Research on Parallel Computation Method for Protein Phylogenetic Analysis</h1>
                    <h2>
                    <span>LI Yichan</span>
                    <span>LING Cheng</span>
            </h2>
                    <h2>
                    <span>Collage of Information Science and Technology, Beijing University of Chemical Technology</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>In current phylogeny studies, most phylogenetic analysis tools cannot analyze protein sequences on the GPU architecture. Therefore, a large scale phylogenetic analysis method tgpMC<sup>3</sup> is proposed. The Conditional Likelihood Probabilities (CLPs) matrix is reconstructed in the form of adding virtual characters to reduce the time consumption caused by the divergence of multi-thread branches. Parallel strategy between half tasks with moderate granularity is designed to increase the number of active thread blocks on Stream Multiprocessor (SM) . The transfer probability matrix with fuzzy state is transmitted by a simple key-value correspondence method to improve the speed of data access. Experimental results show that compared with MrBayes v3. 1. 2 serial version method, this method can achieve a maximum speedup of 117, Compared with taMC<sup>3</sup> method, the parallel analysis performance of this method is better.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=phylogenetic%20analysis&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">phylogenetic analysis;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Conditional%20Likelihood%20Probabilities%20(CLPs)%20&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Conditional Likelihood Probabilities (CLPs) ;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=CUDA%20programming&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">CUDA programming;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=parallel%20computation&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">parallel computation;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=MC3%20algorithm&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">MC3 algorithm;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                                            </p>
                                    <p><b>Received：</b> 2017-12-13</p>
                                    <p>
                                            </p>
            </div>


        <!--brief start-->
                        <h3 id="67" name="67" class="anchor-tag">0概述</h3>
                <div class="p1">
                    <p id="68">系统发育学是研究基因相关物种间进化关系的一门学科, 它提出了不同分类群之间的进化假说, 即系统发育。系统发育通常用一棵树或者分支图代表物种之间的进化关系。系统发育分析是理解进化关系的手段, 主流的系统发育分析方法有距离矩阵法<citation id="124" type="reference"><link href="35" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>、最大简约法<citation id="125" type="reference"><link href="37" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>、最大似然法<citation id="126" type="reference"><link href="39" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>和贝叶斯推理法<citation id="127" type="reference"><link href="41" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>。其中, 最大似然法和贝叶斯推理法是基于显式的演化模型估计抽样树的概率。它们的区别在于, 最大似然法采用标准的统计学方法推断概率分布, 计算特定系统发育树的似然概率, 其需要依据替换模型估计特定变换的概率。贝叶斯方法假设每棵概率树具有先验概率分布, 通过建立马尔可夫链蒙特卡罗 (Markov Chain Moute Carlo, MCMC) 抽样算法和一系列树的移动在树空间中采样, 估计树状态的后验概率。后验概率的值随着相邻样本间似然的估计、先验比和建议比的变化而变化。最后, 从这些树状态的后验分布中总结出最终的一致树。</p>
                </div>
                <div class="p1">
                    <p id="69">系统发育树的似然估计面临的最大挑战是计算的高密集。考虑到现代技术的发展趋势以及图形处理单元 (Graphic Processsing Uuit, GPU) 的广泛应用, 研究人员开始在GPU通用并行计算架构CUDA上加速基于模型的MC<sup>3</sup>方法, 并且其较CPU的大型集群<citation id="128" type="reference"><link href="43" rel="bibliography" /><link href="45" rel="bibliography" /><link href="47" rel="bibliography" /><sup>[<a class="sup">5</a>,<a class="sup">6</a>,<a class="sup">7</a>]</sup></citation>实现了更快的计算速度和更细的并行粒度。gMC<sup>3</sup><citation id="129" type="reference"><link href="49" rel="bibliography" /><link href="51" rel="bibliography" /><sup>[<a class="sup">8</a>,<a class="sup">9</a>]</sup></citation>是第一个基于GPU的MrBayes MC<sup>3</sup>方法, 该方法分配了大量GPU线程并行计算序列位点的条件似然概率 (Conditional Likelihood Probabilities, CLPs) 。在此基础上, 文献<citation id="130" type="reference">[<a class="sup">10</a>]</citation>提出n MC<sup>3</sup>方法, 增加了主机和GPU设备之间的计算并行性。文献<citation id="131" type="reference">[<a class="sup">11</a>]</citation>借鉴标准的GPU优化方法, 提出n MC<sup>3</sup>改进版本。文献<citation id="132" type="reference">[<a class="sup">12</a>]</citation>提出oMC<sup>3</sup>方法, 采用GPU和CPU同时对CLPs并行计算的方式。文献<citation id="135" type="reference">[<a class="sup">13</a>,<a class="sup">14</a>]</citation>提出2个GPU版本方法以提高并行分析的性能。最初版本的tgMC<sup>3</sup>方法通过合并GPU内核减少了内核启动次数和冗余数据的传输。另一版本tgMC<sup>3</sup>方法 (v2.0) 提出对CLPs矩阵进行转置存储, 并且采用BAEGLE负责大部分贝叶斯软件和最大似然系统的核心计算<citation id="133" type="reference"><link href="63" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation>。文献<citation id="134" type="reference">[<a class="sup">16</a>]</citation>提出ta MC<sup>3</sup>方法, 旨在实现GPU上蛋白质序列数据的高性能并行系统发育分析。</p>
                </div>
                <div class="p1">
                    <p id="70">目前, 大部分并行优化方法仅适用于核苷酸序列的系统发育分析, 只有少数能够应用于其他生物序列 (如蛋白质序列) 的分析。为此, 本文设计一种高效的蛋白质序列并行分析方法tgpMC<sup>3</sup>。为与当前先进的蛋白质序列分析方法taMC<sup>3</sup>相对比, 本文保留taMC<sup>3</sup>方法使用的数据存储模型, 同时除去能够提升性能的GPU内核或CPU端的代码优化及任何其他优化手段, 保证实验的公平性。</p>
                </div>
                <h3 id="71" name="71" class="anchor-tag">1 背景知识</h3>
                <h4 class="anchor-tag" id="72" name="72">1.1 贝叶斯系统发育推断</h4>
                <div class="p1">
                    <p id="73">基于特征的贝叶斯推断是构造系统发育树的方法之一。与其他构造树的方法相比, 贝叶斯推断具有较强的信息整合能力, 更易于处理复杂的多参数模型。系统发育推断建立在系统发育树的后验概率之上, 根据贝叶斯定理可以计算出基于数据X的第i棵系统发育树的后验概率, 如式 (1) 所示。</p>
                </div>
                <div class="area_img" id="74">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJC201902049_07400.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="75">其中, f (τ<sub>i</sub>) 表示系统发育树i的先验概率, f (X|τ<sub>i</sub>) 表示已知枝长和替换参数情况下树的似然率。在复杂的替换模型下计算数据X的边缘概率 (式 (1) 的分母) 需要对所有可能树的拓扑结构求和, 对大量的系统发育树参数积分, 其运算比较困难。</p>
                </div>
                <div class="p1">
                    <p id="76">M rBayes在边缘概率估计的难题上取得了一定的突破。它采用标准MCMC和偶联MCMC (MC<sup>3</sup>) 算法近似一棵树的后验概率, 在每个独立分析中运行n条链, 包括1条冷链和n-1条热链。其中, 选定热链与冷链的状态将会周期性地进行交换, 使冷链跃过树空间的局部低谷, 最终收敛到准确的后验密度。MrBayes MC<sup>3</sup>的核心步骤为:递归地调用树重新排列 (移动) 算法来生成新的树状态, 更新转移概率矩阵 (ti Ps) , 评估新生成树状态的似然概率以决定是否接受树的移动操作, 最后总结出系统发育树和参数的后验概率分布。通常假定具有最大后验概率的树的拓扑结构为真实树的一个点估计。在样本中观测到一棵树的概率与这棵树的后验概率近似, 所有树的后验概率之和等于1。因此, 可以将全部树的后验概率从最大到最小排列, 然后在其中选取具有最高后验概率的树的拓扑结构 (可能包括一棵或多棵) , 直到后验概率之和累计到0.95, 形成95%的置信树空间。</p>
                </div>
                <h4 class="anchor-tag" id="77" name="77">1.2 似然估计算法</h4>
                <div class="p1">
                    <p id="78">似然概率的估计是贝叶斯系统发育推断过程中最耗时的部分。在给定参数或参数函数的情况下, 系统发育树的似然概率与观测到数据的概率成比例。定义似然的关键在于选择特定的进化模型, 规定随着树分支的进化序列似然概率的变化规律。一般情况下, 系统发育分析软件默认采用广义时间可逆模型GTR。它涵盖一定比例的不变位点, 指定可变位点间速率变化服从伽马分布, 并提取4个离散的伽马参数表示4个不同的碱基替换速率。</p>
                </div>
                <div class="p1">
                    <p id="79">若待分析的数据集含有s条齐次同源序列, 每条序列包含n个位于不同片段位置 (这里定义为位点) 的残基状态。数据集可以用一个s×n的矩阵表示, 如式 (2) 所示。</p>
                </div>
                <div class="area_img" id="80">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJC201902049_08000.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="81">其中, x<sub>ij</sub>为生物序列的残基状态。令X<sub>h</sub>表示矩阵的第h列, θ代表进化参数 (如分支长度或随着树各个分支变化的碱基替代率等) , f (X<sub>h</sub>|θ) 表示物种m在某个同源位点上的碱基为X<sub>m</sub>的概率。依据式 (1) , 很容易计算系统发育的似然概率。假设序列中不同位点变化相对独立, 生成的系统发育树的似然概率便可以通过大量的位点计算获得, 如式 (3) 所示。</p>
                </div>
                <div class="area_img" id="82">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJC201902049_08200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="83">从系统发育树的拓扑结构来看, 似然概率计算由一系列自下而上的过程组成。祖先节点的似然估计需要从叶子节点开始, 将所有可能的核苷酸或氨基酸组合进行求和, 每个内部节点的条件似然估计又依赖于后代节点的CLPs和不同进化模型下的ti Ps (每个ti Ps蕴含不同的核苷酸或氨基酸的突变概率) 。以DNA序列为例, 图1描述了一棵局部有根树的拓扑结构, 图中的每一排方框代表序列连续位点残基状态的CLPs, 灰色填充的方框描述了一个位点包含的4个CLPs。</p>
                </div>
                <div class="p1">
                    <p id="84">在这里只关注单个位点s的计算。如图1所示, 根节点k有2个后代结点i和j, P (t) 表示残基状态xk经过时间t转化为xi或xj的概率, L (xk) 定义为节点k的核苷酸状态为xk的概率。k节点的条件似然概率可以通过式 (4) 中2项乘积计算得出。若预先假设4个核苷酸残基的先验概率为π, 整个树在位点s上的似然概率如式 (5) 所示。</p>
                </div>
                <div class="area_img" id="85">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902049_08500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 有根树拓扑结构和树节点之间的依赖关系" src="Detail/GetImg?filename=images/JSJC201902049_08500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 有根树拓扑结构和树节点之间的依赖关系  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902049_08500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="86">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJC201902049_08600.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <h3 id="87" name="87" class="anchor-tag">2 相关工作</h3>
                <div class="p1">
                    <p id="88">根据GPU上线程被分配的计算量不同, 研究人员提出的并行优化策略可以分为内部任务并行、任务间并行以及改进的任务间并行。不同存储方式和并行模式的建立情况如图2所示。</p>
                </div>
                <div class="area_img" id="89">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902049_08900.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 不同存储方式和并行模式建立情况" src="Detail/GetImg?filename=images/JSJC201902049_08900.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 不同存储方式和并行模式建立情况  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902049_08900.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="90">gM C<sup>3</sup>和nMC<sup>3</sup>方法采用了内部任务并行策略。以DNA序列16个位点间的并行计算为例, 如图2 (a) 所示, 该策略令一个线程估计一个CLP, 一个半线程束 (half-warp) 估计一个位点的全部CLPs。由于连续线程访问的内存地址连续, 对一个位点16个CLPs的访问可以合并成一个事务。然而, 内部任务并行策略加载ti Ps的开销过大, 它启动了大量的线程, 可能导致占满流多处理器 (Stream Multiprocessor, SM) 。</p>
                </div>
                <div class="p1">
                    <p id="91">如图2 (b) 所示, aMC<sup>3</sup>方法设计了任务间并行策略, 令一个half-warp的16个线程按行估计16个位点的CLPs。通过设置线程块 (TB) 中的线程数等于访问的ti Ps个数, 防止线程级发散。此外, 令每个线程分配更多的计算量, 能够有效减少调用的TB个数, 同时避免重复加载ti Ps矩阵。该并行策略的缺点是不具有拓展性, 一旦进化速率与残基状态数改变, 模型便不能发挥效力。</p>
                </div>
                <div class="p1">
                    <p id="92">如图2 (c) 所示, n MC<sup>3</sup> (v2.1.1) 方法在任务间并行的基础上提出改进的任务间并行策略, 令一个half-w arp线程按列计算16个位点的CLPs。该并行策略的实质是令一个harf-warp纵向访问以行主顺序 (M11、M12、M13…) 存储的矩阵元素。从数据的存储角度出发, 基于行主序存储的方法在实现特定的GPU优化时, 需要设计额外的代码、内存资源和分配计算等以达到高速访问目的。</p>
                </div>
                <h3 id="93" name="93" class="anchor-tag">3 算法设计与实现</h3>
                <h4 class="anchor-tag" id="94" name="94">3.1 输入数据结构重建</h4>
                <div class="p1">
                    <p id="95">在典型的并行系统中, CUDA设备可调度执行的最小并行单元是32个线程 (一个warp) , 成百上千的线程以warp为单位排队等待执行。为挖掘GPU潜在的运算性能, 无论采用哪种并行方法, TB中的线程数都应设定为一个warp线程数的整数倍。taM C<sup>3</sup>是当前最先进的蛋白质并行分析方法, 它设置每个TB包含80个线程, 总共调用的线程块数量等于估计的位点数除以调用的线程数。从执行调度的角度来看, 80个线程可在3个warp (w<sub>1</sub>, w<sub>2</sub>, w<sub>3</sub>) 中执行, 计算一个位点80个CLPs (4×20) 。其中, w<sub>1</sub>和w<sub>2</sub>包含32个线程, w<sub>3</sub>包含16个线程。当SM分配这些warps并行执行相同的指令时, 填充不足的w<sub>3</sub>就会浪费一定的计算资源, 降低计算效率为应有效率的50% (16/32) 。</p>
                </div>
                <div class="p1">
                    <p id="96">针对上述问题, 本文tgpMC<sup>3</sup>方法在似然估计之前, 对输入数据进行预处理。通过对原始的齐次位点末端填充虚字符, 使得序列总长度等于64的倍数。同时, 根据定义的位点数变化, 对涉及系统发育树似然概率估计的其他进化参数 (例如子结点的残基状态和CLPs) 进行重建。鉴于延迟隐藏和硬件占用率方面的考虑, tgpMC<sup>3</sup>设置每个TB包含256个线程。一个TB中的线程可在8个完整的w arps中调用执行。与设置每个TB包含80个线程的taMC<sup>3</sup>方法相比, tgpMC<sup>3</sup>在运行过程中SM上活跃的warps数量更多, 占用率也更高。由于tgpM C<sup>3</sup>不需要判断序列长度和调用线程数量之间的关系, 降低了代码分支成本, 且负责并行估计CLPs部分的内核更易于维护。</p>
                </div>
                <h4 class="anchor-tag" id="97" name="97">3.2 任务间并行优化方法</h4>
                <div class="p1">
                    <p id="98">为提高似然估计过程中参数的访问效率, 本文采用列主序存储方式, 将蛋白质序列中每个位点状态的全部CLPs存储成一列, 一组连续位点按照物理内存地址从低到高的顺序逐列存储。为匹配参数存储的特点, 本文设计一种新的并行策略———半任务间并行策略, 以实现高效多线程计算。为便于说明, 本文给出一些符号并赋予不同的含义。L表示蛋白质数据集中齐次位点的长度。Φ表示模型下位点间替换速率的数量, 设置为4, 其中, Φ<sub>i</sub>为第i个替换速率, i∈{1, 2, 3, 4}。Ω表示在替换速率固定的条件下, 每一个位点需要计算的CLPs的数量, 即为一个位点包含的残基数20。Ψ表示每个TB处理的位点数量, 固定设为64。</p>
                </div>
                <div class="p1">
                    <p id="99">如图3所示, 本文对似然计算的步骤进行如下3个层次的划分。</p>
                </div>
                <div class="area_img" id="100">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902049_10000.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 匹配似然估计的GPU线程和线程块访问的层级划分" src="Detail/GetImg?filename=images/JSJC201902049_10000.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 匹配似然估计的GPU线程和线程块访问的层级划分  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902049_10000.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="101">第1层描述了序列的分组和重建过程。如第3.1节所述, tgpMC<sup>3</sup>将齐次序列以64个位点为单位分成多个组, 在最后一组齐次位点末端填充虚字符, 使整条序列的位点数目L变为64的整数倍。</p>
                </div>
                <div class="p1">
                    <p id="102">第2层体现了似然概率估计之前由全局存储器加载ti Ps矩阵的过程。加载的ti Ps总数与计算的位点数无关, 固定为1 600 (即4个位点间替换速率×大小为400的氨基酸转移概率矩阵) 。为增加线程间的ti Ps重用性, 减少不必要的时间损耗, 本文采用GPU共享内存和寄存器来存储计算参数。对于至少拥有一个叶子结点的内结点, 线程只需从共享存储器中加载ti Ps, 并从寄存器直接读取其叶子结点的CLPs进行内结点的似然计算。由此, 每组位点的估计数目等于5 120 (4×20×64) 。</p>
                </div>
                <div class="p1">
                    <p id="103">第3层表示线程对分组后位点CLPs的并行估计过程。一个TB中线程的分布以及计算一个位点时线程的分配情况如图4所示。令一个64×4的矩阵表示一个TB中的线程组成, blkx和blky分别表示线程块行和列的维度, tid＿x和tid＿y表示线程块行列维度上的索引。根据替换率属性的不同, 每组位点的5 120个似然估计 (即Φ<sub>1</sub>到Φ<sub>4</sub>) 被归为4个子分区。对于每个子分区, 本文采取最直接的方式———垂直划分, 调用一个TB中的64个线程同时估计该分区内的1 280个CLPs。如图4 (a) 所示, tid＿x和tid＿y对应位点的索引和Φ值。为匹配多个线程垂直估计一个位点的访问方式, 半任务间并行策略采用适中的并行粒度, 为每个线程分配20个CLPs的计算任务, 如图4 (b) 所示。</p>
                </div>
                <div class="area_img" id="104">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902049_10400.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 半任务间并行策略的线程分配方式" src="Detail/GetImg?filename=images/JSJC201902049_10400.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 半任务间并行策略的线程分配方式  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902049_10400.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h3 id="105" name="105" class="anchor-tag">4 实验</h3>
                <h4 class="anchor-tag" id="106" name="106">4.1 实验环境</h4>
                <div class="p1">
                    <p id="107">基于MrBayes (v3.1.2) 的方法, 如tgMC<sup>3</sup> (v2.0) 、a MC<sup>3</sup>、n MC<sup>3</sup> (v2.1.1) 和oMC<sup>3</sup> (v1.0) 等均成功实现了GPU异构平台上核苷酸序列的并行分析。这些方法的原理和性能差异具体可参考文献<citation id="136" type="reference">[<a class="sup">14</a>]</citation>。本文主要针对tgpMC<sup>3</sup>与ta MC<sup>3</sup>2种蛋白质序列并行分析方法进行性能对比。2种方法采用的数据结构均为单精度CLFlt型和双精度MrFlt型, 在估计CLPs的过程中均支持扩展函数。为保证实验的标准性和公平性, 所有的操作均在天河一号国家超级计算机的2个分布式节点上进行。该集群上每个计算节点配备8个英特尔 (R) 至强 (R) X7550 CPU, 一块Tesla K20显卡, 内存容量26 GB。在异构集群上, 本文使用gcc (v4.4.7) -O3编译CPU端代码, 同时使用CUDA Toolkit (v7.5) 编译GPU端CUDA代码。</p>
                </div>
                <h4 class="anchor-tag" id="108" name="108">4.2 实验数据</h4>
                <div class="p1">
                    <p id="109">本文实验使用15组由seq-gen v1.3.3生成的模拟数据集, 每组数据集都有固定50个物种, 以及不同的位点个数。对于数据集i (i∈{1, 2, …, 15}) 来说, 每个物种都有1 000×i的氨基酸残基。</p>
                </div>
                <h4 class="anchor-tag" id="110" name="110">4.3 性能对比</h4>
                <div class="p1">
                    <p id="111">本节对比taMC<sup>3</sup>和tgpMC<sup>3</sup>方法在不同环境配置下的运行情况。实验环境为:1) 一个CPU处理核心串行执行的Mr Bayes MC<sup>3</sup>;2) 一个Tesla K20 GPU处理核心并行执行的taMC<sup>3</sup>;3) 一个Tesla K20 GPU处理核心并行执行的tgpMC<sup>3</sup>;4) 一个Tesla K20GPU处理核心并行执行的tgpM C<sup>3</sup>;5) 2个Tesla K20GPU处理核心并行执行的tgpM C<sup>3</sup>。使用上述不同方法分析15个模拟数据集所用的时间对比如表1所示。从表1可以看出, tgpMC<sup>3</sup>在所有测试中的运行时间均少于taMC<sup>3</sup>, 相对MC<sup>3</sup>串行版本的性能提升效果也更加明显。</p>
                </div>
                <div class="area_img" id="112">
                                            <p class="img_tit">
                                                表1 不同方法分析模拟数据集的运行时间比较
                                                
                                                &nbsp;&nbsp;
                                                <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902049_11200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a>
                                                <a class="table downimg" data-tablename="Detail/GetImg?filename=images/JSJC201902049_11200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">下载原表</a>
                                            </p>
                                    <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902049_11200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <img alt="表1 不同方法分析模拟数据集的运行时间比较" src="Detail/GetImg?filename=images/JSJC201902049_11200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                    </a>

                </div>
                <div class="p1">
                    <p id="113">加速比是衡量并行算法性能的重要指标之一。在描述taMC<sup>3</sup>和tgpMC<sup>3</sup>方法的加速比之前, 本文做如下的定义:</p>
                </div>
                <div class="p1">
                    <p id="114">1) X-speedup:1个或2个GPU执行的tgpM C<sup>3</sup>相对串行版本Mr Bayes MC<sup>3</sup>执行速度快的倍数。</p>
                </div>
                <div class="p1">
                    <p id="115">2) T-speedup:1个或2个GPU执行的taM C<sup>3</sup>相对串行版本Mr Bayes MC<sup>3</sup>执行速度快的倍数。</p>
                </div>
                <div class="p1">
                    <p id="116">3) X-T speedup:1个或2个GPU执行的tgpM C<sup>3</sup>相对taMC<sup>3</sup>方法执行速度快的倍数。</p>
                </div>
                <div class="p1">
                    <p id="117">对于X-speedup, 在1个和2个Tesla K20 GPU上执行的ta MC<sup>3</sup>方法, 相对串行版本MrBayes MC<sup>3</sup>分别实现10.2～19.3和22.7～40.3的加速比。在1个GPU上执行的tgpMC<sup>3</sup>相对串行版本MrBayes MC<sup>3</sup>的加速比大约为24.1～51.4。当该方法扩展到2个GPU时, 获得的加速比为46.8～117.0。ta MC<sup>3</sup>和tgpMC<sup>3</sup>方法的加速比都随着采用的GPU数量线性增加。对于X-T speedup, 在1个GPU上执行的tgpMC<sup>3</sup>大约是ta MC<sup>3</sup>运行速度的2.2倍～2.6倍, 在2个GPU上执行的tgpMC<sup>3</sup>大约是ta MC<sup>3</sup>运行速度的2.5倍～2.6倍。在任何情况下, tgpMC<sup>3</sup>加速效果都是最为显著的。</p>
                </div>
                <div class="p1">
                    <p id="118">扩展性是衡量并行算法的另一个重要指标, 用来衡量在计算节点或齐次长度, 即每个数据集拥有的特有位点数量作为变量时加速比的变化情况, 以考量并行方法解决大规模问题的能力。本文做了5组实验, 并记录了2种方法在不同配置下分析固定种群数量, 不同齐次长度的模拟数据集时加速比的变化趋势, 如图5所示。在单个计算节点上, taM C<sup>3</sup>的加速比数值相对稳定, 随着序列长度不断增长, 加速比可以看到略微增长, 最高值达到了40.3。相比之下, tgpM C<sup>3</sup>的加速比曲线一直位于taM C<sup>3</sup>之上, 并且随着序列长度由1 000增加到5 000, tgpM C<sup>3</sup>的加速比出现显著增长。</p>
                </div>
                <div class="area_img" id="119">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201902049_11900.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图5 ta MC3和tgp MC3加速比的比较结果" src="Detail/GetImg?filename=images/JSJC201902049_11900.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图5 ta MC<sup>3</sup>和tgp MC<sup>3</sup>加速比的比较结果  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201902049_11900.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="120">当序列的长度超过6 000时, 加速比的提升相对变缓, 在相对平滑的变化中实现小幅增长。在到达峰值时, tgpMC<sup>3</sup>的加速比为117。由于在2个计算节点上执行2种方法所得到的加速比变化趋势与一个节点的情况相同, 此处不再赘述。通过单个GPU与2个GPU上的方法对比结果可知, tgpMC<sup>3</sup>的可扩展性更为出色。</p>
                </div>
                <h3 id="121" name="121" class="anchor-tag">5 结束语</h3>
                <div class="p1">
                    <p id="122">本文围绕蛋白质序列的高性能并行分析, 提出tgM C<sup>3</sup>方法。在并行策略方面, 设计并采用半任务间并行模型, 大大增加GPU硬件可调用的活跃线程数量。为保证输入序列总长度达到64的整数倍, 采用列主序的存储方式将数据存储在共享内存和寄存器中, 不仅消除了条件分支分散的成本, 而且减少了GPU代码级别的优化。实验结果表明, tgpM C<sup>3</sup>方法具备的良好的并行分析能力, 在所有的测试中, tgpM C<sup>3</sup>的运算速度均快于taMC<sup>3</sup>, 在天河一号超级计算机1个计算节点上的2个NVIDIA Tesla K20GPU上分析15组模拟数据时, tgpM C<sup>3</sup>相对taMC<sup>3</sup>取得2.2～2.6的加速比, 当方法扩展到2个分布式计算节点时, 获得2.5～2.6的加速比。</p>
                </div>
                <div class="p1">
                    <p id="123">本文提出的并行框架能够广泛应用于生物信息学领域, 同时具有良好的计算效率、可扩展性和易于维护性。下一步考虑将tgpMC<sup>3</sup>方法扩展到分布式集群更多的GPU节点上执行, 并将该方法移植到其他系统发育分析工具 (如BEAST) , 以提高似然计算的效率。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="35">
                            <a id="bibliography_1" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CBBD&amp;filename=0007302102708999&amp;v=MjkxMjc0SE5ETXJZaEZiT0lHQlJNOHp4VVNtRGQ5U0g3bjN4RTlmYnZuS3JpZlp1OXVGQ3ZzVTcvSktWMFJWVjI3R2JD&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[1]</b>孙啸, 陆祖宏, 谢建明.生物信息学基础[M].北京:清华大学出版社, 2006.
                            </a>
                        </p>
                        <p id="37">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Toward defining the course of evolution: minimum change for a specific tree topology">

                                <b>[2]</b>FITCH W M.Toward defining the course of evolution:minimum change for a specific tree topology[J].Systematic Zoology, 1971, 20 (4) :406-416.
                            </a>
                        </p>
                        <p id="39">
                            <a id="bibliography_3" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00000361350&amp;v=MTc5NjRSN3FlYnVkdEZDSGxVNzdCSTFzPU5qN0Jhck80SHRITXJJbEVaKzRQWTNrNXpCZGg0ajk5U1hxUnJ4b3hjTUg3&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[3]</b>FELSENSTEIN J.Evolutionary trees from DNAsequences:a maximum likelihood approach[J].Journal of Molecular Evolution, 1981, 17 (6) :368-376.
                            </a>
                        </p>
                        <p id="41">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Monte Carlo sampling methods using Markov chains and their applications">

                                <b>[4]</b>HASTINGS W K.Monte carlo sampling methods usingMarkov chains and their applications[J].Biometrika, 1970, 57 (1) :97-109.
                            </a>
                        </p>
                        <p id="43">
                            <a id="bibliography_5" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CBBD&amp;filename=9787508465432001&amp;v=MTU3NjJzUERSTTh6eFVTbURkOVNIN24zeEU5ZmJ2bktyaWZadTl1RkN2c1U3L0pLVjBSWEZxekdiYTRGdFhLcW90R1p1&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[5]</b>张舒, 褚艳利.GPU高性能运算之CUDA[M].北京:中国水利水电出版社, 2009.
                            </a>
                        </p>
                        <p id="45">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=PBPI:a high performance implementation of Bayesian phylogenetic inference">

                                <b>[6]</b>FENG X, CAMERON K W, BUELL D A.PBPI:a high performance implementation of Bayesian phylogenetic inference[C]//Proceedings of 2006 ACM/IEEEConference on Supercomputing.Washington D.C., USA:IEEE Press, 2006:40.
                            </a>
                        </p>
                        <p id="47">
                            <a id="bibliography_7" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Bayesian phylogeny on grid">

                                <b>[7]</b>WATH R C V D, WATH E V D, CARAPELLI A, et al.Bayesian phylogeny on grid[C]//Proceedings of the 2nd International Conference on Bioinformatics Research and Development.[S.l.]:DBLP, 2008:404-416.
                            </a>
                        </p>
                        <p id="49">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Fine-grain parallelism using multi-core,Cell/BE,and GPU systems">

                                <b>[8]</b>PRATAS F, TRANCOSO P, SOUSA L, et al.Fine-grain parallelism using multi-core, Cell/BE, and GPU systems[C]//Proceedings of 2009 International Conference on Parallel Processing.Washington D.C., USA:IEEE Computer Society, 2009:9-17.
                            </a>
                        </p>
                        <p id="51">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Applying the Stream-Based Computing Model to Design Hardware Accelerators: A Case Study">

                                <b>[9]</b>PRATAS F, SOUSA L.Applying the stream-based computing model to design hardware accelerators:a case study[C]//Proceedings of the 9th International Workshop on Embedded Computer Systems:Architectures, Modeling, and Simulation.Berlin, Germany:Springer, 2009:237-246.
                            </a>
                        </p>
                        <p id="53">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=MrBayes on a Graphics Processing Unit">

                                <b>[10]</b>ZHOU J, LIU X, STONES D S, et al.MrBayes on a graphics processing unit[J].Bioinformatics, 2011, 27 (9) :1255-1261.
                            </a>
                        </p>
                        <p id="55">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Efficient Implementation of MrBayes on Multi-GPU">

                                <b>[11]</b>BAO J, XIA H, ZHOU J, et al.Efficient implementation of MrBayes on multi-GPU[J].Molecular Biologyand Evolution, 2013, 30 (6) :1471-1479.
                            </a>
                        </p>
                        <p id="57">
                            <a id="bibliography_12" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD13101000018526&amp;v=MDU1MTc5SE5yNDlGWk9vSENYNC9vQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMYklJRjhkYVJjPU5qN0Jhcks3SA==&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[12]</b>CHAI J, SU H, WEN M, et al.Resource-efficient utilization of CPU/GPU-based heterogeneous supercomputers for Bayesian phylogenetic inference[J].Journal of Supercomputing, 2013, 66 (1) :364-380.
                            </a>
                        </p>
                        <p id="59">
                            <a id="bibliography_13" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Mrbayes tgMC3:A tight gpu implementation of mrbayes">

                                <b>[13]</b>LING C, HAMADA T, BAI J, et al.MrBayes tgMC3:a tight GPU implementation of MrBayes[J].Plos One, 2013, 8 (4) :1-9.
                            </a>
                        </p>
                        <p id="61">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=MrBayes tgMC3++:a high performance and resource-efficient GPU-oriented phylogenetic analysis method">

                                <b>[14]</b>LING C, HAMADA T, GAO J, et al.MrBayes tgMC3++:a high performance and resource-efficient GPU-oriented phylogenetic analysis method[J].IEEE/ACM Transactions on Computational Biology and Bioinformatics, 2016, 13 (5) :845-854.
                            </a>
                        </p>
                        <p id="63">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=BEAGLE: An Application Programming Interface and High-Performance Computing Library for Statistical Phylogenetics">

                                <b>[15]</b>AYRES D L, DARLING A E, ZWICKL D J, et al.Beagle:an application programming interface and highperformance computing library for statistical phylogenetics[J].Systematic Biology, 2012, 61 (1) :170-173.
                            </a>
                        </p>
                        <p id="65">
                            <a id="bibliography_16" target="_blank" href="http://scholar.cnki.net/result.aspx?q=GPU MrBayes V3.1:MrBayes on graphics processing units for protein sequence data">

                                <b>[16]</b>PANG S, STONES R J, REN M, et al.GPU MrBayes V3.1:MrBayes on graphics processing units for protein sequence data[J].Molecular Biology and Evolution, 2015, 32 (9) :2496-2497.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JSJC201902049" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201902049&amp;v=MDM4NzlHRnJDVVJMT2VaZVJuRnlqa1dyek5MejdCYmJHNEg5ak1yWTlCYllRS0RIODR2UjRUNmo1NE8zenFxQnQ=&amp;uid=WEEvREcwSlJHSldRa1FhcEE0QVN2K0VEcmE1Yy9WeVdIeHJGV1pCK3VXND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
