<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637128889861395000%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJSJC201908012%26RESULT%3d1%26SIGN%3dhe0sL2nfgay3xFCUvtTQ3mMC7sg%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJC201908012&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJC201908012&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201908012&amp;v=MDUwMTdCdEdGckNVUkxPZVplUnFGeS9nVWI3UEx6N0JiYkc0SDlqTXA0OUVab1FLREg4NHZSNFQ2ajU0TzN6cXE=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#51" data-title="0 概述 ">0 概述</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#55" data-title="1 相关工作 ">1 相关工作</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#56" data-title="1.1 SNP选择">1.1 SNP选择</a></li>
                                                <li><a href="#58" data-title="1.2 模糊聚类算法">1.2 模糊聚类算法</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#60" data-title="2 基于改进FCM算法的SNP选择 ">2 基于改进FCM算法的SNP选择</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#61" data-title="2.1 FCM聚类算法原理">2.1 FCM聚类算法原理</a></li>
                                                <li><a href="#68" data-title="2.2 改进的模糊聚类算法GN-FCM">2.2 改进的模糊聚类算法GN-FCM</a></li>
                                                <li><a href="#137" data-title="2.3 GN-FCM在SNP选择中的应用">2.3 GN-FCM在SNP选择中的应用</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#164" data-title="3 实验 ">3 实验</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#165" data-title="3.1 实验环境与数据">3.1 实验环境与数据</a></li>
                                                <li><a href="#169" data-title="3.2 实验评价指标">3.2 实验评价指标</a></li>
                                                <li><a href="#182" data-title="3.3 数据预处理">3.3 数据预处理</a></li>
                                                <li><a href="#185" data-title="3.4 实验结果与分析">3.4 实验结果与分析</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#207" data-title="4 结束语 ">4 结束语</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#139" data-title="&lt;b&gt;图1 SNP选择流程&lt;/b&gt;"><b>图1 SNP选择流程</b></a></li>
                                                <li><a href="#163" data-title="&lt;b&gt;图2 特征选择流程&lt;/b&gt;"><b>图2 特征选择流程</b></a></li>
                                                <li><a href="#168" data-title="&lt;b&gt;表1 数据集描述&lt;/b&gt;"><b>表1 数据集描述</b></a></li>
                                                <li><a href="#233" data-title="图3 混淆矩阵">图3 混淆矩阵</a></li>
                                                <li><a href="#184" data-title="&lt;b&gt;图4 数据集G1000的卡方检验结果&lt;/b&gt;"><b>图4 数据集G1000的卡方检验结果</b></a></li>
                                                <li><a href="#190" data-title="&lt;b&gt;图5 在数据集G1000上的迭代次数比较&lt;/b&gt;"><b>图5 在数据集G1000上的迭代次数比较</b></a></li>
                                                <li><a href="#191" data-title="&lt;b&gt;图6 在数据集E144上的迭代次数比较&lt;/b&gt;"><b>图6 在数据集E144上的迭代次数比较</b></a></li>
                                                <li><a href="#194" data-title="&lt;b&gt;图7 在数据集G1000上的迭代时间比较&lt;/b&gt;"><b>图7 在数据集G1000上的迭代时间比较</b></a></li>
                                                <li><a href="#195" data-title="&lt;b&gt;图8 在数据集E144上的迭代时间比较&lt;/b&gt;"><b>图8 在数据集E144上的迭代时间比较</b></a></li>
                                                <li><a href="#198" data-title="&lt;b&gt;表2 3种聚类方法在数据集E114上的聚类性能&lt;/b&gt;"><b>表2 3种聚类方法在数据集E114上的聚类性能</b></a></li>
                                                <li><a href="#199" data-title="&lt;b&gt;表3 3种聚类方法在数据集E144上的聚类性能&lt;/b&gt;"><b>表3 3种聚类方法在数据集E144上的聚类性能</b></a></li>
                                                <li><a href="#203" data-title="&lt;b&gt;表4 在每个簇中选择的SNP数目&lt;/b&gt;"><b>表4 在每个簇中选择的SNP数目</b></a></li>
                                                <li><a href="#206" data-title="&lt;b&gt;表5 基于不同分类器的分类结果评价&lt;/b&gt;"><b>表5 基于不同分类器的分类结果评价</b></a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="234">


                                    <a id="bibliography_1" title=" RIPKE S, SANDERS A R, KENDLER K S, et al.The schizophrenia psychiatric genome-wide association study (GWAS) consortium genome-wide association study identifies five new schizophrenia loci[J].Nature Genetics, 2011, 42 (1) :36-44." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Genome-wide association study identifies five new schizophrenia loci">
                                        <b>[1]</b>
                                         RIPKE S, SANDERS A R, KENDLER K S, et al.The schizophrenia psychiatric genome-wide association study (GWAS) consortium genome-wide association study identifies five new schizophrenia loci[J].Nature Genetics, 2011, 42 (1) :36-44.
                                    </a>
                                </li>
                                <li id="236">


                                    <a id="bibliography_2" title=" KATE M A, STEPHEN E, GISELA O.Genetics of rheumatoid arthritis:GWAS and beyond[J].Open Access Rheumatology:Research and Reviews, 2011, 3:31-46." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJDP&amp;filename=SJDP120724001467&amp;v=MjMyNzNidm5LcmlmWnU5dUZDdmhVN2pOSWw4VE5pZlBmcks2SHRiT3E0OUZaZThKQ3hNOHp4VVNtRGQ5U0g3bjN4RTlm&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[2]</b>
                                         KATE M A, STEPHEN E, GISELA O.Genetics of rheumatoid arthritis:GWAS and beyond[J].Open Access Rheumatology:Research and Reviews, 2011, 3:31-46.
                                    </a>
                                </li>
                                <li id="238">


                                    <a id="bibliography_3" >
                                        <b>[3]</b>
                                     王巍.基因组关联研究中Tag SNP选择算法优化的研究[D].北京:北京大学, 2008.</a>
                                </li>
                                <li id="240">


                                    <a id="bibliography_4" title=" CONSOLE L, PICARDI C, RIBAUDO M.Process algebras for systems diagnosis[J].Artificial Intelligence, 2002, 142 (1) :19-51." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011702006871&amp;v=MDUzNzVPZmJLN0h0RE5xSTlIWk9zSkJIczRvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMdklKMXNXYXhVPU5pZg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[4]</b>
                                         CONSOLE L, PICARDI C, RIBAUDO M.Process algebras for systems diagnosis[J].Artificial Intelligence, 2002, 142 (1) :19-51.
                                    </a>
                                </li>
                                <li id="242">


                                    <a id="bibliography_5" title=" LI Xiong, LIAO Bo, CAI Lijun, et al.Informative SNPs selection based on two-locus and multilocus linkage disequilibrium:criteria of max-correlation and min-redundancy[J].IEEE/ACM Transactions on Computational Biology and Bioinformatics, 2013, 10 (3) :688-695." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM9D0BFDA045FB346D2A93BD2C903C14F4&amp;v=MTY3OTV2RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOeGh3NzI5d2E4PU5pZklZN3JNSHFPNjIvNUZZTzU1Zm44OXlXSVJtelorT2d2ZzN4czFlc0dWUWN5YkNPTg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[5]</b>
                                         LI Xiong, LIAO Bo, CAI Lijun, et al.Informative SNPs selection based on two-locus and multilocus linkage disequilibrium:criteria of max-correlation and min-redundancy[J].IEEE/ACM Transactions on Computational Biology and Bioinformatics, 2013, 10 (3) :688-695.
                                    </a>
                                </li>
                                <li id="244">


                                    <a id="bibliography_6" title=" ROBNIKŠIKONJA M, KONONENKO I.Theoretical and empirical analysis of ReliefF and RReliefF[J].Machine Learning, 2003, 53 (1/2) :23-69." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00001340690&amp;v=MTkwMzg5OVNYcVJyeG94Y01IN1I3cWVidWR0RkN6bFZMcktJVms9Tmo3QmFyTzRIdEhOckl0Rll1SVBZM2s1ekJkaDRq&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[6]</b>
                                         ROBNIKŠIKONJA M, KONONENKO I.Theoretical and empirical analysis of ReliefF and RReliefF[J].Machine Learning, 2003, 53 (1/2) :23-69.
                                    </a>
                                </li>
                                <li id="246">


                                    <a id="bibliography_7" title=" ALZUBI R, RAMZAN N, ALZOUBI H, et al.A hybrid feature selection method for complex diseases SNPs[J].IEEE Access, 2017, 6:1292-1301." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A hybrid feature selection method for complex diseases SNPs">
                                        <b>[7]</b>
                                         ALZUBI R, RAMZAN N, ALZOUBI H, et al.A hybrid feature selection method for complex diseases SNPs[J].IEEE Access, 2017, 6:1292-1301.
                                    </a>
                                </li>
                                <li id="248">


                                    <a id="bibliography_8" title=" ILHAN I, TEZEL G.Tag SNP selection using clonal selection and majority voting algorithms[J].International Journal of Data Mining and Bioinformatics, 2016, 16 (4) :290-311." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCW&amp;filename=SJCW9DA6838C92FEA77E257ADE92EBAF25AB&amp;v=MjYzMDc5d2E4PU5pZkllYnJNYjlmRXJJYzJiZWw1ZVEwK3lHTVI3emdNUEFycnJtZEhDTVNXUU12dENPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU54aHc3Mg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[8]</b>
                                         ILHAN I, TEZEL G.Tag SNP selection using clonal selection and majority voting algorithms[J].International Journal of Data Mining and Bioinformatics, 2016, 16 (4) :290-311.
                                    </a>
                                </li>
                                <li id="250">


                                    <a id="bibliography_9" title=" ELANDS R J, SIMONS C C, RIEMENSCHNEIDER M, et al.A systematic SNP selection approach to identify mechanisms underlying disease aetiology:linking height to post-menopausal breast and colorectal cancer risk[EB/OL]. (2017-01-24) [2018-07-01].http://europepmc.org/backend/ptpmcrend er.fcgi?accid=PMC5259777&amp;amp; blobtype=pdf." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A systematic SNP selection approach to identify mechanisms underlying disease aetiology:linking height to post-menopausal breast and colorectal cancer risk">
                                        <b>[9]</b>
                                         ELANDS R J, SIMONS C C, RIEMENSCHNEIDER M, et al.A systematic SNP selection approach to identify mechanisms underlying disease aetiology:linking height to post-menopausal breast and colorectal cancer risk[EB/OL]. (2017-01-24) [2018-07-01].http://europepmc.org/backend/ptpmcrend er.fcgi?accid=PMC5259777&amp;amp; blobtype=pdf.
                                    </a>
                                </li>
                                <li id="252">


                                    <a id="bibliography_10" title=" LIMPITI T, INTARAPANICH A, TONGSIMA S.PCA-based informative SNP selection for analyzing population structure[C]//Proceedings of the 7th International Con-ference on Computational Systems-Biology and Bioinfor-matics.New York, USA:ACM Press, 2016:36-40." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=PCA-based informative SNP selection for analyzing population structure">
                                        <b>[10]</b>
                                         LIMPITI T, INTARAPANICH A, TONGSIMA S.PCA-based informative SNP selection for analyzing population structure[C]//Proceedings of the 7th International Con-ference on Computational Systems-Biology and Bioinfor-matics.New York, USA:ACM Press, 2016:36-40.
                                    </a>
                                </li>
                                <li id="254">


                                    <a id="bibliography_11" title=" SOLLERO B P, JUNQUEIRA V S, GOMES C C G, et al.Tag SNP selection for prediction of tick resistance in Brazilian Braford and Hereford cattle breeds using Bayesian methods[J].Genetics Selection Evolution, 2017, 49 (1) :49." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDF905A110DDA360776C673FC6C379378B&amp;v=MDE1OTNUbDZTd21ScW1FMmZydVhRckx0Q09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TnhodzcyOXdhOD1OajdCYXNXeEh0UzlybzVGRUo5K0QzbzV5QkVWbQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[11]</b>
                                         SOLLERO B P, JUNQUEIRA V S, GOMES C C G, et al.Tag SNP selection for prediction of tick resistance in Brazilian Braford and Hereford cattle breeds using Bayesian methods[J].Genetics Selection Evolution, 2017, 49 (1) :49.
                                    </a>
                                </li>
                                <li id="256">


                                    <a id="bibliography_12" title=" SHI Wei, CAO Zhi, LI Xiong, et al.Informative SNPs selection based on fuzzy clustering and genetic algorithm[J].Journal of Computational and Theoretical Nanoscience, 2017, 14 (3) :1440-1445." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJAH&amp;filename=SJAH17E13DAE125232B3371EA9C00544DDA8&amp;v=MTYzNDZwbWFCdUhZZk9HUWxmQ3BiUTM1TnhodzcyOXdhOD1OaWZLWnJLL2E5RFAyLzR3WmVrS0RuODd2UlVRN1Q0SU9YYVJyQkl3ZmJiZ01jdVhDT052RlNpV1dyN0pJRg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[12]</b>
                                         SHI Wei, CAO Zhi, LI Xiong, et al.Informative SNPs selection based on fuzzy clustering and genetic algorithm[J].Journal of Computational and Theoretical Nanoscience, 2017, 14 (3) :1440-1445.
                                    </a>
                                </li>
                                <li id="258">


                                    <a id="bibliography_13" title=" CHORMUNGE S, JENA S.Correlation based feature selection with clustering for high dimensional data[J].Journal of Electrical Systems and Information Technology, 2018, 5 (3) :542-549." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJESC430A0FF603B7C0BEF48EA5FFCBF760D&amp;v=MTA1Mjk4U1RRN3JyQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TnhodzcyOXdhOD1OaWZPZmNDOEhkRzlyL2t6WXVzTWZudEt6MlJtbkR0MVBRN24ybVJHQw==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[13]</b>
                                         CHORMUNGE S, JENA S.Correlation based feature selection with clustering for high dimensional data[J].Journal of Electrical Systems and Information Technology, 2018, 5 (3) :542-549.
                                    </a>
                                </li>
                                <li id="260">


                                    <a id="bibliography_14" title=" BEZDEK J C, EHRLICH R, FULL W.FCM:the fuzzy c-means clustering algorithm[J].Computers and Geosciences, 1984, 10 (2) :191-203." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=FCM: The fuzzy c -means clustering algorithm">
                                        <b>[14]</b>
                                         BEZDEK J C, EHRLICH R, FULL W.FCM:the fuzzy c-means clustering algorithm[J].Computers and Geosciences, 1984, 10 (2) :191-203.
                                    </a>
                                </li>
                                <li id="262">


                                    <a id="bibliography_15" title=" RACHMAN A A, RUSTAM Z.Cancer classification using fuzzy c-means with feature selection[C]//Proceedings of International Conference on Mathematics, Statistics, and Their Applications.Washington D.C., USA:IEEE Press, 2017:31-34." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Cancer classification using fuzzy c-means with feature selection">
                                        <b>[15]</b>
                                         RACHMAN A A, RUSTAM Z.Cancer classification using fuzzy c-means with feature selection[C]//Proceedings of International Conference on Mathematics, Statistics, and Their Applications.Washington D.C., USA:IEEE Press, 2017:31-34.
                                    </a>
                                </li>
                                <li id="264">


                                    <a id="bibliography_16" title=" 唐成龙, 王石刚, 徐威.基于数据加权策略的模糊聚类改进算法[J].电子与信息学报, 2010, 32 (6) :1277-1283." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=DZYX201006004&amp;v=MjA1MzhRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJxRnkvZ1ViN1BJVGZTZHJHNEg5SE1xWTlGWUk=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[16]</b>
                                         唐成龙, 王石刚, 徐威.基于数据加权策略的模糊聚类改进算法[J].电子与信息学报, 2010, 32 (6) :1277-1283.
                                    </a>
                                </li>
                                <li id="266">


                                    <a id="bibliography_17" title=" 王印松, 商丹丹, 王艳飞, 等.改进的模糊聚类在控制系统故障诊断中的应用[J].计算机工程与科学, 2018, 40 (2) :326-330." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJK201802020&amp;v=MDA0NjNVUkxPZVplUnFGeS9nVWI3UEx6N0JaYkc0SDluTXJZOUhaSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckM=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[17]</b>
                                         王印松, 商丹丹, 王艳飞, 等.改进的模糊聚类在控制系统故障诊断中的应用[J].计算机工程与科学, 2018, 40 (2) :326-330.
                                    </a>
                                </li>
                                <li id="268">


                                    <a id="bibliography_18" title=" DING Yi, FU Xian.Kernel-based fuzzy c-means clustering algorithm based on genetic algorithm[J].Neurocomputing, 2016, 188:233-238." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES9E8E156E2769126EB1B66195883385FE&amp;v=MTg3ODZ3SkJYMDd5V05oNjAxN1RuN3JxUm85ZXJHY1FNenFDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcGJRMzVOeGh3NzI5d2E4PU5pZk9mYnJORnFUTnFva3dadQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[18]</b>
                                         DING Yi, FU Xian.Kernel-based fuzzy c-means clustering algorithm based on genetic algorithm[J].Neurocomputing, 2016, 188:233-238.
                                    </a>
                                </li>
                                <li id="270">


                                    <a id="bibliography_19" title=" LI Min, ZHANG Limei, XIANG Zhikang, et al.An improved fuzzy c-means algorithm for brain MRI image segmenta-tion[C]//Proceedings of International Conference on Progress in Informatics and Computing.Washington D.C., USA:IEEE Press, 2017:336-339." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=An improved fuzzy c-means algorithm for brain MRI image segmenta-tion">
                                        <b>[19]</b>
                                         LI Min, ZHANG Limei, XIANG Zhikang, et al.An improved fuzzy c-means algorithm for brain MRI image segmenta-tion[C]//Proceedings of International Conference on Progress in Informatics and Computing.Washington D.C., USA:IEEE Press, 2017:336-339.
                                    </a>
                                </li>
                                <li id="272">


                                    <a id="bibliography_20" title=" CAO Hongbao, DENG Hongwen, WANG Yuping.Segmentation of M-FISH images for improved classification of chromosomes with an adaptive fuzzy c-means clustering algorithm[J].IEEE Transactions on Fuzzy Systems, 2012, 20 (1) :1-8." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Segmentation of M-FISH images for improved classification of chromosomes with an adaptive fuzzy C-means clustering algorithm">
                                        <b>[20]</b>
                                         CAO Hongbao, DENG Hongwen, WANG Yuping.Segmentation of M-FISH images for improved classification of chromosomes with an adaptive fuzzy c-means clustering algorithm[J].IEEE Transactions on Fuzzy Systems, 2012, 20 (1) :1-8.
                                    </a>
                                </li>
                                <li id="274">


                                    <a id="bibliography_21" title=" DUNN J C.A fuzzy relative of the ISODATA process and its use in detecting compact well-separated clusters[J].Journal of Cybernetics, 1974, 3 (3) :32-57." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A fuzzy relative of the ISODATA process and its use in detecting compact, well-separated clusters">
                                        <b>[21]</b>
                                         DUNN J C.A fuzzy relative of the ISODATA process and its use in detecting compact well-separated clusters[J].Journal of Cybernetics, 1974, 3 (3) :32-57.
                                    </a>
                                </li>
                                <li id="276">


                                    <a id="bibliography_22" title=" KANNAN S S, RAMARAJ N.A novel hybrid feature selection via symmetrical uncertainty ranking based local memetic search algorithm[J].Knowledge-Based Systems, 2010, 23 (6) :580-585." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011501719364&amp;v=MTA5ODFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMdklKMXNXYXhVPU5pZk9mYks3SHRETnFvOUVZK29HRDNvOW9CTVQ2VDRQUQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[22]</b>
                                         KANNAN S S, RAMARAJ N.A novel hybrid feature selection via symmetrical uncertainty ranking based local memetic search algorithm[J].Knowledge-Based Systems, 2010, 23 (6) :580-585.
                                    </a>
                                </li>
                                <li id="278">


                                    <a id="bibliography_23" title=" 王连喜, 蒋盛益.一种基于特征聚类的特征选择方法[J].计算机应用研究, 2015, 32 (5) :1305-1308." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSYJ201505007&amp;v=MTM4NDllUnFGeS9nVWI3UEx6N1NaTEc0SDlUTXFvOUZZNFFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVo=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[23]</b>
                                         王连喜, 蒋盛益.一种基于特征聚类的特征选择方法[J].计算机应用研究, 2015, 32 (5) :1305-1308.
                                    </a>
                                </li>
                                <li id="280">


                                    <a id="bibliography_24" title=" PENG Hanchuan, LONG Fuhui, DING C.Feature selection based on mutual information:criteria of max-dependency, max-relevance, and min-redundancy[J].IEEE Transactions on Pattern Analysis and Machine Intelligence, 2005, 27 (8) :1226-1238." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Feature selection based on mutual information criteria of max-dependency, max-relevance, and min-redundancy">
                                        <b>[24]</b>
                                         PENG Hanchuan, LONG Fuhui, DING C.Feature selection based on mutual information:criteria of max-dependency, max-relevance, and min-redundancy[J].IEEE Transactions on Pattern Analysis and Machine Intelligence, 2005, 27 (8) :1226-1238.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JSJC" target="_blank">计算机工程</a>
                2019,45(08),66-74 DOI:10.19678/j.issn.1000-3428.0052286            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>一种面向SNP选择的模糊聚类算法</b></span>
 <span class="shoufa"></span>                                     </h1>

                <div class="btn-downloads  btn-downloads-new">

                        <a class="read read-btn-special" target="_blank" href=http://x.cnki.net/search/common/testlunbo?dbcode=CJFD&amp;tablename=CJFDLAST2019&amp;filename=JSJC201908012&amp;filesourcetype=1><i class="i-btn i-care"></i>精读</a>
                    <div class="read-btn-l">
                        <a class="caj" target="_blank" href="http://kns.cnki.net/kns/download.aspx?filename=CVmYJB1Qw1EVhN0S4tEVZJmVMJzRw1Ud4dzSqh1QLV3aUhUOItWZINWdid3RLNjVptmbahWY4xWT5FXbnBnYmZXZTZVend3cvE0cPZHMWZGUVJ1c2Anajp1ZolXNPJjTLh3K3dXdsxGRiZ1T29UNN1kTJZDdiZVR&tablename=CJFDLAST2019"><i class="i-btn i-caj"></i> CAJ下载</a>
                        <a class="pdf" target="_blank" href="http://kns.cnki.net/kns/download.aspx?filename=CVmYJB1Qw1EVhN0S4tEVZJmVMJzRw1Ud4dzSqh1QLV3aUhUOItWZINWdid3RLNjVptmbahWY4xWT5FXbnBnYmZXZTZVend3cvE0cPZHMWZGUVJ1c2Anajp1ZolXNPJjTLh3K3dXdsxGRiZ1T29UNN1kTJZDdiZVR&tablename=CJFDLAST2019&dflag=pdfdown"><i class="i-btn i-pdf"></i>PDF下载</a>

                        <p>永久保存本文,请下载至本地</p>
                    </div>

                </div>
            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BC%A0%E6%B3%A2&amp;code=07763616&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">张波</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%91%A8%E4%BB%8E%E5%8D%8E&amp;code=10811332&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">周从华</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BC%A0%E4%BB%98%E5%85%A8&amp;code=30152651&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">张付全</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BC%A0%E5%A9%B7&amp;code=21984463&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">张婷</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E8%92%8B%E8%B7%83%E6%98%8E&amp;code=40538250&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蒋跃明</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E6%B1%9F%E8%8B%8F%E5%A4%A7%E5%AD%A6%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%A7%91%E5%AD%A6%E4%B8%8E%E9%80%9A%E4%BF%A1%E5%B7%A5%E7%A8%8B%E5%AD%A6%E9%99%A2&amp;code=0154461&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">江苏大学计算机科学与通信工程学院</a>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E6%97%A0%E9%94%A1%E5%B8%82%E7%B2%BE%E7%A5%9E%E5%8D%AB%E7%94%9F%E4%B8%AD%E5%BF%83&amp;code=0133336&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">无锡市精神卫生中心</a>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E6%97%A0%E9%94%A1%E5%B8%82%E5%A6%87%E5%B9%BC%E4%BF%9D%E5%81%A5%E9%99%A2&amp;code=0232773&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">无锡市妇幼保健院</a>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E6%97%A0%E9%94%A1%E5%B8%82%E7%AC%AC%E4%BA%94%E4%BA%BA%E6%B0%91%E5%8C%BB%E9%99%A2&amp;code=0102125&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">无锡市第五人民医院</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>在对高维少样本的遗传数据进行单核苷酸多态性 (SNP) 选择时, 为能使所选SNP子集高度代表所有SNP信息, 实现数据降维, 在模糊C均值 (FCM) 算法的基础上提出一种改进方法GN-FCM。通过引入SNP权重因子量化SNP位点重要程度的差异性, 同时将重点SNP邻域正则项引入模糊聚类的损失函数中, 挖掘高度重要SNP与同邻域内其他SNP的关联性。实验结果表明, GN-FCM具有较好的收敛性, 与DW-FCM算法相比, 其构造的SNP子集在支持向量机、决策树和朴素贝叶斯分类中准确率分别提升5.73%、3.40%和3.79%, F1值分别提升4.01%、3.20%和2.22%。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8D%95%E6%A0%B8%E8%8B%B7%E9%85%B8%E5%A4%9A%E6%80%81%E6%80%A7%E9%80%89%E6%8B%A9&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">单核苷酸多态性选择;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%A8%A1%E7%B3%8A%E8%81%9A%E7%B1%BB&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">模糊聚类;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%89%B9%E5%BE%81%E9%80%89%E6%8B%A9&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">特征选择;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%94%AF%E6%8C%81%E5%90%91%E9%87%8F%E6%9C%BA&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">支持向量机;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%86%B3%E7%AD%96%E6%A0%91&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">决策树;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%9C%B4%E7%B4%A0%E8%B4%9D%E5%8F%B6%E6%96%AF%E5%88%86%E7%B1%BB&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">朴素贝叶斯分类;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    张波 (1993—) , 男, 硕士研究生, 主研方向为数据挖掘;E-mail: zhangbo_superby@ 163. com;
                                </span>
                                <span>
                                    周从华, 教授、博士;;
                                </span>
                                <span>
                                    张付全, 博士后;;
                                </span>
                                <span>
                                    张婷, 博士。;
                                </span>
                                <span>
                                    蒋跃明, 博士。;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-08-03</p>

                    <p>

                            <b>基金：</b>
                                                        <span>江苏省重点研发计划社会发展项目 (BE2016630, BE2017628);</span>
                                <span>无锡市卫生计生委科研项目 (Z201603);</span>
                    </p>
            </div>
                    <h1><b>A Fuzzy Clustering Algorithm for SNP Selection</b></h1>
                    <h2>
                    <span>ZHANG Bo</span>
                    <span>ZHOU Conghua</span>
                    <span>ZHANG Fuquan</span>
                    <span>ZHANG Ting</span>
                    <span>JIANG Yueming</span>
            </h2>
                    <h2>
                    <span>School of Computer Science and Communication Engineering, Jiangsu University</span>
                    <span>Wuxi Mental Health Center</span>
                    <span>Wuxi Hospital for Maternity and Child Health Care Hospital</span>
                    <span>Wuxi No.5 People's Hospital</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>In the selection of Single Nucleotide Polymorphism (SNP) from high-dimensional genetic data with few samples, in order to make the selected SNP subset highly represent all SNP information and achieve data dimension reduction, an improved method is proposed on the basis of Fuzzy C-Mean (FCM) algorithm, which is named GN-FCM.By introducing the weight factor of SNP, the difference of importance degree of SNP site is quantified.Meanwhile, the neighborhood regular term of key SNP is introduced into the loss function of fuzzy clustering, so as to mine the correlation between highly important SNP and other SNPs in the neighborhood.Experimental results show that GN-FCM has better convergence.Compared with DW-FCM algorithm, the accuracy of the constructed SNP subsets by this algorithm in Support Vector Machine (SVM) , Decision Tree (DT) and Naïve Bayesian (NB) classification is improved by 5.73%, 3.40% and 3.79% respectively, and the F1 value is improved by 4.01%, 3.20% and 2.22% respectively.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Single%20Nucleotide%20Polymorphism%20(SNP)%20selection&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Single Nucleotide Polymorphism (SNP) selection;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=fuzzy%20clustering&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">fuzzy clustering;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=feature%20selection&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">feature selection;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Support%20Vector%20Machine%20(SVM)%20&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Support Vector Machine (SVM) ;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Decision%20Tree%20(DT)%20&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Decision Tree (DT) ;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Na%C3%AFve%20Bayesian%20(NB)%20classification&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Naïve Bayesian (NB) classification;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                                            </p>
                                    <p><b>Received：</b> 2018-08-03</p>
                                    <p>
                                            </p>
            </div>


        <!--brief start-->
                        <h3 id="51" name="51" class="anchor-tag">0 概述</h3>
                <div class="p1">
                    <p id="52">遗传疾病是由于遗传物质的改变所导致的疾病, 这种类型的疾病病种多、发病率高, 目前发现的遗传疾病已高达3 000多种。近年来, 随着DNA微阵列技术取得巨大进步, 人们可以获得数以万计的基因表达谱, 从而能够从基因层面深入了解疾病, 为疾病发病机理的研究提供强有力的支持。人类全基因组研究 (Genome-Wide Association Study, GWAS) 的日益推进, 使得精神分裂症、类风湿关节疾病等疾病的研究取得了良好的进展<citation id="283" type="reference"><link href="234" rel="bibliography" /><link href="236" rel="bibliography" /><sup>[<a class="sup">1</a>,<a class="sup">2</a>]</sup></citation>。GWAS是一种检测特定物种中不同个体间的全部或大部分基因, 从而了解不同个体间基因变化的一种方法。该方法使人们发现了许多未知基因以及染色体区域, 从而更深入地研究复杂疾病, 其研究是建立在单核苷酸多态性 (Single Nucleotide Polymorphisms, SNP) 的基础之上的, 通过对比患病组和健康组的SNP, 可以发现与疾病最为相关的致病基因<citation id="282" type="reference"><link href="238" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>。SNP是指基因组上单个核苷酸的变异, 包括转化、颠换、缺失和插入。SNP数量多、分布广, 平均每1 000个～1 500个碱基对就会有一个SNP, 目前总数达到400万以上。研究表明, 人体许多的表型差异以及对疾病的易感性都与SNP有着紧密的联系。然而, 如此之多的SNP, 并不是每个对于生物表型差异都起决定性作用的, 换言之, 存在着很多冗余的SNP。如果不对这些冗余的SNP进行筛选或者剔除, 会导致维数灾难, 对后续的研究造成极大的麻烦。因此, 关于SNP的选择成为一个重要的课题。</p>
                </div>
                <div class="p1">
                    <p id="53">SNP选择问题在某种程度上其实也可以看作是特征选择的子问题, 然而由于SNP的特殊性, 其高维特性以及SNP与SNP之间并非完全独立的特点, 使得一般的特征选择方法难免会遗漏许多内在的遗传信息。聚类可以有效缓解这种问题:首先通过SNP之间的关系将其划分成不同的簇, 然后基于指定的规则或者指标再从每个簇中选择符合要求的SNP进行最终的子集构造。</p>
                </div>
                <div class="p1">
                    <p id="54">本文结合遗传学指标, 提出一种改进的模糊聚类算法GN-FCM用于SNP选择。通过基于假设性检验的方法初步剔除生物意义较小的SNP, 针对模糊C均值 (Fuzzy C-Mean, FCM) 聚类算法未考虑每个SNP的重要性且无法挖掘SNP局部之间关联性的不足, 使用GN-FCM对初步筛选后的数据进行聚类, 并根据对称的不确定性 (Symmetrical Uncertainty, SU) 对得到的每个簇进行筛选, 构造最终的SNP集合。</p>
                </div>
                <h3 id="55" name="55" class="anchor-tag">1 相关工作</h3>
                <h4 class="anchor-tag" id="56" name="56">1.1 SNP选择</h4>
                <div class="p1">
                    <p id="57">SNP的选择问题本质上就是机器学习领域中的特征选择问题。目前比较成熟的SNP选择方法主要是基于过滤式和包裹式的选择方法。过滤式选择是指通过设定一个评价指标来衡量每个SNP, 其优点是计算简单, 但不一定能获得最优的特征子集;相反包裹式是将学习器和评价指标结合起来, 评价的目的是为了学习器能获得更好的性能, 虽然能获得良好的选择效果, 但是计算量却比较大<citation id="284" type="reference"><link href="240" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>。因此, 研究者基于这两种方法提出多种改进方法。文献<citation id="285" type="reference">[<a class="sup">5</a>]</citation>针对传统包裹式选择方法计算量太大, 而且忽视了基因之间的连锁不平衡度 (Linkage Disequilibrium, LD) 的问题, 提出了一种基于两位点和多位点的连锁不平衡的最大相关和最小冗余选择方法 (MCMR) 来构造信息SNP子集。文献<citation id="286" type="reference">[<a class="sup">6</a>]</citation>使用基于ReliefF算法进行特征加权的选择方法。文献<citation id="287" type="reference">[<a class="sup">7</a>]</citation>融合了过滤式和包裹式方法, 提出了一种混合特征选择方法用来构造信息SNP子集, 从而获得了更高的预测准确度。文献<citation id="288" type="reference">[<a class="sup">8</a>]</citation>定义了一种自然测量方法, 用于评估标签SNP的预测准确性, 并基于该方法提出了一种新的预测算法用于标签SNP选择。文献<citation id="289" type="reference">[<a class="sup">9</a>]</citation>基于搜索策略与聚类方法, 并将身高和绝经后乳腺癌和结肠直肠癌联系起来, 从而提出了一种可以系统性地识别疾病病因的SNP选择方法。文献<citation id="290" type="reference">[<a class="sup">10</a>]</citation>介绍了一种基于个体SNP变异模式的基函数拓展而来的信息量分数, 并结合改进的PCA方法选择具有最佳分数的SNP子集。文献<citation id="291" type="reference">[<a class="sup">11</a>]</citation>使用聚类算法对不同品种的牛的SNP进行选择并研究了他们的抗病性。文献<citation id="292" type="reference">[<a class="sup">12</a>]</citation>使用模糊聚类和遗传方法相结合的策略进行信息SNP的选择, 在降低问题规模的同时还提高了选择和预测的精度。</p>
                </div>
                <h4 class="anchor-tag" id="58" name="58">1.2 模糊聚类算法</h4>
                <div class="p1">
                    <p id="59">聚类作为一种无监督的学习方法, 同样也可以在数据特征选择方面发挥重要作用。聚类方法是根据数据内部的相似程度将数据划分为不同的簇, 使得簇内数据的相似度高, 而簇间的相似度低。其中, k-means是一种流行且有效的方法。文献<citation id="293" type="reference">[<a class="sup">13</a>]</citation>提出一种基于聚类的新方法来解决高维数据的特征选择问题, 先使用k-Means聚类算法来消除不相关的特征, 再使用相似性度量方法从每个簇中选择非冗余的特征。但是传统的k-means聚类算法严格要求每个数据至多只能属于一个簇, 导致算法对异常值较为敏感。针对这个问题, k-means的改进FCM聚类算法<citation id="294" type="reference"><link href="260" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>应运而生。FCM同样也可以应用到特征选择的领域。文献<citation id="295" type="reference">[<a class="sup">15</a>]</citation>结合Fisher’s Ratio方法和FCM对癌症数据进行了特征选择, 最终获得了最高89.68%的分类准确率。虽然FCM在数据聚类分析时具有很大的优势, 也取得了很好的效果, 但在面对不同规模的数据或者不同的数据类型时, 原始的FCM已经难以满足人们的要求, 因此, 为了适应不同的业务场景, 研究者又提出许多改进的FCM方法。文献<citation id="296" type="reference">[<a class="sup">16</a>]</citation>提出了一种指数加权的模糊聚类策略, 使得算法在聚类的过程中可以差异化处理每个数据, 并结合G-K算法提出了一种全新的模糊聚类算法来提高聚类的质量, 同时更准确地挖掘离群点。文献<citation id="297" type="reference">[<a class="sup">17</a>]</citation>针对故障诊断的应用场景, 提出了一种基于高斯密度特征加权的模糊聚类算法, 使得算法不仅能识别不同设备的故障, 而且还能识别同一设备的不同类型的故障。针对高维而且线性不可分的数据的建模, 始终是研究者关注的热点。常用的方法是借助模式识别中的核方法, 将数据从低维空间映射到高维空间, 然后再进行线性分割。同样, 研究者将核方法引入模糊聚类算法中, 在聚类效果上取得了较大提升。文献<citation id="298" type="reference">[<a class="sup">18</a>]</citation>针对传统的FCM算法不能自动确定聚类中心的树木的问题, 提出了一种改进的自适应的方法GAKFCM, 首先使用遗传算法 (Genetic Algorithm, GA) 来初始化类中心, 然后使用基于核方法的改进模糊聚类策略KFCM进行聚类, 使聚类效果得到优化。FCM在图像特征提取方面也能发挥很大的优势。文献<citation id="299" type="reference">[<a class="sup">19</a>]</citation>针对传统FCM不能很好处理MRI图像亮度不平衡的问题, 将图像周围局部的影响考虑在内, 在原来的FCM中引入了邻域项和偏置项, 最终取得了比传统的EM和FCM更好的效果。文献<citation id="300" type="reference">[<a class="sup">20</a>]</citation>考虑了M-FISH图像亮度对每个类别的增益, 引入基于增益的正则项, 最终获得比传统的FCM更低的分类错误率。</p>
                </div>
                <h3 id="60" name="60" class="anchor-tag">2 基于改进FCM算法的SNP选择</h3>
                <h4 class="anchor-tag" id="61" name="61">2.1 FCM聚类算法原理</h4>
                <div class="p1">
                    <p id="62">FCM聚类算法<citation id="301" type="reference"><link href="274" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>是建立在模糊理论的基础之上, 与k-means中每个数据只能属于一个簇不同, 它建立了一个模糊矩阵, 用来表示每个数据属于不同的类别的可能性, 这种不确定性使得算法更加灵活, 也能在某些场景更好地表现数据内在的特性。</p>
                </div>
                <div class="p1">
                    <p id="63">假设一个数据集<i>X</i>={<i>x</i><sub>1</sub>, <i>x</i><sub>2</sub>, …, <i>x</i><sub><i>n</i></sub>}, 聚类的最终目的就是将这<i>n</i>个数据划分到<i>c</i>个簇中。FCM的目标函数如式 (1) 所示。</p>
                </div>
                <div class="p1">
                    <p id="64" class="code-formula">
                        <mathml id="64"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>J</mi><msub><mrow></mrow><mrow><mtext>F</mtext><mtext>C</mtext><mtext>Μ</mtext></mrow></msub><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mi>u</mi></mstyle></mrow></mstyle><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="65">其中:<i>u</i><sub><i>ij</i></sub>为隶属度矩阵元素, 其满足的约束条件为<i>u</i><sub><i>ij</i></sub>∈[0, 1]并且<mathml id="66"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mi>u</mi></mstyle><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mn>1</mn></mrow></math></mathml>;<i>m</i>是模糊因子, 一般取值为1.5～2.5;<i>v</i><sub><i>i</i></sub>表示第<i>i</i>个簇的簇中心。通过使用拉格朗日法可以求得在每一次的迭代过程中, 隶属度矩阵元素<i>u</i><sub><i>ij</i></sub>和簇中心<i>v</i><sub><i>i</i></sub>的更新式分别如式 (2) 和式 (3) 所示。</p>
                </div>
                <div class="p1">
                    <p id="67" class="code-formula">
                        <mathml id="67"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>u</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mfrac><mn>1</mn><mrow><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mrow><mrow><mo> (</mo><mrow><mfrac><mrow><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup></mrow><mrow><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup></mrow></mfrac></mrow><mo>) </mo></mrow></mrow></mstyle></mrow><msup><mrow></mrow><mrow><mfrac><mn>1</mn><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></mfrac></mrow></msup></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>2</mn><mo stretchy="false">) </mo></mtd></mtr><mtr><mtd><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo>=</mo><mfrac><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mi>u</mi></mstyle><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mi>u</mi></mstyle><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>3</mn><mo stretchy="false">) </mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <h4 class="anchor-tag" id="68" name="68">2.2 改进的模糊聚类算法GN-FCM</h4>
                <div class="p1">
                    <p id="69">虽然已有很多研究表明FCM相比其他算法在某些特定的场景具有很明显的优势, 但是面对一些特殊的需求如图像、高维序列等时, 原始的FCM已经不能满足人们的需求。本文研究的内容是SNP的筛选, 传统的FCM方法或者相关的选择方法, 有些完全基于统计或者假设性检验, 导致不能充分挖掘数据内在的联系, 有些则忽略了SNP数据本身包含的遗传信息, 使得最后的子集构造结果丢失重要信息。SNP数据本身具有以下2个特性:1) 每个SNP对表型的影响是不同的, 有的SNP与表型高度关联, 有的则没有关联;2) SNP存在高变区和热点区, 即与表型高度关联的SNP与周围的SNP之间往往也是高度关联的。结合SNP的上述特性, 本文在原模糊聚类算法的基础上, 提出一种基于信息增益加权和局部信息影响的模糊聚类算法GN-FCM。</p>
                </div>
                <h4 class="anchor-tag" id="70" name="70">2.2.1 GN-FCM算法的目标函数</h4>
                <div class="p1">
                    <p id="71">GN-FCM算法主要涵盖以下2个方面:</p>
                </div>
                <div class="p1">
                    <p id="72">1) 信息增益 (Information Gain, IG) 可以用来衡量使用某个SNP进行样本划分时的“纯度”。假定样本<i>S</i>中的某个SNP a有<i>V</i>个可能取值{<i>a</i><sub>1</sub>, <i>a</i><sub>2</sub>, …, <i>a</i><sub><i>V</i></sub>}, 则IG的定义如式 (4) 所示。</p>
                </div>
                <div class="p1">
                    <p id="73" class="code-formula">
                        <mathml id="73"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ι</mi><mi>G</mi><mo stretchy="false"> (</mo><mi>a</mi><mo stretchy="false">) </mo><mo>=</mo><mi>Η</mi><mo stretchy="false"> (</mo><mi>S</mi><mo stretchy="false">) </mo><mo>-</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>v</mi><mo>=</mo><mn>1</mn></mrow><mi>V</mi></munderover><mrow><mfrac><mrow><mrow><mo>|</mo><mrow><mi>S</mi><msup><mrow></mrow><mi>v</mi></msup></mrow><mo>|</mo></mrow></mrow><mrow><mrow><mo>|</mo><mi>S</mi><mo>|</mo></mrow></mrow></mfrac></mrow></mstyle><mi>Η</mi><mo stretchy="false"> (</mo><mi>S</mi><msup><mrow></mrow><mi>v</mi></msup><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>4</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="74">其中:<i>S</i><sup><i>v</i></sup>表示SNP取值为<i>a</i><sub><i>v</i></sub>的样本集合;<i>H</i> (<i>S</i>) 表示样本集合<i>S</i>中的信息熵。</p>
                </div>
                <div class="p1">
                    <p id="75" class="code-formula">
                        <mathml id="75"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Η</mi><mo stretchy="false"> (</mo><mi>S</mi><mo stretchy="false">) </mo><mo>=</mo><mo>-</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mrow><mo>|</mo><mi>Κ</mi><mo>|</mo></mrow></mrow></munderover><mi>p</mi></mstyle><msub><mrow></mrow><mi>k</mi></msub><mtext>l</mtext><mtext>b</mtext><mtext> </mtext><mi>p</mi><msub><mrow></mrow><mi>k</mi></msub><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>5</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="76">其中, <i>p</i><sub><i>k</i></sub>表示<i>S</i>中第<i>k</i>类样本的比例。</p>
                </div>
                <div class="p1">
                    <p id="77">2) 由于SNP局部的高度关联性, 那些对基因表型差异影响最大的SNP与其周围的SNP之间相互影响, 因此针对每个SNP进行局部信息的分析是很有必要的。</p>
                </div>
                <div class="p1">
                    <p id="78">基于上述两方面, GN-FCM算法的目标函数如式 (6) 所示。</p>
                </div>
                <div class="p1">
                    <p id="79" class="code-formula">
                        <mathml id="79"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>J</mi><msub><mrow></mrow><mrow><mtext>G</mtext><mtext>Ν</mtext><mo>-</mo><mtext>F</mtext><mtext>C</mtext><mtext>Μ</mtext></mrow></msub><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mi>g</mi></mstyle><msub><mrow></mrow><mi>j</mi></msub><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mi>u</mi></mstyle><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mo>+</mo></mtd></mtr><mtr><mtd><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext></mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mrow></mrow></mstyle></mrow><mtext> </mtext><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mrow></mrow></mstyle></mrow><mi>u</mi><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup><mrow><mo> (</mo><mrow><mfrac><mn>1</mn><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></munder><mrow><mo stretchy="false">∥</mo><mi>x</mi><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow></mstyle><msup><mrow></mrow><mn>2</mn></msup></mrow><mo>) </mo></mrow></mtd></mtr><mtr><mtd><mtext>s</mtext><mo>.</mo><mtext>t</mtext><mo>.</mo><mtext> </mtext><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mi>u</mi></mstyle><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mn>1</mn><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>6</mn><mo stretchy="false">) </mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="80">其中:<i>D</i><sub><i>j</i></sub>表示与样本<i>x</i><sub><i>j</i></sub>距离最近的样本的集合;<mathml id="81"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></math></mathml>为集合D<sub>j</sub>的样本数;g<sub>j</sub>是归一化后的每个<i>SNP</i>的信息增益, 其值介于0到1之间, 如式 (7) 所示。</p>
                </div>
                <div class="p1">
                    <p id="82" class="code-formula">
                        <mathml id="82"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>g</mi><msub><mrow></mrow><mi>j</mi></msub><mo>=</mo><mfrac><mrow><mi>Ι</mi><mi>G</mi><mo stretchy="false"> (</mo><mi>j</mi><mo stretchy="false">) </mo></mrow><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mi>Ι</mi></mstyle><mi>G</mi><mo stretchy="false"> (</mo><mi>k</mi><mo stretchy="false">) </mo></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>7</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="83">λ<sub>j</sub>是一个系数, 用来限制当某个<i>SNP</i>足够重要时才考虑其周围的<i>SNP</i>的关联性, 因此, 有:</p>
                </div>
                <div class="p1">
                    <p id="84" class="code-formula">
                        <mathml id="84"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub><mo>=</mo><mrow><mo>{</mo><mrow><mtable><mtr><mtd columnalign="left"><mn>1</mn><mo>, </mo><mi>g</mi><msub><mrow></mrow><mi>j</mi></msub><mo>∈</mo><mi>t</mi><mi>o</mi><mi>p</mi><msub><mrow></mrow><mrow><mi>t</mi><mi>%</mi></mrow></msub><mo stretchy="false"> (</mo><mi>G</mi><mo stretchy="false">) </mo></mtd></mtr><mtr><mtd columnalign="left"><mn>0</mn><mo>, </mo><mtext>其</mtext><mtext>他</mtext></mtd></mtr></mtable></mrow></mrow></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="85">其中, top<sub>t%</sub> (<i>G</i>) 表示当前的SNP的重要程度是所有SNP的前<i>t</i>%。</p>
                </div>
                <h4 class="anchor-tag" id="86" name="86">2.2.2 GN-FCM算法迭代</h4>
                <div class="p1">
                    <p id="87">GN-FCM算法使用拉格朗日法来计算聚类过程中隶属度矩阵元素<i>u</i><sub><i>ij</i></sub>和簇中心<i>v</i><sub><i>i</i></sub>的迭代公式, 构造的拉格朗日函数如式 (8) 所示。</p>
                </div>
                <div class="p1">
                    <p id="88" class="code-formula">
                        <mathml id="88"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>L</mi><msub><mrow></mrow><mrow><mi>u</mi><mo>, </mo><mi>v</mi><mo>, </mo><mtext>ϕ</mtext></mrow></msub><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mi>g</mi></mstyle><msub><mrow></mrow><mi>j</mi></msub><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mi>u</mi></mstyle><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mo>+</mo></mtd></mtr><mtr><mtd><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext></mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mrow></mrow></mstyle></mrow><mtext> </mtext><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mrow></mrow></mstyle></mrow><mi>u</mi><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup><mrow><mo> (</mo><mrow><mfrac><mn>1</mn><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></munder><mrow><mo stretchy="false">∥</mo><mi>x</mi><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow></mstyle><msup><mrow></mrow><mn>2</mn></msup></mrow><mo>) </mo></mrow><mo>+</mo></mtd></mtr><mtr><mtd><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext></mrow><mtext>ϕ</mtext><mo stretchy="false"> (</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mi>u</mi></mstyle><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>8</mn><mo stretchy="false">) </mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="89">具体步骤如下:</p>
                </div>
                <h4 class="anchor-tag" id="90" name="90">1) 更新隶属度矩阵元素<i>u</i><sub><i>ij</i></sub>。</h4>
                <div class="p1">
                    <p id="91">对式 (8) 中<i>L</i><sub><i>u</i>, <i>v</i>, ϕ</sub>的<i>u</i><sub><i>ij</i></sub>求偏导:</p>
                </div>
                <div class="p1">
                    <p id="92" class="code-formula">
                        <mathml id="92"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mrow><mi>δ</mi><mi>L</mi><msub><mrow></mrow><mrow><mi>u</mi><mo>, </mo><mi>v</mi><mo>, </mo><mtext>ϕ</mtext></mrow></msub></mrow><mrow><mi>δ</mi><mi>u</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mfrac><mo>=</mo><mi>m</mi><mi>u</mi><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></msubsup><mrow><mo> (</mo><mrow><mi>g</mi><msub><mrow></mrow><mi>j</mi></msub><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mo>-</mo><mfrac><mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></munder><mrow><mo stretchy="false">∥</mo><mi>x</mi><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow></mstyle><msup><mrow></mrow><mn>2</mn></msup></mrow><mo>) </mo></mrow><mo>+</mo><mtext>ϕ</mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>9</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="93">令偏导<mathml id="94"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mrow><mi>δ</mi><mi>L</mi><msub><mrow></mrow><mrow><mi>u</mi><mo>, </mo><mi>v</mi><mo>, </mo><mtext>ϕ</mtext></mrow></msub></mrow><mrow><mi>δ</mi><mi>u</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mfrac><mo>=</mo><mn>0</mn></mrow></math></mathml>, 可以得到:</p>
                </div>
                <div class="p1">
                    <p id="95" class="code-formula">
                        <mathml id="95"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mtext>ϕ</mtext><mo>=</mo><mo>-</mo><mi>m</mi><mi>u</mi><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></msubsup><mrow><mo> (</mo><mrow><mi>g</mi><msub><mrow></mrow><mi>j</mi></msub><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mo>-</mo><mfrac><mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></munder><mrow><mo stretchy="false">∥</mo><mi>x</mi><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow></mstyle><msup><mrow></mrow><mn>2</mn></msup></mrow><mo>) </mo></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>0</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="96">对式 (10) 进行变形, 可以得到:</p>
                </div>
                <div class="p1">
                    <p id="97" class="code-formula">
                        <mathml id="97"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>u</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mfrac><mrow><mrow><mrow><mo> (</mo><mrow><mfrac><mrow><mo>-</mo><mtext>ϕ</mtext></mrow><mi>m</mi></mfrac></mrow><mo>) </mo></mrow></mrow><msup><mrow></mrow><mrow><mfrac><mn>1</mn><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></mfrac></mrow></msup></mrow><mrow><mrow><mrow><mo> (</mo><mrow><mi>g</mi><msub><mrow></mrow><mi>j</mi></msub><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mo>-</mo><mfrac><mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></munder><mrow><mo stretchy="false">∥</mo><mi>x</mi><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow></mstyle><msup><mrow></mrow><mn>2</mn></msup></mrow><mo>) </mo></mrow></mrow><msup><mrow></mrow><mrow><mfrac><mn>1</mn><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></mfrac></mrow></msup></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>1</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="98">结合约束条件<mathml id="99"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mi>u</mi></mstyle><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mn>1</mn></mrow></math></mathml>, 可以得到:</p>
                </div>
                <div class="p1">
                    <p id="100" class="code-formula"><mathml id="101"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mrow></mrow></mstyle></mrow></math></mathml><mathml id="102"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo> (</mo><mrow><mfrac><mrow><mrow><mrow><mo> (</mo><mrow><mfrac><mrow><mo>-</mo><mtext>ϕ</mtext></mrow><mi>m</mi></mfrac></mrow><mo>) </mo></mrow></mrow><msup><mrow></mrow><mrow><mfrac><mn>1</mn><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></mfrac></mrow></msup></mrow><mrow><mrow><mrow><mo> (</mo><mrow><mi>g</mi><msub><mrow></mrow><mi>j</mi></msub><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mo>-</mo><mfrac><mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></munder><mrow><mo stretchy="false">∥</mo><mi>x</mi><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow></mstyle><msup><mrow></mrow><mn>2</mn></msup></mrow><mo>) </mo></mrow></mrow><msup><mrow></mrow><mrow><mfrac><mn>1</mn><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></mfrac></mrow></msup></mrow></mfrac></mrow><mo>) </mo></mrow><mo>=</mo><mn>1</mn><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>2</mn><mo stretchy="false">) </mo></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="103">结合式 (11) 和式 (12) , 可以得到最终的隶属度的迭代公式, 如式 (13) 所示。</p>
                </div>
                <div class="p1">
                    <p id="104"><i>u</i><sub><i>ij</i></sub>=<mathml id="105"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mrow></mrow></mstyle></mrow></math></mathml><mathml id="106"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mrow><mo> (</mo><mrow><mfrac><mrow><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mo>-</mo><mfrac><mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></munder><mrow><mo stretchy="false">∥</mo><mi>x</mi><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">∥</mo></mrow></mstyle><msup><mrow></mrow><mn>2</mn></msup></mrow><mrow><mrow><mo stretchy="false">∥</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mo>-</mo><mfrac><mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></munder><mrow><mo stretchy="false">∥</mo><mi>x</mi><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">∥</mo></mrow></mstyle><msup><mrow></mrow><mn>2</mn></msup></mrow></mfrac></mrow><mo>) </mo></mrow></mrow><msup><mrow></mrow><mrow><mfrac><mn>1</mn><mrow><mn>1</mn><mo>-</mo><mi>m</mi></mrow></mfrac></mrow></msup><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>3</mn><mo stretchy="false">) </mo></mrow></math></mathml></p>
                </div>
                <h4 class="anchor-tag" id="107" name="107">2) 更新簇中心<i>v</i><sub><i>i</i></sub>。</h4>
                <div class="p1">
                    <p id="108">对式 (8) 中<i>L</i><sub><i>u</i>, <i>v</i>, ϕ</sub>的<i>v</i><sub><i>i</i></sub>求偏导:</p>
                </div>
                <div class="p1">
                    <p id="109" class="code-formula">
                        <mathml id="109"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mrow><mi>δ</mi><mi>L</mi><msub><mrow></mrow><mrow><mi>u</mi><mo>, </mo><mi>v</mi><mo>, </mo><mtext>ϕ</mtext></mrow></msub></mrow><mrow><mi>δ</mi><mi>u</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mfrac><mo>=</mo><mo>-</mo><mn>2</mn><mrow><mo> (</mo><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mi>u</mi></mstyle><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup><mrow><mo> (</mo><mrow><mi>g</mi><msub><mrow></mrow><mi>j</mi></msub><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">) </mo><mo>+</mo><mfrac><mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></munder><mo stretchy="false"> (</mo></mstyle><mi>x</mi><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">) </mo></mrow><mo>) </mo></mrow></mrow><mo>) </mo></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>4</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="110">令偏导<mathml id="111"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mrow><mi>δ</mi><mi>L</mi><msub><mrow></mrow><mrow><mi>u</mi><mo>, </mo><mi>v</mi><mo>, </mo><mtext>ϕ</mtext></mrow></msub></mrow><mrow><mi>δ</mi><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub></mrow></mfrac><mo>=</mo><mn>0</mn></mrow></math></mathml>, 可以得到最终的簇中心的迭代公式:</p>
                </div>
                <div class="p1">
                    <p id="112" class="code-formula">
                        <mathml id="112"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo>=</mo><mfrac><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mi>g</mi></mstyle><msub><mrow></mrow><mi>j</mi></msub><mi>u</mi><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup><mrow><mo> (</mo><mrow><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>+</mo><mfrac><mrow><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mrow><mrow><mo>|</mo><mrow><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow><mo>|</mo></mrow></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>D</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></munder><mi>x</mi></mstyle></mrow><mo>) </mo></mrow></mrow><mrow><mo stretchy="false"> (</mo><mn>1</mn><mo>+</mo><mi>λ</mi><msub><mrow></mrow><mi>j</mi></msub><mo stretchy="false">) </mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mi>u</mi></mstyle><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>5</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="113">在每一次迭代的过程中, 都需要不断地按照式 (13) 和式 (15) 来更新隶属度和簇中心, 当前后的隶属度的差异小于某个阈值即<mathml id="114"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>|</mo><mrow><mi>U</mi><msup><mrow></mrow><mi>t</mi></msup><mo>-</mo><mi>U</mi><msup><mrow></mrow><mrow><mi>t</mi><mo>-</mo><mn>1</mn></mrow></msup></mrow><mo>|</mo></mrow><mo>≤</mo><mi>ε</mi></mrow></math></mathml>, 或者当前的迭代次数达到算法设定的最大迭代次数时, 则认定为算法达到收敛。聚类结束后, 可能有些SNP属于任何一个类的可能性都很小, 此时可以当作噪声数据处理。</p>
                </div>
                <h4 class="anchor-tag" id="115" name="115">2.2.3 GN-FCM算法</h4>
                <div class="p1">
                    <p id="116">GN-FCM算法描述如下:</p>
                </div>
                <div class="p1">
                    <p id="117"><b>算法</b> GN-FCM</p>
                </div>
                <div class="p1">
                    <p id="118"><b>输入</b><i>n</i>条<i>k</i>维的数据集<i>D</i>, 模糊因子<i>m</i>, 需要聚成的簇数<i>c</i>, 终止条件阈值<i>ε</i>, 分类阈值<i>τ</i>, 最大迭代次数<i>MAX</i>_<i>N</i></p>
                </div>
                <div class="p1">
                    <p id="119"><b>输出</b> 每个SNP所属于的类别</p>
                </div>
                <div class="p1">
                    <p id="120">1.计算每个SNP的g<sub>j</sub>并存于表中</p>
                </div>
                <div class="p1">
                    <p id="121">2.初始化隶属度矩阵, 使得满足约束条件<mathml id="122"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mtext>i</mtext><mo>=</mo><mn>1</mn></mrow><mtext>c</mtext></munderover><mtext>u</mtext></mstyle><msub><mrow></mrow><mrow><mtext>i</mtext><mtext>j</mtext></mrow></msub><mo>=</mo><mn>1</mn></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="123">3.根据式 (16) 计算初始时的簇中心矩阵</p>
                </div>
                <div class="p1">
                    <p id="124">4.for t=1 to MAX_N do</p>
                </div>
                <div class="p1">
                    <p id="125">5.U<sup>t</sup>←U<sup>t-1</sup>//记录上次迭代的隶属度</p>
                </div>
                <div class="p1">
                    <p id="126">6.for j=1 to n do</p>
                </div>
                <div class="p1">
                    <p id="127">7.for i=1 to c do</p>
                </div>
                <div class="p1">
                    <p id="128">8.根据式 (13) 计算U<sup>t</sup>←u[i][j]</p>
                </div>
                <div class="p1">
                    <p id="129">9.根据式 (15) 计算V<sup>t</sup>←v[i]</p>
                </div>
                <div class="p1">
                    <p id="130">10.end for</p>
                </div>
                <div class="p1">
                    <p id="131">11.end for</p>
                </div>
                <div class="p1">
                    <p id="132">12.until<mathml id="133"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mtext>Δ</mtext><mo>=</mo><mrow><mo>|</mo><mrow><mtext>U</mtext><msup><mrow></mrow><mtext>t</mtext></msup><mo>-</mo><mtext>U</mtext><msup><mrow></mrow><mrow><mtext>t</mtext><mo>-</mo><mn>1</mn></mrow></msup></mrow><mo>|</mo></mrow><mo>≤</mo><mtext>τ</mtext><mspace width="0.25em" /><mtext>o</mtext><mtext>r</mtext><mspace width="0.25em" /><mtext>t</mtext><mo>≥</mo><mtext>Μ</mtext><mtext>A</mtext><mtext>X</mtext><mo>_</mo><mtext>Ν</mtext></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="134">13.计算U中每行的最小值min_row, 如果min_row≤τ, 删除该特征</p>
                </div>
                <div class="p1">
                    <p id="135">14.将U中的每一行的最大值置1, 其他全部置0, 确定每个样本的最终分类</p>
                </div>
                <div class="p1">
                    <p id="136">在GN-FCM算法中, 第1行是计算每个SNP的权重<i>g</i><sub><i>j</i></sub>, 并将其保存在表中, 以便在使用时直接读取;第2行和第3行是初始化隶属度矩阵和簇中心;第4行～第11行表示根据式 (13) 和式 (15) 来不断地更新隶属度矩阵和簇中心;第12行是算法的停止条件, 当前后两次的<i>u</i><sub><i>ij</i></sub>变化很小时说明算法已经是收敛;第13行表示将那些所属类别不明确的SNP直接删除, 最后将<i>u</i><sub><i>ij</i></sub>调整为0或1以确定每个SNP的所属类别。</p>
                </div>
                <h4 class="anchor-tag" id="137" name="137">2.3 GN-FCM在SNP选择中的应用</h4>
                <div class="p1">
                    <p id="138">下文将结合假设性检验和GN-FCM对SNP进行选择。首先基于假设性检验, 初步剔除那些不符合遗传统计的基本规律的和数据, 达到数据清洗和初步降维的目的;然后对剩下的SNP进行聚类;最后通过基于对称的不确定性 (SU) 方法<citation id="302" type="reference"><link href="276" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>从每个簇中选择SNP, 形成最终的SNP子集。SNP选择流程如图1所示。</p>
                </div>
                <div class="area_img" id="139">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201908012_139.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 SNP选择流程" src="Detail/GetImg?filename=images/JSJC201908012_139.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图1 SNP选择流程</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201908012_139.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="140" name="140">2.3.1 基于假设性检验的初步筛选</h4>
                <div class="p1">
                    <p id="141">遗传相关的分析通常使用最小等位基因频率 (Minor Allele Frequency, MAF) 这个指标。在Hapmap研究中, 通常不将MAF小于0.05的SNP作为研究对象, 原因是过小的MAF会使得统计效能降低, 从而导致假阴性的结果。因此, 可以事先对SNP进行MAF的统计, 将MAF小于0.05的SNP从数据集中剔除。此外, 哈代-温伯格平衡法则也可以在SNP的筛选方面起很大的作用。哈代-温伯格定律是指在一个相对理想的条件下 (指不发生突变、迁移和选择无限大的相互交配) , 群体的基因频率和基因型频率将保持不变。根据该法则, 如果基因A的频率是<i>p</i>, a的基因频率是<i>q</i>, 则基因型AA的频率为<i>p</i><sup>2</sup>, aa的频率为<i>q</i><sup>2</sup>, Aa的频率为2<i>pq</i>, 并且<i>p</i><sup>2</sup>+<i>q</i><sup>2</sup>+2<i>pq</i>=1。通过卡方检验, 就可以检验实际数据集中的频率和由平衡法则得到的期望的频率之间的误差。卡方检验公式如式 (16) 所示。</p>
                </div>
                <div class="p1">
                    <p id="142" class="code-formula">
                        <mathml id="142"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>χ</mi><msup><mrow></mrow><mn>2</mn></msup><mo>=</mo><mstyle displaystyle="true"><mo>∑</mo><mrow><mfrac><mrow><mrow><mo stretchy="false"> (</mo><mi>R</mi><mo>-</mo><mi>E</mi><mo stretchy="false">) </mo></mrow><msup><mrow></mrow><mn>2</mn></msup></mrow><mi>E</mi></mfrac></mrow></mstyle><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>6</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="143">其中, <i>R</i>表示实际值, <i>E</i>表示理论值。</p>
                </div>
                <h4 class="anchor-tag" id="144" name="144">2.3.2 基于SU的选择策略</h4>
                <div class="p1">
                    <p id="145">聚类后从每个簇中选择合适的特征是常用的方法。一般地, 如果使用基于模型的贪婪选择方法, 由于每训练一次只能确定一个特征, 因此会导致整个选择阶段的时间开销太大, 而且选择得到的特征子集是局部最优的。本文使用了一种在聚类里常用的方法——对称的不确定性<citation id="303" type="reference"><link href="276" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>来特征选择。SU衡量的是2个离散变量之间的依赖程度, 也可以表示他们之间包含的共同信息量。SU的定义如下:</p>
                </div>
                <div class="p1">
                    <p id="146" class="code-formula">
                        <mathml id="146"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>S</mi><mi>U</mi><mo stretchy="false"> (</mo><mi>X</mi><mo>, </mo><mi>Y</mi><mo stretchy="false">) </mo><mo>=</mo><mn>2</mn><mo>×</mo><mfrac><mrow><mi>Η</mi><mo stretchy="false"> (</mo><mi>X</mi><mo stretchy="false">) </mo><mo>-</mo><mi>Η</mi><mo stretchy="false"> (</mo><mi>X</mi><mo stretchy="false">|</mo><mi>Y</mi><mo stretchy="false">) </mo></mrow><mrow><mi>Η</mi><mo stretchy="false"> (</mo><mi>X</mi><mo stretchy="false">) </mo><mo>+</mo><mi>Η</mi><mo stretchy="false"> (</mo><mi>Y</mi><mo stretchy="false">) </mo></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>7</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="147">其中, <i>H</i> (<i>X</i>) 表示的是变量<i>X</i>的信息熵, 如式 (5) 所示, <i>H</i> (<i>X</i>|<i>Y</i>) 定义如式 (18) 所示。</p>
                </div>
                <div class="p1">
                    <p id="148" class="code-formula">
                        <mathml id="148"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Η</mi><mo stretchy="false"> (</mo><mi>X</mi><mo stretchy="false">|</mo><mi>Y</mi><mo stretchy="false">) </mo><mo>=</mo><mo>-</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mi>j</mi></munder><mi>Ρ</mi></mstyle><mo stretchy="false"> (</mo><mi>y</mi><msub><mrow></mrow><mi>j</mi></msub><mo stretchy="false">) </mo><mstyle displaystyle="true"><munder><mo>∑</mo><mi>i</mi></munder><mi>Ρ</mi></mstyle><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">|</mo><mi>y</mi><msub><mrow></mrow><mi>j</mi></msub><mo stretchy="false">) </mo><mtext>l</mtext><mtext>b</mtext><mtext> </mtext><mi>Ρ</mi><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">|</mo><mi>y</mi><msub><mrow></mrow><mi>j</mi></msub><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>8</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="149">其中, <i>P</i> (<i>y</i><sub><i>j</i></sub>) 是变量<i>Y</i>取值为<i>y</i><sub><i>j</i></sub>时的概率, <i>P</i> (<i>x</i><sub><i>i</i></sub>|<i>y</i><sub><i>j</i></sub>) 是对于给定的变量<i>Y</i>在取值为<i>y</i><sub><i>j</i></sub>时变量<i>X</i>的取值为<i>x</i><sub><i>i</i></sub>的概率。按照“最大相关-最小冗余”原则, 一个特征子集要能代表整个特征集合, 则该特征子集是由与分类结果最相关的特征以及与该特征冗余最小的特征共同组成<citation id="304" type="reference"><link href="278" rel="bibliography" /><sup>[<a class="sup">23</a>]</sup></citation>, 如式 (19) 所示。</p>
                </div>
                <div class="p1">
                    <p id="150" class="code-formula">
                        <mathml id="150"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>s</mi><mi>u</mi><mi>b</mi><mo stretchy="false"> (</mo><mi>c</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">) </mo><mo>=</mo><mo stretchy="false">{</mo><mrow><mi>arg</mi></mrow><munder><mstyle mathsize="140%" displaystyle="true"><mrow><mi>max</mi></mrow></mstyle><mi>i</mi></munder><mfrac><mn>1</mn><mi>n</mi></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>i</mi><mo>∈</mo><mi>c</mi><msub><mrow></mrow><mi>k</mi></msub><mstyle displaystyle="true"><mo>∩</mo><mi>i</mi></mstyle><mo>≠</mo><mi>j</mi></mrow></munder><mi>S</mi></mstyle><mi>U</mi><mo stretchy="false"> (</mo><mi>i</mi><mo>, </mo><mi>j</mi><mo stretchy="false">) </mo><mo stretchy="false">}</mo><mo>+</mo></mtd></mtr><mtr><mtd><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">{</mo><mrow><mi>arg</mi></mrow><munder><mstyle mathsize="140%" displaystyle="true"><mrow><mi>min</mi></mrow></mstyle><mrow><mi>k</mi><mo>∈</mo><mi>c</mi><msub><mrow></mrow><mi>k</mi></msub><mo>\</mo><mi>i</mi></mrow></munder><mi>S</mi><mi>U</mi><mo stretchy="false"> (</mo><mi>i</mi><mo>, </mo><mi>k</mi><mo stretchy="false">) </mo><mo stretchy="false">}</mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>9</mn><mo stretchy="false">) </mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="151">其中, <i>c</i><sub><i>k</i></sub>表示的是聚类后的每个簇。等式右边的第1部分用簇中具有最大平均SU的一个特征来表示最相关的特征, 第2部分用具有与该特征最小SU的特征来表示最不冗余的特征。但是考虑到不同的簇可能包含的SNP的个数有所差异, 如果同等地看待每个簇自然是不合理的, 因此, 可以根据簇的大小来调整选择的个数。定义如下运算:</p>
                </div>
                <div class="p1">
                    <p id="152"><i>sub</i><sup>*</sup> (<i>c</i><sub><i>k</i></sub>) =<i>η</i><sub><i>k</i></sub>⊙<i>sub</i> (<i>c</i><sub><i>k</i></sub>)      (20) </p>
                </div>
                <div class="p1">
                    <p id="153">其中, ⊙运算表示重复执行<i>sub</i> (<i>c</i><sub><i>k</i></sub>) 选择策略<i>η</i><sub><i>k</i></sub>次, 这样就可以形成一个2<i>η</i><sub><i>k</i></sub>大小的子集。<i>η</i><sub><i>k</i></sub>是一个大于等于1的整数系数, 定义如下:</p>
                </div>
                <div class="p1">
                    <p id="154"><i>η</i><sub><i>k</i></sub>=<image href="images/JSJC201908012_155.jpg" type="" display="inline" placement="inline"><alt></alt></image><mathml id="156"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mrow><mrow><mo>|</mo><mrow><mi>c</mi><msub><mrow></mrow><mrow><mi>max</mi></mrow></msub></mrow><mo>|</mo></mrow><mo>-</mo><mrow><mo>|</mo><mrow><mi>c</mi><msub><mrow></mrow><mrow><mi>min</mi></mrow></msub></mrow><mo>|</mo></mrow></mrow><mrow><mrow><mo>|</mo><mrow><mrow><mo>|</mo><mrow><mi>c</mi><msub><mrow></mrow><mi>k</mi></msub></mrow><mo>|</mo></mrow><mo>-</mo><mover accent="true"><mrow><mrow><mo>|</mo><mi>c</mi><mo>|</mo></mrow></mrow><mo stretchy="true">¯</mo></mover></mrow><mo>|</mo></mrow></mrow></mfrac></mrow></math></mathml><image href="images/JSJC201908012_157.jpg" type="" display="inline" placement="inline"><alt></alt></image> (21) </p>
                </div>
                <div class="p1">
                    <p id="158">其中, <mathml id="159"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>|</mo><mrow><mi>c</mi><msub><mrow></mrow><mrow><mi>max</mi></mrow></msub></mrow><mo>|</mo></mrow></mrow></math></mathml>、<mathml id="160"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>|</mo><mrow><mi>c</mi><msub><mrow></mrow><mrow><mi>min</mi></mrow></msub></mrow><mo>|</mo></mrow></mrow></math></mathml>、<mathml id="161"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>|</mo><mrow><mi>c</mi><msub><mrow></mrow><mi>k</mi></msub></mrow><mo>|</mo></mrow></mrow></math></mathml>以及<mathml id="162"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mover accent="true"><mrow><mrow><mo>|</mo><mi>c</mi><mo>|</mo></mrow></mrow><mo stretchy="true">¯</mo></mover></mrow></math></mathml>表示所有簇中最大的簇的大小、最小簇的大小、当前某一个簇的大小以及所有簇的平均大小。从每个簇中选择特征的流程如图2所示。</p>
                </div>
                <div class="area_img" id="163">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201908012_163.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 特征选择流程" src="Detail/GetImg?filename=images/JSJC201908012_163.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图2 特征选择流程</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201908012_163.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h3 id="164" name="164" class="anchor-tag">3 实验</h3>
                <h4 class="anchor-tag" id="165" name="165">3.1 实验环境与数据</h4>
                <div class="p1">
                    <p id="166">实验环境:编译工具Python 3.6.0, 操作系统Windows 10, CPU/Intel (R) Core (TM) i5-3210M双核处理器, 主频2.5 GHz, 内存8 GB, 硬盘容量750 GB。</p>
                </div>
                <div class="p1">
                    <p id="167">实验数据:本次实验所使用的数据均由无锡市精神卫生中心提供。数据格式为遗传病的SNP基因型格式, 并且每个样本都标明健康或者患病的状态。数据的描述如表1所示。</p>
                </div>
                <div class="area_img" id="168">
                    <p class="img_tit"><b>表1 数据集描述</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="168" border="1"><tr><td><br />数据集</td><td>样本个数</td><td>SNP个数</td><td>健康/患病数目</td></tr><tr><td><br />G1000</td><td>1 000</td><td>9 445</td><td>500/500</td></tr><tr><td><br />E144</td><td>144</td><td>251 482</td><td>72/72</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h4 class="anchor-tag" id="169" name="169">3.2 实验评价指标</h4>
                <div class="p1">
                    <p id="170">实验评价指标如下:</p>
                </div>
                <h4 class="anchor-tag" id="171" name="171">1) 聚类效果的评价指标</h4>
                <div class="p1">
                    <p id="172">本文使用聚类评价方法中最常用的簇内紧致度<i>Com</i>与簇间分散度<i>Spt</i>之间的比值衡量聚类效果。两者的定义如式 (22) 和式 (23) 所示。</p>
                </div>
                <div class="p1">
                    <p id="173" class="code-formula">
                        <mathml id="173"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>C</mi><mi>o</mi><mi>m</mi><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>c</mi></munderover><mi>u</mi></mstyle></mrow></mstyle><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mi>m</mi></msubsup><mrow><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">) </mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>2</mn><mn>2</mn><mo stretchy="false">) </mo></mtd></mtr><mtr><mtd><mi>S</mi><mi>p</mi><mi>t</mi><mo>=</mo><munder><mstyle mathsize="140%" displaystyle="true"><mrow><mi>min</mi></mrow></mstyle><mrow><mi>i</mi><mo>≠</mo><mi>k</mi></mrow></munder><mrow><mo stretchy="false">∥</mo><mi>v</mi><msub><mrow></mrow><mi>i</mi></msub><mo>-</mo><mi>v</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">∥</mo></mrow><msup><mrow></mrow><mn>2</mn></msup><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>2</mn><mn>3</mn><mo stretchy="false">) </mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <h4 class="anchor-tag" id="174" name="174">2) 分类预测的评价指标</h4>
                <div class="p1">
                    <p id="175">对于分类实验的评价, 本文使用精准率<i>precision</i>、召回率<i>recall</i>、F1指标<i>F</i>-<i>Measure</i>以及预测的准确率<i>accuracy</i>来评价, 评价指标可以通过混淆矩阵来计算, 如图3所示。</p>
                </div>
                <div class="area_img" id="233">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201908012_23300.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 混淆矩阵" src="Detail/GetImg?filename=images/JSJC201908012_23300.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 混淆矩阵  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201908012_23300.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="178">根据上述矩阵, 计算如式 (24) ～式 (27) 所示的评价指标。</p>
                </div>
                <div class="p1">
                    <p id="179"><i>precision</i>=<i>TP</i>/ (<i>TP</i>+<i>FP</i>)      (24) </p>
                </div>
                <div class="p1">
                    <p id="180"><i>recall</i>=<i>TP</i>/ (<i>TP</i>+<i>FN</i>)      (25) </p>
                </div>
                <div class="p1">
                    <p id="181" class="code-formula">
                        <mathml id="181"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>F</mi><mo>-</mo><mi>m</mi><mi>e</mi><mi>a</mi><mi>s</mi><mi>u</mi><mi>r</mi><mi>e</mi><mo>=</mo><mfrac><mrow><mn>2</mn><mo>×</mo><mi>p</mi><mi>r</mi><mi>e</mi><mi>c</mi><mi>i</mi><mi>s</mi><mi>i</mi><mi>o</mi><mi>n</mi><mo>×</mo><mi>r</mi><mi>e</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>l</mi></mrow><mrow><mi>p</mi><mi>r</mi><mi>e</mi><mi>c</mi><mi>i</mi><mi>s</mi><mi>i</mi><mi>o</mi><mi>n</mi><mo>+</mo><mi>r</mi><mi>e</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>l</mi></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>2</mn><mn>6</mn><mo stretchy="false">) </mo></mtd></mtr><mtr><mtd><mi>a</mi><mi>c</mi><mi>c</mi><mi>u</mi><mi>r</mi><mi>a</mi><mi>c</mi><mi>y</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>Τ</mi><mi>Ν</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi><mo>+</mo><mi>Τ</mi><mi>Ν</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>2</mn><mn>7</mn><mo stretchy="false">) </mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <h4 class="anchor-tag" id="182" name="182">3.3 数据预处理</h4>
                <div class="p1">
                    <p id="183">由于实验的原始数据是SNP的基因型表示, 而这种数据不利于后续的聚类操作, 因此首先要对数据进行重新编码, 将其转换成利于操作的数值型。本次实验采用的编码方式是“0-1-2”编码, 分别表示AA、Aa以及aa。编码完成后, 还需要进一步对缺失值进行填充。考虑到SNP局部可能存在关联性, 所以使用K近邻方法对其进行填充, 与此同时, 将缺失值数量大于10%的样本进行删除。在假设性检验时, 先将MAF的阈值设为0.05, 根据实验结果, 数据集G1000的MAF值均大于0.05, 而数据集E144中MAF值小于0.05的共有32 655条;将卡方检验的P-value阈值设为7.387, 结果在数据集G1000上有228条SNP的P-value大于7.387, 在数据集E144上有5 172条。将不满足条件的SNP剔除, 最后2个数据集分别剩下9 217和213 733条有效的SNP, 用于下一阶段的实验。假设性检验的结果以数据E1000为例, 如图4所示。</p>
                </div>
                <div class="area_img" id="184">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201908012_184.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 数据集G1000的卡方检验结果" src="Detail/GetImg?filename=images/JSJC201908012_184.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图4 数据集G1000的卡方检验结果</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201908012_184.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="185" name="185">3.4 实验结果与分析</h4>
                <h4 class="anchor-tag" id="186" name="186">3.4.1 聚类实验及结果分析</h4>
                <div class="p1">
                    <p id="187">该部分的聚类实验主要包括3个算法的对比, 分别是FCM、DW-FCM和GN-FCM。实验主要从算法的迭代次数、算法的运行时间以及算法聚类效果这三个方面来进行对比。实验部分统一设置模糊因子<i>m</i>=2, 终止条件<i>ε</i>=0.001。</p>
                </div>
                <h4 class="anchor-tag" id="188" name="188">1) 迭代次数比较</h4>
                <div class="p1">
                    <p id="189">在2个数据集上分别对FCM、DW-FCM和GN-FCM 3种算法进行实验, 并比较每种算法达到收敛时的迭代次数, 以此来比较三者的收敛性。每个算法分别在2个数据集上进行5组实验, 对于给定的聚类个数<i>c</i>, 结果如图5和图6所示 (每组实验重复3次并对结果取均值) 。可以看出, 随着聚类个数<i>c</i>的增加, 算法达到收敛时需要的迭代次数也相应增加, 但是相比FCM、DW-FCM, GN-FCM需要更少的迭代次数就可以达到收敛, 几乎是FCM的一半, 这说明GN-FCM具有更好的收敛性。</p>
                </div>
                <div class="area_img" id="190">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201908012_190.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图5 在数据集G1000上的迭代次数比较" src="Detail/GetImg?filename=images/JSJC201908012_190.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图5 在数据集G1000上的迭代次数比较</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201908012_190.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="191">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201908012_191.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图6 在数据集E144上的迭代次数比较" src="Detail/GetImg?filename=images/JSJC201908012_191.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图6 在数据集E144上的迭代次数比较</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201908012_191.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="192" name="192">2) 运行时间比较</h4>
                <div class="p1">
                    <p id="193">在2个数据集上分别对FCM、DW-FCM和GN-FCM 3种算法进行实验, 并比较每种算法达到收敛时的运行时间。每个算法分别在2个数据集上进行5组实验, 对于给定的聚类个数<i>c</i>, 结果如图7和图8所示 (每组实验重复3次并对结果取均值) 。可以看出, 随着聚类个数<i>c</i>的增加, 算法达到收敛时需要的时间也相应增加。DW-FCM所需要的时间最短, 紧接着的是FCM, 而GN-FCM需要的时间相比前两个略长。造成该结果的原因是, GN-FCM中考虑了高权重的SNP与局部SNP的影响, 所以在迭代的过程中增加了额外的距离计算开销。然而在聚类数目较少时, 这种时间开销与FCM和DW-FCM相比差距并不大。</p>
                </div>
                <div class="area_img" id="194">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201908012_194.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图7 在数据集G1000上的迭代时间比较" src="Detail/GetImg?filename=images/JSJC201908012_194.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图7 在数据集G1000上的迭代时间比较</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201908012_194.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="195">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201908012_195.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图8 在数据集E144上的迭代时间比较" src="Detail/GetImg?filename=images/JSJC201908012_195.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图8 在数据集E144上的迭代时间比较</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201908012_195.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="196" name="196">3) 聚类效果比较</h4>
                <div class="p1">
                    <p id="197">实验采用<i>Com</i>和<i>Spt</i>之间的比值来对比3种算法的效果, 比值越大说明不同簇间的差异性越大, 同一簇内部的紧致度越高, 从而说明聚类的效果越好。3种算法分别在数据集G1000和E144上的聚类性能如表2和表3所示, 其中加粗数据表示最大数据。</p>
                </div>
                <div class="area_img" id="198">
                    <p class="img_tit"><b>表2 3种聚类方法在数据集E114上的聚类性能</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="198" border="1"><tr><td><br />聚类个数</td><td>FCM算法</td><td>DW-FCM算法</td><td>GN-FCM算法</td></tr><tr><td><br />5</td><td>36 998.16</td><td><b>38 421.67</b></td><td>37 381.73</td></tr><tr><td><br />6</td><td>38 804.89</td><td>38 990.21</td><td><b>39 041.49</b></td></tr><tr><td><br />7</td><td>37 856.24</td><td>37 224.09</td><td><b>38 291.21</b></td></tr><tr><td><br />8</td><td>28 965.64</td><td>27 982.19</td><td><b>29 934.42</b></td></tr><tr><td><br />9</td><td>26 650.44</td><td>27 553.81</td><td><b>28 037.20</b></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="area_img" id="199">
                    <p class="img_tit"><b>表3 3种聚类方法在数据集E144上的聚类性能</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="199" border="1"><tr><td><br />聚类个数</td><td>FCM算法</td><td>DW-FCM算法</td><td>GN-FCM算法</td></tr><tr><td><br />5</td><td>36 960.65</td><td>41 029.28</td><td><b>45 152.95</b></td></tr><tr><td><br />6</td><td>38 757.27</td><td>38 056.51</td><td><b>52 590.30</b></td></tr><tr><td><br />7</td><td>31 127.70</td><td><b>41 022.38</b></td><td>39 920.49</td></tr><tr><td><br />8</td><td>28 867.67</td><td>30 127.56</td><td><b>38 561.68</b></td></tr><tr><td><br />9</td><td>25 304.22</td><td>25 559.15</td><td><b>26 041.40</b></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="200">对比表2和表3可知, GN-FCM在2个数据集上都表现出较好的聚类性能, 在5组实验中均有4次指标达到最大, 而DW-FCM各有一次效果达到最好。而对于GN-FCM而言, 在聚类的个数<i>c</i>=6时, 在2个数据集上表现最好, 因此, 在后续的实验中将<i>c</i>=6定为终要聚类的数目。</p>
                </div>
                <h4 class="anchor-tag" id="201" name="201">3.4.2 SNP子集评价</h4>
                <div class="p1">
                    <p id="202">在数据集G1000和数据集E144上, 当聚类个数<i>c</i>=6的时候效果最好, 因此, 选择这2个聚类结果, 然后根据2.3节中的选择策略进行构造SNP子集, 最终在两个数据集上分别选出了52个和72个SNP, 每个簇中选择出的SNP的具体情况如表4所示。</p>
                </div>
                <div class="area_img" id="203">
                    <p class="img_tit"><b>表4 在每个簇中选择的SNP数目</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="203" border="1"><tr><td><br />数据集</td><td>簇1</td><td>簇2</td><td>簇3</td><td>簇4</td><td>簇5</td><td>簇6</td></tr><tr><td>G1000</td><td>8</td><td>12</td><td>4</td><td>14</td><td>8</td><td>6</td></tr><tr><td><br />E144</td><td>18</td><td>6</td><td>12</td><td>8</td><td>12</td><td>6</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="204">为进一步检验最终构造的SNP子集所包含信息的重要程度, 本实验使用支持向量机 (Support Vector Machine, SVM) 、决策树 (Decision Tree, DT) 以及朴素贝叶斯 (Naïve Bayesian, NB) 分类器分别进行分类预测, 使用<i>F</i>-<i>measure</i>和分类的准确率<i>accuracy</i>作为评价指标。参与比较的SNP选择方法包括DW-FCM、ReliefF、MRMR<citation id="305" type="reference"><link href="280" rel="bibliography" /><sup>[<a class="sup">24</a>]</sup></citation>以及本文提出的GN-FCM。经过10折交叉验证训练后, 实验数据如表5所示。其中, “-”表示不加假设性检验的结果, “+”表示加入假设性检验后的结果, 加粗数据表示最好的结果。</p>
                </div>
                <div class="p1">
                    <p id="205">由表5中的结果可知, 在使用不同的分类器的分类实验中, 采用新提出的选择方法GN-FCM构造的SNP子集较DW-FCM方法在各项评价指标上均提升了6%～9%, 说明GN-FCM在SNP的选择中具有较大的优势。值得注意的是, 使用基于假设性检验的初步筛选方法, 使得分类的准确率进一步提升了5%～10%, 表明了预先剔除遗传意义较小的SNP对于构造最终的SNP子集具有良好的效果。另外, ReliefF和MRMR方法在不同的数据集上都有不错的表现, 尤其是MRMR, 在准确率和F1指标上接近甚至在个别分类实验中超过GN-FCM。但总体来说, GN-FCM选择方法的分类效果更好。</p>
                </div>
                <div class="area_img" id="206">
                    <p class="img_tit"><b>表5 基于不同分类器的分类结果评价</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note">%</p>
                    <table id="206" border="1"><tr><td rowspan="2">数据集</td><td rowspan="2">选择策略</td><td colspan="2"><br />SVM分类器</td><td colspan="2"><br />DT分类器</td><td colspan="2"><br />NB分类器</td></tr><tr><td><br /><i>accuracy</i></td><td><i>F</i>-<i>measure</i></td><td><br /><i>accuracy</i></td><td><i>F</i>-<i>measure</i></td><td><br /><i>accuracy</i></td><td><i>F</i>-<i>measure</i></td></tr><tr><td rowspan="5"><br />G1000</td><td>DW-FCM (-) </td><td>62.98</td><td>49.91</td><td>59.18</td><td>49.16</td><td>53.11</td><td>50.26</td></tr><tr><td><br />GN-FCM (-) </td><td>67.37</td><td>55.59</td><td>64.30</td><td>49.52</td><td>68.58</td><td>60.17</td></tr><tr><td><br />GN-FCM (+) </td><td><b>74.60</b></td><td><b>63.86</b></td><td><b>68.25</b></td><td><b>60.57</b></td><td><b>73.99</b></td><td>64.27</td></tr><tr><td><br />ReliefF</td><td>65.49</td><td>50.20</td><td>60.19</td><td>54.26</td><td>70.51</td><td><b>69.41</b></td></tr><tr><td><br />MRMR</td><td>70.25</td><td>59.81</td><td>65.54</td><td>55.72</td><td>68.88</td><td>60.46</td></tr><tr><td rowspan="5"><br />E144</td><td>DW-FCM (-) </td><td>59.09</td><td>49.40</td><td>55.22</td><td>50.46</td><td>58.59</td><td>50.41</td></tr><tr><td><br />GN-FCM (-) </td><td>64.74</td><td>50.89</td><td>62.15</td><td>51.97</td><td>64.62</td><td>60.37</td></tr><tr><td><br />GN-FCM (+) </td><td><b>80.15</b></td><td>67.10</td><td><b>65.84</b></td><td><b>57.95</b></td><td><b>78.92</b></td><td><b>66.42</b></td></tr><tr><td><br />ReliefF</td><td>68.68</td><td><b>70.51</b></td><td>61.77</td><td>53.06</td><td>66.50</td><td>59.89</td></tr><tr><td><br />MRMR</td><td>72.84</td><td>64.36</td><td>63.39</td><td>56.40</td><td>74.82</td><td>64.21</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h3 id="207" name="207" class="anchor-tag">4 结束语</h3>
                <div class="p1">
                    <p id="208">针对SNP高维少样本特性以及自身存在的遗传规律, 本文同时考虑单个SNP对分类结果的影响以及局部区域SNP之间的相互关联性, 提出一种基于FCM的方法GN-FCM, 将其运用到SNP选择中。聚类效果评估和分类实验结果均表明, 该方法能够提高SNP选择的准确性。下一步将考虑局部SNP相互影响导致的额外计算开销, 同时研究聚类个数的确定方法, 以期在初始化阶段即确定最优的聚类个数。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="234">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Genome-wide association study identifies five new schizophrenia loci">

                                <b>[1]</b> RIPKE S, SANDERS A R, KENDLER K S, et al.The schizophrenia psychiatric genome-wide association study (GWAS) consortium genome-wide association study identifies five new schizophrenia loci[J].Nature Genetics, 2011, 42 (1) :36-44.
                            </a>
                        </p>
                        <p id="236">
                            <a id="bibliography_2" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJDP&amp;filename=SJDP120724001467&amp;v=MTUzNDY5dUZDdmhVN2pOSWw4VE5pZlBmcks2SHRiT3E0OUZaZThKQ3hNOHp4VVNtRGQ5U0g3bjN4RTlmYnZuS3JpZlp1&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[2]</b> KATE M A, STEPHEN E, GISELA O.Genetics of rheumatoid arthritis:GWAS and beyond[J].Open Access Rheumatology:Research and Reviews, 2011, 3:31-46.
                            </a>
                        </p>
                        <p id="238">
                            <a id="bibliography_3" >
                                    <b>[3]</b>
                                 王巍.基因组关联研究中Tag SNP选择算法优化的研究[D].北京:北京大学, 2008.
                            </a>
                        </p>
                        <p id="240">
                            <a id="bibliography_4" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011702006871&amp;v=MjA0NzBmT2ZiSzdIdEROcUk5SFpPc0pCSHM0b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadUh5am1VTHZJSjFzV2F4VT1OaQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[4]</b> CONSOLE L, PICARDI C, RIBAUDO M.Process algebras for systems diagnosis[J].Artificial Intelligence, 2002, 142 (1) :19-51.
                            </a>
                        </p>
                        <p id="242">
                            <a id="bibliography_5" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM9D0BFDA045FB346D2A93BD2C903C14F4&amp;v=MDM5NTU9TmlmSVk3ck1IcU82Mi81RllPNTVmbjg5eVdJUm16WitPZ3ZnM3hzMWVzR1ZRY3liQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TnhodzcyOXdhOA==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[5]</b> LI Xiong, LIAO Bo, CAI Lijun, et al.Informative SNPs selection based on two-locus and multilocus linkage disequilibrium:criteria of max-correlation and min-redundancy[J].IEEE/ACM Transactions on Computational Biology and Bioinformatics, 2013, 10 (3) :688-695.
                            </a>
                        </p>
                        <p id="244">
                            <a id="bibliography_6" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00001340690&amp;v=MjI5OTVQWTNrNXpCZGg0ajk5U1hxUnJ4b3hjTUg3UjdxZWJ1ZHRGQ3psVkxyS0lWaz1OajdCYXJPNEh0SE5ySXRGWXVJ&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[6]</b> ROBNIKŠIKONJA M, KONONENKO I.Theoretical and empirical analysis of ReliefF and RReliefF[J].Machine Learning, 2003, 53 (1/2) :23-69.
                            </a>
                        </p>
                        <p id="246">
                            <a id="bibliography_7" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A hybrid feature selection method for complex diseases SNPs">

                                <b>[7]</b> ALZUBI R, RAMZAN N, ALZOUBI H, et al.A hybrid feature selection method for complex diseases SNPs[J].IEEE Access, 2017, 6:1292-1301.
                            </a>
                        </p>
                        <p id="248">
                            <a id="bibliography_8" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCW&amp;filename=SJCW9DA6838C92FEA77E257ADE92EBAF25AB&amp;v=MTE3MzlXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU54aHc3Mjl3YTg9TmlmSWVick1iOWZFckljMmJlbDVlUTAreUdNUjd6Z01QQXJycm1kSENNU1dRTXZ0Q09OdkZTaQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[8]</b> ILHAN I, TEZEL G.Tag SNP selection using clonal selection and majority voting algorithms[J].International Journal of Data Mining and Bioinformatics, 2016, 16 (4) :290-311.
                            </a>
                        </p>
                        <p id="250">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A systematic SNP selection approach to identify mechanisms underlying disease aetiology:linking height to post-menopausal breast and colorectal cancer risk">

                                <b>[9]</b> ELANDS R J, SIMONS C C, RIEMENSCHNEIDER M, et al.A systematic SNP selection approach to identify mechanisms underlying disease aetiology:linking height to post-menopausal breast and colorectal cancer risk[EB/OL]. (2017-01-24) [2018-07-01].http://europepmc.org/backend/ptpmcrend er.fcgi?accid=PMC5259777&amp; blobtype=pdf.
                            </a>
                        </p>
                        <p id="252">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=PCA-based informative SNP selection for analyzing population structure">

                                <b>[10]</b> LIMPITI T, INTARAPANICH A, TONGSIMA S.PCA-based informative SNP selection for analyzing population structure[C]//Proceedings of the 7th International Con-ference on Computational Systems-Biology and Bioinfor-matics.New York, USA:ACM Press, 2016:36-40.
                            </a>
                        </p>
                        <p id="254">
                            <a id="bibliography_11" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDF905A110DDA360776C673FC6C379378B&amp;v=Mjg0NzJRckx0Q09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQ3BiUTM1TnhodzcyOXdhOD1OajdCYXNXeEh0UzlybzVGRUo5K0QzbzV5QkVWbVRsNlN3bVJxbUUyZnJ1WA==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[11]</b> SOLLERO B P, JUNQUEIRA V S, GOMES C C G, et al.Tag SNP selection for prediction of tick resistance in Brazilian Braford and Hereford cattle breeds using Bayesian methods[J].Genetics Selection Evolution, 2017, 49 (1) :49.
                            </a>
                        </p>
                        <p id="256">
                            <a id="bibliography_12" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJAH&amp;filename=SJAH17E13DAE125232B3371EA9C00544DDA8&amp;v=MTE3NjFiUTM1TnhodzcyOXdhOD1OaWZLWnJLL2E5RFAyLzR3WmVrS0RuODd2UlVRN1Q0SU9YYVJyQkl3ZmJiZ01jdVhDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZDcA==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[12]</b> SHI Wei, CAO Zhi, LI Xiong, et al.Informative SNPs selection based on fuzzy clustering and genetic algorithm[J].Journal of Computational and Theoretical Nanoscience, 2017, 14 (3) :1440-1445.
                            </a>
                        </p>
                        <p id="258">
                            <a id="bibliography_13" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJESC430A0FF603B7C0BEF48EA5FFCBF760D&amp;v=MzA2NjEyUm1uRHQxUFE3bjJtUkdDOFNUUTdyckNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU54aHc3Mjl3YTg9TmlmT2ZjQzhIZEc5ci9rell1c01mbnRLeg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[13]</b> CHORMUNGE S, JENA S.Correlation based feature selection with clustering for high dimensional data[J].Journal of Electrical Systems and Information Technology, 2018, 5 (3) :542-549.
                            </a>
                        </p>
                        <p id="260">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=FCM: The fuzzy c -means clustering algorithm">

                                <b>[14]</b> BEZDEK J C, EHRLICH R, FULL W.FCM:the fuzzy c-means clustering algorithm[J].Computers and Geosciences, 1984, 10 (2) :191-203.
                            </a>
                        </p>
                        <p id="262">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Cancer classification using fuzzy c-means with feature selection">

                                <b>[15]</b> RACHMAN A A, RUSTAM Z.Cancer classification using fuzzy c-means with feature selection[C]//Proceedings of International Conference on Mathematics, Statistics, and Their Applications.Washington D.C., USA:IEEE Press, 2017:31-34.
                            </a>
                        </p>
                        <p id="264">
                            <a id="bibliography_16" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=DZYX201006004&amp;v=MDc0MjZPZVplUnFGeS9nVWI3UElUZlNkckc0SDlITXFZOUZZSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkw=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[16]</b> 唐成龙, 王石刚, 徐威.基于数据加权策略的模糊聚类改进算法[J].电子与信息学报, 2010, 32 (6) :1277-1283.
                            </a>
                        </p>
                        <p id="266">
                            <a id="bibliography_17" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJK201802020&amp;v=MTAwMDNVUkxPZVplUnFGeS9nVWI3UEx6N0JaYkc0SDluTXJZOUhaSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckM=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[17]</b> 王印松, 商丹丹, 王艳飞, 等.改进的模糊聚类在控制系统故障诊断中的应用[J].计算机工程与科学, 2018, 40 (2) :326-330.
                            </a>
                        </p>
                        <p id="268">
                            <a id="bibliography_18" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES9E8E156E2769126EB1B66195883385FE&amp;v=MjE5NDJ6cUNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkNwYlEzNU54aHc3Mjl3YTg9TmlmT2Zick5GcVROcW9rd1p1d0pCWDA3eVdOaDYwMTdUbjdycVJvOWVyR2NRTQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[18]</b> DING Yi, FU Xian.Kernel-based fuzzy c-means clustering algorithm based on genetic algorithm[J].Neurocomputing, 2016, 188:233-238.
                            </a>
                        </p>
                        <p id="270">
                            <a id="bibliography_19" target="_blank" href="http://scholar.cnki.net/result.aspx?q=An improved fuzzy c-means algorithm for brain MRI image segmenta-tion">

                                <b>[19]</b> LI Min, ZHANG Limei, XIANG Zhikang, et al.An improved fuzzy c-means algorithm for brain MRI image segmenta-tion[C]//Proceedings of International Conference on Progress in Informatics and Computing.Washington D.C., USA:IEEE Press, 2017:336-339.
                            </a>
                        </p>
                        <p id="272">
                            <a id="bibliography_20" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Segmentation of M-FISH images for improved classification of chromosomes with an adaptive fuzzy C-means clustering algorithm">

                                <b>[20]</b> CAO Hongbao, DENG Hongwen, WANG Yuping.Segmentation of M-FISH images for improved classification of chromosomes with an adaptive fuzzy c-means clustering algorithm[J].IEEE Transactions on Fuzzy Systems, 2012, 20 (1) :1-8.
                            </a>
                        </p>
                        <p id="274">
                            <a id="bibliography_21" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A fuzzy relative of the ISODATA process and its use in detecting compact, well-separated clusters">

                                <b>[21]</b> DUNN J C.A fuzzy relative of the ISODATA process and its use in detecting compact well-separated clusters[J].Journal of Cybernetics, 1974, 3 (3) :32-57.
                            </a>
                        </p>
                        <p id="276">
                            <a id="bibliography_22" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011501719364&amp;v=Mjk4ODI0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMdklKMXNXYXhVPU5pZk9mYks3SHRETnFvOUVZK29HRDNvOW9CTVQ2VA==&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[22]</b> KANNAN S S, RAMARAJ N.A novel hybrid feature selection via symmetrical uncertainty ranking based local memetic search algorithm[J].Knowledge-Based Systems, 2010, 23 (6) :580-585.
                            </a>
                        </p>
                        <p id="278">
                            <a id="bibliography_23" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSYJ201505007&amp;v=MDg4NDlqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmVScUZ5L2dVYjdQTHo3U1pMRzRIOVRNcW85Rlk0UUtESDg0dlI0VDY=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[23]</b> 王连喜, 蒋盛益.一种基于特征聚类的特征选择方法[J].计算机应用研究, 2015, 32 (5) :1305-1308.
                            </a>
                        </p>
                        <p id="280">
                            <a id="bibliography_24" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Feature selection based on mutual information criteria of max-dependency, max-relevance, and min-redundancy">

                                <b>[24]</b> PENG Hanchuan, LONG Fuhui, DING C.Feature selection based on mutual information:criteria of max-dependency, max-relevance, and min-redundancy[J].IEEE Transactions on Pattern Analysis and Machine Intelligence, 2005, 27 (8) :1226-1238.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->

            <div class="btn-downloads">
                <span>
                    <a class="caj" target="_blank" href="http://kns.cnki.net/kns/download.aspx?filename=CVmYJB1Qw1EVhN0S4tEVZJmVMJzRw1Ud4dzSqh1QLV3aUhUOItWZINWdid3RLNjVptmbahWY4xWT5FXbnBnYmZXZTZVend3cvE0cPZHMWZGUVJ1c2Anajp1ZolXNPJjTLh3K3dXdsxGRiZ1T29UNN1kTJZDdiZVR&tablename=CJFDLAST2019">CAJ下载</a>
                    <a class="pdf" target="_blank" href="http://kns.cnki.net/kns/download.aspx?filename=CVmYJB1Qw1EVhN0S4tEVZJmVMJzRw1Ud4dzSqh1QLV3aUhUOItWZINWdid3RLNjVptmbahWY4xWT5FXbnBnYmZXZTZVend3cvE0cPZHMWZGUVJ1c2Anajp1ZolXNPJjTLh3K3dXdsxGRiZ1T29UNN1kTJZDdiZVR&tablename=CJFDLAST2019&dflag=pdfdown">PDF下载</a>
                </span>
                <p>永久保存本文,请下载至本地</p>
            </div>


    </div>

        <input id="fileid" type="hidden" value="JSJC201908012" />
        <input id="dpi" type="hidden" value="600" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201908012&amp;v=MDUwMTdCdEdGckNVUkxPZVplUnFGeS9nVWI3UEx6N0JiYkc0SDlqTXA0OUVab1FLREg4NHZSNFQ2ajU0TzN6cXE=&amp;uid=WEEvREcwSlJHSldRa1FhcTdWa2FjcW9xck9nSjc1VXR3SDJsaTcyc24rQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
            <a class="icon" id="copytext" target="_blank" href="http://x.cnki.net/search/common/testlunbo?dbcode=CJFD&amp;tablename=CJFDLAST2019&amp;filename=JSJC201908012&amp;filesourcetype=1">划线</a>
            <a class="icon" id="copytext" target="_blank" href="http://x.cnki.net/search/common/testlunbo?dbcode=CJFD&amp;tablename=CJFDLAST2019&amp;filename=JSJC201908012&amp;filesourcetype=1">笔记</a>
            <a class="icon" id="copytext" target="_blank" href="http://x.cnki.net/search/common/testlunbo?dbcode=CJFD&amp;tablename=CJFDLAST2019&amp;filename=JSJC201908012&amp;filesourcetype=1">文摘</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
