<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637129918027176250%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJSJC201907005%26RESULT%3d1%26SIGN%3dGgRfoGrQgoVoWLyZAD5pb%252bXe49k%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJC201907005&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJC201907005&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201907005&amp;v=MTEwOTdyclBMejdCYmJHNEg5ak1xSTlGWVlRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJwRkNqblU=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#41" data-title="0 概述 ">0 概述</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#45" data-title="1 跨平台分类模式 ">1 跨平台分类模式</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#46" data-title="1.1 最高得分对">1.1 最高得分对</a></li>
                                                <li><a href="#52" data-title="1.2 层级规则树">1.2 层级规则树</a></li>
                                                <li><a href="#64" data-title="1.3 k-HRT算法">1.3 k-HRT算法</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#69" data-title="2 快速挖掘算法 ">2 快速挖掘算法</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#93" data-title="2.1 数据转换">2.1 数据转换</a></li>
                                                <li><a href="#96" data-title="2.2 相对偏移表">2.2 相对偏移表</a></li>
                                                <li><a href="#105" data-title="2.3 层级规则树挖掘">2.3 层级规则树挖掘</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#112" data-title="3 实验结果与分析 ">3 实验结果与分析</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#114" data-title="3.1 数据预处理">3.1 数据预处理</a></li>
                                                <li><a href="#117" data-title="3.2 实验设置">3.2 实验设置</a></li>
                                                <li><a href="#123" data-title="3.3 结果分析">3.3 结果分析</a></li>
                                                <li><a href="#134" data-title="3.4 生物学解释">3.4 生物学解释</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#140" data-title="4 结束语 ">4 结束语</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#58" data-title="&lt;b&gt;表1 数据分布示例&lt;/b&gt;"><b>表1 数据分布示例</b></a></li>
                                                <li><a href="#61" data-title="&lt;b&gt;图1 层级规则树示例&lt;/b&gt;"><b>图1 层级规则树示例</b></a></li>
                                                <li><a href="#95" data-title="&lt;b&gt;图2 数据转换示例&lt;/b&gt;"><b>图2 数据转换示例</b></a></li>
                                                <li><a href="#104" data-title="&lt;b&gt;表2 相对偏移量示例&lt;/b&gt;"><b>表2 相对偏移量示例</b></a></li>
                                                <li><a href="#116" data-title="&lt;b&gt;表3 基因表达数据集信息&lt;/b&gt;"><b>表3 基因表达数据集信息</b></a></li>
                                                <li><a href="#127" data-title="&lt;b&gt;图3 实验1中各算法在性别数据集上的准确度对比&lt;/b&gt;"><b>图3 实验1中各算法在性别数据集上的准确度对比</b></a></li>
                                                <li><a href="#128" data-title="&lt;b&gt;图4 实验1中各算法在年龄数据集上的准确度对比&lt;/b&gt;"><b>图4 实验1中各算法在年龄数据集上的准确度对比</b></a></li>
                                                <li><a href="#129" data-title="&lt;b&gt;图5 实验1中各算法在性别数据集上的运行时间对比&lt;/b&gt;"><b>图5 实验1中各算法在性别数据集上的运行时间对比</b></a></li>
                                                <li><a href="#130" data-title="&lt;b&gt;图6 实验1中各算法在年龄数据集上的运行时间对比&lt;/b&gt;"><b>图6 实验1中各算法在年龄数据集上的运行时间对比</b></a></li>
                                                <li><a href="#131" data-title="&lt;b&gt;表4 实验1中各算法在最大样本量时的分类精度对比&lt;/b&gt;"><b>表4 实验1中各算法在最大样本量时的分类精度对比</b></a></li>
                                                <li><a href="#133" data-title="&lt;b&gt;表5 实验2中各算法的分类精度对比&lt;/b&gt;"><b>表5 实验2中各算法的分类精度对比</b></a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="160">


                                    <a id="bibliography_1" title=" CLAES P, ROOSENBOOM J, WHITE J D, et al.Genome-wide mapping of global-to-local genetic effects on human facial shape[J].Nature Genetics, 2018, 50 (3) :414-420." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Genome-wide mapping of global-to-local genetic effects on human facial shape">
                                        <b>[1]</b>
                                         CLAES P, ROOSENBOOM J, WHITE J D, et al.Genome-wide mapping of global-to-local genetic effects on human facial shape[J].Nature Genetics, 2018, 50 (3) :414-420.
                                    </a>
                                </li>
                                <li id="162">


                                    <a id="bibliography_2" title=" CASPI A, SUGDEN K, MOFFITT T E, et al.Influence of life stress on depression:moderation by a polymorphism in the 5-HTT gene[J].Science, 2003, 301 (5631) :386-389." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Influence of Life Stress on Depression: Moderation by a Polymorphism in the 5-HTT Gene">
                                        <b>[2]</b>
                                         CASPI A, SUGDEN K, MOFFITT T E, et al.Influence of life stress on depression:moderation by a polymorphism in the 5-HTT gene[J].Science, 2003, 301 (5631) :386-389.
                                    </a>
                                </li>
                                <li id="164">


                                    <a id="bibliography_3" title=" GUYON I, WESTON J, BARNHILL S, et al.Gene selection for cancer classification using support vector machines[J].Machine Learning, 2002, 46 (1-3) :389-422." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00001340375&amp;v=MDMwNjNtVTczSkpWaz1OajdCYXJPNEh0SE5ySXRGWit3S1kzazV6QmRoNGo5OVNYcVJyeG94Y01IN1I3cWVidWR0RkMv&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[3]</b>
                                         GUYON I, WESTON J, BARNHILL S, et al.Gene selection for cancer classification using support vector machines[J].Machine Learning, 2002, 46 (1-3) :389-422.
                                    </a>
                                </li>
                                <li id="166">


                                    <a id="bibliography_4" title=" LECUN Y, BENGIO Y, HINTON G.Deep learning[J].Nature, 2015, 521 (7553) :436-440." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Deep learning">
                                        <b>[4]</b>
                                         LECUN Y, BENGIO Y, HINTON G.Deep learning[J].Nature, 2015, 521 (7553) :436-440.
                                    </a>
                                </li>
                                <li id="168">


                                    <a id="bibliography_5" title=" LIU Bing, HSU W, MA Yiming.Integrating classification and association rule mining[C]//Proceedings of International Conference on Knowledge Discovery and Data Mining.New York, USA:AAAI Press, 1998:80-86." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Integrating classification and association rule mining">
                                        <b>[5]</b>
                                         LIU Bing, HSU W, MA Yiming.Integrating classification and association rule mining[C]//Proceedings of International Conference on Knowledge Discovery and Data Mining.New York, USA:AAAI Press, 1998:80-86.
                                    </a>
                                </li>
                                <li id="170">


                                    <a id="bibliography_6" title=" CONG G, TAN K L, TUNG A K, et al.Mining top-k covering rule groups for gene expression data[C]//Proceedings of 2005 ACM SIGMOD International Conference on Management of Data.New York, USA:ACM Press, 2005:670-681." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Mining top-k coveringrule groups for gene expression data">
                                        <b>[6]</b>
                                         CONG G, TAN K L, TUNG A K, et al.Mining top-k covering rule groups for gene expression data[C]//Proceedings of 2005 ACM SIGMOD International Conference on Management of Data.New York, USA:ACM Press, 2005:670-681.
                                    </a>
                                </li>
                                <li id="172">


                                    <a id="bibliography_7" title=" TAN A C, NAIMAN D Q, XU Lei, et al.Simple decision rules for classifying human cancers from gene expression profiles[J].Bioinformatics, 2005, 21 (20) :3896-3904." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Simple decision rules for classifying human cancers from gene expression profiles">
                                        <b>[7]</b>
                                         TAN A C, NAIMAN D Q, XU Lei, et al.Simple decision rules for classifying human cancers from gene expression profiles[J].Bioinformatics, 2005, 21 (20) :3896-3904.
                                    </a>
                                </li>
                                <li id="174">


                                    <a id="bibliography_8" title=" CAI Ruichu, HAO Zhifeng, YANG Xiaowei, et al.An efficient gene selection algorithm based on mutual information[J].Neurocomputing, 2009, 72 (4-6) :991-999." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011501912314&amp;v=MDMzMTBud1plWnVIeWptVUxqTElGd1ZieFU9TmlmT2ZiSzdIdEROcW85RWJlb05EMzA5b0JNVDZUNFBRSC9pclJkR2VycVFUTQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[8]</b>
                                         CAI Ruichu, HAO Zhifeng, YANG Xiaowei, et al.An efficient gene selection algorithm based on mutual information[J].Neurocomputing, 2009, 72 (4-6) :991-999.
                                    </a>
                                </li>
                                <li id="176">


                                    <a id="bibliography_9" title=" CAI Ruichu, TUNG A K, ZHANG Zhenjie, et al.What is unequal among the equals?Ranking equivalent rules from gene expression data[J].IEEE Transactions on Knowledge and Data Engineering, 2011, 23 (11) :1735-1747." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=What is unequal among the equals? Ranking equivalent rules from gene expression data">
                                        <b>[9]</b>
                                         CAI Ruichu, TUNG A K, ZHANG Zhenjie, et al.What is unequal among the equals?Ranking equivalent rules from gene expression data[J].IEEE Transactions on Knowledge and Data Engineering, 2011, 23 (11) :1735-1747.
                                    </a>
                                </li>
                                <li id="178">


                                    <a id="bibliography_10" title=" 蔡瑞初, 王美华, 郝志峰, 等.基于最大间隔的基因表达规则筛选[J].计算机工程与应用, 2011, 47 (26) :11-13." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSGG201126006&amp;v=MjM2ODJDVVJMT2VaZVJwRkNqblVyclBMejdNYWJHNEg5RE9xWTlGWW9RS0RIODR2UjRUNmo1NE8zenFxQnRHRnI=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[10]</b>
                                         蔡瑞初, 王美华, 郝志峰, 等.基于最大间隔的基因表达规则筛选[J].计算机工程与应用, 2011, 47 (26) :11-13.
                                    </a>
                                </li>
                                <li id="180">


                                    <a id="bibliography_11" title=" GEMAN D, D’AVIGNON C, NAIMAN D Q, et al.Classifying gene expression profiles from pairwise mRNA comparisons[J].Statistical Applications in Genetics and Molecular Biology, 2004, 3 (1) :1-19." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Classifying Gene Expression Profiles from Pairwise mRNA Comparisons">
                                        <b>[11]</b>
                                         GEMAN D, D’AVIGNON C, NAIMAN D Q, et al.Classifying gene expression profiles from pairwise mRNA comparisons[J].Statistical Applications in Genetics and Molecular Biology, 2004, 3 (1) :1-19.
                                    </a>
                                </li>
                                <li id="182">


                                    <a id="bibliography_12" title=" 蔡毅, 朱秀芳, 孙章丽, 等.半监督集成学习综述[J].计算机科学, 2017, 44 (增刊) :7-13." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJA2017S1002&amp;v=MDE0MTZxQnRHRnJDVVJMT2VaZVJwRkNqblVyclBMejdCYjdHNEg5YXZybzlGWm9RS0RIODR2UjRUNmo1NE8zenE=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[12]</b>
                                         蔡毅, 朱秀芳, 孙章丽, 等.半监督集成学习综述[J].计算机科学, 2017, 44 (增刊) :7-13.
                                    </a>
                                </li>
                                <li id="184">


                                    <a id="bibliography_13" title=" HAAS P J, HELLERSTEIN J M.Ripple joins for online aggregation[J].ACM SIGMOD Record, 1999, 28 (2) :287-298." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM13091000069003&amp;v=MjE5MjFCTVQ2VDRQUUgvaXJSZEdlcnFRVE1ud1plWnVIeWptVUxqTElGd1ZieFU9TmlmSVk3SzdIdGpOcjQ5RlpPMEdESHc2bw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[13]</b>
                                         HAAS P J, HELLERSTEIN J M.Ripple joins for online aggregation[J].ACM SIGMOD Record, 1999, 28 (2) :287-298.
                                    </a>
                                </li>
                                <li id="186">


                                    <a id="bibliography_14" title=" BAHAR R, HARTMANN C H, RODRIGUEZ K A, et al.Increased cell-to-cell variation in gene expression in ageing mouse heart[J].Nature, 2006, 441 (7096) :1011-1012." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Increased cell-to-cell variation in gene expression in ageing mouse heart">
                                        <b>[14]</b>
                                         BAHAR R, HARTMANN C H, RODRIGUEZ K A, et al.Increased cell-to-cell variation in gene expression in ageing mouse heart[J].Nature, 2006, 441 (7096) :1011-1012.
                                    </a>
                                </li>
                                <li id="188">


                                    <a id="bibliography_15" title=" FEDELES B I, SINGH V, DELANEY J C, et al.The AlkB family of Fe (ii) /α-Ketoglutarate-dependent dioxygenases:repairing nucleic acid alkylation damage and beyond[J].Journal of Biological Chemistry, 2015, 290 (34) :20734-20742." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The AlkB family of Fe (ii)/α-Ketoglutarate-dependent dioxygenases:repairing nucleic acid alkylation damage and beyond">
                                        <b>[15]</b>
                                         FEDELES B I, SINGH V, DELANEY J C, et al.The AlkB family of Fe (ii) /α-Ketoglutarate-dependent dioxygenases:repairing nucleic acid alkylation damage and beyond[J].Journal of Biological Chemistry, 2015, 290 (34) :20734-20742.
                                    </a>
                                </li>
                                <li id="190">


                                    <a id="bibliography_16" title=" OUGLAND R, JONSON I, MOEN M N, et al.Role of ALKBH1 in the core transcriptional network of embryonic stem cells[J].Cellular Physiology and Biochemistry, 2016, 38 (1) :173-184." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Role of ALKBH1 in the core transcriptional network of embryonic stem cells">
                                        <b>[16]</b>
                                         OUGLAND R, JONSON I, MOEN M N, et al.Role of ALKBH1 in the core transcriptional network of embryonic stem cells[J].Cellular Physiology and Biochemistry, 2016, 38 (1) :173-184.
                                    </a>
                                </li>
                                <li id="192">


                                    <a id="bibliography_17" title=" FONSECA R R D, KOSIOL C, TOM&#193; V, et al.Positive selection on apoptosis related genes[J].Febs Letters, 2010, 584 (3) :469-476." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011601597117&amp;v=MDMyMDlZOUVZZUlJRFgwK29CTVQ2VDRQUUgvaXJSZEdlcnFRVE1ud1plWnVIeWptVUxqTElGd1ZieFU9TmlmT2ZiSzdIdEROcQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[17]</b>
                                         FONSECA R R D, KOSIOL C, TOM&#193; V, et al.Positive selection on apoptosis related genes[J].Febs Letters, 2010, 584 (3) :469-476.
                                    </a>
                                </li>
                                <li id="194">


                                    <a id="bibliography_18" title=" TAN Qihua, ZHAO Jinghua, LI Shuxia, et al.Differential and correlation analyses of microarray gene expression data in the CEPH Utah families[J].Genomics, 2008, 92 (2) :94-100." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13012100138100&amp;v=MDYxOTBmT2ZiSzdIdERPcm85RlplZ0hEWHc1b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadUh5am1VTGpMSUZ3VmJ4VT1OaQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[18]</b>
                                         TAN Qihua, ZHAO Jinghua, LI Shuxia, et al.Differential and correlation analyses of microarray gene expression data in the CEPH Utah families[J].Genomics, 2008, 92 (2) :94-100.
                                    </a>
                                </li>
                                <li id="196">


                                    <a id="bibliography_19" title=" GLERUM D M, SHTANKO A, TZAGOLOFF A.Charac-terization of COX17, a yeast gene involved in copper metabolism and assembly of cytochrome oxidase[J].Journal of Biological Chemistry, 1996, 271 (24) :14504-14509." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Characterization of COX17, a yeast gene involved in copper metabolism and assembly of cytochrome oxidase">
                                        <b>[19]</b>
                                         GLERUM D M, SHTANKO A, TZAGOLOFF A.Charac-terization of COX17, a yeast gene involved in copper metabolism and assembly of cytochrome oxidase[J].Journal of Biological Chemistry, 1996, 271 (24) :14504-14509.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JSJC" target="_blank">计算机工程</a>
                2019,45(07),26-31 DOI:10.19678/j.issn.1000-3428.0051856            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>基于层级规则树的跨平台基因表达数据分类</b></span>
 <span class="shoufa"></span>                                     </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E8%94%A1%E7%91%9E%E5%88%9D&amp;code=26284587&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蔡瑞初</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E4%BE%AF%E6%B0%B8%E6%9D%B0&amp;code=39549650&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">侯永杰</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E9%83%9D%E5%BF%97%E5%B3%B0&amp;code=06748053&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">郝志峰</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%B9%BF%E4%B8%9C%E5%B7%A5%E4%B8%9A%E5%A4%A7%E5%AD%A6%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%AD%A6%E9%99%A2&amp;code=0139774&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">广东工业大学计算机学院</a>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E4%BD%9B%E5%B1%B1%E7%A7%91%E5%AD%A6%E6%8A%80%E6%9C%AF%E5%AD%A6%E9%99%A2%E6%95%B0%E5%AD%A6%E4%B8%8E%E5%A4%A7%E6%95%B0%E6%8D%AE%E5%AD%A6%E9%99%A2&amp;code=0041535&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">佛山科学技术学院数学与大数据学院</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>基因检测技术运用至今已积累大量来自不同平台的数据, 针对传统数据分类模式难以在不同平台间进行有效迁移的问题, 提出一种基于层级规则树的基因表达数据分类算法k-HRT。设计数据转换与规则预筛选策略, 实现算法的快速挖掘, 以解决由跨平台特性所带来的大规模数据问题。在真实基因表达数据集上的实验结果表明, 相对k-TSP算法、SVM-RFE算法, k-HRT算法能够有效提高分类精度。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%95%B0%E6%8D%AE%E5%88%86%E7%B1%BB&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">数据分类;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%B7%A8%E5%B9%B3%E5%8F%B0&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">跨平台;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%A7%84%E5%88%99%E5%AD%A6%E4%B9%A0&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">规则学习;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%89%B9%E5%BE%81%E9%80%89%E6%8B%A9&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">特征选择;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%9F%BA%E5%9B%A0%E8%A1%A8%E8%BE%BE%E6%95%B0%E6%8D%AE&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">基因表达数据;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    蔡瑞初 (1983—) , 男, 教授、博士, 主研方向为大数据分析、因果关系发现、机器学习, E-mail: cairuichu@ gmail. com;
                                </span>
                                <span>
                                    侯永杰, 硕士研究生;;
                                </span>
                                <span>
                                    郝志峰, 教授。;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-06-19</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金 (61472089);</span>
                                <span>NSFC-广东联合基金 (U1501254);</span>
                                <span>广东省自然科学基金 (2014A030306004, 2014A030308008);</span>
                                <span>广东省科技计划项目 (2015B010108006, 2015B010131015);</span>
                                <span>广东特支计划 (2015TQ01X140);</span>
                                <span>广州市科技计划项目 (201604016075);</span>
                                <span>广州市珠江科技新星专项 (201610010101);</span>
                    </p>
            </div>
                    <h1><b>Cross-platform Gene Expression Data Classification Based on Hierarchical Rule Tree</b></h1>
                    <h2>
                    <span>CAI Ruichu</span>
                    <span>HOU Yongjie</span>
                    <span>HAO Zhifeng</span>
            </h2>
                    <h2>
                    <span>School of Computers, Guangdong University of Technology</span>
                    <span>School of Mathematics and Big Data, Foshan University</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>The application of genetic testing technology has accumulated a large amount of data from different platforms.To address the problem that it is difficult to migrate traditional data classification modes across different platforms, this paper proposes a gene expression data classification algorithm k-HRT based on Hierarchy Rule Tree (HRT) .The strategy of data conversion and rule pre-screening is designed to realize the fast mining of the algorithm to solve the large-scale data problems caused by cross-platform characteristics.Experimental results on real gene expression datasets show that, compared with k-TSP algorithm and SVM-RFE algorithm, k-HRT algorithm can effectively improve classification accuracy.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=data%20classification&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">data classification;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=cross-platform&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">cross-platform;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=rule%20learning&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">rule learning;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=feature%20selection&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">feature selection;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=gene%20expression%20data&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">gene expression data;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                                            </p>
                                    <p><b>Received：</b> 2018-06-19</p>
                                    <p>
                                            </p>
            </div>


        <!--brief start-->
                        <h3 id="41" name="41" class="anchor-tag">0 概述</h3>
                <div class="p1">
                    <p id="42">人的精神状况、行为偏好由先天基因组与外界刺激共同决定<citation id="200" type="reference"><link href="160" rel="bibliography" /><link href="162" rel="bibliography" /><sup>[<a class="sup">1</a>,<a class="sup">2</a>]</sup></citation>。基因检测技术的发展使得从全基因组表达水平定量检测基因转录产物mRNA得以实现。“基因→mRNA→蛋白质”为基因表达过程, 可以通过分析mRNA数据来研究发生变化的基因表达。运用机器学习方法对基因表达数据进行研究, 对医学临床诊断具有重要的参考意义。但不同于普通的分类问题, 由于伦理和法律方面的约束与潜在风险, 在医学诊断中对基因表达数据进行研究的目标是得到精确且可解释的分类机制。机器学习领域主流的SVM<citation id="198" type="reference"><link href="164" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>、神经网络<citation id="199" type="reference"><link href="166" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>等方法对数据具有很强的拟合能力, 但其结果的可解释性较差, 应用于医学诊断领域存在一定局限。</p>
                </div>
                <div class="p1">
                    <p id="43">针对上述问题, 有学者将关联规则引入到分类问题中。文献<citation id="201" type="reference">[<a class="sup">5</a>]</citation>提出CBA算法, 其挖掘基因数据间的关联规则并用于分类。基于关联的分类规则具有直观的可解释性, 但其在高维基因数据上会产生大量的冗余规则, 且其规则挖掘与分类器构建为两阶段算法, 导致即使有巨量规则, 仍会有大量样本未被分类器所覆盖。文献<citation id="202" type="reference">[<a class="sup">6</a>]</citation>从样本覆盖的角度出发, 提出一种Top-k算法, 其以样本枚举的方式挖掘可以交叉覆盖到全部样本的多个规则组, 以进行分类器构建。以上算法在数据预处理阶段需将连续的基因表达水平值离散化, 即根据阈值的大小将基因表达水平值划分为激活态和抑制态。跨平台基因表达数据的特点是各平台的样本具有相同的特征和标签, 但由于平台的特性, 数据值之间存在尺度差异, 因此不同平台间的阈值也不同, 导致已有分类模式只适用于单一平台。</p>
                </div>
                <div class="p1">
                    <p id="44">受k-TSP分类模式中“相对表达反转”概念<citation id="203" type="reference"><link href="172" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>的启发, 本文设计一种层级规则树 (Hierarchy Rule Tree, HRT) 分类模式。传统有监督学习算法推广到基因表达数据领域时, 一般通过基因选择和关联规则发现2种方式<citation id="204" type="reference"><link href="174" rel="bibliography" /><link href="176" rel="bibliography" /><link href="178" rel="bibliography" /><sup>[<a class="sup">8</a>,<a class="sup">9</a>,<a class="sup">10</a>]</sup></citation>。HRT分类模式将上述2种方式进行结合, 基于基因间相对表达反转的特性, 设计相应的数据转换和规则预筛选策略, 进行迭代式分类规则挖掘, 以提高跨平台基因表达数据挖掘的效率。</p>
                </div>
                <h3 id="45" name="45" class="anchor-tag">1 跨平台分类模式</h3>
                <h4 class="anchor-tag" id="46" name="46">1.1 最高得分对</h4>
                <div class="p1">
                    <p id="47">文献<citation id="205" type="reference">[<a class="sup">11</a>]</citation>提出一种最高得分对 (Top Scoring Pair, TSP) 算法, 该算法是纯数据驱动的机器学习方法, 无需使用者设定参数, 避免了精确调整参数所带来的过拟合问题。TSP分类器的出发点是不同基因表达水平之间的相对差异性, 即寻找事件{<i>V</i><sub><i>a</i></sub>&lt;<i>V</i><sub><i>b</i></sub>} (<i>V</i><sub><i>a</i></sub> (<i>V</i><sub><i>b</i></sub>) 代表基因<i>a</i> (<i>b</i>) 的表达值) 在不同类样本中发生概率显著不同的那些标志性基因对 (<i>a</i>, <i>b</i>) 。TSP算法关注事件{<i>V</i><sub><i>a</i></sub>&lt;<i>V</i><sub><i>b</i></sub>}在每一类样本中发生的概率<i>p</i><sub><i>ab</i></sub> (<i>C</i><sub><i>m</i></sub>) , 其公式定义如下:</p>
                </div>
                <div class="p1">
                    <p id="48"><i>p</i><sub><i>ab</i></sub> (<i>C</i><sub><i>m</i></sub>) =<i>Prob</i>{<i>V</i><sub><i>a</i></sub>&lt;<i>V</i><sub><i>b</i></sub>|<i>Y</i>=<i>C</i><sub><i>m</i></sub>}</p>
                </div>
                <div class="p1">
                    <p id="49">其中, <i>C</i><sub><i>m</i></sub>为样本类别, 此处仅考虑二分类问题, <i>m</i>={0, 1}。用<i>Δ</i><sub><i>ab</i></sub>代表每一个基因对 (<i>a</i>, <i>b</i>) 的得分, <mathml id="50"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Δ</mi><msub><mrow></mrow><mrow><mi>a</mi><mi>b</mi></mrow></msub><mo>=</mo><mrow><mo>|</mo><mrow><mi>p</mi><msub><mrow></mrow><mrow><mi>a</mi><mi>b</mi></mrow></msub><mo stretchy="false"> (</mo><mi>C</mi><msub><mrow></mrow><mn>0</mn></msub><mo stretchy="false">) </mo><mo>-</mo><mi>p</mi><msub><mrow></mrow><mrow><mi>a</mi><mi>b</mi></mrow></msub><mo stretchy="false"> (</mo><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">) </mo></mrow><mo>|</mo></mrow><mo>, </mo></mrow></math></mathml>以得分最高的基因对作为分类器的判定依据。基因数据具有高维度特性, 上万个基因组合出的基因对的数量是亿级, 因此, 有可能出现得分一致的基因对。如果出现这种情况, TSP算法会计算第2个评分指标, 即平均排序差, 以选取唯一的最高得分对。</p>
                </div>
                <div class="p1">
                    <p id="51">TSP算法理论上可以作为跨平台分类模式的基础, 但以单个基因对作为全部样本的分类模式过于简单。有研究者基于机器学习集成思想, 以TSP作为基分类器, 提出改进的k-TSP算法, 但仍难以拟合复杂的数据分布。</p>
                </div>
                <h4 class="anchor-tag" id="52" name="52">1.2 层级规则树</h4>
                <div class="p1">
                    <p id="53">从规则构成的角度出发, 以基因对 (<i>a</i>, <i>b</i>) 之间表达值反转为基础的分类模式, 是由2个独立的分类规则{&lt;<i>a</i>, <i>b</i>&gt;→<i>C</i><sub><i>m</i></sub>}、{¬&lt;<i>a</i>, <i>b</i>&gt;→<i>C</i><sub><i>m</i></sub>} (<i>m</i>={0, 1}) 所构成, 其中, &lt;<i>a</i>, <i>b</i>&gt;代表{<i>V</i><sub><i>a</i></sub>&lt;<i>V</i><sub><i>b</i></sub>}, ¬&lt;<i>a</i>, <i>b</i>&gt;代表{<i>V</i><sub><i>a</i></sub>≥<i>V</i><sub><i>b</i></sub>}。</p>
                </div>
                <div class="p1">
                    <p id="54">本文提出的层级规则树以支持度 (<i>supp</i>) 和置信度 (<i>conf</i>) 作为规则兴趣度的度量。支持度和置信度是传统关联规则挖掘中的度量标准, 现已扩展到分类规则挖掘中。给定一个数据集<i>R</i>, <i>d</i>为数据集样本, <i>X</i>和<i>Y</i>为样本属性, 对于分类规则<i>X</i>→<i>Y</i>, <i>supp</i>和<i>conf</i>公式定义分别如下:</p>
                </div>
                <div class="p1">
                    <p id="55" class="code-formula">
                        <mathml id="55"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>s</mi><mi>u</mi><mi>p</mi><mi>p</mi><mo stretchy="false"> (</mo><mi>X</mi><mo>→</mo><mi>Y</mi><mo stretchy="false">) </mo><mo>=</mo><mfrac><mrow><mrow><mo>|</mo><mrow><mo stretchy="false">{</mo><mi>d</mi><mo>∈</mo><mi>R</mi><mo>;</mo><mi>X</mi><mo>⊆</mo><mi>d</mi><mo>, </mo><mi>Y</mi><mo>⊆</mo><mi>d</mi><mo stretchy="false">}</mo></mrow><mo>|</mo></mrow></mrow><mrow><mrow><mo>|</mo><mi>R</mi><mo>|</mo></mrow></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mo stretchy="false">) </mo></mtd></mtr><mtr><mtd><mi>c</mi><mi>o</mi><mi>n</mi><mi>f</mi><mo stretchy="false"> (</mo><mi>X</mi><mo>→</mo><mi>Y</mi><mo stretchy="false">) </mo><mo>=</mo><mfrac><mrow><mrow><mo>|</mo><mrow><mo stretchy="false">{</mo><mi>d</mi><mo>∈</mo><mi>R</mi><mo>;</mo><mi>X</mi><mo>⊆</mo><mi>d</mi><mo>, </mo><mi>Y</mi><mo>⊆</mo><mi>d</mi><mo stretchy="false">}</mo></mrow><mo>|</mo></mrow></mrow><mrow><mrow><mo>|</mo><mrow><mo stretchy="false">{</mo><mi>d</mi><mo>∈</mo><mi>R</mi><mo>;</mo><mi>X</mi><mo>⊆</mo><mi>d</mi><mo stretchy="false">}</mo></mrow><mo>|</mo></mrow></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>2</mn><mo stretchy="false">) </mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="56">以表1中的数据分布为例, 有:</p>
                </div>
                <div class="p1">
                    <p id="57" class="code-formula">
                        <mathml id="57"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>s</mi><mi>u</mi><mi>p</mi><mi>p</mi><mo stretchy="false"> (</mo><mo>&lt;</mo><mi>a</mi><mo>, </mo><mi>b</mi><mo>&gt;</mo><mo>→</mo><mi>C</mi><msub><mrow></mrow><mn>0</mn></msub><mo stretchy="false">) </mo><mo>=</mo><mfrac><mrow><mi>n</mi><msub><mrow></mrow><mn>1</mn></msub></mrow><mrow><mi>n</mi><msub><mrow></mrow><mn>1</mn></msub><mo>+</mo><mi>n</mi><msub><mrow></mrow><mn>2</mn></msub><mo>+</mo><mi>n</mi><msub><mrow></mrow><mn>3</mn></msub><mo>+</mo><mi>n</mi><msub><mrow></mrow><mn>4</mn></msub></mrow></mfrac></mtd></mtr><mtr><mtd><mi>s</mi><mi>u</mi><mi>p</mi><mi>p</mi><mo stretchy="false"> (</mo><mo>&lt;</mo><mi>a</mi><mo>, </mo><mi>b</mi><mo>&gt;</mo><mo>→</mo><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">) </mo><mo>=</mo><mfrac><mrow><mi>n</mi><msub><mrow></mrow><mn>3</mn></msub></mrow><mrow><mi>n</mi><msub><mrow></mrow><mn>1</mn></msub><mo>+</mo><mi>n</mi><msub><mrow></mrow><mn>2</mn></msub><mo>+</mo><mi>n</mi><msub><mrow></mrow><mn>3</mn></msub><mo>+</mo><mi>n</mi><msub><mrow></mrow><mn>4</mn></msub></mrow></mfrac></mtd></mtr><mtr><mtd><mi>c</mi><mi>o</mi><mi>n</mi><mi>f</mi><mo stretchy="false"> (</mo><mo>&lt;</mo><mi>a</mi><mo>, </mo><mi>b</mi><mo>&gt;</mo><mo>→</mo><mi>C</mi><msub><mrow></mrow><mn>0</mn></msub><mo stretchy="false">) </mo><mo>=</mo><mfrac><mrow><mi>n</mi><msub><mrow></mrow><mn>1</mn></msub></mrow><mrow><mi>n</mi><msub><mrow></mrow><mn>1</mn></msub><mo>+</mo><mi>n</mi><msub><mrow></mrow><mn>2</mn></msub></mrow></mfrac></mtd></mtr><mtr><mtd><mi>c</mi><mi>o</mi><mi>n</mi><mi>f</mi><mo stretchy="false"> (</mo><mo>&lt;</mo><mi>a</mi><mo>, </mo><mi>b</mi><mo>&gt;</mo><mo>→</mo><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">) </mo><mo>=</mo><mfrac><mrow><mi>n</mi><msub><mrow></mrow><mn>3</mn></msub></mrow><mrow><mi>n</mi><msub><mrow></mrow><mn>3</mn></msub><mo>+</mo><mi>n</mi><msub><mrow></mrow><mn>4</mn></msub></mrow></mfrac></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="area_img" id="58">
                    <p class="img_tit"><b>表1 数据分布示例</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="58" border="1"><tr><td>特征</td><td>标签</td><td>样本数量</td></tr><tr><td rowspan="2"><br />&lt;<i>a</i>, <i>b</i>&gt;</td><td><br /><i>C</i><sub>0</sub></td><td><i>n</i><sub>1</sub></td></tr><tr><td><br /><i>C</i><sub>1</sub></td><td><i>n</i><sub>2</sub></td></tr><tr><td rowspan="2"><br />¬&lt;<i>a</i>, <i>b</i>&gt;</td><td><br /><i>C</i><sub>0</sub></td><td><i>n</i><sub>3</sub></td></tr><tr><td><br /><i>C</i><sub>1</sub></td><td><i>n</i><sub>4</sub></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="59">可以看出, &lt;<i>a</i>, <i>b</i>&gt;、¬&lt;<i>a</i>, <i>b</i>&gt;这2个分类规则基于完全不同的样本子集而得出, 两者之间并不存在关联关系。有可能其中一个规则具有很强的分类能力, 而另外一个规则没有参考价值。将两者相结合进行衡量, 会限制分类器对数据的拟合能力。基于此, 本文以其中的单个规则为基础, 建立一种层级规则树分类模式。</p>
                </div>
                <div class="p1">
                    <p id="60">层级规则树首先对置信度最高的规则特征&lt;<i>a</i>, <i>b</i>&gt;所覆盖的样本进行有效分类, 对于剩余样本 (特征¬&lt;<i>a</i>, <i>b</i>&gt;所覆盖的样本) , 寻找更精确的新规则 (如&lt;<i>c</i>, <i>d</i>&gt;) 作为分类依据, 迭代挖掘直至样本集中的样本为空或小于设定阈值。后续规则形式可解释为{¬&lt;<i>a</i>, <i>b</i>&gt;∧&lt;<i>c</i>, <i>d</i>&gt;→<i>C</i><sub><i>m</i></sub>}, {¬&lt;<i>a</i>, <i>b</i>&gt;∧¬&lt;<i>c</i>, <i>d</i>&gt;∧&lt;<i>e</i>, <i>f</i>&gt;→<i>C</i><sub><i>m</i></sub>}, …。如图1所示, 在预测样本类别时, 以挖掘出的顺序作为优先级依次使用多个分类规则, 构成层级规则树分类器<i>CL</i><sub>HRT</sub>。</p>
                </div>
                <div class="area_img" id="61">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201907005_061.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 层级规则树示例" src="Detail/GetImg?filename=images/JSJC201907005_061.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图1 层级规则树示例</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201907005_061.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="62">上述过程是HRT分类模式在二分类问题中的应用, 在解决多分类问题时, 采用如下方式:</p>
                </div>
                <div class="p1">
                    <p id="63">给定一个多分类标签<i>C</i>={<i>C</i><sub>0</sub>, <i>C</i><sub>1</sub>, …, <i>C</i><sub><i>m</i>-1</sub>}, 将多分类问题分解为<i>m</i>个二分类问题, 即{<i>C</i><sub>0</sub>, ～<i>C</i><sub>0</sub>}, {<i>C</i><sub>1</sub>, ～<i>C</i><sub>1</sub>}, …, {<i>C</i><sub><i>m</i>-1</sub>, ～<i>C</i><sub><i>m</i>-1</sub>}, 由此得到<i>m</i>个HRT分类器。若<i>m</i>个分类器预测结果无冲突, 则输出此唯一类别;若存在分类器冲突, 则选择缺省类 (样本中的多数类) 作为最终结果。</p>
                </div>
                <h4 class="anchor-tag" id="64" name="64">1.3 k-HRT算法</h4>
                <div class="p1">
                    <p id="65">集成学习是一种通用的机器学习策略, 与具体的算法模型无关, 其利用多个弱分类器进行集成以提升分类器的最终效果<citation id="206" type="reference"><link href="182" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>。常用的集成学习方法有bagging、boosting, 以及由两者衍生出的随机森林、Adaboost、GBDT等。为提升分类器在跨平台基因表达数据集上的泛化能力, 本文借鉴bagging的思想, 以HRT作为基分类器进行集成学习, 得到k-HRT算法。从训练集中随机去除小部分样本, 对多个基分类器<i>CL</i><sub><i>i</i></sub>进行<i>k</i>次重复训练, 构成分类器<i>h</i><sub>k-HRT</sub>, 对<i>k</i>个基分类器的预测结果进行无权投票, 将得分最高的类别作为样本的最终预测类别并输出。设<i>x</i><sub>new</sub>是等待预测的样本, 待预测样本类别为<i>C</i>={<i>C</i><sub>0</sub>, <i>C</i><sub>1</sub>, …, <i>C</i><sub><i>m</i>-1</sub>}, 则分类器<i>h</i><sub>k-HRT</sub>定义如下:</p>
                </div>
                <div class="p1">
                    <p id="66" class="code-formula">
                        <mathml id="66"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>h</mi><msub><mrow></mrow><mrow><mtext>k</mtext><mo>-</mo><mtext>Η</mtext><mtext>R</mtext><mtext>Τ</mtext></mrow></msub><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mrow><mtext>n</mtext><mtext>e</mtext><mtext>w</mtext></mrow></msub><mo stretchy="false">) </mo><mo>=</mo><mrow><munder><mstyle mathsize="140%" displaystyle="true"><mrow><mtext>a</mtext><mtext>r</mtext><mtext>g</mtext><mtext>m</mtext><mtext>a</mtext><mtext>x</mtext></mrow></mstyle><mi>C</mi></munder></mrow><mtext> </mtext><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>k</mi></munderover><mi>Ι</mi></mstyle><mo stretchy="false"> (</mo><mi>C</mi><mi>L</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mrow><mtext>n</mtext><mtext>e</mtext><mtext>w</mtext></mrow></msub><mo stretchy="false">) </mo><mo>, </mo><mi>C</mi><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>3</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="67"><i>I</i> (<i>CL</i><sub><i>i</i></sub> (<i>x</i><sub>new</sub>) , <i>C</i>) 函数定义如下:</p>
                </div>
                <div class="p1">
                    <p id="68" class="code-formula">
                        <mathml id="68"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ι</mi><mo stretchy="false"> (</mo><mi>C</mi><mi>L</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mrow><mtext>n</mtext><mtext>e</mtext><mtext>w</mtext></mrow></msub><mo stretchy="false">) </mo><mo>, </mo><mi>C</mi><mo stretchy="false">) </mo><mo>=</mo><mrow><mo>{</mo><mrow><mtable><mtr><mtd columnalign="left"><mn>1</mn><mo>, </mo><mi>C</mi><mi>L</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mrow><mtext>n</mtext><mtext>e</mtext><mtext>w</mtext></mrow></msub><mo stretchy="false">) </mo><mo>=</mo><mi>C</mi></mtd></mtr><mtr><mtd columnalign="left"><mn>0</mn><mo>, </mo><mtext>其</mtext><mtext>他</mtext></mtd></mtr></mtable></mrow></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>4</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <h3 id="69" name="69" class="anchor-tag">2 快速挖掘算法</h3>
                <div class="p1">
                    <p id="70">在大规模基因表达数据集中, 快速挖掘算法基于相对偏移表 (Relative Shifting Table, RST) 实现规则预筛选, 以避免对规则空间进行暴力搜索。用<i>N</i>×<i>P</i>的矩阵<b><i>R</i></b>表示数据集, 基因表达数据样本数量为<i>N</i>, 每个样本记录的基因 (即特征) 值的数量为<i>P</i>。算法首先对矩阵<b><i>R</i></b>进行数据转换, 然后构建RST进行规则预筛选, 从候选集中选择得分最高的分类规则, 删去该规则所命中的样本, 最后迭代构建层次规则树。完整的层次规则树挖掘算法BuildHRT描述如下。</p>
                </div>
                <div class="p1">
                    <p id="71"><b>算法1</b> BuildHRT算法</p>
                </div>
                <div class="p1">
                    <p id="72"><b>输入</b> 数据集<b><i>R</i></b>, 最小支持度<i>θ</i>, 候选组数量<i>m</i></p>
                </div>
                <div class="p1">
                    <p id="73"><b>输出</b> 分类器<i>CL</i></p>
                </div>
                <div class="p1">
                    <p id="74">1.DataConvert (R) ;</p>
                </div>
                <div class="p1">
                    <p id="75">2.while<mathml id="76"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>|</mo><mtext>R</mtext><mo>|</mo></mrow><mspace width="0.25em" /><mo>≥</mo><mtext>Ν</mtext><mo>×</mo><mtext>θ</mtext><mspace width="0.25em" /><mtext>d</mtext><mtext>o</mtext></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="77">3.RST=RelativeShiftTable (R) ;</p>
                </div>
                <div class="p1">
                    <p id="78">4.Can=GetCandidates (T, m) ;</p>
                </div>
                <div class="p1">
                    <p id="79">5.γ=GetTop (RST, Can, θ) ;</p>
                </div>
                <div class="p1">
                    <p id="80">6.if γ!=NULL then</p>
                </div>
                <div class="p1">
                    <p id="81">7.insert γ into CL;</p>
                </div>
                <div class="p1">
                    <p id="82">8.for each d∈R do</p>
                </div>
                <div class="p1">
                    <p id="83">9.if d satisfies γ then</p>
                </div>
                <div class="p1">
                    <p id="84">10.delete d from R;</p>
                </div>
                <div class="p1">
                    <p id="85">11.else</p>
                </div>
                <div class="p1">
                    <p id="86">12.break</p>
                </div>
                <div class="p1">
                    <p id="87">13.Set majority class of remain R as default class of CL.</p>
                </div>
                <div class="p1">
                    <p id="88">BuildHRT算法的时间复杂度和其在数据集<b><i>R</i></b>上的迭代效率有关。在最佳情况下, 迭代一次算法终止, 即HRT算法为一层, 算法时间复杂度为<i>O</i> (<i>mN</i>) 。在最差情况下, 迭代<mathml id="89"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mn>1</mn><mi>θ</mi></mfrac></mrow></math></mathml>次算法终止, 算法时间复杂度为<mathml id="90"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ο</mi><mrow><mo> (</mo><mrow><mfrac><mrow><mn>1</mn><mo>+</mo><mi>θ</mi></mrow><mrow><mn>2</mn><mi>θ</mi></mrow></mfrac><mi>m</mi><mi>Ν</mi></mrow><mo>) </mo></mrow><mo>, </mo></mrow></math></mathml>此时最小支持度θ决定了时间复杂度, θ的理论取值范围为<mathml id="91"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>[</mo><mrow><mfrac><mn>1</mn><mi>Ν</mi></mfrac><mo>, </mo><mn>1</mn></mrow><mo>]</mo></mrow><mo>, </mo><mi>θ</mi></mrow></math></mathml>值越小, 算法复杂度越高, 取最小值<mathml id="92"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mn>1</mn><mi>Ν</mi></mfrac></mrow></math></mathml>时算法复杂度为O (mN<sup>2</sup>) 。</p>
                </div>
                <h4 class="anchor-tag" id="93" name="93">2.1 数据转换</h4>
                <div class="p1">
                    <p id="94">对于以基因对表达反转为基础的分类模式, 基因表达数据上有意义的是各基因之间表达值的大小关系, 而非单个基因的表达值。因此, 在BuildHRT算法的第1行, <i>DataConvert</i> (<b><i>R</i></b>) 把记录基因表达值的矩阵转换为基因序列矩阵, 以此作为HRT分类模式挖掘算法的基础。具体地, 将矩阵<b><i>R</i></b>中每一个样本的基因表达值和基因名组合成一个二元组, 对每个样本内的<i>P</i>个二元组, 以基因表达值为键进行排序, 删去二元组中的基因表达值, 从而将样本转换为由基因名构成的有序序列, 该过程如图2所示。</p>
                </div>
                <div class="area_img" id="95">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201907005_095.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 数据转换示例" src="Detail/GetImg?filename=images/JSJC201907005_095.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图2 数据转换示例</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201907005_095.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="96" name="96">2.2 相对偏移表</h4>
                <div class="p1">
                    <p id="97">解决分类问题的关键是发现在不同类样本中表现差异最大的基因表达模式, 并对其进行高效率的挖掘。为避免在高维度、大样本的跨平台基因表达数据集中进行暴力搜索, 减小规则搜索空间, 本文在数据转换的基础上设计一种相对偏移表RST, 以进行规则预筛选。具体过程如下:</p>
                </div>
                <div class="p1">
                    <p id="98">1) 计算正、负类样本特征。根据式 (5) 计算样本的<i>P</i>个基因在矩阵<b><i>R</i></b>正类样本<i>C</i><sub>0</sub>中的总得分值<i>Score</i><mathml id="99"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>g</mi><mrow><mi>C</mi><msub><mrow></mrow><mn>0</mn></msub></mrow></msubsup></mrow></math></mathml>, 根据总得分值对P个基因由高到低排序, 将排序后得到的基因序列定义为正类样本特征EP。同理可以获取负类样本特征EN。</p>
                </div>
                <div class="p1">
                    <p id="100" class="code-formula">
                        <mathml id="100"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>S</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi><msubsup><mrow></mrow><mi>g</mi><mrow><mi>C</mi><msub><mrow></mrow><mn>0</mn></msub></mrow></msubsup><mo>=</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>d</mi><mo>∈</mo><mi>C</mi><msub><mrow></mrow><mn>0</mn></msub></mrow></munder><mi>Ι</mi></mstyle><mo stretchy="false"> (</mo><mi>d</mi><mo>, </mo><mi>g</mi><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>5</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="101">2) 根据式 (6) 计算每个基因g在正类样本特征EP上相对于负类样本特征EN中的相对偏移量Dis<sub>g</sub>。</p>
                </div>
                <div class="p1">
                    <p id="102">Dis<sub>g</sub>=I (EP, g) -I (EN, g)      (6) </p>
                </div>
                <div class="p1">
                    <p id="103">3) 对P个基因按照偏移量得分值进行排序, 得到的基因序列即为相对偏移表<i>RST</i>。假设在数据矩阵<b><i>R</i></b>上得到正类样本特征<i>EP</i>=<i>daceb</i>、负类样本特征<i>EN</i>=<i>dbeca</i>, 则计算的相对偏移量<i>Dis</i><sub><i>g</i></sub>如表2所示, 最终求得相对偏移表<i>RST</i>=<i>acdeb</i>。</p>
                </div>
                <div class="area_img" id="104">
                    <p class="img_tit"><b>表2 相对偏移量示例</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="104" border="1"><tr><td><br />基因</td><td><i>Dis</i><sub><i>g</i></sub></td></tr><tr><td><br /><i>a</i></td><td>-3</td></tr><tr><td><br /><i>b</i></td><td>3</td></tr><tr><td><br /><i>c</i></td><td>-1</td></tr><tr><td><br /><i>d</i></td><td>0</td></tr><tr><td><br /><i>e</i></td><td>1</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h4 class="anchor-tag" id="105" name="105">2.3 层级规则树挖掘</h4>
                <div class="p1">
                    <p id="106">利用相对偏移表RST进行规则预筛选, 目标是找到在不同类中相对表达反转的基因对 (<i>a</i>, <i>b</i>) 。令<i>x</i>为负类样本特征<i>EN</i>相对于正类样本特征<i>EP</i>向前偏移最多的基因, <i>y</i>为<i>EN</i>相对于<i>EP</i>向后偏移最多的基因, 则基因对 (<i>x</i>, <i>y</i>) 有极大概率在不同类样本上完成相对表达反转。向前、向后偏移最多的基因分别位于相对偏移表<i>T</i>的首、尾位置。因此, 为了提高算法的稳定性, 本文从相对偏移表首、尾各取出<i>m</i>个基因组合为候选基因对<i>Can</i> (算法1中的第4行) , 并从中选取置信度最高的基因对作为目标分类规则 (算法1中的第5行) 。</p>
                </div>
                <div class="p1">
                    <p id="107">在每一层寻找分类规则<i>γ</i>的具体过程如下:</p>
                </div>
                <div class="p1">
                    <p id="108">1) 从相对偏移表RST的首、尾各随机取出<i>m</i>个基因, 按照位置对应组合为<i>m</i>个候选基因对。</p>
                </div>
                <div class="p1">
                    <p id="109">2) 每一个候选基因对 (<i>a</i>, <i>b</i>) 可以产生4个分类规则:{&lt;<i>a</i>, <i>b</i>&gt;→<i>C</i><sub>0</sub>}, {&lt;<i>a</i>, <i>b</i>&gt;→<i>C</i><sub>1</sub>}, {¬&lt;<i>a</i>, <i>b</i>&gt;→<i>C</i><sub>0</sub>}, {¬&lt;<i>a</i>, <i>b</i>&gt;→<i>C</i><sub>1</sub>}。从中选取置信度得分最高的作为基因对 (<i>a</i>, <i>b</i>) 的唯一分类规则, 共生成<i>m</i>个候选分类规则。</p>
                </div>
                <div class="p1">
                    <p id="110">3) 从<i>m</i>个候选分类规则中选取置信度得分最高且满足最小支持度<i>θ</i>的规则, 作为目标分类规则<i>γ</i>。</p>
                </div>
                <div class="p1">
                    <p id="111">将分类规则<i>γ</i>加入层次规则树分类器中, 从训练数据集上删去被分类规则<i>γ</i>前项所命中的样本, 重复算法1的第3行～第12行, 直至剩余样本数量小于最小支持度<i>θ</i>, 将剩余样本中的多数类作为层次规则树分类器中的缺省类。</p>
                </div>
                <h3 id="112" name="112" class="anchor-tag">3 实验结果与分析</h3>
                <div class="p1">
                    <p id="113">为验证本文分类模式的性能, 从NCBI上获取多个平台的基因数据集作为原始实验数据, 进行仿真对比与分析, 多个数据集所共有的标签是性别与年龄。</p>
                </div>
                <h4 class="anchor-tag" id="114" name="114">3.1 数据预处理</h4>
                <div class="p1">
                    <p id="115">原始基因表达数据样本的维度为20 660, 在这些属性上存在很多缺失值。由于这些样本来自不同的GPL平台, 各样本上缺失值的数量与位置不同, 因此需要找到缺失值最少的样本与维度。本文受数据库搜索方法ripple join<citation id="207" type="reference"><link href="184" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>的启发, 使用类似的方法进行缺失值数据的消除。首先将样本及其维度根据NA值的数量从高到低分别进行排序, 然后记录维度映射表。初始化一个空矩阵, 迭代扩展行和列, 找到最大的无缺失样本矩阵。经过数据清洗后, 最终可以得到无缺失值的2个样本集, 数据分布如表3所示。其中, 性别数据集来自7个平台:GPL10558, GPL6102, GPL6884, GPL4133, GPL6947, GPL6480, GPL570;年龄数据集来自6个平台:GPL10558, GPL6102, GPL6884, GPL6947, GPL6480, GPL570。</p>
                </div>
                <div class="area_img" id="116">
                    <p class="img_tit"><b>表3 基因表达数据集信息</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="116" border="1"><tr><td>数据集</td><td>样本数量 (正∶负) </td><td>平台数量</td><td>特征数量</td></tr><tr><td>性别</td><td>5 729 (Male) ∶5 077 (Female) </td><td>7</td><td>13 876</td></tr><tr><td><br />年龄</td><td>9 261 (Young) ∶3 502 (Old) </td><td>6</td><td>12 763</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h4 class="anchor-tag" id="117" name="117">3.2 实验设置</h4>
                <div class="p1">
                    <p id="118">性别数据集的2个标签为Male、Female, 在数据预处理阶段, 去除两性分别独有的基因特征, 以男性、女性所共有的基因作为样本特征。年龄数据集的正、负类标签为Young、Old, 由于年龄是连续的数值, 年龄数据集对Old标签的定义是年龄大于60岁的人群。</p>
                </div>
                <div class="p1">
                    <p id="119">实验中的对比算法为本文k-HRT算法、k-TSP算法<citation id="208" type="reference"><link href="172" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>、SVM-RFE算法<citation id="209" type="reference"><link href="164" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>。实验分为2组:</p>
                </div>
                <div class="p1">
                    <p id="120"><b>实验1</b> 2个数据集上的无平台信息实验。</p>
                </div>
                <div class="p1">
                    <p id="121"><b>实验2</b> 2个数据集上的平台迁移实验。</p>
                </div>
                <div class="p1">
                    <p id="122">本文自行从NCBI获取并整合实验数据, 无训练集与测试集的分别。对于第1组无平台信息实验, 训练集与测试集按8∶2的比例进行无偏划分;对于第2组平台迁移实验, 将源平台作为训练集、目标平台作为测试集。对于BuildHRT算法中的2个自定义参数, 最小支持度<i>θ</i>设定为20/<i>N</i>, 候选基因对数量<i>m</i>为50。用于集成的层级规则树数目<i>k</i>设定为3, 对比算法k-TSP中的<i>k</i>值也为3。</p>
                </div>
                <h4 class="anchor-tag" id="123" name="123">3.3 结果分析</h4>
                <div class="p1">
                    <p id="124">在无平台信息实验中, 图3、图4分别为性别、年龄2个数据集上各算法的准确度随样本量的变化情况。由图3、图4可以看出, 在类均衡的性别数据集上, k-HRT算法与k-TSP算法均优于无法消除平台差异性的SVM-RFE算法;在类不均衡的年龄数据集上, k-HRT算法的稳定性与准确性相比其他算法均具有较大优势。图5、图6分别为性别、年龄2个数据集上各算法的CPU运行时间随样本量的变化情况。由图5、图6可以看出, 相比其他2种算法, k-TSP算法效率较差, 难以应对大规模数据集。表4所示为2个数据集上最大样本量时各算法的准确度对比。</p>
                </div>
                <div class="area_img" id="127">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201907005_127.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 实验1中各算法在性别数据集上的准确度对比" src="Detail/GetImg?filename=images/JSJC201907005_127.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图3 实验1中各算法在性别数据集上的准确度对比</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201907005_127.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="128">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201907005_128.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 实验1中各算法在年龄数据集上的准确度对比" src="Detail/GetImg?filename=images/JSJC201907005_128.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图4 实验1中各算法在年龄数据集上的准确度对比</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201907005_128.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="129">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201907005_129.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图5 实验1中各算法在性别数据集上的运行时间对比" src="Detail/GetImg?filename=images/JSJC201907005_129.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图5 实验1中各算法在性别数据集上的运行时间对比</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201907005_129.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="130">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJC201907005_130.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图6 实验1中各算法在年龄数据集上的运行时间对比" src="Detail/GetImg?filename=images/JSJC201907005_130.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图6 实验1中各算法在年龄数据集上的运行时间对比</b>  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJC201907005_130.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="131">
                    <p class="img_tit"><b>表4 实验1中各算法在最大样本量时的分类精度对比</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="131" border="1"><tr><td rowspan="2"><br />数据集</td><td colspan="3"><br />分类精度</td></tr><tr><td><br />k-HRT</td><td>k-TSP</td><td>SVM-RFE</td></tr><tr><td><br />性别</td><td>0.950</td><td>0.941</td><td>0.806</td></tr><tr><td><br />年龄</td><td>0.934</td><td>0.753</td><td>0.627</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="132">对于平台迁移实验, 表5所示为性别、年龄2个数据集上各算法的准确度对比, 其中, 每个数据集上准确度最高的值用加粗标出。由表5可以看出, 该实验的结论与无平台信息实验基本一致。</p>
                </div>
                <div class="area_img" id="133">
                    <p class="img_tit"><b>表5 实验2中各算法的分类精度对比</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="133" border="1"><tr><td rowspan="2">数据集</td><td rowspan="2">源平台</td><td rowspan="2">目标平台</td><td colspan="3"><br />分类精度</td></tr><tr><td><br />k-HRT</td><td>k-TSP</td><td>SVM-RFE</td></tr><tr><td rowspan="3"><br />性别</td><td>GPL10558</td><td>GPL6884</td><td>0.843</td><td><b>0.892</b></td><td>0.799</td></tr><tr><td><br />GPL6480</td><td>GPL4133</td><td><b>0.949</b></td><td>0.943</td><td>0.558</td></tr><tr><td><br />GPL6884</td><td>GPL4133</td><td>0.910</td><td><b>0.925</b></td><td>0.575</td></tr><tr><td rowspan="3"><br />年龄</td><td>GPL6947</td><td>GPL6480</td><td><b>0.727</b></td><td>0.568</td><td>0.618</td></tr><tr><td><br />GPL10558</td><td>GPL6480</td><td><b>0.726</b></td><td>0.561</td><td>0.603</td></tr><tr><td><br />GPL 6947</td><td>GPL10558</td><td><b>0.769</b></td><td>0.634</td><td>0.674</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h4 class="anchor-tag" id="134" name="134">3.4 生物学解释</h4>
                <div class="p1">
                    <p id="135">生物的衰老过程和某些基因表达水平的变化有关<citation id="210" type="reference"><link href="186" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>。本文对年龄组实验中k-HRT分类器上的如下分类规则进行生物学分析:</p>
                </div>
                <div class="p1">
                    <p id="136">If IFNA17 &lt;ALKBH1, then Young.</p>
                </div>
                <div class="p1">
                    <p id="137">If BCORL1 &lt;COX17, then Young.</p>
                </div>
                <div class="p1">
                    <p id="138">If BCORL1≥COX17 and MRAS &lt; ZNF75D, then Old.</p>
                </div>
                <div class="p1">
                    <p id="139">ALKBH1与DNA烷基化损伤修复机制有关<citation id="211" type="reference"><link href="188" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation>。文献<citation id="212" type="reference">[<a class="sup">16</a>]</citation>研究表明, ALKBH1在胚胎干细胞转录网络中扮演着核心角色。IFNA17由巨噬细胞产生, 具有抗病毒活性, 文献<citation id="213" type="reference">[<a class="sup">17</a>]</citation>发现其和细胞凋亡相关。文献<citation id="214" type="reference">[<a class="sup">18</a>]</citation>在CEPH Uath家族数据集上的研究结果也表明, IFNA17的表达水平和年龄有关。COX17则与生物的新陈代谢功能相关<citation id="215" type="reference"><link href="196" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>。BCORL1编码的蛋白质是转录辅阻遏物, 目前尚未发现BCORL1在生物衰老方面的相关特性。</p>
                </div>
                <h3 id="140" name="140" class="anchor-tag">4 结束语</h3>
                <div class="p1">
                    <p id="141">传统分类模式在跨平台基因表达数据平台间难以有效迁移。为此, 本文提出一种基于层级规则树的k-HRT算法。针对由跨平台特性所带来的大规模数据问题, 设计与分类模式相切合的数据转换和规则预筛选策略。同步进行规则挖掘与分类器构建, 以解决规则冗余与样本覆盖等问题。在真实基因表达数据集上的实验结果验证了k-HRT算法的可行性与高效性。本文所提出的数据转换与规则预筛选策略能够为相关的数据集挖掘工作提供一种新思路。下一步考虑将k-HRT算法应用到连续数值型跨平台数据集中, 以进行数据分类。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="160">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Genome-wide mapping of global-to-local genetic effects on human facial shape">

                                <b>[1]</b> CLAES P, ROOSENBOOM J, WHITE J D, et al.Genome-wide mapping of global-to-local genetic effects on human facial shape[J].Nature Genetics, 2018, 50 (3) :414-420.
                            </a>
                        </p>
                        <p id="162">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Influence of Life Stress on Depression: Moderation by a Polymorphism in the 5-HTT Gene">

                                <b>[2]</b> CASPI A, SUGDEN K, MOFFITT T E, et al.Influence of life stress on depression:moderation by a polymorphism in the 5-HTT gene[J].Science, 2003, 301 (5631) :386-389.
                            </a>
                        </p>
                        <p id="164">
                            <a id="bibliography_3" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00001340375&amp;v=MTI4ODRZM2s1ekJkaDRqOTlTWHFScnhveGNNSDdSN3FlYnVkdEZDL21VNzNKSlZrPU5qN0Jhck80SHRITnJJdEZaK3dL&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[3]</b> GUYON I, WESTON J, BARNHILL S, et al.Gene selection for cancer classification using support vector machines[J].Machine Learning, 2002, 46 (1-3) :389-422.
                            </a>
                        </p>
                        <p id="166">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Deep learning">

                                <b>[4]</b> LECUN Y, BENGIO Y, HINTON G.Deep learning[J].Nature, 2015, 521 (7553) :436-440.
                            </a>
                        </p>
                        <p id="168">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Integrating classification and association rule mining">

                                <b>[5]</b> LIU Bing, HSU W, MA Yiming.Integrating classification and association rule mining[C]//Proceedings of International Conference on Knowledge Discovery and Data Mining.New York, USA:AAAI Press, 1998:80-86.
                            </a>
                        </p>
                        <p id="170">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Mining top-k coveringrule groups for gene expression data">

                                <b>[6]</b> CONG G, TAN K L, TUNG A K, et al.Mining top-k covering rule groups for gene expression data[C]//Proceedings of 2005 ACM SIGMOD International Conference on Management of Data.New York, USA:ACM Press, 2005:670-681.
                            </a>
                        </p>
                        <p id="172">
                            <a id="bibliography_7" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Simple decision rules for classifying human cancers from gene expression profiles">

                                <b>[7]</b> TAN A C, NAIMAN D Q, XU Lei, et al.Simple decision rules for classifying human cancers from gene expression profiles[J].Bioinformatics, 2005, 21 (20) :3896-3904.
                            </a>
                        </p>
                        <p id="174">
                            <a id="bibliography_8" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011501912314&amp;v=MDA1MjhIeWptVUxqTElGd1ZieFU9TmlmT2ZiSzdIdEROcW85RWJlb05EMzA5b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[8]</b> CAI Ruichu, HAO Zhifeng, YANG Xiaowei, et al.An efficient gene selection algorithm based on mutual information[J].Neurocomputing, 2009, 72 (4-6) :991-999.
                            </a>
                        </p>
                        <p id="176">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=What is unequal among the equals? Ranking equivalent rules from gene expression data">

                                <b>[9]</b> CAI Ruichu, TUNG A K, ZHANG Zhenjie, et al.What is unequal among the equals?Ranking equivalent rules from gene expression data[J].IEEE Transactions on Knowledge and Data Engineering, 2011, 23 (11) :1735-1747.
                            </a>
                        </p>
                        <p id="178">
                            <a id="bibliography_10" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSGG201126006&amp;v=MzA3Mjc0SDlET3FZOUZZb1FLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUkxPZVplUnBGQ2puVXJyUEx6N01hYkc=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[10]</b> 蔡瑞初, 王美华, 郝志峰, 等.基于最大间隔的基因表达规则筛选[J].计算机工程与应用, 2011, 47 (26) :11-13.
                            </a>
                        </p>
                        <p id="180">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Classifying Gene Expression Profiles from Pairwise mRNA Comparisons">

                                <b>[11]</b> GEMAN D, D’AVIGNON C, NAIMAN D Q, et al.Classifying gene expression profiles from pairwise mRNA comparisons[J].Statistical Applications in Genetics and Molecular Biology, 2004, 3 (1) :1-19.
                            </a>
                        </p>
                        <p id="182">
                            <a id="bibliography_12" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJA2017S1002&amp;v=MjY1MTBScEZDam5VcnJQTHo3QmI3RzRIOWF2cm85RlpvUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSTE9lWmU=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[12]</b> 蔡毅, 朱秀芳, 孙章丽, 等.半监督集成学习综述[J].计算机科学, 2017, 44 (增刊) :7-13.
                            </a>
                        </p>
                        <p id="184">
                            <a id="bibliography_13" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM13091000069003&amp;v=MjUwNTdIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMakxJRndWYnhVPU5pZklZN0s3SHRqTnI0OUZaTzBHREh3Nm9CTVQ2VDRQUQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[13]</b> HAAS P J, HELLERSTEIN J M.Ripple joins for online aggregation[J].ACM SIGMOD Record, 1999, 28 (2) :287-298.
                            </a>
                        </p>
                        <p id="186">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Increased cell-to-cell variation in gene expression in ageing mouse heart">

                                <b>[14]</b> BAHAR R, HARTMANN C H, RODRIGUEZ K A, et al.Increased cell-to-cell variation in gene expression in ageing mouse heart[J].Nature, 2006, 441 (7096) :1011-1012.
                            </a>
                        </p>
                        <p id="188">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The AlkB family of Fe (ii)/α-Ketoglutarate-dependent dioxygenases:repairing nucleic acid alkylation damage and beyond">

                                <b>[15]</b> FEDELES B I, SINGH V, DELANEY J C, et al.The AlkB family of Fe (ii) /α-Ketoglutarate-dependent dioxygenases:repairing nucleic acid alkylation damage and beyond[J].Journal of Biological Chemistry, 2015, 290 (34) :20734-20742.
                            </a>
                        </p>
                        <p id="190">
                            <a id="bibliography_16" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Role of ALKBH1 in the core transcriptional network of embryonic stem cells">

                                <b>[16]</b> OUGLAND R, JONSON I, MOEN M N, et al.Role of ALKBH1 in the core transcriptional network of embryonic stem cells[J].Cellular Physiology and Biochemistry, 2016, 38 (1) :173-184.
                            </a>
                        </p>
                        <p id="192">
                            <a id="bibliography_17" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011601597117&amp;v=MDgzNTVMSUZ3VmJ4VT1OaWZPZmJLN0h0RE5xWTlFWWVJSURYMCtvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp1SHlqbVVMag==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[17]</b> FONSECA R R D, KOSIOL C, TOMÁ V, et al.Positive selection on apoptosis related genes[J].Febs Letters, 2010, 584 (3) :469-476.
                            </a>
                        </p>
                        <p id="194">
                            <a id="bibliography_18" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13012100138100&amp;v=MDA0NzhmT2ZiSzdIdERPcm85RlplZ0hEWHc1b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadUh5am1VTGpMSUZ3VmJ4VT1OaQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[18]</b> TAN Qihua, ZHAO Jinghua, LI Shuxia, et al.Differential and correlation analyses of microarray gene expression data in the CEPH Utah families[J].Genomics, 2008, 92 (2) :94-100.
                            </a>
                        </p>
                        <p id="196">
                            <a id="bibliography_19" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Characterization of COX17, a yeast gene involved in copper metabolism and assembly of cytochrome oxidase">

                                <b>[19]</b> GLERUM D M, SHTANKO A, TZAGOLOFF A.Charac-terization of COX17, a yeast gene involved in copper metabolism and assembly of cytochrome oxidase[J].Journal of Biological Chemistry, 1996, 271 (24) :14504-14509.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JSJC201907005" />
        <input id="dpi" type="hidden" value="600" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201907005&amp;v=MTEwOTdyclBMejdCYmJHNEg5ak1xSTlGWVlRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVJMT2VaZVJwRkNqblU=&amp;uid=WEEvREcwSlJHSldRa1FhdXNXaEhoRFQwam9BWm83cU5adCswdFNtUXRBUT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
