<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=NOOK-mvTMnxl11s6oPYV0boPXN3iP4N7Rc-A56nk4KI1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2fKXReader%2fDetail%3fTIMESTAMP%3d637141081962943750%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dZGWT201904002%26RESULT%3d1%26SIGN%3dQ1S6dqbR11hPUub%252ftseakU4WdeU%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201904002&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201904002&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201904002&amp;v=MDc2MzlLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1Wm5GeURtVWIvS1B5cmNlckc0SDlqTXE0OUZab1E=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#25" data-title="原理与方法 ">原理与方法</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#26" data-title="1.高斯图模型">1.高斯图模型</a></li>
                                                <li><a href="#38" data-title="2.差异网络分析">2.差异网络分析</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#48" data-title="模拟实验 ">模拟实验</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#56" data-title="实例分析 ">实例分析</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#64" data-title="讨  论 ">讨  论</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#51" data-title="&lt;b&gt;图1&lt;/b&gt; 模拟实验1设置的真实差异网络关系图"><b>图1</b> 模拟实验1设置的真实差异网络关系图</a></li>
                                                <li><a href="#52" data-title="&lt;b&gt;表1&lt;/b&gt; 真实差异网络中差异边"><b>表1</b> 真实差异网络中差异边</a></li>
                                                <li><a href="#54" data-title="&lt;b&gt;图2&lt;/b&gt; 模拟实验各评价指标随样本量不同的变化情况"><b>图2</b> 模拟实验各评价指标随样本量不同的变化情况</a></li>
                                                <li><a href="#55" data-title="&lt;b&gt;图3&lt;/b&gt; 三种方法在不同样本量下准确度和假发现率箱式图"><b>图3</b> 三种方法在不同样本量下准确度和假发现率箱式图</a></li>
                                                <li><a href="#60" data-title="&lt;b&gt;图4&lt;/b&gt; 卵巢癌患者复发与非复发状态下的差异基因调控网络"><b>图4</b> 卵巢癌患者复发与非复发状态下的差异基因调控网络</a></li>
                                                <li><a href="#62" data-title="&lt;b&gt;表2&lt;/b&gt; 卵巢癌患者复发状态差异网络中的差异调控关系"><b>表2</b> 卵巢癌患者复发状态差异网络中的差异调控关系</a></li>
                                                <li><a href="#63" data-title="&lt;b&gt;表3&lt;/b&gt; 卵巢癌患者复发状态差异网络中的差异基因及其对应的差异调控关系频数"><b>表3</b> 卵巢癌患者复发状态差异网络中的差异基因及其对应的差异调控关系频数</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="4">


                                    <a id="bibliography_1" title=" 游顶云,李康,贝叶斯网络方法在基因调控研究中的应用.中国卫生统计,2009,26(1):83-86." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT200901032&amp;v=MzE1NTBabkZ5RG1VYi9LUHlyY2VyRzRIdGpNcm85R1pvUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnU=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[1]</b>
                                         游顶云,李康,贝叶斯网络方法在基因调控研究中的应用.中国卫生统计,2009,26(1):83-86.
                                    </a>
                                </li>
                                <li id="6">


                                    <a id="bibliography_2" title=" Opgen-Rhein R,Strimmer K.From correlation to causation networks:a simple approximate learning algorithm and its application to high-dimensional plant gene expression data.BMC SystBiol,2007,1:37." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=From correlation to causation networks: a simple approximate learning algorithm and its application to high-dimensional plant gene expression data">
                                        <b>[2]</b>
                                         Opgen-Rhein R,Strimmer K.From correlation to causation networks:a simple approximate learning algorithm and its application to high-dimensional plant gene expression data.BMC SystBiol,2007,1:37.
                                    </a>
                                </li>
                                <li id="8">


                                    <a id="bibliography_3" title=" Caron F,Doucet A.Sparse Bayesian Nonparametric Regression.In ICML&#39; 08,2008:88-95." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Sparse Bayesian Nonparametric Regression">
                                        <b>[3]</b>
                                         Caron F,Doucet A.Sparse Bayesian Nonparametric Regression.In ICML&#39; 08,2008:88-95.
                                    </a>
                                </li>
                                <li id="10">


                                    <a id="bibliography_4" title=" Green PJ.Reversible jump Markov chain Monte Carlo computation and Bayesian model determination.Biometrika,1995,82(4):711-732." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Reversible jump Markov chain Monte Carlo computation and Bayesian model determination">
                                        <b>[4]</b>
                                         Green PJ.Reversible jump Markov chain Monte Carlo computation and Bayesian model determination.Biometrika,1995,82(4):711-732.
                                    </a>
                                </li>
                                <li id="12">


                                    <a id="bibliography_5" title=" Wang T,et al.FastGGM:An Efficient Algorithm for the Inference of Gaussian Graphical Model in Biological Networks.PLoS Comput Biol,2016,12(2):e1004755." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJLA&amp;filename=SJLA85764D0CA1ADB3A89CA35202643013F4&amp;v=MjQ2ODI0YW1VNStUWDNpcmhReGVyS1ZSc3liQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQnJMVTA1dEZoekx1OXdLbz1OaWZIYjd1OUdkZkkyNDgyRmVwK2VBNDZ2aA==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[5]</b>
                                         Wang T,et al.FastGGM:An Efficient Algorithm for the Inference of Gaussian Graphical Model in Biological Networks.PLoS Comput Biol,2016,12(2):e1004755.
                                    </a>
                                </li>
                                <li id="14">


                                    <a id="bibliography_6" title=" Deng K,Zhang F,Song W,et al.Identification of pathway-based recurrence-associated signatures in optimally debulked patients with serous ovarian cancer.J Cell Biochem,2018:1-10." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD1DD1CDA709026ADDD08DE5E3D5D248BA&amp;v=MjM5MjJmY2FyTE1hdEMvMi81Q1pPSVBEbnBJdTJKbjZqY0pQWHFYcjJZd0RiQ1FUY2p1Q09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQnJMVTA1dEZoekx1OXdLbz1OaQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[6]</b>
                                         Deng K,Zhang F,Song W,et al.Identification of pathway-based recurrence-associated signatures in optimally debulked patients with serous ovarian cancer.J Cell Biochem,2018:1-10.
                                    </a>
                                </li>
                                <li id="16">


                                    <a id="bibliography_7" title=" Paik JH,et al.FoxOs are lineage-restricted redundant tumor suppressors and regulate endothelial cell homeostasis.Cell,2007,128(2):309-23." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011600217979&amp;v=MTI5NjZud1plWnRGaW5sVXJiSUtGMFdhaEE9TmlmT2ZiSzdIdEROcVk5Rlp1b0lCWHN3b0JNVDZUNFBRSC9pclJkR2VycVFUTQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[7]</b>
                                         Paik JH,et al.FoxOs are lineage-restricted redundant tumor suppressors and regulate endothelial cell homeostasis.Cell,2007,128(2):309-23.
                                    </a>
                                </li>
                                <li id="18">


                                    <a id="bibliography_8" title=" Park SH,et al.Auranofin displays anticancer activity against ovarian cancer cells through FOXO3 activation independent of p53.Int J Oncol,2014,45(4):1691-8." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJSD&amp;filename=SJSD14101400000249&amp;v=MDYyMjdHZXJxUVRNbndaZVp0RmlubFVyYklLRjBXYWhBPU5pZllhcks4SDlITnE0OUZaT3NQRG5nd29CTVQ2VDRQUUgvaXJSZA==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[8]</b>
                                         Park SH,et al.Auranofin displays anticancer activity against ovarian cancer cells through FOXO3 activation independent of p53.Int J Oncol,2014,45(4):1691-8.
                                    </a>
                                </li>
                                <li id="20">


                                    <a id="bibliography_9" title=" Zhang Y,et al.Expression of NLK and its potential effect in ovarian cancer chemotherapy.Int J Gynecol Cancer,2011,21(8):1380-1387." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWK&amp;filename=SJWK12111300062770&amp;v=MDYzNjdRVE1ud1plWnRGaW5sVXJiSUtGMFdhaEE9TmlmY1piSzZIOUROckk5RlpPME5DM3M1b0JNVDZUNFBRSC9pclJkR2VycQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[9]</b>
                                         Zhang Y,et al.Expression of NLK and its potential effect in ovarian cancer chemotherapy.Int J Gynecol Cancer,2011,21(8):1380-1387.
                                    </a>
                                </li>
                                <li id="22">


                                    <a id="bibliography_10" title=" Psyrri A,et al.Effect of epidermal growth factor receptor expression level on survival in patients with epithelial ovarian cancer.Clin Cancer Res,2005,11(24 Pt 1):8637-8643." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Effect of epidermal growth factor receptor expression level on survival in patients with epithelial ovarian cancer">
                                        <b>[10]</b>
                                         Psyrri A,et al.Effect of epidermal growth factor receptor expression level on survival in patients with epithelial ovarian cancer.Clin Cancer Res,2005,11(24 Pt 1):8637-8643.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=ZGWT" target="_blank">中国卫生统计</a>
                2019,36(04),484-488             </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm">基于贝叶斯准则的差异网络分析方法研究与应用</span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%AE%8B%E5%BE%AE&amp;code=42823056&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">宋微</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E7%8E%8B%E6%96%87%E6%9D%B0&amp;code=36315944&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">王文杰</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BE%90%E6%AC%A2&amp;code=42823057&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">徐欢</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E8%94%A1%E9%9B%A8%E6%99%B4&amp;code=42823058&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蔡雨晴</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%9D%8E%E5%BA%B7&amp;code=07019514&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">李康</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%93%88%E5%B0%94%E6%BB%A8%E5%8C%BB%E7%A7%91%E5%A4%A7%E5%AD%A6%E5%8D%AB%E7%94%9F%E7%BB%9F%E8%AE%A1%E5%AD%A6%E6%95%99%E7%A0%94%E5%AE%A4&amp;code=0161188&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">哈尔滨医科大学卫生统计学教研室</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p><b>目的</b> 探讨基于贝叶斯准则的差异网络分析方法的性能,并将其应用于卵巢癌基因表达谱数据分析。<b>方法</b> 通过模拟实验评价其识别差异边及差异节点的准确性,并与传统方法做对比。同时应用上皮性卵巢癌基因组学数据,构建差异网络模型。<b>结果</b> 模拟试验结果表明,基于贝叶斯准则的差异网络分析方法识别差异边能力明显优于高斯图模型方法;实例分析结果表明,本文方法构建的差异网络模型具有实际意义。<b>结论</b> 应用基于贝叶斯准则的差异网络分析方法能得出准确度较高的差异网络,效果优于传统方法。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%B4%9D%E5%8F%B6%E6%96%AF%E6%A8%A1%E5%9E%8B%E9%80%89%E6%8B%A9&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">贝叶斯模型选择;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E9%AB%98%E6%96%AF%E5%9B%BE%E6%A8%A1%E5%9E%8B&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">高斯图模型;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%B7%AE%E5%BC%82%E7%BD%91%E7%BB%9C&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">差异网络;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    *李康,E-mail:likang@ems.hrbmu.edu.cn;
                                </span>
                    </p>
                
                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金(81773551);</span>
                    </p>
            </div>
                    <h1><b>Differential Gene Regulatory Network Inference Method Based on Bayesian Model Selection and its Application</b></h1>
                    <h2>
                    <span>Song Wei</span>
                    <span>Wang Wenjie</span>
                    <span>Xu Huan</span>
            </h2>
                    <h2>
                    <span>Department of Medical Statistics,Harbin Medical University</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p><b>Objective</b> To explore the performance of differential gene regulatory network inference method baseds on Bayesian model selection,and to apply it to the analysis of ovarian cancer gene expression data.<b>Methods</b> The efficiency of the algorithm given in this article was testified with simulation data and compared with traditional methods,and this algorithm was used to construct differential gene regulatory networks with ovarian cancer gene expression data.<b>Results</b> Differential gene regulatory network inference method based on Bayesian model selection performed better than the general Gaussian graphical models in differential edges inference.The results of gene expression data analysis also indicated that this algorithm could provide valuable differential network structures.<b>Conclusion</b> Differential gene regulatory network inference method based on Bayesian model selection can establish highly precise differential network models, performed better than traditional methods.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Bayesian%20model%20selection&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Bayesian model selection;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Gaussian%20graphical%20models&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Gaussian graphical models;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Differential%20gene%20regulatory%20network&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Differential gene regulatory network;</a>
                </p>
            </div>
        

        <!--brief start-->
                        <div class="p1">
                    <p id="24">基因的作用通常不是独立的,而是基因之间相互影响、相互制约,形成复杂的基因调控网络。某些疾病发生时,基因表达量变化不明显,而基因网络中调控关系却发生明显改变,这种情况传统的差异分析方法通常无法识别。而在实验条件下,由于独立误差、人为因素、试验条件限制等原因,有些差异调控关系往往不能被研究者发现。然而,在大数据网络结构分析中便可被发现具有统计学显著性。因此,在疾病研究领域,差异网络的研究已越来越受到研究者的重视。目前最常用的基因调控网络模型有布尔网络模型、线性组合模型、加权矩阵模型、高斯图模型和贝叶斯网络模型等<citation id="94" type="reference"><link href="4" rel="bibliography" /><link href="6" rel="bibliography" /><sup>[<a class="sup">1</a>,<a class="sup">2</a>]</sup></citation>。其中图模型是在变量条件独立的情况下构建基因网络,排除了其他变量的影响,因此能在复杂的相互关系中准确提取出两变量间的真实调控关系。贝叶斯因子(Bayes factor,BF)是基于贝叶斯准则研究得出的方法,广泛用于模型比较,通过计算BF可得出差异模型的直接证据。本文将高斯图模型与贝叶斯准则相结合,建立差异网络模型,推断出网络中差异的调控关系,并与一般的高斯图模型方法相比较,考察其有效性。最后运用本文给出的方法对卵巢癌基因表达谱数据进行分析,做出生物学解释。</p>
                </div>
                <h3 id="25" name="25" class="anchor-tag">原理与方法</h3>
                <h4 class="anchor-tag" id="26" name="26">1.高斯图模型</h4>
                <div class="p1">
                    <p id="27">高斯图模型中,随机变量<i>X</i><sub>1</sub>,……,<i>X</i><sub><i>m</i></sub>符合多元正态分布X～<i>N</i><mathml id="69"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>(</mo><mrow><mi>μ</mi><mo>,</mo><mstyle displaystyle="true"><mo>∑</mo><mrow></mrow></mstyle></mrow><mo>)</mo></mrow></mrow></math></mathml>,若节点<i>i</i>和节点<i>j</i>条件独立,则二者之间没有边存在。条件独立,即协方差矩阵的逆矩阵∑<mathml id="70"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msubsup></mrow></math></mathml>=0。判断这种条件独立关系是否成立,常用的方法是求解随机变量<i>X</i><sub><i>i</i></sub>,<i>X</i><sub><i>j</i></sub>关于剩余变量<i>X</i><sub>-</sub><sub><i>i</i></sub><sub>,-</sub><sub><i>j</i></sub>的偏相关系数,即:</p>
                </div>
                <div class="p1">
                    <p id="28" class="code-formula"><mathml id="71"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>ρ</mi><msub><mrow></mrow><mrow><mi>X</mi><msub><mrow></mrow><mi>i</mi></msub></mrow></msub><msub><mrow></mrow><mrow><mi>X</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></msub><msub><mrow></mrow><mrow><mrow><mo>|</mo><mrow><mi>X</mi><msub><mrow></mrow><mrow><mo>-</mo><mi>i</mi><mo>,</mo><mo>-</mo><mi>j</mi></mrow></msub></mrow></mrow></mrow></msub><mo>=</mo></mrow></math></mathml><mathml id="72"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mrow><mo>-</mo><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msubsup><mrow></mrow></mstyle></mrow><mrow><msqrt><mrow><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>,</mo><mi>i</mi></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msubsup><mrow></mrow></mstyle><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>,</mo><mi>j</mi></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msubsup><mrow></mrow></mstyle></mrow></msqrt></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="29">基于以上高斯理论,当<mathml id="73"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>ρ</mi><msub><mrow></mrow><mrow><mi>X</mi><msub><mrow></mrow><mi>i</mi></msub></mrow></msub><msub><mrow></mrow><mrow><mi>X</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></msub><msub><mrow></mrow><mrow><mrow><mo>|</mo><mrow><mi>X</mi><msub><mrow></mrow><mrow><mo>-</mo><mi>i</mi><mo>,</mo><mo>-</mo><mi>j</mi></mrow></msub></mrow></mrow></mrow></msub><mo>≠</mo><mn>0</mn></mrow></math></mathml>时,变量<i>i</i>与变量<i>j</i>之间的边存在于高斯图模型中。</p>
                </div>
                <div class="p1">
                    <p id="30">在处理高维组学数据时,往往会遇到“<i>m</i>≪<i>n</i>”的问题,即变量数<i>m</i>远大于样本数<i>n</i>,此时协方差矩阵不是唯一的,因此传统的统计模型无法求出偏相关系数。针对此种情况,有学者提出应用调整回归模型推导两变量的偏相关系数,将变量<i>X</i><sub><i>i</i></sub>作为因变量,剩余变量<i>X</i><sub><i>j</i></sub>(<i>j</i>≠<i>i</i>)作为自变量,构建回归方程:</p>
                </div>
                <div class="p1">
                    <p id="31"><i>X</i><sub><i>i</i></sub>=<mathml id="74"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>≠</mo><mi>i</mi></mrow><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></munderover><mi>β</mi></mstyle><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup><mi>X</mi><msub><mrow></mrow><mi>j</mi></msub><mo>+</mo><mi>ε</mi><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>2</mn><mo stretchy="false">)</mo></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="32">其中<i>β</i><mathml id="75"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></mathml>表示第<i>i</i>个节点对其他变量的回归系数,误差<i>ε</i>服从以0为均数的正态分布,即<i>ε</i>～<i>N</i>(0,<i>σ</i><sup>2</sup>)。采用稀疏贝叶斯非参数回归方法(sparse Bayesian nonparametric regression)<citation id="95" type="reference"><link href="8" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>对回归系数<i>β</i><mathml id="76"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></mathml>进行估计。通过混合高斯分布标准化对β<mathml id="77"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></mathml>进行稀疏处理,得出β<mathml id="78"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></mathml>的边缘概率分布函数:</p>
                </div>
                <div class="p1">
                    <p id="33"><mathml id="79"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>p</mi><mrow><mo>(</mo><mrow><mi>β</mi><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow><mo>)</mo></mrow><mo>=</mo></mrow></math></mathml>∫<i>N</i>(<i>β</i><mathml id="80"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></mathml>;0,<i>ζ</i><mathml id="81"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mn>2</mn></msubsup></mrow></math></mathml>)<i>p</i>(<i>ζ</i><mathml id="82"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mn>2</mn></msubsup></mrow></math></mathml>)<i>dζ</i><mathml id="83"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mn>2</mn></msubsup></mrow></math></mathml>      (3)</p>
                </div>
                <div class="p1">
                    <p id="34">这里的ζ<mathml id="84"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mn>2</mn></msubsup></mrow></math></mathml>服从逆高斯分布,即</p>
                </div>
                <div class="p1">
                    <p id="35">ζ<mathml id="85"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mn>2</mn></msubsup></mrow></math></mathml>～<i>IG</i><mathml id="86"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>(</mo><mrow><mi>α</mi><mo>,</mo><mi>γ</mi></mrow><mo>)</mo></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>4</mn><mo stretchy="false">)</mo></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="36">公式中的α和γ控制对β<mathml id="87"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></mathml>的惩罚。通过迭代更新σ<sup>2</sup>、ζ<mathml id="88"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mn>2</mn></msubsup></mrow></math></mathml>、β<mathml id="89"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></mathml>的值,求出β<mathml id="90"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></mathml>的最大后验估计值(<i>maximum a posteriori</i>,<i>MAP</i>)。最后利用以下公式估计出变量i和变量j的偏相关系数:</p>
                </div>
                <div class="p1">
                    <p id="37" class="code-formula">
                        <mathml id="37"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>ρ</mi><msub><mrow></mrow><mrow><mi>X</mi><msub><mrow></mrow><mi>i</mi></msub></mrow></msub><msub><mrow></mrow><mrow><mi>X</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></msub><msub><mrow></mrow><mrow><mrow><mo>|</mo><mrow><mi>X</mi><msub><mrow></mrow><mrow><mo>-</mo><mi>i</mi><mo>,</mo><mo>-</mo><mi>j</mi></mrow></msub></mrow></mrow></mrow></msub><mo>=</mo></mtd></mtr><mtr><mtd><mrow><mo>(</mo><mrow><mtable><mtr><mtd columnalign="left"><mtext>s</mtext><mtext>i</mtext><mtext>g</mtext><mtext>n</mtext><mo stretchy="false">(</mo><mi>β</mi><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup><mo stretchy="false">)</mo><mi>min</mi><mrow><mo>(</mo><mrow><mn>1</mn><mo>,</mo><msqrt><mrow><mi>β</mi><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup><mi>β</mi><msubsup><mrow></mrow><mi>i</mi><mrow><mo stretchy="false">(</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></msqrt></mrow><mo>)</mo></mrow><mo>,</mo></mtd><mtd columnalign="left"><mtext>s</mtext><mtext>i</mtext><mtext>g</mtext><mtext>n</mtext><mo stretchy="false">(</mo><mi>β</mi><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup><mo stretchy="false">)</mo><mo>=</mo><mtext>s</mtext><mtext>i</mtext><mtext>g</mtext><mtext>n</mtext><mrow><mo>(</mo><mrow><mi>β</mi><msubsup><mrow></mrow><mi>i</mi><mrow><mo stretchy="false">(</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow><mo>)</mo></mrow></mtd></mtr><mtr><mtd columnalign="left"><mn>0</mn><mo>,</mo></mtd><mtd columnalign="left"><mtext>s</mtext><mtext>i</mtext><mtext>g</mtext><mtext>n</mtext><mrow><mo>(</mo><mrow><mi>β</mi><msubsup><mrow></mrow><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow><mo>)</mo></mrow><mo>≠</mo><mtext>s</mtext><mtext>i</mtext><mtext>g</mtext><mtext>n</mtext><mrow><mo>(</mo><mrow><mi>β</mi><msubsup><mrow></mrow><mi>i</mi><mrow><mo stretchy="false">(</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow><mo>)</mo></mrow></mtd></mtr></mtable></mrow></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>5</mn><mo stretchy="false">)</mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <h4 class="anchor-tag" id="38" name="38">2.差异网络分析</h4>
                <div class="p1">
                    <p id="39">为推断不同条件下网络中的差异结构,将含有m个变量的数据按K个分类标签分成不同数据集。对于公式(2),记Y=X<sub>i</sub>。根据极大似然和平均场近似估计理论,可以直接得出K个独立网络结构满足:</p>
                </div>
                <div class="p1">
                    <p id="40">p(β<sub>k</sub>,ζ<mathml id="91"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>k</mi><mn>2</mn></msubsup></mrow></math></mathml>,σ<sup>2</sup>|Y<sub>k</sub>,X<sub>k</sub>)</p>
                </div>
                <div class="p1">
                    <p id="41" class="code-formula">
                        <mathml id="41"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mo>∝</mo><mstyle displaystyle="true"><munderover><mo>∏</mo><mi>k</mi><mi>Κ</mi></munderover><mo stretchy="false">[</mo></mstyle><mi>p</mi><mo stretchy="false">(</mo><mi>Y</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">|</mo><mi>X</mi><msub><mrow></mrow><mi>k</mi></msub><mo>,</mo><mi>β</mi><msub><mrow></mrow><mi>k</mi></msub><mo>,</mo><mi>σ</mi><msup><mrow></mrow><mn>2</mn></msup><mo stretchy="false">)</mo><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>j</mi><mo>≠</mo><mi>i</mi></mrow><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></munderover><mo stretchy="false">(</mo></mstyle><mi>p</mi><mo stretchy="false">(</mo><mi>β</mi><msub><mrow></mrow><mrow><mi>k</mi><mi>j</mi></mrow></msub><mo stretchy="false">|</mo><mi>ζ</mi><msubsup><mrow></mrow><mrow><mi>k</mi><mi>j</mi></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo><mi>p</mi><mo stretchy="false">(</mo><mi>ζ</mi><msubsup><mrow></mrow><mrow><mi>k</mi><mi>j</mi></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo stretchy="false">]</mo><mi>p</mi><mo stretchy="false">(</mo><mi>σ</mi><msup><mrow></mrow><mn>2</mn></msup><mo stretchy="false">)</mo></mtd></mtr><mtr><mtd><mo>⇁</mo><mstyle displaystyle="true"><munderover><mo>∏</mo><mi>k</mi><mi>Κ</mi></munderover><mi>q</mi></mstyle><mo stretchy="false">(</mo><mi>β</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">)</mo><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>j</mi><mo>≠</mo><mi>i</mi></mrow><mrow><mi>m</mi><mo>-</mo><mn>1</mn></mrow></munderover><mi>q</mi></mstyle><mo stretchy="false">(</mo><mi>ζ</mi><msub><mrow></mrow><mrow><mi>k</mi><mi>j</mi></mrow></msub><mo stretchy="false">)</mo><mi>q</mi><mo stretchy="false">(</mo><mi>σ</mi><msup><mrow></mrow><mn>2</mn></msup><mo stretchy="false">)</mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>6</mn><mo stretchy="false">)</mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="42">其中<i>β</i><sub><i>k</i></sub>表示第<i>k</i>类的回归系数向量,<i>q</i>为模型参数的分布,可通过平均场近似估计理论结合变分贝叶斯推断理论求解出最优分布<mathml id="92"><math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mi>q</mi><mo>^</mo></mover></math></mathml>,代入公式得出模型参数的近似后验分布<i>p</i>(<i>β</i><sub><i>k</i></sub>,<i>ζ</i><mathml id="93"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>k</mi><mn>2</mn></msubsup></mrow></math></mathml>,<i>σ</i><sup>2</sup>|<i>Y</i><sub><i>k</i></sub>,<i>X</i><sub><i>k</i></sub>)。</p>
                </div>
                <div class="p1">
                    <p id="43">对于给定的模型<i>M</i>(分类),基于模型参数的变分推断方法,结合变分下限和可逆跳跃马尔科夫链蒙特卡洛方法(RJMCMC)<citation id="96" type="reference"><link href="10" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>,可以求得模型证据<i>p</i>(<i>X</i>|<i>M</i>)的估计值:</p>
                </div>
                <div class="p1">
                    <p id="44"><i>p</i>(<i>X</i>|<i>M</i>)=∫<i>p</i>(<i>X</i>|<i>θ</i>,<i>M</i>)<i>p</i>(<i>θ</i>|<i>M</i>)<i>d</i>(<i>θ</i>)      (7)</p>
                </div>
                <div class="p1">
                    <p id="45">公式中<i>X</i>表示模型中所有变量的观察值,<i>θ</i>代表所有模型参数的集合。对于<i>K</i>=1和<i>K</i>=2两个模型,通过计算贝叶斯因子</p>
                </div>
                <div class="p1">
                    <p id="46" class="code-formula">
                        <mathml id="46"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>B</mi><mi>F</mi><msup><mrow></mrow><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msup><mo>=</mo><mfrac><mrow><mi>p</mi><mo stretchy="false">(</mo><mi>X</mi><mo stretchy="false">|</mo><mi>Μ</mi><msub><mrow></mrow><mrow><mi>Κ</mi><mo>=</mo><mn>2</mn></mrow></msub><mo stretchy="false">)</mo></mrow><mrow><mi>p</mi><mo stretchy="false">(</mo><mi>X</mi><mo stretchy="false">|</mo><mi>Μ</mi><msub><mrow></mrow><mrow><mi>Κ</mi><mo>=</mo><mn>1</mn></mrow></msub><mo stretchy="false">)</mo></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>8</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="47">判断支持哪个模型成立的证据更为充分,即各变量(如基因)对其他变量调控关系在两类中是否有显著的差异。对每个变量<i>X</i><sub>1</sub>,……,<i>X</i><sub><i>m</i></sub>计算贝叶斯因子,如果大于给定阈值,则进一步根据偏相关系数判断差异边,最后根据给定的假发现率<i>FDR</i>值判定网络中的调控边,得出差异网络。</p>
                </div>
                <h3 id="48" name="48" class="anchor-tag">模拟实验</h3>
                <div class="p1">
                    <p id="49">研究通过对设置的网络结构偏相关系数矩阵的运算产生不同样本量的模拟数据<citation id="97" type="reference"><link href="6" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>。检验基于贝叶斯准则的差异网络分析方法(记为Bayes),将贝叶斯因子选择的差异网络模型与设置的真实差异网络结构进行比较,若分析结果能够获得更多的已知差异网络中存在的边,则说明该方法有更好的分析效果。根据前期研究结果和大量文献经验结果,本模拟取<i>BF</i><sup>(</sup><sup><i>i</i></sup><sup>)</sup>&gt;2和<i>FDR</i>&lt;0.025。</p>
                </div>
                <div class="p1">
                    <p id="50">模拟产生两组数据,网络中共包含50个节点,其中有6个为差异节点(G30、G38、G1、G37、G4、G42),共有49条共同边和8条差异边(黑色),49条共同边偏相关系数在(-1,1)随机抽取,8条差异边的偏相关系数在(0.2,0.5)随机抽取。黑色边上的数字“1”和“2”分别表示两种不同情况下网络中的差异边。数字“1”为仅在第一组数据中有调控关系,数字“2”为仅在第二组数据中有调控关系(参见图1和表1)。模拟选取样本量<i>n</i><sub>1</sub>=<i>n</i><sub>2</sub>={30,60,90,120,150},模拟实验重复100次。</p>
                </div>
                <div class="area_img" id="51">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201904002_051.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 模拟实验1设置的真实差异网络关系图" src="Detail/GetImg?filename=images/ZGWT201904002_051.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图1</b> 模拟实验1设置的真实差异网络关系图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201904002_051.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="52">
                    <p class="img_tit"><b>表1</b> 真实差异网络中差异边 <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="52" border="1"><tr><td><br />差异边</td><td>节点1</td><td>节点2</td><td>第一组</td><td>第二组</td></tr><tr><td><br />1</td><td>G30</td><td>G1</td><td>1</td><td>0</td></tr><tr><td><br />2</td><td>G30</td><td>G4</td><td>1</td><td>0</td></tr><tr><td><br />3</td><td>G30</td><td>G37</td><td>1</td><td>0</td></tr><tr><td><br />4</td><td>G30</td><td>G42</td><td>1</td><td>0</td></tr><tr><td><br />5</td><td>G38</td><td>G1</td><td>0</td><td>1</td></tr><tr><td><br />6</td><td>G38</td><td>G4</td><td>0</td><td>1</td></tr><tr><td><br />7</td><td>G38</td><td>G37</td><td>0</td><td>1</td></tr><tr><td><br />8</td><td>G38</td><td>G42</td><td>0</td><td>1</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="53">对数据分别使用Bayes、GeneNet、FastGGM三种方法进行分析,其中后两种方法是两种常用的高斯图模型方法<citation id="98" type="reference"><link href="6" rel="bibliography" /><link href="12" rel="bibliography" /><sup>[<a class="sup">2</a>,<a class="sup">5</a>]</sup></citation>。模型的评价指标选用准确度、假发现率、灵敏度和特异度,其中准确度为真阳性边占检查出的阳性边的比例,相当于诊断试验中的阳性预测值,假发现率为假阳性边占检查出的阳性边的比例。Bayes方法模拟实验结果显示在图2中,可以看到:每组样本量为60时,准确度维持在0.6左右;在每组样本量为90时,差异节点的灵敏度达到1,差异边的灵敏度在0.75附近。图3显示了三种不同方法在不同样本量下的准确度和假发现率,可以看到,Bayes方法明显优于另外两种常用的方法。</p>
                </div>
                <div class="area_img" id="54">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201904002_054.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 模拟实验各评价指标随样本量不同的变化情况" src="Detail/GetImg?filename=images/ZGWT201904002_054.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图2</b> 模拟实验各评价指标随样本量不同的变化情况  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201904002_054.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="55">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201904002_055.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 三种方法在不同样本量下准确度和假发现率箱式图" src="Detail/GetImg?filename=images/ZGWT201904002_055.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图3</b> 三种方法在不同样本量下准确度和假发现率箱式图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201904002_055.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h3 id="56" name="56" class="anchor-tag">实例分析</h3>
                <div class="p1">
                    <p id="57">本研究通过对卵巢癌患者复发情况及基因表达数据进行分析,应用基于贝叶斯准则的差异网络分析方法构建差异网络模型,得出基因间差异调控关系,及存在差异调控关系的基因。通过结合生物学知识、通路数据库、文献查询,对差异网络结果进行生物学解释,从基因组学角度为卵巢癌复发机制提供线索。</p>
                </div>
                <div class="p1">
                    <p id="58">本研究从R包curatedOvarianData (version:1.8.0)中下载Ⅱ-Ⅳ期上皮性卵巢癌患者基因表达谱数据(GSE49997),排除了临床信息中缺失复发时间以及复发时间&lt;90天的患者,基因表达数据应用<i>Z</i>-score法进行标准化。将患者根据生存情况分为复发组124例和非复发组70例,全基因组表达谱数据一共测得16024个基因的表达值。由于基因数目过多,需要先筛选出与上皮性卵巢癌复发相关的基因通路,再对通路中的全部基因构建差异网络模型。变量筛选有助于提高建模效率,使得差异网络模型更加合理。本研究使用基于LASSO的Cox比例风险回归模型,对通路富集后的基因进行得分计算,并通过得分矩阵对通路进行筛选<citation id="99" type="reference"><link href="14" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>,最终给出与卵巢癌复发有相关的12条通路。其中,FoxO信号通路经动物实验证明有抑癌作用<citation id="100" type="reference"><link href="16" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>,同时,研究表明FoxO蛋白表达量与卵巢癌生存期相关<citation id="101" type="reference"><link href="18" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>。基于以上结果,本研究选取FoxO信号通路中全部基因,在复发组和非复发组中进行差异网络模型的构建与分析。</p>
                </div>
                <div class="p1">
                    <p id="59">将映射在FoxO信号通路中的119个基因的表达数据整理出来,标准化处理后结合复发状态数据进行差异网络模型的构建。应用基于贝叶斯准则的差异网络分析方法构建差异网络模型,结果如图4所示,其中节点代表FoxO信号通路中的基因,黑色虚线代表复发与非复发状态下的共同边,13条黑色虚线代表只存在于复发患者中的差异边,9条黑色实线则代表只存在于非复发患者中的差异边(表2)。存在差异调控关系的基因如表3所示,其对应的差异调控关系频数越大,越可能成为差异网络中的枢纽基因(Hub Gene)。</p>
                </div>
                <div class="area_img" id="60">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201904002_060.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 卵巢癌患者复发与非复发状态下的差异基因调控网络" src="Detail/GetImg?filename=images/ZGWT201904002_060.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图4</b> 卵巢癌患者复发与非复发状态下的差异基因调控网络  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201904002_060.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="61">通过查询GeneMANIA及KEGG基因数据库,发现有8条差异边出现在数据库中,例如NLK与PRKAA1、FOXO4和SGK2三个基因间的调控关系。表3中NLK、STK4、HOMER1、PRKAA1、EGFR五个基因与多个基因间存在差异调控关系,可以将他们视为差异网络中的枢纽基因。有文献报道NLK与卵巢癌分期、分级、化疗和预后有关,NLK基因的高表达加速了顺铂治疗中的细胞凋亡过程,从而延长病人生存期<citation id="102" type="reference"><link href="20" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>。另一枢纽基因EGFR与肿瘤细胞的增殖、血管生成、肿瘤侵袭、转移及细胞凋亡的抑制有关,EGFR的表达水平可作为上皮性卵巢癌患者生存期的预后因子,其高表达与生存期的降低相关<citation id="103" type="reference"><link href="22" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>。</p>
                </div>
                <div class="area_img" id="62">
                                            <p class="img_tit">
                                                <b>表2</b> 卵巢癌患者复发状态差异网络中的差异调控关系
                                                
                                                &nbsp;&nbsp;
                                                <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201904002_06200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a>
                                                <a class="table downimg" data-tablename="Detail/GetImg?filename=images/ZGWT201904002_06200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">下载原表</a>
                                            </p>
                                    <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201904002_06200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <img alt="表2 卵巢癌患者复发状态差异网络中的差异调控关系" src="Detail/GetImg?filename=images/ZGWT201904002_06200.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                    </a>
                                            <p class="img_tit">
                                                &nbsp;&nbsp;
                                                <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201904002_06201.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a>
                                                <a class="table downimg" data-tablename="Detail/GetImg?filename=images/ZGWT201904002_06201.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">下载原表</a>
                                            </p>
                                    <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201904002_06201.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <img alt="表2 卵巢癌患者复发状态差异网络中的差异调控关系" src="Detail/GetImg?filename=images/ZGWT201904002_06201.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                    </a>

                </div>
                <div class="area_img" id="63">
                                            <p class="img_tit">
                                                <b>表3</b> 卵巢癌患者复发状态差异网络中的差异基因及其对应的差异调控关系频数
                                                
                                                &nbsp;&nbsp;
                                                <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201904002_06300.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a>
                                                <a class="table downimg" data-tablename="Detail/GetImg?filename=images/ZGWT201904002_06300.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">下载原表</a>
                                            </p>
                                    <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201904002_06300.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <img alt="表3 卵巢癌患者复发状态差异网络中的差异基因及其对应的差异调控关系频数" src="Detail/GetImg?filename=images/ZGWT201904002_06300.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                    </a>
                                            <p class="img_tit">
                                                &nbsp;&nbsp;
                                                <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201904002_06301.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a>
                                                <a class="table downimg" data-tablename="Detail/GetImg?filename=images/ZGWT201904002_06301.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">下载原表</a>
                                            </p>
                                    <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201904002_06301.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <img alt="表3 卵巢癌患者复发状态差异网络中的差异基因及其对应的差异调控关系频数" src="Detail/GetImg?filename=images/ZGWT201904002_06301.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                    </a>

                </div>
                <h3 id="64" name="64" class="anchor-tag">讨  论</h3>
                <div class="p1">
                    <p id="65">传统变量筛选方法主要针对不同条件下基因表达量的差异,无法准确识别表达量未发生明显变化但调控关系改变的基因。基于贝叶斯准则的差异网络分析方法是在高斯图模型基础上,给出的一种能够识别不同条件下调控关系改变的变量和调控边的一种方法。该方法利用贝叶斯原理,在假定的先验分布上对模型的参数进行估计,获得偏相关系数,在此基础上从整体网络的角度进行差异推断。</p>
                </div>
                <div class="p1">
                    <p id="66">模拟实验结果表明,差异贝叶斯方法具有较好的识别差异边的能力,本研究得到以下几点结论:①基于贝叶斯准则的差异网络分析方法结果可靠,即在保证高准确度的同时,还具有较低假发现率的特点;②该方法受样本量的影响较小,在样本量小于变量数的情况下也能识别差异网络中的差异调控关系;③传统的高斯图模型推断差异网络时,需要在不同情况下单独建网后比较差异边,相比之下,加入贝叶斯因子进行模型选择,提高了高斯图模型识别差异边的能力。</p>
                </div>
                <div class="p1">
                    <p id="67">通过实例分析,对上皮性卵巢癌复发和非复发的调控网络进行了对比,得出两组在FOXO信号通路中差异调控关系及差异网络的枢纽基因,为卵巢癌复发的基因调整网络机制的研究提供了线索。</p>
                </div>
                <div class="p1">
                    <p id="68">应用该方法理论上对变量的数目没有限制,因此适用任何差异基因调控网络的比较,并能准确地进行差异调控边的定位。这种方法可以通过贝叶斯因子得出差异模型证据,进而从数值上检验差异存在的可能性大小。然而,生物网络通常是非常复杂的,要确证不同条件下的真实差异调控关系,还必须通过生物学实验进行验证。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="4">
                            <a id="bibliography_1" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT200901032&amp;v=MDc3MzBabkZ5RG1VYi9LUHlyY2VyRzRIdGpNcm85R1pvUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnU=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[1]</b> 游顶云,李康,贝叶斯网络方法在基因调控研究中的应用.中国卫生统计,2009,26(1):83-86.
                            </a>
                        </p>
                        <p id="6">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=From correlation to causation networks: a simple approximate learning algorithm and its application to high-dimensional plant gene expression data">

                                <b>[2]</b> Opgen-Rhein R,Strimmer K.From correlation to causation networks:a simple approximate learning algorithm and its application to high-dimensional plant gene expression data.BMC SystBiol,2007,1:37.
                            </a>
                        </p>
                        <p id="8">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Sparse Bayesian Nonparametric Regression">

                                <b>[3]</b> Caron F,Doucet A.Sparse Bayesian Nonparametric Regression.In ICML' 08,2008:88-95.
                            </a>
                        </p>
                        <p id="10">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Reversible jump Markov chain Monte Carlo computation and Bayesian model determination">

                                <b>[4]</b> Green PJ.Reversible jump Markov chain Monte Carlo computation and Bayesian model determination.Biometrika,1995,82(4):711-732.
                            </a>
                        </p>
                        <p id="12">
                            <a id="bibliography_5" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJLA&amp;filename=SJLA85764D0CA1ADB3A89CA35202643013F4&amp;v=MjYxODJpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0Rmh6THU5d0tvPU5pZkhiN3U5R2RmSTI0ODJGZXArZUE0NnZoNGFtVTUrVFgzaXJoUXhlcktWUnN5YkNPTnZGUw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[5]</b> Wang T,et al.FastGGM:An Efficient Algorithm for the Inference of Gaussian Graphical Model in Biological Networks.PLoS Comput Biol,2016,12(2):e1004755.
                            </a>
                        </p>
                        <p id="14">
                            <a id="bibliography_6" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD1DD1CDA709026ADDD08DE5E3D5D248BA&amp;v=MzI1MzFXV3I3SklGcG1hQnVIWWZPR1FsZkJyTFUwNXRGaHpMdTl3S289TmlmY2FyTE1hdEMvMi81Q1pPSVBEbnBJdTJKbjZqY0pQWHFYcjJZd0RiQ1FUY2p1Q09OdkZTaQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[6]</b> Deng K,Zhang F,Song W,et al.Identification of pathway-based recurrence-associated signatures in optimally debulked patients with serous ovarian cancer.J Cell Biochem,2018:1-10.
                            </a>
                        </p>
                        <p id="16">
                            <a id="bibliography_7" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011600217979&amp;v=MTM4MjJvSUJYc3dvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp0RmlubFVyYklLRjBXYWhBPU5pZk9mYks3SHRETnFZOUZadQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[7]</b> Paik JH,et al.FoxOs are lineage-restricted redundant tumor suppressors and regulate endothelial cell homeostasis.Cell,2007,128(2):309-23.
                            </a>
                        </p>
                        <p id="18">
                            <a id="bibliography_8" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJSD&amp;filename=SJSD14101400000249&amp;v=MjA0NzNySzhIOUhOcTQ5RlpPc1BEbmd3b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadEZpbmxVcmJJS0YwV2FoQT1OaWZZYQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[8]</b> Park SH,et al.Auranofin displays anticancer activity against ovarian cancer cells through FOXO3 activation independent of p53.Int J Oncol,2014,45(4):1691-8.
                            </a>
                        </p>
                        <p id="20">
                            <a id="bibliography_9" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWK&amp;filename=SJWK12111300062770&amp;v=MTU4MTFpclJkR2VycVFUTW53WmVadEZpbmxVcmJJS0YwV2FoQT1OaWZjWmJLNkg5RE5ySTlGWk8wTkMzczVvQk1UNlQ0UFFILw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[9]</b> Zhang Y,et al.Expression of NLK and its potential effect in ovarian cancer chemotherapy.Int J Gynecol Cancer,2011,21(8):1380-1387.
                            </a>
                        </p>
                        <p id="22">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Effect of epidermal growth factor receptor expression level on survival in patients with epithelial ovarian cancer">

                                <b>[10]</b> Psyrri A,et al.Effect of epidermal growth factor receptor expression level on survival in patients with epithelial ovarian cancer.Clin Cancer Res,2005,11(24 Pt 1):8637-8643.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="ZGWT201904002" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201904002&amp;v=MDc2MzlLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1Wm5GeURtVWIvS1B5cmNlckc0SDlqTXE0OUZab1E=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=842_VlRz_g1CMCLi-M0twVw-4EOFG_noWyi_OsMnrAg1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
