

<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>

</head>

<body>

    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2fKXReader%2fDetail%3fTIMESTAMP%3d637141086130416250%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dZGWT201905040%26RESULT%3d1%26SIGN%3deQthji3YR39FSIPTEbCfzBEJIUE%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201905040&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201905040&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>


    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201905040&amp;v=MTAyNTBabkZ5RG5Vci9OUHlyY2VyRzRIOWpNcW85QlpJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnU=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#12" data-title="原理和方法 ">原理和方法</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#14" data-title="1.数据预处理">1.数据预处理</a></li>
                                                <li><a href="#25" data-title="2.确定共同矩阵和独立矩阵的秩">2.确定共同矩阵和独立矩阵的秩</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#33" data-title="实例分析 ">实例分析</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#44" data-title="讨  论 ">讨  论</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#36" data-title="&lt;b&gt;图1&lt;/b&gt; JIVE分解得到三部分解释方差比"><b>图1</b> JIVE分解得到三部分解释方差比</a></li>
                                                <li><a href="#38" data-title="&lt;b&gt;图2&lt;/b&gt; mRNA及JIVE分解前后miRNA表达数据PCA分类图对比"><b>图2</b> mRNA及JIVE分解前后miRNA表达数据PCA分类图对比</a></li>
                                                <li><a href="#41" data-title="&lt;b&gt;表1&lt;/b&gt; 变量在前两个主成分中的因子载荷"><b>表1</b> 变量在前两个主成分中的因子载荷</a></li>
                                                <li><a href="#42" data-title="&lt;b&gt;图3&lt;/b&gt; 第一主成分的样本得分概率密度分布图"><b>图3</b> 第一主成分的样本得分概率密度分布图</a></li>
                                                <li><a href="#43" data-title="&lt;b&gt;图4&lt;/b&gt; 第二主成分的样本得分概率密度分布图"><b>图4</b> 第二主成分的样本得分概率密度分布图</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="3">


                                    <a id="bibliography_1" title=" Lock EF,Hoadley KA,Marron JS,et al.Joint and individual variation explained(JIVE)for integrated analysis of multiple data types.Annals of Applied Statistics,2013,7(1):523-542." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJST&amp;filename=SJSTD50AD57236D42A6FAB7ACD73B73A11C5&amp;v=Mjg4NDBSTW1hQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQnJMVTA1dEZoekxxK3dLMD1OaWZZZXNlOUhxQzRxb2hIWisxN0NINUl5V0JpbURnTU93dmxyMkF5ZXNPVg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[1]</b>
                                         Lock EF,Hoadley KA,Marron JS,et al.Joint and individual variation explained(JIVE)for integrated analysis of multiple data types.Annals of Applied Statistics,2013,7(1):523-542.
                                    </a>
                                </li>
                                <li id="5">


                                    <a id="bibliography_2" title=" O′Connell MJ,Lock EF,R.JIVE for exploration of multi-source molecular data.Bioinformatics,2016,32(18):2877-2879." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=JIVE for exploration of multi-source molecular data">
                                        <b>[2]</b>
                                         O′Connell MJ,Lock EF,R.JIVE for exploration of multi-source molecular data.Bioinformatics,2016,32(18):2877-2879.
                                    </a>
                                </li>
                                <li id="7">


                                    <a id="bibliography_3" title=" Network TCGA,Integrated genomic analyses of ovarian carcinoma.Nature,2011,474(7353):609-615." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Integrated genomic analyses of ovarian carcinoma">
                                        <b>[3]</b>
                                         Network TCGA,Integrated genomic analyses of ovarian carcinoma.Nature,2011,474(7353):609-615.
                                    </a>
                                </li>
                                <li id="9">


                                    <a id="bibliography_4" title=" Winterhoff B,Hamidi H,Wang C,et al.,Molecular classification of high grade endometrioid and clear cell ovarian cancer using TCGA gene expression signatures.Gynecologic Oncology,2016,141(1):95-100." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Molecular classification of high grade endometrioid and clear cell ovarian cancer using TCGA gene expression signatures">
                                        <b>[4]</b>
                                         Winterhoff B,Hamidi H,Wang C,et al.,Molecular classification of high grade endometrioid and clear cell ovarian cancer using TCGA gene expression signatures.Gynecologic Oncology,2016,141(1):95-100.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=ZGWT" target="_blank">中国卫生统计</a>
                2019,36(05),774-776             </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm">JIVE<b>方法在卵巢癌多组学数据整合分析中的应用</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BE%90%E6%AC%A2&amp;code=42823057&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">徐欢</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%AE%8B%E5%BE%AE&amp;code=42823056&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">宋微</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E8%94%A1%E9%9B%A8%E6%99%B4&amp;code=42823058&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蔡雨晴</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E4%BE%AF%E8%89%B3&amp;code=11211630&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">侯艳</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%9D%8E%E5%BA%B7&amp;code=07019514&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">李康</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%93%88%E5%B0%94%E6%BB%A8%E5%8C%BB%E7%A7%91%E5%A4%A7%E5%AD%A6%E5%8D%AB%E7%94%9F%E7%BB%9F%E8%AE%A1%E5%AD%A6%E6%95%99%E7%A0%94%E5%AE%A4&amp;code=0161188&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">哈尔滨医科大学卫生统计学教研室</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p><b>目的</b> 引入JIVE方法对多组学数据进行整合分析,并应用于肿瘤分子分型研究。<b>方法</b> 使用TCGA数据库中卵巢癌mRNA和miRNA的组学数据,应用JIVE方法整合分析两个组学数据,提取两不同组学数据的共同特征,然后通过对其具有共同结构的数据做主成分分析,给出卵巢癌miRNA分子分型的结果。<b>结果</b> 经过JIVE方法整合分析后,使miRNA数据具有明显与mRNA相一致的分型结构,从而进一步支持了mRNA的分型结果,同时揭示了两组学之间在组织分子分型上具有一定的调控关系。<b>结论</b> JIVE方法可以用于提取组学之间存在的共同结构矩阵,从而进行多组学数据的整合分析。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=JIVE%E6%96%B9%E6%B3%95&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">JIVE方法;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%A4%9A%E7%BB%84%E5%AD%A6%E6%95%B0%E6%8D%AE%E6%95%B4%E5%90%88&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">多组学数据整合;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%89%B9%E5%BE%81%E6%8F%90%E5%8F%96&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">特征提取;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    *李康,E-mail:likang@ems.hrbmu.edu.cn;
                                </span>
                    </p>
                
                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金(81773551,81573256);</span>
                    </p>
            </div>
                                                        

        <!--brief start-->
                        <div class="p1">
                    <p id="11">组学数据(omics data)是指通过测序仪、生物芯片、磁共振、色谱-质谱联用等高通量实验技术得到的基因组、转录组、蛋白质组和代谢组等数据。通过对高维多组学进行整合分析,可以研究相关疾病的分子分型及标志物,对疾病的早期诊断、临床治疗和预后具有重要意义。本文引进JIVE(joint and individual variation explained)方法<citation id="59" type="reference"><link href="3" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>,将其应用于卵巢癌mRNA和miRNA两组学数据的分析,揭示两种组学之间潜在的生物学关系,为研究其分子分型的调控机制提供有价值的分析结果。</p>
                </div>
                <h3 id="12" name="12" class="anchor-tag">原理和方法</h3>
                <div class="p1">
                    <p id="13">设有<i>k</i>个组学数据矩阵<i>X</i><sub>1</sub>,<i>X</i><sub>2</sub>,…,<i>X</i><sub><i>k</i></sub>,分别为<i>p</i><sub>1</sub>、…、<i>p</i><sub><i>k</i></sub>行和<i>n</i>列,其中行为各组学数据的变量,列为样本。JIVE方法的基础是矩阵的奇异值分解,即将不同组学的数据矩阵合并后分解为三部分,即描述不同组学数据的共同结构矩阵(<i>J</i>)、独立结构矩阵(<i>A</i>)和残差矩阵(<i>R</i>)。具体步骤如下:</p>
                </div>
                <h4 class="anchor-tag" id="14" name="14">1.数据预处理</h4>
                <div class="p1">
                    <p id="15">首先将多个矩阵按行合并,合并的数据矩阵:</p>
                </div>
                <div class="p1">
                    <p id="16" class="code-formula">
                        <mathml id="16"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>X</mi><mo>=</mo><mrow><mrow><mo>[</mo><mrow><mtable><mtr><mtd columnalign="left"><mi>X</mi><msub><mrow></mrow><mn>1</mn></msub></mtd></mtr><mtr><mtd columnalign="left"><mo>⋮</mo></mtd></mtr><mtr><mtd columnalign="left"><mi>X</mi><msub><mrow></mrow><mi>k</mi></msub></mtd></mtr></mtable></mrow><mo>]</mo></mrow></mrow><msub><mrow></mrow><mrow><mi>p</mi><mo>×</mo><mi>n</mi></mrow></msub><mo>,</mo><mi>p</mi><mo>=</mo><mi>p</mi><msub><mrow></mrow><mn>1</mn></msub><mo>+</mo><mi>p</mi><msub><mrow></mrow><mn>2</mn></msub><mo>+</mo><mo>⋯</mo><mo>+</mo><mi>p</mi><msub><mrow></mrow><mi>k</mi></msub><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>1</mn><mo stretchy="false">)</mo></mtd></mtr><mtr><mtd><mi>X</mi><msub><mrow></mrow><mi>i</mi></msub><mo>=</mo><mrow><mo>[</mo><mrow><mtable><mtr><mtd columnalign="left"><mi>X</mi><msub><mrow></mrow><mrow><mn>1</mn><mn>1</mn></mrow></msub><mo>,</mo><mi>X</mi><msub><mrow></mrow><mrow><mn>1</mn><mn>2</mn></mrow></msub><mo>,</mo><mo>⋯</mo><mo>,</mo><mi>X</mi><msub><mrow></mrow><mrow><mn>1</mn><mi>n</mi></mrow></msub></mtd></mtr><mtr><mtd columnalign="left"><mo>⋮</mo></mtd></mtr><mtr><mtd columnalign="left"><mi>X</mi><msub><mrow></mrow><mrow><mi>p</mi><msub><mrow></mrow><mi>i</mi></msub><mn>1</mn></mrow></msub><mo>,</mo><mi>X</mi><msub><mrow></mrow><mrow><mi>p</mi><msub><mrow></mrow><mi>i</mi></msub><mn>2</mn></mrow></msub><mo>,</mo><mo>⋯</mo><mo>,</mo><mi>X</mi><msub><mrow></mrow><mrow><mi>p</mi><msub><mrow></mrow><mi>i</mi></msub><mi>n</mi></mrow></msub></mtd></mtr></mtable></mrow><mo>]</mo></mrow></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="17">为了使每个数据集对合并矩阵的总变异贡献相等,需要对其标准化,首先对其中心化去除不同数据集的基线差异。</p>
                </div>
                <div class="p1">
                    <p id="18"><mathml id="48"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>X</mi><msubsup><mrow></mrow><mrow><mi>m</mi><mi>n</mi></mrow><mrow><mi>c</mi><mi>e</mi><mi>n</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>e</mi><mi>d</mi></mrow></msubsup><mo>=</mo><mi>X</mi><msub><mrow></mrow><mrow><mi>m</mi><mi>n</mi></mrow></msub><mo>-</mo><mover accent="true"><mi>X</mi><mo>¯</mo></mover><msub><mrow></mrow><mi>m</mi></msub><mo>,</mo><mi>m</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mo>⋯</mo><mo>,</mo><mi>p</mi></mrow></math></mathml>;<i>n</i>=1,2,…,<i>n</i>      (2)</p>
                </div>
                <div class="p1">
                    <p id="19">然后再将数据矩阵归一化,使合并矩阵中每个不同组学的数据集的总变异贡献相等,对此可以应用Frobenius范数:</p>
                </div>
                <div class="p1">
                    <p id="20">‖<i>A</i>‖<sup>2</sup>=∑<sub><i>i</i></sub><sub>,</sub><sub><i>j</i></sub><i>a</i><mathml id="49"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mn>2</mn></msubsup></mrow></math></mathml>      (3)</p>
                </div>
                <div class="p1">
                    <p id="21">归一化的数据矩阵为</p>
                </div>
                <div class="p1">
                    <p id="22" class="code-formula">
                        <mathml id="22"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>X</mi><msubsup><mrow></mrow><mi>i</mi><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup><mo>=</mo><mfrac><mrow><mi>X</mi><msubsup><mrow></mrow><mi>i</mi><mrow><mi>c</mi><mi>e</mi><mi>n</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>e</mi><mi>d</mi></mrow></msubsup></mrow><mrow><mo stretchy="false">∥</mo><mi>X</mi><msubsup><mrow></mrow><mi>i</mi><mrow><mi>c</mi><mi>e</mi><mi>n</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>e</mi><mi>d</mi></mrow></msubsup><mo stretchy="false">∥</mo></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>4</mn><mo stretchy="false">)</mo></mtd></mtr><mtr><mtd><mi>X</mi><msup><mrow></mrow><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msup><mo>=</mo><mrow><mrow><mo>[</mo><mrow><mtable><mtr><mtd columnalign="left"><mi>X</mi><msubsup><mrow></mrow><mn>1</mn><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup></mtd></mtr><mtr><mtd columnalign="left"><mo>⋮</mo></mtd></mtr><mtr><mtd columnalign="left"><mi>X</mi><msubsup><mrow></mrow><mi>i</mi><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup></mtd></mtr></mtable></mrow><mo>]</mo></mrow></mrow><msub><mrow></mrow><mrow><mi>p</mi><mo>×</mo><mi>n</mi></mrow></msub><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>5</mn><mo stretchy="false">)</mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="23">对合并矩阵可以做如下奇异值分解</p>
                </div>
                <div class="p1">
                    <p id="24"><i>X</i><sup><i>scaled</i></sup>=<i>U</i><sub>1</sub><i>V</i><mathml id="50"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mn>1</mn><mi>Τ</mi></msubsup></mrow></math></mathml><i>λ</i><sub>1</sub>+<i>U</i><sub>2</sub><i>V</i><mathml id="51"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mn>2</mn><mi>Τ</mi></msubsup></mrow></math></mathml><i>λ</i><sub>2</sub>+…+<i>U</i><sub><i>n</i></sub><i>V</i><mathml id="52"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>n</mi><mi>Τ</mi></msubsup></mrow></math></mathml><i>λ</i><sub><i>n</i></sub></p>
                </div>
                <h4 class="anchor-tag" id="25" name="25">2.确定共同矩阵和独立矩阵的秩</h4>
                <div class="p1">
                    <p id="26">这里的秩就是奇异值的个数,通常希望只取较大值的奇异值,舍去较小的奇异值,这样得到的共同矩阵才可能有意义,对此可以应用置换检验。首先计算矩阵<i>X</i><sup><i>scaled</i></sup>的奇异值,然后置换各组学数据<i>X</i><mathml id="53"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>i</mi><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup></mrow></math></mathml>中的列,置换1000次(综合考虑置信区间与计算能力),计算每次置换后的奇异值,最后给出其分布,以95%上分位数λ<mathml id="54"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mrow><mi>p</mi><mi>e</mi><mi>r</mi><mi>m</mi></mrow></msubsup></mrow></math></mathml>作为阈值,若矩阵X<sup>scaled</sup>的第j个奇异值λ<sub>j</sub>&gt;λ<mathml id="55"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>j</mi><mrow><mi>p</mi><mi>e</mi><mi>r</mi><mi>m</mi></mrow></msubsup></mrow></math></mathml>,则满足这一条件的奇异值的个数即为共同结构矩阵J的秩r。计算独立矩阵A<sub>i</sub>的秩时,置换X<sup>scaled</sup>每一行中的数值,置换1000次,给出其分布,用同样的方法可以得到独立矩阵的秩r<sub>i</sub>。</p>
                </div>
                <div class="p1">
                    <p id="27">标准化之后,对合并矩阵X<sup>scaled</sup>进行秩为r的奇异值分解得到共同矩阵J<sup>(0)</sup>,对(X<mathml id="56"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>i</mi><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup></mrow></math></mathml>-J<mathml id="57"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>i</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></mathml>)进行秩为r<sub>i</sub>的奇异值分解得到独立矩阵A<mathml id="58"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>i</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></mathml>。J为p×n的矩阵,其中第一行至第p<sub>1</sub>行为矩阵J<sub>1</sub>,第p<sub>1</sub>+1行至第p<sub>1</sub>+p<sub>2</sub>行为矩阵J<sub>2</sub>,其余类推。则有</p>
                </div>
                <div class="p1">
                    <p id="28" class="code-formula">
                        <mathml id="28"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mtable><mtr><mtd><mi>X</mi><msubsup><mrow></mrow><mn>1</mn><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup><mo>=</mo><mi>J</mi><msubsup><mrow></mrow><mn>1</mn><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><mi>A</mi><msubsup><mrow></mrow><mn>1</mn><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><mi>ε</mi><msubsup><mrow></mrow><mn>1</mn><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup></mtd></mtr><mtr><mtd><mi>X</mi><msubsup><mrow></mrow><mn>2</mn><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup><mo>=</mo><mi>J</mi><msubsup><mrow></mrow><mn>2</mn><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><mi>A</mi><msubsup><mrow></mrow><mn>2</mn><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><mi>ε</mi><msubsup><mrow></mrow><mn>2</mn><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup></mtd></mtr><mtr><mtd><mo>⋮</mo><mtext> </mtext><mtext> </mtext><mo>⋮</mo></mtd></mtr><mtr><mtd><mi>X</mi><msubsup><mrow></mrow><mi>k</mi><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup><mo>=</mo><mi>J</mi><msubsup><mrow></mrow><mi>k</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><mi>A</mi><msubsup><mrow></mrow><mi>k</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><mi>ε</mi><msubsup><mrow></mrow><mi>k</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></msubsup></mtd></mtr></mtable><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>6</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="29">通过最小化残差平方和来确定共同结构与独立结构,即给定矩阵的秩,使‖R‖<sup>2</sup>最小来确定J和A<sub>1</sub>,…,A<sub>k</sub>。这一步通过一个迭代运算来完成,对于第t次迭代有</p>
                </div>
                <div class="p1">
                    <p id="30" class="code-formula">
                        <mathml id="30"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>R</mi><msup><mrow></mrow><mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msup><mo>=</mo><mrow><mo>[</mo><mrow><mtable><mtr><mtd columnalign="left"><mi>R</mi><msubsup><mrow></mrow><mn>1</mn><mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msubsup></mtd></mtr><mtr><mtd columnalign="left"><mi>R</mi><msubsup><mrow></mrow><mn>2</mn><mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msubsup></mtd></mtr><mtr><mtd columnalign="left"><mspace width="0.25em" /><mo>⋮</mo></mtd></mtr><mtr><mtd columnalign="left"><mi>R</mi><msubsup><mrow></mrow><mi>k</mi><mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msubsup></mtd></mtr></mtable></mrow><mo>]</mo></mrow><mo>=</mo><mrow><mo>[</mo><mrow><mtable><mtr><mtd columnalign="left"><mi>X</mi><msubsup><mrow></mrow><mn>1</mn><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup><mo>-</mo><mi>J</mi><msubsup><mrow></mrow><mn>1</mn><mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>-</mo><mi>A</mi><msubsup><mrow></mrow><mn>1</mn><mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msubsup></mtd></mtr><mtr><mtd columnalign="left"><mi>X</mi><msubsup><mrow></mrow><mn>2</mn><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup><mo>-</mo><mi>J</mi><msubsup><mrow></mrow><mn>2</mn><mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>-</mo><mi>A</mi><msubsup><mrow></mrow><mn>2</mn><mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msubsup></mtd></mtr><mtr><mtd columnalign="left"><mtext> </mtext><mtext> </mtext><mtext> </mtext><mo>⋮</mo></mtd></mtr><mtr><mtd columnalign="left"><mi>X</mi><msubsup><mrow></mrow><mi>k</mi><mrow><mi>s</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>e</mi><mi>d</mi></mrow></msubsup><mo>-</mo><mi>J</mi><msubsup><mrow></mrow><mi>k</mi><mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>-</mo><mi>A</mi><msubsup><mrow></mrow><mi>k</mi><mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msubsup></mtd></mtr></mtable></mrow><mo>]</mo></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>7</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="31">其中R是一个p×n的残差矩阵,重复以上步骤直到找到最合适的J和A<sub>1</sub>,…,A<sub>k</sub>使‖R‖<sup>2</sup>最小。</p>
                </div>
                <div class="p1">
                    <p id="32">上述<i>JIVE</i>分解模型中要求共同结构矩阵J与独立结构矩阵A<sub>i</sub>的行是正交的,即当i=1,…,k时JA<sup>T</sup><sub>i</sub>=0<sub>p×pi</sub>以此保证共同结构所代表的不同组学数据间共享的数据信息与独立结构所代表的数据信息互不相关。与主成分分析(<i>principal component analysis</i>,<i>PCA</i>)相似,可以根据样本在各成分中的得分与因子载荷做变量筛选和其他分析。实现<i>JIVE</i>方法的<i>R</i>软件包为<i>r</i>.<i>jive</i><citation id="60" type="reference"><link href="5" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>。</p>
                </div>
                <h3 id="33" name="33" class="anchor-tag">实例分析</h3>
                <div class="p1">
                    <p id="34">研究背景:2011年TCGA团队对TCGA中489例卵巢癌mRNA数据进行聚类分析,结合专业得到了四个分型,即增殖型、间叶细胞型、分化型和免疫反应型,该结果发表在Nature杂志上<citation id="61" type="reference"><link href="7" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>。本例主要分析miRNA是否能够做同样的分型及是否能够说明其在分型上与mRNA具有潜在的调控作用。</p>
                </div>
                <div class="p1">
                    <p id="35">使用TCGA卵巢癌基因表达数据mRNA与miRNA数据(经标准化),进行样本ID匹配后,得到样本量为408的样本。其中mRNA基因数目为20113个,miRNA的数目为680个,共两个数据矩阵。首先对数据进行预处理,去除标准差较小的mRNA,对mRNA设定的阈值为<i>SD</i>≤1.5;与mRNA相比,miRNA具有更强的时空表达异质性,即在一些位置和时间不表达,因此剔除零表达个数超过样本量一半的miRNA。预处理后得到mRNA表达矩阵为302行,408列,miR-NA表达矩阵为351行,408列。将两个数据矩阵合并后进行JIVE方法分析,得到共同结构矩阵秩为3,mRNA独立结构矩阵秩为34,miRNA独立结构矩阵秩为23。共同结构的解释方差及置信区间分别为1.64×10<sup>-9</sup>(1.48×10<sup>-9</sup>,1.78×10<sup>-9</sup>)、2.35×10<sup>-9</sup>(2.16×10<sup>-9</sup>,2.52×10<sup>-9</sup>),独立结构的解释方差及置信区间分别为4.62×10<sup>-9</sup>(4.43×10<sup>-9</sup>,4.78×10<sup>-9</sup>)、3.18×10<sup>-9</sup>(3.01×10<sup>-9</sup>,3.35×10<sup>-9</sup>)。JIVE分解可以得到的三个矩阵,解释方差占比如图1。</p>
                </div>
                <div class="area_img" id="36">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201905040_036.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 JIVE分解得到三部分解释方差比" src="Detail/GetImg?filename=images/ZGWT201905040_036.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图1</b> JIVE分解得到三部分解释方差比  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201905040_036.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="37">分别对数据矩阵<i>X</i><sub>2</sub>和共同结构矩阵<i>J</i><sub>2</sub>做主成分分析,样本前三个主成分得分图如图2。结果显示,对于miRNA数据,原始矩阵<i>X</i><sub>2</sub>主成分解释方差占比分别为10.30%、8.62%、6.86%,散点图中各点混在一起,而其共同结构矩阵<i>J</i><sub>2</sub>则呈现出与mRNA表达分类相同的分类趋势(主成分解释方差占比分别为55.93%、27.11%、16.96%),说明使用JIVE分解方法可以看到两组潜在的调控关系。</p>
                </div>
                <div class="area_img" id="38">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201905040_038.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 mRNA及JIVE分解前后miRNA表达数据PCA分类图对比" src="Detail/GetImg?filename=images/ZGWT201905040_038.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图2</b> mRNA及JIVE分解前后miRNA表达数据PCA分类图对比  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201905040_038.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="39">为了显示其综合分类效果,将JIVE分解得到的共同结构矩阵<i>J</i><sub>1</sub>和<i>J</i><sub>2</sub>合并在一起做主成分分析,使用前两个主成分分别给出其样本得分的密度分布图,参见图3和图4,变量的因子载荷见表1(mRNA、miRNA按照绝对值排序各前5位)。</p>
                </div>
                <div class="p1">
                    <p id="40">结果显示,第一主成分的样本得分可以区分增殖型与间质型样本,第二主成分的样本得分可将增殖型、间质型与另外两种分子分型区分开。计算各共同成分的变量载荷,第一主成分中因子载荷绝对值由大到小前五位的基因分别为SFRP2、POSTN、DLK1、LUM、MMP11,其中基因SFRP2的甲基化与卵巢癌患者复发和生存率相关,POSTN基因高表达的卵巢癌患者总生存期和一线化疗后的无进展生存期明显更短,DLK1在浆液性卵巢癌和浆液性交界性癌中均高表达,第一主成分中因子载荷绝对值由大到小前五位的miRNA分别为miR-508、miR-202、miR-509,其中miR-508、miR-509的高表达与卵巢癌患者更长的生存期有关,第二主成分因子载荷较大的变量中,IGF2的高表达与晚期卵巢癌的发生发展和化疗药物耐药性有关,CLDN6在卵巢癌组织中表达上调,COL3A1是能够独立预测卵巢癌铂基化疗耐药性的基因,miR-483-3p、miR-370与卵巢癌的耐药机制及化疗敏感性相关。</p>
                </div>
                <div class="area_img" id="41">
                    <p class="img_tit"><b>表1</b> 变量在前两个主成分中的因子载荷 <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="41" border="1"><tr><td><br />变量</td><td>第一主成分因子载荷</td><td>变量</td><td>第二主成分因子载荷</td></tr><tr><td><br />SFRP2</td><td>0.1206</td><td>INS-IGF2</td><td>0.1522</td></tr><tr><td><br />POSTN</td><td>0.1184</td><td>IGF2</td><td>0.1512</td></tr><tr><td><br />COLEC11</td><td>-0.1039</td><td>CLDN6</td><td>0.1055</td></tr><tr><td><br />DLK1</td><td>-0.1031</td><td>MFAP2</td><td>0.0927</td></tr><tr><td><br />LUM</td><td>0.0934</td><td>COL3A1</td><td>0.0895</td></tr><tr><td><br />miR-508</td><td>-0.2965</td><td>miR-483</td><td>0.1957</td></tr><tr><td><br />miR-202</td><td>-0.2725</td><td>miR-431</td><td>0.1330</td></tr><tr><td><br />miR-509-3</td><td>-0.2547</td><td>miR-485</td><td>0.1234</td></tr><tr><td><br />miR-509-1</td><td>-0.2508</td><td>miR-432</td><td>0.1228</td></tr><tr><td><br />miR-509-2</td><td>-0.2492</td><td>miR-370</td><td>0.1111</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="area_img" id="42">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201905040_042.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 第一主成分的样本得分概率密度分布图" src="Detail/GetImg?filename=images/ZGWT201905040_042.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图3</b> 第一主成分的样本得分概率密度分布图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201905040_042.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="43">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201905040_043.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 第二主成分的样本得分概率密度分布图" src="Detail/GetImg?filename=images/ZGWT201905040_043.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图4</b> 第二主成分的样本得分概率密度分布图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201905040_043.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h3 id="44" name="44" class="anchor-tag">讨  论</h3>
                <div class="p1">
                    <p id="45">分子分型在临床实际中具有重要意义,如不同分型对特定的药物敏感性不同或预后不同<citation id="62" type="reference"><link href="9" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>。本研究根据由mRNA得到的4个卵巢癌分子分型引入了JIVE方法分析卵巢癌miRNA数据,提取miRNA与mRNA的共同结构,得到了共同结构的分类特征,揭示了miRNA在分子分型上是否与mRNA存在可能的调控关系。</p>
                </div>
                <div class="p1">
                    <p id="46">JIVE方法使用的前提是不同组学数据间有足够的共性可以提取。一般来说,共同结构矩阵的秩决定了不同组学之间调控关系的复杂程度,秩越大,表明数据可能具有更复杂的潜在分型结构。而对于提取的共性结构如何解释则需要结合生物学背景知识。如,本文实例中提取的共性结构表示卵巢癌miRNA的表达与分子分型之间存在相关关系。需要注意,计算秩的方法是置换检验,因此当样本量过小时,亦不容易得到显著的秩,亦难以获得组学之间的关联信息,其他计算秩的方法仍需进一步研究。另外,JIVE对缺失数据不稳健,应先选择合适的方法进行填补再分析。</p>
                </div>
                <div class="p1">
                    <p id="47">奇异值分解的原理与PCA相似,只有在数据中的方差贡献足够大时,其特征才能够被提取出来,因此需要舍弃那些方差贡献小的变量,尽量减少对分析的干扰,以获得更好的分析结果。本研究使用的数据粗筛的方法是“去除<i>SD</i>&lt;1.5的mRNA变量和“0”表达个数超过总样本量的一半的miRNA变量”,实际中也可以使用其他的阈值或方法。用敏感度分析的思路去考察粗筛方法对结果的影响,结果差别不大。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="3">
                            <a id="bibliography_1" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJST&amp;filename=SJSTD50AD57236D42A6FAB7ACD73B73A11C5&amp;v=MDI4NDF0Rmh6THErd0swPU5pZlllc2U5SHFDNHFvaEhaKzE3Q0g1SXlXQmltRGdNT3d2bHIyQXllc09WUk1tYUNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkJyTFUwNQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[1]</b> Lock EF,Hoadley KA,Marron JS,et al.Joint and individual variation explained(JIVE)for integrated analysis of multiple data types.Annals of Applied Statistics,2013,7(1):523-542.
                            </a>
                        </p>
                        <p id="5">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=JIVE for exploration of multi-source molecular data">

                                <b>[2]</b> O′Connell MJ,Lock EF,R.JIVE for exploration of multi-source molecular data.Bioinformatics,2016,32(18):2877-2879.
                            </a>
                        </p>
                        <p id="7">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Integrated genomic analyses of ovarian carcinoma">

                                <b>[3]</b> Network TCGA,Integrated genomic analyses of ovarian carcinoma.Nature,2011,474(7353):609-615.
                            </a>
                        </p>
                        <p id="9">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Molecular classification of high grade endometrioid and clear cell ovarian cancer using TCGA gene expression signatures">

                                <b>[4]</b> Winterhoff B,Hamidi H,Wang C,et al.,Molecular classification of high grade endometrioid and clear cell ovarian cancer using TCGA gene expression signatures.Gynecologic Oncology,2016,141(1):95-100.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="ZGWT201905040" />
        <input id="dpi" type="hidden" value="96" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>


    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201905040&amp;v=MTAyNTBabkZ5RG5Vci9OUHlyY2VyRzRIOWpNcW85QlpJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnU=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="0" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=842_VlRz_g1CMCLi-M0twVw-4EOFG_noWyi_OsMnrAg1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>


    <link href="/kxreader/Content/css/LeftDetail?v=vAscMyvIPP9NePnbGPkqJ0A5tHOvnzl65tGuRDsf9xg1" rel="stylesheet"/>

</body>
</html>

