<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=NOOK-mvTMnxl11s6oPYV0boPXN3iP4N7Rc-A56nk4KI1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2fKXReader%2fDetail%3fTIMESTAMP%3d637141009755912500%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dZGWT201902003%26RESULT%3d1%26SIGN%3dVn%252fFHyZ2KFNgfx0G7OkW3LMB%252bkw%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201902003&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201902003&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201902003&amp;v=MTgyMjU3cWZadVpuRnk3bVVMdktQeXJjZXJHNEg5ak1yWTlGWjRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#59" data-title="&lt;b&gt;材料与方法&lt;/b&gt; "><b>材料与方法</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#60" data-title="1.方法">1.方法</a></li>
                                                <li><a href="#63" data-title="2.模拟数据">2.模拟数据</a></li>
                                                <li><a href="#65" data-title="3.真实数据">3.真实数据</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#67" data-title="&lt;b&gt;结 果&lt;/b&gt; "><b>结 果</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#68" data-title="1.模拟数据结果">1.模拟数据结果</a></li>
                                                <li><a href="#73" data-title="2.真实数据结果">2.真实数据结果</a></li>
                                                <li><a href="#75" data-title="3.运算时间">3.运算时间</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#77" data-title="&lt;b&gt;讨 论&lt;/b&gt; "><b>讨 论</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#72" data-title="&lt;b&gt;图1&lt;/b&gt; 遗传度0.2 (A) 、0.5 (B) 、0.8 (C) 时三种模拟情景下Lasso (红) 、ENET (绿) 、ssLasso (深蓝) 、BVSR (绿) 、LMM (蓝) 、BSLMM1 (Linear-BSLMM) (黄) 、BSLMM2 (Probit-BSLMM) (紫) 、DPR.VB (浅蓝) 、DPR.MCMC (粉) 之间AUC估计值的比较"><b>图1</b> 遗传度0.2 (A) 、0.5 (B) 、0.8 (C) 时三种模拟情景下Lasso (红) 、ENET (绿) 、ssLasso (深蓝) 、BVSR (绿) 、LMM (蓝) 、BSLMM1 (Linear-BSLMM) (黄) 、BSLMM2 (Probit-BSLMM) (紫) 、DPR.VB (浅蓝) 、DPR.MCMC (粉) 之间AUC估计值的比较</a></li>
                                                <li><a href="#79" data-title="&lt;b&gt;图2&lt;/b&gt; BRCA九种方法预测能力的比较"><b>图2</b> BRCA九种方法预测能力的比较</a></li>
                                                <li><a href="#80" data-title="&lt;b&gt;表1&lt;/b&gt; 真实数据集中不同方法的运算时间 (小时) "><b>表1</b> 真实数据集中不同方法的运算时间 (小时) </a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="4">


                                    <a id="bibliography_1" title=" Mancuso N, Rohland N, Rand KA, et al.The contribution of rare variation to prostate cancer heritability.Nature Genetics, 2016, 48 (1) :30-35." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The contribution of rare variation to prostate cancer heritability">
                                        <b>[1]</b>
                                         Mancuso N, Rohland N, Rand KA, et al.The contribution of rare variation to prostate cancer heritability.Nature Genetics, 2016, 48 (1) :30-35.
                                    </a>
                                </li>
                                <li id="6">


                                    <a id="bibliography_2" title=" Fuchsberger C, Flannick J, Teslovich TM, et al.The genetic architecture of type 2 diabetes.Nature, 2016, 536 (7614) :41." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The genetic architecture of type 2 diabetes">
                                        <b>[2]</b>
                                         Fuchsberger C, Flannick J, Teslovich TM, et al.The genetic architecture of type 2 diabetes.Nature, 2016, 536 (7614) :41.
                                    </a>
                                </li>
                                <li id="8">


                                    <a id="bibliography_3" title=" Warren HR, Evangelou E, Cabrera C P, et al.Genome-wide association analysis identifies novel blood pressure loci and offers biological insights into cardiovascular risk.Nature Genetics, 2017, 49 (10) :403." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Genome-wide associa-tion analysis identifies novel blood pressure loci and offers biological insights into cardiovascular risk">
                                        <b>[3]</b>
                                         Warren HR, Evangelou E, Cabrera C P, et al.Genome-wide association analysis identifies novel blood pressure loci and offers biological insights into cardiovascular risk.Nature Genetics, 2017, 49 (10) :403.
                                    </a>
                                </li>
                                <li id="10">


                                    <a id="bibliography_4" title=" Afshari NA, Jr IR, Morris NJ, et al.Genome-wide association study identifies three novel loci in Fuchs endothelial corneal dystrophy.Nature Communications, 2017, 8:14898." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Genomewide association study identifies three novel loci in Fuchs endothelial corneal dystrophy">
                                        <b>[4]</b>
                                         Afshari NA, Jr IR, Morris NJ, et al.Genome-wide association study identifies three novel loci in Fuchs endothelial corneal dystrophy.Nature Communications, 2017, 8:14898.
                                    </a>
                                </li>
                                <li id="12">


                                    <a id="bibliography_5" title=" Burton PR, Clayton DG, Cardon LR, et al.Genome-wide association study of 14, 000 cases of seven common diseases and 3, 000 shared controls.Nature, 2007, 447 (7145) :661-78." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls">
                                        <b>[5]</b>
                                         Burton PR, Clayton DG, Cardon LR, et al.Genome-wide association study of 14, 000 cases of seven common diseases and 3, 000 shared controls.Nature, 2007, 447 (7145) :661-78.
                                    </a>
                                </li>
                                <li id="14">


                                    <a id="bibliography_6" title=" Hoffmann TJ, Passarelli MN, Graff RE, et al.Genome-wide association study of prostate-specific antigen levels identifies novel loci independent of prostate cancer.Nature Communications, 2017, 8:14248." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Genome-wide association study of prostate-specific antigen levels identifies novel loci independent of prostate cancer">
                                        <b>[6]</b>
                                         Hoffmann TJ, Passarelli MN, Graff RE, et al.Genome-wide association study of prostate-specific antigen levels identifies novel loci independent of prostate cancer.Nature Communications, 2017, 8:14248.
                                    </a>
                                </li>
                                <li id="16">


                                    <a id="bibliography_7" title=" Fritsche LG, Igl W, Bailey JNC, et al.A large genome-wide association study of age-related macular degeneration highlights contributions of rare and common variants.Nature Genetics, 2016, 48 (2) :134-143." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A large genome-wide association study of age-related macular degeneration highlights contributions of rare and common variants">
                                        <b>[7]</b>
                                         Fritsche LG, Igl W, Bailey JNC, et al.A large genome-wide association study of age-related macular degeneration highlights contributions of rare and common variants.Nature Genetics, 2016, 48 (2) :134-143.
                                    </a>
                                </li>
                                <li id="18">


                                    <a id="bibliography_8" title=" Makowsky R, Pajewski NM, Klimentidis YC, et al.Beyond missing heritability:prediction of complex traits.Plos Genetics, 2011, 7 (4) :e1002051." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJLA&amp;filename=SJLA3919605C9E6B24048EE72C1B61F2E6F6&amp;v=MzI1NzFIYjdDeEg5aktyNG8yYlo0SmZuNDl6eElibjBwNlNnemozaFEwRDdEaFE4eVpDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0Rmh3cnU4eEtvPU5pZg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[8]</b>
                                         Makowsky R, Pajewski NM, Klimentidis YC, et al.Beyond missing heritability:prediction of complex traits.Plos Genetics, 2011, 7 (4) :e1002051.
                                    </a>
                                </li>
                                <li id="20">


                                    <a id="bibliography_9" title=" Hayes BJ, Pryce J, Chamberlain AJ, et al.Genetic Architecture of Complex Traits and Accuracy of Genomic Prediction:Coat Colour, Milk-Fat Percentage, and Type in Holstein Cattle as Contrasting Model Traits.Plos Genetics, 2010, 6 (9) :e1001139." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJLA&amp;filename=SJLAD7AA60B3E7EB5DDD97C064D179B6BC58&amp;v=MTExNDdHRWV4NmZubE51MklhN1V4OVRudVdyUlU4QzdUbU5yK1hDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0Rmh3cnU4eEtvPU5pZkhiOGUvYjZES3IvMQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[9]</b>
                                         Hayes BJ, Pryce J, Chamberlain AJ, et al.Genetic Architecture of Complex Traits and Accuracy of Genomic Prediction:Coat Colour, Milk-Fat Percentage, and Type in Holstein Cattle as Contrasting Model Traits.Plos Genetics, 2010, 6 (9) :e1001139.
                                    </a>
                                </li>
                                <li id="22">


                                    <a id="bibliography_10" title=" Chatterjee N, Shi J, Garc&#237;aclosas M.Developing and evaluating polygenic risk prediction models for stratified disease prevention.Nature Reviews Genetics, 2016, 17 (7) :392." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Developing and evaluating polygenic risk prediction models for stratified disease prevention">
                                        <b>[10]</b>
                                         Chatterjee N, Shi J, Garc&#237;aclosas M.Developing and evaluating polygenic risk prediction models for stratified disease prevention.Nature Reviews Genetics, 2016, 17 (7) :392.
                                    </a>
                                </li>
                                <li id="24">


                                    <a id="bibliography_11" title=" Gamazon ER, Wheeler HE, Wheeler HE, et al.A gene-based association method for mapping traits using reference transcriptome data.Nature Genetics, 2015, 47 (9) :1091-1098." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A gene-based association method for mapping traits using reference transcriptome data">
                                        <b>[11]</b>
                                         Gamazon ER, Wheeler HE, Wheeler HE, et al.A gene-based association method for mapping traits using reference transcriptome data.Nature Genetics, 2015, 47 (9) :1091-1098.
                                    </a>
                                </li>
                                <li id="26">


                                    <a id="bibliography_12" title=" Handerson CR.Estimation of genetic parameters.Annals of Mathematical Statistics, 1950, 21:309-310." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Estimation of genetic parameters">
                                        <b>[12]</b>
                                         Handerson CR.Estimation of genetic parameters.Annals of Mathematical Statistics, 1950, 21:309-310.
                                    </a>
                                </li>
                                <li id="28">


                                    <a id="bibliography_13" title=" Speed D and Balding DJ.MultiBLUP:improved SNP-based prediction for complex traits.Genome Research, 2014, 24 (9) :1550-1557." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Multi BLUP:improved SNP-based prediction for complex traits">
                                        <b>[13]</b>
                                         Speed D and Balding DJ.MultiBLUP:improved SNP-based prediction for complex traits.Genome Research, 2014, 24 (9) :1550-1557.
                                    </a>
                                </li>
                                <li id="30">


                                    <a id="bibliography_14" title=" Tang Z, Shen Y, Li Y, et al.Group Spike-and-Slab Lasso Generalized Linear Models for Disease Prediction and Associated Genes Detection by Incorporating Pathway Information.Bioinformatics, 2017, 34 (6) :901-910." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Group Spike-and-Slab Lasso Generalized Linear Models for Disease Prediction and Associated Genes Detection by Incorporating Pathway Information">
                                        <b>[14]</b>
                                         Tang Z, Shen Y, Li Y, et al.Group Spike-and-Slab Lasso Generalized Linear Models for Disease Prediction and Associated Genes Detection by Incorporating Pathway Information.Bioinformatics, 2017, 34 (6) :901-910.
                                    </a>
                                </li>
                                <li id="32">


                                    <a id="bibliography_15" title=" Habier D, Fernando RL, Kizilkaya K, et al.Extension of the bayesian alphabet for genomic selection.BMC Bioinformatics, 2011, 12 (1) :186." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Extension of the bayesian alphabet for genomic selection">
                                        <b>[15]</b>
                                         Habier D, Fernando RL, Kizilkaya K, et al.Extension of the bayesian alphabet for genomic selection.BMC Bioinformatics, 2011, 12 (1) :186.
                                    </a>
                                </li>
                                <li id="34">


                                    <a id="bibliography_16" title=" Tibshirani R.Regression Shrinkage and Selection via the Lasso.Journal of the Royal Statistical Society, 2011, 73 (3) :273-282." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Regression Shrinkage and Selection via the Lasso">
                                        <b>[16]</b>
                                         Tibshirani R.Regression Shrinkage and Selection via the Lasso.Journal of the Royal Statistical Society, 2011, 73 (3) :273-282.
                                    </a>
                                </li>
                                <li id="36">


                                    <a id="bibliography_17" title=" Hui Z, Hastie T.Addendum:Regularization and variable selection via the elastic net.Journal of the Royal Statistical Society, 2005, 67 (2) :301-320." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD00001256895&amp;v=Mjg5MjlpZmNhck80SHRITnJZcERiT0lLWTNrNXpCZGg0ajk5U1hxUnJ4b3hjTUg3UjdxZForWnVGaUhsVmJ6TEpGdz1O&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[17]</b>
                                         Hui Z, Hastie T.Addendum:Regularization and variable selection via the elastic net.Journal of the Royal Statistical Society, 2005, 67 (2) :301-320.
                                    </a>
                                </li>
                                <li id="38">


                                    <a id="bibliography_18" title=" Zhou X, Carbonetto P, Stephens M.Polygenic Modeling with Bayesian Sparse Linear Mixed Models.Plos Genetics, 2013, 9 (2) ::e1003264." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJLA&amp;filename=SJLAC493B92DA2C9D54E67A6F0C3412D57CB&amp;v=MTA4NDh3cnU4eEtvPU5pZkhiOEM4RjlLK3BvMHhGZWw4QlFnOHkyTVY3VTU3UG4rUnJ4WTBlOGFSUXNudENPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkJyTFUwNXRGaA==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[18]</b>
                                         Zhou X, Carbonetto P, Stephens M.Polygenic Modeling with Bayesian Sparse Linear Mixed Models.Plos Genetics, 2013, 9 (2) ::e1003264.
                                    </a>
                                </li>
                                <li id="40">


                                    <a id="bibliography_19" title=" Zeng P, Zhou X.Non-parametric genetic prediction of complex traits with latent Dirichlet process regression models.Nature Communications, 2017, 8 (1) :456." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Non-parametric genetic prediction of complex traits with latent Dirichlet process regression models">
                                        <b>[19]</b>
                                         Zeng P, Zhou X.Non-parametric genetic prediction of complex traits with latent Dirichlet process regression models.Nature Communications, 2017, 8 (1) :456.
                                    </a>
                                </li>
                                <li id="42">


                                    <a id="bibliography_20" title=" Tang Z, Shen Y, Zhang X, et al.The Spike-and-Slab Lasso Generalized Linear Models for Prediction and Associated Genes Detection.Genetics, 2016, 205 (1) :77-88." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SPQD&amp;filename=SPQD9469FB1559ADA91A1D5F7008E35526B8&amp;v=MjYxNjRwR2MyZkxlV1E4aVhDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0Rmh3cnU4eEtvPU5qM2FhcnE4R05pNjNZNUFZZUorZUEwd3ptY1Nuam9MVDMvaQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[20]</b>
                                         Tang Z, Shen Y, Zhang X, et al.The Spike-and-Slab Lasso Generalized Linear Models for Prediction and Associated Genes Detection.Genetics, 2016, 205 (1) :77-88.
                                    </a>
                                </li>
                                <li id="44">


                                    <a id="bibliography_21" title=" Guan Y, Stephens M.Bayesian variable selection regression for genome-wide association studies and other large-scale problems.Annals of Applied Statistics, 2011, 5 (3) :1780-1815." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJST&amp;filename=SJST15012900203942&amp;v=MTEwOTJIdERPcG85Rlp1c01CWGc3b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadEZpbmxVcmJJSmwwWGJoQT1OaWZZZXJLOQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[21]</b>
                                         Guan Y, Stephens M.Bayesian variable selection regression for genome-wide association studies and other large-scale problems.Annals of Applied Statistics, 2011, 5 (3) :1780-1815.
                                    </a>
                                </li>
                                <li id="46">


                                    <a id="bibliography_22" title=" Ping Z, Xiang Z, Huang S.Prediction of gene expression with cis-SNPs using mixed models and regularization methods.Bmc Genomics, 2017, 18 (1) :368." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDBE08C68C54CE7FE94C6EB609450A88CD&amp;v=MDgzOTg4WG1Ua0lPbm5pcFJZd2VjT2NUY25yQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQnJMVTA1dEZod3J1OHhLbz1OajdCYXNITkh0bS9xWWMyWWU5OGVYdFB1aA==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[22]</b>
                                         Ping Z, Xiang Z, Huang S.Prediction of gene expression with cis-SNPs using mixed models and regularization methods.Bmc Genomics, 2017, 18 (1) :368.
                                    </a>
                                </li>
                                <li id="48">


                                    <a id="bibliography_23" title=" Moser G, Sang HL, Hayes BJ, et al.Simultaneous Discovery, Estimation and Prediction Analysis of Complex Traits Using a Bayesian Mixture Model.Plos Genetics, 2015, 11 (4) :e1004969." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJLA&amp;filename=SJLA7D965CB316466A212BDAC56956DAA800&amp;v=MDk2NTk9TmlmSGI3VE1GOWZKM1AxR1plMExDbnBJelJjUm1Fc01PM3JrcFJjekRjUGxUYnFmQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQnJMVTA1dEZod3J1OHhLbw==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[23]</b>
                                         Moser G, Sang HL, Hayes BJ, et al.Simultaneous Discovery, Estimation and Prediction Analysis of Complex Traits Using a Bayesian Mixture Model.Plos Genetics, 2015, 11 (4) :e1004969.
                                    </a>
                                </li>
                                <li id="50">


                                    <a id="bibliography_24" title=" Mandt S, Wenzel F, Nakajima S, et al.Sparse probit linear mixed model.Machine Learning, 2015 (7298) :1-22." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Sparse probit linear mixed model">
                                        <b>[24]</b>
                                         Mandt S, Wenzel F, Nakajima S, et al.Sparse probit linear mixed model.Machine Learning, 2015 (7298) :1-22.
                                    </a>
                                </li>
                                <li id="52">


                                    <a id="bibliography_25" title=" Kanehisa M and Goto S.KEGG:kyoto encyclopedia of genes and genomes.Nucleic acids research, 2000, 28 (1) :27-30." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=KEGG: kyoto encyclopedia of genes and genomes">
                                        <b>[25]</b>
                                         Kanehisa M and Goto S.KEGG:kyoto encyclopedia of genes and genomes.Nucleic acids research, 2000, 28 (1) :27-30.
                                    </a>
                                </li>
                                <li id="54">


                                    <a id="bibliography_26" title=" Tang Z, Shen Y, Li Y, et al.Group spike-and-slab lasso generalized linear models for disease prediction and associated genes detection by incorporating pathway information.Bioinformatics, 2017, 34 (6) :901-910." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Group spike-and-slab lasso generalized linear models for disease prediction and associated genes detection by incorporating pathway information">
                                        <b>[26]</b>
                                         Tang Z, Shen Y, Li Y, et al.Group spike-and-slab lasso generalized linear models for disease prediction and associated genes detection by incorporating pathway information.Bioinformatics, 2017, 34 (6) :901-910.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=ZGWT" target="_blank">中国卫生统计</a>
                2019,36(02),171-175             </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm">利用全基因组表达数据对比评价九种遗传预测模型方法</span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E4%BD%99%E6%98%9F%E7%9A%93&amp;code=35883473&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">余星皓</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%88%98%E5%85%B5&amp;code=37844136&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">刘兵</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%9B%BE%E5%B9%B3&amp;code=36315958&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">曾平</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E9%BB%84%E6%B0%B4%E5%B9%B3&amp;code=36315956&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">黄水平</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%BE%90%E5%B7%9E%E5%8C%BB%E7%A7%91%E5%A4%A7%E5%AD%A6%E5%85%AC%E5%85%B1%E5%8D%AB%E7%94%9F%E5%AD%A6%E9%99%A2%E6%B5%81%E8%A1%8C%E7%97%85%E4%B8%8E%E5%8D%AB%E7%94%9F%E7%BB%9F%E8%AE%A1%E5%AD%A6%E7%B3%BB&amp;code=1699588&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">徐州医科大学公共卫生学院流行病与卫生统计学系</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p><b>目的</b> 研究稀疏模型 (Lasso、ENET、ssLasso、贝叶斯变量选择回归模型 (BVSR) ) 与多基因模型 (线性混合模型 (LMM) 、贝叶斯稀疏线性混合模型 (BSLMM) 、狄利克雷回归模型 (DPR) ) 等九种遗传预测方法在全基因组表达数据中对复杂疾病的遗传预测表现。<b>方法</b> 通过模拟研究评价每种方法在不同的较大基因稀疏程度和不同的遗传度下的预测精度, 利用乳腺癌数据进行表型预测。<b>结果</b> 模拟结果显示预测方法在满足各自的模型假设时表现结果最好。在相同模拟假设情况下, 随着遗传度的增高, 模型的预测准确性也逐渐增高。BVSR运算速度和BSLMM运算速度相似, 由于迭代次数的影响, BVSR与BSLMM的运算速度低于LMM。实际的乳腺癌数据显示BSLMM和DPR的预测精度优于其他方法。<b>结论</b> BSLMM和DPR在不同模拟情形下和真实数据中均表现出稳健的预测能力, 值得在实际应用中推荐。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E9%81%97%E4%BC%A0%E9%A2%84%E6%B5%8B&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">遗传预测;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%A4%9A%E5%9F%BA%E5%9B%A0%E6%A8%A1%E5%9E%8B&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">多基因模型;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%A8%80%E7%96%8F%E6%A8%A1%E5%9E%8B&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">稀疏模型;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%9F%BA%E5%9B%A0%E8%A1%A8%E8%BE%BE&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">基因表达;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    *曾平, E-mail:zpstat@xzhmu.edu.cn;;
                                </span>
                                <span>
                                    *黄水平, E-mail:hsp@xzhmu.edu.cn;
                                </span>
                    </p>
                
                    <p>

                            <b>基金：</b>
                                                        <span>教育部人文社会科学研究青年基金项目资助 (18YJC910002);</span>
                                <span>江苏省自然科学基金资助项目 (BK20181472);</span>
                                <span>江苏省高校青蓝工程优秀青年骨干教师项目;</span>
                                <span>中国博士后科学基金资助项目 (2018M630607);</span>
                                <span>徐州医科大学博士后科学基金资助项目;</span>
                                <span>国家自然科学基金青年科学基金资助项目 (81402765);</span>
                                <span>国家统计局全国统计科学研究项目 (2014LY112);</span>
                    </p>
            </div>
                    <h1><b>Comparison of Nine Genetic Prediction Models Using Genome-wide Expression Data</b></h1>
                    <h2>
                    <span>Yu Xinghao</span>
                    <span>Liu Bing</span>
                    <span>Zeng Ping</span>
            </h2>
                    <h2>
                    <span>Department of Epidemiology and Health Statistics, School of Public Health, Xuzhou Medical University</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p><b>Objective</b> To explore the performance of genetic prediction for complex traits with nine methods, including sparse models (e.g.Lasso, ENET, ssLasso and BVSR) and polygenic models (e.g.LMM, BSLMM and DPR) .<b>Methods</b> We evaluated and compared these prediction methods via simulated data obtained under various genetic architectures and real life gene expression dataset of breast cancer.<b>Results</b> The simulation results show that these prediction methods perform best when the own model assumptions are satisfied.For example, scenario ii satisfies the model assumption of BVSR, so that BVSR has the highest prediction accuracy.Simulations also show that the prediction accuracy of these models increase as the increase of PVE.The computation time of BVSR is similar to BSLMM, but the computation speed of BVSR and BSLMM is slower than the LMM due to iterations needed by Bayesian models.In the real data of breast cancer, it shows that BSLMM and DPR are more accurate than other compared methods.<b>Conclusion</b> BSLMM and DPR are two prediction approaches with robust and high accuracy compared with other existing methods, and can be recommended in practical application.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Genetic%20prediction&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Genetic prediction;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Polygenic%20model&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Polygenic model;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Sparse%20model&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Sparse model;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Gene%20expression&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Gene expression;</a>
                </p>
            </div>
        

        <!--brief start-->
                        <div class="p1">
                    <p id="56">全基因组关联研究 (genome-wide association study, GWAS) 已经发现了上万个单核苷酸多态性 (single-nucleotide polymorphism, SNP) 位点与数百种复杂疾病存在关联, 为表型变异的遗传基础提供了前所未有的视角<citation id="86" type="reference"><link href="4" rel="bibliography" /><link href="6" rel="bibliography" /><link href="8" rel="bibliography" /><link href="10" rel="bibliography" /><link href="12" rel="bibliography" /><link href="14" rel="bibliography" /><link href="16" rel="bibliography" /><sup>[<a class="sup">1</a>,<a class="sup">2</a>,<a class="sup">3</a>,<a class="sup">4</a>,<a class="sup">5</a>,<a class="sup">6</a>,<a class="sup">7</a>]</sup></citation>。GWAS的成功也极大促进了利用遗传信息 (除环境和生活方式信息外) 进行复杂疾病风险预测评估的研究<citation id="82" type="reference"><link href="18" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>。在动物或植物中, 具有遗传标记的准确表型预测可以帮助选择理想的育种个体, 从而提高育种计划的有效性<citation id="83" type="reference"><link href="20" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>;在人群中, 利用遗传标记的准确表型预测有利于复杂疾病的早期预防和干预、促进开发个性化药物, 从而制定治疗方案和预测疗效<citation id="84" type="reference"><link href="22" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>。此外, 在跨组学数据分析中, 表型预测也被认为是将功能基因组测序研究与GWAS相结合的关键步骤, 可以在GWAS中构建更高效和具有可解释性的基因集测试, 通过设置SNP权重, 将其从预测模型中推断出来, 并用于表达数量性状位点映射研究<citation id="85" type="reference"><link href="24" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>。</p>
                </div>
                <div class="p1">
                    <p id="57">研究者们发明了许多新的表型预测模型方法以利用全基因组遗传位点信息提高预测精度。表型预测模型本质上是对基因效应大小分布的假设不同, 例如Henderson等人<citation id="87" type="reference"><link href="26" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>提出的线性混合模型 (liner mixed model, LMM) , Speed等人<citation id="88" type="reference"><link href="28" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>提出的最佳无偏估计模型 (multiple best linear unbiased prediction, MultiBLUP) , Tang等人<citation id="89" type="reference"><link href="30" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>提出的Bayes Lasso模型。上述方法多属于参数模型, 所假设的遗传效应分布是高度参数化的、由少数几个参数决定, 其局限在于仅对特定满足模型假设的疾病预测效果良好, 对其它不满足假设的疾病预测精度不佳。理论上, 模型预测精度取决于遗传效应先验与真实分布的接近程度。然而, 真实的遗传结构往往未知, 由于疾病的不同, 在遗传度、关联位点个数、最小等位基因频率和效应大小等方面存在诸多差别。另外, 这些方法先前主要利用全基因组SNP位点信息进行复杂疾病的预测;而越来越多的研究表明基因表达在复杂疾病的发生、发展和转归中起着十分重要的作用, 是SNP对复杂疾病起作用的重要分子中介。因此, 通过全基因组表达数据预测复杂疾病具有重要的科学意义。通过基因表达预测复杂疾病与利用SNP预测复杂疾病的差别在于, 基因表达数据的变量较少, 效应可能更强, 作为一个整体的功能单位, 基因相对SNP更具有结果的可解释性。那些通过全基因组SNP位点获得复杂疾病预测结论未必适合基因表达预测。</p>
                </div>
                <div class="p1">
                    <p id="58">本文主要对九种遗传预测模型方法进行比较, 这些方法大致可分为两类:稀疏模型和多基因模型。前者包括Lasso、elastic net (ENET) 、spike-and-slab Lasso GLMs (ssLasso) 、贝叶斯变量选择回归模型 (Bayesian variable selection regression, BVSR) ;后者包括线性混合模型 (LMM) 、贝叶斯稀疏线性混合模型 (Bayesian sparse linear mixed model, BSLMM) 、狄利克雷回归模型 (Dirichlet process regression, DPR) 等。本文通过癌症患者基因表达数据模拟各种复杂疾病可能的遗传结构, 比较上述方法在不同情况下对于复杂疾病的预测能力, 进一步通过比较各方法在真实肿瘤数据中的预测能力评价这些方法在基因表达数据中的表型预测精度。</p>
                </div>
                <h3 id="59" name="59" class="anchor-tag"><b>材料与方法</b></h3>
                <h4 class="anchor-tag" id="60" name="60">1.方法</h4>
                <div class="p1">
                    <p id="61">比较九种模型 (稀疏模型:BVSR<citation id="90" type="reference"><link href="32" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation>、Lasso<citation id="91" type="reference"><link href="34" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation>、ENET<citation id="92" type="reference"><link href="36" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>和ssLasso, 多基因模型:LMM、Linear-BSLMM、Probit-BSLMM<citation id="93" type="reference"><link href="38" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>、DPR.VB和DPR.MCMC<citation id="94" type="reference"><link href="40" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>) 在遗传预测中的能力, 这些模型方法被广泛运用于人类复杂疾病的表型预测以及动植物繁殖的基因选择中。具体而言, BVSR假设较少部分的基因变异对复杂疾病或性状产生效应, 单个基因变异对表型的效应较大, 假设总体效应大小符合正态分布和零点分布的混合分布;Lasso是在模型的离均差平方和基础上增加一个绝对值的惩罚项, 使其总和小于等于一个常数, 是一种压缩估计方法;ENET是一种Lasso与岭回归组合后的回归分析, 在绝对值惩罚项基础上同时施加平方和惩罚项;ssLasso假设基因的系数服从混合双指数先验, 根据不同的基因型与表型数据, 产生适合的收缩系数, 并保留系数较大的基因;LMM假设所有的基因对表型均产生效应, 且效应均较小, 同时服从正态分布;BSLMM假设基因对表型的效应大小服从两个正态分布组成的混合分布, 两个正态分布的混合程度决定于不同的混合比例, 需要注意的是, 针对病例对照研究 (假设表型编码为0-1) , BSLMM可直接将0-1当作连续变量通过线性模型处理, 称为Linear-BSLMM, 或将Probit链接函数通过广义线性模型处理, 称为Probit-BSLMM, 图1中分别以BSLMM1和BSLMM2表示;DPR假设基因效应大小服从一个无穷混合正态分布, 属于非参数贝叶斯模型范畴, DPR的参数估计可通过传统的马尔科夫蒙特卡罗估计, 称为DPR.MCMC, 也可通过变分贝叶斯方法估计, 称为DPR.VB。</p>
                </div>
                <div class="p1">
                    <p id="62">LMM、BVSR、Linear-BSLMM和Probit-BSLMM方法采用GEMMA<citation id="95" type="reference"><link href="38" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>软件执行, DPR.VB、DPR.MCMC等方法采用DPR<citation id="96" type="reference"><link href="40" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>软件执行, Lasso、ENET通过glmnet软件包执行, ssLasso通过BhGLM<citation id="97" type="reference"><link href="42" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>软件包执行。</p>
                </div>
                <h4 class="anchor-tag" id="63" name="63">2.模拟数据</h4>
                <div class="p1">
                    <p id="64">为更加接近真实数据, 使用916例乳腺癌患者的基因表达数据, 模拟三种情境下的表型数据: (i) 所有基因均对表型产生效应, 分别服从正态分布、<i>t</i>分布或Laplace分布, 设置遗传度分别为0.2、0.5或0.8; (ii) 仅有小部分基因 (分别为10、50或100) 对表型产生效应, 服从正态分布, 大部分剩余的基因对表型均不产生效应, 遗传度分别为0.2、0.5或0.8; (iii) 所有基因对表型均产生效应, 效应基因分为两部分:小部分效应较大的基因和较大部分效应较小的基因, 两组基因的效应大小都服从正态分布, 两部分占遗传度的比例分别为0.2和0.8;较大效应的基因数量分别为10、50或100;遗传度分别为0.2、0.5或0.8。每种情景分别重复模拟20次, 每一次重复中随机将数据中90%的个体作为训练集, 剩余的10%作为测试集。通过训练集数据估计每种方法的模型参数, 并在测试集数据中比较不同方法的预测能力。通过表型与预测值的曲线下面积 (area under the curve, AUC) 评价预测效果。</p>
                </div>
                <h4 class="anchor-tag" id="65" name="65">3.真实数据</h4>
                <div class="p1">
                    <p id="66">真实数据全部来源于癌症和肿瘤基因图谱 (The Cancer Genome Atlas, TCGA) , 通过加利福尼亚大学基因组浏览器 (UCSC Xena, https://xenabrowser.net/) 下载乳腺癌 (Breast Cancer, BRCA) 数据。原始数据包括1215例患者临床数据和1219例患者的20530个基因表达数据, 首先对两份数据进行合并, 删除重复的患者和男性患者, 同时删除2840个零表达值超过50%的基因, 最终获得916例患者的17690个基因表达数据。在数据分析前, 首先将癌症的基因表达数据分成两部分:90%的患者为训练集数据, 剩余10%的患者为测试集数据, 对数据集执行20次重复。通过<i>AUC</i>评价不同模型的预测能力。</p>
                </div>
                <h3 id="67" name="67" class="anchor-tag"><b>结 果</b></h3>
                <h4 class="anchor-tag" id="68" name="68">1.模拟数据结果</h4>
                <div class="p1">
                    <p id="69">情景i中 (图1-i) 当遗传度为0.2时, 效应大小的分布服从正态分布的情况下, 各模型的预测能力接近。效应大小的分布服从<i>t</i>分布和Laplace分布的情况下, 多基因模型的预测能力均高于稀疏模型, 多基因模型的预测能力大小接近。当遗传度为0.5或0.8时, 三种情况下多基因模型中五种模型的预测准确性相对接近, 优于稀疏模型, BVSR预测能力最低。当遗传度为0.8时, 各统计学模型的预测结果<i>AUC</i>均在0.7左右, 高于遗传度为0.2和0.5时的预测结果。</p>
                </div>
                <div class="p1">
                    <p id="70">情景ii中 (图1-ii) 当遗传度为0.2或0.5时, 稀疏模型的预测能力优于多基因模型, 但是, 随着效应基因数量和遗传度增加, 稀疏模型与多基因模型之间的差异逐渐缩小;当遗传度为0.8:效应基因数量为10的时候, BSLMM和DPR的预测能力较强, 其余多基因模型精度稍低。随着效应基因数量增加, 稀疏模型与多基因模型之间的差异越小, 当效应基因数量为100时, 稀疏模型的预测能力要低于多基因模型。</p>
                </div>
                <div class="p1">
                    <p id="71">情景iii中 (图1-iii) 当遗传度为0.2时, 三种情况下多基因模型的预测效果均高于稀疏模型, 其中BVSR的预测能力最差, linear-BSLMM、probit-BSLMM和DPR.MCMC的预测能力较强且较稳定。当遗传度为0.5时, 多基因模型的预测效果优于稀疏模型, BSLMM和DPR的预测能力较稳定, 且精度较高。当较大效应基因数量增加时, Lasso和ENET的预测能力上升;当遗传度为0.8时, 较大效应基因的数量不同时, 模型之间预测结果相似。多基因模型的预测结果相似。稀疏模型的预测结果相对不稳。在不同情况下, 多基因模型的预测能力均稍高于稀疏模型。各统计学模型的预测结果<i>AUC</i>均在0.75左右, 高于遗传度为0.2或0.5时的预测结果。</p>
                </div>
                <div class="area_img" id="72">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201902003_072.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 遗传度0.2 (A) 、0.5 (B) 、0.8 (C) 时三种模拟情景下Lasso (红) 、ENET (绿) 、ssLasso (深蓝) 、BVSR (绿) 、LMM (蓝) 、BSLMM1 (Linear-BSLMM) (黄) 、BSLMM2 (Probit-BSLMM) (紫) 、DPR.VB (浅蓝) 、DPR.MCMC (粉) 之间AUC估计值的比较" src="Detail/GetImg?filename=images/ZGWT201902003_072.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图1</b> 遗传度0.2 (A) 、0.5 (B) 、0.8 (C) 时三种模拟情景下Lasso (红) 、ENET (绿) 、ssLasso (深蓝) 、BVSR (绿) 、LMM (蓝) 、BSLMM1 (Linear-BSLMM) (黄) 、BSLMM2 (Probit-BSLMM) (紫) 、DPR.VB (浅蓝) 、DPR.MCMC (粉) 之间AUC估计值的比较  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201902003_072.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="73" name="73">2.真实数据结果</h4>
                <div class="p1">
                    <p id="74">真实数据来源于上述的乳腺癌数据, 将术后淋巴结阳性转移作为预测的表型数据 (编码为0-1, 视为连续变量应用线性模型处理) 。经过计算, 数据的真实遗传度为0.15。各方法表型预测结果<i>AUC</i>均在0.61左右 (图2A) , 多基因模型的预测能力均高于稀疏模型。多基因模型中, 五种方法的预测能力几乎无差异 (虽然DPR和BSLMM轻微略高) 。稀疏模型中, ENET的预测能力最强, Lasso与BVSR的预测能力接近, ssLasso的预测能力最低。同时, 采用Logistic回归分析各基因的效应大小, 并绘制相应的分布密度函数, 与正态分布、Laplace分布和自由度为4的<i>t</i>分布相比较 (图2B) 。</p>
                </div>
                <h4 class="anchor-tag" id="75" name="75">3.运算时间</h4>
                <div class="p1">
                    <p id="76">最后对比各种方法的运算速度, 以乳腺癌数据为例。这里需要说明的是, 贝叶斯预测方法的运算时间与所使用的迭代次数有关系, 为了节省时间, 这里只进行了100次迭代。结果显示 (表1) , Lasso、ENET和ssLasso的运算速度最快, 其次为LMM和DPR.VB;BVSR、BSLMM和DPR.MCMC运算速度相近, 且时间均较长。</p>
                </div>
                <h3 id="77" name="77" class="anchor-tag"><b>讨 论</b></h3>
                <div class="p1">
                    <p id="78">本文所提及的多种统计学方法在遗传统计学的多个方面都有较好的应用价值, 例如估计遗传度<citation id="98" type="reference"><link href="38" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>和关联映射<citation id="99" type="reference"><link href="44" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>。本文主要致力于多种统计学方法在表型预测方面的研究。国内外已发表的文献<citation id="100" type="reference"><link href="28" rel="bibliography" /><link href="38" rel="bibliography" /><link href="40" rel="bibliography" /><link href="46" rel="bibliography" /><link href="48" rel="bibliography" /><link href="50" rel="bibliography" /><sup>[<a class="sup">13</a>,<a class="sup">18</a>,<a class="sup">19</a>,<a class="sup">22</a>,<a class="sup">23</a>,<a class="sup">24</a>]</sup></citation>主要研究表型预测方法在GWAS中的比较, 已取得较大成果, 本文通过模拟不同的疾病遗传结构, 比较了两类统计学方法在基因表达数据中的预测能力。模拟数据分析结果显示, 各模型在表型预测上的表现不同。当模拟假设全部基因对表型均产生效应时, 多基因模型的预测能力要高于稀疏模型;当模拟假设只有部分基因对表型产生效应时, 稀疏模型的预测能力要高于多基因模型。当模拟假设小部分基因对表型产生较大的效应, 其余大部分对表型产生较小的效应时, 多基因模型中的BSLMM和DPR预测能力要高于其余模型。在相同模拟假设情况下, 随着遗传度的增高, 模型的预测准确性也逐渐增高。当假设全部基因对表型均产生效应、效应大小服从单个正态分布、遗传度值较小时, 多基因模型与稀疏模型的预测能力相似。当遗传度较大时, 多基因模型的准确性稍高于稀疏模型。当假设部分基因对表型产生效应, 且效应大小服从单个正态分布时, 随着相应基因数量的增加, 稀疏模型的预测能力逐渐由高于多基因模型逐渐变化为与多基因模型预测能力相似, 甚至低于多基因模型。真实数据分析结果显示不同统计学方法在表型预测的表现不同, 各方法表型预测结果表型预测结果<i>AUC</i>均在0.61左右, 多基因模型的预测能力均高于稀疏模型, 其中BSLMM和DPR最优。由模拟研究和真实数据分析结果可以看出, 模型预测的准确性与复杂疾病或性状的真实遗传结构有关, 当复杂疾病的真实遗传结构与模型假设相吻合时, 模型的预测表现较好, 而当复杂疾病的真实遗传结构与模型假设差异较大时, 模型的预测表现较差。这与以往研究的结论相同<citation id="101" type="reference"><link href="28" rel="bibliography" /><link href="38" rel="bibliography" /><sup>[<a class="sup">13</a>,<a class="sup">18</a>]</sup></citation>。因此我们认为多基因的模型假设与复杂疾病的真实遗传结构相符合, 少数较大效应的基因和大多数小效应基因共同对复杂疾病产生影响。</p>
                </div>
                <div class="area_img" id="79">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201902003_079.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 BRCA九种方法预测能力的比较" src="Detail/GetImg?filename=images/ZGWT201902003_079.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图2</b> BRCA九种方法预测能力的比较  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201902003_079.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="80">
                    <p class="img_tit"><b>表1</b> 真实数据集中不同方法的运算时间 (小时)  <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="80" border="1"><tr><td rowspan="2"><br />数据集</td><td rowspan="2">Lasso</td><td rowspan="2">ENET</td><td rowspan="2">ssLasso</td><td rowspan="2">BVSR</td><td rowspan="2">LMM</td><td colspan="2"><br />BSLMM</td><td colspan="2">DPR</td></tr><tr><td><br />Linear</td><td>Probit</td><td>VB</td><td>MCMC</td></tr><tr><td><br />BRCA</td><td>0.9<br />1 (0.02) </td><td>0.91<br /> (0.03) </td><td>1.08<br /> (0.02) </td><td>2.39<br /> (0.01) </td><td>2.12<br /> (0.01) </td><td>2.39<br /> (0.01) </td><td>2.41<br /> (0.08) </td><td>1.88<br /> (0.02) </td><td>2.13<br /> (0.38) </td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note">*:Lasso、ENET、ssLasso的调整参数均为通过100倍交叉验证选择的最佳值, BVSR、BSLMM、DPR.MCMC均为进行了100次MCMC迭代计算, 括号中的值为标准差, 均值和标准差都是基于模拟的20次重复计算。</p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="81">本文研究的基因表达数据相对较小, 三种方法的运算速度均较高, 但是这些方法往往并不适用于较大数据集<citation id="102" type="reference"><link href="46" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>。需要注意的是, Lasso、ENET和ssLasso的运算速度同时受到交叉验证次数的影响, 本研究中只通过100倍交叉验证, 并未进一步研究交叉验证数量对于运算速度的影响。LMM、BVSR和BSLMM使用GEMMA软件进行分析, 同时适用于大规模数据集。BVSR和BSLMM的运算速度相似, 由于迭代次数的影响, BVSR与BSLMM的运算速度低于LMM。有研究报道<citation id="103" type="reference"><link href="38" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>, 当纳入模型的自变量较大时, BSLMM的运算速度将远高于BVSR。DPR通过DPR软件进行分析, 其中DPR.VB与LMM运算时间接近, 而DPR.MCMC运算时间同样受到迭代次数的影响, 远高于其它方法。此外, 计算时间还受到工作环境、计算机语言、以及基因和表型之间的关系等因素的影响。虽然目前对于表型预测方法的研究层出不穷, 但是尚未有一种方法对于不同遗传结构的复杂疾病或性状都具有较高的预测能力且运算时间较快。同时, 遗传度较低时, 所有方法的预测表现均较差。此外, 本文所纳入的方法均为通过基因表达数据进行复杂疾病或性状的预测, 都未能考虑到在遗传数据中非常重要的分组结构。例如:SNP可以通过基因或者功能注释的形式分成不同的组;基因之间存在交互作用和共同作用, 基因可以通过通路信息进行分组, 功能相同的基因可以被划分到一组<citation id="104" type="reference"><link href="52" rel="bibliography" /><sup>[<a class="sup">25</a>]</sup></citation>。这些被忽视的遗传信息 (分组结构、非线性效应和交互作用) 可能会对预测结果造成一定的影响。已经有研究表明, 充分利用多组学的信息可以提高遗传预测的精度<citation id="105" type="reference"><link href="54" rel="bibliography" /><sup>[<a class="sup">26</a>]</sup></citation>, 这也将是接下来的研究重点之一。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="4">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The contribution of rare variation to prostate cancer heritability">

                                <b>[1]</b> Mancuso N, Rohland N, Rand KA, et al.The contribution of rare variation to prostate cancer heritability.Nature Genetics, 2016, 48 (1) :30-35.
                            </a>
                        </p>
                        <p id="6">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The genetic architecture of type 2 diabetes">

                                <b>[2]</b> Fuchsberger C, Flannick J, Teslovich TM, et al.The genetic architecture of type 2 diabetes.Nature, 2016, 536 (7614) :41.
                            </a>
                        </p>
                        <p id="8">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Genome-wide associa-tion analysis identifies novel blood pressure loci and offers biological insights into cardiovascular risk">

                                <b>[3]</b> Warren HR, Evangelou E, Cabrera C P, et al.Genome-wide association analysis identifies novel blood pressure loci and offers biological insights into cardiovascular risk.Nature Genetics, 2017, 49 (10) :403.
                            </a>
                        </p>
                        <p id="10">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Genomewide association study identifies three novel loci in Fuchs endothelial corneal dystrophy">

                                <b>[4]</b> Afshari NA, Jr IR, Morris NJ, et al.Genome-wide association study identifies three novel loci in Fuchs endothelial corneal dystrophy.Nature Communications, 2017, 8:14898.
                            </a>
                        </p>
                        <p id="12">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls">

                                <b>[5]</b> Burton PR, Clayton DG, Cardon LR, et al.Genome-wide association study of 14, 000 cases of seven common diseases and 3, 000 shared controls.Nature, 2007, 447 (7145) :661-78.
                            </a>
                        </p>
                        <p id="14">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Genome-wide association study of prostate-specific antigen levels identifies novel loci independent of prostate cancer">

                                <b>[6]</b> Hoffmann TJ, Passarelli MN, Graff RE, et al.Genome-wide association study of prostate-specific antigen levels identifies novel loci independent of prostate cancer.Nature Communications, 2017, 8:14248.
                            </a>
                        </p>
                        <p id="16">
                            <a id="bibliography_7" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A large genome-wide association study of age-related macular degeneration highlights contributions of rare and common variants">

                                <b>[7]</b> Fritsche LG, Igl W, Bailey JNC, et al.A large genome-wide association study of age-related macular degeneration highlights contributions of rare and common variants.Nature Genetics, 2016, 48 (2) :134-143.
                            </a>
                        </p>
                        <p id="18">
                            <a id="bibliography_8" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJLA&amp;filename=SJLA3919605C9E6B24048EE72C1B61F2E6F6&amp;v=MDI4MzVCckxVMDV0Rmh3cnU4eEtvPU5pZkhiN0N4SDlqS3I0bzJiWjRKZm40OXp4SWJuMHA2U2d6ajNoUTBEN0RoUTh5WkNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[8]</b> Makowsky R, Pajewski NM, Klimentidis YC, et al.Beyond missing heritability:prediction of complex traits.Plos Genetics, 2011, 7 (4) :e1002051.
                            </a>
                        </p>
                        <p id="20">
                            <a id="bibliography_9" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJLA&amp;filename=SJLAD7AA60B3E7EB5DDD97C064D179B6BC58&amp;v=MjEwMjZOdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQnJMVTA1dEZod3J1OHhLbz1OaWZIYjhlL2I2REtyLzFHRWV4NmZubE51MklhN1V4OVRudVdyUlU4QzdUbU5yK1hDTw==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[9]</b> Hayes BJ, Pryce J, Chamberlain AJ, et al.Genetic Architecture of Complex Traits and Accuracy of Genomic Prediction:Coat Colour, Milk-Fat Percentage, and Type in Holstein Cattle as Contrasting Model Traits.Plos Genetics, 2010, 6 (9) :e1001139.
                            </a>
                        </p>
                        <p id="22">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Developing and evaluating polygenic risk prediction models for stratified disease prevention">

                                <b>[10]</b> Chatterjee N, Shi J, Garcíaclosas M.Developing and evaluating polygenic risk prediction models for stratified disease prevention.Nature Reviews Genetics, 2016, 17 (7) :392.
                            </a>
                        </p>
                        <p id="24">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A gene-based association method for mapping traits using reference transcriptome data">

                                <b>[11]</b> Gamazon ER, Wheeler HE, Wheeler HE, et al.A gene-based association method for mapping traits using reference transcriptome data.Nature Genetics, 2015, 47 (9) :1091-1098.
                            </a>
                        </p>
                        <p id="26">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Estimation of genetic parameters">

                                <b>[12]</b> Handerson CR.Estimation of genetic parameters.Annals of Mathematical Statistics, 1950, 21:309-310.
                            </a>
                        </p>
                        <p id="28">
                            <a id="bibliography_13" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Multi BLUP:improved SNP-based prediction for complex traits">

                                <b>[13]</b> Speed D and Balding DJ.MultiBLUP:improved SNP-based prediction for complex traits.Genome Research, 2014, 24 (9) :1550-1557.
                            </a>
                        </p>
                        <p id="30">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Group Spike-and-Slab Lasso Generalized Linear Models for Disease Prediction and Associated Genes Detection by Incorporating Pathway Information">

                                <b>[14]</b> Tang Z, Shen Y, Li Y, et al.Group Spike-and-Slab Lasso Generalized Linear Models for Disease Prediction and Associated Genes Detection by Incorporating Pathway Information.Bioinformatics, 2017, 34 (6) :901-910.
                            </a>
                        </p>
                        <p id="32">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Extension of the bayesian alphabet for genomic selection">

                                <b>[15]</b> Habier D, Fernando RL, Kizilkaya K, et al.Extension of the bayesian alphabet for genomic selection.BMC Bioinformatics, 2011, 12 (1) :186.
                            </a>
                        </p>
                        <p id="34">
                            <a id="bibliography_16" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Regression Shrinkage and Selection via the Lasso">

                                <b>[16]</b> Tibshirani R.Regression Shrinkage and Selection via the Lasso.Journal of the Royal Statistical Society, 2011, 73 (3) :273-282.
                            </a>
                        </p>
                        <p id="36">
                            <a id="bibliography_17" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD00001256895&amp;v=MjM5MDhyeG94Y01IN1I3cWRaK1p1RmlIbFZiekxKRnc9TmlmY2FyTzRIdEhOcllwRGJPSUtZM2s1ekJkaDRqOTlTWHFS&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[17]</b> Hui Z, Hastie T.Addendum:Regularization and variable selection via the elastic net.Journal of the Royal Statistical Society, 2005, 67 (2) :301-320.
                            </a>
                        </p>
                        <p id="38">
                            <a id="bibliography_18" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJLA&amp;filename=SJLAC493B92DA2C9D54E67A6F0C3412D57CB&amp;v=MDIzNjlmT0dRbGZCckxVMDV0Rmh3cnU4eEtvPU5pZkhiOEM4RjlLK3BvMHhGZWw4QlFnOHkyTVY3VTU3UG4rUnJ4WTBlOGFSUXNudENPTnZGU2lXV3I3SklGcG1hQnVIWQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[18]</b> Zhou X, Carbonetto P, Stephens M.Polygenic Modeling with Bayesian Sparse Linear Mixed Models.Plos Genetics, 2013, 9 (2) ::e1003264.
                            </a>
                        </p>
                        <p id="40">
                            <a id="bibliography_19" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Non-parametric genetic prediction of complex traits with latent Dirichlet process regression models">

                                <b>[19]</b> Zeng P, Zhou X.Non-parametric genetic prediction of complex traits with latent Dirichlet process regression models.Nature Communications, 2017, 8 (1) :456.
                            </a>
                        </p>
                        <p id="42">
                            <a id="bibliography_20" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SPQD&amp;filename=SPQD9469FB1559ADA91A1D5F7008E35526B8&amp;v=MDQwODEzL2lwR2MyZkxlV1E4aVhDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0Rmh3cnU4eEtvPU5qM2FhcnE4R05pNjNZNUFZZUorZUEwd3ptY1Nuam9MVA==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[20]</b> Tang Z, Shen Y, Zhang X, et al.The Spike-and-Slab Lasso Generalized Linear Models for Prediction and Associated Genes Detection.Genetics, 2016, 205 (1) :77-88.
                            </a>
                        </p>
                        <p id="44">
                            <a id="bibliography_21" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJST&amp;filename=SJST15012900203942&amp;v=MTcyNzVZZXJLOUh0RE9wbzlGWnVzTUJYZzdvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp0RmlubFVyYklKbDBYYmhBPU5pZg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[21]</b> Guan Y, Stephens M.Bayesian variable selection regression for genome-wide association studies and other large-scale problems.Annals of Applied Statistics, 2011, 5 (3) :1780-1815.
                            </a>
                        </p>
                        <p id="46">
                            <a id="bibliography_22" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDBE08C68C54CE7FE94C6EB609450A88CD&amp;v=MjQyNTFXV3I3SklGcG1hQnVIWWZPR1FsZkJyTFUwNXRGaHdydTh4S289Tmo3QmFzSE5IdG0vcVljMlllOThlWHRQdWg4WG1Ua0lPbm5pcFJZd2VjT2NUY25yQ09OdkZTaQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[22]</b> Ping Z, Xiang Z, Huang S.Prediction of gene expression with cis-SNPs using mixed models and regularization methods.Bmc Genomics, 2017, 18 (1) :368.
                            </a>
                        </p>
                        <p id="48">
                            <a id="bibliography_23" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJLA&amp;filename=SJLA7D965CB316466A212BDAC56956DAA800&amp;v=MDEzNjNCckxVMDV0Rmh3cnU4eEtvPU5pZkhiN1RNRjlmSjNQMUdaZTBMQ25wSXpSY1JtRXNNTzNya3BSY3pEY1BsVGJxZkNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[23]</b> Moser G, Sang HL, Hayes BJ, et al.Simultaneous Discovery, Estimation and Prediction Analysis of Complex Traits Using a Bayesian Mixture Model.Plos Genetics, 2015, 11 (4) :e1004969.
                            </a>
                        </p>
                        <p id="50">
                            <a id="bibliography_24" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Sparse probit linear mixed model">

                                <b>[24]</b> Mandt S, Wenzel F, Nakajima S, et al.Sparse probit linear mixed model.Machine Learning, 2015 (7298) :1-22.
                            </a>
                        </p>
                        <p id="52">
                            <a id="bibliography_25" target="_blank" href="http://scholar.cnki.net/result.aspx?q=KEGG: kyoto encyclopedia of genes and genomes">

                                <b>[25]</b> Kanehisa M and Goto S.KEGG:kyoto encyclopedia of genes and genomes.Nucleic acids research, 2000, 28 (1) :27-30.
                            </a>
                        </p>
                        <p id="54">
                            <a id="bibliography_26" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Group spike-and-slab lasso generalized linear models for disease prediction and associated genes detection by incorporating pathway information">

                                <b>[26]</b> Tang Z, Shen Y, Li Y, et al.Group spike-and-slab lasso generalized linear models for disease prediction and associated genes detection by incorporating pathway information.Bioinformatics, 2017, 34 (6) :901-910.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="ZGWT201902003" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201902003&amp;v=MTgyMjU3cWZadVpuRnk3bVVMdktQeXJjZXJHNEg5ak1yWTlGWjRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHemFFMEVVb1VEa0hsL0w5az0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=842_VlRz_g1CMCLi-M0twVw-4EOFG_noWyi_OsMnrAg1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
