<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=NOOK-mvTMnxl11s6oPYV0boPXN3iP4N7Rc-A56nk4KI1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2fKXReader%2fDetail%3fTIMESTAMP%3d637141083429350000%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dZGWT201904045%26RESULT%3d1%26SIGN%3dZYJ4Rqy2CedHBfOfQ9Nsxsro1dM%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201904045&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201904045&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201904045&amp;v=MDAzMzlHRnJDVVI3cWZadVpuRnlEbVY3ckpQeXJjZXJHNEg5ak1xNDlCWVlRS0RIODR2UjRUNmo1NE8zenFxQnQ=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#33" data-title="基于NTS局部改变的分析方法 ">基于NTS局部改变的分析方法</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#37" data-title="1.DiffK算法">1.DiffK算法</a></li>
                                                <li><a href="#41" data-title="2.NC算法">2.NC算法</a></li>
                                                <li><a href="#44" data-title="3.DCloc算法">3.DCloc算法</a></li>
                                                <li><a href="#47" data-title="4.&lt;i&gt;PageRank&lt;/i&gt;算法">4.<i>PageRank</i>算法</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#51" data-title="基于NTS全局改变的分析方法 ">基于NTS全局改变的分析方法</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#61" data-title="结  语 ">结  语</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#35" data-title="&lt;b&gt;图1&lt;/b&gt; 两不同分组的调控网络及差异边"><b>图1</b> 两不同分组的调控网络及差异边</a></li>
                                                <li><a href="#55" data-title="&lt;b&gt;图2&lt;/b&gt; BC算法适用的网络图示"><b>图2</b> BC算法适用的网络图示</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="89">


                                    <a id="bibliography_1" title=" Lichtblau Y,Zimmermann K,Haldemann B,et al.Comparative assessment of differential network analysis methods.Briefings in Bioinformatics,2016,18(5):837." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Comparative assessment of differential network analysis methods">
                                        <b>[1]</b>
                                         Lichtblau Y,Zimmermann K,Haldemann B,et al.Comparative assessment of differential network analysis methods.Briefings in Bioinformatics,2016,18(5):837.
                                    </a>
                                </li>
                                <li id="91">


                                    <a id="bibliography_2" title=" Luscombe NM,Babu MM,YuH,et al.Genomic analysis of regulatory network dynamics reveals large topological changes.Nature,2004,431(7006):308." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Genomic analysis of regulatory network dynamics reveals large topological changes">
                                        <b>[2]</b>
                                         Luscombe NM,Babu MM,YuH,et al.Genomic analysis of regulatory network dynamics reveals large topological changes.Nature,2004,431(7006):308.
                                    </a>
                                </li>
                                <li id="93">


                                    <a id="bibliography_3" title=" 王文杰,侯艳,李康.基因组学数据的网络构建与分析方法.中国卫生统计,2017,34(1):177-180." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201701055&amp;v=MTQzNDVyRzRIOWJNcm85QVlZUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVabkZ5RG1WN3JKUHlyY2U=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[3]</b>
                                         王文杰,侯艳,李康.基因组学数据的网络构建与分析方法.中国卫生统计,2017,34(1):177-180.
                                    </a>
                                </li>
                                <li id="95">


                                    <a id="bibliography_4" title=" Zhang B,Horvath S.A General Framework For Weighted Gene Co-Expression Network Analysis.Statistical Applications in Genetics and Molecular Biology,2005,4(1):Article17." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A general framework for weighted gene co-expression network analysis">
                                        <b>[4]</b>
                                         Zhang B,Horvath S.A General Framework For Weighted Gene Co-Expression Network Analysis.Statistical Applications in Genetics and Molecular Biology,2005,4(1):Article17.
                                    </a>
                                </li>
                                <li id="97">


                                    <a id="bibliography_5" title=" Fuller TF,Ghazalpour A,Aten JE,et al.Weighted gene coexpression network analysis strategies applied to mouse weight.Mammalian Genome,2007,18(6-7):463-472." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00000718128&amp;v=MjM3NjRZM2s1ekJkaDRqOTlTWHFScnhveGNNSDdSN3FkWitadUZpSGxXN3pNSlY4PU5qN0Jhck80SHRITXFJNU5aZWtI&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[5]</b>
                                         Fuller TF,Ghazalpour A,Aten JE,et al.Weighted gene coexpression network analysis strategies applied to mouse weight.Mammalian Genome,2007,18(6-7):463-472.
                                    </a>
                                </li>
                                <li id="99">


                                    <a id="bibliography_6" title=" Li M,Wang J,Wang H,et al.Essential Proteins Discovery from Weighted Protein Interaction Networks.Bioinformatics Research and Applications,2010,8(4):293-297." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Essential proteins discovery from weighted protein interaction networks">
                                        <b>[6]</b>
                                         Li M,Wang J,Wang H,et al.Essential Proteins Discovery from Weighted Protein Interaction Networks.Bioinformatics Research and Applications,2010,8(4):293-297.
                                    </a>
                                </li>
                                <li id="101">


                                    <a id="bibliography_7" title=" Wang J,Li M,Wang H,et al.Identification of essential proteins based on edge clustering coefficient.IEEE/ACM Transactions on Computational Biology and Bioinformatics,2012,9(4):1070-1080." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM50DD78CECE49B7D30F4E0BCE52366242&amp;v=MjIwNDJpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0Rmh6THU3eGFrPU5pZklZN2E0YXFYTHAvd3dGNTRMQlE0K3V4VVRuRHNJU0EyUjJSYzNlclNTUjc2ZENPTnZGUw==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[7]</b>
                                         Wang J,Li M,Wang H,et al.Identification of essential proteins based on edge clustering coefficient.IEEE/ACM Transactions on Computational Biology and Bioinformatics,2012,9(4):1070-1080.
                                    </a>
                                </li>
                                <li id="103">


                                    <a id="bibliography_8" title=" Bockmayr M,Klauschen F,Gy&#246;rffy B,et al.New network topology approaches reveal differential correlation patterns in breast cancer.BMC Systems Biology,2013,7(1):78." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=&amp;quot;New network topology approaches reveal differential correlation patterns in breast cancer.&amp;quot;">
                                        <b>[8]</b>
                                         Bockmayr M,Klauschen F,Gy&#246;rffy B,et al.New network topology approaches reveal differential correlation patterns in breast cancer.BMC Systems Biology,2013,7(1):78.
                                    </a>
                                </li>
                                <li id="105">


                                    <a id="bibliography_9" title=" Page L.The PageRank citation ranking:Bringing order to the web.Stanford Digital Libraries Working Paper,1998,9(1):1-14." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The PageRank citation ranking: Bringing order to the web">
                                        <b>[9]</b>
                                         Page L.The PageRank citation ranking:Bringing order to the web.Stanford Digital Libraries Working Paper,1998,9(1):1-14.
                                    </a>
                                </li>
                                <li id="107">


                                    <a id="bibliography_10" title=" Omranian N,Mueller-Roeber B,Nikoloski Z.PageRank-based identification of signaling crosstalk from transcriptomics data:the case of Arabidopsis thaliana.Molecular Biosystems,2012,8(4):1121-1127." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=PageRank-based identification of signaling crosstalk from transcriptomics data: the case of Arabidopsis thaliana">
                                        <b>[10]</b>
                                         Omranian N,Mueller-Roeber B,Nikoloski Z.PageRank-based identification of signaling crosstalk from transcriptomics data:the case of Arabidopsis thaliana.Molecular Biosystems,2012,8(4):1121-1127.
                                    </a>
                                </li>
                                <li id="109">


                                    <a id="bibliography_11" title=" Freeman LC.A Set of Measures of Centrality Based on Betweenness.Sociometry,1977,40(1):35-41." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJST&amp;filename=SJST8900EAD5B0FD54F04AAB335000D3D738&amp;v=MTEyNzBtWENPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkJyTFUwNXRGaHpMdTd4YWs9TmlmWWVydXhIdEc1M3Z0QUZ1dDVlSGs5dVJZWG0wNFBTM3puckJJMURiSGdRcg==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[11]</b>
                                         Freeman LC.A Set of Measures of Centrality Based on Betweenness.Sociometry,1977,40(1):35-41.
                                    </a>
                                </li>
                                <li id="111">


                                    <a id="bibliography_12" title=" Potapov AP,Voss N,Sasse N,et al.Topology of mammalian transcription networks.Genome Informatics International Conference on Genome Informatics,2005." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Topology of mammalian transcription networks">
                                        <b>[12]</b>
                                         Potapov AP,Voss N,Sasse N,et al.Topology of mammalian transcription networks.Genome Informatics International Conference on Genome Informatics,2005.
                                    </a>
                                </li>
                                <li id="113">


                                    <a id="bibliography_13" title=" Odibat O,Reddy CK.Ranking differential genes in co-expression networks.Proceedings of the 2nd ACM Conference onBioinformatics,Computational Biology and Biomedicine-BCB’11,2011,350-354." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Ranking Differential Genes in Co-expression Networks">
                                        <b>[13]</b>
                                         Odibat O,Reddy CK.Ranking differential genes in co-expression networks.Proceedings of the 2nd ACM Conference onBioinformatics,Computational Biology and Biomedicine-BCB’11,2011,350-354.
                                    </a>
                                </li>
                                <li id="115">


                                    <a id="bibliography_14" title=" Vogelstein B,Papadopoulos N,Velculescu V E,et al.Cancer genome landscapes.Science,2013,339(6127):1546-1558." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Cancer genome landscapes">
                                        <b>[14]</b>
                                         Vogelstein B,Papadopoulos N,Velculescu V E,et al.Cancer genome landscapes.Science,2013,339(6127):1546-1558.
                                    </a>
                                </li>
                                <li id="117">


                                    <a id="bibliography_15" title=" Thorne T.NetDiff-Bayesian model selection for differential gene regulatory network inference.Sci Rep,2016,6(1):39224." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=NetDiff-Bayesian model selection for differential gene regulatory network inference">
                                        <b>[15]</b>
                                         Thorne T.NetDiff-Bayesian model selection for differential gene regulatory network inference.Sci Rep,2016,6(1):39224.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=ZGWT" target="_blank">中国卫生统计</a>
                2019,36(04),635-637             </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>差异网络分析方法在组学数据变量筛选中的应用</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E8%94%A1%E9%9B%A8%E6%99%B4&amp;code=42823058&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蔡雨晴</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%AE%8B%E5%BE%AE&amp;code=42823056&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">宋微</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BE%90%E6%AC%A2&amp;code=42823057&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">徐欢</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%9D%8E%E5%BA%B7&amp;code=07019514&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">李康</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%93%88%E5%B0%94%E6%BB%A8%E5%8C%BB%E7%A7%91%E5%A4%A7%E5%AD%A6%E5%8D%AB%E7%94%9F%E7%BB%9F%E8%AE%A1%E5%AD%A6%E6%95%99%E7%A0%94%E5%AE%A4&amp;code=0161188&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">哈尔滨医科大学卫生统计学教研室</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                                
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    *李康,E-mail:likang@ems.hrbmu.edu.cn;
                                </span>
                    </p>
                
                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金(81773551);</span>
                    </p>
            </div>
                                                        

        <!--brief start-->
                        <div class="p1">
                    <p id="32">组学数据(omics data),如基因组、转录组、蛋白质组和代谢组等数据能够反映疾病的发生、发展和预后的情况。对于组学数据的分析,主要是筛选有用的生物标志物、分析调控网络和建立预测模型。一般是通过变量差异表达量分析不同类别之间的差别。但在实际中,也可能有这样一种情况,即在不同分类中,变量的量值变化不大,但其网络拓扑结构(network topology structure,NTS)却发生了变化,这种情况同样能反映不同的生物学特征,并据此发现重要的生物标记物。差异网络分析方法是近年来新提出的一种以网络为基础的生物信息算法,注重不同状态(如健康或患病等)下NTS的差异,从而发现导致不同生物进程的重要差异物质<citation id="119" type="reference"><link href="89" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>。与传统的差异变量分析方法相比,差异网络分析方法更侧重于分析变量间关系的改变,在调控关系发生变化时使用这种分析方法更为有效<citation id="120" type="reference"><link href="91" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>。本文对近年来提出和发展的几种差异网络分析方法做一综述。</p>
                </div>
                <h3 id="33" name="33" class="anchor-tag">基于NTS局部改变的分析方法</h3>
                <div class="p1">
                    <p id="34">差异网络分析将不同分组情况下的数据分别构建网络,如图1a和图1b为两不同分组的网络调控关系,图1c标示了两组变化的调控边。</p>
                </div>
                <div class="area_img" id="35">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201904045_035.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 两不同分组的调控网络及差异边" src="Detail/GetImg?filename=images/ZGWT201904045_035.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图1</b> 两不同分组的调控网络及差异边  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201904045_035.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="36">网络图中描述局部连接参数的指标较多,其中应用最多的为网络连接权重(connectivity),其意义为节点之间的调控关系强弱。在图1中,节点之间的连接边粗细代表不同权重大小。连接权重的求解方法有互信息法、相关系数法、偏相关系数法、回归系数法及其他非线性回归方法等<citation id="121" type="reference"><link href="93" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>。网络连接权重可以通过随机置换试验选择合适的阈值,减少不显著的假阳性边,得到更为优化的网络关系。Zhang等学者提出权重基因共表达网络分析(weighted gene co-expression network analysis,WGCNA)<citation id="122" type="reference"><link href="95" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>,将WGCNA的思想应用在网络中的连接边上,可以得到比未加权更具稳定性的网络。</p>
                </div>
                <h4 class="anchor-tag" id="37" name="37">1.DiffK算法</h4>
                <div class="p1">
                    <p id="38"><i>DiffK</i>(<i>u</i>)=<mathml id="64"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>|</mo><mrow><mfrac><mrow><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>v</mi><mo>∈</mo><mi>Ν</mi><msub><mrow></mrow><mi>u</mi></msub></mrow></munder><mi>k</mi></mstyle><msup><mrow></mrow><mi>A</mi></msup><mo stretchy="false">(</mo><mi>u</mi><mo>,</mo><mi>v</mi><mo stretchy="false">)</mo></mrow><mrow><mi>max</mi><mo stretchy="false">(</mo><mi>k</mi><msup><mrow></mrow><mi>A</mi></msup><mo stretchy="false">)</mo></mrow></mfrac><mo>-</mo><mfrac><mrow><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>v</mi><mo>∈</mo><mi>Ν</mi><msub><mrow></mrow><mi>u</mi></msub></mrow></munder><mi>k</mi></mstyle><msup><mrow></mrow><mi>B</mi></msup><mo stretchy="false">(</mo><mi>u</mi><mo>,</mo><mi>v</mi><mo stretchy="false">)</mo></mrow><mrow><mi>max</mi><mo stretchy="false">(</mo><mi>k</mi><msup><mrow></mrow><mi>B</mi></msup><mo stretchy="false">)</mo></mrow></mfrac></mrow><mo>|</mo></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="39">其中,<i>u</i>为网络内任一节点,<i>N</i><sub><i>u</i></sub>是在网络中与节点<i>u</i>相关联的节点集合。<i>k</i><sup><i>A</i></sup>(<i>u</i>,<i>v</i>)和<i>k</i><sup><i>B</i></sup>(<i>u</i>,<i>v</i>)分别表示不同网络A、B中节点<i>u</i>与其关联节点的连接权重。<i>DiffK</i>值用于比较节点在不同网络中连接权重的差异,以网络中节点连接权重最大值max(<i>k</i><sup><i>A</i></sup>)和max(<i>k</i><sup><i>B</i></sup>)进行标准化以便比较。从公式(1)中可以看出,节点<i>u</i>与其他节点的连接权重和越大,节点<i>u</i>的<i>DiffK</i>值越大。<i>DiffK</i>值可在一定程度上反映节点在网络中的作用大小。Fuller等人在小鼠基因组数据中应用WGCNA方法,与传统的差异表达分析方法相比,找到了与小鼠体重有关的生物标志物和通路信息,表明结合网络特性的DiffK算法效果更好<citation id="123" type="reference"><link href="97" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>。</p>
                </div>
                <div class="p1">
                    <p id="40">网络连接权重从节点间的关联强度考虑节点重要性。在实际生物学现象中,节点的度(degree)同样重要。度即节点的连接边数量,如在图1a中,节点G1在网络中有4条连接边,即节点G1度为4。实际中的多数网络为无标度网络(scale-free network),其特性是仅少数节点有较大的度数,如中心基因(hub genes),多数节点只有少量的连接边。中心基因表达水平的微小变化虽不容易识别,却能明显改变网络的拓扑结构。在蛋白互作网络中,重要的功能性蛋白通常具有较高的度<citation id="124" type="reference"><link href="99" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>。</p>
                </div>
                <h4 class="anchor-tag" id="41" name="41">2.NC算法</h4>
                <div class="p1">
                    <p id="42" class="code-formula">
                        <mathml id="42"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>Ν</mi><mi>C</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo>=</mo><mrow><mo>|</mo><mrow><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>v</mi><mo>∈</mo><mi>Ν</mi><msub><mrow></mrow><mi>u</mi></msub></mrow></munder><mrow><mrow><mo>(</mo><mrow><mfrac><mrow><mi>Ζ</mi><msubsup><mrow></mrow><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mi>A</mi></msubsup></mrow><mrow><mi>min</mi><mo stretchy="false">(</mo><mi>d</mi><msubsup><mrow></mrow><mi>u</mi><mi>A</mi></msubsup><mo>-</mo><mn>1</mn><mo>,</mo><mi>d</mi><msubsup><mrow></mrow><mi>v</mi><mi>A</mi></msubsup><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></mfrac></mrow><mo>)</mo></mrow></mrow></mstyle></mrow></mrow><mo>-</mo></mtd></mtr><mtr><mtd><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>v</mi><mo>∈</mo><mi>Ν</mi><msub><mrow></mrow><mi>u</mi></msub></mrow></munder><mrow><mrow><mrow><mrow><mo>(</mo><mrow><mfrac><mrow><mi>Ζ</mi><msubsup><mrow></mrow><mrow><mi>u</mi><mo>,</mo><mi>v</mi></mrow><mi>B</mi></msubsup></mrow><mrow><mi>min</mi><mo stretchy="false">(</mo><mi>d</mi><msubsup><mrow></mrow><mi>u</mi><mi>B</mi></msubsup><mo>-</mo><mn>1</mn><mo>,</mo><mi>d</mi><msubsup><mrow></mrow><mi>v</mi><mi>B</mi></msubsup><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></mfrac></mrow><mo>)</mo></mrow></mrow><mo>|</mo></mrow></mrow></mstyle><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>2</mn><mo stretchy="false">)</mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="43">其中,<i>N</i><sub><i>u</i></sub>为与节点<i>u</i>相关联的节点集合,<i>Z</i><sub><i>u</i></sub><sub>,</sub><sub><i>v</i></sub>表示在网络图中由节点<i>u</i>和<i>v</i>与其他任意相关节点连接边所组成的三角形数量,<i>d</i><sub><i>u</i></sub>、<i>d</i><sub><i>v</i></sub>分别代表节点<i>u</i>和<i>v</i>的度。公式(2)每个单项式分母含义为节点<i>u</i>和<i>v</i>与其他节点连接组成的三角形最大数量,分子为相应的三角形实际数量。如图1a,G1与G3的度都为4,理论上由G1和G3构成的三角形最大数量为3个,但实际只有Δ<sub>139</sub>1个。NC算法既能考虑到节点的中心性,也能考虑到节点与相邻节点之间的联系。Wang等人将NC算法应用于三个不同的酵母菌蛋白交互网络中,与其他六种差异网络分析方法对比,NC算法在所有网络中的阳性结果,均能得到更多的必需蛋白质,同时NC算法具有更高的灵敏度和特异度<citation id="125" type="reference"><link href="101" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>。</p>
                </div>
                <h4 class="anchor-tag" id="44" name="44">3.DCloc算法</h4>
                <div class="p1">
                    <p id="45" class="code-formula">
                        <mathml id="45"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>D</mi><mi>C</mi><mi>l</mi><mi>o</mi><mi>c</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo>=</mo><mn>1</mn><mo>-</mo><mfrac><mrow><mrow><mo>|</mo><mrow><mi>Ν</mi><msubsup><mrow></mrow><mi>u</mi><mi>A</mi></msubsup><mstyle displaystyle="true"><mo>∩</mo><mi>Ν</mi></mstyle><msubsup><mrow></mrow><mi>u</mi><mi>B</mi></msubsup></mrow><mo>|</mo></mrow></mrow><mrow><mrow><mo>|</mo><mrow><mi>Ν</mi><msubsup><mrow></mrow><mi>u</mi><mi>A</mi></msubsup><mstyle displaystyle="true"><mo>∪</mo><mi>Ν</mi></mstyle><msubsup><mrow></mrow><mi>u</mi><mi>B</mi></msubsup></mrow><mo>|</mo></mrow></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>3</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="46"><i>N</i><mathml id="65"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>u</mi><mi>A</mi></msubsup></mrow></math></mathml>和<i>N</i><mathml id="66"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>u</mi><mi>B</mi></msubsup></mrow></math></mathml>分别表示在网络<i>A</i>和网络<i>B</i>中节点u的关联节点集合。<i>DCloc</i>算法根据节点u在<i>A</i>和<i>B</i>网络中的相同关联节点数目与占所有关联节点数目的比值表示节点u的网络差异重要性。DCloc(u)值大,说明节点u在网络<i>A</i>、<i>B</i>中调控关系发生了较大变化,即节点u是网络拓扑结构的差异位点。当节点u在<i>A</i>、<i>B</i>两网络下关联点并集元素数量小于3时,DCloc(u)=0,即节点u是网络中的孤立节点。<i>Bockmayr</i>等研究人员经过随机试验和交叉验证证明,<i>DCloc</i>算法结果稳定性较高,假阳性率较低,在乳腺癌基因组学<i>ER</i>和<i>HER</i>2分子亚型研究中有良好效果<citation id="126" type="reference"><link href="103" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>。</p>
                </div>
                <h4 class="anchor-tag" id="47" name="47">4.<i>PageRank</i>算法</h4>
                <div class="p1">
                    <p id="48">&lt;i&gt;Page&lt;/i&gt;在1998年首次在网页浏览重要性应用上提出<i>PageRank</i>概念<citation id="127" type="reference"><link href="105" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>,即网页的重要性取决于网页链接指向该网页的其他网页的重要性。同理,可将其理解为网络图中一节点的连接重要性取决于相邻节点的度。如与某一节点连接的相邻节点的度大,则该节点在网络中可能起到传播或桥梁作用。其计算公式为</p>
                </div>
                <div class="p1">
                    <p id="49">R(u)=<mathml id="67"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>|</mo><mrow><mi>c</mi><msub><mrow></mrow><mi>A</mi></msub><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>v</mi><mo>∈</mo><mi>Ρ</mi><msub><mrow></mrow><mi>u</mi></msub></mrow></munder><mrow><mfrac><mrow><mi>R</mi><msup><mrow></mrow><mi>A</mi></msup><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo></mrow><mrow><mi>Ν</mi><msubsup><mrow></mrow><mi>v</mi><mi>A</mi></msubsup></mrow></mfrac></mrow></mstyle><mo>-</mo><mi>c</mi><msub><mrow></mrow><mi>B</mi></msub><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>v</mi><mo>∈</mo><mi>Ρ</mi><msub><mrow></mrow><mi>u</mi></msub></mrow></munder><mrow><mfrac><mrow><mi>R</mi><msup><mrow></mrow><mi>B</mi></msup><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo></mrow><mrow><mi>Ν</mi><msubsup><mrow></mrow><mi>v</mi><mi>B</mi></msubsup></mrow></mfrac></mrow></mstyle></mrow><mo>|</mo></mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>4</mn><mo stretchy="false">)</mo></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="50">PageRank算法首先对所有节点进行一个简单的排秩,通过不断使用公式(4)迭代计算使其收敛。<i>P</i><sub><i>u</i></sub>表示指向节点<i>u</i>的节点集合,<i>B</i><sub><i>u</i></sub>为节点<i>u</i>指向的节点集合,<i>N</i><sub><i>u</i></sub>为<i>B</i><sub><i>u</i></sub>内节点数量。该方法假设节点<i>u</i>的秩<i>R</i>(<i>u</i>)被其指向的节点集合<i>B</i><sub><i>u</i></sub>均分,由于一些节点可能没有<i>B</i><sub><i>u</i></sub>而损失了<i>u</i>的秩,因此<i>c</i><sub><i>A</i></sub>和<i>c</i><sub><i>B</i></sub>分别为A、B网络的标准化系数,它可使网络中所有节点的秩和恒定。Omranian等人以PageRank算法为基础在拟南芥转录组数据中发现了不同信号通路的关键基因<citation id="128" type="reference"><link href="107" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>。值得注意的是,PageRank算法是基于有向网络的一种方法,对网络中心节点的发现有重要意义,尤其当变量数量较多时。</p>
                </div>
                <h3 id="51" name="51" class="anchor-tag">基于NTS全局改变的分析方法</h3>
                <div class="p1">
                    <p id="52">以上基于网络拓扑结构局部改变的方法都是以节点的直接连接节点出发计算其差异统计量,而无法考虑网络中的所有节点对被分析节点的影响,包括直接连接点和间接连接点。基于此,Freeman在1977年提出中介中心性(between centrality,BC)这一概念<citation id="129" type="reference"><link href="109" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>,其公式为</p>
                </div>
                <div class="p1">
                    <p id="53"><i>BC</i>(<i>u</i>)=<mathml id="68"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>|</mo><mrow><mstyle displaystyle="true"><munder><mo>∑</mo><mi>s</mi></munder><mrow><mstyle displaystyle="true"><munder><mo>∑</mo><mi>t</mi></munder><mrow><mfrac><mrow><mi>ρ</mi><msup><mrow></mrow><mi>A</mi></msup><mo stretchy="false">(</mo><mi>s</mi><mo>,</mo><mi>u</mi><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mrow><mi>ρ</mi><msup><mrow></mrow><mi>A</mi></msup><mo stretchy="false">(</mo><mi>s</mi><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></mfrac></mrow></mstyle></mrow></mstyle><mo>-</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mi>s</mi></munder><mrow><mstyle displaystyle="true"><munder><mo>∑</mo><mi>t</mi></munder><mrow><mfrac><mrow><mi>ρ</mi><msup><mrow></mrow><mi>B</mi></msup><mo stretchy="false">(</mo><mi>s</mi><mo>,</mo><mi>u</mi><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mrow><mi>ρ</mi><msup><mrow></mrow><mi>B</mi></msup><mo stretchy="false">(</mo><mi>s</mi><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></mfrac></mrow></mstyle></mrow></mstyle></mrow><mo>|</mo></mrow><mo>,</mo><mi>s</mi><mo>≠</mo><mi>u</mi><mo>≠</mo><mi>t</mi><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>5</mn><mo stretchy="false">)</mo></mrow></math></mathml></p>
                </div>
                <div class="p1">
                    <p id="54">其中,<i>s</i>、<i>t</i>为网络中任意两点,<i>ρ</i>(<i>s</i>,<i>t</i>)表示网络中以<i>s</i>为起点、<i>t</i>为终点的最短路径的总数,<i>ρ</i>(<i>s</i>,<i>u</i>,<i>t</i>)表示在经过<i>s</i>、<i>t</i>节点的所有最短路径(沿节点<i>s</i>到节点<i>t</i>的所有路径中,各边的权重总和最小的路径)中,同时经过节点<i>u</i>的数量。当节点<i>u</i>的关联节点数目较少或节点之间连接权重较小,却是网络中最短路径的必经节点时(如图2中,G6虽然只与G5和G7相关联,但网络中大部分最短路径都要经过G6),仍可认为<i>u</i>是网络中的重要节点,此时<i>BC</i>(<i>u</i>)值能够反映节点<i>u</i>在网络全局中的重要程度。在蛋白网络应用中,一些度低但中介中心性很高(high betweenness low degree,HBLC)的蛋白也发挥着十分重要的作用<citation id="130" type="reference"><link href="109" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>。Potapov将其应用于哺乳动物转录组数据的两个网络中,发现中介中心性对描述生物网络拓扑结构有重要意义,更具有实际生物学意义<citation id="131" type="reference"><link href="111" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>。</p>
                </div>
                <div class="area_img" id="55">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201904045_055.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 BC算法适用的网络图示" src="Detail/GetImg?filename=images/ZGWT201904045_055.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图2</b> BC算法适用的网络图示  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201904045_055.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="56">基于NTS的全局和局部改变分析方法</p>
                </div>
                <div class="p1">
                    <p id="57">既然网络拓扑结构的局部改变与全局改变同样重要,Odibat即提出DiffRank算法<citation id="132" type="reference"><link href="113" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>。DiffRank算法根据节点对网络差异改变的贡献排序,从而筛选出引起网络差异的重要节点。局部和全局结构改变评价指标为</p>
                </div>
                <div class="p1">
                    <p id="58" class="code-formula">
                        <mathml id="58"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>Δ</mi><mi>C</mi><msup><mrow></mrow><mi>i</mi></msup><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>v</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mrow><mfrac><mrow><mrow><mo>|</mo><mrow><mi>k</mi><msup><mrow></mrow><mi>A</mi></msup><mo stretchy="false">(</mo><mi>u</mi><mo>,</mo><mi>v</mi><mo stretchy="false">)</mo><mo>-</mo><mi>k</mi><msup><mrow></mrow><mi>B</mi></msup><mo stretchy="false">(</mo><mi>u</mi><mo>,</mo><mi>v</mi><mo stretchy="false">)</mo></mrow><mo>|</mo></mrow><mo>×</mo><mi>π</mi><msubsup><mrow></mrow><mi>u</mi><mi>i</mi></msubsup></mrow><mrow><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>z</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></msubsup><mrow><mrow><mo>|</mo><mrow><mi>k</mi><msup><mrow></mrow><mi>A</mi></msup><mo stretchy="false">(</mo><mi>u</mi><mo>,</mo><mi>z</mi><mo stretchy="false">)</mo><mo>-</mo><mi>k</mi><msup><mrow></mrow><mi>B</mi></msup><mo stretchy="false">(</mo><mi>u</mi><mo>,</mo><mi>z</mi><mo stretchy="false">)</mo></mrow><mo>|</mo></mrow></mrow></mstyle></mrow></mfrac></mrow></mstyle><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>6</mn><mo stretchy="false">)</mo></mtd></mtr><mtr><mtd><mi>Δ</mi><mi>B</mi><mi>C</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mi>s</mi><mi>Ν</mi></munderover><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mi>t</mi><mi>Ν</mi></munderover><mrow><mrow><mo>|</mo><mrow><mi>S</mi><mi>Ρ</mi><msubsup><mrow></mrow><mi>u</mi><mi>A</mi></msubsup><mo stretchy="false">(</mo><mi>s</mi><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo><mo>-</mo><mi>S</mi><mi>Ρ</mi><msubsup><mrow></mrow><mi>u</mi><mi>B</mi></msubsup><mo stretchy="false">(</mo><mi>s</mi><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mo>|</mo></mrow></mrow></mstyle></mrow></mstyle><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>7</mn><mo stretchy="false">)</mo></mtd></mtr><mtr><mtd><mi>π</mi><msubsup><mrow></mrow><mi>u</mi><mi>i</mi></msubsup><mo>=</mo><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><mi>λ</mi><mo stretchy="false">)</mo><mo>×</mo><mfrac><mrow><mi>Δ</mi><mi>B</mi><mi>C</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo></mrow><mrow><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>v</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></msubsup><mi>Δ</mi></mstyle><mi>B</mi><mi>C</mi><mo stretchy="false">(</mo><mi>v</mi><mo stretchy="false">)</mo></mrow></mfrac><mo>+</mo><mi>λ</mi><mo>×</mo><mi>Δ</mi><mi>C</mi><msup><mrow></mrow><mi>i</mi></msup><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>8</mn><mo stretchy="false">)</mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="59">其中,网络A和网络B中分别有<i>N</i>个节点,<i>k</i>(<i>u</i>,<i>v</i>)为节点<i>u</i>与相关节点<i>v</i>的连接权重。<i>π</i><mathml id="69"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>u</mi><mi>i</mi></msubsup></mrow></math></mathml>为节点<i>u</i>在网络中第<i>i</i>次迭代的差异评分,用参数<i>λ</i>结合两部分指标综合评价,<i>λ</i>取值范围为[0,1],如<i>λ</i>=0.75。将所有节点的差异评分初始值设为1/<i>N</i>,经过每一次迭代发生变化,当差异评分的改变达到规定的一个比较小的值时循环停止。<i>SP</i><sub><i>u</i></sub>(<i>s</i>,<i>t</i>)表示通过节点<i>u</i>的一个<i>N</i>×<i>N</i>矩阵,在网络中任意两节点<i>s</i>、<i>t</i>的最短路径若通过节点<i>u</i>,则在矩阵中用1表示,否则用0表示。计算通过节点<i>u</i>的最短路径数量来反映其网络中心性。</p>
                </div>
                <div class="p1">
                    <p id="60">基于每一节点的差异评分<i>π</i>给所有节点排序,<i>π</i>越大表示在差异网络中贡献越大,即我们所需要筛选的差异位点。DiffRank算法既考虑了网络中节点的局部信息,又考虑了网络全局信息,Lichtblau评价了十种差异网络分析方法对4个相同数据集的筛选效果,其中包括多个局部与全局NTS差异网络算法,根据给出的差异基因金标准(gold standard list,GSL)<citation id="133" type="reference"><link href="115" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>判断,DiffRank算法在前40个差异位点中重合概率最高,Fisher确切检验表明结果具有统计学意义<citation id="134" type="reference"><link href="89" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>。</p>
                </div>
                <h3 id="61" name="61" class="anchor-tag">结  语</h3>
                <div class="p1">
                    <p id="62">本文主要介绍了网络拓扑结构的一些基本概念以及近年来提出和发展的几种差异网络分析方法,其主要特点是通过不同分组各节点或拓扑结构的变化筛选重要的变量,克服了单纯比较变量均值变化的不足。本文介绍的三类方法各有特点,如局部度数检验能够识别直接调控其他节点的关键调控因子,但不能发现间接调控其他节点的重要节点;全局中介中心性算法考虑了网络中各节点对其他节点的影响,更强调中间调节点;局部与全局结合在一起的DiffRank算法则能够更全面地分析网络调控信息。实际中,当变量数过大时,在差异网络分析之前,可以结合差异表达分析对变量进行筛选或选取与分组信息有关的通路进行分析。</p>
                </div>
                <div class="p1">
                    <p id="63">网络的基本概念还包括接近中心性(closeness centrality,CC)、特征向量中心性(eigenvector centrality,EC)等,基于这些概念的差异网络分析方法也有很多,不仅能通过网络的信息筛选组学数据中的差异位点,还可以得到导致不同结局的差异边,如根据贝叶斯算法判断差异边等<citation id="135" type="reference"><link href="117" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation>。目前大部分差异网络分析都是基于排秩方法判断差异物质,但不同数据情况无法确定取前几位,因此如何通过统计学方法选取合适的显著性差异物质是亟需解决的问题。同时,由于基因间真实的相关关系未知,缺乏可用的金标准来评估差异网络分析方法用于真实数据的可靠性,目前可借助已有的通路信息(如KEGG、GO等数据库)对其进行验证和支持。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="89">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Comparative assessment of differential network analysis methods">

                                <b>[1]</b> Lichtblau Y,Zimmermann K,Haldemann B,et al.Comparative assessment of differential network analysis methods.Briefings in Bioinformatics,2016,18(5):837.
                            </a>
                        </p>
                        <p id="91">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Genomic analysis of regulatory network dynamics reveals large topological changes">

                                <b>[2]</b> Luscombe NM,Babu MM,YuH,et al.Genomic analysis of regulatory network dynamics reveals large topological changes.Nature,2004,431(7006):308.
                            </a>
                        </p>
                        <p id="93">
                            <a id="bibliography_3" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201701055&amp;v=MTUwOTlHRnJDVVI3cWZadVpuRnlEbVY3ckpQeXJjZXJHNEg5Yk1ybzlBWVlRS0RIODR2UjRUNmo1NE8zenFxQnQ=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[3]</b> 王文杰,侯艳,李康.基因组学数据的网络构建与分析方法.中国卫生统计,2017,34(1):177-180.
                            </a>
                        </p>
                        <p id="95">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A general framework for weighted gene co-expression network analysis">

                                <b>[4]</b> Zhang B,Horvath S.A General Framework For Weighted Gene Co-Expression Network Analysis.Statistical Applications in Genetics and Molecular Biology,2005,4(1):Article17.
                            </a>
                        </p>
                        <p id="97">
                            <a id="bibliography_5" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00000718128&amp;v=MDY2ODhTWHFScnhveGNNSDdSN3FkWitadUZpSGxXN3pNSlY4PU5qN0Jhck80SHRITXFJNU5aZWtIWTNrNXpCZGg0ajk5&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[5]</b> Fuller TF,Ghazalpour A,Aten JE,et al.Weighted gene coexpression network analysis strategies applied to mouse weight.Mammalian Genome,2007,18(6-7):463-472.
                            </a>
                        </p>
                        <p id="99">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Essential proteins discovery from weighted protein interaction networks">

                                <b>[6]</b> Li M,Wang J,Wang H,et al.Essential Proteins Discovery from Weighted Protein Interaction Networks.Bioinformatics Research and Applications,2010,8(4):293-297.
                            </a>
                        </p>
                        <p id="101">
                            <a id="bibliography_7" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM50DD78CECE49B7D30F4E0BCE52366242&amp;v=MjQwODlMVTA1dEZoekx1N3hhaz1OaWZJWTdhNGFxWExwL3d3RjU0TEJRNCt1eFVUbkRzSVNBMlIyUmMzZXJTU1I3NmRDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCcg==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[7]</b> Wang J,Li M,Wang H,et al.Identification of essential proteins based on edge clustering coefficient.IEEE/ACM Transactions on Computational Biology and Bioinformatics,2012,9(4):1070-1080.
                            </a>
                        </p>
                        <p id="103">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=&amp;quot;New network topology approaches reveal differential correlation patterns in breast cancer.&amp;quot;">

                                <b>[8]</b> Bockmayr M,Klauschen F,Györffy B,et al.New network topology approaches reveal differential correlation patterns in breast cancer.BMC Systems Biology,2013,7(1):78.
                            </a>
                        </p>
                        <p id="105">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The PageRank citation ranking: Bringing order to the web">

                                <b>[9]</b> Page L.The PageRank citation ranking:Bringing order to the web.Stanford Digital Libraries Working Paper,1998,9(1):1-14.
                            </a>
                        </p>
                        <p id="107">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=PageRank-based identification of signaling crosstalk from transcriptomics data: the case of Arabidopsis thaliana">

                                <b>[10]</b> Omranian N,Mueller-Roeber B,Nikoloski Z.PageRank-based identification of signaling crosstalk from transcriptomics data:the case of Arabidopsis thaliana.Molecular Biosystems,2012,8(4):1121-1127.
                            </a>
                        </p>
                        <p id="109">
                            <a id="bibliography_11" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJST&amp;filename=SJST8900EAD5B0FD54F04AAB335000D3D738&amp;v=MDc1ODlybVhDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0Rmh6THU3eGFrPU5pZlllcnV4SHRHNTN2dEFGdXQ1ZUhrOXVSWVhtMDRQUzN6bnJCSTFEYkhnUQ==&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[11]</b> Freeman LC.A Set of Measures of Centrality Based on Betweenness.Sociometry,1977,40(1):35-41.
                            </a>
                        </p>
                        <p id="111">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Topology of mammalian transcription networks">

                                <b>[12]</b> Potapov AP,Voss N,Sasse N,et al.Topology of mammalian transcription networks.Genome Informatics International Conference on Genome Informatics,2005.
                            </a>
                        </p>
                        <p id="113">
                            <a id="bibliography_13" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Ranking Differential Genes in Co-expression Networks">

                                <b>[13]</b> Odibat O,Reddy CK.Ranking differential genes in co-expression networks.Proceedings of the 2nd ACM Conference onBioinformatics,Computational Biology and Biomedicine-BCB’11,2011,350-354.
                            </a>
                        </p>
                        <p id="115">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Cancer genome landscapes">

                                <b>[14]</b> Vogelstein B,Papadopoulos N,Velculescu V E,et al.Cancer genome landscapes.Science,2013,339(6127):1546-1558.
                            </a>
                        </p>
                        <p id="117">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=NetDiff-Bayesian model selection for differential gene regulatory network inference">

                                <b>[15]</b> Thorne T.NetDiff-Bayesian model selection for differential gene regulatory network inference.Sci Rep,2016,6(1):39224.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="ZGWT201904045" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201904045&amp;v=MDAzMzlHRnJDVVI3cWZadVpuRnlEbVY3ckpQeXJjZXJHNEg5ak1xNDlCWVlRS0RIODR2UjRUNmo1NE8zenFxQnQ=&amp;uid=WEEvREcwSlJHSldRa1FhdXNzY2Z1OVRsbWlROEJvNUc5VFdtU3JEQUFDdz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=842_VlRz_g1CMCLi-M0twVw-4EOFG_noWyi_OsMnrAg1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
