<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=NOOK-mvTMnxl11s6oPYV0boPXN3iP4N7Rc-A56nk4KI1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2fKXReader%2fDetail%3fTIMESTAMP%3d637141023766537500%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dZGWT201903006%26RESULT%3d1%26SIGN%3dEBDosDpWtE1u7AMxXsSTw86hVa0%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201903006&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201903006&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201903006&amp;v=MDczODJyY2VyRzRIOWpNckk5RllvUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVabkZ5N2dWTC9OUHk=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#34" data-title="原理与方法 ">原理与方法</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#43" data-title="实例分析 ">实例分析</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#45" data-title="1.变量选择和样本确定">1.变量选择和样本确定</a></li>
                                                <li><a href="#50" data-title="2.建立模型和变量筛选">2.建立模型和变量筛选</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#59" data-title="讨  论 ">讨  论</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#37" data-title="&lt;b&gt;图1&lt;/b&gt; 支持向量机原理示意图"><b>图1</b> 支持向量机原理示意图</a></li>
                                                <li><a href="#49" data-title="&lt;b&gt;表1&lt;/b&gt; 影响SBP和DBP的23个SNPs位点"><b>表1</b> 影响SBP和DBP的23个SNPs位点</a></li>
                                                <li><a href="#53" data-title="&lt;b&gt;表2&lt;/b&gt; 不同SNPs组合情况下基于MIV的SVM方法的实验结果"><b>表2</b> 不同SNPs组合情况下基于MIV的SVM方法的实验结果</a></li>
                                                <li><a href="#55" data-title="&lt;b&gt;表3&lt;/b&gt; 最优的基因组合的SNPs位点"><b>表3</b> 最优的基因组合的SNPs位点</a></li>
                                                <li><a href="#57" data-title="&lt;b&gt;表4&lt;/b&gt; 针对高血压SNP数据的不同分类方法的结果比较"><b>表4</b> 针对高血压SNP数据的不同分类方法的结果比较</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="4">


                                    <a id="bibliography_1" title=" Ayers KL, Cordell HJ.Analysis of Genetic Analysis Workshop 18 data with gene-based penalized regression.Bmc Proceedings, 2014, 8 (S1) :1-5." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Analysis of Genetic Analysis Workshop 18 data with gene-based penalized regression">
                                        <b>[1]</b>
                                         Ayers KL, Cordell HJ.Analysis of Genetic Analysis Workshop 18 data with gene-based penalized regression.Bmc Proceedings, 2014, 8 (S1) :1-5.
                                    </a>
                                </li>
                                <li id="6">


                                    <a id="bibliography_2" title=" 丁佩剑, 金秀菊, 孙宝信.人类单核苷酸多态性研究进展.临床和实验医学杂志, 2016, 15 (12) :1235-1238." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=SYLC201612035&amp;v=MTk1NDdmWnVabkZ5N2dWTC9OTmpUSGJiRzRIOWZOclk5R1lZUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3E=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[2]</b>
                                         丁佩剑, 金秀菊, 孙宝信.人类单核苷酸多态性研究进展.临床和实验医学杂志, 2016, 15 (12) :1235-1238.
                                    </a>
                                </li>
                                <li id="8">


                                    <a id="bibliography_3" title=" Saeys Y, Inza I, Larra?aga P.A review of feature selection techniques in bioinformatics.Bioinformatics, 2007, 23 (19) :2507-2517." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A review of feature selection techniques in bioinformatics">
                                        <b>[3]</b>
                                         Saeys Y, Inza I, Larra?aga P.A review of feature selection techniques in bioinformatics.Bioinformatics, 2007, 23 (19) :2507-2517.
                                    </a>
                                </li>
                                <li id="10">


                                    <a id="bibliography_4" title=" Wang HY, Chang SC, Lin WY, et al.Machine Learning-Based Method for Obesity Risk Evaluation Using Single-Nucleotide Polymorphisms Derived from Next-Generation Sequencing.J Comput Biol, 2018, " target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Machine Learning-Based Method for Obesity Risk Evaluation Using Single-Nucleotide Polymorphisms Derived from Next-Generation Sequencing">
                                        <b>[4]</b>
                                         Wang HY, Chang SC, Lin WY, et al.Machine Learning-Based Method for Obesity Risk Evaluation Using Single-Nucleotide Polymorphisms Derived from Next-Generation Sequencing.J Comput Biol, 2018, 
                                    </a>
                                </li>
                                <li id="12">


                                    <a id="bibliography_5" title=" 宋欠欠, 李轶群, 侯艳, 等.随机森林的变量捕获方法在高维数据变量筛选中的应用.中国卫生统计, 2015, 32 (1) :49-53." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201501013&amp;v=MTA2MTN5N2dWTC9OUHlyY2VyRzRIOVRNcm85RVo0UUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVabkY=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[5]</b>
                                         宋欠欠, 李轶群, 侯艳, 等.随机森林的变量捕获方法在高维数据变量筛选中的应用.中国卫生统计, 2015, 32 (1) :49-53.
                                    </a>
                                </li>
                                <li id="14">


                                    <a id="bibliography_6" title=" Vapnik VN.The Nature of Statistic Learning Theory.2000." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The Nature of Statistical Learning Theory">
                                        <b>[6]</b>
                                         Vapnik VN.The Nature of Statistic Learning Theory.2000.
                                    </a>
                                </li>
                                <li id="16">


                                    <a id="bibliography_7" title=" 田英杰.支持向量回归机及其应用研究;中国农业大学, 2005." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CDFD&amp;filename=2005085011.nh&amp;v=MDY0MjRHN093RzlITnJwRWJQSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1Wm5GeTdnVkwvTlYxMjc=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[7]</b>
                                         田英杰.支持向量回归机及其应用研究;中国农业大学, 2005.
                                    </a>
                                </li>
                                <li id="18">


                                    <a id="bibliography_8" >
                                        <b>[8]</b>
                                     王小川.MATLAB神经网络43个案例分析.北京航空航天大学出版社, 2013.</a>
                                </li>
                                <li id="20">


                                    <a id="bibliography_9" title=" 邓乃扬, 田英杰.数据挖掘中的新方法:支持向量机.科学出版社, 2004." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CBBD&amp;filename=0007030132815999&amp;v=MDYwMjM3SHREUHJZZEVZZUlHQlJNOHp4VVNtRGQ5U0g3bjN4RTlmYnZuS3JpZlplWnZGeW5zVTduTkoxNFJWVjI3R2JP&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[9]</b>
                                         邓乃扬, 田英杰.数据挖掘中的新方法:支持向量机.科学出版社, 2004.
                                    </a>
                                </li>
                                <li id="22">


                                    <a id="bibliography_10" title=" Burges CJC, Smola AJ.Advances in kernel methods:support vector learning.MIT Press, 1999." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Advances in Kernel Methods-Support Vector Learning">
                                        <b>[10]</b>
                                         Burges CJC, Smola AJ.Advances in kernel methods:support vector learning.MIT Press, 1999.
                                    </a>
                                </li>
                                <li id="24">


                                    <a id="bibliography_11" title=" 吕艳伟, 李文桓, 田伟, 等.基于不同核函数构建的退行性颈椎病支持向量机高危人群筛查模型的比较.中国卫生统计, 2018, 35 (3) :368-371." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201803010&amp;v=MTk4NzVFWklRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpuRnk3Z1ZML05QeXJjZXJHNEg5bk1ySTk=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[11]</b>
                                         吕艳伟, 李文桓, 田伟, 等.基于不同核函数构建的退行性颈椎病支持向量机高危人群筛查模型的比较.中国卫生统计, 2018, 35 (3) :368-371.
                                    </a>
                                </li>
                                <li id="26">


                                    <a id="bibliography_12" title=" Jiang J, Tan Q, Li W, et al.Use of Support Vector Regression Based on Mean Impact Value Model to Identify Active Compounds in a Combination of Curcuma longa L.and Glycyrrhiza extracts.Transactions of Tianjin University, 2017, 23 (3) :237-244." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDCB8E39B7EA2E51D8B21EB2D95316557D&amp;v=MDI3NTZ3SzA9Tmo3QmFzREtGcVRQcHYxQ0Vab05lWGs0dXg1aDZENElPbjJXcFJjMmVMU1JRTDNyQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQnJMVTA1dEZod3IyNA==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[12]</b>
                                         Jiang J, Tan Q, Li W, et al.Use of Support Vector Regression Based on Mean Impact Value Model to Identify Active Compounds in a Combination of Curcuma longa L.and Glycyrrhiza extracts.Transactions of Tianjin University, 2017, 23 (3) :237-244.
                                    </a>
                                </li>
                                <li id="28">


                                    <a id="bibliography_13" title=" Thaker VV, Esteves KM, Towne MC, et al.Whole exome sequencing identifies RAI1 mutation in a morbidly obese child diagnosed with ROHHAD syndrome.J Clin Endocrinol Metab, 2015, 100 (5) :1723-1730." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWK&amp;filename=SJWK15051200000229&amp;v=MDY2NDVlWnRGaW5sVXJiSUpsc1RhaGM9TmlmY1piSzlIdFROclk5RlpPc1BEbjR3b0JNVDZUNFBRSC9pclJkR2VycVFUTW53Wg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[13]</b>
                                         Thaker VV, Esteves KM, Towne MC, et al.Whole exome sequencing identifies RAI1 mutation in a morbidly obese child diagnosed with ROHHAD syndrome.J Clin Endocrinol Metab, 2015, 100 (5) :1723-1730.
                                    </a>
                                </li>
                                <li id="30">


                                    <a id="bibliography_14" title=" Hu WS, Ho TJ, Pai P, et al.Gelsolin (GSN) induces cardiomyocyte hypertrophy and BNP expression via p38 signaling and GATA-4 transcriptional factor activation.Mol Cell Biochem, 2014, 390 (1-2) :263-270." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD14040800003478&amp;v=MTA3MTRkR2VycVFUTW53WmVadEZpbmxVcmJJSmxzVGFoYz1OajdCYXJLOEh0WE1wNDlGWk9zTUNIc3hvQk1UNlQ0UFFIL2lyUg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[14]</b>
                                         Hu WS, Ho TJ, Pai P, et al.Gelsolin (GSN) induces cardiomyocyte hypertrophy and BNP expression via p38 signaling and GATA-4 transcriptional factor activation.Mol Cell Biochem, 2014, 390 (1-2) :263-270.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=ZGWT" target="_blank">中国卫生统计</a>
                2019,36(03),344-347             </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm">基于平均影响值的SVM在遗传数据疾病分类和特征提取中的应用</span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BC%A0%E9%98%B3%E9%98%B3&amp;code=42075827&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">张阳阳</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%9B%B9%E7%BA%A2%E8%89%B3&amp;code=21624985&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">曹红艳</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%AD%A6%E6%B7%91%E7%90%B4&amp;code=21899490&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">武淑琴</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%B1%B1%E8%A5%BF%E5%8C%BB%E7%A7%91%E5%A4%A7%E5%AD%A6%E5%8D%AB%E7%94%9F%E7%BB%9F%E8%AE%A1%E6%95%99%E7%A0%94%E5%AE%A4&amp;code=0049397&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">山西医科大学卫生统计教研室</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p><b>目的</b> 探讨基于平均影响值 (MIV) 的支持向量机 (SVM) 在遗传数据疾病分类预测和变量筛选中的应用, 为遗传数据的疾病分类与特征提取方面提供方法学参考。<b>方法</b> 以GAW18 (genetic analysis workshop 18) 数据为例, 采用基于MIV的SVM建立预测模型, 并和logistic回归模型、SVM、多层感知机和决策树分类模型进行比较分析, 评价基于MIV的SVM预测分类和变量筛选效果。<b>结果</b> 经过平均影响值的支持向量机算法处理后, 六个SNPs位点 (13_28567172、3_127394820、1_1658093、9_123969834、1_174996637、17_17498492) 组合的变量子集, 获得78.125%的分类准确率, 明显优于其他分类模型。<b>结论</b> 基于MIV的SVM能比较有效的在实现遗传数据变量筛选的同时提高分类预测能力, 避免了变量间的交互作用, 为探索各种疾病发病机制和寻找易感SNPs位点提供线索, 具有一定的研究和应用价值。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%94%AF%E6%8C%81%E5%90%91%E9%87%8F%E6%9C%BA&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">支持向量机;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%B9%B3%E5%9D%87%E5%BD%B1%E5%93%8D%E5%80%BC&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">平均影响值;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8D%95%E6%A0%B8%E8%8B%B7%E9%85%B8%E5%A4%9A%E6%80%81%E6%80%A7&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">单核苷酸多态性;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    *武淑琴, E-mail:wsq13.wu@163.com;
                                </span>
                    </p>
                
                    <p>

                            <b>基金：</b>
                                                        <span>山西省回国留学人员科研资助项目 (2017-054);</span>
                    </p>
            </div>
                    <h1><b>Application of SVM based on MIV in the Classification and Feature Extraction of Genetic Data</b></h1>
                    <h2>
                    <span>Zhang Yangyang</span>
                    <span>Cao Hongyan</span>
                    <span>Wu Shuqin</span>
            </h2>
                    <h2>
                    <span>Department of Health Statistics, Shanxi Medical University</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p><b>Objective</b> The application of support vector machine (SVM) based on average impact value (MIV) in genetic data classification, prediction and variable selection is discussed to provide methodological reference for disease classification and feature extraction.<b>Methods</b> Taking GAW18 data as an example, a prediction model was built based on MIV SVM, and compared with the logistic regression model, the SVM, the MLP and the tree algorithms model, and MIV based SVM prediction classification and variable selection effect were evaluated.<b>Results</b> After the support vector machine algorithm with MIV is processed, the subset of six SNPs loci (13_28567172、3_127394820、1_1658093、9_123969834、1_174996637、17_17498492) is combined to get 78.125%, which is obviously better than that of other models.<b>Conclusion</b> The SVM based on MIV can be more effective in improving the classification prediction ability while implementing genetic data variables screening.It avoids the interaction between variables, and provides clues for exploring the pathogenesis of various diseases and finding vulnerable SNPs loci.It has research value and application value.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Support%20vector%20machine&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Support vector machine;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Mean%20impact%20value&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Mean impact value;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=SNPs&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">SNPs;</a>
                </p>
            </div>
        

        <!--brief start-->
                        <div class="p1">
                    <p id="32">随着DNA测序技术的发展和高通量测序的出现, 后续产生的大量遗传数据对统计分析方法提出新的要求。单核苷酸多态性 (single nucleotide polymorphisms, SNPs) <citation id="63" type="reference"><link href="4" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>是基因组水平上单个核苷酸变异所引起的DNA序列多态性, 在人类基因组中广泛存在, 估计其总数可达到300万个甚至更多<citation id="64" type="reference"><link href="6" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>。本文拟采用遗传分析工作组18 (genetic analysis workshop 18, GAW18) 中134个相互独立个体的全基因组测序数据, 每个样本拥有较多SNPs, 高维度和较小的样本量给疾病分类和特征提取带来了极大的挑战。而变量降维和筛选常见的方法有主成分分析、Wrapper法<citation id="65" type="reference"><link href="8" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>、filter法<citation id="66" type="reference"><link href="8" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>、逐步多元线性回归<citation id="67" type="reference"><link href="10" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>和随机森林<citation id="68" type="reference"><link href="12" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>等。本文采用Wrapper法的思想以分类的精度为指标, 在完成变量筛选的同时获得较高的分类准确率。由于该方法的特征寻找依赖于分类器的性能, 所以合适分类模型的选择尤为重要。</p>
                </div>
                <div class="p1">
                    <p id="33">针对分类模型的选择问题, 最初采用由Vapnik于1968年提出的支持向量机 (support vector machine, SVM) <citation id="69" type="reference"><link href="14" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>。支持向量是一种机器学习算法, 广泛应用于数据挖掘、文本分类和图像识别等多个方面。该算法以统计学习理论为基础, 结构风险最小化为原则, 既考虑渐进性能的要求, 又在有限的信息条件下追求最优的结果, 是一种具有坚实理论基础的小样本学习方法<citation id="70" type="reference"><link href="16" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>。该算法的判别函数最终由少数几个支持向量所决定, 在某种意义上避免了“维数灾难”。所以SVM在面对小样本和高维度数据时有较好的优越性。其理论基础和算法本质决定了该算法在生物遗传数据上良好的适应性。但由于SVM是一种将全部变量纳入模型进行训练和预测的机器学习方法, 无法对变量进行特征筛选。因此需要寻找合适的方法来解决该问题。平均影响值 (mean impact value, MIV) 被认为是神经网络中评价变量相关性最好的指标之一<citation id="71" type="reference"><link href="18" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>。故考虑将SVM和MIV结合起来, 通过评价各个变量相对结局变量的影响程度, 找到对结果有较大影响的自变量, 继而实现使用SVM进行变量筛选的过程。因此, 本文提出一种基于平均影响值的支持向量机的方法, 能相对有效的解决生物遗传数据的特征提取问题。使用较少的变量构建更好的分类预测模型, 在提高分类预测模型准确率的同时实现了变量筛选。本文以GAW18数据为例, 采用基于平均影响值的支持向量机进行分析, 一方面为遗传数据的分析和处理提供方法学参考。另一方面, 对特征变量的提取有助于使人们的注意力集中到少数的变量上, 使得实验研究更具有针对性。</p>
                </div>
                <h3 id="34" name="34" class="anchor-tag">原理与方法</h3>
                <div class="p1">
                    <p id="35">支持向量机最初是为了解决分类问题而提出的一种机器学习方法<citation id="72" type="reference"><link href="14" rel="bibliography" /><link href="20" rel="bibliography" /><sup>[<a class="sup">6</a>,<a class="sup">9</a>]</sup></citation>。本研究将SNPs数据的每个样本与样本空间中的样本点一一对应。其基本原理是在样本空间中寻找一超平面, 使得高血压和非高血压样本点分别分布在超平面的两侧。如图1所示, 低维度的样本空间线性不可分, 通过非线性映射到高维空间后变得线性可分。在数据空间中, 设用于分离的超平面方程是<i>ω</i>·<i>x</i>+<i>b</i>=0 (<i>ω</i>是可调的权值向量, <i>b</i>是偏置) 。设最优的<i>ω</i>和<i>b</i>为<i>ω</i><sub>0</sub>和<i>b</i><sub>0</sub>, 则有:<i>ω</i><sub>0</sub><i>x</i>+<i>b</i><sub>0</sub>=0。支持向量是最靠近决策面的样本点 (<i>x</i><sub><i>i</i></sub>, <i>y</i><sub><i>i</i></sub>) , 满足条件:<i>ω</i>·<i>x</i><sub><i>i</i></sub>+<i>b</i>=<i>y</i><sub><i>i</i></sub>, <i>y</i><sub><i>i</i></sub>=±1。设 (<i>x</i><sub>1</sub>, 1) 和 (<i>x</i><sub>2</sub>, -1) 分别为最靠近决策超平面支持向量, 该两点满足<i>ω</i>·0<i>x</i><sub>1</sub>+<i>b</i>=1和<i>ω</i>·<i>x</i><sub>2</sub>+<i>b</i>=-1;则该两点的间隔为<mathml id="36"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>d</mi><mi>i</mi><mi>s</mi><mo>=</mo><mfrac><mi>ω</mi><mrow><mo stretchy="false">∥</mo><mi>ω</mi><mo stretchy="false">∥</mo></mrow></mfrac><mo>⋅</mo><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mn>1</mn></msub><mo>-</mo><mi>x</mi><msub><mrow></mrow><mn>2</mn></msub><mo stretchy="false">) </mo><mo>=</mo><mfrac><mn>2</mn><mrow><mo stretchy="false">∥</mo><mi>ω</mi><mo stretchy="false">∥</mo></mrow></mfrac></mrow></math></mathml>。对于任意的样本点 (<i>x</i><sub><i>i</i></sub>, <i>y</i><sub><i>i</i></sub>) 均满足<i>y</i><sub><i>i</i></sub>· (<i>ω</i>·<i>x</i><sub><i>i</i></sub>+<i>b</i>) ≥1。</p>
                </div>
                <div class="area_img" id="37">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201903006_037.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 支持向量机原理示意图" src="Detail/GetImg?filename=images/ZGWT201903006_037.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图1</b> 支持向量机原理示意图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201903006_037.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="38">支持向量的间隔问题最终归结为一个求最小<mathml id="39"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mfrac><mrow><mo stretchy="false">∥</mo><mtext>ω</mtext><mo stretchy="false">∥</mo><msup><mrow></mrow><mn>2</mn></msup></mrow><mn>2</mn></mfrac></mrow></math></mathml>二次规划问题。使用Lagrange乘子法可以将该问题转换为原问题的对偶问题, 此时可以求得原问题的最优解:<i>w</i><sub>0</sub>=∑<mathml id="40"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></msubsup></mrow></math></mathml><i>a</i><sup>*</sup><sub><i>i</i></sub><i>y</i><sub><i>i</i></sub><i>x</i><sub><i>i</i></sub><i>x</i>;<i>b</i><sub>0</sub>=1-<i>w</i><sub>0</sub><i>x</i><sup>[<i>s</i>]</sup>, <i>y</i><sup> (<i>s</i>) </sup>=1;。至此, 原函数得到解决获得判别函数:<mathml id="41"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mi>s</mi><mi>g</mi><mi>n</mi></mrow><mrow><mo> (</mo><mrow><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></msubsup><mi>a</mi></mstyle><msubsup><mrow></mrow><mi>i</mi><mo>*</mo></msubsup><mi>y</mi><msub><mrow></mrow><mi>i</mi></msub><mi>x</mi><msub><mrow></mrow><mi>i</mi></msub><mi>x</mi><mo>+</mo><mi>b</mi><msub><mrow></mrow><mn>0</mn></msub></mrow><mo>) </mo></mrow></mrow></math></mathml>, 其中<i>sgn</i>是数学中的符号函数, <i>a</i><sup>*</sup><sub><i>i</i></sub>表示最优的Lagrange乘子, <i>x</i>为测试集中的样本。而线性不可分的情况则是通过升维将线性不可分的数据通过一定的方法映射到高维空间中, 在高维空间中使用线性回归对空间中的数据点进行分类或者回归。其中需要注意的是, 高维空间中训练点的内积可以用原来空间中的函数 (核函数) 进行替代。支持向量机 (SVM) 就是该算法的典型代表<citation id="74" type="reference"><link href="22" rel="bibliography" /><link href="24" rel="bibliography" /><sup>[<a class="sup">10</a>,<a class="sup">11</a>]</sup></citation>。核函数与支持向量机的性能好坏有着直接的关系。目前最常用的核函数有四类, 分别为:高斯径向基核、多项式核、sigmoid核、B-样条核<citation id="73" type="reference"><link href="16" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>。</p>
                </div>
                <div class="p1">
                    <p id="42">平均影响值 (mean impact value, MIV) 是评价各变量对结局变量影响程度的重要指标, 其符号代表相关的方向, 绝对值大小代表影响的程度<citation id="75" type="reference"><link href="26" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>。MIV的具体做法为:构建一个SVM模型, 再将SNPs的训练样本<i>p</i>的每个变量在原来的基础上分别加或减某个值构成两个新的训练样本<i>P</i>1、<i>P</i>2, 将<i>P</i>1、<i>P</i>2作为测试集带入训练好的模型进行预测, 得到两个结果<i>A</i>1、<i>A</i>2, 两者的差值就是该变量变动后对输出产生的影响变化值 (impact value, IV) 。再将IV按照例数进行平均得到该自变量SNP相对于因变量高血压的MIV。计算出每一个SNPs变量相对高血压的MIV, 按照绝对值降序排列, 然后依次去除对结局变量影响最小的若干个SNPs变量, 剩下的变量作为SNPs变量子集, 分析其对样本的分类能力, 从中寻找具有最小SNPs数和最高分类率的候选子集。</p>
                </div>
                <h3 id="43" name="43" class="anchor-tag">实例分析</h3>
                <div class="p1">
                    <p id="44">本研究通过对GAW18数据进行分析和研究, 探寻高血压发生和SNPs位点间的关系, 将有助于利用遗传数据从SNPs的角度对高血压患者进行筛查和预测。并提供一些可能与高血压发生相关的候选SNPs位点。</p>
                </div>
                <h4 class="anchor-tag" id="45" name="45">1.变量选择和样本确定</h4>
                <div class="p1">
                    <p id="46">本次研究采用的数据源于GAW18, 一个国际上公开的用于研究稀有变异关联方法的模拟数据平台, 研究者针对GAW的组织者所发布的遗传数据进行分析。该数据包括20个墨西哥裔美国家庭的1043个个体的全基因组测序数据 (whole genome sequencing data, WGS) 及其相对应的纵向血压表型数据。</p>
                </div>
                <div class="p1">
                    <p id="47">GAW18数据仅提供了奇数染色体遗传数据, 包括四种数据类型:基因型数据 (chrN-geno.csv.gz) , GWAS数据 (chrN-geno.csv.gz) , 原始测序数据 (chrN-seq.vcf.gz) 以及变异剂量数据 (chrN-dose.csv.gz) 。本文针对变异剂量数据 (chrN-dose.csv.gz) 进行分析。GAW18的表型数据包括真实测量数据和模拟数据。GAW18的“GAW18-TheAnswers”中提供了具体的模拟研究设置情况, 包括主要的效应基因和功能SNPs。本研究将信息完整的134个无亲缘关系、互不相关的个体作为研究对象。高血压的诊断标准为收缩压 (SBP) &gt;140mmHg, 舒张压 (DBP) &gt;90mmHg, 测量期间使用高血压药物也诊断为高血压。</p>
                </div>
                <div class="p1">
                    <p id="48">由于四次血压测量中第一次测量缺失最少, 因此选择第一次测量结果作为结局变量。自变量则选择影响SBP和DBP的SNPs位点。前十五个分别影响收缩压和舒张压的SNPs位点中共有7个SNPs的位点同时影响SBP和DBP。将其合并后总共23个SNPs位点作为自变量。利用MATLAB软件实现, 采用基于平均影响值的支持向量机对该高血压数据进行分析。 (表1) </p>
                </div>
                <div class="area_img" id="49">
                    <p class="img_tit"><b>表1</b> 影响SBP和DBP的23个SNPs位点 <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="49" border="1"><tr><td>Number</td><td>SNPs</td><td>染色体</td><td>RS编号</td><td>累计方差解释率 (%) </td></tr><tr><td>1</td><td>1_174996637</td><td>1</td><td>rs35996757</td><td>0.0408</td></tr><tr><td><br />2</td><td>1_65912722</td><td>1</td><td>none</td><td>0.0250</td></tr><tr><td><br />3</td><td>1_151491026</td><td>1</td><td>none</td><td>0.0083</td></tr><tr><td><br />4</td><td>1_1658093</td><td>1</td><td>none</td><td>0.0036</td></tr><tr><td><br />5</td><td>1_53712727</td><td>1</td><td>rs5174</td><td>0.0017</td></tr><tr><td><br />6</td><td>3_58089761</td><td>3</td><td>none</td><td>0.0029</td></tr><tr><td><br />7</td><td>3_47912898</td><td>3</td><td>none</td><td>0.0648</td></tr><tr><td><br />8</td><td>3_127394820</td><td>3</td><td>rs35703629</td><td>0.0013</td></tr><tr><td><br />9</td><td>5_7870973</td><td>5</td><td>rs1801394</td><td>0.0036</td></tr><tr><td><br />10</td><td>7_129252980</td><td>7</td><td>none</td><td>0.0265</td></tr><tr><td><br />11</td><td>7_150041836</td><td>7</td><td>rs6964110</td><td>0.002</td></tr><tr><td><br />12</td><td>7_73822336</td><td>7</td><td>none</td><td>0.0036</td></tr><tr><td><br />13</td><td>9_115803080</td><td>9</td><td>none</td><td>0.0092</td></tr><tr><td><br />14</td><td>9_123969834</td><td>9</td><td>rs7030849</td><td>0.0021</td></tr><tr><td><br />15</td><td>9_123575167</td><td>9</td><td>none</td><td>0.0016</td></tr><tr><td><br />16</td><td>11_67288594</td><td>11</td><td>rs2276118</td><td>0.0021</td></tr><tr><td><br />17</td><td>11_77935643</td><td>11</td><td>none</td><td>0.0014</td></tr><tr><td><br />18</td><td>13_28567172</td><td>13</td><td>rs2481968</td><td>0.0122</td></tr><tr><td><br />19</td><td>17_17498492</td><td>17</td><td>rs114892192</td><td>0.0032</td></tr><tr><td><br />20</td><td>19_12513424</td><td>19</td><td>none</td><td>0.0034</td></tr><tr><td><br />21</td><td>19_58740339</td><td>19</td><td>rs260438</td><td>0.0021</td></tr><tr><td><br />22</td><td>21_46266768</td><td>21</td><td>rs35846107</td><td>0.0028</td></tr><tr><td><br />23</td><td>21_32253513</td><td>21</td><td>rs9636845</td><td>0.0013</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h4 class="anchor-tag" id="50" name="50">2.建立模型和变量筛选</h4>
                <div class="p1">
                    <p id="51">针对与高血压相关性较强的23个SNP位点进行进一步筛选以获得更好的自变量期望维数。采用MIV与SVM相结合的方法来进一步的筛选SNPs信息。</p>
                </div>
                <div class="p1">
                    <p id="52">从样本集中随机抽取70个高血压患者和30个非高血压患者作为训练集, 剩余样本作为测试集。训练集和测试集样本量的比例约为3∶1。使用训练集对模型进行训练和预测, 并记录测试集的分类准确率。把所有的SNPs纳入作为自变量, 高血压结局作为因变量进行模型训练, 使用10-折交叉验证进行模型寻优, 寻找到最佳参数, 使用训练好的模型对预测样本进行预测, 记录分类准确率。按照变量的重要程度进行排序, 得到结果见表2。</p>
                </div>
                <div class="area_img" id="53">
                    <p class="img_tit"><b>表2</b> 不同SNPs组合情况下基于MIV的SVM方法的实验结果 <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="53" border="1"><tr><td><br />SNP个数</td><td><i>c</i></td><td><i>g</i></td><td><i>Best</i></td><td>预测准确 (%) </td></tr><tr><td>23</td><td>1</td><td>0.5</td><td>70.2970297</td><td>71.875</td></tr><tr><td><br />21</td><td>1</td><td>0.5</td><td>70.2970297</td><td>71.875</td></tr><tr><td><br />20</td><td>1</td><td>0.5</td><td>70.2970297</td><td>71.875</td></tr><tr><td><br />19</td><td>1</td><td>0.5</td><td>70.2970297</td><td>71.875</td></tr><tr><td><br />18</td><td>1</td><td>0.5</td><td>70.2970297</td><td>71.875</td></tr><tr><td><br />17</td><td>1</td><td>0.5</td><td>70.2970297</td><td>71.875</td></tr><tr><td><br />16</td><td>1</td><td>0.5</td><td>70.2970297</td><td>71.875</td></tr><tr><td><br />15</td><td>1</td><td>1</td><td>70.2970297</td><td>75.000</td></tr><tr><td><br />14</td><td>1</td><td>1</td><td>70.2970297</td><td>75.000</td></tr><tr><td><br />13</td><td>1</td><td>1</td><td>72.27722772</td><td>75.000</td></tr><tr><td><br />12</td><td>1</td><td>1</td><td>72.27722772</td><td>75.000</td></tr><tr><td><br />11</td><td>1</td><td>1</td><td>73.26732673</td><td>75.000</td></tr><tr><td><br />10</td><td>1</td><td>1</td><td>75.24752475</td><td>75.000</td></tr><tr><td><br />9</td><td>1</td><td>1</td><td>75.24752475</td><td>75.000</td></tr><tr><td><br />8</td><td>1</td><td>1</td><td>75.24752475</td><td>75.000</td></tr><tr><td><br />7</td><td>1</td><td>1</td><td>75.24752475</td><td>75.000</td></tr><tr><td><br />6</td><td>1024</td><td>1024</td><td>74.25742574</td><td>78.125</td></tr><tr><td><br />5</td><td>0.5</td><td>1</td><td>72.27722772</td><td>68.750</td></tr><tr><td><br />4</td><td>0.5</td><td>1</td><td>71.28712871</td><td>68.750</td></tr><tr><td><br />3</td><td>1024</td><td>1024</td><td>71.28712871</td><td>75.000</td></tr><tr><td><br />2</td><td>1024</td><td>0.001953125</td><td>69.30693069</td><td>71.875</td></tr><tr><td><br />1</td><td>1024</td><td>1024</td><td>69.30693069</td><td>68.750</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="54">从表2中可以看到, 当变量个数为6时, 预测样本的分类准确率最高为78.125%, 高于其他自变量组合的分类准确率。这6个SNPs可能与高血压疾病相关, 它们的信息如表3所示。</p>
                </div>
                <div class="area_img" id="55">
                    <p class="img_tit"><b>表3</b> 最优的基因组合的SNPs位点 <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="55" border="1"><tr><td><br />SNP编号</td><td>SNP所在基因</td><td>Reference SNP</td></tr><tr><td><br />13_28567172</td><td>FLT3</td><td>rs2481968</td></tr><tr><td><br />3_127394820</td><td>ABTB1</td><td>rs35703629</td></tr><tr><td><br />1_1658093</td><td>SLC35E2</td><td>无</td></tr><tr><td><br />9_123969834</td><td>GSN</td><td>rs7030849</td></tr><tr><td><br />1_174996637</td><td>TNN</td><td>rs35996757</td></tr><tr><td><br />17_17498492</td><td>RAI1</td><td>rs114892192</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="56">logistic回归模型采用二分类非条件logistic回归建立模型, 进入变量的方法为逐步筛选。支持向量机的方法, 将样本划分为训练集和测试集, 经验证核函数选择RBF时分类准确率最高, 为71.88%。神经网络采用多层感知机, 其识别精度为68.00%。决策树模型采用常用的C4.5建立模型, 建立好的模型精度为34.37%。表4给出了针对高血压的SNP表达数据采用不用方法进行比较的结果。</p>
                </div>
                <div class="area_img" id="57">
                    <p class="img_tit"><b>表4</b> 针对高血压SNP数据的不同分类方法的结果比较 <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="57" border="1"><tr><td><br />序号</td><td>方法</td><td>分类器</td><td>基因数</td><td>识别精度</td></tr><tr><td>1</td><td>MIVSVM</td><td>SVM</td><td>6</td><td>78.00%</td></tr><tr><td><br />2</td><td>基于logistic的变量筛选</td><td>logistic</td><td>7</td><td>74.40%</td></tr><tr><td><br />3</td><td>不进行变量筛选的SVM</td><td>SVM (RBF) </td><td>23</td><td>71.88%</td></tr><tr><td><br />4</td><td>不进行变量筛选的神经网络</td><td>多层感知机</td><td>23</td><td>68.20%</td></tr><tr><td><br />5</td><td>决策树</td><td>c4.5</td><td>23</td><td>34.37%</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="58">比较发现, 基于平均影响值的支持向量机相较传统的logistic回归能更好的筛选变量并有更高的识别精度。相比其他机器学习算法在分类准确率上也有较大提高。本文提出的基于MIV的SVM方法与其他方法相比, 具有较好的变量筛选能力和预测效果。</p>
                </div>
                <h3 id="59" name="59" class="anchor-tag">讨  论</h3>
                <div class="p1">
                    <p id="60">基于MIV的SVM方法通过考虑不同变量的改变对结局变量的影响, 同时实现了对遗传数据的特征提取和模型构建。本文以高血压遗传数据的SNPs位点分析为例, 识别了与高血压的发生可能相关的SNPs位点, 并与其他方法进行了比较, 说明基于MIV的SVM在生物遗传数据的分析中有较好的变量筛选和预测能力。</p>
                </div>
                <div class="p1">
                    <p id="61">本文通过基于平均影响值的支持向量机的机器学习算法, 构建不同的SNPs集合, 避免了SNPs位点间的交互作用。该算法在构建了一个利用SNP信息数据对遗传数据进行预测分类模型的同时实现了变量的筛选, 即为疾病的发病机制提供线索, 又为疾病易感SNPs的研究提供了候选位点。此外, 通过相关生物医学方面的文献发现本文获得特征SNP突变位点与高血压存在一定联系:基因RAI1 (retinoic acid induced 1) 在基因的表达和转录过程中起着重要作用<citation id="76" type="reference"><link href="28" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>, 与体内所有的生物化学功能相关;基因TNN (tenascin N) 与细胞的生长、移行相关;基因GSN (gelsolin) 与肌动蛋白的相关活动有关<citation id="77" type="reference"><link href="30" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>, 通过切断和封盖调节动态肌动蛋白丝组织, 而肌动蛋白与血管收缩的功能密切相关;基因SLC35E2 (solute carrier family35, memberE2) 与离子的跨膜运输有关, 钠离子和钾离子对高血压的影响广为人知;基因FLT3 (fms-related tyrosine kinase3) 与细胞的增殖和凋亡有关。以上结果均显示筛选出的SNPs位点与高血压有一定的相关性, 有可能是高血压发生的中间环节。需要指出的是, 基于MIV的SVM筛选出的位点, 仍然需要进一步的生物学方面的实验验证。</p>
                </div>
                <div class="p1">
                    <p id="62">总之, 基于平均影响值的支持向量机不但考虑到生物遗传数据的分析难点, 而且对高维变量进行了降维, 在较低的维度内获得了分类准确率更高的分类预测模型, 是一种针对遗传数据比较有力的分析方法。一方面实现了疾病在SNPs层面的预测和分类模型的构建;另一方面, 将人们的注意力从大量的遗传变量转移到较少的变量上, 为科学研究提供方向。基于MIV的SVM为探索疾病发生时体内的生物学过程和病理机制提供遗传方面的线索, 将有可能在日益繁复的遗传数据分析中发挥重要作用。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="4">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Analysis of Genetic Analysis Workshop 18 data with gene-based penalized regression">

                                <b>[1]</b> Ayers KL, Cordell HJ.Analysis of Genetic Analysis Workshop 18 data with gene-based penalized regression.Bmc Proceedings, 2014, 8 (S1) :1-5.
                            </a>
                        </p>
                        <p id="6">
                            <a id="bibliography_2" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=SYLC201612035&amp;v=MjAwMTZxQnRHRnJDVVI3cWZadVpuRnk3Z1ZML05OalRIYmJHNEg5Zk5yWTlHWVlRS0RIODR2UjRUNmo1NE8zenE=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[2]</b> 丁佩剑, 金秀菊, 孙宝信.人类单核苷酸多态性研究进展.临床和实验医学杂志, 2016, 15 (12) :1235-1238.
                            </a>
                        </p>
                        <p id="8">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A review of feature selection techniques in bioinformatics">

                                <b>[3]</b> Saeys Y, Inza I, Larra?aga P.A review of feature selection techniques in bioinformatics.Bioinformatics, 2007, 23 (19) :2507-2517.
                            </a>
                        </p>
                        <p id="10">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Machine Learning-Based Method for Obesity Risk Evaluation Using Single-Nucleotide Polymorphisms Derived from Next-Generation Sequencing">

                                <b>[4]</b> Wang HY, Chang SC, Lin WY, et al.Machine Learning-Based Method for Obesity Risk Evaluation Using Single-Nucleotide Polymorphisms Derived from Next-Generation Sequencing.J Comput Biol, 2018, 
                            </a>
                        </p>
                        <p id="12">
                            <a id="bibliography_5" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201501013&amp;v=MzI3MTNvOUVaNFFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1Wm5GeTdnVkwvTlB5cmNlckc0SDlUTXI=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[5]</b> 宋欠欠, 李轶群, 侯艳, 等.随机森林的变量捕获方法在高维数据变量筛选中的应用.中国卫生统计, 2015, 32 (1) :49-53.
                            </a>
                        </p>
                        <p id="14">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The Nature of Statistical Learning Theory">

                                <b>[6]</b> Vapnik VN.The Nature of Statistic Learning Theory.2000.
                            </a>
                        </p>
                        <p id="16">
                            <a id="bibliography_7" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CDFD&amp;filename=2005085011.nh&amp;v=MzI1NDFIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpuRnk3Z1ZML05WMTI3RzdPd0c5SE5ycEViUElRS0Q=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[7]</b> 田英杰.支持向量回归机及其应用研究;中国农业大学, 2005.
                            </a>
                        </p>
                        <p id="18">
                            <a id="bibliography_8" >
                                    <b>[8]</b>
                                 王小川.MATLAB神经网络43个案例分析.北京航空航天大学出版社, 2013.
                            </a>
                        </p>
                        <p id="20">
                            <a id="bibliography_9" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CBBD&amp;filename=0007030132815999&amp;v=MTM1NjBVN25OSjE0UlZWMjdHYk83SHREUHJZZEVZZUlHQlJNOHp4VVNtRGQ5U0g3bjN4RTlmYnZuS3JpZlplWnZGeW5z&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[9]</b> 邓乃扬, 田英杰.数据挖掘中的新方法:支持向量机.科学出版社, 2004.
                            </a>
                        </p>
                        <p id="22">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Advances in Kernel Methods-Support Vector Learning">

                                <b>[10]</b> Burges CJC, Smola AJ.Advances in kernel methods:support vector learning.MIT Press, 1999.
                            </a>
                        </p>
                        <p id="24">
                            <a id="bibliography_11" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201803010&amp;v=MjY0NTVnVkwvTlB5cmNlckc0SDluTXJJOUVaSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1Wm5GeTc=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[11]</b> 吕艳伟, 李文桓, 田伟, 等.基于不同核函数构建的退行性颈椎病支持向量机高危人群筛查模型的比较.中国卫生统计, 2018, 35 (3) :368-371.
                            </a>
                        </p>
                        <p id="26">
                            <a id="bibliography_12" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDCB8E39B7EA2E51D8B21EB2D95316557D&amp;v=MTc5NDNCYXNES0ZxVFBwdjFDRVpvTmVYazR1eDVoNkQ0SU9uMldwUmMyZUxTUlFMM3JDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0Rmh3cjI0d0swPU5qNw==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[12]</b> Jiang J, Tan Q, Li W, et al.Use of Support Vector Regression Based on Mean Impact Value Model to Identify Active Compounds in a Combination of Curcuma longa L.and Glycyrrhiza extracts.Transactions of Tianjin University, 2017, 23 (3) :237-244.
                            </a>
                        </p>
                        <p id="28">
                            <a id="bibliography_13" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWK&amp;filename=SJWK15051200000229&amp;v=MTMxMDJud1plWnRGaW5sVXJiSUpsc1RhaGM9TmlmY1piSzlIdFROclk5RlpPc1BEbjR3b0JNVDZUNFBRSC9pclJkR2VycVFUTQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[13]</b> Thaker VV, Esteves KM, Towne MC, et al.Whole exome sequencing identifies RAI1 mutation in a morbidly obese child diagnosed with ROHHAD syndrome.J Clin Endocrinol Metab, 2015, 100 (5) :1723-1730.
                            </a>
                        </p>
                        <p id="30">
                            <a id="bibliography_14" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD14040800003478&amp;v=MjAyNDVlWnRGaW5sVXJiSUpsc1RhaGM9Tmo3QmFySzhIdFhNcDQ5RlpPc01DSHN4b0JNVDZUNFBRSC9pclJkR2VycVFUTW53Wg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[14]</b> Hu WS, Ho TJ, Pai P, et al.Gelsolin (GSN) induces cardiomyocyte hypertrophy and BNP expression via p38 signaling and GATA-4 transcriptional factor activation.Mol Cell Biochem, 2014, 390 (1-2) :263-270.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="ZGWT201903006" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201903006&amp;v=MDczODJyY2VyRzRIOWpNckk5RllvUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVabkZ5N2dWTC9OUHk=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hESGJHMVNKUVUyNW92ZnE0bzFYQT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=842_VlRz_g1CMCLi-M0twVw-4EOFG_noWyi_OsMnrAg1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
