

<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>

</head>

<body>

    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2fKXReader%2fDetail%3fTIMESTAMP%3d637141084777135000%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dZGWT201905003%26RESULT%3d1%26SIGN%3dCyXCJXVLwNNYd7SBzNiP1GN4Zwo%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201905003&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=ZGWT201905003&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>


    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201905003&amp;v=MjEwNTFuRnlEbVZidkJQeXJjZXJHNEg5ak1xbzlGWjRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVo=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#33" data-title="CS原理与方法 ">CS原理与方法</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#34" data-title="1.CS理论简介">1.CS理论简介</a></li>
                                                <li><a href="#37" data-title="2.基于CS理论的变量筛选方法">2.基于CS理论的变量筛选方法</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#57" data-title="模拟实验 ">模拟实验</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#59" data-title="1.模拟实验条件设置">1.模拟实验条件设置</a></li>
                                                <li><a href="#61" data-title="2.模拟实验结果">2.模拟实验结果</a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#65" data-title="实例应用 ">实例应用</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#69" data-title="讨论与结论 ">讨论与结论</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#63" data-title="&lt;b&gt;图1&lt;/b&gt;&lt;i&gt;p&lt;/i&gt;=50为例,不同相关系数条件下三种方法筛选变量的效果比较"><b>图1</b><i>p</i>=50为例,不同相关系数条件下三种方法筛选变量的效果比较</a></li>
                                                <li><a href="#64" data-title="&lt;b&gt;图2&lt;/b&gt;&lt;i&gt;r&lt;/i&gt;=0.4为例,不同差异变量比例条件下三种方法筛选变量的效果比较"><b>图2</b><i>r</i>=0.4为例,不同差异变量比例条件下三种方法筛选变量的效果比较</a></li>
                                                <li><a href="#67" data-title="&lt;b&gt;表1&lt;/b&gt; 筛选出的22个差异蛋白质鉴定结果"><b>表1</b> 筛选出的22个差异蛋白质鉴定结果</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="4">


                                    <a id="bibliography_1" title=" Rai AJ,Chan DW.Cancer Proteomics:Serum Diagnostics for Tumor Marker Discovery.Annals of the New York Academy of Sciences,2004,1022(1):286-294." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD00001101437&amp;v=MzA5OTVJWTNrNXpCZGg0ajk5U1hxUnJ4b3hjTUg3UjdxZForWnVGaUhsVzd6T0pGYz1OaWZjYXJPNEh0SE5ybzlFWU9n&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[1]</b>
                                         Rai AJ,Chan DW.Cancer Proteomics:Serum Diagnostics for Tumor Marker Discovery.Annals of the New York Academy of Sciences,2004,1022(1):286-294.
                                    </a>
                                </li>
                                <li id="6">


                                    <a id="bibliography_2" title=" Donoho DL.Compressed sensing.IEEE Transactions on Information Theory,2006,52(4):1289-1306." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Compressed sensing">
                                        <b>[2]</b>
                                         Donoho DL.Compressed sensing.IEEE Transactions on Information Theory,2006,52(4):1289-1306.
                                    </a>
                                </li>
                                <li id="8">


                                    <a id="bibliography_3" title=" Wang A,Lin F,Jin Z,et al.A Configurable Energy-Efficient Compressed Sensing Architecture With Its Application on Body Sensor Networks.IEEE Transactions on Industrial Informatics,2017,12(1):15-27." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A Configurable Energy-Efficient Compressed Sensing Architecture With Its Application on Body Sensor Networks">
                                        <b>[3]</b>
                                         Wang A,Lin F,Jin Z,et al.A Configurable Energy-Efficient Compressed Sensing Architecture With Its Application on Body Sensor Networks.IEEE Transactions on Industrial Informatics,2017,12(1):15-27.
                                    </a>
                                </li>
                                <li id="10">


                                    <a id="bibliography_4" title=" Tivive FHC,Bouzerdoum A.A compressed sensing method for complex-valued signals with application to through-the-wall radar imaging.IEEE International Conference on Acoustics,Speech and Signal Processing.IEEE,2013:2144-2148." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A compressed sensing method for complex-valued signals with application to through-the-wall radar imaging">
                                        <b>[4]</b>
                                         Tivive FHC,Bouzerdoum A.A compressed sensing method for complex-valued signals with application to through-the-wall radar imaging.IEEE International Conference on Acoustics,Speech and Signal Processing.IEEE,2013:2144-2148.
                                    </a>
                                </li>
                                <li id="12">


                                    <a id="bibliography_5" title=" Lustig M,Santos JM,Lee JH,et al.Application of “Compressed Sensing” for Rapid MR Imaging.Spars,2010,58(6):1182-1195." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Sparse MRI: The application of compressed sensing for rapid MR imaging">
                                        <b>[5]</b>
                                         Lustig M,Santos JM,Lee JH,et al.Application of “Compressed Sensing” for Rapid MR Imaging.Spars,2010,58(6):1182-1195.
                                    </a>
                                </li>
                                <li id="14">


                                    <a id="bibliography_6" title=" Li X,Luo S.A compressed sensing-based iterative algorithm for CT reconstruction and its possible application to phase contrast imaging.BioMedical Engineering OnLine,2011,10(1):73." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A compressed sensing-based iterative algorithm for CT reconstruction and its possible application to phase contrast imaging">
                                        <b>[6]</b>
                                         Li X,Luo S.A compressed sensing-based iterative algorithm for CT reconstruction and its possible application to phase contrast imaging.BioMedical Engineering OnLine,2011,10(1):73.
                                    </a>
                                </li>
                                <li id="16">


                                    <a id="bibliography_7" title=" 李珅,马彩文,李艳,等.压缩感知重构算法综述.红外与激光工程,2013,42(s1):225-232." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=HWYJ2013S1044&amp;v=MDgzNjM0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVabkZ5RG1WYnZCTFRyU1pMRzRIOUt2cm85QllJUUtESDg=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[7]</b>
                                         李珅,马彩文,李艳,等.压缩感知重构算法综述.红外与激光工程,2013,42(s1):225-232.
                                    </a>
                                </li>
                                <li id="18">


                                    <a id="bibliography_8" title=" 柯朝甫,张涛,武晓岩,等.代谢组学数据分析的统计学方法.中国卫生统计,2014,31(2):357-359." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201402052&amp;v=MDM4Mjl1Wm5GeURtVmJ2QlB5cmNlckc0SDlYTXJZOUFab1FLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlo=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[8]</b>
                                         柯朝甫,张涛,武晓岩,等.代谢组学数据分析的统计学方法.中国卫生统计,2014,31(2):357-359.
                                    </a>
                                </li>
                                <li id="20">


                                    <a id="bibliography_9" title=" Terry J,Leung S,Laskin J,et al.Optimal immunohistochemical markers for distinguishing lung adenocarcinomas from squamous cell carcinomas in small tumor samples.Am J Surg Pathol,2010,34:1805-1811." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWK&amp;filename=SJWK121016063204&amp;v=MTAzODlCTTh6eFVTbURkOVNIN24zeEU5ZmJ2bktyaWZaZVp2Rnluc1U3ZkxKbG9kTmlmY1piSzZIOUhOcVk5RFora1BD&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[9]</b>
                                         Terry J,Leung S,Laskin J,et al.Optimal immunohistochemical markers for distinguishing lung adenocarcinomas from squamous cell carcinomas in small tumor samples.Am J Surg Pathol,2010,34:1805-1811.
                                    </a>
                                </li>
                                <li id="22">


                                    <a id="bibliography_10" title=" Schwarz J,Ayim AA,Jager S,et al.Differential expression of desmosomal plakophilins in various types of carcinomas:correlation with cell type and differentiation.Human Pathology,2006,37(5):613-622." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011600344146&amp;v=MTMxMDRaZVp0RmlubFVyYklLRjBTYmhzPU5pZk9mYks3SHRETnFZOUZaKzhMRFhnL29CTVQ2VDRQUUgvaXJSZEdlcnFRVE1udw==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[10]</b>
                                         Schwarz J,Ayim AA,Jager S,et al.Differential expression of desmosomal plakophilins in various types of carcinomas:correlation with cell type and differentiation.Human Pathology,2006,37(5):613-622.
                                    </a>
                                </li>
                                <li id="24">


                                    <a id="bibliography_11" title=" Shi I,Hashemi Sadraei N,Duan ZH,et al.Aberrant Signaling Pathways in Squamous Cell Lung Car-cinoma.Cancer Inform,2011,10:273-285." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Aberrant Signaling Pathways in Squamous Cell Lung Carcino-ma">
                                        <b>[11]</b>
                                         Shi I,Hashemi Sadraei N,Duan ZH,et al.Aberrant Signaling Pathways in Squamous Cell Lung Car-cinoma.Cancer Inform,2011,10:273-285.
                                    </a>
                                </li>
                                <li id="26">


                                    <a id="bibliography_12" title=" Rekhtman N,Ang DC,Sima CS,et al.Immunohistochemical algorithm for differentia-tion of lung adenocarcinoma and squamous cell carcinoma based on large series of whole-tissue sections with validation in small specimens.Mod Pathol Off J U S Can Acad Pathol Inc,2011,24:1348-1359." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Immunohistochemical algorithm for differentiation of lung adenocarcinoma and squamous cell carcinoma based on large series of whole-tissue sections with validation in small specimens">
                                        <b>[12]</b>
                                         Rekhtman N,Ang DC,Sima CS,et al.Immunohistochemical algorithm for differentia-tion of lung adenocarcinoma and squamous cell carcinoma based on large series of whole-tissue sections with validation in small specimens.Mod Pathol Off J U S Can Acad Pathol Inc,2011,24:1348-1359.
                                    </a>
                                </li>
                                <li id="28">


                                    <a id="bibliography_13" title=" G&#243;mez-Morales M,C&#225;mara-Pulido M,Miranda-Le&#243;n MT,et al.Differential immunohistochemical localization of desmosomal plaque-related proteins in non-small-cell lung cancer.Histopathology,2013,63:103-113." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD13062800007835&amp;v=MTU5MDJiSUtGMFNiaHM9TmlmY2FySzdIdGZPcDQ5RlpPc0lCSDg4b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadEZpbmxVcg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[13]</b>
                                         G&#243;mez-Morales M,C&#225;mara-Pulido M,Miranda-Le&#243;n MT,et al.Differential immunohistochemical localization of desmosomal plaque-related proteins in non-small-cell lung cancer.Histopathology,2013,63:103-113.
                                    </a>
                                </li>
                                <li id="30">


                                    <a id="bibliography_14" title=" Stewart PA,Parapatics K,Welsh EA,et al.A Pilot Proteogenomic Study with Data Integration Identifies MCT1 and GLUT1 as Prognostic Markers in Lung Adenocarcinoma.Plos One,2015,10(11):e0142162." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A Pilot Proteogenomic Study with Data Integration Identifies MCT1 and GLUT1 as Prognostic Markers in Lung Adenocarcinoma">
                                        <b>[14]</b>
                                         Stewart PA,Parapatics K,Welsh EA,et al.A Pilot Proteogenomic Study with Data Integration Identifies MCT1 and GLUT1 as Prognostic Markers in Lung Adenocarcinoma.Plos One,2015,10(11):e0142162.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=ZGWT" target="_blank">中国卫生统计</a>
                2019,36(05),649-652+657             </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm">压缩感知理论在小样本量蛋白质组学变量筛选研究中的应用</span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BC%A0%E8%96%87&amp;code=07021088&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">张薇</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BC%A0%E7%A7%8B%E8%8F%8A&amp;code=10612857&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">张秋菊</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E7%8E%8B%E7%8E%89%E9%B9%8F&amp;code=27548074&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">王玉鹏</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E8%B0%A2%E5%BD%AA&amp;code=33921518&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">谢彪</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%AD%99%E7%90%B3&amp;code=36315966&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">孙琳</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E9%AB%98%E5%85%B5&amp;code=36315965&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">高兵</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%8F%B6%E5%80%A9&amp;code=41907445&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">叶倩</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E7%94%B0%E4%BC%9F&amp;code=43252173&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">田伟</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E4%BE%AF%E5%B0%8F%E6%96%87&amp;code=43252174&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">侯小文</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%88%98%E7%BE%8E%E5%A8%9C&amp;code=07019596&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">刘美娜</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%93%88%E5%B0%94%E6%BB%A8%E5%8C%BB%E7%A7%91%E5%A4%A7%E5%AD%A6%E5%8D%AB%E7%94%9F%E7%BB%9F%E8%AE%A1%E5%AD%A6%E6%95%99%E7%A0%94%E5%AE%A4&amp;code=0161188&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">哈尔滨医科大学卫生统计学教研室</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p><b>目的</b> 探索基于压缩感知理论变量筛选方法在小样本量蛋白质组学研究中应用的效果和特点,为小样本量的蛋白质组学的变量筛选提供更灵敏、可靠的方法。<b>方法</b> 模拟实验比较基于CS理论的变量筛选方法与偏最小二乘(PLS)及随机森林(RF)筛选变量的能力,通过灵敏度、特异度及平衡准确度评价其变量筛选效果;利用CS变量筛选方法筛选非小细胞肺癌两亚型组(腺癌和鳞状细胞癌)的差异蛋白。<b>结果</b> 模拟实验表明,CS理论的变量筛选方法在样本量较小时具有较好的变量筛选效果,灵敏度、特异度及平衡准确度均较高;利用基于CS理论的变量筛选方法筛选,获得肺腺癌和鳞状细胞癌间差异表达蛋白22种,被证明是肺腺癌和鳞状细胞癌间有差异的蛋白为:Cytokeratin 6A、Cytokeratin 6B、Cytokeratin 6C、PKP1、P63、MCT1。<b>结论</b> 基于CS理论的变量筛选方法在样本量特别少时,筛选变量的效果优于PLS和RF,更适用于小样本蛋白质组学数据变量筛选研究。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8E%8B%E7%BC%A9%E6%84%9F%E7%9F%A5&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">压缩感知;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%BB%84%E5%AD%A6&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蛋白质组学;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8F%98%E9%87%8F%E7%AD%9B%E9%80%89&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">变量筛选;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%B0%8F%E6%A0%B7%E6%9C%AC&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">小样本;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    *刘美娜,E-mail:liumeina369@163.com;
                                </span>
                    </p>
                
                    <p>

                            <b>基金：</b>
                                                        <span>国家重大专项课题(2016ZX08011005);</span>
                                <span>国家自然基金青年基金(81502889);</span>
                                <span>2017年度哈尔滨医科大学创新科学研究基金(2017JCZX22);</span>
                    </p>
            </div>
                    <h1><b>Application of Compressive Sensing Theory in the Study of Small Sample Proteomics Variables Selection</b></h1>
                    <h2>
                    <span>Zhang Wei</span>
                    <span>Zhang Qiuju</span>
                    <span>Wang Yupeng</span>
            </h2>
                    <h2>
                    <span>Department of Medical Statistics,Harbin Medical University</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p><b>Objective</b> To explore the effect and characteristics of the application of the CS theory variable selection method in the small sample size proteomics research,and to provide a more sensitive and reliable method for small sample size proteomics.<b>Methods</b> Compare the ability of variable selection method based on CS theory with partial least squares(PLS)and random forest(RF)to select variables by simulation experiment.Using the sensitivity,specificity and the balance accuracy to evaluate the effect of variable selection.The differential proteins of two subtypes(adenocarcinoma and squamous cell carcinoma)of non-small cell lung cancer were selected by CS variable selection method.<b>Results</b> Simulation experiment shows that the CS variable selection method has a good effect in the variable selection when the sample size is small with higher sensitivity,specificity and balance accuracy.There are 22 differentially expressed proteins between lung adenocarcinoma and squamous cell carcinoma by using the CS variable selection method.Cytokeratin 6 A,Cytokeratin 6 B,Cytokeratin 6 C,PKP1,P63,MCT1 have been proved to be the different proteins among them.<b>Conclusion</b> The CS variable selection method is better than PLS and RF when the sample size is very small,which is more suitable for small sample proteomics data variable selection research.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Compressive%20sensing&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Compressive sensing;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Proteomics&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Proteomics;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Variable%20selection&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Variable selection;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Small%20sample%20size&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Small sample size;</a>
                </p>
            </div>
        

        <!--brief start-->
                        <div class="p1">
                    <p id="32">目前使用的蛋白质组学定量技术多以质谱为基础,主要分成两类:第一种是稳定同位素标记的定量蛋白质组学(如iTRAQ、TMT);第二种是非标记的定量蛋白质组学技术即label-free。通过质谱技术获得高维的蛋白质组学数据,可利用单变量或多变量等统计学分析方法筛选患者和健康对照之间的差异蛋白质。由于蛋白质之间的相互作用,单变量特征筛选方法(如<i>t</i>检验、ANOVA等)会忽略变量之间的相关性,损失重要的生物学信息,同时存在多重比较问题;目前常用的多变量特征筛选方法有偏最小二乘(PLS)、随机森林(RF)等,能够考虑到变量之间的多重相关性,但蛋白质组学的检测费用昂贵,尤其是当研究某些罕见疾病时,样本量通常很小 (有时样本量不足10),此时PLS、RF等方法筛选差异变量的能力可能受到限制<citation id="83" type="reference"><link href="4" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>。因此本研究介绍一种基于压缩感知(compressive sensing,CS)理论变量筛选方法,通过模拟实验,比较基于CS的变量筛选方法和PLS、RF在小样本蛋白质组学数据变量筛选研究中的效果;并将该方法应用于实际数据,进行肺腺癌和鳞状细胞癌差异蛋白筛选。</p>
                </div>
                <h3 id="33" name="33" class="anchor-tag">CS原理与方法</h3>
                <h4 class="anchor-tag" id="34" name="34">1.CS理论简介</h4>
                <div class="p1">
                    <p id="35">Candés和Donoho在相关研究的基础上于2006年正式提出了压缩感知的概念,为信号采集技术带来了革命性的突破<citation id="84" type="reference"><link href="6" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>。CS理论的主要原理是只要信号在某个变换域是稀疏的,就可以用一个与变换基无关的测量矩阵将稀疏的高维变换域信号投影到低维空间,通过优化求解从低维空间以高概率重构出原信号,极大地降低了存储空间和计算复杂度。</p>
                </div>
                <div class="p1">
                    <p id="36">CS理论将信号采样与压缩相结合,在信号处理领域应用广泛。Wang A等人将可配置的节能压缩感知结构应用于人体传感网络,解决了人体传感网络应用中无线电通讯部分耗能大的问题<citation id="86" type="reference"><link href="8" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>;在雷达信号处理方面,Tivive FHC等人提出了一种基于多重测量向量压缩感知模型的复值信号压缩感知方法,并将其应用于压缩传感穿墙雷达成像问题<citation id="87" type="reference"><link href="10" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>;在医学上压缩感知理论用于核磁共振成像,降低噪声信号干扰<citation id="88" type="reference"><link href="12" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>,也用于CT断层扫描以缩短扫描时间,降低辐射剂量<citation id="89" type="reference"><link href="14" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>。</p>
                </div>
                <h4 class="anchor-tag" id="37" name="37">2.基于CS理论的变量筛选方法</h4>
                <h4 class="anchor-tag" id="38" name="38">(1)基于CS理论变量筛选的基本思想</h4>
                <div class="p1">
                    <p id="39">CS的主要思想是从线性测量<i>y</i>=<i>Aω</i>中重构未知向量<i>ω</i>,在高维蛋白质组学中,<i>A</i>∈R<sup><i>n</i></sup><sup>×</sup><sup><i>d</i></sup>是由包含<i>n</i>个样本和<i>d</i>个变量的高维蛋白质组学质谱数据构成的测量矩阵,<i>y</i>∈R<sup><i>n</i></sup>为由<i>n</i>个样本的应变量测量值组成的向量,通过测量矩阵<i>A</i>和测量值<i>y</i>重构向量<i>ω</i>。</p>
                </div>
                <div class="p1">
                    <p id="40">现阶段CS的重构算法大致可以分为以下几类:第一类是贪婪迭代算法,该类算法基本原则就是通过迭代的方式寻找稀疏向量的支撑集,并使用受限支撑最小二乘估计来重构信号,计算速度快但是需要的测量数据多且精度低;第二类是凸优化算法,这类方法通过将非凸问题转化为凸问题求解找到信号的逼近,其中最常用的方法为基追踪算法,该类算法计算速度慢,但需要的测量数据少且精度高;第三类算法是基于贝叶斯框架提出的重构算法,该类算法考虑到了信号的时间相关性,特别是当信号具有较强的时间相关性时,能够提供比其他重构算法更优越的重构精度<citation id="90" type="reference"><link href="16" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>。本研究的目的是利用CS理论筛选两组之间差异表达的蛋白质,未涉及时间相关性问题,且蛋白质组学研究中样本量通常是非常小的,所以选择凸优化基追踪算法作为本研究的CS重构方法。</p>
                </div>
                <div class="p1">
                    <p id="41">CS方法的先验信息是假设向量<i>ω</i>是稀疏的,即其中大部分元素值为0,或者特别小,当样本量<i>n</i>远小于变量数<i>d</i>时,用基追踪方法求<i>y</i>=<i>Aω</i>的最稀疏解<i>ω</i>:</p>
                </div>
                <div class="p1">
                    <p id="42" class="code-formula">
                        <mathml id="42"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mi>arg</mi></mrow><mspace width="0.25em" /><mrow><mi>min</mi></mrow><munder><mstyle mathsize="140%" displaystyle="true"><mrow><mrow><mo stretchy="false">∥</mo><mi>ω</mi><mo stretchy="false">∥</mo></mrow><msub><mrow></mrow><mn>1</mn></msub></mrow></mstyle><mrow><mi>ω</mi><mo>∈</mo><mtext>R</mtext><msup><mrow></mrow><mi>d</mi></msup></mrow></munder><mtext>s</mtext><mtext>u</mtext><mtext>b</mtext><mtext>j</mtext><mtext>e</mtext><mtext>c</mtext><mtext>t</mtext><mspace width="0.25em" /><mtext>t</mtext><mtext>o</mtext><mspace width="0.25em" /><mi>y</mi><mo>=</mo><mi>A</mi><mi>ω</mi><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="43">随着CS及其相关领域研究的发展,新的算法如核范数最小化、1-bit压缩感知等相继被提出,这些方法在理论研究和真实数据研究方面效果都非常好。其中1-bit压缩感知将测量值<i>y</i>进行分类化,研究应变量为二分类的问题,只保留其符号信息:</p>
                </div>
                <div class="p1">
                    <p id="44"><i>y</i><sub><i>i</i></sub>=sign(〈<i>a</i><sub><i>i</i></sub>,<i>ω</i>〉),<i>i</i>=1,…,<i>n</i>       (2)</p>
                </div>
                <div class="p1">
                    <p id="45">其中<i>a</i><sub>1</sub>,…,<i>a</i><sub><i>n</i></sub>∈R<sup><i>d</i></sup>是测量矩阵<i>A</i>∈R<sup><i>n</i></sup><sup>×</sup><sup><i>d</i></sup>每一个样本的各变量值组成的向量。</p>
                </div>
                <h4 class="anchor-tag" id="46" name="46">(2)基于CS理论筛选变量的步骤</h4>
                <div class="p1">
                    <p id="47">原始数据的样本量为<i>n</i>,每个样本预处理后蛋白质组学质谱数据<i>x</i><sub><i>i</i></sub>∈R<sup><i>d</i></sup>,<i>i</i>=1,…,<i>n</i>由其<i>d</i>个变量相应的强度值组成的向量进行归一化、平滑化及标准化得到,已知每个样本的分类标签<i>y</i><sub><i>i</i></sub>∈{-1,+1},<i>i</i>=1,…,<i>n</i>。基于CS的变量筛选方法实际上就是要重构出能够将两组正确分开的稀疏向量<i>ω</i>,并找到其中非零元素所在位置的集合。</p>
                </div>
                <div class="p1">
                    <p id="48">①对原始质谱数据进行预处理,包括归一化、平滑化及标准化,得到预处理后蛋白质组学数据<i>x</i><sub><i>i</i></sub>∈R<sup><i>d</i></sup>,<i>i</i>=1,…,<i>n</i>;</p>
                </div>
                <div class="p1">
                    <p id="49">②构建线性决定函数<i>f</i><sub><i>ω</i></sub>(<i>x</i><sub><i>i</i></sub>):向量<i>ω</i>=(<i>ω</i><sub>1</sub>,…,<i>ω</i><sub><i>d</i></sub>)∈R<sup><i>d</i></sup>为目标稀疏向量,在空间中,以<i>ω</i>为法向量的超平面可以适当的将不同类别的样本点分开,从数学角度可以理解为向量<i>ω</i>可以使大多数样本满足<i>y</i><sub><i>i</i></sub>=sign(<i>f</i><sub><i>ω</i></sub>(<i>x</i><sub><i>i</i></sub>)),<i>i</i>=1,…,<i>n</i>,其中<i>f</i><sub><i>ω</i></sub>(<i>x</i><sub><i>i</i></sub>)为线性决定函数,<i>f</i><sub><i>ω</i></sub>(<i>x</i><sub><i>i</i></sub>)=〈<i>ω</i>,<i>x</i><sub><i>i</i></sub>〉=∑<sup><i>d</i></sup><sub><i>j</i></sub><sub>=1</sub><i>ω</i><sub><i>j</i></sub><i>x</i><sub><i>i</i></sub><sub>,</sub><sub><i>j</i></sub>,<i>i</i>=1,…,<i>n</i>,<i>j</i>=1,…,<i>d</i>,用于帮助预测样本分类;</p>
                </div>
                <div class="p1">
                    <p id="50">③判定<i>y</i>的分类:根据1-bit压缩感知理论,仅保留样本观测值的符号信息,利用<i>ω</i>对样本进行预测分类:<mathml id="74"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mover accent="true"><mi>y</mi><mo>^</mo></mover><msub><mrow></mrow><mi>i</mi></msub><mo>=</mo><mtext>s</mtext><mtext>i</mtext><mtext>g</mtext><mtext>n</mtext></mrow></math></mathml>(〈<i>x</i><sub><i>i</i></sub>,<i>ω</i>〉),<i>i</i>=1,…,<i>n</i>,当<mathml id="75"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mo>〈</mo><mover accent="true"><mi>x</mi><mo>⌣</mo></mover><msub><mrow></mrow><mi>i</mi></msub><mo>,</mo><mi>ω</mi><mo>〉</mo><mo>&gt;</mo><mn>0</mn></mrow></math></mathml>时,<mathml id="76"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mover accent="true"><mi>y</mi><mo>^</mo></mover><msub><mrow></mrow><mi>i</mi></msub><mo>=</mo><mn>1</mn><mo>,</mo></mrow></math></mathml>反之,<mathml id="77"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mover accent="true"><mi>y</mi><mo>^</mo></mover><msub><mrow></mrow><mi>i</mi></msub><mo>=</mo><mo>-</mo><mn>1</mn></mrow></math></mathml>。</p>
                </div>
                <div class="p1">
                    <p id="51">④获取稀疏向量<i>ω</i>:如果向量<i>ω</i>能将样本正确分类,那么实际分类标签<i>y</i><sub><i>i</i></sub>与〈<i>x</i><sub><i>i</i></sub>,<i>ω</i>〉符号相同,即<i>y</i><sub><i>i</i></sub>〈<i>x</i><sub><i>i</i></sub>,<i>ω</i>〉&gt;0,如果多数样本都能够被向量<i>ω</i>正确分类,目标函数<mathml id="78"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mtext>i</mtext><mo>=</mo><mn>1</mn></mrow><mtext>n</mtext></munderover><mrow></mrow></mstyle></mrow></math></mathml><mathml id="79"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>y</mi><msub><mrow></mrow><mi>i</mi></msub><mo>〈</mo><mover accent="true"><mi>x</mi><mo>⌣</mo></mover><msub><mrow></mrow><mi>i</mi></msub><mo>,</mo><mi>ω</mi><mo>〉</mo></mrow></math></mathml>的值将会特别大,此时的<i>ω</i>即为可以将两个类别适当分开的向量。为了保证<i>ω</i>的可解释性和泛化能力,对<i>ω</i>的解空间进行L2范数化约束,找到使目标函数值最大的最优解:</p>
                </div>
                <div class="area_img" id="96">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/ZGWT201905003_09600.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="54" class="code-formula">
                        <mathml id="54"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mover accent="true"><mi>ω</mi><mo>^</mo></mover><mo>=</mo><mrow><mo>{</mo><mrow><mtable><mtr><mtd columnalign="left"><mover accent="true"><mi>ω</mi><mo>^</mo></mover><msubsup><mrow></mrow><mtext>k</mtext><mo>´</mo></msubsup><mo>,</mo><mtext>i</mtext><mtext>f</mtext><mrow><mo>|</mo><mrow><mover accent="true"><mi>ω</mi><mo>^</mo></mover><msubsup><mrow></mrow><mtext>k</mtext><mo>´</mo></msubsup></mrow><mo>|</mo></mrow><mo>&gt;</mo><mi>ξ</mi></mtd></mtr><mtr><mtd columnalign="left"><mn>0</mn><mo>,</mo><mtext>o</mtext><mtext>t</mtext><mtext>h</mtext><mtext>e</mtext><mtext>r</mtext><mtext>w</mtext><mtext>i</mtext><mtext>s</mtext><mtext>e</mtext><mo>,</mo></mtd></mtr></mtable></mrow></mrow><mi>k</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>⋯</mo><mo>,</mo><mi>d</mi><mo>.</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="55">其中<i>x</i><sub><i>i</i></sub>为标准化后的质谱数据,<i>y</i><sub><i>i</i></sub>∈{-1,+1},<i>i</i>=1,…,<i>n</i>为样本已知的分类标签,阈值<i>ξ</i>&gt;0,根据欲筛选变量个数的需要人为设置。<mathml id="81"><math xmlns="http://www.w3.org/1998/Math/MathML"><msup><mover accent="true"><mi>ω</mi><mo>^</mo></mover><mo>′</mo></msup></math></mathml>相当于一个线性分类器,其中的每一个元素都对应质谱图中相应的位置,将每个位置元素值的绝对值与预先设定的阈值<i>ξ</i>比较,小于阈值的元素所在位置记为0,得到最终的稀疏向量<mathml id="82"><math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mi>ω</mi><mo>^</mo></mover></math></mathml>,其中的非零元素即为两组之间有差异的特征变量,非零元素值的绝对值相当于变量的重要性。</p>
                </div>
                <div class="p1">
                    <p id="56">上述过程可以用<i>Matlab</i>软件实现。</p>
                </div>
                <h3 id="57" name="57" class="anchor-tag">模拟实验</h3>
                <div class="p1">
                    <p id="58">构建具有相关性的差异变量,加入一定数目的噪声变量,设置不同的样本数,考察基于压缩感知理论变量筛选方法的筛选效果,同时与常用方法PLS及RF进行比较,因为这两种方法是目前高维组学进行变量筛选常用的方法,筛选出的变量有重要性排序<citation id="91" type="reference"><link href="18" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>,在与本研究的变量筛选方法利用平衡准确度比较筛选变量的效果时更有可比性。</p>
                </div>
                <h4 class="anchor-tag" id="59" name="59">1.模拟实验条件设置</h4>
                <div class="p1">
                    <p id="60">设置模拟数据的总样本数分别为<i>N</i>=6,8,10,12,18,24,30,病例组和对照组的样本量相等;样本中总变量数为1000,差异变量比例设为3%、5%、8%、10%;变量之间的相关性设为0.2、0.4、0.6、0.8。不同差异变量比例及变量之间相关性进行组合,抽取符合多元正态分布的数据组成模拟数据,每种情况重复100次,分别利用基于CS的变量筛选方法、PLS和RF进行变量筛选,计算每种方法在各种差异变量比例与变量相关性组合的情况下筛选变量的平均平衡准确度(平衡准确度=(灵敏度+特异度)/2)。</p>
                </div>
                <h4 class="anchor-tag" id="61" name="61">2.模拟实验结果</h4>
                <div class="p1">
                    <p id="62">不同差异变量比例与相关系数的组合有很多,以<i>p</i>=50为例,不同相关系数条件下三种方法筛选变量的效果比较(图1)和以<i>r</i>=0.4为例,不同差异变量比例条件下三种方法筛选变量的效果比较(图2)如图所示。结果显示:在样本量较小时,基于CS理论的变量筛选方法筛选变量的效果均为三种方法中最优,其次为PLS,RF最差,不同差异变量比例以及相关系数条件下均得到同样结果;尤其是当样本量<i>N</i>≤12时,基于CS理论的变量筛选方法优势更为明显;随着样本量增加,其变量筛选效果与PLS越来越接近。</p>
                </div>
                <div class="area_img" id="63">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201905003_063.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1p=50为例,不同相关系数条件下三种方法筛选变量的效果比较" src="Detail/GetImg?filename=images/ZGWT201905003_063.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图1</b><i>p</i>=50为例,不同相关系数条件下三种方法筛选变量的效果比较  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201905003_063.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="64">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/ZGWT201905003_064.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2r=0.4为例,不同差异变量比例条件下三种方法筛选变量的效果比较" src="Detail/GetImg?filename=images/ZGWT201905003_064.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit"><b>图2</b><i>r</i>=0.4为例,不同差异变量比例条件下三种方法筛选变量的效果比较  <a class="btn-zoomin" href="Detail/GetImg?filename=images/ZGWT201905003_064.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h3 id="65" name="65" class="anchor-tag">实例应用</h3>
                <div class="p1">
                    <p id="66">实例数据分析中,使用的是ProteomeXchange数据库中的PXD002622数据集,用于非小细胞肺癌的两种亚型(腺癌和鳞状细胞癌)间的差异蛋白质筛选研究。其中腺癌(ADC)和鳞状细胞癌(SCC)患者各3例,采集患者组织学样本,利用TMT标记定量蛋白质组学技术及MaxQuant软件,鉴定出51001个多肽和7241个蛋白质。将基于CS理论的变量筛选方法应用于上述数据,阈值设为0.09,筛选出22个ADC和SCC之间的差异蛋白质,表1为其相对应的蛋白质名称。</p>
                </div>
                <div class="area_img" id="67">
                    <p class="img_tit"><b>表1</b> 筛选出的22个差异蛋白质鉴定结果 <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="67" border="1"><tr><td><br />No</td><td>Uniprot ID</td><td>Protein</td></tr><tr><td><br />1</td><td>Q86SG5</td><td>S100 calcium-binding protein A7A</td></tr><tr><td><br />2</td><td>P52926</td><td>High mobility group protein HMGI-C</td></tr><tr><td><br />3</td><td>P31151</td><td>Psoriasin</td></tr><tr><td><br />4</td><td>Q14145</td><td>Kelch-like ECH-associated protein 1</td></tr><tr><td><br />5</td><td>P09238</td><td>Stromelysin-2</td></tr><tr><td><br />6</td><td>P48668</td><td>Cytokeratin 6C</td></tr><tr><td><br />7</td><td>P02538</td><td>Cytokeratin 6A</td></tr><tr><td><br />8</td><td>Q14134</td><td>Tripartite motif-containing protein 29</td></tr><tr><td><br />9</td><td>O95832</td><td>Claudin-1</td></tr><tr><td><br />10</td><td>P04259</td><td>Cytokeratin 6B</td></tr><tr><td><br />11</td><td>Q9UMD9</td><td>Collagen alpha-1(XVII)chain</td></tr><tr><td><br />12</td><td>Q13835</td><td>PKP1</td></tr><tr><td><br />13</td><td>Q9H3D4</td><td>P63</td></tr><tr><td><br />14</td><td>A6NDB9</td><td>Paralemmin-3</td></tr><tr><td><br />15</td><td>P29508</td><td>Serpin B3</td></tr><tr><td><br />16</td><td>Q96C23</td><td>Aldose 1-epimerase</td></tr><tr><td><br />17</td><td>O95049</td><td>Tight junction protein ZO-3</td></tr><tr><td><br />18</td><td>P16383</td><td>GC-rich sequence DNA-binding factor 2</td></tr><tr><td><br />19</td><td>O15350</td><td>P73</td></tr><tr><td><br />20</td><td>Q9UM07</td><td>Protein-arginine deiminase type-4</td></tr><tr><td><br />21</td><td>P09467</td><td>Fructose-1,6-bisphosphatase 1</td></tr><tr><td><br />22</td><td>P53985</td><td>MCT1</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="68">P63是一种肿瘤蛋白,在Terry J等<citation id="92" type="reference"><link href="20" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>的研究中,用单个生物标志物区分ADC和SCC时,P63是最有意义的(灵敏度为84%,特异度为85%);PKP1为血小板亲和蛋白1,Schwarz J等<citation id="93" type="reference"><link href="22" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>认为它是SCC中的特异表达标志物,其表达水平与癌症的恶性程度成反比;Cytokeratin 6A、Cytokeratin 6B、Cytokeratin 6C是常用的肿瘤免疫组织化学标记物,在本研究中ADC和SCC之间的表达差异与Terry J等<citation id="95" type="reference"><link href="20" rel="bibliography" /><link href="24" rel="bibliography" /><link href="26" rel="bibliography" /><link href="28" rel="bibliography" /><sup>[<a class="sup">9</a>,<a class="sup">11</a>,<a class="sup">12</a>,<a class="sup">13</a>]</sup></citation>研究的结果一致;MCT1为单羧酸转运蛋白1,是一类跨膜转运蛋白,涉及多种生物学功能,包括促进营养物质吸收、影响代谢动态平衡、调节胞内pH值以及参与药物输送等,Stewart PA等<citation id="94" type="reference"><link href="30" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>认为MCT1可能是腺癌和鳞状细胞癌的潜在诊断标志物及药物治疗靶向蛋白。</p>
                </div>
                <h3 id="69" name="69" class="anchor-tag">讨论与结论</h3>
                <div class="p1">
                    <p id="70">本方法不仅能应用于蛋白质组学数据,对于具有相似数据结构的其他具有小样本量(尤其是<i>N</i>≤12当时)的高维组学数据(如转录组学、代谢组学等)的差异变量筛选也具有一定的应用意义,为小样本量高维组学数据的研究提供了新思路。</p>
                </div>
                <div class="p1">
                    <p id="71">本研究基于CS理论的变量筛选方法的核心是解决一个有约束的最优化问题,本文的约束条件使用的是L2范数,实际上也可以根据分析需要,使用L1范数或L1范数与L2范数结合的约束条件对本方法进行优化。</p>
                </div>
                <div class="p1">
                    <p id="72">本文模拟小样本量蛋白质组学变量筛选研究,在样本量小时(尤其是当<i>N</i>≤12时),基于CS的变量筛选方法筛选差异变量的能力优于PLS和RF,随着样本量增加,其变量筛选效果逼近PLS,在不同的差异变量比例及相关系数条件下结果一致。模拟实验为了验证小样本量时三种变量筛选方法的效果,设置的最大样本量为30,所以当继续增大样本量时本方法与PLS的优劣还需要进一步研究。</p>
                </div>
                <div class="p1">
                    <p id="73">本文用基于CS理论的变量筛选方法分析了非小细胞肺癌数据,筛选出的差异蛋白集合中变量数目少,同时又包含了Cytokeratin 6A、Cytokeratin 6B、Cytokeratin 6C、PKP1、P63、MCT1等已经被验证是ADC与SCC之间差异表达的蛋白质。其中Cytokeratin 6系列是常用的肿瘤免疫组织化学标记物;PKP1为血小板亲和蛋白,与表皮的形态形成有关;P63是肿瘤蛋白63,由一段结合转录激活剂或抑制剂的特异DNA序列翻译而来;MCT1为单羧酸转运蛋白1,是一类跨膜转运蛋白,涉及多种生物学功能,包括促进营养物质吸收、影响代谢动态平衡、调节胞内pH值以及参与药物输送等。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="4">
                            <a id="bibliography_1" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD00001101437&amp;v=MTg4MjBhck80SHRITnJvOUVZT2dJWTNrNXpCZGg0ajk5U1hxUnJ4b3hjTUg3UjdxZForWnVGaUhsVzd6T0pGYz1OaWZj&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[1]</b> Rai AJ,Chan DW.Cancer Proteomics:Serum Diagnostics for Tumor Marker Discovery.Annals of the New York Academy of Sciences,2004,1022(1):286-294.
                            </a>
                        </p>
                        <p id="6">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Compressed sensing">

                                <b>[2]</b> Donoho DL.Compressed sensing.IEEE Transactions on Information Theory,2006,52(4):1289-1306.
                            </a>
                        </p>
                        <p id="8">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A Configurable Energy-Efficient Compressed Sensing Architecture With Its Application on Body Sensor Networks">

                                <b>[3]</b> Wang A,Lin F,Jin Z,et al.A Configurable Energy-Efficient Compressed Sensing Architecture With Its Application on Body Sensor Networks.IEEE Transactions on Industrial Informatics,2017,12(1):15-27.
                            </a>
                        </p>
                        <p id="10">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A compressed sensing method for complex-valued signals with application to through-the-wall radar imaging">

                                <b>[4]</b> Tivive FHC,Bouzerdoum A.A compressed sensing method for complex-valued signals with application to through-the-wall radar imaging.IEEE International Conference on Acoustics,Speech and Signal Processing.IEEE,2013:2144-2148.
                            </a>
                        </p>
                        <p id="12">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Sparse MRI: The application of compressed sensing for rapid MR imaging">

                                <b>[5]</b> Lustig M,Santos JM,Lee JH,et al.Application of “Compressed Sensing” for Rapid MR Imaging.Spars,2010,58(6):1182-1195.
                            </a>
                        </p>
                        <p id="14">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A compressed sensing-based iterative algorithm for CT reconstruction and its possible application to phase contrast imaging">

                                <b>[6]</b> Li X,Luo S.A compressed sensing-based iterative algorithm for CT reconstruction and its possible application to phase contrast imaging.BioMedical Engineering OnLine,2011,10(1):73.
                            </a>
                        </p>
                        <p id="16">
                            <a id="bibliography_7" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=HWYJ2013S1044&amp;v=MjM5MjRSN3FmWnVabkZ5RG1WYnZCTFRyU1pMRzRIOUt2cm85QllJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1U=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[7]</b> 李珅,马彩文,李艳,等.压缩感知重构算法综述.红外与激光工程,2013,42(s1):225-232.
                            </a>
                        </p>
                        <p id="18">
                            <a id="bibliography_8" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201402052&amp;v=MDM4ODJDVVI3cWZadVpuRnlEbVZidkJQeXJjZXJHNEg5WE1yWTlBWm9RS0RIODR2UjRUNmo1NE8zenFxQnRHRnI=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[8]</b> 柯朝甫,张涛,武晓岩,等.代谢组学数据分析的统计学方法.中国卫生统计,2014,31(2):357-359.
                            </a>
                        </p>
                        <p id="20">
                            <a id="bibliography_9" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWK&amp;filename=SJWK121016063204&amp;v=MTY4MDZFOWZidm5LcmlmWmVadkZ5bnNVN2ZMSmxvZE5pZmNaYks2SDlITnFZOURaK2tQQ0JNOHp4VVNtRGQ5U0g3bjN4&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[9]</b> Terry J,Leung S,Laskin J,et al.Optimal immunohistochemical markers for distinguishing lung adenocarcinomas from squamous cell carcinomas in small tumor samples.Am J Surg Pathol,2010,34:1805-1811.
                            </a>
                        </p>
                        <p id="22">
                            <a id="bibliography_10" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011600344146&amp;v=MjUyOTAwU2Jocz1OaWZPZmJLN0h0RE5xWTlGWis4TERYZy9vQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp0RmlubFVyYklLRg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[10]</b> Schwarz J,Ayim AA,Jager S,et al.Differential expression of desmosomal plakophilins in various types of carcinomas:correlation with cell type and differentiation.Human Pathology,2006,37(5):613-622.
                            </a>
                        </p>
                        <p id="24">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Aberrant Signaling Pathways in Squamous Cell Lung Carcino-ma">

                                <b>[11]</b> Shi I,Hashemi Sadraei N,Duan ZH,et al.Aberrant Signaling Pathways in Squamous Cell Lung Car-cinoma.Cancer Inform,2011,10:273-285.
                            </a>
                        </p>
                        <p id="26">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Immunohistochemical algorithm for differentiation of lung adenocarcinoma and squamous cell carcinoma based on large series of whole-tissue sections with validation in small specimens">

                                <b>[12]</b> Rekhtman N,Ang DC,Sima CS,et al.Immunohistochemical algorithm for differentia-tion of lung adenocarcinoma and squamous cell carcinoma based on large series of whole-tissue sections with validation in small specimens.Mod Pathol Off J U S Can Acad Pathol Inc,2011,24:1348-1359.
                            </a>
                        </p>
                        <p id="28">
                            <a id="bibliography_13" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD13062800007835&amp;v=MjY0MzV0RmlubFVyYklLRjBTYmhzPU5pZmNhcks3SHRmT3A0OUZaT3NJQkg4OG9CTVQ2VDRQUUgvaXJSZEdlcnFRVE1ud1plWg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[13]</b> Gómez-Morales M,Cámara-Pulido M,Miranda-León MT,et al.Differential immunohistochemical localization of desmosomal plaque-related proteins in non-small-cell lung cancer.Histopathology,2013,63:103-113.
                            </a>
                        </p>
                        <p id="30">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A Pilot Proteogenomic Study with Data Integration Identifies MCT1 and GLUT1 as Prognostic Markers in Lung Adenocarcinoma">

                                <b>[14]</b> Stewart PA,Parapatics K,Welsh EA,et al.A Pilot Proteogenomic Study with Data Integration Identifies MCT1 and GLUT1 as Prognostic Markers in Lung Adenocarcinoma.Plos One,2015,10(11):e0142162.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="ZGWT201905003" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>


    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZGWT201905003&amp;v=MjEwNTFuRnlEbVZidkJQeXJjZXJHNEg5ak1xbzlGWjRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVo=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZEYUp6R0R1dU16ZHF4cmNVV1BTYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="0" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=842_VlRz_g1CMCLi-M0twVw-4EOFG_noWyi_OsMnrAg1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>


    <link href="/kxreader/Content/css/LeftDetail?v=vAscMyvIPP9NePnbGPkqJ0A5tHOvnzl65tGuRDsf9xg1" rel="stylesheet"/>

</body>
</html>

