<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637136444616221250%26DBCODE%3dCJFD%26TABLEName%3dCJFDTEMP%26FileName%3dJSJY201911008%26RESULT%3d1%26SIGN%3dg6mJ8kEugBtS3ZCksnPydlaREpA%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJY201911008&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJY201911008&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJY201911008&amp;v=MDYwMzA1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlublZyL09MejdCZDdHNEg5ak5ybzlGYklRS0RIODR2UjRUNmo=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#63" data-title="0 引言 ">0 引言</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#69" data-title="1 数据集 ">1 数据集</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#72" data-title="2 蛋白质残基的特征表示 ">2 蛋白质残基的特征表示</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#74" data-title="2.1 &lt;b&gt;特征选取&lt;/b&gt;">2.1 <b>特征选取</b></a></li>
                                                <li><a href="#82" data-title="2.2 &lt;b&gt;随机下采样&lt;/b&gt;">2.2 <b>随机下采样</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#84" data-title="3 卷积神经网络模型的构建和训练 ">3 卷积神经网络模型的构建和训练</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#85" data-title="3.1 &lt;b&gt;网络模型结构&lt;/b&gt;">3.1 <b>网络模型结构</b></a></li>
                                                <li><a href="#101" data-title="3.2 &lt;b&gt;模型训练&lt;/b&gt;">3.2 <b>模型训练</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#106" data-title="4 实验结果和比较分析 ">4 实验结果和比较分析</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#107" data-title="4.1 &lt;b&gt;评判标准&lt;/b&gt;">4.1 <b>评判标准</b></a></li>
                                                <li><a href="#117" data-title="4.2 &lt;b&gt;实验结果&lt;/b&gt;">4.2 <b>实验结果</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#125" data-title="5 讨论 ">5 讨论</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#126" data-title="5.1 &lt;b&gt;一维卷积神经网络还是二维卷积神经网络&lt;/b&gt;">5.1 <b>一维卷积神经网络还是二维卷积神经网络</b></a></li>
                                                <li><a href="#133" data-title="5.2 &lt;b&gt;深度卷积神经网络还是普通卷积神经网络&lt;/b&gt;">5.2 <b>深度卷积神经网络还是普通卷积神经网络</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#138" data-title="6 结语 ">6 结语</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#71" data-title="&lt;b&gt;表&lt;/b&gt;1 &lt;b&gt;训练集和独立测试集的构成&lt;/b&gt;"><b>表</b>1 <b>训练集和独立测试集的构成</b></a></li>
                                                <li><a href="#87" data-title="图1 神经网络模型的基本结构">图1 神经网络模型的基本结构</a></li>
                                                <li><a href="#109" data-title="&lt;b&gt;表&lt;/b&gt;2 &lt;b&gt;混淆矩阵&lt;/b&gt;"><b>表</b>2 <b>混淆矩阵</b></a></li>
                                                <li><a href="#119" data-title="图2 两层卷积层300卷积核的&lt;i&gt;CNN&lt;/i&gt;模型的训练过程">图2 两层卷积层300卷积核的<i>CNN</i>模型的训练过程</a></li>
                                                <li><a href="#123" data-title="&lt;b&gt;表&lt;/b&gt;3 &lt;b&gt;所提方法和其他预测器 在独立测试集上的性能对比 &lt;/b&gt;"><b>表</b>3 <b>所提方法和其他预测器 在独立测试集上的性能对比 </b></a></li>
                                                <li><a href="#130" data-title="图3 一维卷积网络和二维卷积网络的对比">图3 一维卷积网络和二维卷积网络的对比</a></li>
                                                <li><a href="#136" data-title="图4 不同结构的卷积神经网络的对比">图4 不同结构的卷积神经网络的对比</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="163">


                                    <a id="bibliography_1" title=" GAO M,SKOLNICK J.The distribution of ligand-binding pockets around protein-protein interfaces suggests a general mechanism for pocket formation[J].Proceedings of the National Academy of Sciences of the United States of America,2012,109(10):3784-3789." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The distribution of ligand-binding pockets around protein-protein interfaces suggests a general mechanism for pocket formation">
                                        <b>[1]</b>
                                         GAO M,SKOLNICK J.The distribution of ligand-binding pockets around protein-protein interfaces suggests a general mechanism for pocket formation[J].Proceedings of the National Academy of Sciences of the United States of America,2012,109(10):3784-3789.
                                    </a>
                                </li>
                                <li id="165">


                                    <a id="bibliography_2" title=" TURTON D A,SENN H M,HARWOOD T,et al.Terahertz underdamped vibrational motion governs protein-ligand binding in solution [EB/OL].[2018- 11- 20].http://europacat.co.uk/staff/wynne/pubs/r/2014-NatComm-proteins.pdf." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Terahertz underdamped vibrational motion governs protein-ligand binding in solution">
                                        <b>[2]</b>
                                         TURTON D A,SENN H M,HARWOOD T,et al.Terahertz underdamped vibrational motion governs protein-ligand binding in solution [EB/OL].[2018- 11- 20].http://europacat.co.uk/staff/wynne/pubs/r/2014-NatComm-proteins.pdf.
                                    </a>
                                </li>
                                <li id="167">


                                    <a id="bibliography_3" title=" SIRIMULLA S,BAILEY J B,VEGESNA R,et al.Halogen interactions in protein-ligand complexes:implications of halogen bonding for rational drug design [J].Journal of Chemical Information and Modeling,2013,53(11):2781-2791." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Halogen Interactions in Protein-Ligand Complexes: Implications of Halogen Bonding for Rational Drug Design">
                                        <b>[3]</b>
                                         SIRIMULLA S,BAILEY J B,VEGESNA R,et al.Halogen interactions in protein-ligand complexes:implications of halogen bonding for rational drug design [J].Journal of Chemical Information and Modeling,2013,53(11):2781-2791.
                                    </a>
                                </li>
                                <li id="169">


                                    <a id="bibliography_4" title=" AMARI S,AIZAWA M,ZHANG J,et al.VISCANA:visualized cluster analysis of protein-ligand interaction based on the ab initio fragment molecular orbital method for virtual ligand screening[J].Journal of Chemical Information and Modeling,2006,46(1):221-230." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=VISCANA: Visualized cluster analysis of protein - Ligand interaction based on the ab initio fragment molecular orbital method for virtual ligand screening">
                                        <b>[4]</b>
                                         AMARI S,AIZAWA M,ZHANG J,et al.VISCANA:visualized cluster analysis of protein-ligand interaction based on the ab initio fragment molecular orbital method for virtual ligand screening[J].Journal of Chemical Information and Modeling,2006,46(1):221-230.
                                    </a>
                                </li>
                                <li id="171">


                                    <a id="bibliography_5" title=" BILOFSKY H S,BURKS C.The GenBank genetic sequence data bank[J].Nucleic Acids Research,1988,16(5):1861-1863." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The GenBank genetic sequence data bank">
                                        <b>[5]</b>
                                         BILOFSKY H S,BURKS C.The GenBank genetic sequence data bank[J].Nucleic Acids Research,1988,16(5):1861-1863.
                                    </a>
                                </li>
                                <li id="173">


                                    <a id="bibliography_6" title=" LEVITT D G,BANASZAK L J.POCKET:a computer graphics method for identifying and displaying protein cavities and their surrounding amino acids[J].Journal of Molecular Graphics,1992,10(4):229-234." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=POCKET: a computer graphics method for identifying and displaying protein cavities and their surrounding amino acids">
                                        <b>[6]</b>
                                         LEVITT D G,BANASZAK L J.POCKET:a computer graphics method for identifying and displaying protein cavities and their surrounding amino acids[J].Journal of Molecular Graphics,1992,10(4):229-234.
                                    </a>
                                </li>
                                <li id="175">


                                    <a id="bibliography_7" title=" LASKOWSKI R A.SURFNET:a program for visualizing molecular surfaces,cavities,and intermolecular interactions[J].Journal of Molecular Graphics,1995,13(5):323-330." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13012201343676&amp;v=MTYxNjVlWnRGaW5sVXIzSUlWd1JhaFE9TmlmT2ZiSzdIdERPclk5RVorOE1DbnMvb0JNVDZUNFBRSC9pclJkR2VycVFUTW53Wg==&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[7]</b>
                                         LASKOWSKI R A.SURFNET:a program for visualizing molecular surfaces,cavities,and intermolecular interactions[J].Journal of Molecular Graphics,1995,13(5):323-330.
                                    </a>
                                </li>
                                <li id="177">


                                    <a id="bibliography_8" title=" CHEN K,MIZIANTY M J,KURGAN L.ATPsite:sequence-based prediction of ATP-binding residues[J].Proteome Science,2011,9(S1):S4." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=ATPsite: sequence-based prediction of ATP-binding residues">
                                        <b>[8]</b>
                                         CHEN K,MIZIANTY M J,KURGAN L.ATPsite:sequence-based prediction of ATP-binding residues[J].Proteome Science,2011,9(S1):S4.
                                    </a>
                                </li>
                                <li id="179">


                                    <a id="bibliography_9" title=" 石大宏.基于序列的蛋白质—核苷酸绑定位点预测研究[D].南京:南京理工大学,2015.(SHI D H.Sequential protein-GDP binding residues prediction[D].Nanjing:Nanjing University of Science and Technology,2015.)" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CMFD&amp;filename=1015358422.nh&amp;v=MDczNDM0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVac0Z5bm5Wci9PVkYyNkc3QzlGdFhPclpFYlBJUUtESDg=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[9]</b>
                                         石大宏.基于序列的蛋白质—核苷酸绑定位点预测研究[D].南京:南京理工大学,2015.(SHI D H.Sequential protein-GDP binding residues prediction[D].Nanjing:Nanjing University of Science and Technology,2015.)
                                    </a>
                                </li>
                                <li id="181">


                                    <a id="bibliography_10" title=" JONES D T,WARD J J.Prediction of disordered regions in proteins from position specific score matrices[J].Proteins-Structure Function and Bioinformatics,2003,53(S6):573-578." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prediction of disordered regions in proteins from position specific score matrices">
                                        <b>[10]</b>
                                         JONES D T,WARD J J.Prediction of disordered regions in proteins from position specific score matrices[J].Proteins-Structure Function and Bioinformatics,2003,53(S6):573-578.
                                    </a>
                                </li>
                                <li id="183">


                                    <a id="bibliography_11" title=" LAURIE A T R,JACKSON R M.Q-SiteFinder:an energy-based method for the prediction of protein-ligand binding sites [J].Bioinformatics,2005,21(9):1908-1916." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Q-SiteFinder: an energy-based method for the prediction of protein-ligand binding sites">
                                        <b>[11]</b>
                                         LAURIE A T R,JACKSON R M.Q-SiteFinder:an energy-based method for the prediction of protein-ligand binding sites [J].Bioinformatics,2005,21(9):1908-1916.
                                    </a>
                                </li>
                                <li id="185">


                                    <a id="bibliography_12" title=" ZHANG Y,SKOLNICK J.TM-align:a protein structure alignment algorithm based on the TM-score[J].Nucleic Acids Research,2005,33(7):2302-2309." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=TM-align: a protein structure alignment algorithm based on the TM-score">
                                        <b>[12]</b>
                                         ZHANG Y,SKOLNICK J.TM-align:a protein structure alignment algorithm based on the TM-score[J].Nucleic Acids Research,2005,33(7):2302-2309.
                                    </a>
                                </li>
                                <li id="187">


                                    <a id="bibliography_13" title=" 杨骥.基于序列与结构特征结合的蛋白质与DNA绑定位点预测[J].计算机与现代化,2016(1):20-25.(YANG J.Prediction of DNA-protein binding sites based on combining sequence with structure information[J].Computer and Modernization,2016(1):20-25.)" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYXH201601005&amp;v=MTYyMDhIOWZNcm85RllZUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVac0Z5bm5Wci9PTHpUVFpyRzQ=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[13]</b>
                                         杨骥.基于序列与结构特征结合的蛋白质与DNA绑定位点预测[J].计算机与现代化,2016(1):20-25.(YANG J.Prediction of DNA-protein binding sites based on combining sequence with structure information[J].Computer and Modernization,2016(1):20-25.)
                                    </a>
                                </li>
                                <li id="189">


                                    <a id="bibliography_14" title=" 於东军,胡俊,於铉.基于查询驱动的蛋白质-配体绑定位点预测方法:201310573950.9[P].2014- 03- 05.(YU D J,HU J,YU X.Prediction of protein-ligand binding sites based on query-driven:201310573950.9[P].2014- 03- 05.)" target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SCPD&amp;filename=CN103617203A&amp;v=MTU2NDkrVnVGaXJnVTdrPUppTzZIckMrSDliT3I0dzBDKzRQRDMxTHh4WVQ2em9PUzNmbXBXRmFlN0tXUmJxZVo=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[14]</b>
                                         於东军,胡俊,於铉.基于查询驱动的蛋白质-配体绑定位点预测方法:201310573950.9[P].2014- 03- 05.(YU D J,HU J,YU X.Prediction of protein-ligand binding sites based on query-driven:201310573950.9[P].2014- 03- 05.)
                                    </a>
                                </li>
                                <li id="191">


                                    <a id="bibliography_15" title=" HU J,LI Y,ZHANG Y,et al.ATPbind:accurate protein-ATP binding site prediction by combining sequence-profiling and structure-based comparisons[J].Journal of Chemical Information and Modeling,2018,58(2):501-510." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=ATPbind:accurate protein-ATP binding site prediction by combining sequence-profiling and structure-based comparisons">
                                        <b>[15]</b>
                                         HU J,LI Y,ZHANG Y,et al.ATPbind:accurate protein-ATP binding site prediction by combining sequence-profiling and structure-based comparisons[J].Journal of Chemical Information and Modeling,2018,58(2):501-510.
                                    </a>
                                </li>
                                <li id="193">


                                    <a id="bibliography_16" title=" ALIPANAHI B,DELONG A,WEIRAUCH M T,et al.Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning [J].Nature Biotechnology,2015,33(8):831-838." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Predicting the sequence specificities of DNA-and RNA-binding proteins by deep learning">
                                        <b>[16]</b>
                                         ALIPANAHI B,DELONG A,WEIRAUCH M T,et al.Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning [J].Nature Biotechnology,2015,33(8):831-838.
                                    </a>
                                </li>
                                <li id="195">


                                    <a id="bibliography_17" title=" JIMENEZ J,DOERR S,MARTINEZ-ROSELL G,et al.DeepSite:protein-binding site predictor using 3D-convolutional neural networks [J].Bioinformatics,2017,33(19):3036-3042." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=DeepSite:Protein binding site predictor using 3D-convolutional neural networks">
                                        <b>[17]</b>
                                         JIMENEZ J,DOERR S,MARTINEZ-ROSELL G,et al.DeepSite:protein-binding site predictor using 3D-convolutional neural networks [J].Bioinformatics,2017,33(19):3036-3042.
                                    </a>
                                </li>
                                <li id="197">


                                    <a id="bibliography_18" title=" DESHPANDE N,ADDESS K J,BLUHM W F,et al.The RCSB protein data bank:a redesigned query system and relational database based on the mmCIF schema[J].Nucleic Acids Research,2005,33(S1):D233-D237." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The RCSB Protein Data Bank: a redesigned query system and relational database based on the mmCIF schema">
                                        <b>[18]</b>
                                         DESHPANDE N,ADDESS K J,BLUHM W F,et al.The RCSB protein data bank:a redesigned query system and relational database based on the mmCIF schema[J].Nucleic Acids Research,2005,33(S1):D233-D237.
                                    </a>
                                </li>
                                <li id="199">


                                    <a id="bibliography_19" title=" LI W,GODZIK A.Cd-hit:a fast program for clustering and comparing large sets of protein or nucleotide sequences[J].Bioinformatics,2006,22(13):1658-1659." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences">
                                        <b>[19]</b>
                                         LI W,GODZIK A.Cd-hit:a fast program for clustering and comparing large sets of protein or nucleotide sequences[J].Bioinformatics,2006,22(13):1658-1659.
                                    </a>
                                </li>
                                <li id="201">


                                    <a id="bibliography_20" title=" SCHAFFER A A,ARAVIND L,MADDEN T L,et al.Improving the accuracy of PSI-BLAST protein database searches with composition-based statistics and other refinements[J].Nucleic Acids Research,2001,29(14):2994-3005." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Improving the accuracy of PSI-BLAST protein database searches with composition-based statistics and other refinements">
                                        <b>[20]</b>
                                         SCHAFFER A A,ARAVIND L,MADDEN T L,et al.Improving the accuracy of PSI-BLAST protein database searches with composition-based statistics and other refinements[J].Nucleic Acids Research,2001,29(14):2994-3005.
                                    </a>
                                </li>
                                <li id="203">


                                    <a id="bibliography_21" title=" BAIROCH A,APWEILER R.The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000[J].Nucleic Acids Research,2000,28(1):45-48." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000">
                                        <b>[21]</b>
                                         BAIROCH A,APWEILER R.The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000[J].Nucleic Acids Research,2000,28(1):45-48.
                                    </a>
                                </li>
                                <li id="205">


                                    <a id="bibliography_22" title=" JONES D T.Protein secondary structure prediction based on position-specific scoring matrices[J].Journal of Molecular Biology,1999,292(2):195-202." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011601148133&amp;v=MDMxNDI5RVplOEhEWDg2b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadEZpbmxVcjNJSVZ3UmFoUT1OaWZPZmJLN0h0RE5xWQ==&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[22]</b>
                                         JONES D T.Protein secondary structure prediction based on position-specific scoring matrices[J].Journal of Molecular Biology,1999,292(2):195-202.
                                    </a>
                                </li>
                                <li id="207">


                                    <a id="bibliography_23" title=" JOO K,LEE S J,LEE J.Sann:solvent accessibility prediction of proteins by nearest neighbor method[J].Proteins,2012,80(7):1791-1797." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD120614000788&amp;v=MDYxNDI0U05pZmNhcks2SHRmTnE0OUZaT3dIQkJNOHp4VVNtRGQ5U0g3bjN4RTlmYnZuS3JpZlplWnZGeW5uVTc3S0pW&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[23]</b>
                                         JOO K,LEE S J,LEE J.Sann:solvent accessibility prediction of proteins by nearest neighbor method[J].Proteins,2012,80(7):1791-1797.
                                    </a>
                                </li>
                                <li id="209">


                                    <a id="bibliography_24" title=" YU D J,HU J,TANG Z M,et al.Improving protein-ATP binding residues prediction by boosting SVMs with random under-sampling [J].Neurocomputing,2013,104:180-190." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011501913228&amp;v=MjA2MjFlb01EbjR4b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadEZpbmxVcjNJSVZ3UmFoUT1OaWZPZmJLN0h0RE5xbzlFYg==&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[24]</b>
                                         YU D J,HU J,TANG Z M,et al.Improving protein-ATP binding residues prediction by boosting SVMs with random under-sampling [J].Neurocomputing,2013,104:180-190.
                                    </a>
                                </li>
                                <li id="211">


                                    <a id="bibliography_25" title=" COHEN G,HILARIO M,SAX H,et al.Learning from imbalanced data in surveillance of nosocomial infection [J].Artificial Intelligence in Medicine,2006,37(1):7-18." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011501753849&amp;v=MzIyOThRPU5pZk9mYks3SHRETnFvOUVZKzRNQkhnd29CTVQ2VDRQUUgvaXJSZEdlcnFRVE1ud1plWnRGaW5sVXIzSUlWd1JhaA==&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[25]</b>
                                         COHEN G,HILARIO M,SAX H,et al.Learning from imbalanced data in surveillance of nosocomial infection [J].Artificial Intelligence in Medicine,2006,37(1):7-18.
                                    </a>
                                </li>
                                <li id="213">


                                    <a id="bibliography_26" >
                                        <b>[26]</b>
                                     SRIVASTAVA N,HINTON G,KRIZHEVSKY A,et al.Dropout:a simple way to prevent neural networks from overfitting[J].The Journal of Machine Learning Research,2014,15(1):1929-1958.</a>
                                </li>
                                <li id="215">


                                    <a id="bibliography_27" title=" ABADI M.TensorFlow:learning functions at scale[J].ACM SIGPLAN Notices,2016,51(9):1." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM66BB49F2D1AF8309D25C9A64BB131E06&amp;v=Mjg3MTRQSXB2bEhFT3ArZW5RNnp4OW42RG9PUVE3a3FHQkhlTEdWTUxxWkNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkJyTFUwNXRwaHhicTZ3SzQ9TmlmSVk3VytiSw==&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[27]</b>
                                         ABADI M.TensorFlow:learning functions at scale[J].ACM SIGPLAN Notices,2016,51(9):1.
                                    </a>
                                </li>
                                <li id="217">


                                    <a id="bibliography_28" title=" CHEN K,MIZIANTY M J,KURGAN L.Prediction and analysis of nucleotide-binding residues using sequence and sequence-derived structural descriptors [J].Bioinformatics,2012,28(3):331-341." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prediction and analysis of nucleotide-binding residues using sequence and sequence-derived structural descriptors">
                                        <b>[28]</b>
                                         CHEN K,MIZIANTY M J,KURGAN L.Prediction and analysis of nucleotide-binding residues using sequence and sequence-derived structural descriptors [J].Bioinformatics,2012,28(3):331-341.
                                    </a>
                                </li>
                                <li id="219">


                                    <a id="bibliography_29" title=" YU D J,HU J,HUANG Y,et al.TargetATPsite:a template-free method for ATP-binding sites prediction with residue evolution image sparse representation and classifier ensemble [J].Journal of Computational Chemistry,2013,34(11):974-985." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD130321006336&amp;v=MjY1MThNOHp4VVNtRGQ5U0g3bjN4RTlmYnZuS3JpZlplWnZGeW5uVTc3S0pWNFNOaWZjYXJLN0h0TE9ybzlGWXVnTUNo&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[29]</b>
                                         YU D J,HU J,HUANG Y,et al.TargetATPsite:a template-free method for ATP-binding sites prediction with residue evolution image sparse representation and classifier ensemble [J].Journal of Computational Chemistry,2013,34(11):974-985.
                                    </a>
                                </li>
                                <li id="221">


                                    <a id="bibliography_30" title=" ZENG H,EDWARDS M D,LIU G,et al.Convolutional neural network architectures for predicting DNA-protein binding[J].Bioinformatics,2016,32(12):i121-i127." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Convolutional neural network architectures for predicting DNA–protein binding:">
                                        <b>[30]</b>
                                         ZENG H,EDWARDS M D,LIU G,et al.Convolutional neural network architectures for predicting DNA-protein binding[J].Bioinformatics,2016,32(12):i121-i127.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">

    <div class="head-tag">   
            <p>
               <b> 网络首发时间: 2019-07-23 13:17</b>
            </p>     
    </div>


        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JSJY" target="_blank">计算机应用</a>
                2019,39(11),3146-3150 DOI:10.11772/j.issn.1001-9081.2019050865            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>基于一维卷积神经网络的蛋白质-ATP绑定位点预测</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%BC%A0%E5%AF%93&amp;code=43224060&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">张寓</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%96%BC%E4%B8%9C%E5%86%9B&amp;code=08067053&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">於东军</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%8D%97%E4%BA%AC%E7%90%86%E5%B7%A5%E5%A4%A7%E5%AD%A6%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%A7%91%E5%AD%A6%E4%B8%8E%E5%B7%A5%E7%A8%8B%E5%AD%A6%E9%99%A2&amp;code=0077991&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">南京理工大学计算机科学与工程学院</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>为了提高预测腺嘌呤核苷三磷酸(ATP)绑定位点的准确率,提出了一种基于一维卷积神经网络(1D-CNN)的方法。首先,以蛋白质的序列信息为基础,融合位置特异性得分矩阵信息、二级结构信息和水溶性信息,使用随机下采样的方法消除数据不平衡的影响,再对缺失的特征进行再编码补齐,得到训练特征。训练一个1D-CNN来预测蛋白质-ATP绑定位点,优化网络结构,并且进行实验来对比所提方法和其他机器学习方法的优劣。实验结果展示了所提方法的有效性,并且该方法与传统支持向量机(SVM)相比在AUC指标上有部分的提升。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%9B%8B%E7%99%BD%E8%B4%A8-ATP&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蛋白质-ATP;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8D%B7%E7%A7%AF%E7%A5%9E%E7%BB%8F%E7%BD%91%E7%BB%9C&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">卷积神经网络;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%95%B0%E6%8D%AE%E4%B8%8D%E5%B9%B3%E8%A1%A1%E9%97%AE%E9%A2%98&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">数据不平衡问题;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%88%86%E7%B1%BB&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">分类;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    张寓(1995—),男,江苏扬州人,硕士研究生,主要研究方向:生物信息计算、模式识别;;
                                </span>
                                <span>
                                    *於东军(1975—),男,江苏镇江人,教授,博士,主要研究方向:生物信息计算、机器学习、模式识别、智能系统,电子邮箱njyudj@njust.edu.cn;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2019-05-06</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金资助项目(61772273,61373062);</span>
                    </p>
            </div>
                    <h1><b>Protein-ATP binding site prediction based on</b> 1<b>D-convolutional neural network</b></h1>
                    <h2>
                    <span>ZHANG Yu</span>
                    <span>YU Dongjun</span>
            </h2>
                    <h2>
                    <span>School of Computer Science and Engineering, Nanjing University of Science and Technology</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>To improve the accuracy of protein-ATP(Adenosine TriPhosphate) binding sites, an algorithm was proposed by using One Dimensional Convolutional Neural Network(1 D-CNN). Firstly, based on the protein sequence information, position specific score matrix information, secondary structure information and water solubility information were combined and random under-sampling was used to eliminate the impact of data imbalance. Then, the missing features were completed by recoding. Finally, the training features were obtained. A 1 D-CNN was trained to predict protein-ATP binding sites, the network structure was optimized, and experiments were carried out to compare the proposed method and other machine learning methods. Experimental results show that the proposed method is effective and can achieve better performance on AUC(Area Under Curve) compared to the traditional Support Vector Machine(SVM).</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=protein-ATP(Adenosine%20TriPhosphate)&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">protein-ATP(Adenosine TriPhosphate);</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Convolutional%20Neural%20Network(CNN)&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Convolutional Neural Network(CNN);</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=data%20imbalance%20problem&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">data imbalance problem;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=classification&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">classification;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                            <b>Author：</b>
                                                        <span>
                                    ZHANG Yu, born in 1995, M. S. candidate. His research interests include bioinformatics computing, pattern recognition. ;
                                </span>
                                <span>
                                    YU Dongjun, born in 1975, Ph. D., professor. His research interests include bioinformatics computing, machine learning, pattern recognition, intelligence system.;
                                </span>
                    </p>
                                    <p><b>Received：</b> 2019-05-06</p>
                                    <p>
                            <b>Fund：</b>
                                                        <span>supported by the National Natural Science Foundation of China(61772273,61373062);</span>
                    </p>
            </div>


        <!--brief start-->
                        <h3 id="63" name="63" class="anchor-tag">0 引言</h3>
                <div class="p1">
                    <p id="64">腺嘌呤核苷三磷酸(Adenosine TriPhosphate, ATP)是大部分生物体力最直接的能量来源,它水解时能释放出大量的能量,这些能量参与了很多人类的基础的生命活动, 因此,ATP是人体内极为重要的一种分子<citation id="223" type="reference"><link href="163" rel="bibliography" /><link href="165" rel="bibliography" /><sup>[<a class="sup">1</a>,<a class="sup">2</a>]</sup></citation>。蛋白质-ATP绑定位点指的是在绑定有ATP的蛋白质的残基中和ATP配体距离小于一定距离的残基,如果能够正确定位蛋白质-ATP绑定位点,就能变相地确定ATP分子的位置,这对药物设计和进一步的生物生命活动研究都有着重大的意义<citation id="224" type="reference"><link href="167" rel="bibliography" /><link href="169" rel="bibliography" /><sup>[<a class="sup">3</a>,<a class="sup">4</a>]</sup></citation>。</p>
                </div>
                <div class="p1">
                    <p id="65">在传统的生物学实验中,通常要通过生物湿实验来测定蛋白质的ATP绑定位点,但是做生物湿实验较为耗时而且经济成本高昂。随着数字化信息时代的开始,很多的生物信息以数字化的方式得以保存<citation id="225" type="reference"><link href="171" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>,随后就出现了使用已知的生物信息来预测未知的信息的方法,最开始使用的方法一般是基于模板的匹配算法<citation id="226" type="reference"><link href="173" rel="bibliography" /><link href="175" rel="bibliography" /><sup>[<a class="sup">6</a>,<a class="sup">7</a>]</sup></citation>,随着机器学习技术的发展,开始出现使用机器学习来预测未知的绑定位点<citation id="227" type="reference"><link href="177" rel="bibliography" /><link href="179" rel="bibliography" /><sup>[<a class="sup">8</a>,<a class="sup">9</a>]</sup></citation>的方法。</p>
                </div>
                <div class="p1">
                    <p id="66">在机器学习方法中,有效的特征提取是提高预测性能的关键。在蛋白质的特征工程方面,开始只是使用原本的序列信息来预测,后来,蛋白质序列中的一些隐藏的信息被挖掘了出来,例如:位置特异性得分矩阵(Position Specific Score Matrix, PSSM)<citation id="228" type="reference"><link href="181" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>、二级结构(Secondary Structure, SS)信息、溶液接触面积(Solvent Accessibility, SA)信息、物理化学属性<citation id="229" type="reference"><link href="183" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>、三级结构信息<citation id="230" type="reference"><link href="185" rel="bibliography" /><link href="187" rel="bibliography" /><sup>[<a class="sup">12</a>,<a class="sup">13</a>]</sup></citation>等,这些信息也被用于参与预测,而且提高了预测的性能; 然而这些信息很多也是未经过实验测定的。针对这个问题,后续出现了使用预测器来预测这些隐藏信息,并且把预测结果作为特征参与预测的集成预测器<citation id="231" type="reference"><link href="189" rel="bibliography" /><link href="191" rel="bibliography" /><sup>[<a class="sup">14</a>,<a class="sup">15</a>]</sup></citation>。随着方法的不断改进,预测的准确率也越来越高。</p>
                </div>
                <div class="p1">
                    <p id="67">近几年来,深度学习技术开始逐渐应用于生物信息学,DeepBind<citation id="232" type="reference"><link href="193" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation>使用了深度卷积神经网络预测了脱氧核糖核酸(DeoxyriboNucleic Acid, DNA)和核糖核酸(RiboNucleic Acid,RNA)的蛋白质绑定位点。DeepSite<citation id="233" type="reference"><link href="195" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>直接使用蛋白质的三级结构特征训练一个深度三维卷积神经网络,对绑定口袋在蛋白质三维空间中的位置进行预测。随着这些深度学习的方法的产生,预测的准确率又被提高了一个层次。</p>
                </div>
                <div class="p1">
                    <p id="68">本文提出了一种基于一维卷积神经网络(One Dimensional Convolutional Neural Network, 1D-CNN)的方法来预测蛋白质-ATP绑定位点,使用蛋白质的序列信息、PSSM矩阵、二级结构特征、溶液接触面积特征来训练模型,预测每个残基是绑定位点的概率。最后和其他使用支持向量机(Support Vector Machine, SVM)的方法进行比较,分析本文方法的优点和缺点。</p>
                </div>
                <h3 id="69" name="69" class="anchor-tag">1 数据集</h3>
                <div class="p1">
                    <p id="70">本文所使用的数据集为论文ATPbind<citation id="234" type="reference"><link href="189" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>中所使用的数据集,该数据集由429个和ATP绑定的蛋白质链构成。该数据集来自2016年5月之前PDB(Protein Data Bank)<citation id="235" type="reference"><link href="197" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>中的数据,并且使用CD-hit<citation id="236" type="reference"><link href="199" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>软件将同源性超过40%的蛋白质全部除去。在这429个蛋白质链中,将其中388个作为训练集(PATP-388),剩下41个蛋白质链作为独立测试集(PATP-TEST)。PATP-388包含5 657个ATP绑定位点残基和142 086个非ATP绑定位点残基,PATP-TEST包含674个ATP绑点位点残基和14 159个非ATP绑点位点残基。</p>
                </div>
                <div class="area_img" id="71">
                    <p class="img_tit"><b>表</b>1 <b>训练集和独立测试集的构成</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit">Tab. 1 Composition of training set and independent test set</p>
                    <p class="img_note"></p>
                    <table id="71" border="1"><tr><td><br />数据集</td><td>序列数量</td><td>正样本</td><td>负样本</td><td>负正比</td></tr><tr><td>PATP-388</td><td>388</td><td>5 657</td><td>142 086</td><td>25.12</td></tr><tr><td><br />PATP-TEST</td><td>41</td><td>674</td><td>14 159</td><td>21.01</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note">注:负正比为负样本数和正样本数的比值。</p>
                    <p class="img_note"></p>
                </div>
                <h3 id="72" name="72" class="anchor-tag">2 蛋白质残基的特征表示</h3>
                <div class="p1">
                    <p id="73">蛋白质的每个残基都是20种氨基酸脱水缩合后中的一种,一条蛋白质链可以简单表示为一个由20种残基组成的字符串,但是每条蛋白质链中的信息远多于一个字符串所表达的信息。蛋白质的特征工程问题一直是生物信息学中复杂而又重要的问题。本文选取了几个常用而又有效的特征,在对其进行一些处理后,作为最终的训练特征。</p>
                </div>
                <h4 class="anchor-tag" id="74" name="74">2.1 <b>特征选取</b></h4>
                <div class="p1">
                    <p id="75">本文选用了PSSM特异性得分矩阵、蛋白质二级结构特征、溶液接触面积作为基本的特征组成成分。</p>
                </div>
                <div class="p1">
                    <p id="76">PSSM矩阵可以反映出该蛋白的序列信息和族谱信息。通常使用PSI-BLAST<citation id="237" type="reference"><link href="201" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>程序将待比对蛋白的信息放到Swiss-Prot<citation id="238" type="reference"><link href="203" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>蛋白质数据库中搜索并和结果进行比对,最后得到PSSM矩阵。PSSM矩阵是一个<i>N</i>×20的矩阵,<i>N</i>为蛋白质残基的数量,每个残基对应着20个元素,这20个元素代表着20种氨基酸的每一种出现的可信度得分。在得到PSSM矩阵后,使用归一化函数将PSSM矩阵的每个得分进行归一化处理。归一化函数如下:</p>
                </div>
                <div class="p1">
                    <p id="77" class="code-formula">
                        <mathml id="77"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><mi>x</mi><mo>-</mo><mi>m</mi><mi>i</mi><mi>n</mi></mrow><mrow><mi>m</mi><mi>a</mi><mi>x</mi><mo>-</mo><mi>m</mi><mi>i</mi><mi>n</mi></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="78">蛋白质的二级结构(SS)指的是多个残基在局部所表现出的一种链接形状。蛋白质的二级结构被分为α螺旋(H)、β折叠(E)和无规律的卷曲(C)。本文中用已有的预测器psipred<citation id="239" type="reference"><link href="205" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>来预测蛋白质的每个残基属于某一种二级结构的概率,并把得到的结果作为特征。预测器得到的结果为<i>N</i>×3的矩阵,即每个残基有3个元素,分别指3种不同二级结构的概率。</p>
                </div>
                <div class="p1">
                    <p id="79">蛋白质的溶液接触面积(SA)是指蛋白质每个残基可以和水接触的面积大小,间接反映了这个残基是否可溶于水。每个残基对应着三种状态:易于和水接触、不易于和水接触、中性。本文中使用已有的预测器Sann<citation id="240" type="reference"><link href="207" rel="bibliography" /><sup>[<a class="sup">23</a>]</sup></citation>来预测,得到<i>N</i>×3的矩阵,即每个残基对应3个元素,分别代表3种状态的概率。</p>
                </div>
                <div class="p1">
                    <p id="80">融合以上特征,每个残基一共有20+3+3=26维的基本向量。然而,一个ATP绑定位点的确定和邻近的残基有相互作用的关系,所以,一般使用一个滑动窗口把指定残基的邻近残基也作为特征输入,针对ATP绑定位点预测问题的滑动窗口大小一般都取17<citation id="241" type="reference"><link href="209" rel="bibliography" /><sup>[<a class="sup">24</a>]</sup></citation>,指定残基的前8个和后8个残基也都在滑动窗口之内。但是一条蛋白质链的前8个残基和后8个残基的滑动窗口区域并不存在17个残基。针对这个问题,本文使用了补位的方法,在头部和尾部补上缺失的残基,对于这些补位的残基,使用了一种类似于one-hot编码的方法。针对选取的3种基本特征,本文在这3个基本特征中每个额外加入一个维度也就变成了21(PSSM)、4(SS)、4(SA),一共29个维度。对于正常的残基,这3个额外的维度值都为0,而针对补位的残基,这3个维度值为1而其余的维度值为0。</p>
                </div>
                <div class="p1">
                    <p id="81">最后,每个残基的维度数量就变成了29×17=493个,这493个维度也是作为最终的输入向量的构成。</p>
                </div>
                <h4 class="anchor-tag" id="82" name="82">2.2 <b>随机下采样</b></h4>
                <div class="p1">
                    <p id="83">由于正负样本数量极为不平衡,正负类比达到1∶24。这就造成了不平衡问题<citation id="242" type="reference"><link href="211" rel="bibliography" /><sup>[<a class="sup">25</a>]</sup></citation>。如果不对数据进行处理,直接进行训练,那么分类器将倾向把所有样本都预测为负类。针对不平衡问题,通常会使用上采样或者下采样方法来平衡样本。文中所使用的是最常用的随机下采样,即随机选取指定数目的负类样本,使之数目和正类样本平衡。每次随机拿取正类样本数目的负类样本,和所有正类样本混合作为一轮神经网络训练的训练样本。</p>
                </div>
                <h3 id="84" name="84" class="anchor-tag">3 卷积神经网络模型的构建和训练</h3>
                <h4 class="anchor-tag" id="85" name="85">3.1 <b>网络模型结构</b></h4>
                <div class="p1">
                    <p id="86">由于蛋白质序列信息是一维信息而且具有局部关联性,本文使用一维卷积神经网络作为模型进行训练。神经网络的大致结构如图1所示。神经网络的输入是一个17×29的矩阵,即长度为17,深度为29。</p>
                </div>
                <div class="area_img" id="87">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201911008_087.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 神经网络模型的基本结构" src="Detail/GetImg?filename=images/JSJY201911008_087.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 神经网络模型的基本结构  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201911008_087.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 1 Basic structure of neural network model</p>

                </div>
                <div class="p1">
                    <p id="88">网络的第一层和第二层都为卷积层(Conv Layer),卷积的窗口大小为2,步长为1,第一层卷积核的数量为400,第二层卷积核的数量也为400,所以经过2层卷积之后,输出的数据的深度就为400。</p>
                </div>
                <div class="p1">
                    <p id="89">由于序列所携带的信息量比较少,在图像领域中的卷积神经网络中常用的池化层在本文中并没有使用。</p>
                </div>
                <div class="p1">
                    <p id="90">第三层为平铺层(Flatten Layer),用来连接卷积层和全连接层。</p>
                </div>
                <div class="p1">
                    <p id="91">第四层和第五层都为全连接层(Dense Layer),第四层的神经元的数量为500,第五层为200。</p>
                </div>
                <div class="p1">
                    <p id="92">第六层是带有丢弃<citation id="243" type="reference"><link href="213" rel="bibliography" /><sup>[<a class="sup">26</a>]</sup></citation>(Drop out)的全连接层,随机丢弃一些输入来防止神经网络过拟合,本文中丢弃输入的概率设为50%。</p>
                </div>
                <div class="p1">
                    <p id="93">最后一层为输出层(Output),神经元的数量为2。这两个神经元的输出就分别代表着正类和负类的预测置信度。</p>
                </div>
                <div class="p1">
                    <p id="94">在每一层得到输入计算出结果后,会使用激活函数对结果进行调整。除最后输出层外,每一层都使用了ELU激活函数。第五层则使用了sigmoid函数。ELU和sigmoid函数如下:</p>
                </div>
                <div class="p1">
                    <p id="95" class="code-formula">
                        <mathml id="95"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>e</mi><mi>l</mi><mi>u</mi><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>=</mo><mrow><mo>{</mo><mrow><mtable><mtr><mtd columnalign="left"><mi>x</mi><mo>,</mo><mspace width="0.25em" /></mtd><mtd columnalign="left"><mi>x</mi><mo>≥</mo><mn>0</mn></mtd></mtr><mtr><mtd columnalign="left"><mi>α</mi><mo stretchy="false">(</mo><mi>exp</mi><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo><mo>,</mo></mtd><mtd columnalign="left"><mi>x</mi><mo>&lt;</mo><mn>0</mn></mtd></mtr></mtable></mrow></mrow><mspace width="0.25em" /><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>2</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="96" class="code-formula">
                        <mathml id="96"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>s</mi><mi>i</mi><mi>g</mi><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mn>1</mn><mrow><mn>1</mn><mo>+</mo><mtext>e</mtext><msup><mrow></mrow><mrow><mo>-</mo><mi>x</mi></mrow></msup></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>3</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="97">最后对这两个输出结果使用Softmax函数,得出正类和负类的预测概率。Softmax函数如下:</p>
                </div>
                <div class="p1">
                    <p id="98" class="code-formula">
                        <mathml id="98"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>σ</mi><mo stretchy="false">(</mo><mi>z</mi><mo stretchy="false">)</mo><msub><mrow></mrow><mi>j</mi></msub><mo>=</mo><mfrac><mrow><mtext>e</mtext><msup><mrow></mrow><mrow><mi mathvariant="bold-italic">z</mi><msub><mrow></mrow><mi>j</mi></msub></mrow></msup></mrow><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>Κ</mi></munderover><mtext>e</mtext></mstyle><msup><mrow></mrow><mrow><mi mathvariant="bold-italic">z</mi><msub><mrow></mrow><mi>k</mi></msub></mrow></msup></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>4</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="99">本次实验使用TensorFlow<citation id="244" type="reference"><link href="215" rel="bibliography" /><sup>[<a class="sup">27</a>]</sup></citation>进行神经网络的构建和训练,在硬件上使用了两块GTX Titan XP,并且使用CUDA(Compute Unified Device Architecture)运算平台进行加速计算。</p>
                </div>
                <div class="p1">
                    <p id="100">为了优化网络结构,本文还构建了一些不同结构的神经网络进行对比实验。文中主要通过改变网络模型中卷积层的层数和每层卷积层中卷积核的个数来改变结构。层数越多,卷积核数量越多的网络的学习能力也就越强,但是同时也会导致学习的速度慢、梯度爆炸和梯度消失的情况发生。</p>
                </div>
                <h4 class="anchor-tag" id="101" name="101">3.2 <b>模型训练</b></h4>
                <div class="p1">
                    <p id="102">本文将一次随机下采样的样本作为一轮的训练样本进行训练,将每次<i>Softmax</i>函数得到的正类和负类的预测概率和真实类别的交叉熵函数作为损失函数,使用梯度下降法对网络模型进行优化。交叉熵函数如下:</p>
                </div>
                <div class="p1">
                    <p id="103" class="code-formula">
                        <mathml id="103"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Η</mi><mo stretchy="false">(</mo><mi mathvariant="bold-italic">p</mi><mo>,</mo><mi mathvariant="bold-italic">q</mi><mo stretchy="false">)</mo><mo>=</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mi>i</mi></munder><mi>p</mi></mstyle><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>*</mo><mspace width="0.25em" /><mrow><mi>ln</mi></mrow><mspace width="0.25em" /><mo stretchy="false">(</mo><mn>1</mn><mo>/</mo><mi>q</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>5</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="104">其中:<i><b>p</b></i>为样本的真实值; <i><b>q</b></i>为样本的预测值; <i>p</i>(<i>i</i>)、<i>q</i>(<i>i</i>)表示向量<i><b>p</b></i>、<i><b>q</b></i>的第<i>i</i>个元素。</p>
                </div>
                <div class="p1">
                    <p id="105">为了使模型收敛的速度加快,将正负样本交替输入进模型进行训练,每输入一个残基的特征就进行一次梯度下降操作,即<i>batch size</i>=1。本文以对独立测试集预测结果的AUC(Area Under Curve)作为第一评估指标,训练多轮直至AUC收敛。</p>
                </div>
                <h3 id="106" name="106" class="anchor-tag">4 实验结果和比较分析</h3>
                <h4 class="anchor-tag" id="107" name="107">4.1 <b>评判标准</b></h4>
                <div class="p1">
                    <p id="108">本文涉及的是一个二分类问题,所有样本只有正类和负类。根据独立测试集的真实标签和预测标签,可以把每个样本的预测结果分成4种情况:<i>TP</i>(<i>True Positives</i>)、<i>FP</i>(<i>False Positives</i>)、<i>TN</i>(<i>True Negatives</i>)和<i>FN</i>(<i>False Negatives</i>)。</p>
                </div>
                <div class="area_img" id="109">
                    <p class="img_tit"><b>表</b>2 <b>混淆矩阵</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><i>Tab</i>. 2 <i>Confusion matrix</i></p>
                    <p class="img_note"></p>
                    <table id="109" border="1"><tr><td rowspan="2"><br />预测类别</td><td colspan="2"><br />实际类别</td></tr><tr><td><br />实际正类</td><td>实际负类</td></tr><tr><td><br />预测正类</td><td><i>TP</i></td><td><i>FP</i></td></tr><tr><td><br />预测负类</td><td><i>FN</i></td><td><i>TN</i></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="110">根据这些基础数据,演化出了特异性(<i>Specificity</i>,<i>Spe</i>)、敏感性(<i>Sensitivity</i>,<i>Sen</i>)、准确性(<i>Accuracy</i>,<i>Acc</i>)、马修斯相关系数(<i>Matthews Correlation Coefficient</i>,<i>MCC</i>)。由于本文的分类器是软分类器,即输出的是属于正类的概率,必须先确定一个阈值才能确定预测的样本是正类还是负类,如果得到的正类概率大于阈值那么这个样本就会被预测为正类; 反之,就会被预测为负类。所以,一个已经被预测好的结果的<i>TP</i>、<i>FP</i>、<i>FN</i>、<i>TN</i>是会随着阈值而改变的,<i>MCC</i>等数据也会随之改变。</p>
                </div>
                <div class="p1">
                    <p id="111" class="code-formula">
                        <mathml id="111"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>S</mi><mi>p</mi><mi>e</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ν</mi></mrow><mrow><mi>Τ</mi><mi>Ν</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>6</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="112" class="code-formula">
                        <mathml id="112"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>S</mi><mi>e</mi><mi>n</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>7</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="113" class="code-formula">
                        <mathml id="113"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>A</mi><mi>c</mi><mi>c</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>Τ</mi><mi>Ν</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>Τ</mi><mi>Ν</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>8</mn><mo stretchy="false">)</mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="114" class="code-formula">
                        <mathml id="114"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>Μ</mi><mi>C</mi><mi>C</mi><mo>=</mo></mtd></mtr><mtr><mtd><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi><mo>*</mo><mi>Τ</mi><mi>Ν</mi><mo>-</mo><mi>F</mi><mi>Ρ</mi><mo>*</mo><mi>F</mi><mi>Ν</mi></mrow><mrow><msqrt><mrow><mo stretchy="false">(</mo><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi><mo stretchy="false">)</mo><mo>*</mo><mo stretchy="false">(</mo><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi><mo stretchy="false">)</mo><mo>*</mo><mo stretchy="false">(</mo><mi>Τ</mi><mi>Ν</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi><mo stretchy="false">)</mo><mo>*</mo><mo stretchy="false">(</mo><mi>Τ</mi><mi>Ν</mi><mo>+</mo><mi>F</mi><mi>Ν</mi><mo stretchy="false">)</mo></mrow></msqrt></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false">(</mo><mn>9</mn><mo stretchy="false">)</mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="115">AUC可以更加稳定地去描述一个预测结果的好坏,将阈值从0到100%遍历,根据<i>TP</i>/<i>TP</i>+<i>FN</i>(纵坐标)和<i>FP</i>/<i>FP</i>+<i>TN</i>(横坐标),画出ROC(Receiver Operating Characteristic)曲线,AUC为ROC曲线下与坐标轴围成的面积,而这个值是不需要一个确定的阈值的。</p>
                </div>
                <div class="p1">
                    <p id="116">由于本文所涉及的问题是个样本不平衡问题,如果使用Acc作为衡量标准是无法判别预测器好坏的,因为即使把所有类都预测为多数类,Acc也能达到很高的数值, 所以,一般用AUC指标作为分类器的评判标准。</p>
                </div>
                <h4 class="anchor-tag" id="117" name="117">4.2 <b>实验结果</b></h4>
                <div class="p1">
                    <p id="118">本文将随机下采样后得到的一组数据作为一轮训练的所需数据。针对每种结构不同的神经网络,会动态调正学习速率,并且让其训练多轮。在每轮训练结束后,使用独立测试集对当前的网络进行一次测试。前170轮次在独立测试集上的结果如图2所示。</p>
                </div>
                <div class="area_img" id="119">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201911008_119.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 两层卷积层300卷积核的CNN模型的训练过程" src="Detail/GetImg?filename=images/JSJY201911008_119.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 两层卷积层300卷积核的<i>CNN</i>模型的训练过程  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201911008_119.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"><i>Fig</i>. 2 <i>Training process of CNN with</i> 2 <i>layers and</i> 300 <i>kernels</i></p>

                </div>
                <div class="p1">
                    <p id="120">可以看出,大概第50轮次训练开始,<i>AUC</i>指数就已经收敛并且趋向于稳定。但是,从40轮开始,<i>MCC</i>却以非常慢的速度上升,而且60轮往后的<i>MCC</i>表现得非常不稳定。随着损失的不断减小,<i>AUC</i>和<i>MCC</i>并没有出现下降的趋势,说明并没有发生过拟合的现象。</p>
                </div>
                <div class="p1">
                    <p id="121">可以得出,卷积神经网络模型在本文的问题中对<i>AUC</i>指标的优化是非常有效的,但是对<i>MCC</i>指标的优化却有所不足,而且从50轮开始,<i>AUC</i>数值基本稳定,但是<i>MCC</i>却缓慢提高。<i>MCC</i>指标优化缓慢的问题有可能和本文神经网络所使用的交叉熵损失函数核梯度下降的优化方法有关。</p>
                </div>
                <div class="p1">
                    <p id="122">将这种方法在独立测试集上,和其他方法进行比较,结果如表3所示。</p>
                </div>
                <div class="area_img" id="123">
                    <p class="img_tit"><b>表</b>3 <b>所提方法和其他预测器 在独立测试集上的性能对比 </b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><i>Tab</i>. 3 <i>Performance comparison of the proposed method</i><i>and other predictors on independent test dataset</i></p>
                    <p class="img_note">单位:%</p>
                    <table id="123" border="1"><tr><td><br />预测器</td><td><i>Sen</i>/%</td><td><i>Spe</i>/%</td><td><i>Acc</i>/%</td><td><i>AUC</i>/%</td></tr><tr><td><br /><i>NsitePred</i><sup>[28]</sup></td><td>46.74</td><td>97.70</td><td>95.39</td><td>85.2</td></tr><tr><td><br /><i>TargetATPsite</i><sup>[29]</sup></td><td>41.25</td><td>99.49</td><td>96.84</td><td>85.3</td></tr><tr><td><br /><i>TargetS</i><sup>[24]</sup></td><td>51.63</td><td>98.89</td><td>96.74</td><td>87.2</td></tr><tr><td><br /><i>ATPseq</i><sup>[15]</sup></td><td>54.45</td><td>99.27</td><td>99.27</td><td>87.8</td></tr><tr><td><br />本文方法</td><td>47.73</td><td>99.22</td><td>96.86</td><td>88.2</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="124">从表3可以看出:虽然<i>Acc</i>并不是特别优秀,不及<i>ATPseq</i>的99.27%,但也达到了一般预测器96%的水平;而本文所提出的方法较于其他方法在<i>AUC</i>上有不少提升,比最好的<i>ATPseq</i>也要高出0.4个百分点,达到了88.2%。这可能就是神经网络着重于<i>AUC</i>的优化才导致了这一结果。</p>
                </div>
                <h3 id="125" name="125" class="anchor-tag">5 讨论</h3>
                <h4 class="anchor-tag" id="126" name="126">5.1 <b>一维卷积神经网络还是二维卷积神经网络</b></h4>
                <div class="p1">
                    <p id="127">二维卷积神经网络目前被广泛应用于图像领域,而且取得了很好的成果。然而,在本文所讨论的问题中,我们认为并没有必要使用二维卷积神经网络。</p>
                </div>
                <div class="p1">
                    <p id="128">本文所使用的序列信息是一个一维的信息,虽然产生的<i>PSSM</i>、<i>SS</i>、<i>SA</i>矩阵看似是个二维的信息,但是本文中只有残基之间是有顺序关系的,也就是矩阵中的各个横列是顺序关系的。而矩阵中的纵列只是残基在各个指标上的得分,各个指标之间并没有明显的顺序关系。</p>
                </div>
                <div class="p1">
                    <p id="129">本文使用了规模大体一致的一维卷积神经网络(1<i>D</i>-<i>CNN</i>)和二维卷积神经网络(<i>Two Dimensional Convolutional Neural Network</i>, 2<i>D</i>-<i>CNN</i>)进行训练,并且对比了它们的训练过程中在独立测试集上的结果,如图3所示。</p>
                </div>
                <div class="area_img" id="130">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201911008_130.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 一维卷积网络和二维卷积网络的对比" src="Detail/GetImg?filename=images/JSJY201911008_130.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 一维卷积网络和二维卷积网络的对比  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201911008_130.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"><i>Fig</i>. 3 <i>Comparison between</i> 1<i>D</i>-<i>CNN and</i> 2<i>D</i>-<i>CNN</i></p>

                </div>
                <div class="p1">
                    <p id="131">一维卷积网络使用的是2层卷积层、400卷积核的结构。二维卷积网络的卷积窗口大小为2×2,因此卷积窗口的数量就变多了。为了保证网络大小规模的平等性,所以削减了卷积核的数目,为2层卷积层、20个卷积核的结构。</p>
                </div>
                <div class="p1">
                    <p id="132">通过实验可以看出,在40轮训练前,二维网络的<i>AUC</i>收敛地稍微快一些。在40轮训练后,两个网络在<i>AUC</i>指标上基本就没有什么明显的差距,而且一维的网络比二维的网络更为稳定。因此,针对序列信息的预测方面,二维卷积网络和一维卷积网络并没有明显的差距。</p>
                </div>
                <h4 class="anchor-tag" id="133" name="133">5.2 <b>深度卷积神经网络还是普通卷积神经网络</b></h4>
                <div class="p1">
                    <p id="134">一般来说,深度神经网络的学习能力和泛化能力都强于浅层的神经网络,但是针对本文中的序列信息来说,浅层的神经网络已经足够学习到里面的规律。在<i>DNA</i>-蛋白质绑定位点的预测问题上,有学者使用了只有1层卷积层的卷积神经网络<citation id="245" type="reference"><link href="221" rel="bibliography" /><sup>[<a class="sup">30</a>]</sup></citation>,就完成了对绑点位点较好的预测。</p>
                </div>
                <div class="p1">
                    <p id="135">为此,本文使用了3折交叉验证比较了不同规格的卷积神经网络,其结果如图4所示(1<i>layer</i>_400表示1层卷积层400卷积核)。</p>
                </div>
                <div class="area_img" id="136">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201911008_136.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 不同结构的卷积神经网络的对比" src="Detail/GetImg?filename=images/JSJY201911008_136.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 不同结构的卷积神经网络的对比  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201911008_136.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"><i>Fig</i>. 4 <i>Comparison between CNNs with different structures</i></p>

                </div>
                <div class="p1">
                    <p id="137">可以看出,2层400卷积核的神经网络模型已经有很好的学习能力和学习速率,较深网络可能出现了梯度爆炸的问题导致收敛困难,而且在卷积神经网络的深度和神经元数量达到一定规模时,再增加深度或者神经元数量并不会对最终结果产生很大的改变。</p>
                </div>
                <h3 id="138" name="138" class="anchor-tag">6 结语</h3>
                <div class="p1">
                    <p id="139">本文主要研究了蛋白质-<i>ATP</i>绑定位点的预测问题,使用了蛋白质的特异性得分矩阵、二级结构、溶液接触面积的融合特征,基于一维卷积神经网络的算法,并且采用随机下采样解决了不平衡问题,完成了对蛋白质-<i>ATP</i>绑定位点的较好的预测,并且在独立测试集上得到了验证。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="163">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The distribution of ligand-binding pockets around protein-protein interfaces suggests a general mechanism for pocket formation">

                                <b>[1]</b> GAO M,SKOLNICK J.The distribution of ligand-binding pockets around protein-protein interfaces suggests a general mechanism for pocket formation[J].Proceedings of the National Academy of Sciences of the United States of America,2012,109(10):3784-3789.
                            </a>
                        </p>
                        <p id="165">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Terahertz underdamped vibrational motion governs protein-ligand binding in solution">

                                <b>[2]</b> TURTON D A,SENN H M,HARWOOD T,et al.Terahertz underdamped vibrational motion governs protein-ligand binding in solution [EB/OL].[2018- 11- 20].http://europacat.co.uk/staff/wynne/pubs/r/2014-NatComm-proteins.pdf.
                            </a>
                        </p>
                        <p id="167">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Halogen Interactions in Protein-Ligand Complexes: Implications of Halogen Bonding for Rational Drug Design">

                                <b>[3]</b> SIRIMULLA S,BAILEY J B,VEGESNA R,et al.Halogen interactions in protein-ligand complexes:implications of halogen bonding for rational drug design [J].Journal of Chemical Information and Modeling,2013,53(11):2781-2791.
                            </a>
                        </p>
                        <p id="169">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=VISCANA: Visualized cluster analysis of protein - Ligand interaction based on the ab initio fragment molecular orbital method for virtual ligand screening">

                                <b>[4]</b> AMARI S,AIZAWA M,ZHANG J,et al.VISCANA:visualized cluster analysis of protein-ligand interaction based on the ab initio fragment molecular orbital method for virtual ligand screening[J].Journal of Chemical Information and Modeling,2006,46(1):221-230.
                            </a>
                        </p>
                        <p id="171">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The GenBank genetic sequence data bank">

                                <b>[5]</b> BILOFSKY H S,BURKS C.The GenBank genetic sequence data bank[J].Nucleic Acids Research,1988,16(5):1861-1863.
                            </a>
                        </p>
                        <p id="173">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=POCKET: a computer graphics method for identifying and displaying protein cavities and their surrounding amino acids">

                                <b>[6]</b> LEVITT D G,BANASZAK L J.POCKET:a computer graphics method for identifying and displaying protein cavities and their surrounding amino acids[J].Journal of Molecular Graphics,1992,10(4):229-234.
                            </a>
                        </p>
                        <p id="175">
                            <a id="bibliography_7" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13012201343676&amp;v=MDc5OThxUVRNbndaZVp0RmlubFVyM0lJVndSYWhRPU5pZk9mYks3SHRET3JZOUVaKzhNQ25zL29CTVQ2VDRQUUgvaXJSZEdlcg==&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[7]</b> LASKOWSKI R A.SURFNET:a program for visualizing molecular surfaces,cavities,and intermolecular interactions[J].Journal of Molecular Graphics,1995,13(5):323-330.
                            </a>
                        </p>
                        <p id="177">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=ATPsite: sequence-based prediction of ATP-binding residues">

                                <b>[8]</b> CHEN K,MIZIANTY M J,KURGAN L.ATPsite:sequence-based prediction of ATP-binding residues[J].Proteome Science,2011,9(S1):S4.
                            </a>
                        </p>
                        <p id="179">
                            <a id="bibliography_9" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CMFD&amp;filename=1015358422.nh&amp;v=MTk0MzgvT1ZGMjZHN0M5RnRYT3JaRWJQSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnNGeW5uVnI=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[9]</b> 石大宏.基于序列的蛋白质—核苷酸绑定位点预测研究[D].南京:南京理工大学,2015.(SHI D H.Sequential protein-GDP binding residues prediction[D].Nanjing:Nanjing University of Science and Technology,2015.)
                            </a>
                        </p>
                        <p id="181">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prediction of disordered regions in proteins from position specific score matrices">

                                <b>[10]</b> JONES D T,WARD J J.Prediction of disordered regions in proteins from position specific score matrices[J].Proteins-Structure Function and Bioinformatics,2003,53(S6):573-578.
                            </a>
                        </p>
                        <p id="183">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Q-SiteFinder: an energy-based method for the prediction of protein-ligand binding sites">

                                <b>[11]</b> LAURIE A T R,JACKSON R M.Q-SiteFinder:an energy-based method for the prediction of protein-ligand binding sites [J].Bioinformatics,2005,21(9):1908-1916.
                            </a>
                        </p>
                        <p id="185">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=TM-align: a protein structure alignment algorithm based on the TM-score">

                                <b>[12]</b> ZHANG Y,SKOLNICK J.TM-align:a protein structure alignment algorithm based on the TM-score[J].Nucleic Acids Research,2005,33(7):2302-2309.
                            </a>
                        </p>
                        <p id="187">
                            <a id="bibliography_13" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYXH201601005&amp;v=MDA2NTZWci9PTHpUVFpyRzRIOWZNcm85RllZUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVac0Z5bm4=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[13]</b> 杨骥.基于序列与结构特征结合的蛋白质与DNA绑定位点预测[J].计算机与现代化,2016(1):20-25.(YANG J.Prediction of DNA-protein binding sites based on combining sequence with structure information[J].Computer and Modernization,2016(1):20-25.)
                            </a>
                        </p>
                        <p id="189">
                            <a id="bibliography_14" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SCPD&amp;filename=CN103617203A&amp;v=MTY3NzZwV0ZhZTdLV1JicWVaK1Z1RmlyZ1U3az1KaU82SHJDK0g5Yk9yNHcwQys0UEQzMUx4eFlUNnpvT1MzZm0=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[14]</b> 於东军,胡俊,於铉.基于查询驱动的蛋白质-配体绑定位点预测方法:201310573950.9[P].2014- 03- 05.(YU D J,HU J,YU X.Prediction of protein-ligand binding sites based on query-driven:201310573950.9[P].2014- 03- 05.)
                            </a>
                        </p>
                        <p id="191">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=ATPbind:accurate protein-ATP binding site prediction by combining sequence-profiling and structure-based comparisons">

                                <b>[15]</b> HU J,LI Y,ZHANG Y,et al.ATPbind:accurate protein-ATP binding site prediction by combining sequence-profiling and structure-based comparisons[J].Journal of Chemical Information and Modeling,2018,58(2):501-510.
                            </a>
                        </p>
                        <p id="193">
                            <a id="bibliography_16" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Predicting the sequence specificities of DNA-and RNA-binding proteins by deep learning">

                                <b>[16]</b> ALIPANAHI B,DELONG A,WEIRAUCH M T,et al.Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning [J].Nature Biotechnology,2015,33(8):831-838.
                            </a>
                        </p>
                        <p id="195">
                            <a id="bibliography_17" target="_blank" href="http://scholar.cnki.net/result.aspx?q=DeepSite:Protein binding site predictor using 3D-convolutional neural networks">

                                <b>[17]</b> JIMENEZ J,DOERR S,MARTINEZ-ROSELL G,et al.DeepSite:protein-binding site predictor using 3D-convolutional neural networks [J].Bioinformatics,2017,33(19):3036-3042.
                            </a>
                        </p>
                        <p id="197">
                            <a id="bibliography_18" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The RCSB Protein Data Bank: a redesigned query system and relational database based on the mmCIF schema">

                                <b>[18]</b> DESHPANDE N,ADDESS K J,BLUHM W F,et al.The RCSB protein data bank:a redesigned query system and relational database based on the mmCIF schema[J].Nucleic Acids Research,2005,33(S1):D233-D237.
                            </a>
                        </p>
                        <p id="199">
                            <a id="bibliography_19" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences">

                                <b>[19]</b> LI W,GODZIK A.Cd-hit:a fast program for clustering and comparing large sets of protein or nucleotide sequences[J].Bioinformatics,2006,22(13):1658-1659.
                            </a>
                        </p>
                        <p id="201">
                            <a id="bibliography_20" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Improving the accuracy of PSI-BLAST protein database searches with composition-based statistics and other refinements">

                                <b>[20]</b> SCHAFFER A A,ARAVIND L,MADDEN T L,et al.Improving the accuracy of PSI-BLAST protein database searches with composition-based statistics and other refinements[J].Nucleic Acids Research,2001,29(14):2994-3005.
                            </a>
                        </p>
                        <p id="203">
                            <a id="bibliography_21" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000">

                                <b>[21]</b> BAIROCH A,APWEILER R.The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000[J].Nucleic Acids Research,2000,28(1):45-48.
                            </a>
                        </p>
                        <p id="205">
                            <a id="bibliography_22" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011601148133&amp;v=MjA3OTZVcjNJSVZ3UmFoUT1OaWZPZmJLN0h0RE5xWTlFWmU4SERYODZvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp0RmlubA==&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[22]</b> JONES D T.Protein secondary structure prediction based on position-specific scoring matrices[J].Journal of Molecular Biology,1999,292(2):195-202.
                            </a>
                        </p>
                        <p id="207">
                            <a id="bibliography_23" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD120614000788&amp;v=MDczNTJVNzdLSlY0U05pZmNhcks2SHRmTnE0OUZaT3dIQkJNOHp4VVNtRGQ5U0g3bjN4RTlmYnZuS3JpZlplWnZGeW5u&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[23]</b> JOO K,LEE S J,LEE J.Sann:solvent accessibility prediction of proteins by nearest neighbor method[J].Proteins,2012,80(7):1791-1797.
                            </a>
                        </p>
                        <p id="209">
                            <a id="bibliography_24" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011501913228&amp;v=MjY5MjVoUT1OaWZPZmJLN0h0RE5xbzlFYmVvTURuNHhvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp0RmlubFVyM0lJVndSYQ==&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[24]</b> YU D J,HU J,TANG Z M,et al.Improving protein-ATP binding residues prediction by boosting SVMs with random under-sampling [J].Neurocomputing,2013,104:180-190.
                            </a>
                        </p>
                        <p id="211">
                            <a id="bibliography_25" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13011501753849&amp;v=MjMzMDFNbndaZVp0RmlubFVyM0lJVndSYWhRPU5pZk9mYks3SHRETnFvOUVZKzRNQkhnd29CTVQ2VDRQUUgvaXJSZEdlcnFRVA==&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[25]</b> COHEN G,HILARIO M,SAX H,et al.Learning from imbalanced data in surveillance of nosocomial infection [J].Artificial Intelligence in Medicine,2006,37(1):7-18.
                            </a>
                        </p>
                        <p id="213">
                            <a id="bibliography_26" >
                                    <b>[26]</b>
                                 SRIVASTAVA N,HINTON G,KRIZHEVSKY A,et al.Dropout:a simple way to prevent neural networks from overfitting[J].The Journal of Machine Learning Research,2014,15(1):1929-1958.
                            </a>
                        </p>
                        <p id="215">
                            <a id="bibliography_27" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM66BB49F2D1AF8309D25C9A64BB131E06&amp;v=MDUxMjZ1SFlmT0dRbGZCckxVMDV0cGh4YnE2d0s0PU5pZklZN1crYktQSXB2bEhFT3ArZW5RNnp4OW42RG9PUVE3a3FHQkhlTEdWTUxxWkNPTnZGU2lXV3I3SklGcG1hQg==&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[27]</b> ABADI M.TensorFlow:learning functions at scale[J].ACM SIGPLAN Notices,2016,51(9):1.
                            </a>
                        </p>
                        <p id="217">
                            <a id="bibliography_28" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prediction and analysis of nucleotide-binding residues using sequence and sequence-derived structural descriptors">

                                <b>[28]</b> CHEN K,MIZIANTY M J,KURGAN L.Prediction and analysis of nucleotide-binding residues using sequence and sequence-derived structural descriptors [J].Bioinformatics,2012,28(3):331-341.
                            </a>
                        </p>
                        <p id="219">
                            <a id="bibliography_29" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD130321006336&amp;v=MTY1OThNOHp4VVNtRGQ5U0g3bjN4RTlmYnZuS3JpZlplWnZGeW5uVTc3S0pWNFNOaWZjYXJLN0h0TE9ybzlGWXVnTUNo&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[29]</b> YU D J,HU J,HUANG Y,et al.TargetATPsite:a template-free method for ATP-binding sites prediction with residue evolution image sparse representation and classifier ensemble [J].Journal of Computational Chemistry,2013,34(11):974-985.
                            </a>
                        </p>
                        <p id="221">
                            <a id="bibliography_30" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Convolutional neural network architectures for predicting DNA–protein binding:">

                                <b>[30]</b> ZENG H,EDWARDS M D,LIU G,et al.Convolutional neural network architectures for predicting DNA-protein binding[J].Bioinformatics,2016,32(12):i121-i127.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JSJY201911008" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJY201911008&amp;v=MDYwMzA1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlublZyL09MejdCZDdHNEg5ak5ybzlGYklRS0RIODR2UjRUNmo=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c0dERktQNlNvSG5VV2tqWXgrbz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="0" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
