<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637136775677783750%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJSJY201904017%26RESULT%3d1%26SIGN%3d4hfEA6PAyWSArmlhBmlNv2puywc%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJY201904017&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JSJY201904017&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJY201904017&amp;v=MzAyNzMzenFxQnRHRnJDVVI3cWZadVpzRnlEZ1ZMN0tMejdCZDdHNEg5ak1xNDlFWTRRS0RIODR2UjRUNmo1NE8=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#85" data-title="0 引言 ">0 引言</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#91" data-title="1 基本概念 ">1 基本概念</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#109" data-title="2 本文FSC-FM算法 ">2 本文FSC-FM算法</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#110" data-title="2.1 &lt;b&gt;模糊谱聚类算法&lt;/b&gt;">2.1 <b>模糊谱聚类算法</b></a></li>
                                                <li><a href="#112" data-title="2.2 FSC-FM&lt;b&gt;方法的优化策略&lt;/b&gt;">2.2 FSC-FM<b>方法的优化策略</b></a></li>
                                                <li><a href="#164" data-title="2.3 FSC-FM&lt;b&gt;方法&lt;/b&gt;">2.3 FSC-FM<b>方法</b></a></li>
                                                <li><a href="#170" data-title="2.4 &lt;b&gt;方法分析&lt;/b&gt;">2.4 <b>方法分析</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#172" data-title="3 实验与结果分析 ">3 实验与结果分析</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#173" data-title="3.1 &lt;b&gt;实验环境&lt;/b&gt;">3.1 <b>实验环境</b></a></li>
                                                <li><a href="#175" data-title="3.2 &lt;b&gt;实验数据集&lt;/b&gt;">3.2 <b>实验数据集</b></a></li>
                                                <li><a href="#180" data-title="3.3 &lt;b&gt;评价指标&lt;/b&gt;">3.3 <b>评价指标</b></a></li>
                                                <li><a href="#194" data-title="3.4 &lt;b&gt;参数影响分析&lt;/b&gt;">3.4 <b>参数影响分析</b></a></li>
                                                <li><a href="#204" data-title="3.5 FEC&lt;b&gt;策略的有效性分析&lt;/b&gt;">3.5 FEC<b>策略的有效性分析</b></a></li>
                                                <li><a href="#208" data-title="3.6 DPCS&lt;b&gt;和&lt;/b&gt;EED&lt;b&gt;策略的有效性分析&lt;/b&gt;">3.6 DPCS<b>和</b>EED<b>策略的有效性分析</b></a></li>
                                                <li><a href="#212" data-title="3.7 &lt;b&gt;算法性能的比较分析&lt;/b&gt;">3.7 <b>算法性能的比较分析</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#238" data-title="4 结语 ">4 结语</a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#115" data-title="图1 不确定网络的构建">图1 不确定网络的构建</a></li>
                                                <li><a href="#198" data-title="&lt;b&gt;表&lt;/b&gt;1 &lt;b&gt;实验参数设置&lt;/b&gt;"><b>表</b>1 <b>实验参数设置</b></a></li>
                                                <li><a href="#199" data-title="图2 &lt;i&gt;ε&lt;/i&gt;和&lt;i&gt;δ&lt;/i&gt;取不同值时, &lt;i&gt;F&lt;/i&gt;-&lt;i&gt;measure&lt;/i&gt;值和 匹配的蛋白质功能模块比例变化">图2 <i>ε</i>和<i>δ</i>取不同值时, <i>F</i>-<i>measure</i>值和 匹配的蛋白质功能模块比例变化</a></li>
                                                <li><a href="#202" data-title="图3 参数&lt;i&gt;T&lt;/i&gt;对&lt;i&gt;F&lt;/i&gt;-&lt;i&gt;measure&lt;/i&gt;的影响">图3 参数<i>T</i>对<i>F</i>-<i>measure</i>的影响</a></li>
                                                <li><a href="#207" data-title="图4 FEC策略对比分析">图4 FEC策略对比分析</a></li>
                                                <li><a href="#211" data-title="图5 DPCS和EED策略对比分析">图5 DPCS和EED策略对比分析</a></li>
                                                <li><a href="#216" data-title="&lt;b&gt;表&lt;/b&gt;2 &lt;b&gt;各算法挖掘的功能模块基本信息&lt;/b&gt;"><b>表</b>2 <b>各算法挖掘的功能模块基本信息</b></a></li>
                                                <li><a href="#219" data-title="图7 各个算法的功能模块挖掘可视化比较">图7 各个算法的功能模块挖掘可视化比较</a></li>
                                                <li><a href="#220" data-title="图6 不同算法的&lt;i&gt;F&lt;/i&gt;-&lt;i&gt;measure&lt;/i&gt;、&lt;i&gt;Sp&lt;/i&gt;和&lt;i&gt;Sn&lt;/i&gt;值对比">图6 不同算法的<i>F</i>-<i>measure</i>、<i>Sp</i>和<i>Sn</i>值对比</a></li>
                                                <li><a href="#226" data-title="&lt;b&gt;表&lt;/b&gt;3 &lt;b&gt;各个算法挖掘的功能模块的显著性统计信息&lt;/b&gt;"><b>表</b>3 <b>各个算法挖掘的功能模块的显著性统计信息</b></a></li>
                                                <li><a href="#228" data-title="图8 各个算法的&lt;i&gt;P&lt;/i&gt;-&lt;i&gt;value&lt;/i&gt;区间对比分析">图8 各个算法的<i>P</i>-<i>value</i>区间对比分析</a></li>
                                                <li><a href="#232" data-title="&lt;b&gt;表&lt;/b&gt;4 &lt;i&gt;FSC&lt;/i&gt;-&lt;i&gt;FM&lt;/i&gt;&lt;b&gt;和其他四种算法效率分析&lt;/b&gt;"><b>表</b>4 <i>FSC</i>-<i>FM</i><b>和其他四种算法效率分析</b></a></li>
                                                <li><a href="#236" data-title="&lt;b&gt;表&lt;/b&gt;5 &lt;b&gt;在&lt;/b&gt;&lt;i&gt;Krogan&lt;/i&gt;&lt;b&gt;数据集上各个算法的运行效率分析&lt;/b&gt;"><b>表</b>5 <b>在</b><i>Krogan</i><b>数据集上各个算法的运行效率分析</b></a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="290">


                                    <a id="bibliography_1" title="冀俊忠, 高光轩.基于文化算法的PPI网络功能模块检测方法[J].北京工业大学学报, 2017, 43 (1) :13-21. (JI J Z, GAO GX.Detecting functional module method based on cultural algorithm in protein-protein interaction networks[J].Journal of Beijing University of Technology, 2017, 43 (1) :13-21.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=BJGD201701006&amp;v=MDg5NjBESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVac0Z5RGdWTDdLSnlmTWFyRzRIOWJNcm85RllvUUs=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[1]</b>
                                        冀俊忠, 高光轩.基于文化算法的PPI网络功能模块检测方法[J].北京工业大学学报, 2017, 43 (1) :13-21. (JI J Z, GAO GX.Detecting functional module method based on cultural algorithm in protein-protein interaction networks[J].Journal of Beijing University of Technology, 2017, 43 (1) :13-21.) 
                                    </a>
                                </li>
                                <li id="292">


                                    <a id="bibliography_2" title="鱼亮, 高琳, 孙鹏.蛋白质网络中复合体和功能模块预测算法研究[J].计算机学报, 2011, 34 (7) :1239-1251. (YU L, GAOL, SUN P.Research on algorithms for complexes and functional modules prediction in protein-protein interaction networks[J].Chinese Journal of Computer, 2011, 34 (7) :1239-1251.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJX201107007&amp;v=MzIwOTZWTDdLTHo3QmRyRzRIOURNcUk5Rlk0UUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVac0Z5RGc=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[2]</b>
                                        鱼亮, 高琳, 孙鹏.蛋白质网络中复合体和功能模块预测算法研究[J].计算机学报, 2011, 34 (7) :1239-1251. (YU L, GAOL, SUN P.Research on algorithms for complexes and functional modules prediction in protein-protein interaction networks[J].Chinese Journal of Computer, 2011, 34 (7) :1239-1251.) 
                                    </a>
                                </li>
                                <li id="294">


                                    <a id="bibliography_3" title="倪问尹, 王建新, 熊慧军, 等.基于不确定数据的功能模块预测[J].四川大学学报 (工程科学版) , 2013, 45 (5) :80-87. (NI W Y, WANG J X, XIONG H J, et al.Research of detecting functional modules based on uncertainty data[J].Journal of Sichuan University (Engineering Science Edition) , 2013, 45 (5) :80-87.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=SCLH201305014&amp;v=MjE1MjU3cWZadVpzRnlEZ1ZMN0tOaTdIWnJHNEg5TE1xbzlFWUlRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[3]</b>
                                        倪问尹, 王建新, 熊慧军, 等.基于不确定数据的功能模块预测[J].四川大学学报 (工程科学版) , 2013, 45 (5) :80-87. (NI W Y, WANG J X, XIONG H J, et al.Research of detecting functional modules based on uncertainty data[J].Journal of Sichuan University (Engineering Science Edition) , 2013, 45 (5) :80-87.) 
                                    </a>
                                </li>
                                <li id="296">


                                    <a id="bibliography_4" title="冀俊忠, 刘志军, 刘红欣, 等.蛋白质相互作用网络功能模块检测的研究综述[J].自动化学报, 2014, 40 (4) :577-593. (JI JZ, LIU Z J, LIU H X, et al.An overview research on functional module detection for protein-protein interaction networks[J].Acta Automatica Sinica, 2014, 40 (4) :577-593.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=MOTO201404002&amp;v=MTgyNzh0R0ZyQ1VSN3FmWnVac0Z5RGdWTDdLS0NMZlliRzRIOVhNcTQ5RlpvUUtESDg0dlI0VDZqNTRPM3pxcUI=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[4]</b>
                                        冀俊忠, 刘志军, 刘红欣, 等.蛋白质相互作用网络功能模块检测的研究综述[J].自动化学报, 2014, 40 (4) :577-593. (JI JZ, LIU Z J, LIU H X, et al.An overview research on functional module detection for protein-protein interaction networks[J].Acta Automatica Sinica, 2014, 40 (4) :577-593.) 
                                    </a>
                                </li>
                                <li id="298">


                                    <a id="bibliography_5" title="李敏, 王建新, 刘彬彬, 等.基于极大团扩展的蛋白质复合物识别算法[J].中南大学学报 (自然科学版) , 2010, 41 (2) :560-565. (LI M, WANG J X, LIU B B, et al.An algorithm for identifying protein complexes based on maximal clique extension[J].Journal of Central South University (Science and Technology) , 2010, 41 (2) :560-565.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZNGD201002032&amp;v=MTk1NDJDVVI3cWZadVpzRnlEZ1ZMN0tQeVBNYXJHNEg5SE1yWTlHWm9RS0RIODR2UjRUNmo1NE8zenFxQnRHRnI=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[5]</b>
                                        李敏, 王建新, 刘彬彬, 等.基于极大团扩展的蛋白质复合物识别算法[J].中南大学学报 (自然科学版) , 2010, 41 (2) :560-565. (LI M, WANG J X, LIU B B, et al.An algorithm for identifying protein complexes based on maximal clique extension[J].Journal of Central South University (Science and Technology) , 2010, 41 (2) :560-565.) 
                                    </a>
                                </li>
                                <li id="300">


                                    <a id="bibliography_6" title="KESSLER J, ANDRUSHCHENKO V, KAPITAN J, et al.Insight into vibrational circular dichroism of proteins by density functional modeling[J].Physical Chemistry Chemical Physics, 2018, 20 (7) :4926-4935." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Insight into vibrational circular dichroism of proteins by density functional modeling">
                                        <b>[6]</b>
                                        KESSLER J, ANDRUSHCHENKO V, KAPITAN J, et al.Insight into vibrational circular dichroism of proteins by density functional modeling[J].Physical Chemistry Chemical Physics, 2018, 20 (7) :4926-4935.
                                    </a>
                                </li>
                                <li id="302">


                                    <a id="bibliography_7" title="ALDECO R, MARIN I.Jerarca:efficient analysis of complex networks using hierarchical clustering[J].PLo S ONE, 2010, 5 (7) :11585-11591." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Jerarca: efficient analysis of complex networks using hierarchical clustering">
                                        <b>[7]</b>
                                        ALDECO R, MARIN I.Jerarca:efficient analysis of complex networks using hierarchical clustering[J].PLo S ONE, 2010, 5 (7) :11585-11591.
                                    </a>
                                </li>
                                <li id="304">


                                    <a id="bibliography_8" title="ABEYSIRIGUNAWARDENA S C, KIM H, LAI J, et al.Evolution of protein-coupled RNA dynamics during hierarchical assembly of ribosomal complexes[J].Nature Communications, 2017, 8 (1) :492-500." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Evolution of protein-coupled RNA dynamics during hierarchical assembly of ribosomal complexes">
                                        <b>[8]</b>
                                        ABEYSIRIGUNAWARDENA S C, KIM H, LAI J, et al.Evolution of protein-coupled RNA dynamics during hierarchical assembly of ribosomal complexes[J].Nature Communications, 2017, 8 (1) :492-500.
                                    </a>
                                </li>
                                <li id="306">


                                    <a id="bibliography_9" title="雷秀娟, 高银, 郭玲.基于拓扑势加权的动态PPI网络复合物挖掘方法[J].电子学报, 2018, 46 (1) :145-151. (LEI X J, GAOY, GUO L.Mining protein complexes based on topology potential weight in dynamic protein-protein interaction networks[J].Acta Electronica Sinica, 2018, 46 (1) :145-151.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=DZXU201801020&amp;v=MDQ0MzJGeURnVkw3S0lUZlRlN0c0SDluTXJvOUhaSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnM=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[9]</b>
                                        雷秀娟, 高银, 郭玲.基于拓扑势加权的动态PPI网络复合物挖掘方法[J].电子学报, 2018, 46 (1) :145-151. (LEI X J, GAOY, GUO L.Mining protein complexes based on topology potential weight in dynamic protein-protein interaction networks[J].Acta Electronica Sinica, 2018, 46 (1) :145-151.) 
                                    </a>
                                </li>
                                <li id="308">


                                    <a id="bibliography_10" title="YAO X H, YAN J W, LIU K F, et al.Tissue-specific networkbased genome wide study of amygdala imaging phenotypes to identify functional interaction modules[J].Bioinformatics, 2017, 33 (20) :3250-3257." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Tissue-specific networkbased genome wide study of amygdala imaging phenotypes to identify functional interaction modules">
                                        <b>[10]</b>
                                        YAO X H, YAN J W, LIU K F, et al.Tissue-specific networkbased genome wide study of amygdala imaging phenotypes to identify functional interaction modules[J].Bioinformatics, 2017, 33 (20) :3250-3257.
                                    </a>
                                </li>
                                <li id="310">


                                    <a id="bibliography_11" title="范子静, 罗泽, 马永征.一种基于模糊核聚类的谱聚类算法[J].计算机工程, 2017, 43 (11) :161-165. (FAN Z J, LUOZ, MA Y Z.A spectral clustering algorithm based on fuzzy kernel clustering[J].Computer Engineering, 2017, 43 (11) :161-165.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201711027&amp;v=MTM3NDNCYmJHNEg5Yk5ybzlIWTRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlEZ1ZMN0tMejc=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[11]</b>
                                        范子静, 罗泽, 马永征.一种基于模糊核聚类的谱聚类算法[J].计算机工程, 2017, 43 (11) :161-165. (FAN Z J, LUOZ, MA Y Z.A spectral clustering algorithm based on fuzzy kernel clustering[J].Computer Engineering, 2017, 43 (11) :161-165.) 
                                    </a>
                                </li>
                                <li id="312">


                                    <a id="bibliography_12" title="MADANI S, FAEZ K, AMINGHAFARI M.Identifying similar functional modules by a new hybrid spectral clustering method[J].IET Systems Biology, 2012, 6 (5) :175-186." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Identifying similar functional modules by a new hybrid spectral clustering method">
                                        <b>[12]</b>
                                        MADANI S, FAEZ K, AMINGHAFARI M.Identifying similar functional modules by a new hybrid spectral clustering method[J].IET Systems Biology, 2012, 6 (5) :175-186.
                                    </a>
                                </li>
                                <li id="314">


                                    <a id="bibliography_13" title="QIN G M, GAO L.Spectral clustering for protein complexes in Protein-Protein Interaction (PPI) networks[J].Mathematical and Computer Modelling, 2010, 52 (11/12) :2066-2074." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13012300032579&amp;v=MjUyNDRaT2dOQ1hzd29CTVQ2VDRQUUgvaXJSZEdlcnFRVE1ud1plWnRGaW5sVXIzSUtGc1RheEE9TmlmT2ZiSzdIdERPckk5Rg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[13]</b>
                                        QIN G M, GAO L.Spectral clustering for protein complexes in Protein-Protein Interaction (PPI) networks[J].Mathematical and Computer Modelling, 2010, 52 (11/12) :2066-2074.
                                    </a>
                                </li>
                                <li id="316">


                                    <a id="bibliography_14" title="INOUE K, LI W J, KURATA H.Diffusion model based spectral clustering for protein-protein interaction networks[J].PLo S ONE, 2010, 5 (9) :12623-12632." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Diffusion model based spectral clustering for protein-protein interaction networks">
                                        <b>[14]</b>
                                        INOUE K, LI W J, KURATA H.Diffusion model based spectral clustering for protein-protein interaction networks[J].PLo S ONE, 2010, 5 (9) :12623-12632.
                                    </a>
                                </li>
                                <li id="318">


                                    <a id="bibliography_15" title="那第尔.识别蛋白质相互作用网络中的复合物[D].长沙:中南大学, 2012:22-34. (NA D E.Exploiting fuzzy spectral clustering in protein-complex detection[D].Changsha:Central South University, 2012:22-34.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CMFD&amp;filename=1012477561.nh&amp;v=MjkxNTZWTDdLVkYyNkhMZS9HZFRLcnBFYlBJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVac0Z5RGc=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[15]</b>
                                        那第尔.识别蛋白质相互作用网络中的复合物[D].长沙:中南大学, 2012:22-34. (NA D E.Exploiting fuzzy spectral clustering in protein-complex detection[D].Changsha:Central South University, 2012:22-34.) 
                                    </a>
                                </li>
                                <li id="320">


                                    <a id="bibliography_16" title="TRIVODALIEV K, CINGOVSKA I, KALAJDZISKI S.Protein function prediction by spectral clustering of protein interaction network[C]//Proceedings of the 2011 Database Theory and Application, Bio-Science and Bio-Technology.Berlin:Springer, 2011:108-117." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Protein Function Prediction by Spectral Clustering of Protein Interaction Network">
                                        <b>[16]</b>
                                        TRIVODALIEV K, CINGOVSKA I, KALAJDZISKI S.Protein function prediction by spectral clustering of protein interaction network[C]//Proceedings of the 2011 Database Theory and Application, Bio-Science and Bio-Technology.Berlin:Springer, 2011:108-117.
                                    </a>
                                </li>
                                <li id="322">


                                    <a id="bibliography_17" title="ZOU Z N, LI J Z, GAO H, et al.Mining frequent subgraph patterns from uncertain graph data[J].IEEE Transactions on Knowledge and Data Engineering, 2010, 22 (9) :1203-1218." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Mining Frequent Subgraph Patterns from Uncertain Graph Data">
                                        <b>[17]</b>
                                        ZOU Z N, LI J Z, GAO H, et al.Mining frequent subgraph patterns from uncertain graph data[J].IEEE Transactions on Knowledge and Data Engineering, 2010, 22 (9) :1203-1218.
                                    </a>
                                </li>
                                <li id="324">


                                    <a id="bibliography_18" title="ZHANG Y J, LIN H F, YANG Z H, et al.An uncertain modelbased approach for identifying protein complexes in uncertain protein-protein interaction networks[J].BMC Genomics, 2017, 18 (7) :743-752." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD4733C6586C7FB6FEAFD8D6CAB85AFD54&amp;v=MDU0NTdFdDFQSG1SM1dBOWZNUGlNYitiQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQnJMVTA1dHBoekwyNHdhbz1OajdCYXJlL0hkSy9xWXBOWXBnSWVnNC91V05pbg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[18]</b>
                                        ZHANG Y J, LIN H F, YANG Z H, et al.An uncertain modelbased approach for identifying protein complexes in uncertain protein-protein interaction networks[J].BMC Genomics, 2017, 18 (7) :743-752.
                                    </a>
                                </li>
                                <li id="326">


                                    <a id="bibliography_19" title="ZHAO B H, WANG J X, LI M.Detecting protein complexes based on uncertain graph model[J].IEEE/ACM Transactions on Computational Biology&amp;amp;Bioinformatics, 2014, 11 (3) :486-497." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM89D31A271C531AB878079906FAB42375&amp;v=MTU1OTQyYUNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkJyTFUwNXRwaHpMMjR3YW89TmlmSVk3dXhhdExOM28xQ1paZ0tEMzFJdlI0VTRqOTZRWGJpcW1SRUM3YVdScg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[19]</b>
                                        ZHAO B H, WANG J X, LI M.Detecting protein complexes based on uncertain graph model[J].IEEE/ACM Transactions on Computational Biology&amp;amp;Bioinformatics, 2014, 11 (3) :486-497.
                                    </a>
                                </li>
                                <li id="328">


                                    <a id="bibliography_20" title="HALIM Z, WAQAS M, HUSSAIN S F.Clustering large probabilistic graphs using multi-population evolutionary algorithm[J].Information Sciences, 2015, 317 (1) :78-95." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES15120801000224&amp;v=MTYyNDFyM0lLRnNUYXhBPU5pZk9mYks5SDlQTXA0OUVaT3NQRG40OW9CTVQ2VDRQUUgvaXJSZEdlcnFRVE1ud1plWnRGaW5sVQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[20]</b>
                                        HALIM Z, WAQAS M, HUSSAIN S F.Clustering large probabilistic graphs using multi-population evolutionary algorithm[J].Information Sciences, 2015, 317 (1) :78-95.
                                    </a>
                                </li>
                                <li id="330">


                                    <a id="bibliography_21" title="BANO R, RAO K.Graph based gene/protein prediction and clustering over uncertain medical databases[J].Journal of Theoretical and Applied Information Technology, 2015, 82 (3) :347-352." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Graph based gene/protein prediction and clustering over uncertain medical databases">
                                        <b>[21]</b>
                                        BANO R, RAO K.Graph based gene/protein prediction and clustering over uncertain medical databases[J].Journal of Theoretical and Applied Information Technology, 2015, 82 (3) :347-352.
                                    </a>
                                </li>
                                <li id="332">


                                    <a id="bibliography_22" title="GAO Y J, MIAO X Y, CHEN G, et al.On efficiently finding reverse k-nearest neighbors over uncertain graphs[J].VLDB Journal, 2017, 26 (4) :1-26." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDC6960E2C87630F6FEB62B811B99CDF44&amp;v=MzI1MzFXV3I3SklGcG1hQnVIWWZPR1FsZkJyTFUwNXRwaHpMMjR3YW89Tmo3QmFzQytGOWZNMm8wMmJPd0pEM3hQeVdCbW1EbC9PbmZqcldBOGNNSGdNNzZiQ09OdkZTaQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[22]</b>
                                        GAO Y J, MIAO X Y, CHEN G, et al.On efficiently finding reverse k-nearest neighbors over uncertain graphs[J].VLDB Journal, 2017, 26 (4) :1-26.
                                    </a>
                                </li>
                                <li id="334">


                                    <a id="bibliography_23" title="李敏, 张含会, 费耀平.融合PPI和基因表达数据的关键蛋白质识别方法[J].中南大学学报 (自然科学版) , 2013, 44 (3) :1024-1039. (LI M, ZHANG H H, FEI Y P.Essential protein discovery method based on integration of PPI and gene expression data[J].Journal of Central South University (Science and Technology) , 2013, 44 (3) :1024-1039.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZNGD201303027&amp;v=MDQzNDdUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlEZ1ZMN0tQeVBNYXJHNEg5TE1ySTlIWTRRS0RIODR2UjQ=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[23]</b>
                                        李敏, 张含会, 费耀平.融合PPI和基因表达数据的关键蛋白质识别方法[J].中南大学学报 (自然科学版) , 2013, 44 (3) :1024-1039. (LI M, ZHANG H H, FEI Y P.Essential protein discovery method based on integration of PPI and gene expression data[J].Journal of Central South University (Science and Technology) , 2013, 44 (3) :1024-1039.) 
                                    </a>
                                </li>
                                <li id="336">


                                    <a id="bibliography_24" title="黄链, 邓磊.拟-偏b-度量空间中α-φ-压缩映象不动点的存在性[J].西南大学学报 (自然科学版) , 2018, 40 (3) :115-120. (HUANG L, DENG L.α-φ-contractive mappings on quasi-partial b-metric spaces[J].Journal of Southwest University (Natural Science Edition) , 2018, 40 (3) :115-120.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=XNND201803018&amp;v=MTM3MTE0TzN6cXFCdEdGckNVUjdxZlp1WnNGeURnVkw3S1BTUEZhckc0SDluTXJJOUViSVFLREg4NHZSNFQ2ajU=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[24]</b>
                                        黄链, 邓磊.拟-偏b-度量空间中α-φ-压缩映象不动点的存在性[J].西南大学学报 (自然科学版) , 2018, 40 (3) :115-120. (HUANG L, DENG L.α-φ-contractive mappings on quasi-partial b-metric spaces[J].Journal of Southwest University (Natural Science Edition) , 2018, 40 (3) :115-120.) 
                                    </a>
                                </li>
                                <li id="338">


                                    <a id="bibliography_25" title="朱镕, 邹兆年, 李建中.不确定图上的Top-k稠密子图挖掘算法[J].计算机学报, 2016, 39 (8) :1570-1582. (ZHU R, ZOUZ N, LI J Z.Mining Top-k dense subgraphs from uncertain graphs[J].Chinese Journal of Computers, 2016, 39 (8) :1570-1582.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJX201608006&amp;v=MDQ4MjR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlEZ1ZMN0tMejdCZHJHNEg5Zk1wNDlGWW9RS0RIODQ=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[25]</b>
                                        朱镕, 邹兆年, 李建中.不确定图上的Top-k稠密子图挖掘算法[J].计算机学报, 2016, 39 (8) :1570-1582. (ZHU R, ZOUZ N, LI J Z.Mining Top-k dense subgraphs from uncertain graphs[J].Chinese Journal of Computers, 2016, 39 (8) :1570-1582.) 
                                    </a>
                                </li>
                                <li id="340">


                                    <a id="bibliography_26" title="胡赛, 熊慧军, 陈治平, 等.基于不确定网络的关键蛋白质识别[J].四川大学学报 (工程科学版) , 2014, 46 (5) :116-120. (HU S, XIONG H J, CHEN Z P, et al.Identification of essential proteins based on uncertain networks[J].Journal of Sichuan University (Engineering Science Edition) , 2014, 46 (5) :116-120.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=SCLH201405017&amp;v=MTk0ODFyQ1VSN3FmWnVac0Z5RGdWTDdLTmk3SFpyRzRIOVhNcW85RVk0UUtESDg0dlI0VDZqNTRPM3pxcUJ0R0Y=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[26]</b>
                                        胡赛, 熊慧军, 陈治平, 等.基于不确定网络的关键蛋白质识别[J].四川大学学报 (工程科学版) , 2014, 46 (5) :116-120. (HU S, XIONG H J, CHEN Z P, et al.Identification of essential proteins based on uncertain networks[J].Journal of Sichuan University (Engineering Science Edition) , 2014, 46 (5) :116-120.) 
                                    </a>
                                </li>
                                <li id="342">


                                    <a id="bibliography_27" title="王玲, 薄列峰, 焦李成.密度敏感的谱聚类[J].电子学报, 2007, 35 (8) :1577-1581. (WANG L, BO L F, JIAO L C.Density-sensitive spectral clustering[J].Acta Electronica Sinica, 2007, 35 (8) :1577-1581.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=DZXU200708030&amp;v=MjIxMzJGeURnVkw3S0lUZlRlN0c0SHRiTXA0OUdaSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnM=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[27]</b>
                                        王玲, 薄列峰, 焦李成.密度敏感的谱聚类[J].电子学报, 2007, 35 (8) :1577-1581. (WANG L, BO L F, JIAO L C.Density-sensitive spectral clustering[J].Acta Electronica Sinica, 2007, 35 (8) :1577-1581.) 
                                    </a>
                                </li>
                                <li id="344">


                                    <a id="bibliography_28" title="RAFAILIDIS D, CONSTANTINOU E, MANOLOPOULOS Y.Landmark selection for spectral clustering based on weighted PageRank[J].Future Generation Computer Systems, 2017, 68 (3) :465-472." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Landmark Selection for Spectral Clustering Based on Weighted PageRank">
                                        <b>[28]</b>
                                        RAFAILIDIS D, CONSTANTINOU E, MANOLOPOULOS Y.Landmark selection for spectral clustering based on weighted PageRank[J].Future Generation Computer Systems, 2017, 68 (3) :465-472.
                                    </a>
                                </li>
                                <li id="346">


                                    <a id="bibliography_29" title="KESEMEN O, TEZEL O, OZKUL E.Fuzzy C-means clustering algorithm for directional data (FCM4DD) [J].Expert Systems with Applications, 2016, 58:76-82." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Fuzzy c-means clustering algorithm for directional data (FCM4DD).">
                                        <b>[29]</b>
                                        KESEMEN O, TEZEL O, OZKUL E.Fuzzy C-means clustering algorithm for directional data (FCM4DD) [J].Expert Systems with Applications, 2016, 58:76-82.
                                    </a>
                                </li>
                                <li id="348">


                                    <a id="bibliography_30" title="XENARIOS I, SALWINSKI L, DUAN X J, et al.DIP, the database of interacting proteins:a research tool for studying cellular networks of protein interactions[J].Nucleic Acids Research, 2002, 30 (1) :303-305." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=DIP, the Database of Interacting Proteins: a research tool for studying cellular networks of protein interactions">
                                        <b>[30]</b>
                                        XENARIOS I, SALWINSKI L, DUAN X J, et al.DIP, the database of interacting proteins:a research tool for studying cellular networks of protein interactions[J].Nucleic Acids Research, 2002, 30 (1) :303-305.
                                    </a>
                                </li>
                                <li id="350">


                                    <a id="bibliography_31" title="PU S, WONG J, TURNER B, et al.Up-to-date catalogues of yeast protein complexes[J].Nucleic Acids Research, 2009, 37 (3) :825-831." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Up-to-date catalogues of yeast protein complexes">
                                        <b>[31]</b>
                                        PU S, WONG J, TURNER B, et al.Up-to-date catalogues of yeast protein complexes[J].Nucleic Acids Research, 2009, 37 (3) :825-831.
                                    </a>
                                </li>
                                <li id="352">


                                    <a id="bibliography_32" title="KROGAN N, CAGNEY G, YU H, et al.Global landscape of protein complexes in the yeast Saccharomyces cerevisiae[J].Nature, 2006, 440 (7084) :637-643." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Global landscape of protein complexes in the yeast Saccharomyces cerevisiae">
                                        <b>[32]</b>
                                        KROGAN N, CAGNEY G, YU H, et al.Global landscape of protein complexes in the yeast Saccharomyces cerevisiae[J].Nature, 2006, 440 (7084) :637-643.
                                    </a>
                                </li>
                                <li id="354">


                                    <a id="bibliography_33" title="胡赛, 熊慧军, 李学勇, 等.多关系蛋白质网络构建及其应用研究[J].自动化学报, 2015, 41 (12) :2155-2163. (HU S, XIONG H J, LI X Y, et al.Construction of multi-relation protein networks and its application[J].Acta Automatica Sinica, 2015, 41 (12) :2155-2163.) " target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=MOTO201512016&amp;v=MjI3MTVxcUJ0R0ZyQ1VSN3FmWnVac0Z5RGdWTDdLS0NMZlliRzRIOVROclk5RVlvUUtESDg0dlI0VDZqNTRPM3o=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[33]</b>
                                        胡赛, 熊慧军, 李学勇, 等.多关系蛋白质网络构建及其应用研究[J].自动化学报, 2015, 41 (12) :2155-2163. (HU S, XIONG H J, LI X Y, et al.Construction of multi-relation protein networks and its application[J].Acta Automatica Sinica, 2015, 41 (12) :2155-2163.) 
                                    </a>
                                </li>
                                <li id="356">


                                    <a id="bibliography_34" title="LEI X J, WU S, LIANG G, et al.Clustering and overlapping modules detection in PPI network based on IBFO[J].Proteomics, 2013, 13 (2) :278-290." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD15121501608629&amp;v=MjUyOTN4QT1OaWZjYXJLOUg5UE5xbzlFWXVzSENuNHdvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp0RmlubFVyM0lLRnNUYQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[34]</b>
                                        LEI X J, WU S, LIANG G, et al.Clustering and overlapping modules detection in PPI network based on IBFO[J].Proteomics, 2013, 13 (2) :278-290.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">

    <div class="head-tag">   
            <p>
               <b> 网络首发时间: 2018-11-12 13:24</b>
            </p>     
    </div>


        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JSJY" target="_blank">计算机应用</a>
                2019,39(04),1032-1040 DOI:10.11772/j.issn.1001-9081.2018091880            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>基于模糊谱聚类的不确定蛋白质相互作用网络功能模块挖掘</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%AF%9B%E4%BC%8A%E6%95%8F&amp;code=10557422&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">毛伊敏</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%88%98%E9%93%B6%E8%90%8D&amp;code=41172130&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">刘银萍</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%A2%81%E7%94%B0&amp;code=43182144&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">梁田</a>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E6%AF%9B%E4%B8%81%E6%85%A7&amp;code=37944838&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">毛丁慧</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E6%B1%9F%E8%A5%BF%E7%90%86%E5%B7%A5%E5%A4%A7%E5%AD%A6%E4%BF%A1%E6%81%AF%E5%B7%A5%E7%A8%8B%E5%AD%A6%E9%99%A2&amp;code=0217990&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">江西理工大学信息工程学院</a>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E6%B1%9F%E8%A5%BF%E7%90%86%E5%B7%A5%E5%A4%A7%E5%AD%A6%E5%BA%94%E7%94%A8%E7%A7%91%E5%AD%A6%E5%AD%A6%E9%99%A2&amp;code=1750197&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">江西理工大学应用科学学院</a>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E4%B8%AD%E9%99%95%E6%A0%B8%E5%B7%A5%E4%B8%9A%E9%9B%86%E5%9B%A2%E4%BA%8C%E4%B8%80%E4%B8%80%E5%A4%A7%E9%98%9F%E6%9C%89%E9%99%90%E5%85%AC%E5%8F%B8&amp;code=1701851&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">中陕核工业集团二一一大队有限公司</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>针对谱聚类融合模糊<i>C</i>-means (FCM) 聚类的蛋白质相互作用 (PPI) 网络功能模块挖掘方法准确率不高、执行效率较低和易受假阳性影响的问题, 提出一种基于模糊谱聚类的不确定PPI网络功能模块挖掘 (FSC-FM) 方法。首先, 构建一个不确定PPI网络模型, 使用边聚集系数给每一条蛋白质交互作用赋予一个存在概率测度, 克服假阳性对实验结果的影响;第二, 利用基于边聚集系数流行距离 (FEC) 策略改进谱聚类中的相似度计算, 解决谱聚类算法对尺度参数敏感的问题, 进而利用谱聚类算法对不确定PPI网络数据进行预处理, 降低数据的维数, 提高聚类的准确率;第三, 设计基于密度的概率中心选取策略 (DPCS) 解决模糊<i>C</i>-means算法对初始聚类中心和聚类数目敏感的问题, 并对预处理后的PPI数据进行FCM聚类, 提高聚类的执行效率以及灵敏度;最后, 采用改进的边期望稠密度 (EED) 对挖掘出的蛋白质功能模块进行过滤。在酵母菌DIP数据集上运行各个算法可知, FSC-FM与基于不确定图模型的检测蛋白质复合物 (DCU) 算法相比, F-measure值提高了27.92%, 执行效率提高了27.92%;与在动态蛋白质相互作用网络中识别复合物的方法 (CDUN) 、演化算法 (EA) 、医学基因或蛋白质预测算法 (MGPPA) 相比也有更高的F-measure值和执行效率。实验结果表明, 在不确定PPI网络中, FSC-FM适合用于功能模块的挖掘。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E4%B8%8D%E7%A1%AE%E5%AE%9A%E6%95%B0%E6%8D%AE&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">不确定数据;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%9B%B8%E4%BA%92%E4%BD%9C%E7%94%A8&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蛋白质相互作用;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%B0%B1%E8%81%9A%E7%B1%BB%E7%AE%97%E6%B3%95&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">谱聚类算法;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%A8%A1%E7%B3%8A%3Ci%3EC%3C%2Fi%3E-means&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">模糊<i>C</i>-means;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8A%9F%E8%83%BD%E6%A8%A1%E5%9D%97&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">功能模块;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%9C%9F%E6%9C%9B%E7%A8%A0%E5%AF%86%E5%BA%A6&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">期望稠密度;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    毛伊敏 (1970—) , 女, 江西赣州人, 教授, 博士, 主要研究方向:数据挖掘、地理信息系统;;
                                </span>
                                <span>
                                    *刘银萍 (1993—) , 女, 山西朔州人, 硕士研究生, 主要研究方向:数据挖掘、地理信息系统;电子邮箱2304676691@qq.com;
                                </span>
                                <span>
                                    梁田 (1983—) , 女, 江西赣州人, 讲师, 博士, 主要研究方向:数据挖掘;;
                                </span>
                                <span>
                                    毛丁慧 (1993—) , 女, 新疆伊宁人, 工程师, 主要研究方向:数据挖掘。;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-09-10</p>

                    <p>

                            <b>基金：</b>
                                                        <span>国家自然科学基金资助项目 (41562019);</span>
                                <span>江西省教育厅科技项目 (GJJ161566);</span>
                    </p>
            </div>
                    <h1><b>Functional module mining in uncertain protein-protein interaction network based on fuzzy spectral clustering</b></h1>
                    <h2>
                    <span>MAO Yimin</span>
                    <span>LIU Yinping</span>
                    <span>LIANG Tian</span>
                    <span>MAO Dinghui</span>
            </h2>
                    <h2>
                    <span>School of Information Engineering, Jiangxi University of Science and Technology</span>
                    <span>College of Applied Science, Jiangxi University of Science and Technology</span>
                    <span>211 Battalion Company Limited, Sino Shaanxi Nuclear Industry Group</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>Aiming at the problem that Protein-Protein Interaction (PPI) network functional module mining method based on spectral clustering and Fuzzy <i>C</i>-Means (FCM) clustering has low accuracy and low running efficiency, and is susceptible to false positive, a method for Functional Module mining in uncertain PPI network based on Fuzzy Spectral Clustering (FSC-FM) was proposed. Firstly, in order to overcome the effect of false positives, an uncertain PPI network was constructed, in which every protein-protein interaction was endowed with a existence probability measure by using edge aggregation coefficient. Secondly, based on edge aggregation coefficient and flow distance, the similarity calculation of spectral clustering was modified using Flow distance of Edge Clustering coefficient (FEC) strategy to overcome the sensitivity problem of the spectral clustering to the scaling parameters. Then the spectral clustering algorithm was used to preprocess the uncertain PPI network data, reducing the dimension of the data and improving the accuracy of clustering. Thirdly, Density-based Probability Center Selection (DPCS) strategy was designed to solve the problem that FCM algorithm was sensitive to the initial cluster center and clustering numbers, and the processed PPI data was clustered by using FCM algorithm to improve the running efficiency and sensitivity of the clustering. Finally, the mined functional module was filtered by Edge-Expected Density (EED) strategy. Experiments on yeast DIP dataset show that, compared with Detecting protein Complexes based on Uncertain graph model (DCU) algorithm, FSC-FM has F-measure increased by 27.92%, running efficiency increased by 27.92%; compared with an uncertain model-based approach for identifying Dynamic protein Complexes in Uncertain protein-protein interaction Networks (CDUN) , Evolutionary Algorithm (EA) and Medical Gene or Protein Prediction Algorithm (MGPPA) , FSC-FM also has higher F-measure and running efficiency. The experimental results show that FSC-FM is suitable for the functional module mining in the uncertain PPI network.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=uncertain%20data&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">uncertain data;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Protein-Protein%20Interaction%20(PPI)%20&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Protein-Protein Interaction (PPI) ;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=spectral%20clustering%20algorithm&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">spectral clustering algorithm;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Fuzzy%20%3Ci%3EC%3C%2Fi%3E-Means%20(FCM)%20&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Fuzzy <i>C</i>-Means (FCM) ;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=functional%20module&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">functional module;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=expected%20density&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">expected density;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                            <b>Author：</b>
                                                        <span>
                                    MAO Yimin, born in 1970, Ph. D. , professor. Her research interests include data mining, geographic information system.;
                                </span>
                                <span>
                                    LIU Yinping, born in 1993, M. S. candidate. Her research interests include data mining, geographic information system.;
                                </span>
                                <span>
                                    LIANG Tian, born in 1983, Ph. D. , lecturer. Her research interests include data mining.;
                                </span>
                                <span>
                                    MAO Dinghui, born in 1993, engineer. Her research interests include data mining.;
                                </span>
                    </p>
                                    <p><b>Received：</b> 2018-09-10</p>
                                    <p>
                            <b>Fund：</b>
                                                        <span>partially supported by the National Natural Science Foundation of China (41562019);</span>
                                <span>the Science and Technology Project of Education Department of Jiangxi Province (GJJ161566);</span>
                    </p>
            </div>


        <!--brief start-->
                        <h3 id="85" name="85" class="anchor-tag">0 引言</h3>
                <div class="p1">
                    <p id="86">蛋白质组是一个在空间和时间上动态变化的整体, 其功能往往通过蛋白质之间或核酸之间的相互作用而表现出来, 这种相互作用存在于机体细胞的生命活动过程中, 相互交叉形成蛋白质相互作用 (Protein-Protein Interaction, PPI) 网络<citation id="358" type="reference"><link href="290" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>。在一个PPI网络中, 不同时间和空间阶段通过相互作用完成某一特定分子进程的蛋白质集合称为蛋白质功能模块<citation id="359" type="reference"><link href="292" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>。大量的生物实验和计算方法实验产生了大量的蛋白质间相互作用数据, 这些数据是挖掘蛋白质功能模块的基石, 而功能模块对于了解细胞的功能组织结构、执行生理功能方面又是至关重要的<citation id="360" type="reference"><link href="294" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>, 因此, 挖掘蛋白质相互作用的功能模块具有重要的意义。</p>
                </div>
                <div class="p1">
                    <p id="87">迄今为止, 利用计算方法进行蛋白质功能模块挖掘已经是后基因组时代生物信息学领域中一个非常活跃的研究领域<citation id="361" type="reference"><link href="296" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>。根据计算机制的不同, 挖掘蛋白质功能模块的算法大体分为:基于密度的聚类方法<citation id="363" type="reference"><link href="298" rel="bibliography" /><link href="300" rel="bibliography" /><sup>[<a class="sup">5</a>,<a class="sup">6</a>]</sup></citation>、基于层次的聚类方法<citation id="364" type="reference"><link href="302" rel="bibliography" /><link href="304" rel="bibliography" /><sup>[<a class="sup">7</a>,<a class="sup">8</a>]</sup></citation>、基于划分的聚类方法<citation id="365" type="reference"><link href="306" rel="bibliography" /><link href="308" rel="bibliography" /><sup>[<a class="sup">9</a>,<a class="sup">10</a>]</sup></citation>和基于谱分析的聚类方法等。其中:基于密度的聚类方法很难对网络中大量的稀疏节点进行聚类, 算法挖掘的功能模块的准确率不高;基于层次的聚类方法难以检测出节点交叠的功能模块, 聚类结果对网络的噪声非常敏感;基于划分的聚类方法需要事先确定聚类数目, 不能检测出重叠的功能模块;而基于图论的谱聚类算法实现简单, 不局限于原始数据的分布形状, 可以收敛于全局最优解<citation id="362" type="reference"><link href="310" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>, 因此, 目前谱聚类算法已成功应用于PPI网络功能模块挖掘, 成为该领域的研究热点。</p>
                </div>
                <div class="p1">
                    <p id="88">Madani等<citation id="366" type="reference"><link href="312" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>提出了一种新的基于谱聚类的功能模块挖掘算法, 用于挖掘整个PPI网络最相似的功能模块。Qin等<citation id="367" type="reference"><link href="314" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>利用谱聚类方法对PPI网络模块识别进行了研究, 提出一种基于PPI网络属性确定模块数的方法, 并且进行了相关验证。Inoue等<citation id="368" type="reference"><link href="316" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>提出了一种可调扩散矩阵谱聚类 (Adjustable Diffusion Matrix-based Spectral Clustering, ADMSC) 方法, 该方法用于PPI网络模块划分挖掘。这些算法根据谱聚类算法中的特征向量将数据划分到不相交的类中, 属于且仅属于一个类, 可以自动确定聚类数目, 是一种硬划分方法, 不能准确反映样本间的实际关系;另计算相似度矩阵时, 实验结果容易受到尺度参数的影响, 导致功能模块挖掘过程中不能充分考虑节点的局部一致性和全局一致性, 进而使得算法的运行效率降低以及准确性不高。为了解决谱聚类算法的硬划分问题, 文献<citation id="370" type="reference">[<a class="sup">15</a>,<a class="sup">16</a>]</citation>提出将模糊<i>C</i>-means (Fuzzy <i>C</i>-Means, FCM) 与谱聚类算法相结合用于蛋白质模块挖掘, 利用FCM算法中的模糊因子改进谱聚类的硬划分问题, 不断更新聚类中心隶属度来划分簇;但划分结果存在对初始聚类中心以及聚类数目敏感的问题, 导致功能模块挖掘的过程中容易陷入局部最优, 算法的预测精度降低以及特异性和灵敏度不高。然而上述研究都是将PPI网络有效地用无向图模型来描述, 只关注于精确的、完全的确定图, 忽略了生物信息学中的PPI网络数据以及其他的一些生物数据常常会由于实验检测方法的局限性而呈现出不确定性<citation id="369" type="reference"><link href="322" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>, 实验结果容易受到假阳性的影响, 因此, 将PPI网络作为不确定图来研究更为合理。</p>
                </div>
                <div class="p1">
                    <p id="89">目前从不确定性的数据中挖掘蛋白质功能模块信息越来越受到人们的关注。基于不确定模型, Zhang等<citation id="371" type="reference"><link href="324" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>提出了一种在动态蛋白质相互作用网络中识别复合物的方法 (an uncertain model-based approach for identifying Dynamic protein Complexes in Uncertain protein-protein interaction Networks, CDUN) 用于识别蛋白质功能模块;Zhao等<citation id="372" type="reference"><link href="326" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>提出了一种基于不确定图模型的检测蛋白质复合物 (Detecting protein Complexes based on Uncertain graph model, DCU) 算法;Halim等<citation id="373" type="reference"><link href="328" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>提出了一种从不确定蛋白质网络概率图中聚类子图模块的演化算法 (Evolutionary Algorithm, EA) ;Bano等<citation id="374" type="reference"><link href="330" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>在不确定数据基础上提出了医学基因或蛋白质预测算法 (Medical Gene or Protein Prediction Algorithm, MGPPA) 应用于蛋白质簇的挖掘。这些方法克服了假阳性对实验结果的影响, 有很好的预测精度和很强的鲁棒性, 但是聚类结果的灵敏度和准确率不高。虽然基于不确定PPI网络的功能模块挖掘取得了一定的成效, 但是如何有效地构建不确定PPI, 如何克服谱聚类融合FCM算法对尺度参数、聚类中心和聚类数目敏感等导致的准确率、灵敏度不高以及执行效率低等缺陷, 仍是亟待解决的问题。</p>
                </div>
                <div class="p1">
                    <p id="90">针对以上问题, 本文提出了基于模糊谱聚类的不确定PPI网络功能模块挖掘 (Functional Module mining in uncertain PPI network based on Fuzzy Spectral Clustering, FSC-FM) 方法。 本文主要工作为:1) 利用边聚集系数构建不确定PPI网络;2) 结合边聚集系数和流行距离, 提出了边聚集系数流行距离 (Flow distance of Edge Clustering coefficient, FEC) 策略来计算蛋白质节点之间的相似度矩阵, 克服了谱聚类算法对尺度参数的敏感的缺陷;3) 根据基于密度的概率中心优化策略 (Density-based Probability Center Selection, DPCS) , 优化FCM算法对初始聚类中心的选取, 降低离群数据对整个数据的影响, 确定聚类数目, 进而提高算法的运行效率;4) 利用改进的边期望稠密度 (Edge-Expected Density, EED) 度量来对挖掘出的模块进行过滤。实验结果表明本文方法收敛快、聚类精度高、运行效率高, 聚类结果的准确率以及灵敏度较高。</p>
                </div>
                <h3 id="91" name="91" class="anchor-tag">1 基本概念</h3>
                <div class="p1">
                    <p id="92">由于PPI网络可以模型化为一个图, 节点代表蛋白质, 边代表蛋白质之间的相互作用, 因此, 具有不确定性的PPI网络可用不确定图来表示, 下面给出基本概念。</p>
                </div>
                <div class="p1">
                    <p id="93"><b>定义</b>1 不确定图<citation id="375" type="reference"><link href="332" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>。无向图<i>G</i>= (<i>V</i>, <i>E</i>, <i>P</i>) , 其中:<i>V</i>= (<i>v</i><sub>1</sub>, <i>v</i><sub>2</sub>, …, <i>v</i><sub><i>n</i></sub>) , 代表图中顶点集合;<i>E</i>= (<i>e</i><sub>1</sub>, <i>e</i><sub>2</sub>, …, <i>e</i><sub><i>m</i></sub>) , 代表图中边的集合;<i>E</i>中的每条边存在的概率用<i>P</i>表示, <i>P</i>= (<i>p</i> (<i>e</i><sub>1</sub>) , <i>p</i> (<i>e</i><sub>2</sub>) , …, <i>p</i> (<i>e</i><sub><i>m</i></sub>) ) 。</p>
                </div>
                <div class="p1">
                    <p id="94"><b>定义</b>2 边聚集系数<citation id="376" type="reference"><link href="334" rel="bibliography" /><sup>[<a class="sup">23</a>]</sup></citation>。无向图<i>G</i>= (<i>V</i>, <i>E</i>, <i>P</i>) , 其中:节点<i>u</i>, <i>v</i>∈<i>V</i>; (<i>u</i>, <i>v</i>) ∈<i>E</i>;<i>H</i><sub><i>c</i></sub>表示节点<i>u</i>和<i>v</i>所共有的邻居节点数;<i>d</i><sub><i>u</i></sub>和<i>d</i><sub><i>v</i></sub>分别代表节点<i>u</i>和<i>v</i>节点的度。无向图中的边聚集系数 (Edge Clustering Coefficient, ECC) 定义为:</p>
                </div>
                <div class="p1">
                    <p id="95" class="code-formula">
                        <mathml id="95"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>E</mi><mi>C</mi><mi>C</mi><mo>=</mo><mfrac><mrow><mi>Η</mi><msub><mrow></mrow><mi>c</mi></msub></mrow><mrow><mi>min</mi><mo stretchy="false"> (</mo><mi>d</mi><msub><mrow></mrow><mi>u</mi></msub><mo>-</mo><mn>1</mn><mo>, </mo><mi>d</mi><msub><mrow></mrow><mi>v</mi></msub><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="96"><b>定义</b>3 度量空间<citation id="377" type="reference"><link href="336" rel="bibliography" /><sup>[<a class="sup">24</a>]</sup></citation>。设<i>X</i>为一非空集合, 对于∀<i>x</i>, <i>y</i>, <i>z</i>∈<i>X</i>, 如果映射<i>h</i>满足:</p>
                </div>
                <div class="p1">
                    <p id="97">1) 对称性, <i>h</i> (<i>x</i>, <i>y</i>) =<i>h</i> (<i>y</i>, <i>x</i>) ;</p>
                </div>
                <div class="p1">
                    <p id="98">2) 非负性, <i>h</i> (<i>x</i>, <i>y</i>) ≥0;</p>
                </div>
                <div class="p1">
                    <p id="99">3) 自反性, <i>h</i> (<i>x</i>, <i>y</i>) =0当且仅当<i>x</i>=<i>y</i>;</p>
                </div>
                <div class="p1">
                    <p id="100">4) 三角不等式, ∀<i>x</i>, <i>y</i>, <i>z</i>, <i>h</i> (<i>x</i>, <i>z</i>) ≤<i>h</i> (<i>x</i>, <i>y</i>) +<i>h</i> (<i>y</i>, <i>z</i>) 。</p>
                </div>
                <div class="p1">
                    <p id="101">则称映射<i>h</i>为集合<i>X</i>的一个度量或者距离函数, 而抽象系统{<i>X</i>, <i>h</i>}称为一个度量空间。</p>
                </div>
                <div class="p1">
                    <p id="102"><b>定义</b>4 可能实例图<citation id="378" type="reference"><link href="338" rel="bibliography" /><sup>[<a class="sup">25</a>]</sup></citation>。不确定图<i>G</i>= (<i>V</i>, <i>E</i>, <i>P</i>) 的一种可能实例图记作<i>G</i>′= (<i>V</i>′, <i>E</i>′) , 其中:<i>V</i>′⊆<i>V</i>, <i>E</i>′⊆<i>E</i>, <i>G</i>′与<i>G</i>之间的关系记作<i>G</i>′ ⇒ <i>G</i>。实例图被不确定图蕴含的概率为:</p>
                </div>
                <div class="p1">
                    <p id="103" class="code-formula">
                        <mathml id="103"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ρ</mi><mo stretchy="false"> (</mo><msup><mi>G</mi><mo>′</mo></msup><mspace width="0.25em" /><mo>⇒</mo><mspace width="0.25em" /><mi>G</mi><mo stretchy="false">) </mo><mo>=</mo><mstyle displaystyle="true"><munder><mo>∏</mo><mrow><mi>e</mi><mo>∈</mo><msup><mi>E</mi><mo>′</mo></msup></mrow></munder><mi>p</mi></mstyle><mo stretchy="false"> (</mo><mi>e</mi><mo stretchy="false">) </mo><mstyle displaystyle="true"><munder><mo>∏</mo><mrow><mi>e</mi><mo>∈</mo><mo stretchy="false"> (</mo><mi>E</mi><mo>\</mo><msup><mi>E</mi><mo>′</mo></msup><mo stretchy="false">) </mo></mrow></munder><mo stretchy="false"> (</mo></mstyle><mn>1</mn><mo>-</mo><mi>p</mi><mo stretchy="false"> (</mo><mi>e</mi><mo stretchy="false">) </mo><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>2</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="104"><b>定义</b>5 期望稠密度<citation id="379" type="reference"><link href="340" rel="bibliography" /><sup>[<a class="sup">26</a>]</sup></citation>。不确定图<i>G</i>= (<i>V</i>, <i>E</i>, <i>P</i>) , <i>G</i>所蕴含的确定图为集合为<i>DG</i>={<i>g</i><sub>1</sub>, <i>g</i><sub>2</sub>, …, <i>g</i><sub><i>n</i></sub>}, 其中<i>n</i>=2<sup>|<i>E</i>|</sup>, 则出现在不确定图<i>G</i>中的子图<i>S</i>= (<i>V</i>", <i>E</i>", <i>P</i>") 的期望稠密度 (Expected Density, ED) 为:</p>
                </div>
                <div class="p1">
                    <p id="105" class="code-formula">
                        <mathml id="105"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>E</mi><mi>D</mi><mo>=</mo><mfrac><mn>2</mn><mrow><mo stretchy="false">|</mo><mi>V</mi><mo>"</mo><mo stretchy="false">|</mo><mo stretchy="false"> (</mo><mo stretchy="false">|</mo><mi>V</mi><mo>"</mo><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>g</mi><mo>∈</mo><mi>D</mi><mi>G</mi></mrow></munder><mi>Ρ</mi></mstyle><mo stretchy="false"> (</mo><mi>G</mi><mspace width="0.25em" /><mo>⇒</mo><mspace width="0.25em" /><mi>g</mi><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>3</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="106"><b>定义</b>6 流行距离<citation id="380" type="reference"><link href="342" rel="bibliography" /><sup>[<a class="sup">27</a>]</sup></citation>。在图<i>G</i>上任意两个顶点<i>b</i><sub>0</sub>, <i>b</i><sub><i>s</i></sub>存在顶点序列<i>r</i>= (<i>b</i><sub>0</sub>, <i>b</i><sub>1</sub>, …, <i>b</i><sub><i>w</i></sub>) 表示连接这两个顶点的路径, 其中<i>b</i><sub><i>q</i></sub>∈<i>V</i> (0≤<i>q</i>≤<i>w</i>) , (<i>b</i><sub><i>q</i></sub>, <i>b</i><sub><i>q</i>+1</sub>) ∈<i>E</i> (0≤<i>q</i>≤<i>w</i>) , 令<i>R</i><sub>0<i>j</i></sub>表示<i>G</i>上连接两节点之间全部可能的到达路径集合, 顶点<i>b</i><sub>0</sub>, <i>b</i><sub><i>j</i></sub>之间的流行距离 (Flow Distance, FD) 定义如下:</p>
                </div>
                <div class="p1">
                    <p id="107" class="code-formula">
                        <mathml id="107"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>F</mi><mi>D</mi><mo stretchy="false"> (</mo><mi>b</mi><msub><mrow></mrow><mn>0</mn></msub><mo>, </mo><mi>b</mi><msub><mrow></mrow><mi>j</mi></msub><mo stretchy="false">) </mo><mo>=</mo><mfrac><mn>1</mn><mrow><munder><mstyle mathsize="140%" displaystyle="true"><mrow><mi>min</mi></mrow></mstyle><mrow><mi>b</mi><mo>∈</mo><mi>R</mi><msub><mrow></mrow><mrow><mi>o</mi><mi>j</mi></mrow></msub></mrow></munder><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>q</mi><mo>=</mo><mn>1</mn></mrow><mrow><mo stretchy="false">|</mo><mi>w</mi><mo stretchy="false">|</mo><mo>-</mo><mn>1</mn></mrow></munderover><mo stretchy="false"> (</mo></mstyle><mi>e</mi><msup><mrow></mrow><mrow><mi>ρ</mi><mi>d</mi><msup><mrow></mrow><mn>2</mn></msup><mo stretchy="false"> (</mo><mi>b</mi><msub><mrow></mrow><mi>q</mi></msub><mo>, </mo><mi>b</mi><msub><mrow></mrow><mrow><mi>q</mi><mo>+</mo><mn>1</mn></mrow></msub><mo stretchy="false">) </mo></mrow></msup><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo><mo>+</mo><mn>1</mn></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>4</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="108">其中:<i>de</i> (<i>b</i><sub><i>q</i></sub>, <i>b</i><sub><i>q</i>+1</sub>) 表示数据节点<i>b</i><sub><i>q</i></sub>和<i>b</i><sub><i>q</i>+1</sub>间的欧氏距离;伸缩因子<i>ρ</i> (<i>ρ</i>&gt;1) 是可调节的参数。</p>
                </div>
                <h3 id="109" name="109" class="anchor-tag">2 本文FSC-FM算法</h3>
                <h4 class="anchor-tag" id="110" name="110">2.1 <b>模糊谱聚类算法</b></h4>
                <div class="p1">
                    <p id="111">模糊谱聚类是将谱聚类与FCM算法融合在一起所得到的, 其中谱聚类算法是建立在谱图划分理论基础上, 将数据点看成是一个无向图<i>G</i>= (<i>V</i>, <i>W</i>) 的顶点<i>V</i>, 边权重的集合<i>W</i>={<b><i>U</i></b><sub><i>ij</i></sub>}表示基于高斯核函数度量的两个数据点之间的相似度, <b><i>U</i></b>表示待聚类数据点间的相似度矩阵, 其本质是利用相似度矩阵的特征向量以及结合FCM完成聚类。划分的准则是:子图内的相关性最大, 各个子图间的相关性最小<citation id="381" type="reference"><link href="344" rel="bibliography" /><sup>[<a class="sup">28</a>]</sup></citation>。FCM算法<citation id="382" type="reference"><link href="346" rel="bibliography" /><sup>[<a class="sup">29</a>]</sup></citation>的基本思想是基于目标函数的隶属度矩阵来确定每个样本与所有簇的关联强度, 不断更新聚类中心和隶属度将样本划分到与其关联强度最大的簇中完成聚类。目前, 大量研究者将模糊谱聚类应用到蛋白质网络中, 用于功能模块挖掘<citation id="383" type="reference"><link href="318" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation><sup>24, </sup><citation id="384" type="reference"><link href="320" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation><sup>112</sup>。由于蛋白质相互作用网络本身存在的不确定性, 功能模块挖掘容易受到假阳性的影响;谱聚类算法中的数据降维处理效率受到尺度参数影响较大以及FCM聚类结果受初始聚类中心、聚类数目敏感。为提高算法的执行效率、准确率、灵敏度以及避免假阳性的影响, 本文提出了一种有效的挖掘蛋白质功能模块方法FSC-FM。FSC-FM方法包括:不确定PPI网络的构建、相似度改进的FEC策略、概率密度中心的DPCS策略和期望稠密度优化的EED度量。</p>
                </div>
                <h4 class="anchor-tag" id="112" name="112">2.2 FSC-FM<b>方法的优化策略</b></h4>
                <h4 class="anchor-tag" id="113" name="113">2.2.1 不确定<i>PPI</i>网络的构建</h4>
                <div class="p1">
                    <p id="114">由于受到实验检测条件的局限性以及蛋白质网络的拓扑特性, 蛋白质相互作用网络和生物信息学中的一些生物数据存在不确定性, 实验结果容易受到假阳性的影响。为了降低实验结果受假阳性的影响, 融合不确定数据处理技术提高<i>PPI</i>网络功能模块预测的准确率, 本文将<i>PPI</i>网络用不确定图来表示。通过计算<i>PPI</i>网络图中连接每条边的两个节点的公共邻居节点数以及选取这两个节点度的最小值, 利用边聚集系数定义公式来测度每一组相互作用, 构建不确定<i>PPI</i>网络。图1描述了如何将一个<i>PPI</i>网络构建成一个不确定网络, 其中:图1 (<i>a</i>) 给出包含8个蛋白质和18个蛋白质间相互作用;图1 (<i>b</i>) 是构造的不确定网络, 每一个相互作用的测度通过边聚集系数计算得到。构造的不确定网络由2<sup>18</sup>个可能的实例网络组成。</p>
                </div>
                <div class="area_img" id="115">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201904017_115.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 不确定网络的构建" src="Detail/GetImg?filename=images/JSJY201904017_115.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 不确定网络的构建  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201904017_115.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"><i>Fig</i>. 1 <i>Construction of uncertain network</i></p>

                </div>
                <h4 class="anchor-tag" id="116" name="116">2.2.2 相似度改进的<i>FEC</i>策略</h4>
                <div class="p1">
                    <p id="117">针对谱聚类算法采用传统的高斯核函数来度量蛋白质节点间的相似性, 仅仅能反映聚类结构的局部一致性特征, 而且构造相似度矩阵时对尺度参数比较敏感, 计算复杂度较高, 导致执行效率和准确率降低。为了解决这问题, 在不确定<i>PPI</i>网络中, 根据蛋白质网络的拓扑特性即聚集程度以及流行距离来改进相似性度量, 提出了<i>FEC</i>策略。</p>
                </div>
                <div class="p1">
                    <p id="118"><b>定理</b>1 <i>FEC</i>策略。在不确定图<i>G</i>= (<i>V</i>, <i>E</i>, <i>P</i>) 中, <i>V</i>代表蛋白质, <i>E</i>代表蛋白质之间的相互作用, <i>P</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) =<i>ECC</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) 表示任意边 (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) ∈<i>E</i>的存在概率, <i>FD</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) 表示这两个节点之间的流行距离, 则两个蛋白质<i>b</i><sub><i>i</i></sub>、<i>b</i><sub><i>j</i></sub>之间的相似性可以通过式 (5) 计算得到:</p>
                </div>
                <div class="p1">
                    <p id="119"><i>FEC</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) =<i>ECC</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) ×<i>FD</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>)      (5) </p>
                </div>
                <div class="p1">
                    <p id="120">证明</p>
                </div>
                <div class="p1">
                    <p id="121">1) 对于∀<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>, <i>FEC</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) =<i>FEC</i> (<i>b</i><sub><i>j</i></sub>, <i>b</i><sub><i>i</i></sub>) , 对称性满足;</p>
                </div>
                <div class="p1">
                    <p id="122">2) 对于∀<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>, <i>ECC</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) ≥0且<i>FD</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) ≥0, 则<i>FEC</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) ≥0, 非负性满足;</p>
                </div>
                <div class="p1">
                    <p id="123">3) 对于∀<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>, <i>b</i><sub><i>z</i></sub>, <i>FEC</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>j</i></sub>) +<i>FEC</i> (<i>b</i><sub><i>j</i></sub>, <i>b</i><sub><i>z</i></sub>) ≥<i>FEC</i> (<i>b</i><sub><i>i</i></sub>, <i>b</i><sub><i>z</i></sub>) , 三角不等式满足。</p>
                </div>
                <div class="p1">
                    <p id="124">因此式 (5) 满足度量空间定义的基本条件, 是距离度量公式。</p>
                </div>
                <h4 class="anchor-tag" id="125" name="125">2.2.3 概率密度中心的DPCS策略</h4>
                <div class="p1">
                    <p id="126">针对<i>FCM</i>算法融合谱聚类用于蛋白质功能模块的挖掘, 利用<i>FCM</i>算法中的模糊因子改进谱聚类算法的硬划分问题, 不断更新聚类中心以及隶属度来划分簇, 划分结果却存在对初始聚类中心以及聚类数目敏感的问题。若初始聚类中心选择存在偏差, 可能会导致聚类结果与实际情况存在较大差异, 挖掘功能模块容易陷入局部最优解, 算法的精度以及准确率降低。本文通过计算样本数据间的几何分布紧密程度得到相应的密度中心, 并将得到的样本数据密度中心代入<i>FSC</i>-<i>FM</i>算法近似模拟全体数据的初始聚类中心, 对使用<i>FEC</i>策略的谱聚类算法预处理后的数据实现蛋白质功能模块的挖掘。该方法可以避免<i>FCM</i>算法陷入局部最优并且减少算法迭代次数, 能够提高算法的运行效率和精度。<i>DPCS</i>策略算法思想如下:</p>
                </div>
                <div class="p1">
                    <p id="127">步骤1 利用FEC距离度量公式计算数据节点<i>X</i>={<i>x</i><sub>1</sub>, <i>x</i><sub>2</sub>, …, <i>x</i><sub><i>n</i></sub>}之间的相似度。</p>
                </div>
                <div class="p1">
                    <p id="128">步骤2 在步骤1的基础上计算样本的概率密度函数, 对象<i>x</i><sub><i>i</i></sub>处的概率函数定义为:</p>
                </div>
                <div class="p1">
                    <p id="129" class="code-formula">
                        <mathml id="129"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>D</mi><msubsup><mrow></mrow><mi>i</mi><mrow><mo stretchy="false"> (</mo><mn>0</mn><mo stretchy="false">) </mo></mrow></msubsup><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mspace width="0.25em" /></mstyle><mfrac><mn>1</mn><mrow><mn>1</mn><mo>+</mo><mfrac><mrow><mn>4</mn><mi>F</mi><mi>E</mi><mi>C</mi><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mi>i</mi></msub><mo>, </mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo stretchy="false">) </mo></mrow><mrow><mi>r</mi><msubsup><mrow></mrow><mi>d</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>6</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="130">其中<i>r</i><sub><i>d</i></sub>为邻域半径, 其数值是<i>N</i>个对象的平均距离度量, 即:</p>
                </div>
                <div class="p1">
                    <p id="131" class="code-formula">
                        <mathml id="131"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>r</mi><msub><mrow></mrow><mi>d</mi></msub><mo>=</mo><msqrt><mrow><mfrac><mn>1</mn><mrow><mi>Ν</mi><mo stretchy="false"> (</mo><mi>Ν</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mi>F</mi></mstyle></mrow></mstyle><mi>E</mi><mi>C</mi><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mi>i</mi></msub><mo>, </mo><mi>x</mi><msub><mrow></mrow><mi>j</mi></msub><mo stretchy="false">) </mo></mrow></msqrt><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>7</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="132">显然, <i>x</i><sub><i>i</i></sub>与其他元素的紧密度越大, <i>r</i><sub><i>d</i></sub>越小, 相对应的概率函数值越大。</p>
                </div>
                <div class="p1">
                    <p id="133">步骤3 得到初始聚类中心。</p>
                </div>
                <div class="p1">
                    <p id="134">设<i>D</i><sup>*</sup><sub><i>k</i></sub>=max{<i>D</i><mathml id="135"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>i</mi><mrow><mo stretchy="false"> (</mo><mi>k</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></msubsup></mrow></math></mathml>|<i>i</i>=1, 2, …, <i>N</i>}为第<i>k</i>个聚类中心点 (即<i>c</i><sup>*</sup><sub><i>k</i></sub>) , 则概率函数的更新公式如式 (8) 所示:</p>
                </div>
                <div class="p1">
                    <p id="136"><mathml id="137"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>D</mi><msubsup><mrow></mrow><mi>i</mi><mrow><mo stretchy="false"> (</mo><mi>k</mi><mo stretchy="false">) </mo></mrow></msubsup><mo>=</mo><mi>D</mi><msubsup><mrow></mrow><mi>i</mi><mrow><mo stretchy="false"> (</mo><mi>k</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></msubsup><mo>-</mo><mi>D</mi><msubsup><mrow></mrow><mi>k</mi><mo>*</mo></msubsup><mfrac><mn>1</mn><mrow><mn>1</mn><mo>+</mo><mfrac><mrow><mn>4</mn><mi>F</mi><mi>E</mi><mi>C</mi><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mi>i</mi></msub><mo>, </mo><mi>c</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">) </mo></mrow><mrow><mi>r</mi><msubsup><mrow></mrow><mi>d</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></mfrac></mrow></math></mathml>; <i>k</i>=1, 2, …, <i>N</i>      (8) </p>
                </div>
                <div class="p1">
                    <p id="138">步骤4 在步骤3获得初始聚类中心的基础上, 结合FCM算法的目标函数更新聚类中心和隶属度, 那么目标函数的定义如式 (9) 所示:</p>
                </div>
                <div class="p1">
                    <p id="139" class="code-formula">
                        <mathml id="139"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>J</mi><mo stretchy="false"> (</mo><mi>u</mi><mo>, </mo><mi>c</mi><mo stretchy="false">) </mo><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mi>u</mi></mstyle></mrow></mstyle><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><msup><mrow></mrow><mi>m</mi></msup><mi>d</mi><mi>e</mi><mo stretchy="false"> (</mo><mi>x</mi><msub><mrow></mrow><mi>i</mi></msub><mo>, </mo><mi>c</mi><msub><mrow></mrow><mi>j</mi></msub><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>9</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="140">最小化目标函数得到隶属度<i>u</i><sub><i>ij</i></sub>和聚类中心<i>c</i><sub><i>j</i></sub>的迭代更新表达如式 (10) ～ (11) 所示:</p>
                </div>
                <div class="area_img" id="289">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JSJY201904017_28900.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="143"><mathml id="144"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>c</mi><msub><mrow></mrow><mi>j</mi></msub><mo>=</mo><mfrac><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mi>u</mi></mstyle><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><msup><mrow></mrow><mi>m</mi></msup><mo>⋅</mo><mi>x</mi><msub><mrow></mrow><mi>i</mi></msub></mrow><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mi>u</mi></mstyle><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><msup><mrow></mrow><mi>m</mi></msup></mrow></mfrac></mrow></math></mathml>; <i>j</i>=1, 2, …, <i>K</i>      (11) </p>
                </div>
                <div class="p1">
                    <p id="145">其中:<i>m</i>&gt;1为模糊加权指数;<i>K</i>为类别数;<i>u</i><sub><i>ij</i></sub>为节点<i>x</i><sub><i>i</i></sub>在<i>j</i>类中的模糊隶属度。</p>
                </div>
                <div class="p1">
                    <p id="146">由式 (8) 可看出, 距离聚类中心点越近, 对应的概率更新值就越小。当<i>D</i><sup>*</sup><sub><i>c</i></sub>&lt;<i>δD</i><sup>*</sup><sub>1</sub>迭代停止, 这样可以得到<i>K</i>个全局密度较大的数据点作为聚类的初始聚类中心, 利用FCM聚类算法进行功能模块挖掘。如此过程, 高密度样本而非边缘离群点处于类别的中心处, 使得选取的类中心点尽量属于不同的类别, 可以得到<i>K</i>个初始类别中心点, 降低噪声点对实验结果的影响。根据FCM目标函数来不断迭代更新聚类中心以及隶属度, 优化FCM算法对初始聚类中心敏感的问题, 进而挖掘蛋白质功能模块。</p>
                </div>
                <h4 class="anchor-tag" id="147" name="147">2.2.4 期望稠密度优化的EED度量</h4>
                <div class="p1">
                    <p id="148">随着数据的逐渐增多, 图的规模也相应地增加, 不确定图所蕴含的确定图数目呈指数形式增加, 不确定图蕴含的确定图的期望密度的计算量是指数级的, 导致子图模式在不确定图中的期望稠密度的计算十分复杂。针对此问题, 基于2.1.1节边聚集系数构建的不确定<i>PPI</i>网络图, 提出了子图在不确定图中的期望稠密度优化<i>EED</i>度量, 充分考虑节点的邻域信息以及<i>PPI</i>网络内部聚集程度, 降低计算复杂度, 进而提高计算效率。本文利用<i>EED</i>优化策略对算法挖掘出的功能模块进行过滤, 将低于<i>EED</i>阈值<i>T</i>的模块过滤掉, 避免重复划分, 提高算法的预测率。</p>
                </div>
                <div class="p1">
                    <p id="149"><b>定理</b>2 EED度量。设不确定图<i>G</i>= (<i>V</i>, <i>E</i>, <i>P</i>) 中存在的子图<i>S</i>= (<i>V</i>", <i>E</i>", <i>P</i>") , 其中<i>V</i>"⊆<i>V</i>, <i>E</i>"⊆<i>E</i>, <i>P</i> (<i>e</i>) =<i>ECC</i> (<i>e</i>) 表示边<i>e</i>存在的概率, 则<i>S</i>在<i>G</i>中的期望稠密度定义为:</p>
                </div>
                <div class="p1">
                    <p id="150" class="code-formula">
                        <mathml id="150"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>E</mi><mi>E</mi><mi>D</mi><mo>=</mo><mfrac><mn>2</mn><mrow><mo stretchy="false">|</mo><mi>V</mi><mo>"</mo><mo stretchy="false">|</mo><mo>×</mo><mo stretchy="false"> (</mo><mo stretchy="false">|</mo><mi>V</mi><mo>"</mo><mo stretchy="false">|</mo><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mi>e</mi><mo>∈</mo><mi>E</mi><mo>"</mo></mrow></munder><mi>E</mi></mstyle><mi>C</mi><mi>C</mi><mo stretchy="false"> (</mo><mi>e</mi><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>2</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="151">证明 设子图<i>S</i>具有<i>M</i>个顶点、<i>I</i>条边, 用边聚集系数<i>ECC</i><sub><i>i</i></sub>来测度边<i>i</i>∈<i>I</i>的存在概率, 故</p>
                </div>
                <div class="p1">
                    <p id="152" class="code-formula">
                        <mathml id="152"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>E</mi><mi>E</mi><mi>D</mi><mo>=</mo><mfrac><mn>2</mn><mrow><mi>Μ</mi><mo>×</mo><mo stretchy="false"> (</mo><mi>Μ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>Ι</mi></munderover><mi>E</mi></mstyle><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>i</mi></msub><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>3</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="153">对于子图<i>S</i>, 蕴含2<sup><i>n</i></sup>个确定图, 按照边的数目递增的顺序标记为<i>S</i><sub><i>fj</i></sub> (<i>f</i>=1, 2, …, <i>I</i>; <i>j</i>=1, 2, …, <i>C</i><mathml id="154"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msubsup><mrow></mrow><mi>Ι</mi><mi>f</mi></msubsup></mrow></math></mathml>) 。其中: <i>f</i>代表确定图中的边数; <i>j</i>代表相同边数下确定图的编号, 则相应的稠密度的定义为:</p>
                </div>
                <div class="p1">
                    <p id="155" class="code-formula">
                        <mathml id="155"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>d</mi><mi>e</mi><mi>n</mi><mo stretchy="false"> (</mo><mi>S</mi><msub><mrow></mrow><mrow><mi>f</mi><mi>j</mi></mrow></msub><mo stretchy="false">) </mo><mo>=</mo><mfrac><mrow><mn>2</mn><mi>f</mi></mrow><mrow><mi>Μ</mi><mo stretchy="false"> (</mo><mi>Μ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>4</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="156">因为<i>G</i>中的边都是独立存在, 子图<i>S</i>以外的边对<i>S</i>的稠密度计算无影响, 所以:</p>
                </div>
                <div class="p1">
                    <p id="157" class="code-formula">
                        <mathml id="157"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>E</mi><mi>E</mi><mi>D</mi><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>f</mi><mo>=</mo><mn>1</mn></mrow><mi>Ι</mi></munderover><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>C</mi><msubsup><mrow></mrow><mi>Ι</mi><mi>f</mi></msubsup></mrow></munderover><mi>d</mi></mstyle></mrow></mstyle><mi>e</mi><mi>n</mi><mo stretchy="false"> (</mo><mi>S</mi><msub><mrow></mrow><mrow><mi>f</mi><mi>j</mi></mrow></msub><mo stretchy="false">) </mo><mi>E</mi><mi>C</mi><mi>C</mi><mo stretchy="false"> (</mo><mi>S</mi><msub><mrow></mrow><mrow><mi>f</mi><mi>j</mi></mrow></msub><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>5</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="158">将式 (14) 代入式 (15) 中得到:</p>
                </div>
                <div class="p1">
                    <p id="159" class="code-formula">
                        <mathml id="159"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mi>E</mi><mi>E</mi><mi>D</mi><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>f</mi><mo>=</mo><mn>1</mn></mrow><mi>Ι</mi></munderover><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>C</mi><msubsup><mrow></mrow><mi>Ι</mi><mi>f</mi></msubsup></mrow></munderover><mspace width="0.25em" /></mstyle></mrow></mstyle><mfrac><mrow><mn>2</mn><mi>f</mi></mrow><mrow><mi>Μ</mi><mo>×</mo><mo stretchy="false"> (</mo><mi>Μ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mi>E</mi><mi>C</mi><mi>C</mi><mo stretchy="false"> (</mo><mi>S</mi><msub><mrow></mrow><mrow><mi>f</mi><mi>j</mi></mrow></msub><mo stretchy="false">) </mo><mo>=</mo></mtd></mtr><mtr><mtd><mfrac><mn>2</mn><mrow><mi>Μ</mi><mo>×</mo><mo stretchy="false"> (</mo><mi>Μ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>f</mi><mo>=</mo><mn>1</mn></mrow><mi>Ι</mi></munderover><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>C</mi><msubsup><mrow></mrow><mi>Ι</mi><mi>f</mi></msubsup></mrow></munderover><mspace width="0.25em" /></mstyle></mrow></mstyle><mi>f</mi><mi>E</mi><mi>C</mi><mi>C</mi><mo stretchy="false"> (</mo><mi>S</mi><msub><mrow></mrow><mrow><mi>f</mi><mi>j</mi></mrow></msub><mo stretchy="false">) </mo><mo>=</mo></mtd></mtr><mtr><mtd><mfrac><mn>2</mn><mrow><mi>Μ</mi><mo>×</mo><mo stretchy="false"> (</mo><mi>Μ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mo stretchy="false">[</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo stretchy="false">) </mo><mo>⋯</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>n</mi></msub><mo stretchy="false">) </mo><mo>+</mo></mtd></mtr><mtr><mtd><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">) </mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo>⋯</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">) </mo><mo>+</mo><mo>⋯</mo><mo>+</mo></mtd></mtr><mtr><mtd><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">) </mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo stretchy="false">) </mo><mo>⋯</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>n</mi></msub><mo>+</mo><mn>2</mn><mo>×</mo></mtd></mtr><mtr><mtd><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>3</mn></msub><mo stretchy="false">) </mo><mo>⋯</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">) </mo><mo>+</mo></mtd></mtr><mtr><mtd><mn>2</mn><mo>×</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">) </mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>3</mn></msub><mo>⋯</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">) </mo><mo>+</mo><mo>⋯</mo><mo>+</mo></mtd></mtr><mtr><mtd><mi>Ι</mi><mo>×</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo>⋯</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="160">而式 (13) 的右侧展开是:</p>
                </div>
                <div class="p1">
                    <p id="161" class="code-formula">
                        <mathml id="161"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mfrac><mrow><mn>2</mn><mo>×</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>Ι</mi></munderover><mi>E</mi></mstyle><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>i</mi></msub></mrow><mrow><mi>Μ</mi><mo>×</mo><mo stretchy="false"> (</mo><mi>Μ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mo>=</mo><mfrac><mn>2</mn><mrow><mi>Μ</mi><mo>×</mo><mo stretchy="false"> (</mo><mi>Μ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mo stretchy="false"> (</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo>+</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo>+</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>3</mn></msub><mo>+</mo></mtd></mtr><mtr><mtd><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext></mrow><mo>⋯</mo><mo>+</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">) </mo><mo>=</mo><mfrac><mn>2</mn><mrow><mi>Μ</mi><mo>×</mo><mo stretchy="false"> (</mo><mi>Μ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mo stretchy="false">[</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">[</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo>+</mo></mtd></mtr><mtr><mtd><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo stretchy="false">) </mo><mo stretchy="false">]</mo><mo stretchy="false">[</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>3</mn></msub><mo>+</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>3</mn></msub><mo stretchy="false">) </mo><mo>⋯</mo></mtd></mtr><mtr><mtd><mo stretchy="false">[</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo>+</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">) </mo><mo stretchy="false">]</mo><mo>+</mo></mtd></mtr><mtr><mtd><mo stretchy="false">[</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo>+</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">) </mo><mo stretchy="false">]</mo><mo stretchy="false">[</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo>+</mo></mtd></mtr><mtr><mtd><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo stretchy="false">) </mo><mo stretchy="false">]</mo><mo stretchy="false">[</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>3</mn></msub><mo>+</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>3</mn></msub><mo stretchy="false">) </mo><mo stretchy="false">]</mo><mo>⋯</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">]</mo><mo>=</mo></mtd></mtr><mtr><mtd><mfrac><mn>2</mn><mrow><mi>Μ</mi><mo>×</mo><mo stretchy="false"> (</mo><mi>Μ</mi><mo>-</mo><mn>1</mn><mo stretchy="false">) </mo></mrow></mfrac><mo stretchy="false">[</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo stretchy="false">) </mo><mo>⋯</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">) </mo><mo>+</mo></mtd></mtr><mtr><mtd><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">) </mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo>⋯</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">) </mo><mo>+</mo><mo>⋯</mo><mo>+</mo></mtd></mtr><mtr><mtd><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">) </mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo stretchy="false">) </mo><mo>⋯</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo>+</mo></mtd></mtr><mtr><mtd><mn>2</mn><mo>×</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>3</mn></msub><mo stretchy="false">) </mo><mo>⋯</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">) </mo><mo>+</mo></mtd></mtr><mtr><mtd><mn>2</mn><mo>×</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mo stretchy="false">) </mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>3</mn></msub><mo>⋯</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">) </mo><mo>+</mo><mo>⋯</mo><mo>+</mo></mtd></mtr><mtr><mtd><mi>Ι</mi><mo>×</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>1</mn></msub><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mn>2</mn></msub><mo>⋯</mo><mi>E</mi><mi>C</mi><mi>C</mi><msub><mrow></mrow><mi>Ι</mi></msub><mo stretchy="false">]</mo><mspace width="0.25em" /></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="162">结论得证。</p>
                </div>
                <div class="p1">
                    <p id="163">运用这个定理, 本文把指数级的期望稠密度计算量降低到了线性级。</p>
                </div>
                <h4 class="anchor-tag" id="164" name="164">2.3 FSC-FM<b>方法</b></h4>
                <div class="p1">
                    <p id="165"><i>FSC</i>-<i>FM</i>方法的具体实现步骤:</p>
                </div>
                <div class="p1">
                    <p id="166">步骤1 利用边聚集系数计算<i>PPI</i>网络中每组相互作用间的概率, 从而构建不确定<i>PPI</i>网络图。</p>
                </div>
                <div class="p1">
                    <p id="167">步骤2 根据式 (5) 计算<i>PPI</i>网络中的蛋白质节点间的相似度, 计算<i>PPI</i>网络中每组相互作用的相似度矩阵, 并采用改进相似度度量后的谱聚类算法预处理<i>PPI</i>数据, 得到维数较低的矩阵<b><i>Y</i></b>。</p>
                </div>
                <div class="p1">
                    <p id="168">步骤3 通过DPCS方法, 获取<i>K</i>个初始聚类中心;以初始聚类中心为起点, 不断迭代根据式 (10) ～ (11) 更新聚类中心以及隶属度, 根据式 (9) 计算目标函数, 实现网络功能模块的划分, 直到所有的节点都被遍历完或与上次目标函数值进行比较的出的变化量小于阈值<i>ε</i>。</p>
                </div>
                <div class="p1">
                    <p id="169">步骤4 根据式 (12) 计算挖掘的模块的密度, 过滤边期望稠密度小于阈值<i>T</i>的模块。本文设定<i>T</i>=0.1。</p>
                </div>
                <h4 class="anchor-tag" id="170" name="170">2.4 <b>方法分析</b></h4>
                <div class="p1">
                    <p id="171"><i>FSC</i>-<i>FM</i>方法的计算复杂度由以下几个步骤构成:采用边聚集系数构建不确定<i>PPI</i>网络的时间复杂度为<i>O</i> (|<i>E</i>|) ;采用FEC策略改进相似性度量的谱聚类算法的时间复杂度主要取决于计算相似度矩阵以及特征分解, 其中计算相似度矩阵的时间复杂度为<i>O</i> (<i>N</i><sup>2</sup>) , 计算特征分解的时间复杂度为<i>O</i> (<i>N</i>) , 谱聚类算法的整体时间复杂度为<i>O</i> (<i>N</i>) ;采用DPCS策略选取初始聚类中心的FCM算法的时间复杂度主要取决于计算概率密度函数以及搜索最大值, 其中计算概率密度函数的时间复杂度为<i>O</i> (<i>N</i>) , 搜索最大值的时间复杂度为<i>O</i> (<i>N</i>) , FCM算法的整体时间复杂度为<i>O</i> (<i>N</i><sup>2</sup>+<i>N</i>) 即<i>O</i> (<i>N</i><sup>2</sup>) ;采用EED度量过滤蛋白质功能模块的时间复杂度为<i>O</i> (<i>K</i>) 。因此, FSC-FM方法的时间复杂度为<i>O</i> (|<i>E</i>|+<i>N</i><sup>3</sup>+<i>N</i><sup>2</sup>+<i>K</i>) 即<i>O</i> (<i>N</i><sup>3</sup>) 。而在CDUN算法中, 算法的时间复杂度主要取决于基于基因表达数据和PPI高通量数据构建的不确定PPI网络检测候选蛋白质模块以及删除高度重叠蛋白质模块, 即<i>O</i> (<i>KLN</i><sup>3</sup>) ;在DCU算法中, 算法的时间复杂度主要取决于产生候选蛋白质集以及候选附件蛋白质, 即<i>O</i> (<i>KN</i><sup>3</sup>) ;在EA中, 算法的时间复杂度主要取决于种群演化以及初始化算法, 即<i>O</i> (<i>αN</i><sup>3</sup><i>R</i>) ;在MGPPA中, 算法的时间复杂度主要取决于蛋白质簇形成的过程, 即<i>O</i> (<i>KθN</i><sup>3</sup>) 。上述提及的<i>L</i>、<i>α</i>、<i>R</i>和<i>θ</i>分别表示基因表达时刻数、集群个数、迭代次数和数据库属性数目值。</p>
                </div>
                <h3 id="172" name="172" class="anchor-tag">3 实验与结果分析</h3>
                <h4 class="anchor-tag" id="173" name="173">3.1 <b>实验环境</b></h4>
                <div class="p1">
                    <p id="174"><i>FSC</i>-<i>FM</i>方法实验的编程环境为<i>Python</i>3.5.2;操作系统为<i>Windows</i> 10家庭中文版;内存12 <i>GB</i>; <i>CPU</i>为<i>Intel Core i</i>5-4200<i>H</i> 2.8 <i>GHz</i>。</p>
                </div>
                <h4 class="anchor-tag" id="175" name="175">3.2 <b>实验数据集</b></h4>
                <div class="p1">
                    <p id="176">为验证本文方法的有效性, 选用蛋白质相互作用数据相对完整和可靠的酵母蛋白质相互作用网络数据作为实验数据。具体实验数据如下所示:</p>
                </div>
                <div class="p1">
                    <p id="177">1) 酵母<i>PPI</i>网络数据来源于<i>DIP</i>数据库<citation id="385" type="reference"><link href="348" rel="bibliography" /><sup>[<a class="sup">30</a>]</sup></citation>, 去除重复的相互作用, 该数据库包含4 995个蛋白质和21 554对相互作用。</p>
                </div>
                <div class="p1">
                    <p id="178">2) 本文采用<i>CYC</i>2008<citation id="386" type="reference"><link href="350" rel="bibliography" /><sup>[<a class="sup">31</a>]</sup></citation>作为已知蛋白质功能模块集, <i>CYC</i>2008包含408个通过生物实验预测得到的功能模块。</p>
                </div>
                <div class="p1">
                    <p id="179">3) <i>Krogan</i>数据<citation id="387" type="reference"><link href="352" rel="bibliography" /><sup>[<a class="sup">32</a>]</sup></citation>是用串联亲和纯化来处理4 562不同标签酵母蛋白质, 去除自相互作用和重复相互作用后, 该网络中包含3 672个蛋白质和14 317条可靠的相互作用。</p>
                </div>
                <h4 class="anchor-tag" id="180" name="180">3.3 <b>评价指标</b></h4>
                <h4 class="anchor-tag" id="181" name="181">3.3.1 特异性、灵敏度和F-measure度量</h4>
                <div class="p1">
                    <p id="182">本文使用文献<citation id="388" type="reference">[<a class="sup">33</a>]</citation>中的特异性 (<i>Specificity</i>, <i>Sp</i>) 、灵敏度 (<i>Sensitivity</i>, <i>Sn</i>) 和F-measure指标来进行算法评价对比。特异性是指算法识别的功能模块中成功匹配的模块在挖掘出的模块数目中所占比例, 其定义为:</p>
                </div>
                <div class="p1">
                    <p id="183" class="code-formula">
                        <mathml id="183"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>S</mi><mi>p</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>6</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="184">灵敏度是指匹配成功的功能模块在基准模块中所占比例, 其定义为:</p>
                </div>
                <div class="p1">
                    <p id="185" class="code-formula">
                        <mathml id="185"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>S</mi><mi>n</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>7</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="186">其中:<i>TP</i>表示算法识别的功能模块中与已知功能模块匹配程度<i>OS</i> (<i>A</i>, <i>B</i>) ≥0.2的数量;<i>FP</i>表示预测功能模块中没有匹配成功的数量;<i>FN</i>表示基准模块中没有被成功匹配的数量。</p>
                </div>
                <div class="p1">
                    <p id="187">为评估算法的有效性, 对于算法挖掘出来的功能模块<i>A</i>和已知功能模块<i>B</i>之间的匹配程度通过<mathml id="188"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ο</mi><mi>S</mi><mo stretchy="false"> (</mo><mi>A</mi><mo>, </mo><mi>B</mi><mo stretchy="false">) </mo><mo>=</mo><mfrac><mrow><mo stretchy="false">|</mo><mi>A</mi><mstyle displaystyle="true"><mo>∩</mo><mi>B</mi></mstyle><mo stretchy="false">|</mo><msup><mrow></mrow><mn>2</mn></msup></mrow><mrow><mo stretchy="false">|</mo><mi>A</mi><mo stretchy="false">|</mo><mo stretchy="false">|</mo><mi>B</mi><mo stretchy="false">|</mo></mrow></mfrac></mrow></math></mathml>计算得到。若识别出的功能模块<i>A</i>与已知功能模块<i>B</i>的匹配程度超过给定阈值, 则称该已知功能模块被标识, 本文根据文献<citation id="389" type="reference">[<a class="sup">33</a>]</citation>将该阈值设置为0.2。若<i>OS</i> (<i>A</i>, <i>B</i>) =1, 则称该已知功能模块被完全标识。为了避免灵敏度和特异性所带来的偏见, 采用<i>F</i>-<i>measure</i>综合评价指标来评估整体算法的性能, 其计算公式如式 (18) 所示:</p>
                </div>
                <div class="p1">
                    <p id="189" class="code-formula">
                        <mathml id="189"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>F</mi><mo>-</mo><mi>m</mi><mi>e</mi><mi>a</mi><mi>s</mi><mi>u</mi><mi>r</mi><mi>e</mi><mo>=</mo><mfrac><mrow><mn>2</mn><mo>×</mo><mi>S</mi><mi>n</mi><mo>×</mo><mi>S</mi><mi>p</mi></mrow><mrow><mi>S</mi><mi>n</mi><mo>+</mo><mi>S</mi><mi>p</mi></mrow></mfrac><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>8</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <h4 class="anchor-tag" id="190" name="190">3.3.2 <i>P</i>值度量</h4>
                <div class="p1">
                    <p id="191">随着蛋白质组学研究的深入, 使得一个蛋白质与其功能注释向对应成为可能, 蛋白质簇发生对于一个给定功能注释在统计学上的意义就可以通过一个超几何分布的等式来进行计算<citation id="390" type="reference"><link href="356" rel="bibliography" /><sup>[<a class="sup">34</a>]</sup></citation>:</p>
                </div>
                <div class="p1">
                    <p id="192" class="code-formula">
                        <mathml id="192"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ρ</mi><mo>-</mo><mi>v</mi><mi>a</mi><mi>l</mi><mi>u</mi><mi>e</mi><mo>=</mo><mn>1</mn><mo>-</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>0</mn></mrow><mrow><mi>D</mi><mo>-</mo><mn>1</mn></mrow></munderover><mrow><mfrac><mrow><mrow><mo> (</mo><mrow><mtable><mtr><mtd><mi>F</mi></mtd></mtr><mtr><mtd><mi>i</mi></mtd></mtr></mtable></mrow><mo>) </mo></mrow><mspace width="0.25em" /><mrow><mo> (</mo><mrow><mtable><mtr><mtd><mi>C</mi><mo>-</mo><mi>F</mi></mtd></mtr><mtr><mtd><mi>B</mi><mo>-</mo><mi>i</mi></mtd></mtr></mtable></mrow><mo>) </mo></mrow></mrow><mrow><mrow><mo> (</mo><mrow><mtable><mtr><mtd><mi>C</mi></mtd></mtr><mtr><mtd><mi>B</mi></mtd></mtr></mtable></mrow><mo>) </mo></mrow></mrow></mfrac></mrow></mstyle><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>9</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="193">其中:<i>C</i>代表PPI网络中包含的蛋白质总数;<i>B</i>为预测挖掘出的模块数;<i>F</i>为一个功能组数量;<i>D</i>为<i>B</i>中包含<i>F</i>中的蛋白质数目。</p>
                </div>
                <h4 class="anchor-tag" id="194" name="194">3.4 <b>参数影响分析</b></h4>
                <h4 class="anchor-tag" id="195" name="195">3.4.1 参数<i>δ</i>和<i>ε</i>的影响分析</h4>
                <div class="p1">
                    <p id="196">FSC-FM方法中, 由于参数<i>δ</i>和<i>ε</i>的取值影响实验的聚类效果, 因此本文在15组<i>δ</i>和<i>ε</i>的参数取值上独立运行20次实验, 取20次实验的平均值进行分析。实验使用到的参数设置如下:<i>m</i>=2, <i>ρ</i>=3。表1给出了具体参数设置情况, 其中<i>Set</i><sub><i>i</i></sub>代表第<i>i</i>组参数, <i>Q</i>值表示不同的<i>F</i>-<i>measure</i>值或匹配的蛋白质功能模块比例。实验结果如图2所示。</p>
                </div>
                <div class="p1">
                    <p id="197">实验结果表明, 随着<i>δ</i>从0～0.3逐渐增大, <i>F</i>-<i>measure</i>的值在<i>ε</i>不同取值之下也逐渐增大, 实验挖掘出的功能模块和已知的功能模块的匹配比例也逐渐增加;随着<i>δ</i>从0.3～0.5逐渐增大, <i>F</i>-<i>measure</i>的值在<i>ε</i>不同取值之下逐渐降低, 实验挖掘出的功能模块和已知的功能模块的匹配比例也逐渐降低。这是因为采用DPCS策略选取合适的初始聚类中心时, 算法需要多次迭代达到收敛效果, 需要运行很长时间;且存在初始聚类中心选择不理想, 满足条件的相互作用减少, 模块识别的覆盖率降低, 能够匹配的功能模块要求更加严格, 功能模块识别的数量较少, 算法的精确度增加, 导致<i>F</i>-<i>measure</i>值和匹配比例先增加后降低。通过观察发现存在一对合理取值即<i>ε</i>=0.000 15, <i>δ</i>=0.3使<i>F</i>-<i>measure</i>达到最大值0.59且匹配比例达到68.834 7%。</p>
                </div>
                <div class="area_img" id="198">
                    <p class="img_tit"><b>表</b>1 <b>实验参数设置</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit">Tab. 1 Experimental parameter setting</p>
                    <p class="img_note"></p>
                    <table id="198" border="1"><tr><td rowspan="2"><br /><i>ε</i></td><td colspan="5"><br />不同<i>δ</i>值所对应的<i>Q</i>值</td></tr><tr><td><br />0.1</td><td>0.2</td><td>0.3</td><td>0.4</td><td>0.5</td></tr><tr><td><br />0.000 15</td><td><i>Set</i><sub>1</sub></td><td><i>Set</i><sub>2</sub></td><td><i>Set</i><sub>3</sub></td><td><i>Set</i><sub>4</sub></td><td><i>Set</i><sub>5</sub></td></tr><tr><td><br />0.000 45</td><td><i>Set</i><sub>6</sub></td><td><i>Set</i><sub>7</sub></td><td><i>Set</i><sub>8</sub></td><td><i>Set</i><sub>9</sub></td><td><i>Set</i><sub>10</sub></td></tr><tr><td><br />0.000 75</td><td><i>Set</i><sub>11</sub></td><td><i>Set</i><sub>12</sub></td><td><i>Set</i><sub>13</sub></td><td><i>Set</i><sub>14</sub></td><td><i>Set</i><sub>15</sub></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="area_img" id="199">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201904017_199.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 ε和δ取不同值时, F-measure值和 匹配的蛋白质功能模块比例变化" src="Detail/GetImg?filename=images/JSJY201904017_199.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 <i>ε</i>和<i>δ</i>取不同值时, <i>F</i>-<i>measure</i>值和 匹配的蛋白质功能模块比例变化  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201904017_199.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 2 <i>F</i>-<i>measure</i> and matching ratio of protein function module scale change with <i>ε</i> and <i>δ</i></p>

                </div>
                <h4 class="anchor-tag" id="200" name="200">3.4.2 阈值<i>T</i>的分析</h4>
                <div class="p1">
                    <p id="201">FSC-FM算法中, 根据改进的期望稠密度EED对挖掘出的蛋白质功能模块进行过滤, 引入自定义参数<i>T</i>描述模块的EED阈值, 由定理2, <i>T</i>∈[0, 1]。图3显示了<i>T</i>取不同值, FSC-FM算法的<i>F</i>-<i>measure</i>值的变化情况。</p>
                </div>
                <div class="area_img" id="202">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201904017_202.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 参数T对F-measure的影响" src="Detail/GetImg?filename=images/JSJY201904017_202.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 参数<i>T</i>对<i>F</i>-<i>measure</i>的影响  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201904017_202.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 3 Effect of parameter <i>T</i> on <i>F</i>-<i>measure</i></p>

                </div>
                <div class="p1">
                    <p id="203">由图3可看出, 当<i>T</i>=0.1时, FSC-FM方法可以得到最高的<i>F</i>-<i>measure</i>值, 为此, 本文设定<i>T</i>=0.1。</p>
                </div>
                <h4 class="anchor-tag" id="204" name="204">3.5 FEC<b>策略的有效性分析</b></h4>
                <div class="p1">
                    <p id="205">为了验证<i>FSC</i>-<i>FM</i>方法使用改进的相似度<i>FEC</i>策略的有效性, 分别基于使用<i>FEC</i>策略改进相似度计算的<i>FSC</i>-<i>FM</i>方法和未使用<i>FEC</i>策略的<i>FSC</i>-<i>FM</i>方法, 在<i>DIP</i>数据库上进行功能模块的挖掘, 实验得到的F-measure和匹配比例如图4所示。</p>
                </div>
                <div class="p1">
                    <p id="206">由图4显示, 使用改进相似度<i>FEC</i>策略的<i>FSC</i>-<i>FM</i>方法在<i>Sn</i>、<i>Sp</i>、<i>F</i>-<i>measure</i>取值和匹配的蛋白质功能模块比例都比未使用FEC策略的取值要高。具体<i>Sn</i>的取值比未使用FEC策略提高15.29%, <i>Sp</i>的取值比未使用FEC策略提高17.27%, <i>F</i>-<i>measure</i>的取值比未使用FEC策略提高5.12%, 匹配的蛋白质模块比未使用FEC策略提高12.39%。实验结果说明, 使用改进的FEC策略的方法的聚类效果得到了提高。</p>
                </div>
                <div class="area_img" id="207">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201904017_207.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 FEC策略对比分析" src="Detail/GetImg?filename=images/JSJY201904017_207.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 FEC策略对比分析  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201904017_207.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 4 Comparative analysis of FEC strategy</p>

                </div>
                <h4 class="anchor-tag" id="208" name="208">3.6 DPCS<b>和</b>EED<b>策略的有效性分析</b></h4>
                <div class="p1">
                    <p id="209">为了验证<i>FSC</i>-<i>FM</i>方法使用改进的相似度<i>DPCS</i>策略和<i>EED</i>度量的有效性, 分别基于<i>DPCS</i>策略以及过滤模块的<i>EED</i>度量的<i>FSC</i>-<i>FM</i>方法和未使用这两种策略的<i>FSC</i>-<i>FM</i>方法, 在<i>DIP</i>数据库独立执行20次进行功能模块的挖掘, 实验检测结果如图5所示。</p>
                </div>
                <div class="p1">
                    <p id="210">图5显示的是使用<i>DPCS</i>和<i>EED</i>策略的<i>FSC</i>-<i>FM</i>方法在<i>Sn</i>、<i>Sp</i>、<i>F</i>-<i>measure</i>取值和匹配的蛋白质功能模块比例与未使用这两种策略的对比情况, 其中使用这两种策略的<i>Sn</i>的取值比未使用这两种策略提高12.50%, <i>Sp</i>的取值比未使用这两种策略提高30.86%, <i>F</i>-<i>measure</i>的取值比未使用这两种策略提高9.63%, 匹配的蛋白质模块比未使用这两种策略提高7.05%。这是因为, 未使用DPCS策略和EED度量的算法挖掘出的功能模块的预测结果存在过度的重叠特性, 这种过度的重叠特性造成了预测结果太大而无法与一些较小的真实功能模块相匹配;相反采用DPCS选择初始聚类中心进行功能模块挖掘, 对得到的功能模块采用EED度量进行模块过滤, 可以避免网络数据噪声对聚类结果造成的影响, 避免过度重叠划分, 方法的聚类结果的特异性、灵敏度和<i>F</i>-<i>measure</i>值都较高, 挖掘出的无用模块数目以及重复划分模块数目较少。实验结果说明, 使用这两种策略的方法的聚类效果较优。</p>
                </div>
                <div class="area_img" id="211">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201904017_211.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图5 DPCS和EED策略对比分析" src="Detail/GetImg?filename=images/JSJY201904017_211.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图5 DPCS和EED策略对比分析  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201904017_211.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 5 Comparative analysis of DPCS and EED strategies</p>

                </div>
                <h4 class="anchor-tag" id="212" name="212">3.7 <b>算法性能的比较分析</b></h4>
                <div class="p1">
                    <p id="213">本节将<i>FSC</i>-<i>FM</i>分别从功能模块挖掘的比较分析、功能富集的比较分析以及方法运行效率的比较分析与<i>CDUN</i><citation id="391" type="reference"><link href="324" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>、<i>DCU</i><citation id="392" type="reference"><link href="326" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>、<i>EA</i><citation id="393" type="reference"><link href="328" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>和<i>MGPPA</i><citation id="394" type="reference"><link href="330" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>进行比较分析, 重复迭代次数为20。实验中使用的参数设置如下:取<i>m</i>=2, <i>ε</i>=0.000 15, <i>δ</i>=0.3, <i>ρ</i>=3, <i>T</i>=0.1。</p>
                </div>
                <h4 class="anchor-tag" id="214" name="214">3.7.1 功能模块挖掘的比较分析</h4>
                <div class="p1">
                    <p id="215">为了验证本文方法的性能, 将<i>FSC</i>-<i>FM</i>方法与其他4种算法独立运行20次, 取实验结果的平均值进行分析, 得到各个算法挖掘的功能模块基本信息以及实验评价指标对比分析如表2和图6所示。</p>
                </div>
                <div class="area_img" id="216">
                    <p class="img_tit"><b>表</b>2 <b>各算法挖掘的功能模块基本信息</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><i>Tab</i>. 2 <i>Basic information of functional</i><i>modules mined by different algorithms</i></p>
                    <p class="img_note"></p>
                    <table id="216" border="1"><tr><td>算法</td><td>PM</td><td>Full</td><td>TP</td><td>算法</td><td>PM</td><td>Full</td><td>TP</td></tr><tr><td><i>CDUN</i></td><td>126</td><td>6</td><td>54</td><td><i>MGPPA</i></td><td>354</td><td>7</td><td>92</td></tr><tr><td><br /><i>DCU</i></td><td>254</td><td>9</td><td>153</td><td><i>FSC</i>-<i>FM</i></td><td>369</td><td>18</td><td>254</td></tr><tr><td><br /><i>EA</i></td><td>378</td><td>8</td><td>127</td><td></td><td></td><td></td><td></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="217">在表2中, <i>PM</i>表示算法挖掘出的功能模块总数, <i>Full</i>是指已知的功能模块集中被完全标识的功能模块数。从表2可以知道, FSC-FM方法挖掘的功能模块中有254个被匹配, 在所有算法中匹配数量最多, 相比较而言本文方法对于挖掘蛋白质功能模块算法具有更高的效率。</p>
                </div>
                <div class="p1">
                    <p id="218">图6显示各种方法在DIP数据集中识别的功能模块计算的<i>Sn</i>、<i>Sp</i>和<i>F</i>-<i>measure</i>对比分析。</p>
                </div>
                <div class="area_img" id="219">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201904017_219.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图7 各个算法的功能模块挖掘可视化比较" src="Detail/GetImg?filename=images/JSJY201904017_219.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图7 各个算法的功能模块挖掘可视化比较  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201904017_219.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 7 Visualization comparison of functional modules mined by different algorithms</p>

                </div>
                <div class="area_img" id="220">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201904017_220.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图6 不同算法的F-measure、Sp和Sn值对比" src="Detail/GetImg?filename=images/JSJY201904017_220.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图6 不同算法的<i>F</i>-<i>measure</i>、<i>Sp</i>和<i>Sn</i>值对比  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201904017_220.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 6 Comparison of <i>F</i>-<i>measure</i>, <i>Sp</i> and <i>Sn</i> values of different algorithms</p>

                </div>
                <div class="p1">
                    <p id="221">由图6显示, 本文方法具有较高的<i>F</i>-<i>measure</i>、<i>Sp</i>和<i>Sn</i>值, <i>F</i>-<i>measure</i>的值较CDUN、DCU、EA和MGPPA提高了192.37%、27.92%、82.98%、182.23%, 本文识别的功能模块中识别正确的部分所占比例较高, 因此本文方法取得了较好的优化效率。</p>
                </div>
                <div class="p1">
                    <p id="222">图7显示了不同算法检测到的Elongator holoenzyme模块结果, 它真实存在于酵母菌细胞内。图7 (a) 是该标准模块所包含的蛋白质相互作用情况, 其他是不同算法的检测结果。</p>
                </div>
                <div class="p1">
                    <p id="223">通过图7显示, 本文方法能够准确地挖掘蛋白质功能模块;CDUN算法识别出标准复合物中的6个蛋白质, 但是也包含了4个非Elongator holoenzyme模块内的蛋白质;DCU算法识别出标准模块中的6个蛋白质, 但是也包含了1个非Elongator holoenzyme模块内的蛋白质;EA识别出标准模块中的6个蛋白质, 但是也包含了2个非Elongator holoenzyme模块内的蛋白质;MGPPA识别出标准模块中的5个蛋白质。实验结果表明, 本文方法在挖掘蛋白质功能模块上具有较好的聚类效果。</p>
                </div>
                <h4 class="anchor-tag" id="224" name="224">3.7.2 功能富集的比较分析</h4>
                <div class="p1">
                    <p id="225">为了测试算法挖掘的功能模块的生物学意义, 本文采用功能富集分析评价挖掘的模块的统计和生物特性。挖掘的模块的低值<i>P</i>-<i>value</i>表明该功能模块具有很高的统计学意义, 将<i>P</i>-<i>value</i>的最小值对应的功能作为该功能模块的主要功能, 通过给每个挖掘的模块赋予最小的<i>P</i>-<i>value</i>值对应的功能, 可以识别预测功能模块的功能。若一个模块的<i>P</i>-<i>value</i>&lt;0.01, 则认为这个模块是显著的, 显著的模块数量在挖掘出的模块总数中所占的比例可以很好地评价各个算法的整体性。具体各个算法性能比较分析如表3所示。</p>
                </div>
                <div class="area_img" id="226">
                    <p class="img_tit"><b>表</b>3 <b>各个算法挖掘的功能模块的显著性统计信息</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit">Tab. 3 Significant statistics of functional modules mined by each algorithm</p>
                    <p class="img_note"></p>
                    <table id="226" border="1"><tr><td>算法</td><td><i>PM</i></td><td><i>SC</i></td><td>显著性<br />比例/%</td><td>算法</td><td><i>PM</i></td><td><i>SC</i></td><td>显著性<br />比例/%</td></tr><tr><td>CDUN</td><td>126</td><td>63</td><td>50.00</td><td>MGPPA</td><td>354</td><td>180</td><td>50.85</td></tr><tr><td><br />DCU</td><td>254</td><td>167</td><td>65.75</td><td>FSC-FM</td><td>369</td><td>307</td><td>83.20</td></tr><tr><td><br />EA</td><td>378</td><td>208</td><td>55.03</td><td></td><td></td><td></td><td></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="227">在表3中:<i>PM</i>表示算法挖掘出的功能模块总数, <i>SC</i>是具有显著意义的模块数目。本文方法FSC-FM挖掘的模块数目中显著性模块的比例达到83.20%, 相对于CDUN<citation id="395" type="reference"><link href="324" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>、DCU<citation id="396" type="reference"><link href="326" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>、EA<citation id="397" type="reference"><link href="328" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>和MGPPA<citation id="398" type="reference"><link href="330" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>分别提高了66.4%、26.54%、51.19%、63.62%, 由此可见, FSC-FM方法挖掘的功能模块具有很强的生物统计学意义。为了更加深入分析和全面对比, 对各个算法预测得到的功能模块根据<i>P</i>-<i>value</i>区间值进行对比分析, 可分为两个区间, 即 (0, E-10) 和[E-10, 0.01) 。图8显示了分区间对比情况。</p>
                </div>
                <div class="area_img" id="228">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JSJY201904017_228.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图8 各个算法的P-value区间对比分析" src="Detail/GetImg?filename=images/JSJY201904017_228.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图8 各个算法的<i>P</i>-<i>value</i>区间对比分析  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JSJY201904017_228.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit">Fig. 8 Comparative analysis of <i>P</i>-<i>value</i> range of different algorithms</p>

                </div>
                <div class="p1">
                    <p id="229">从图8可看出本文方法FSC-FM挖掘的功能模块中<i>P</i>-<i>value</i>&lt;E-10的显著性模块数量累计比例最高, 达到了58.51%, 而其他算法的显著性比例分别为48.32%、55.23%、52.06%、50.17%。根据表3和图8分析可知, FSC-FM方法挖掘出的功能模块更具有生物意义。</p>
                </div>
                <h4 class="anchor-tag" id="230" name="230">3.7.3 算法效率的比较分析</h4>
                <div class="p1">
                    <p id="231">为进一步分析比较本文方法<i>FSC</i>-<i>FM</i>的执行效率, 将其与<i>CDUN</i><citation id="399" type="reference"><link href="324" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>、<i>DCU</i><citation id="400" type="reference"><link href="326" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>、<i>EA</i><citation id="401" type="reference"><link href="328" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>和<i>MGPPA</i><citation id="402" type="reference"><link href="330" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>在各自算法优化参数之下, 在<i>DIP</i>数据库上运行20次, 取实验的平均值来比较分析, 得到各个算法的平均运行效率对比如表4所示。</p>
                </div>
                <div class="area_img" id="232">
                    <p class="img_tit"><b>表</b>4 <i>FSC</i>-<i>FM</i><b>和其他四种算法效率分析</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><i>Tab</i>. 4 <i>Efficiency analysis of FSC</i>-<i>FM algorithm and</i><i>other four algorithms</i></p>
                    <p class="img_note"></p>
                    <table id="232" border="1"><tr><td>算法</td><td>挖掘的<br />模块数量</td><td>模块规模<br />范围</td><td>匹配率/%</td><td>运行时间/<i>s</i></td></tr><tr><td><br /><i>CDUN</i></td><td>126</td><td>4～38</td><td>42.875 0</td><td>1 485.949 0</td></tr><tr><td><br /><i>DCU</i></td><td>254</td><td>4～167</td><td>60.236 2</td><td>650.163 9</td></tr><tr><td><br /><i>EA</i></td><td>378</td><td>4～117</td><td>33.597 9</td><td>929.983 2</td></tr><tr><td><br /><i>MGPPA</i></td><td>354</td><td>4～97</td><td>25.988 7</td><td>1 434.433 0</td></tr><tr><td><br /><i>FSC</i>-<i>FM</i></td><td>369</td><td>4～295</td><td>68.834 7</td><td>508.250 0</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="233">在表4中, 模块数量是指算法挖掘的模块规模大于3的蛋白质数目, 匹配率是挖掘的蛋白质和基准模块匹配的数目比例。从表4可知, 本文方法挖掘蛋白质功能模块所需的时间相对较少, 实验运行的时间复杂度较低, 是508.25 <i>s</i>。本文方法相对其他四种算法的平均运行时间都超过600 <i>s</i>, 比<i>DCU</i>算法执行效率提高了27.92%。由此可见, 本文方法可以应用于相对规模较大的不确定<i>PPI</i>网络, 进而挖掘蛋白质功能模块。主要是因为, 本文方法基于不确定<i>PPI</i>网络, 使用改进相似度度量的谱聚类算法以及融合优化初始聚类中心选取的<i>FCM</i>算法来挖掘蛋白质功能模块, 进而采用不确定<i>PPI</i>网络拓扑特性的边期望稠密来过滤模块。因此, 本文方法在挖掘蛋白质模块具有很好的运行效率。</p>
                </div>
                <div class="p1">
                    <p id="234">为进一步分析比较数据规模对个算法运行效率的影响, 将<i>FSC</i>-<i>FM</i>方法与<i>CDUN</i><citation id="403" type="reference"><link href="324" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>、<i>DCU</i><citation id="404" type="reference"><link href="326" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>、<i>EA</i><citation id="405" type="reference"><link href="328" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>和<i>MGPPA</i><citation id="406" type="reference"><link href="330" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>在各自算法优化参数之下, 在<i>Krogan</i>数据集上运行20次, 取实验的平均值来比较分析, 得到各个算法的平均运行效率对比如表5所示。</p>
                </div>
                <div class="p1">
                    <p id="235">从表5可知, 相对于<i>DIP</i>数据库上的运行结果, 各个算法在数据规模较小的<i>Krogan</i>数据集上的执行效率以及匹配率相对都有所提高。具体来说, <i>CDUN</i>算法挖掘模块的匹配率提高了1.815 51%, 运行时间降低了1.247 9%;<i>DCU</i>算法挖掘模块的匹配率提高了4.409 1%, 运行时间降低了1.383 2%;<i>EA</i>挖掘模块的匹配率提高了1.907 6%, 运行时间降低了4.881 9%;<i>MGPPA</i>的匹配率提高了0.532 154 4%, 运行时间降低了0.052 3%;<i>FSC</i>-<i>FM</i>方法挖掘模块的匹配率提高了4.711 9%, 运行时间降低了5.176%。从表5可知, <i>DCU</i>算法和<i>FSC</i>-<i>FM</i>处理数据规模较大的数据集的执行效率相对较高, 而数据规模对<i>MGPPA</i>的执行效率影响不大, <i>CDUN</i>和<i>EA</i>处理数据规模较小的数据集的执行效率相对较高。</p>
                </div>
                <div class="area_img" id="236">
                    <p class="img_tit"><b>表</b>5 <b>在</b><i>Krogan</i><b>数据集上各个算法的运行效率分析</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"><i>Tab</i>. 5 <i>Running efficiency analysis of</i><i>different algorithms on Krogan dataset</i></p>
                    <p class="img_note"></p>
                    <table id="236" border="1"><tr><td>算法</td><td>挖掘的<br />模块数量</td><td>模块规模<br />范围</td><td>匹配率/%</td><td>运行时间/<i>s</i></td></tr><tr><td><br /><i>CDUN</i></td><td>130</td><td>4～35</td><td>43.653 4</td><td>1 467.634 1</td></tr><tr><td><br /><i>DCU</i></td><td>267</td><td>4～153</td><td>62.892 1</td><td>641.293 6</td></tr><tr><td><br /><i>EA</i></td><td>384</td><td>4～98</td><td>34.238 8</td><td>886.695 3</td></tr><tr><td><br /><i>MGPPA</i></td><td>356</td><td>4～89</td><td>26.127 0</td><td>1 433.683 0</td></tr><tr><td><br /><i>FSC</i>-<i>FM</i></td><td>375</td><td>4～246</td><td>72.238 5</td><td>483.237 6</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="237">综合分析表4～5, 本文提出的挖掘功能模块<i>FSC</i>-<i>FM</i>方法的运行效率较高。</p>
                </div>
                <h3 id="238" name="238" class="anchor-tag">4 结语</h3>
                <div class="p1">
                    <p id="239">本文基于不确定蛋白质相互作用网络, 提出一种基于模糊谱聚类的不确定<i>PPI</i>网络功能模块挖掘方法<i>FSC</i>-<i>FM</i>。该方法利用边聚集系数构建不确定蛋白质网络, 提高功能模块挖掘的准确率;其次采用<i>FEC</i>策略改进谱聚类算法中相似矩阵计算对尺度参数敏感的缺陷;通过<i>DPCS</i>策略优化<i>FCM</i>算法对初始聚类中心、聚类数目敏感的问题;采用<i>EED</i>度量过滤算法挖掘出的模块。为了评估方法的性能, 本文将<i>FSC</i>-<i>FM</i>方法与<i>CDUN</i>、<i>DCU</i>、<i>EA</i>和<i>MGPPA</i>进行了对比, 实验结果表明, <i>FSC</i>-<i>FM</i>方法具有更高的准确率、灵敏度和执行效率, 识别的功能模块具有更强的生物统计意义。对蛋白质功能模块挖掘今后的研究, 可以从两个方面入手:1) 深入研究<i>PPI</i>网络拓扑结构, 综合考虑蛋白质生物信息来构建动态蛋白质网络以降低数据噪声的影响;2) 结合多元生物数据的方法以提升挖掘结果。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="290">
                            <a id="bibliography_1" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=BJGD201701006&amp;v=Mjk0NjF5Zk1hckc0SDliTXJvOUZZb1FLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnNGeURnVkw3S0o=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[1]</b>冀俊忠, 高光轩.基于文化算法的PPI网络功能模块检测方法[J].北京工业大学学报, 2017, 43 (1) :13-21. (JI J Z, GAO GX.Detecting functional module method based on cultural algorithm in protein-protein interaction networks[J].Journal of Beijing University of Technology, 2017, 43 (1) :13-21.) 
                            </a>
                        </p>
                        <p id="292">
                            <a id="bibliography_2" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJX201107007&amp;v=MjEzMDNCZHJHNEg5RE1xSTlGWTRRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlEZ1ZMN0tMejc=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[2]</b>鱼亮, 高琳, 孙鹏.蛋白质网络中复合体和功能模块预测算法研究[J].计算机学报, 2011, 34 (7) :1239-1251. (YU L, GAOL, SUN P.Research on algorithms for complexes and functional modules prediction in protein-protein interaction networks[J].Chinese Journal of Computer, 2011, 34 (7) :1239-1251.) 
                            </a>
                        </p>
                        <p id="294">
                            <a id="bibliography_3" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=SCLH201305014&amp;v=MjcyODBGckNVUjdxZlp1WnNGeURnVkw3S05pN0hackc0SDlMTXFvOUVZSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEc=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[3]</b>倪问尹, 王建新, 熊慧军, 等.基于不确定数据的功能模块预测[J].四川大学学报 (工程科学版) , 2013, 45 (5) :80-87. (NI W Y, WANG J X, XIONG H J, et al.Research of detecting functional modules based on uncertainty data[J].Journal of Sichuan University (Engineering Science Edition) , 2013, 45 (5) :80-87.) 
                            </a>
                        </p>
                        <p id="296">
                            <a id="bibliography_4" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=MOTO201404002&amp;v=MzIwNTVGWm9RS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlEZ1ZMN0tLQ0xmWWJHNEg5WE1xNDk=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[4]</b>冀俊忠, 刘志军, 刘红欣, 等.蛋白质相互作用网络功能模块检测的研究综述[J].自动化学报, 2014, 40 (4) :577-593. (JI JZ, LIU Z J, LIU H X, et al.An overview research on functional module detection for protein-protein interaction networks[J].Acta Automatica Sinica, 2014, 40 (4) :577-593.) 
                            </a>
                        </p>
                        <p id="298">
                            <a id="bibliography_5" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZNGD201002032&amp;v=MzE1NzlLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnNGeURnVkw3S1B5UE1hckc0SDlITXJZOUdab1E=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[5]</b>李敏, 王建新, 刘彬彬, 等.基于极大团扩展的蛋白质复合物识别算法[J].中南大学学报 (自然科学版) , 2010, 41 (2) :560-565. (LI M, WANG J X, LIU B B, et al.An algorithm for identifying protein complexes based on maximal clique extension[J].Journal of Central South University (Science and Technology) , 2010, 41 (2) :560-565.) 
                            </a>
                        </p>
                        <p id="300">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Insight into vibrational circular dichroism of proteins by density functional modeling">

                                <b>[6]</b>KESSLER J, ANDRUSHCHENKO V, KAPITAN J, et al.Insight into vibrational circular dichroism of proteins by density functional modeling[J].Physical Chemistry Chemical Physics, 2018, 20 (7) :4926-4935.
                            </a>
                        </p>
                        <p id="302">
                            <a id="bibliography_7" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Jerarca: efficient analysis of complex networks using hierarchical clustering">

                                <b>[7]</b>ALDECO R, MARIN I.Jerarca:efficient analysis of complex networks using hierarchical clustering[J].PLo S ONE, 2010, 5 (7) :11585-11591.
                            </a>
                        </p>
                        <p id="304">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Evolution of protein-coupled RNA dynamics during hierarchical assembly of ribosomal complexes">

                                <b>[8]</b>ABEYSIRIGUNAWARDENA S C, KIM H, LAI J, et al.Evolution of protein-coupled RNA dynamics during hierarchical assembly of ribosomal complexes[J].Nature Communications, 2017, 8 (1) :492-500.
                            </a>
                        </p>
                        <p id="306">
                            <a id="bibliography_9" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=DZXU201801020&amp;v=MjQwODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlEZ1ZMN0tJVGZUZTdHNEg5bk1ybzlIWklRS0RIODQ=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[9]</b>雷秀娟, 高银, 郭玲.基于拓扑势加权的动态PPI网络复合物挖掘方法[J].电子学报, 2018, 46 (1) :145-151. (LEI X J, GAOY, GUO L.Mining protein complexes based on topology potential weight in dynamic protein-protein interaction networks[J].Acta Electronica Sinica, 2018, 46 (1) :145-151.) 
                            </a>
                        </p>
                        <p id="308">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Tissue-specific networkbased genome wide study of amygdala imaging phenotypes to identify functional interaction modules">

                                <b>[10]</b>YAO X H, YAN J W, LIU K F, et al.Tissue-specific networkbased genome wide study of amygdala imaging phenotypes to identify functional interaction modules[J].Bioinformatics, 2017, 33 (20) :3250-3257.
                            </a>
                        </p>
                        <p id="310">
                            <a id="bibliography_11" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJC201711027&amp;v=MjQ2MThRS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlEZ1ZMN0tMejdCYmJHNEg5Yk5ybzlIWTQ=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[11]</b>范子静, 罗泽, 马永征.一种基于模糊核聚类的谱聚类算法[J].计算机工程, 2017, 43 (11) :161-165. (FAN Z J, LUOZ, MA Y Z.A spectral clustering algorithm based on fuzzy kernel clustering[J].Computer Engineering, 2017, 43 (11) :161-165.) 
                            </a>
                        </p>
                        <p id="312">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Identifying similar functional modules by a new hybrid spectral clustering method">

                                <b>[12]</b>MADANI S, FAEZ K, AMINGHAFARI M.Identifying similar functional modules by a new hybrid spectral clustering method[J].IET Systems Biology, 2012, 6 (5) :175-186.
                            </a>
                        </p>
                        <p id="314">
                            <a id="bibliography_13" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES13012300032579&amp;v=MjIwMzRmT2ZiSzdIdERPckk5RlpPZ05DWHN3b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadEZpbmxVcjNJS0ZzVGF4QT1OaQ==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[13]</b>QIN G M, GAO L.Spectral clustering for protein complexes in Protein-Protein Interaction (PPI) networks[J].Mathematical and Computer Modelling, 2010, 52 (11/12) :2066-2074.
                            </a>
                        </p>
                        <p id="316">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Diffusion model based spectral clustering for protein-protein interaction networks">

                                <b>[14]</b>INOUE K, LI W J, KURATA H.Diffusion model based spectral clustering for protein-protein interaction networks[J].PLo S ONE, 2010, 5 (9) :12623-12632.
                            </a>
                        </p>
                        <p id="318">
                            <a id="bibliography_15" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CMFD&amp;filename=1012477561.nh&amp;v=MjAyMzZQSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnNGeURnVkw3S1ZGMjZITGUvR2RUS3JwRWI=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[15]</b>那第尔.识别蛋白质相互作用网络中的复合物[D].长沙:中南大学, 2012:22-34. (NA D E.Exploiting fuzzy spectral clustering in protein-complex detection[D].Changsha:Central South University, 2012:22-34.) 
                            </a>
                        </p>
                        <p id="320">
                            <a id="bibliography_16" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Protein Function Prediction by Spectral Clustering of Protein Interaction Network">

                                <b>[16]</b>TRIVODALIEV K, CINGOVSKA I, KALAJDZISKI S.Protein function prediction by spectral clustering of protein interaction network[C]//Proceedings of the 2011 Database Theory and Application, Bio-Science and Bio-Technology.Berlin:Springer, 2011:108-117.
                            </a>
                        </p>
                        <p id="322">
                            <a id="bibliography_17" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Mining Frequent Subgraph Patterns from Uncertain Graph Data">

                                <b>[17]</b>ZOU Z N, LI J Z, GAO H, et al.Mining frequent subgraph patterns from uncertain graph data[J].IEEE Transactions on Knowledge and Data Engineering, 2010, 22 (9) :1203-1218.
                            </a>
                        </p>
                        <p id="324">
                            <a id="bibliography_18" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD4733C6586C7FB6FEAFD8D6CAB85AFD54&amp;v=MDk4NDhhcmUvSGRLL3FZcE5ZcGdJZWc0L3VXTmluRXQxUEhtUjNXQTlmTVBpTWIrYkNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkJyTFUwNXRwaHpMMjR3YW89Tmo3Qg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[18]</b>ZHANG Y J, LIN H F, YANG Z H, et al.An uncertain modelbased approach for identifying protein complexes in uncertain protein-protein interaction networks[J].BMC Genomics, 2017, 18 (7) :743-752.
                            </a>
                        </p>
                        <p id="326">
                            <a id="bibliography_19" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJCM&amp;filename=SJCM89D31A271C531AB878079906FAB42375&amp;v=MDg2NjFvMUNaWmdLRDMxSXZSNFU0ajk2UVhiaXFtUkVDN2FXUnIyYUNPTnZGU2lXV3I3SklGcG1hQnVIWWZPR1FsZkJyTFUwNXRwaHpMMjR3YW89TmlmSVk3dXhhdExOMw==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[19]</b>ZHAO B H, WANG J X, LI M.Detecting protein complexes based on uncertain graph model[J].IEEE/ACM Transactions on Computational Biology&amp;Bioinformatics, 2014, 11 (3) :486-497.
                            </a>
                        </p>
                        <p id="328">
                            <a id="bibliography_20" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES15120801000224&amp;v=MDQzMDFPc1BEbjQ5b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadEZpbmxVcjNJS0ZzVGF4QT1OaWZPZmJLOUg5UE1wNDlFWg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[20]</b>HALIM Z, WAQAS M, HUSSAIN S F.Clustering large probabilistic graphs using multi-population evolutionary algorithm[J].Information Sciences, 2015, 317 (1) :78-95.
                            </a>
                        </p>
                        <p id="330">
                            <a id="bibliography_21" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Graph based gene/protein prediction and clustering over uncertain medical databases">

                                <b>[21]</b>BANO R, RAO K.Graph based gene/protein prediction and clustering over uncertain medical databases[J].Journal of Theoretical and Applied Information Technology, 2015, 82 (3) :347-352.
                            </a>
                        </p>
                        <p id="332">
                            <a id="bibliography_22" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDC6960E2C87630F6FEB62B811B99CDF44&amp;v=MTIwMTF2RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0cGh6TDI0d2FvPU5qN0Jhc0MrRjlmTTJvMDJiT3dKRDN4UHlXQm1tRGwvT25manJXQThjTUhnTTc2YkNPTg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[22]</b>GAO Y J, MIAO X Y, CHEN G, et al.On efficiently finding reverse k-nearest neighbors over uncertain graphs[J].VLDB Journal, 2017, 26 (4) :1-26.
                            </a>
                        </p>
                        <p id="334">
                            <a id="bibliography_23" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZNGD201303027&amp;v=MTA0NzlHRnJDVVI3cWZadVpzRnlEZ1ZMN0tQeVBNYXJHNEg5TE1ySTlIWTRRS0RIODR2UjRUNmo1NE8zenFxQnQ=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[23]</b>李敏, 张含会, 费耀平.融合PPI和基因表达数据的关键蛋白质识别方法[J].中南大学学报 (自然科学版) , 2013, 44 (3) :1024-1039. (LI M, ZHANG H H, FEI Y P.Essential protein discovery method based on integration of PPI and gene expression data[J].Journal of Central South University (Science and Technology) , 2013, 44 (3) :1024-1039.) 
                            </a>
                        </p>
                        <p id="336">
                            <a id="bibliography_24" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=XNND201803018&amp;v=MTU3NDFIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlEZ1ZMN0tQU1BGYXJHNEg5bk1ySTlFYklRS0Q=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[24]</b>黄链, 邓磊.拟-偏b-度量空间中α-φ-压缩映象不动点的存在性[J].西南大学学报 (自然科学版) , 2018, 40 (3) :115-120. (HUANG L, DENG L.α-φ-contractive mappings on quasi-partial b-metric spaces[J].Journal of Southwest University (Natural Science Edition) , 2018, 40 (3) :115-120.) 
                            </a>
                        </p>
                        <p id="338">
                            <a id="bibliography_25" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJX201608006&amp;v=MTQ1NzdMN0tMejdCZHJHNEg5Zk1wNDlGWW9RS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVpzRnlEZ1Y=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[25]</b>朱镕, 邹兆年, 李建中.不确定图上的Top-k稠密子图挖掘算法[J].计算机学报, 2016, 39 (8) :1570-1582. (ZHU R, ZOUZ N, LI J Z.Mining Top-k dense subgraphs from uncertain graphs[J].Chinese Journal of Computers, 2016, 39 (8) :1570-1582.) 
                            </a>
                        </p>
                        <p id="340">
                            <a id="bibliography_26" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=SCLH201405017&amp;v=MDYyNTFNcW85RVk0UUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVac0Z5RGdWTDdLTmk3SFpyRzRIOVg=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[26]</b>胡赛, 熊慧军, 陈治平, 等.基于不确定网络的关键蛋白质识别[J].四川大学学报 (工程科学版) , 2014, 46 (5) :116-120. (HU S, XIONG H J, CHEN Z P, et al.Identification of essential proteins based on uncertain networks[J].Journal of Sichuan University (Engineering Science Edition) , 2014, 46 (5) :116-120.) 
                            </a>
                        </p>
                        <p id="342">
                            <a id="bibliography_27" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=DZXU200708030&amp;v=MTk2ODc0SHRiTXA0OUdaSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnNGeURnVkw3S0lUZlRlN0c=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[27]</b>王玲, 薄列峰, 焦李成.密度敏感的谱聚类[J].电子学报, 2007, 35 (8) :1577-1581. (WANG L, BO L F, JIAO L C.Density-sensitive spectral clustering[J].Acta Electronica Sinica, 2007, 35 (8) :1577-1581.) 
                            </a>
                        </p>
                        <p id="344">
                            <a id="bibliography_28" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Landmark Selection for Spectral Clustering Based on Weighted PageRank">

                                <b>[28]</b>RAFAILIDIS D, CONSTANTINOU E, MANOLOPOULOS Y.Landmark selection for spectral clustering based on weighted PageRank[J].Future Generation Computer Systems, 2017, 68 (3) :465-472.
                            </a>
                        </p>
                        <p id="346">
                            <a id="bibliography_29" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Fuzzy c-means clustering algorithm for directional data (FCM4DD).">

                                <b>[29]</b>KESEMEN O, TEZEL O, OZKUL E.Fuzzy C-means clustering algorithm for directional data (FCM4DD) [J].Expert Systems with Applications, 2016, 58:76-82.
                            </a>
                        </p>
                        <p id="348">
                            <a id="bibliography_30" target="_blank" href="http://scholar.cnki.net/result.aspx?q=DIP, the Database of Interacting Proteins: a research tool for studying cellular networks of protein interactions">

                                <b>[30]</b>XENARIOS I, SALWINSKI L, DUAN X J, et al.DIP, the database of interacting proteins:a research tool for studying cellular networks of protein interactions[J].Nucleic Acids Research, 2002, 30 (1) :303-305.
                            </a>
                        </p>
                        <p id="350">
                            <a id="bibliography_31" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Up-to-date catalogues of yeast protein complexes">

                                <b>[31]</b>PU S, WONG J, TURNER B, et al.Up-to-date catalogues of yeast protein complexes[J].Nucleic Acids Research, 2009, 37 (3) :825-831.
                            </a>
                        </p>
                        <p id="352">
                            <a id="bibliography_32" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Global landscape of protein complexes in the yeast Saccharomyces cerevisiae">

                                <b>[32]</b>KROGAN N, CAGNEY G, YU H, et al.Global landscape of protein complexes in the yeast Saccharomyces cerevisiae[J].Nature, 2006, 440 (7084) :637-643.
                            </a>
                        </p>
                        <p id="354">
                            <a id="bibliography_33" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=MOTO201512016&amp;v=MDc1MjY0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVac0Z5RGdWTDdLS0NMZlliRzRIOVROclk5RVlvUUtESDg0dlI=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[33]</b>胡赛, 熊慧军, 李学勇, 等.多关系蛋白质网络构建及其应用研究[J].自动化学报, 2015, 41 (12) :2155-2163. (HU S, XIONG H J, LI X Y, et al.Construction of multi-relation protein networks and its application[J].Acta Automatica Sinica, 2015, 41 (12) :2155-2163.) 
                            </a>
                        </p>
                        <p id="356">
                            <a id="bibliography_34" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD15121501608629&amp;v=MjgxNjRxbzlFWXVzSENuNHdvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp0RmlubFVyM0lLRnNUYXhBPU5pZmNhcks5SDlQTg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[34]</b>LEI X J, WU S, LIANG G, et al.Clustering and overlapping modules detection in PPI network based on IBFO[J].Proteomics, 2013, 13 (2) :278-290.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JSJY201904017" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSJY201904017&amp;v=MzAyNzMzenFxQnRHRnJDVVI3cWZadVpzRnlEZ1ZMN0tMejdCZDdHNEg5ak1xNDlFWTRRS0RIODR2UjRUNmo1NE8=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZJTWYxaVVtWjVZcnN4dE5mQUIrND0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
