<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637135764724346250%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJYRJ201904045%26RESULT%3d1%26SIGN%3dLIZtR1BINpKO48Od5aBL3cmroXs%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JYRJ201904045&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JYRJ201904045&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYRJ201904045&amp;v=MDIzMDdUNmo1NE8zenFxQnRHRnJDVVI3cWZadVp0Rnl6Z1U3N0JMelRaWkxHNEg5ak1xNDlCWVlRS0RIODR2UjQ=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#79" data-title="&lt;b&gt;0 引 言&lt;/b&gt; "><b>0 引 言</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#83" data-title="&lt;b&gt;1 基础方法&lt;/b&gt; "><b>1 基础方法</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#84" data-title="&lt;b&gt;1.1 局部描述符&lt;/b&gt;"><b>1.1 局部描述符</b></a></li>
                                                <li><a href="#91" data-title="&lt;b&gt;1.2 深度神经网络&lt;/b&gt;"><b>1.2 深度神经网络</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#99" data-title="&lt;b&gt;2 模型构建&lt;/b&gt; "><b>2 模型构建</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#100" data-title="&lt;b&gt;2.1 数据集&lt;/b&gt;"><b>2.1 数据集</b></a></li>
                                                <li><a href="#103" data-title="&lt;b&gt;2.2 性能评价指标&lt;/b&gt;"><b>2.2 性能评价指标</b></a></li>
                                                <li><a href="#112" data-title="&lt;b&gt;2.3 构建流程&lt;/b&gt;"><b>2.3 构建流程</b></a></li>
                                                <li><a href="#115" data-title="&lt;b&gt;2.4 参数调整&lt;/b&gt;"><b>2.4 参数调整</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#127" data-title="&lt;b&gt;3 实 验&lt;/b&gt; "><b>3 实 验</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#128" data-title="&lt;b&gt;3.1 DPPI模型实验&lt;/b&gt;"><b>3.1 DPPI模型实验</b></a></li>
                                                <li><a href="#137" data-title="&lt;b&gt;3.2 方法比较&lt;/b&gt;"><b>3.2 方法比较</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#144" data-title="&lt;b&gt;4 结 语&lt;/b&gt; "><b>4 结 语</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#87" data-title="&lt;b&gt;表1 基于侧链的偶极子和体积的氨基酸分类&lt;/b&gt;"><b>表1 基于侧链的偶极子和体积的氨基酸分类</b></a></li>
                                                <li><a href="#90" data-title="图1 一条蛋白质划分为10个区域的划分方法示意图">图1 一条蛋白质划分为10个区域的划分方法示意图</a></li>
                                                <li><a href="#93" data-title="图2 深度神经网络结构">图2 深度神经网络结构</a></li>
                                                <li><a href="#114" data-title="图3 DPPI模型构建流程图">图3 DPPI模型构建流程图</a></li>
                                                <li><a href="#118" data-title="&lt;b&gt;表2 DPPI模型学习率的调整&lt;/b&gt;"><b>表2 DPPI模型学习率的调整</b></a></li>
                                                <li><a href="#120" data-title="&lt;b&gt;表3 DPPI模型宽度的调整&lt;/b&gt;"><b>表3 DPPI模型宽度的调整</b></a></li>
                                                <li><a href="#122" data-title="&lt;b&gt;表4 DPPI模型深度的调整&lt;/b&gt;"><b>表4 DPPI模型深度的调整</b></a></li>
                                                <li><a href="#124" data-title="&lt;b&gt;表5 DPPI模型的最优预测性能&lt;/b&gt;"><b>表5 DPPI模型的最优预测性能</b></a></li>
                                                <li><a href="#126" data-title="&lt;b&gt;表6 DPPI模型参数表&lt;/b&gt;"><b>表6 DPPI模型参数表</b></a></li>
                                                <li><a href="#130" data-title="&lt;b&gt;表7 DPPI模型预测性能平均值&lt;/b&gt;"><b>表7 DPPI模型预测性能平均值</b></a></li>
                                                <li><a href="#135" data-title="图4 DPPI模型各个指标预测趋势图">图4 DPPI模型各个指标预测趋势图</a></li>
                                                <li><a href="#139" data-title="&lt;b&gt;表8 不同方法的预测性能的比较&lt;/b&gt;"><b>表8 不同方法的预测性能的比较</b></a></li>
                                                <li><a href="#140" data-title="&lt;b&gt;表9 不同算法采用LD编码方式结果比较&lt;/b&gt;"><b>表9 不同算法采用LD编码方式结果比较</b></a></li>
                                                <li><a href="#143" data-title="&lt;b&gt;表10 AC和LD编码时间的比较&lt;/b&gt;"><b>表10 AC和LD编码时间的比较</b></a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="179">


                                    <a id="bibliography_1" title=" Fields S, Song O. A Novel Genetic System to Detect Protein Protein Interactions[J]. Nature, 1989, 340 (6230) :245-246." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A novel genetic system to detect protein-protein interactions">
                                        <b>[1]</b>
                                         Fields S, Song O. A Novel Genetic System to Detect Protein Protein Interactions[J]. Nature, 1989, 340 (6230) :245-246.
                                    </a>
                                </li>
                                <li id="181">


                                    <a id="bibliography_2" title=" Ho Y, Gruhler A, Herlbut A, et al. Systematic Identification of Protein Complexes in Saccharomyces Cerevisiae by Mass Spectrometry[J]. Nature, 2002, 415 (6868) :180-183." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry">
                                        <b>[2]</b>
                                         Ho Y, Gruhler A, Herlbut A, et al. Systematic Identification of Protein Complexes in Saccharomyces Cerevisiae by Mass Spectrometry[J]. Nature, 2002, 415 (6868) :180-183.
                                    </a>
                                </li>
                                <li id="183">


                                    <a id="bibliography_3" title=" Gavin A C, Bosche M, Krause R, et al. Functional Organization of the Yeast Proteome by Systematic Analysis of Protein Complexes[J]. Nature, 2002, 415 (6868) : 141-147." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Functional organization of the yeast proteome by systematic analysis of protein complexes">
                                        <b>[3]</b>
                                         Gavin A C, Bosche M, Krause R, et al. Functional Organization of the Yeast Proteome by Systematic Analysis of Protein Complexes[J]. Nature, 2002, 415 (6868) : 141-147.
                                    </a>
                                </li>
                                <li id="185">


                                    <a id="bibliography_4" title=" Heng Z, Metin B, Rhonda B, et al. Global Analysis of Protein Activities Using Proteome Chips[J]. Science, 2001, 293 (5537) :2101-2105." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Global Analysis of Protein Activities Using Proteome Chips">
                                        <b>[4]</b>
                                         Heng Z, Metin B, Rhonda B, et al. Global Analysis of Protein Activities Using Proteome Chips[J]. Science, 2001, 293 (5537) :2101-2105.
                                    </a>
                                </li>
                                <li id="187">


                                    <a id="bibliography_5" title=" Chatterjee P, Basu S, Kundu M, et al. PPI_SVM: Prediction of protein-protein interactions using machine learning, domain-domain affinities and frequency tables[J]. Cellular &amp;amp; Molecular Biology Letters, 2011, 16 (2) :264-278." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=PPI_SVM: Prediction of protein-protein interactions using machine learning, domain-domain affinities and frequency tables">
                                        <b>[5]</b>
                                         Chatterjee P, Basu S, Kundu M, et al. PPI_SVM: Prediction of protein-protein interactions using machine learning, domain-domain affinities and frequency tables[J]. Cellular &amp;amp; Molecular Biology Letters, 2011, 16 (2) :264-278.
                                    </a>
                                </li>
                                <li id="189">


                                    <a id="bibliography_6" title=" Fariselli P, Pazos F, Valencia A, et al. Prediction of protein—protein interaction sites in heterocomplexes with neural networks[J]. European Journal of Biochemistry, 2002, 269 (5) : 1356-1361." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prediction of protein--protein interaction sites in heterocomplexes with neural networks">
                                        <b>[6]</b>
                                         Fariselli P, Pazos F, Valencia A, et al. Prediction of protein—protein interaction sites in heterocomplexes with neural networks[J]. European Journal of Biochemistry, 2002, 269 (5) : 1356-1361.
                                    </a>
                                </li>
                                <li id="191">


                                    <a id="bibliography_7" title=" Lin X, Chen X W. Heterogeneous Data Integration by Tree Augmented Naive Bayes for Protein-Protein Interactions Prediction[J]. Proteomics, 2013, 13 (2) : 261-268." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD15121501608600&amp;v=MjYwMzR6SUpGc1VheHM9TmlmY2FySzlIOVBOcW85RVl1c0hDbnc1b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadEZpbmxVcg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[7]</b>
                                         Lin X, Chen X W. Heterogeneous Data Integration by Tree Augmented Naive Bayes for Protein-Protein Interactions Prediction[J]. Proteomics, 2013, 13 (2) : 261-268.
                                    </a>
                                </li>
                                <li id="193">


                                    <a id="bibliography_8" title=" Browne F, Wang H, Zheng H, et al. Supervised Statistical and Machine Learning Approaches to Inferring Pairwise and Module Based Protein Interaction Networks[C]//Proceedings of the 7th IEEE International Conference on Bioinformatics and Bioengineering, New York, USA: IEEE, 2007: 1365-1369." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Supervised Statistical and Machine Learning Approaches to Inferring Pairwise and Module Based Protein Interaction Networks">
                                        <b>[8]</b>
                                         Browne F, Wang H, Zheng H, et al. Supervised Statistical and Machine Learning Approaches to Inferring Pairwise and Module Based Protein Interaction Networks[C]//Proceedings of the 7th IEEE International Conference on Bioinformatics and Bioengineering, New York, USA: IEEE, 2007: 1365-1369.
                                    </a>
                                </li>
                                <li id="195">


                                    <a id="bibliography_9" title=" Valente G T, Acencio M L, Martins C, et al. The Development of a Universal in Silico Predictor of Protein-Protein Interactions[J]. PLoS One, 2013, 8 (5) : e65587." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The development of a universal in silico predictor of protein-protein interactions">
                                        <b>[9]</b>
                                         Valente G T, Acencio M L, Martins C, et al. The Development of a Universal in Silico Predictor of Protein-Protein Interactions[J]. PLoS One, 2013, 8 (5) : e65587.
                                    </a>
                                </li>
                                <li id="197">


                                    <a id="bibliography_10" title=" Chen X W, Liu M. Prediction of Protein-Protein Interactions Using Random Decision Forest Framework[J]. Bioinformatics, 2005, 21 (24) : 4394-4400." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prediction of protein?protein interactions using random decision forest framework">
                                        <b>[10]</b>
                                         Chen X W, Liu M. Prediction of Protein-Protein Interactions Using Random Decision Forest Framework[J]. Bioinformatics, 2005, 21 (24) : 4394-4400.
                                    </a>
                                </li>
                                <li id="199">


                                    <a id="bibliography_11" title=" Saha I, Zubek J, Klingstr&#246;m T, et al. Ensemble Learning Prediction of Protein-Protein Interactions Using Proteins Functional Annotations[J]. Molecular Biosystems, 2014, 10 (4) : 820-830." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Ensemble learning prediction of protein-protein interactions using proteins functional annotations">
                                        <b>[11]</b>
                                         Saha I, Zubek J, Klingstr&#246;m T, et al. Ensemble Learning Prediction of Protein-Protein Interactions Using Proteins Functional Annotations[J]. Molecular Biosystems, 2014, 10 (4) : 820-830.
                                    </a>
                                </li>
                                <li id="201">


                                    <a id="bibliography_12" title=" Mohanty S P, Hughes D P, Salath&#233; M. Using Deep Learning for Image-Based Plant Disease Detection[J]. Frontiers in Plant Science, 2016, 7:1419." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Using deep learning for image-based plant disease detection">
                                        <b>[12]</b>
                                         Mohanty S P, Hughes D P, Salath&#233; M. Using Deep Learning for Image-Based Plant Disease Detection[J]. Frontiers in Plant Science, 2016, 7:1419.
                                    </a>
                                </li>
                                <li id="203">


                                    <a id="bibliography_13" title=" Collobert G, Weston J. A Unified Architecture for Natural Language Processing: Deep Neural Networks with Multitask Learning[C]//International Conference on Machine Learning, 2008:160-167." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A Unified Architecture for Natural Language Processing:Deep Neural Networks with Multitask Learning">
                                        <b>[13]</b>
                                         Collobert G, Weston J. A Unified Architecture for Natural Language Processing: Deep Neural Networks with Multitask Learning[C]//International Conference on Machine Learning, 2008:160-167.
                                    </a>
                                </li>
                                <li id="205">


                                    <a id="bibliography_14" title=" Silver D, Huang A, Maddison C J, et al. Mastering the Game of Go with Deep Neural Networks and Tree Search[J]. Nature, 2016, 529 (7587) :484-489." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Mastering the game of Go with deep neural networks and tree search">
                                        <b>[14]</b>
                                         Silver D, Huang A, Maddison C J, et al. Mastering the Game of Go with Deep Neural Networks and Tree Search[J]. Nature, 2016, 529 (7587) :484-489.
                                    </a>
                                </li>
                                <li id="207">


                                    <a id="bibliography_15" title=" Du X Q, Sun S W, Hu CL, et al. DeepPPI: Boosting Prediction of Protein-Protein Interactions with Deep Neural Networks[J]. Journal of Chemical Information &amp;amp; Modeling, 2017, 57 (6) :1499-1510." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Deep PPI:Boosting prediction of protein-protein interactions with deep neural networks">
                                        <b>[15]</b>
                                         Du X Q, Sun S W, Hu CL, et al. DeepPPI: Boosting Prediction of Protein-Protein Interactions with Deep Neural Networks[J]. Journal of Chemical Information &amp;amp; Modeling, 2017, 57 (6) :1499-1510.
                                    </a>
                                </li>
                                <li id="209">


                                    <a id="bibliography_16" title=" Leung M K, Xiong H Y, Lee L J, et al. Deep Learning of the Tissue-Regulated Splicing Code[J]. Bioinformatics, 2014, 30 (12) : i121-i129." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Deep learning of the tissue-regulated splicing code">
                                        <b>[16]</b>
                                         Leung M K, Xiong H Y, Lee L J, et al. Deep Learning of the Tissue-Regulated Splicing Code[J]. Bioinformatics, 2014, 30 (12) : i121-i129.
                                    </a>
                                </li>
                                <li id="211">


                                    <a id="bibliography_17" title=" Zhou J, Troyanskaya O G. Predicting Effects of Noncoding Variants with Deep Learning-Based Sequence Model[J]. Nature Methods, 2015, 12 (10) : 931-934." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Predicting effects of noncoding variants with deep learning-based sequence model">
                                        <b>[17]</b>
                                         Zhou J, Troyanskaya O G. Predicting Effects of Noncoding Variants with Deep Learning-Based Sequence Model[J]. Nature Methods, 2015, 12 (10) : 931-934.
                                    </a>
                                </li>
                                <li id="213">


                                    <a id="bibliography_18" title=" Bengio Y, Courville A, Vincent P. Representation Learning: A Review and New Perspectives[J]. IEEE Transactions on Pattern Analysis &amp;amp; Machine Intelligence, 2012, 35 (8) :1798-1828." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Representation learning:A review and new perspectives">
                                        <b>[18]</b>
                                         Bengio Y, Courville A, Vincent P. Representation Learning: A Review and New Perspectives[J]. IEEE Transactions on Pattern Analysis &amp;amp; Machine Intelligence, 2012, 35 (8) :1798-1828.
                                    </a>
                                </li>
                                <li id="215">


                                    <a id="bibliography_19" title=" Alipanahi B, Delong A, Weirauch M T, et al. Predicting the sequence specificities of DNA—and RNA-binding proteins by deep learning[J]. Nature Biotechnology, 2015, 33 (8) : 831-838." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Predicting the sequence specificities of DNA-and RNA-binding proteins by deep learning">
                                        <b>[19]</b>
                                         Alipanahi B, Delong A, Weirauch M T, et al. Predicting the sequence specificities of DNA—and RNA-binding proteins by deep learning[J]. Nature Biotechnology, 2015, 33 (8) : 831-838.
                                    </a>
                                </li>
                                <li id="217">


                                    <a id="bibliography_20" title=" Krizhevsky A, Sutskever I, Hinton G E. Imagenet Classification with Deep Convolutional Neural Networks[J]. International Conference on Neural Information Processing Systems, 2012, 60 (2) : 1097-1105." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Image Net classification with deep convolutional neural networks">
                                        <b>[20]</b>
                                         Krizhevsky A, Sutskever I, Hinton G E. Imagenet Classification with Deep Convolutional Neural Networks[J]. International Conference on Neural Information Processing Systems, 2012, 60 (2) : 1097-1105.
                                    </a>
                                </li>
                                <li id="219">


                                    <a id="bibliography_21" title=" Tian K, Shao M Y, Wang Y, et al. Boosting Compound-Protein Interaction Prediction by Deep Learning[J]. Methods, 2016, 110: 64-72." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES56B53E38E4B3FBB22783327E47E04993&amp;v=MTg3MTRZeURMS1FUTE9jQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQnJMVTA1dHRod0wyL3dhRT1OaWZPZmJhK2JOVFAyb3hORWU5OUR3cEx2UlFSN1RkK1MzM2wyUg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[21]</b>
                                         Tian K, Shao M Y, Wang Y, et al. Boosting Compound-Protein Interaction Prediction by Deep Learning[J]. Methods, 2016, 110: 64-72.
                                    </a>
                                </li>
                                <li id="221">


                                    <a id="bibliography_22" >
                                        <b>[22]</b>
                                     Yang L, Xia J F, Gui F. Prediction of Protein-Protein Interactions from Protein Sequence Using Local Descriptors[J]. Protein &amp;amp; Peptide Letters, 2010, 17 (19) :1085-1090.</a>
                                </li>
                                <li id="223">


                                    <a id="bibliography_23" title=" Davies M N, Secker A, Freitas A A, et al. Optimizing Amino Acid Groupings for GPCR Classification[J]. Bioinformatics, 2008, 24 (18) :1980-1986." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Optimizing amino acid groupings for GPCR classification">
                                        <b>[23]</b>
                                         Davies M N, Secker A, Freitas A A, et al. Optimizing Amino Acid Groupings for GPCR Classification[J]. Bioinformatics, 2008, 24 (18) :1980-1986.
                                    </a>
                                </li>
                                <li id="225">


                                    <a id="bibliography_24" >
                                        <b>[24]</b>
                                     Hinton G E, Salakhutdinov R R. Reducing the Dimensionality of Data with Neural Networks[J]. Science, 2006, 313 (5786) : 504-507.</a>
                                </li>
                                <li id="227">


                                    <a id="bibliography_25" title=" Hinton G E, Osindero S, Teh Y W. A Fast Learning Algorithm for Deep Belief Nets[J]. Neural Computation, 2014, 18 (7) : 1527-1554." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJBK&amp;filename=SJBK15090500012538&amp;v=MTAwMzRkR2VycVFUTW53WmVadEZpbmxVcnpJSkZzVWF4cz1OaWZKWmJLOUh0ak1xbzlGWk9vTkNYOHhvQk1UNlQ0UFFIL2lyUg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[25]</b>
                                         Hinton G E, Osindero S, Teh Y W. A Fast Learning Algorithm for Deep Belief Nets[J]. Neural Computation, 2014, 18 (7) : 1527-1554.
                                    </a>
                                </li>
                                <li id="229">


                                    <a id="bibliography_26" title=" Glorot X, Bordes A, Bengio Y. Deep Sparse Rectifier Neural Networks[C]//Proceedings of the 14th International Conference on Artificial Intelligence and Statistics (AISTATS) , 2010: 315-323." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Deep sparse rectifier neural networks">
                                        <b>[26]</b>
                                         Glorot X, Bordes A, Bengio Y. Deep Sparse Rectifier Neural Networks[C]//Proceedings of the 14th International Conference on Artificial Intelligence and Statistics (AISTATS) , 2010: 315-323.
                                    </a>
                                </li>
                                <li id="231">


                                    <a id="bibliography_27" title=" Srivastava N, Hinton G, Krizhevsky A, et al. Dropout: A Simple Way to Prevent Neural Networks from Over Fitting. Journal of Machine Learning Research, 2014, 15 (1) : 1929-1958." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Dropout:A simple way to prevent neural networks from overfitting">
                                        <b>[27]</b>
                                         Srivastava N, Hinton G, Krizhevsky A, et al. Dropout: A Simple Way to Prevent Neural Networks from Over Fitting. Journal of Machine Learning Research, 2014, 15 (1) : 1929-1958.
                                    </a>
                                </li>
                                <li id="233">


                                    <a id="bibliography_28" title=" Kingma D, Ba J. Adam: A Method for Stochastic Optimization[DB]. arXiv:1412.6980V8, 2014." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Adam: A Method for Stochastic Optimization[DB]">
                                        <b>[28]</b>
                                         Kingma D, Ba J. Adam: A Method for Stochastic Optimization[DB]. arXiv:1412.6980V8, 2014.
                                    </a>
                                </li>
                                <li id="235">


                                    <a id="bibliography_29" title=" Bengio Y. Practical Recommendations for Gradient-Based Training of Deep Architectures[J]. Springer Berlin Heidelberg, 2012, 7700 (1-3) : 437-478." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Practical Recommendations for Gradient-Based Training of Deep Architectures">
                                        <b>[29]</b>
                                         Bengio Y. Practical Recommendations for Gradient-Based Training of Deep Architectures[J]. Springer Berlin Heidelberg, 2012, 7700 (1-3) : 437-478.
                                    </a>
                                </li>
                                <li id="237">


                                    <a id="bibliography_30" title=" Shen J, Zhang J, Luo X, et al. Predicting protein-protein interactions based only on sequences information[J]. Proceedings of the National Academy of Sciences, 2007, 104 (11) :4337-4341." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Predicting protein–protein interactions based only on sequences information">
                                        <b>[30]</b>
                                         Shen J, Zhang J, Luo X, et al. Predicting protein-protein interactions based only on sequences information[J]. Proceedings of the National Academy of Sciences, 2007, 104 (11) :4337-4341.
                                    </a>
                                </li>
                                <li id="239">


                                    <a id="bibliography_31" title=" You Z H, Li S, Gao X, et al. Large-Scale Protein-Protein Interactions Detection by Integrating Big Biosensing Data with Computational Model[J]. Biomed Research International, 2014, 2014 (2) :598129." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJHD&amp;filename=SJHD14090200000112&amp;v=MjE5OTZRSC9pclJkR2VycVFUTW53WmVadEZpbmxVcnpJSkZzVWF4cz1OaWZEYXJLOEh0ak1yWTlGWk9zUERYMDdvQk1UNlQ0UA==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[31]</b>
                                         You Z H, Li S, Gao X, et al. Large-Scale Protein-Protein Interactions Detection by Integrating Big Biosensing Data with Computational Model[J]. Biomed Research International, 2014, 2014 (2) :598129.
                                    </a>
                                </li>
                                <li id="241">


                                    <a id="bibliography_32" title=" Guo Y, Li M, Pu X, et al. PRED_PPI: A Server for Predicting Protein-Protein Interactions Based on Sequence Data with Probability Assignment[J]. Bmc Research Notes, 2010, 3 (1) : 145-152." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=PRED_PPI: a server for predicting protein-protein interactions based on sequence data with probability assignment">
                                        <b>[32]</b>
                                         Guo Y, Li M, Pu X, et al. PRED_PPI: A Server for Predicting Protein-Protein Interactions Based on Sequence Data with Probability Assignment[J]. Bmc Research Notes, 2010, 3 (1) : 145-152.
                                    </a>
                                </li>
                                <li id="243">


                                    <a id="bibliography_33" title=" Pan X Y, Zhang Y N, Shen H B. Large-scale Prediction of Human Protein-Protein Interactions from Amino Acid Sequence Based on Latent Topic Features[J]. Journal of Proteome Research 2010, 9 (10) :4992-5001." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Large-Scale Prediction of Human Protein-Protein Interactions from Amino Acid Sequence Based on Latent Topic Features">
                                        <b>[33]</b>
                                         Pan X Y, Zhang Y N, Shen H B. Large-scale Prediction of Human Protein-Protein Interactions from Amino Acid Sequence Based on Latent Topic Features[J]. Journal of Proteome Research 2010, 9 (10) :4992-5001.
                                    </a>
                                </li>
                                <li id="245">


                                    <a id="bibliography_34" title=" Sun T, Zhou B, Lai H, et al. Sequence-Based Prediction of Protein Protein Interaction Using a Deep-Learning Algorithm[J]. BMC Bioinformatics, 2017, 18 (1) : 277-285." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDD8E623236FB51C19591C69B5C2F3C3F0&amp;v=MDIyODVMVTA1dHRod0wyL3dhRT1OajdCYXNld2E5Zk9ySTFHWXAxOUNYMUt6aDhXNHo0T1RuYVFxV0UzRDdIblJzeWZDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCcg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[34]</b>
                                         Sun T, Zhou B, Lai H, et al. Sequence-Based Prediction of Protein Protein Interaction Using a Deep-Learning Algorithm[J]. BMC Bioinformatics, 2017, 18 (1) : 277-285.
                                    </a>
                                </li>
                                <li id="247">


                                    <a id="bibliography_35" title=" You Z H, Lei Y K, Zhu L, et al. Prediction of Protein-Protein Interactions from Amino Acid Sequences with Ensemble Extreme Learning Machines and Principal Component Analysis[J]. BMC Bioinformatics, 2013, 14 (S8) : S10." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prediction\sof protein-protein interactions from amino acid sequences\swith ensemble extreme learning machines and principal component analysis">
                                        <b>[35]</b>
                                         You Z H, Lei Y K, Zhu L, et al. Prediction of Protein-Protein Interactions from Amino Acid Sequences with Ensemble Extreme Learning Machines and Principal Component Analysis[J]. BMC Bioinformatics, 2013, 14 (S8) : S10.
                                    </a>
                                </li>
                                <li id="249">


                                    <a id="bibliography_36" title=" Zhou Y Z, Gao Y, Zheng Y Y. Prediction of Protein-Protein Interactions Using Local Description of Amino Acid Sequence[J]. Communications in Computer &amp;amp; Information Science, 2011, 202:254-262." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prediction of protein-protein interactions using local description of amino acid sequence">
                                        <b>[36]</b>
                                         Zhou Y Z, Gao Y, Zheng Y Y. Prediction of Protein-Protein Interactions Using Local Description of Amino Acid Sequence[J]. Communications in Computer &amp;amp; Information Science, 2011, 202:254-262.
                                    </a>
                                </li>
                                <li id="251">


                                    <a id="bibliography_37" title=" Yang L, Xia J F, Gui J. Prediction of Protein-Protein Interactions from Protein Sequence Using Local Descriptors[J]. Protein &amp;amp; Peptide Letters, 2010, 17 (9) :1085-1090." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Prediction of protein-protein interactions from protein sequence using local descriptors">
                                        <b>[37]</b>
                                         Yang L, Xia J F, Gui J. Prediction of Protein-Protein Interactions from Protein Sequence Using Local Descriptors[J]. Protein &amp;amp; Peptide Letters, 2010, 17 (9) :1085-1090.
                                    </a>
                                </li>
                                <li id="253">


                                    <a id="bibliography_38" title=" Wang J, Zhang L, Jia L Y, et al. Protein-Protein Interactions Prediction Using a Novel Local Conjoint Triad Descriptor of Amino Acid Sequences[J]. International Journal of Molecular Sciences, 2017, 18 (11) :2373." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Protein-Protein Interactions Prediction Using a Novel Local Conjoint Triad Descriptor of Amino Acid Sequences">
                                        <b>[38]</b>
                                         Wang J, Zhang L, Jia L Y, et al. Protein-Protein Interactions Prediction Using a Novel Local Conjoint Triad Descriptor of Amino Acid Sequences[J]. International Journal of Molecular Sciences, 2017, 18 (11) :2373.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JYRJ" target="_blank">计算机应用与软件</a>
                2019,36(04),273-280 DOI:10.3969/j.issn.1000-386x.2019.04.044            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>基于深度神经网络和局部描述符的大规模蛋白质互作预测方法</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="javascript:;">桂元苗</a>
                                <a href="javascript:;">王儒敬</a>
                                <a href="javascript:;">王雪</a>
                                <a href="javascript:;">魏圆圆</a>
                </h2>
                    <h2>

                    <span>中国科学院合肥物质科学研究院智能机械研究所</span>
                    <span>中国科学技术大学信息技术学院</span>
                    <span>中国科学院合肥物质科学研究院技术生物与农业工程研究所</span>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>蛋白质相互作用PPI (Protein-Protein Interaction) 是生物体中众多生命活动过程的重要组成部分, 蛋白质互作预测是研究蛋白质互作的重要途径。为了提高蛋白质互作预测性能, 构建一个用于预测蛋白质互作的深度神经网络模型DPPI。采用局部描述符将氨基酸序列编码成具鉴别性的特定维数向量;使用训练集训练DPPI模型, 并使用测试集对DPPI模型进行测试和评价;根据测试和评价的结果调整各参数, 优化DPPI模型;使用优化后的DPPI模型, 来对蛋白质互作进行预测。结果表明, DPPI模型编码简单、代码简洁, 实验获得的较高的准确率, 可以作为大规模蛋白质互作预测的有益补充。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%B7%B1%E5%BA%A6%E7%A5%9E%E7%BB%8F%E7%BD%91%E7%BB%9C&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">深度神经网络;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%B1%80%E9%83%A8%E6%8F%8F%E8%BF%B0%E7%AC%A6&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">局部描述符;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E8%9B%8B%E7%99%BD%E8%B4%A8%E4%BA%92%E4%BD%9C&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">蛋白质互作;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    桂元苗, 助理研究员, 主研领域:机器学习, 模式识别。;
                                </span>
                                <span>
                                    王儒敬, 研究员。;
                                </span>
                                <span>
                                    王雪, 助理研究员。;
                                </span>
                                <span>
                                    魏圆圆, 副研究员。;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-10-19</p>

            </div>
                    <h1><b>A LARGE-SCALE PREDICTION OF PROTEIN-PROTEIN INTERACTIONS BASED ON DEEP NEURAL NETWORK COMBINED WITH LOCAL DESCRIPTOR</b></h1>
                    <h2>
                    <span>Gui Yuanmiao</span>
                    <span>Wang Rujing</span>
                    <span>Wang Xue</span>
                    <span>Wei Yuanyuan</span>
            </h2>
                    <h2>
                    <span>Institute of Intelligent Machine, Hefei Institutes of Physical Science, Chinese Academy of Sciences</span>
                    <span>Institute of Information Technology, University of Science and Technology of China</span>
                    <span>Institute of Technical Biology and Agriculture Engineering, Hefei Institutes of Physical Science, Chinese Academy of Sciences</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>Protein-protein interaction (PPI) plays an important role in many biological processes. PPI prediction is an important way to study protein interaction. In order to improve the prediction performance of protein interaction, a DPPI model was constructed to protein-protein interactions. The local descriptor was used to encode the amino acid sequence into a discriminative specific dimension vector. Then, the training set was used to train DPPI model, and the test set was used for testing and evaluation. We adjusted the parameters according to the results of the test and evaluation to optimize the DPPI model. The optimized DPPI model was used to predict protein interactions. The results find that the DPPI model has simple encoding, simple code, and higher accuracy. It can be used as a useful supplement for large-scale protein interaction prediction.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Deep%20neural%20network&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Deep neural network;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Local%20descriptor&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Local descriptor;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Protein-protein%20interaction%20(PPI)%20&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Protein-protein interaction (PPI) ;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                                            </p>
                                    <p><b>Received：</b> 2018-10-19</p>
                            </div>


        <!--brief start-->
                        <h3 id="79" name="79" class="anchor-tag"><b>0 引 言</b></h3>
                <div class="p1">
                    <p id="80">蛋白质相互作用PPI是生物体中众多生命活动过程的重要组成部分, 在许多细胞生物学过程中起着重要的作用。新陈代谢、信号转导、细胞周期调控、新陈代谢、细胞凋亡及免疫应答等一系列生命活动都是通过蛋白质相互作用实现的。蛋白质互作预测是研究蛋白质互作的重要途径。近几年涌现了诸多预测蛋白质相互作用的高通量实验方法, 例如:酵母双杂交方法<citation id="255" type="reference"><link href="179" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>、质谱蛋白质复合物鉴别<citation id="256" type="reference"><link href="181" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>、质谱分析<citation id="257" type="reference"><link href="183" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>以及蛋白质芯片方法<citation id="258" type="reference"><link href="185" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>等。然而, 这些使用化学实验的方法, 需要耗费大量的人力、财力和时间, 难以应用于大规模的蛋白互作预测。机器学习的出现, 使大规模的蛋白互作预测成为可能。到目前为止, 已经出现的大量机器学习模型, 包括支持向量SVM<citation id="259" type="reference"><link href="187" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>、神经网络NN (Neural Networks) <citation id="260" type="reference"><link href="189" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>、朴素贝叶斯<citation id="261" type="reference"><link href="191" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>、K-最近邻<citation id="262" type="reference"><link href="193" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>等已经被用来预测PPI。 尽管上述PPI预测方法很流行, 但仍然存在一定的局限性, 一般的机器学习模型无法很好地处理蛋白序列噪声输入中的隐性关联<citation id="263" type="reference"><link href="195" rel="bibliography" /><link href="197" rel="bibliography" /><link href="199" rel="bibliography" /><sup>[<a class="sup">9</a>,<a class="sup">10</a>,<a class="sup">11</a>]</sup></citation>。深度神经网络的出现, 为这类问题提供了强有力的解决方案。</p>
                </div>
                <div class="p1">
                    <p id="81">深度神经网络 (DNN) 是机器学习中最活跃的领域之一, 可自动从数据中提取高层抽象信息, 用于复杂预测任务, 如语音和图像识别<citation id="264" type="reference"><link href="201" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>、自然语言理解<citation id="265" type="reference"><link href="203" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>、决策制定<citation id="266" type="reference"><link href="205" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>以及最近的计算生物学<citation id="274" type="reference"><link href="207" rel="bibliography" /><link href="209" rel="bibliography" /><link href="211" rel="bibliography" /><sup>[<a class="sup">15</a>,<a class="sup">16</a>,<a class="sup">17</a>]</sup></citation>。Leung等<citation id="267" type="reference"><link href="209" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation>利用深度神经网络和rna-seq数据, 建立了一个预测个体组织和组织间剪接模式差异的模型;Zhou等<citation id="268" type="reference"><link href="211" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>使用深度神经网络开发了深度学习框架 (DeepSEA http://deepsea.princeton.edu/) 。该框架可以从染色质谱分析序列中学习调控序列代码, 并且可以改进功能变体的优先级。与基于序列的其他机器学习方法相比, 深度神经网络具有几个优点: (1) Bengio等<citation id="269" type="reference"><link href="213" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>证明深度神经网络能够减少噪声对原始数据的影响, 并学习真正隐藏的高层特征; (2) Alipanahi等<citation id="270" type="reference"><link href="215" rel="bibliography" /><sup>[<a class="sup">19</a>]</sup></citation>发现, 深度神经网络可以使用各种实验数据和评估指标自动学习蛋白质的特定序列基序; (3) Krizhevsky等<citation id="271" type="reference"><link href="217" rel="bibliography" /><sup>[<a class="sup">20</a>]</sup></citation>人为地将噪声引入基于深度神经网络的方法来减少过度拟合, 并且揭示深度神经网络可以增强模型泛化。最近, 深度神经网络用于蛋白网络互作也取得了良好的结果<citation id="275" type="reference"><link href="207" rel="bibliography" /><link href="219" rel="bibliography" /><sup>[<a class="sup">15</a>,<a class="sup">21</a>]</sup></citation>, Tian等<citation id="272" type="reference"><link href="219" rel="bibliography" /><sup>[<a class="sup">21</a>]</sup></citation>提出了一种称为DL-CPI (复合蛋白质相互作用预测的深度学习的缩写) 的方法。该方法使用深度神经网络通过分层提取来学习复合蛋白对的有用特征, 从而在平衡数据集和不平衡数据集上取得了比现有方法更好的预测性能, 有效地提高复合蛋白互作的预测性能。Du等<citation id="273" type="reference"><link href="207" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation>使用深度神经网络基于氨基酸序列来研究蛋白互作预测, 并分别获得了92.50%的准确率和90.50%的召回率。尽管深度神经网络算法在蛋白质互作预测中取得了成功的结果, 但基于深度神经网络和局部描述符的蛋白互作预测的研究很少见。</p>
                </div>
                <div class="p1">
                    <p id="82">本文首先采用局部描述符将蛋白质序列编码成固定长度的向量, 并随机分成训练集和测试集;然后将训练集输入深度神经网络, 调整并优化网络结构和学习率、丢弃率等模型参数, 训练蛋白互作预测模型DPPI;最后DPPI模型经过测试和验证用于蛋白质互作预测, 并将预测的结果和前人的蛋白质互作预测方法进行比较。</p>
                </div>
                <h3 id="83" name="83" class="anchor-tag"><b>1 基础方法</b></h3>
                <h4 class="anchor-tag" id="84" name="84"><b>1.1 局部描述符</b></h4>
                <div class="p1">
                    <p id="85">深度学习等机器学习方法的输入均为某一维数空间中的向量。为使深度学习方法学习并预测蛋白互作关系成为可能, 必然要求将长度不统一的蛋白序列编码成某一维数空间中的向量。为了将蛋白质序列编码成维数相同的空间向量, Yang等<citation id="276" type="reference"><link href="221" rel="bibliography" /><sup>[<a class="sup">22</a>]</sup></citation>首次将局部描述符LD (Local Descriptor) 应用于蛋白互作预测, 在酒酿酵母数据集上达到86.15%的准确率。LD<citation id="277" type="reference"><link href="223" rel="bibliography" /><sup>[<a class="sup">23</a>]</sup></citation>是一种无需序列比对的方法, 其效果在很大程度上取决于潜在的氨基酸分类。首先, 依据氨基酸侧链的偶极性和体积将20种氨基酸分成7组 (见表1) , 并将蛋白序列中的所有氨基酸替换成对应的分组编码。例如, 蛋白序列“MESSKKMDSPGALQTNP”转换成“36335536321124342”。</p>
                </div>
                <div class="area_img" id="87">
                    <p class="img_tit"><b>表1 基于侧链的偶极子和体积的氨基酸分类</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="87" border="1"><tr><td><br />分组</td><td>氨基酸种类</td></tr><tr><td><br />1</td><td>Ala (A) , Gly (G) , Val (V) </td></tr><tr><td><br />2</td><td>Ile (I) , Leu (L) , Phe (F) , Pro (P) </td></tr><tr><td><br />3</td><td>Tyr (Y) , Met (M) , Thr (T) , Ser (S) </td></tr><tr><td><br />4</td><td>His (H) , Asn (N) , Gln (Q) , Trp (W) </td></tr><tr><td><br />5</td><td>Arg (R) , Lys (K) </td></tr><tr><td><br />6</td><td>Asp (D) , Glu (E) </td></tr><tr><td><br />7</td><td>Cys (C) </td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="88">然后, 依据氨基酸官能团在蛋白质初级序列中发生的变化计算Composition (C) 、Transition (T) 和Distribution (D) 。其中:Composition为各组氨基酸在整条蛋白序列中所占的比例;Transition指一组氨基酸中的氨基酸残基和另外一组氨基酸中的氨基酸残基相邻的频率;Distribution指在一条蛋白质序列中每组氨基酸的氨基酸残基数目的第一个、25%、50%、75%和100%在整个蛋白质序列中所占位置的比例。所以一个氨基酸片段可以用63维的向量表示:7 (计算C得到的) +21 (计算T得到的 (7×6) /2) +35 (计算D得到的7×5) 。</p>
                </div>
                <div class="p1">
                    <p id="89">为了更好地从蛋白质的氨基酸片段中捕捉蛋白质相互作用信息, 本实验将每条蛋白质序列划分为10个局部区域 (A-J) , 见图1。区域 (A-D) 是把一条蛋白质序列分成四个相等的区域;区域 (E-F) 是把一条蛋白质序列分成二个相等的区域;区域G表示位于蛋白质序列中间的50%氨基酸片段;区域H表示整条蛋白质序列的前75%的氨基酸片段;区域I表示整条蛋白质序列的后75%的氨基酸片段;区域J表示整条蛋白质序列的中间75%的氨基酸片段。一条蛋白质序列的所有局部区域氨基酸片段的编码, 串联在一起就形成了一条蛋白质序列的编码, 得到630维向量。因此, 本文构造了一个1 260维向量来表示每个蛋白质对, 并将其作为DPPI模型的输入向量。</p>
                </div>
                <div class="area_img" id="90">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201904045_090.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 一条蛋白质划分为10个区域的划分方法示意图" src="Detail/GetImg?filename=images/JYRJ201904045_090.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 一条蛋白质划分为10个区域的划分方法示意图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201904045_090.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="91" name="91"><b>1.2 深度神经网络</b></h4>
                <div class="p1">
                    <p id="92">DNN是指一组模仿人类大脑设计的, 旨在识别模式的算法。DNN由输入层、一个或多个隐藏层以及输出层三部分组成, 如图2所示。一般来说, 第一层是输入层, 最后一层是输出层, 而中间的层都是隐藏层。相邻层之间全连接, 即第<i>i</i>层的任意一个神经元与第<i>i</i>+1层的任意一个神经元相连。DNN类似于一般的人工神经网络, 然而, 隐藏层的数量和训练过程是不同的。Hinton等<citation id="278" type="reference"><link href="225" rel="bibliography" /><sup>[<a class="sup">24</a>]</sup></citation>利用预训练方法缓解了局部最优解问题, 将隐含层推动到了7层后, DNN才有了真正意义上的“深度”。DNN在输入层接收数据, 在各个节点中将输入数据与权重相结合以非线性方式转换这些数据, 通过计算平均梯度并相应地调整权重和激活函数, 最后在输出层计算最终输出。</p>
                </div>
                <div class="area_img" id="93">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201904045_093.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 深度神经网络结构" src="Detail/GetImg?filename=images/JYRJ201904045_093.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 深度神经网络结构  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201904045_093.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="94">虽然DNN很复杂, 但是从局部模型来说, 还是和感知机一样, 由一个线性关系加上一个非线性激活函数组成, 用矩阵法表示, 第<i>l</i>层的输出为:</p>
                </div>
                <div class="p1">
                    <p id="95"><b><i>a</i></b><sup><i>l</i></sup>=<i>δ</i> (<b><i>Z</i></b><sup><i>i</i></sup>) =<i>δ</i> (<b><i>w</i></b><sup><i>l</i></sup><b><i>a</i></b><sup><i>l</i>-1</sup>+<b><i>b</i></b><sup><i>l</i></sup>)      (1) </p>
                </div>
                <div class="p1">
                    <p id="96">式中:<i>l</i>=1, 2, …, <i>N</i>;<i>a</i><sup><i>l</i>-1</sup>是第<i>l</i>层的输入数据;<i>w</i><sup><i>l</i></sup>是第 (<i>l</i>-1) 层和第<i>l</i>层之间的连接权重矩阵;<b><i>b</i></b><sup><i>l</i></sup>是第<i>l</i>层的偏置, <i>δ</i>表示第<i>l</i>层的激活函数。</p>
                </div>
                <div class="p1">
                    <p id="97">目前, 在DNN中, 通常使用ReLU (Rectified linear unit) 作为神经元的激活函数。如式 (2) 所示, ReLU具有单侧抑制特性, 把所有的负值都变为0, 而正值不变。这种单侧抑制会使神经网络中的神经元具有稀疏激活性, 实现稀疏后的模型能够更好地挖掘相关特征和拟合训练数据。</p>
                </div>
                <div class="p1">
                    <p id="98"><i>δ</i> (<i>z</i>) =max (0, <i>z</i>)      (2) </p>
                </div>
                <h3 id="99" name="99" class="anchor-tag"><b>2 模型构建</b></h3>
                <h4 class="anchor-tag" id="100" name="100"><b>2.1 数据集</b></h4>
                <div class="p1">
                    <p id="101">本实验采用由Pan等<citation id="279" type="reference"><link href="237" rel="bibliography" /><sup>[<a class="sup">30</a>]</sup></citation>发布于http://www.csbio.sjtu.edu.cn/bioinf/LR_PPI/Data.htm的人类蛋白序列对数据集。该数据集包含36 630条阳性样本 (有互作关系蛋白质序列对) 和36 480条阴性样本 (无互作关系蛋白质序列对) 。其中:阳性样本取自人类蛋白质参考数据库 (HPRD) 2007版;阴性样本取自瑞士Swiss-Prot数据库57.3版。</p>
                </div>
                <div class="p1">
                    <p id="102">实验侧重20种常见氨基酸组成的蛋白质序列, 并且蛋白质序列编码方法要求蛋白质序列的长度不易太短。所以, 实验过程中除去蛋白质序列长度少于50及含有B、J、O、U、X、Z的蛋白质序列对, 得到36 591对阳性样本和36 324对阴性样本, 分别从阳性样本和阴性样本随机选取30 000条蛋白质序列对组成训练集, 剩下的12 915条蛋白质序列对作为测试集。</p>
                </div>
                <h4 class="anchor-tag" id="103" name="103"><b>2.2 性能评价指标</b></h4>
                <div class="p1">
                    <p id="104">实验采用准确率 (Accuracy) 、召回率 (Recall) 、损失率 (Loss) 和受试者工作特征曲线下面积AUC四个指标来评价模型性能。其中准确率和召回率计算公式如下:</p>
                </div>
                <div class="p1">
                    <p id="105"><mathml id="106"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>A</mi><msub><mrow></mrow><mrow><mtext>c</mtext><mtext>c</mtext><mtext>u</mtext><mtext>r</mtext><mtext>a</mtext><mtext>c</mtext><mtext>y</mtext></mrow></msub><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>Τ</mi><mi>Ν</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>Τ</mi><mi>Ν</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac></mrow></math></mathml>      (3) </p>
                </div>
                <div class="p1">
                    <p id="107"><mathml id="108"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>R</mi><msub><mrow></mrow><mrow><mtext>e</mtext><mtext>c</mtext><mtext>a</mtext><mtext>l</mtext><mtext>l</mtext></mrow></msub><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac></mrow></math></mathml>      (4) </p>
                </div>
                <div class="p1">
                    <p id="109">式中:<i>TP</i>、<i>TN</i>、<i>FP</i>和<i>FN</i>分别代表真正、真负、假正和假负。AUC通过开源代码计算<citation id="280" type="reference"><link href="251" rel="bibliography" /><sup>[<a class="sup">37</a>]</sup></citation>。损失率是用来衡量模型的实际输出与期望输出的距离, 损失函数越小, 表示两个概率分布就越接近, 模型的拟合性和鲁棒性就越好。损失率通过交叉熵函数计算公式如下:</p>
                </div>
                <div class="p1">
                    <p id="110" class="code-formula">
                        <mathml id="110"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>l</mi><mi>o</mi><mi>s</mi><mi>s</mi><mo stretchy="false"> (</mo><mi>y</mi><mo>, </mo><mi>y</mi><msup><mrow></mrow><mo>*</mo></msup><mo stretchy="false">) </mo><mo>=</mo><mo>-</mo><mfrac><mn>1</mn><mi>n</mi></mfrac><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mo stretchy="false"> (</mo></mstyle><mi>y</mi><msubsup><mrow></mrow><mi>i</mi><mo>*</mo></msubsup><mrow><mi>ln</mi></mrow><mi>y</mi><msub><mrow></mrow><mi>i</mi></msub><mo>+</mo><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>y</mi><msubsup><mrow></mrow><mi>i</mi><mo>*</mo></msubsup><mo stretchy="false">) </mo><mrow><mi>ln</mi></mrow><mo stretchy="false"> (</mo><mn>1</mn><mo>-</mo><mi>y</mi><msub><mrow></mrow><mi>i</mi></msub><mo stretchy="false">) </mo><mo stretchy="false">) </mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>5</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="111">式中:<i>y</i>= (<i>y</i><sub>1</sub>, <i>y</i><sub>2</sub>, …, <i>y</i><sub><i>n</i></sub>) , 为实际输出;<i>y</i><sup>*</sup>= (<i>y</i><sup>*</sup><sub>1</sub>, <i>y</i><sup>*</sup><sub>2</sub>, …, <i>y</i><sup>*</sup><sub><i>n</i></sub>) 为期望输出。</p>
                </div>
                <h4 class="anchor-tag" id="112" name="112"><b>2.3 构建流程</b></h4>
                <div class="p1">
                    <p id="113">DPPI模型是基于Tensorflow平台在Python环境下构建的, 其构建流程如图3所示。主要包括以下几步:首先, 使用LD分别对有关序列对和无关序列对的氨基酸序列进行编码, 生成有关序列集和无关序列集;其次, 使用随机选择的60 000条训练集数据对模型进行训练, 生成DPPI模型;接着, 使用剩下的12 915条测试集数据对DPPI模型进行测试;最后, 对DPPI模型的预测性能进行评价, 并根据评价结果调整参数, 优化DPPI模型。</p>
                </div>
                <div class="area_img" id="114">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201904045_114.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 DPPI模型构建流程图" src="Detail/GetImg?filename=images/JYRJ201904045_114.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 DPPI模型构建流程图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201904045_114.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="115" name="115"><b>2.4 参数调整</b></h4>
                <div class="p1">
                    <p id="116">参数调整是模型训练过程中很重要的一步, 是训练出健壮模型的关键要素之一。实验中, 激活函数使用ReLU、优化器选择Adam、代价函数使用交叉熵。在优化器方面, 目前已经开发了诸如RMSprop, Adagrad<citation id="281" type="reference"><link href="231" rel="bibliography" /><sup>[<a class="sup">27</a>]</sup></citation>和Adam<citation id="282" type="reference"><link href="233" rel="bibliography" /><sup>[<a class="sup">28</a>]</sup></citation>等优化方法, 其中, Adam集合了RMSprop和Adagrad两个算法的优点, 能够较好地处理噪声样本。交叉熵代价函数是用来衡量深度神经网络的预测值与实际值的一种方式, 可以弥补sigmoid型函数的导数形式易发生饱和的缺陷, 使训练更快收敛。</p>
                </div>
                <div class="p1">
                    <p id="117">学习率决定了权值更新的速度, 设置得太大易越过最优值, 出现振荡现象;太小会使下降速度过慢, 长时间无法收敛。因此, 学习率直接决定着算法的性能表现。Bengio<citation id="283" type="reference"><link href="235" rel="bibliography" /><sup>[<a class="sup">29</a>]</sup></citation>认为一般常用的学习率有0.000 01、0.000 1、0.001、0.01、0.1, 推荐使用的是0.01, 同时Bengio也指出, 学习率的选择要根据数据集的大小、特征提取方法等实际情况来确定。实验中, 设定隐含层节点数为64、激活函数为ReLU、优化算法为Adam、批处理大小为128、迭代次数为300 000次, 调整学习率的结果如表2所示。可以看出, 学习率为0.001时的准确率最高、平均损失最小。</p>
                </div>
                <div class="area_img" id="118">
                    <p class="img_tit"><b>表2 DPPI模型学习率的调整</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="118" border="1"><tr><td><br />学习率</td><td>准确率/%</td><td>平均损失/%</td></tr><tr><td><br />0.01</td><td>83.23±2.05</td><td>37.87±3.49</td></tr><tr><td><br />0.001</td><td>92.23±3.19</td><td>21.75±5.42</td></tr><tr><td><br />0.000 1</td><td>89.55±5.51</td><td>26.17±9.82</td></tr><tr><td><br />0.000 01</td><td>78.05±4.75</td><td>46.39±6.61</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="119">为了确定模型的宽度, 设定激活函数为ReLU、优化算法为Adam、学习率为0.001、批处理大小为128、迭代次数为300 000次, 调整模型宽度的结果如表3所示。可以看出, 宽度为512时, 模型的准确率最高, 同时训练时间比宽度为256时增加了近一倍, 而准确率、平均损失分别比宽度为256时仅仅提高了0.003 1、0.001 3, 考虑到时间复杂度, 本模型宽度选择256。</p>
                </div>
                <div class="area_img" id="120">
                    <p class="img_tit"><b>表3 DPPI模型宽度的调整</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="120" border="1"><tr><td><br />宽度</td><td>准确率/%</td><td>平均损失/%</td><td>100步训练时间/s</td></tr><tr><td><br />64</td><td>92.23±3.19</td><td>21.75±5.42</td><td>0.629 2±0.102 4</td></tr><tr><td><br />128</td><td>93.90±2.58</td><td>19.14±4.49</td><td>0.847 7±0.106 7</td></tr><tr><td><br />256</td><td>94.52±2.48</td><td>19.64±4.36</td><td>1.223 0±0.109 8</td></tr><tr><td><br />512</td><td>94.83±2.06</td><td>19.51±3.57</td><td>2.282 8±0.997 8</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="121">模型的宽度、激活函数、优化算法、学习率等确定之后, 本文通过调整隐含层层数来确定模型的深度, 调整深度的结果如表4所示。根据表4, 可知网络深度为[256-128-64-32]时的准确率较高、平均损失较低, 训练时间较短。</p>
                </div>
                <div class="area_img" id="122">
                    <p class="img_tit"><b>表4 DPPI模型深度的调整</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="122" border="1"><tr><td>深度</td><td>准确率/%</td><td>平均损失/%</td><td>100步训练时间/s</td></tr><tr><td><br />256-128-<br />64-32</td><td>95.74±1.05</td><td>22.67±4.18</td><td>1.397±0.120 3</td></tr><tr><td><br />256-128-64</td><td>95.46±1.39</td><td>24.36±4.55</td><td>1.398 7±0.121 5</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="123">丢弃率是DNN中防止过拟合、提高性能的一个很重要的参数。为了优化DPPI模型, 本文通过调整丢弃率得到7个预测模型, 各预测模型的最优结果如表5所示。从表5可知, 丢弃率为0.025时准确率最高, 达到96.81%, 平均损失为15.72%;丢弃率为0.05时, 准确率比丢弃率为0.025时的准确率降低了0.08%, 同时平均损失降低了2.52%;不使用丢弃率时, 准确率、AUC、Recall、平均损失分别为95.84%、97.44%、98.25%、45.38%。虽然不使用丢弃率时准确率和使用丢弃率时准确率差别不大, 但不使用丢弃率的平均损失较高, 所以不推荐使用。由此, DPPI-2和DPPI-3的性能较好, 可以作为DPPI的最终预测模型。</p>
                </div>
                <div class="area_img" id="124">
                    <p class="img_tit"><b>表5 DPPI模型的最优预测性能</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="124" border="1"><tr><td>模型</td><td>准确率<br />/%</td><td>AUC<br />/%</td><td>召回率<br />/%</td><td>平均<br />损失/%</td><td>训练迭代<br />次数</td><td>丢弃率<br />/%</td></tr><tr><td><br />DPPI- 1</td><td>95.84</td><td>97.44</td><td>98.25</td><td>45.38</td><td>570 000</td><td>N/A</td></tr><tr><td><br />DPPI- 2</td><td>96.81</td><td>98.74</td><td>99.06</td><td>15.72</td><td>600 000</td><td>2.5</td></tr><tr><td><br />DPPI- 3</td><td>96.73</td><td>99.00</td><td>99.21</td><td>13.20</td><td>440 000</td><td>5.0</td></tr><tr><td><br />DPPI- 4</td><td>96.41</td><td>98.92</td><td>99.03</td><td>15.26</td><td>560 000</td><td>7.5</td></tr><tr><td><br />DPPI- 5</td><td>93.45</td><td>97.83</td><td>98.17</td><td>18.02</td><td>380 000</td><td>10</td></tr><tr><td><br />DPPI- 6</td><td>82.61</td><td>91.18</td><td>92.04</td><td>39.92</td><td>30 000</td><td>30</td></tr><tr><td><br />DPPI- 7</td><td>81.81</td><td>90.84</td><td>91.78</td><td>44.80</td><td>30 000</td><td>50</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="125">经过超参数调整后, 本文构建了一个包含4个隐藏层, 各隐藏层节点数分别为256、128、64、32的DPPI模型。经过大量实验和调试, 总结了本实验采用的参数, 如表6所示。</p>
                </div>
                <div class="area_img" id="126">
                    <p class="img_tit"><b>表6 DPPI模型参数表</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="126" border="1"><tr><td><br />参数名称</td><td>范围</td><td>参数取值</td></tr><tr><td><br />深度</td><td>2, 3, 4, 5, 6</td><td>4</td></tr><tr><td><br />宽度</td><td>128, 256, 1 024, 2 048</td><td>256-128-64-32</td></tr><tr><td><br />激活函数</td><td>ReLU, tanh, sigmoid, softmax, ELU</td><td>ReLU</td></tr><tr><td><br />优化算法</td><td>Adadelta, RMSprop, Adam</td><td>Adam</td></tr><tr><td><br />学习率</td><td>0.01, 0.001, 0.000 1, 0.000 01</td><td>0.001</td></tr><tr><td><br />代价函数</td><td>cross-entropy</td><td>cross-entropy</td></tr><tr><td><br />丢弃率</td><td>0.025, 0.05, 0.075, 0.1, 0.3, 0.5</td><td>0.025, 0.05</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h3 id="127" name="127" class="anchor-tag"><b>3 实 验</b></h3>
                <h4 class="anchor-tag" id="128" name="128"><b>3.1 DPPI模型实验</b></h4>
                <div class="p1">
                    <p id="129">参照表6的DPPI模型参数, 根据使用和不使用丢弃率以及不同的丢弃率值, 使用7个不同的DPPI模型进行计算。每个模型各迭代60万次, 每1万次输出一个测试结果, 各输出60个实验结果。所有实验结果的统计值见表7。可以看到, DPPI-2的准确率最高, DPPI-1和DPPI-3稍微次之。DPPI-3的AUC、召回率和平均损失这三个指标比其他6组表现更优异。特别地, DPPI-3的平均损失 (15.34%) 比DPPI-2的平均损失 (16.99%) 降低了1.65%。</p>
                </div>
                <div class="area_img" id="130">
                    <p class="img_tit"><b>表7 DPPI模型预测性能平均值</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="130" border="1"><tr><td><br />模型</td><td>准确率<br />/%</td><td>AUC<br />/%</td><td>召回率<br />/%</td><td>平均损失<br />/%</td><td>丢弃率<br />/%</td></tr><tr><td><br />DPPI- 1</td><td>95.30±<br />0.87</td><td>97.57±<br />0.43</td><td>98.28±<br />0.30</td><td>36.23±<br />9.24</td><td>N/A</td></tr><tr><td><br />DPPI- 2</td><td>95.77±<br />1.30</td><td>98.50±<br />0.490</td><td>98.87±<br />0.46</td><td>16.99±<br />2.34</td><td>2.5</td></tr><tr><td><br />DPPI- 3</td><td>95.60±<br />1.67</td><td>98.65±<br />0.71</td><td>98.89±<br />0.65</td><td>15.34±<br />2.98</td><td>5.0</td></tr><tr><td><br />DPPI- 4</td><td>94.16±<br />1.96</td><td>98.13±<br />0.88</td><td>98.37±<br />0.78</td><td>17.79±<br />3.37</td><td>7.5</td></tr><tr><td><br />DPPI- 5</td><td>90.20±<br />2.74</td><td>96.53±<br />1.35</td><td>97.01±<br />1.17</td><td>24.28±<br />4.73</td><td>10</td></tr><tr><td><br />DPPI- 6</td><td>74.06±<br />4.59</td><td>94.07±<br />1.67</td><td>94.89±<br />1.51</td><td>53.81±<br />11.49</td><td>30</td></tr><tr><td><br />DPPI- 7</td><td>60.58±<br />8.71</td><td>93.92±<br />2.14</td><td>94.74±<br />1.86</td><td>76.62±<br />19.31</td><td>50</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="131">图4为7个DPPI预测模型不同评价指标的趋势图。其中:1代表DPPI-1预测指标趋势图;2代表DPPI-2预测指标趋势图;3代表DPPI-3预测指标趋势图;4代表DPPI-4预测指标趋势图;5代表DPPI-5预测指标趋势图;6代表DPPI-6预测指标趋势图;7代表DPPI-7预测指标趋势图。从图4 (a) 中可以看出, 模型DPPI-6和DPPI-7随着迭代次数的增加, 准确率呈下降趋势。DPPI-5的准确率虽然比DPPI-6和DPPI-3好, 但DPPI-5准确率不稳定, 振荡幅度稍大。从准确率来看, 模型DPPI-1、DPPI-2、DPPI-3和DPPI-4准确率后期都比较稳定, DPPI-2的准确率最好。 (b) 是7个DPPI模型AUC趋势图, 可以看出, 模型DPPI-2、DPPI-3和DPPI-4后期结果比较平稳, 虽然DPPI-1的结果开始较好, 但后期较差。 (c) 是7个模型的召回率趋势图, DPPI-3的召回率性能较好, DPPI-5、DPPI-6和DPPI-7召回率较低, 不平稳且振荡幅度较大。 (d) 是7个模型平均损失的趋势图, 趋势图显示模型DPPI-2、DPPI-3和DPPI-4结果较好, 但DPPI-2振荡幅度比DPPI-3稍大。</p>
                </div>
                <div class="area_img" id="135">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201904045_13500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 DPPI模型各个指标预测趋势图" src="Detail/GetImg?filename=images/JYRJ201904045_13500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 DPPI模型各个指标预测趋势图  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201904045_13500.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="136">综合表7和图4得到最终的预测模型DPPI-3, 其准确率为95.6%, 平均损失为15.34%。</p>
                </div>
                <h4 class="anchor-tag" id="137" name="137"><b>3.2 方法比较</b></h4>
                <div class="p1">
                    <p id="138">近几年, 已经有许多研究者对人蛋白互作预测提出了不同的计算方法。这些人蛋白互作预测方法以及DPPI的性能比较结果见表8。可以看出, 所列方法获得的精度均在83.90%和97.19%之间, 同时, 除了Sun<citation id="284" type="reference"><link href="245" rel="bibliography" /><sup>[<a class="sup">34</a>]</sup></citation>使用SAE+AC方法获得97.19%的准确率以外, DPPI模型获得了最好的准确率。和Sun的结果相比, 虽然DPPI的准确率不算突出, 但是Sun使用SAE+CT的准确率没有DPPI模型的准确率高。从表9可见, 在酒酿酵母数据集上, DPPI模型和其他采用LD编码的方法比较, 也取得最高准确率。</p>
                </div>
                <div class="area_img" id="139">
                    <p class="img_tit"><b>表8 不同方法的预测性能的比较</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="139" border="1"><tr><td>参照</td><td>方法</td><td>准确率</td><td>数据集划分方式</td></tr><tr><td><br />Shen等研究结果<sup>[30]</sup></td><td>SVM</td><td>83.90%</td><td>Holdout</td></tr><tr><td><br />You等研究结果<sup>[31]</sup></td><td>ELM</td><td>84.80%</td><td>Holdout</td></tr><tr><td><br />Guo等研究结果<sup>[32]</sup></td><td>SVM</td><td>90.67%</td><td>Holdout</td></tr><tr><td><br />Du等研究结果<sup>[15]</sup></td><td>DNN</td><td>89.00%</td><td>Holdout</td></tr><tr><td><br />Pan等研究结果<sup>[33]</sup></td><td>PseAAC<br />+SVM</td><td>91.20%</td><td>五倍交叉</td></tr><tr><td><br />Sun等研究结果<sup>[34]</sup></td><td>SAE+CT</td><td>94.52%</td><td>十倍交叉</td></tr><tr><td><br />Sun等研究结果<sup>[34]</sup></td><td>SAE+AC</td><td>97.19%</td><td>十倍交叉</td></tr><tr><td><br />DPPI</td><td>DNN+LD</td><td>95.60%</td><td>Holdout</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="area_img" id="140">
                    <p class="img_tit"><b>表9 不同算法采用LD编码方式结果比较</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="140" border="1"><tr><td><br />参照</td><td>方法</td><td>数据集</td><td>准确率</td></tr><tr><td><br />You等研究结果<sup>[35]</sup></td><td>ELM+LD</td><td>酒酿酵母</td><td>83.90%</td></tr><tr><td><br />Zhou等研究结果<sup>[36]</sup></td><td>SVM+LD</td><td>酒酿酵母</td><td>88.76%</td></tr><tr><td><br />Yang等研究结果<sup>[37]</sup></td><td>kNN+LD</td><td>酒酿酵母</td><td>84.81%</td></tr><tr><td><br />Wang等研究结果<sup>[38]</sup></td><td>DNN+LD</td><td>酒酿酵母</td><td>90.19%</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="141">SAE+AC结果优于DPPI的原因可能在于特征提取方法的不同。AC编码是通过选择物理化学性质, 解释了氨基酸与序列中相隔一定数量的氨基酸之间的相互作用, 该方法考虑了最长序列30 bp的邻近效应<citation id="285" type="reference"><link href="241" rel="bibliography" /><sup>[<a class="sup">32</a>]</sup></citation>。LD为了更好地从蛋白质的氨基酸片段中捕捉蛋白质相互作用信息, 将一条蛋白质序列划分为10个局部区域, 这样分组, 局部信息突出不明显, 致使丢失某些关键信息<citation id="286" type="reference"><link href="251" rel="bibliography" /><sup>[<a class="sup">37</a>]</sup></citation>。LD的这种缺陷在以后的研究中, 可以通过增加局部区域的划分等方法来减少特征信息的丢失。</p>
                </div>
                <div class="p1">
                    <p id="142">虽然DPPI模型准确率和SAE+AC方法相比不算突出, 但在蛋白互作预测方面也取得了良好的结果, 且DNN去噪能力优于SAE, 代码也比SAE简洁, LD编码简单、速度快。特别地, 由表10可知, LD和AC编码72 915对人蛋白质的时间可见在相同软硬件计算环境下LD编码速度比AC编码速度快3.5倍以上。通过上面的比较可知, 本文的DPPI模型与其他方法相比可以显著提高大规模蛋白互作预测性能。</p>
                </div>
                <div class="area_img" id="143">
                    <p class="img_tit"><b>表10 AC和LD编码时间的比较</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="143" border="1"><tr><td><br />编码方式</td><td>编码时间/s</td><td>数据集</td></tr><tr><td><br />AC</td><td>0.028 65</td><td>HPRD (36591) +Swiss-Port (36324) </td></tr><tr><td><br />LD</td><td>0.007 97</td><td>HPRD (36591) +Swiss-Port (36324) </td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h3 id="144" name="144" class="anchor-tag"><b>4 结 语</b></h3>
                <div class="p1">
                    <p id="145">深度学习算法已经涉足许多领域, 但是在蛋白互作的研究中还没有被广泛的应用。因此, 本文采用深度神经网络DNN和LD蛋白质序列编码方法相结合的方法构建了蛋白互作预测模型DPPI。DPPI模型获得准确率96.73%、AUC 99.00%、召回率99.21%和平均损失13.2%的最优性能, 以及95.60%准确率、98.65%AUC、98.89%召回率和15.34%平均损失的平均性能。和其他研究者提出的人蛋白互作预测方法比较, DPPI模型的准确率优于Shen、You、Guo、Du、Pan等结果, 但DPPI的结果没有Sun采用的SAE+AC方法预测结果性能好。LD的这种缺陷在以后的研究中, 可以通过增加局部区域等方法来减少特征信息的丢失。</p>
                </div>
                <div class="p1">
                    <p id="146">本文首次采用DNN结合LD对人蛋白互作数据集构建的, 用于蛋白互作预测的模型DPPI。该模型具有较强的去噪能力、编码简单、代码简洁、计算速度快、运行时间段等优点, 可以通过分层抽象学习蛋白质对的有用特征, 从数据中自动学习内部分布式特征表示。鉴于以上优点, DPPI模型可以作为蛋白互作预测的有益补充。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="179">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A novel genetic system to detect protein-protein interactions">

                                <b>[1]</b> Fields S, Song O. A Novel Genetic System to Detect Protein Protein Interactions[J]. Nature, 1989, 340 (6230) :245-246.
                            </a>
                        </p>
                        <p id="181">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry">

                                <b>[2]</b> Ho Y, Gruhler A, Herlbut A, et al. Systematic Identification of Protein Complexes in Saccharomyces Cerevisiae by Mass Spectrometry[J]. Nature, 2002, 415 (6868) :180-183.
                            </a>
                        </p>
                        <p id="183">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Functional organization of the yeast proteome by systematic analysis of protein complexes">

                                <b>[3]</b> Gavin A C, Bosche M, Krause R, et al. Functional Organization of the Yeast Proteome by Systematic Analysis of Protein Complexes[J]. Nature, 2002, 415 (6868) : 141-147.
                            </a>
                        </p>
                        <p id="185">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Global Analysis of Protein Activities Using Proteome Chips">

                                <b>[4]</b> Heng Z, Metin B, Rhonda B, et al. Global Analysis of Protein Activities Using Proteome Chips[J]. Science, 2001, 293 (5537) :2101-2105.
                            </a>
                        </p>
                        <p id="187">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=PPI_SVM: Prediction of protein-protein interactions using machine learning, domain-domain affinities and frequency tables">

                                <b>[5]</b> Chatterjee P, Basu S, Kundu M, et al. PPI_SVM: Prediction of protein-protein interactions using machine learning, domain-domain affinities and frequency tables[J]. Cellular &amp; Molecular Biology Letters, 2011, 16 (2) :264-278.
                            </a>
                        </p>
                        <p id="189">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prediction of protein--protein interaction sites in heterocomplexes with neural networks">

                                <b>[6]</b> Fariselli P, Pazos F, Valencia A, et al. Prediction of protein—protein interaction sites in heterocomplexes with neural networks[J]. European Journal of Biochemistry, 2002, 269 (5) : 1356-1361.
                            </a>
                        </p>
                        <p id="191">
                            <a id="bibliography_7" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJWD&amp;filename=SJWD15121501608600&amp;v=MDkxNjhVcnpJSkZzVWF4cz1OaWZjYXJLOUg5UE5xbzlFWXVzSENudzVvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp0RmlubA==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[7]</b> Lin X, Chen X W. Heterogeneous Data Integration by Tree Augmented Naive Bayes for Protein-Protein Interactions Prediction[J]. Proteomics, 2013, 13 (2) : 261-268.
                            </a>
                        </p>
                        <p id="193">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Supervised Statistical and Machine Learning Approaches to Inferring Pairwise and Module Based Protein Interaction Networks">

                                <b>[8]</b> Browne F, Wang H, Zheng H, et al. Supervised Statistical and Machine Learning Approaches to Inferring Pairwise and Module Based Protein Interaction Networks[C]//Proceedings of the 7th IEEE International Conference on Bioinformatics and Bioengineering, New York, USA: IEEE, 2007: 1365-1369.
                            </a>
                        </p>
                        <p id="195">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The development of a universal in silico predictor of protein-protein interactions">

                                <b>[9]</b> Valente G T, Acencio M L, Martins C, et al. The Development of a Universal in Silico Predictor of Protein-Protein Interactions[J]. PLoS One, 2013, 8 (5) : e65587.
                            </a>
                        </p>
                        <p id="197">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prediction of protein?protein interactions using random decision forest framework">

                                <b>[10]</b> Chen X W, Liu M. Prediction of Protein-Protein Interactions Using Random Decision Forest Framework[J]. Bioinformatics, 2005, 21 (24) : 4394-4400.
                            </a>
                        </p>
                        <p id="199">
                            <a id="bibliography_11" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Ensemble learning prediction of protein-protein interactions using proteins functional annotations">

                                <b>[11]</b> Saha I, Zubek J, Klingström T, et al. Ensemble Learning Prediction of Protein-Protein Interactions Using Proteins Functional Annotations[J]. Molecular Biosystems, 2014, 10 (4) : 820-830.
                            </a>
                        </p>
                        <p id="201">
                            <a id="bibliography_12" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Using deep learning for image-based plant disease detection">

                                <b>[12]</b> Mohanty S P, Hughes D P, Salathé M. Using Deep Learning for Image-Based Plant Disease Detection[J]. Frontiers in Plant Science, 2016, 7:1419.
                            </a>
                        </p>
                        <p id="203">
                            <a id="bibliography_13" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A Unified Architecture for Natural Language Processing:Deep Neural Networks with Multitask Learning">

                                <b>[13]</b> Collobert G, Weston J. A Unified Architecture for Natural Language Processing: Deep Neural Networks with Multitask Learning[C]//International Conference on Machine Learning, 2008:160-167.
                            </a>
                        </p>
                        <p id="205">
                            <a id="bibliography_14" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Mastering the game of Go with deep neural networks and tree search">

                                <b>[14]</b> Silver D, Huang A, Maddison C J, et al. Mastering the Game of Go with Deep Neural Networks and Tree Search[J]. Nature, 2016, 529 (7587) :484-489.
                            </a>
                        </p>
                        <p id="207">
                            <a id="bibliography_15" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Deep PPI:Boosting prediction of protein-protein interactions with deep neural networks">

                                <b>[15]</b> Du X Q, Sun S W, Hu CL, et al. DeepPPI: Boosting Prediction of Protein-Protein Interactions with Deep Neural Networks[J]. Journal of Chemical Information &amp; Modeling, 2017, 57 (6) :1499-1510.
                            </a>
                        </p>
                        <p id="209">
                            <a id="bibliography_16" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Deep learning of the tissue-regulated splicing code">

                                <b>[16]</b> Leung M K, Xiong H Y, Lee L J, et al. Deep Learning of the Tissue-Regulated Splicing Code[J]. Bioinformatics, 2014, 30 (12) : i121-i129.
                            </a>
                        </p>
                        <p id="211">
                            <a id="bibliography_17" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Predicting effects of noncoding variants with deep learning-based sequence model">

                                <b>[17]</b> Zhou J, Troyanskaya O G. Predicting Effects of Noncoding Variants with Deep Learning-Based Sequence Model[J]. Nature Methods, 2015, 12 (10) : 931-934.
                            </a>
                        </p>
                        <p id="213">
                            <a id="bibliography_18" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Representation learning:A review and new perspectives">

                                <b>[18]</b> Bengio Y, Courville A, Vincent P. Representation Learning: A Review and New Perspectives[J]. IEEE Transactions on Pattern Analysis &amp; Machine Intelligence, 2012, 35 (8) :1798-1828.
                            </a>
                        </p>
                        <p id="215">
                            <a id="bibliography_19" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Predicting the sequence specificities of DNA-and RNA-binding proteins by deep learning">

                                <b>[19]</b> Alipanahi B, Delong A, Weirauch M T, et al. Predicting the sequence specificities of DNA—and RNA-binding proteins by deep learning[J]. Nature Biotechnology, 2015, 33 (8) : 831-838.
                            </a>
                        </p>
                        <p id="217">
                            <a id="bibliography_20" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Image Net classification with deep convolutional neural networks">

                                <b>[20]</b> Krizhevsky A, Sutskever I, Hinton G E. Imagenet Classification with Deep Convolutional Neural Networks[J]. International Conference on Neural Information Processing Systems, 2012, 60 (2) : 1097-1105.
                            </a>
                        </p>
                        <p id="219">
                            <a id="bibliography_21" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES56B53E38E4B3FBB22783327E47E04993&amp;v=MDg5OTVHUWxmQnJMVTA1dHRod0wyL3dhRT1OaWZPZmJhK2JOVFAyb3hORWU5OUR3cEx2UlFSN1RkK1MzM2wyUll5RExLUVRMT2NDT052RlNpV1dyN0pJRnBtYUJ1SFlmTw==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[21]</b> Tian K, Shao M Y, Wang Y, et al. Boosting Compound-Protein Interaction Prediction by Deep Learning[J]. Methods, 2016, 110: 64-72.
                            </a>
                        </p>
                        <p id="221">
                            <a id="bibliography_22" >
                                    <b>[22]</b>
                                 Yang L, Xia J F, Gui F. Prediction of Protein-Protein Interactions from Protein Sequence Using Local Descriptors[J]. Protein &amp; Peptide Letters, 2010, 17 (19) :1085-1090.
                            </a>
                        </p>
                        <p id="223">
                            <a id="bibliography_23" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Optimizing amino acid groupings for GPCR classification">

                                <b>[23]</b> Davies M N, Secker A, Freitas A A, et al. Optimizing Amino Acid Groupings for GPCR Classification[J]. Bioinformatics, 2008, 24 (18) :1980-1986.
                            </a>
                        </p>
                        <p id="225">
                            <a id="bibliography_24" >
                                    <b>[24]</b>
                                 Hinton G E, Salakhutdinov R R. Reducing the Dimensionality of Data with Neural Networks[J]. Science, 2006, 313 (5786) : 504-507.
                            </a>
                        </p>
                        <p id="227">
                            <a id="bibliography_25" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJBK&amp;filename=SJBK15090500012538&amp;v=MTI4NDJ6SUpGc1VheHM9TmlmSlpiSzlIdGpNcW85RlpPb05DWDh4b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadEZpbmxVcg==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[25]</b> Hinton G E, Osindero S, Teh Y W. A Fast Learning Algorithm for Deep Belief Nets[J]. Neural Computation, 2014, 18 (7) : 1527-1554.
                            </a>
                        </p>
                        <p id="229">
                            <a id="bibliography_26" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Deep sparse rectifier neural networks">

                                <b>[26]</b> Glorot X, Bordes A, Bengio Y. Deep Sparse Rectifier Neural Networks[C]//Proceedings of the 14th International Conference on Artificial Intelligence and Statistics (AISTATS) , 2010: 315-323.
                            </a>
                        </p>
                        <p id="231">
                            <a id="bibliography_27" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Dropout:A simple way to prevent neural networks from overfitting">

                                <b>[27]</b> Srivastava N, Hinton G, Krizhevsky A, et al. Dropout: A Simple Way to Prevent Neural Networks from Over Fitting. Journal of Machine Learning Research, 2014, 15 (1) : 1929-1958.
                            </a>
                        </p>
                        <p id="233">
                            <a id="bibliography_28" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Adam: A Method for Stochastic Optimization[DB]">

                                <b>[28]</b> Kingma D, Ba J. Adam: A Method for Stochastic Optimization[DB]. arXiv:1412.6980V8, 2014.
                            </a>
                        </p>
                        <p id="235">
                            <a id="bibliography_29" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Practical Recommendations for Gradient-Based Training of Deep Architectures">

                                <b>[29]</b> Bengio Y. Practical Recommendations for Gradient-Based Training of Deep Architectures[J]. Springer Berlin Heidelberg, 2012, 7700 (1-3) : 437-478.
                            </a>
                        </p>
                        <p id="237">
                            <a id="bibliography_30" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Predicting protein–protein interactions based only on sequences information">

                                <b>[30]</b> Shen J, Zhang J, Luo X, et al. Predicting protein-protein interactions based only on sequences information[J]. Proceedings of the National Academy of Sciences, 2007, 104 (11) :4337-4341.
                            </a>
                        </p>
                        <p id="239">
                            <a id="bibliography_31" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJHD&amp;filename=SJHD14090200000112&amp;v=MTAwNTJVcnpJSkZzVWF4cz1OaWZEYXJLOEh0ak1yWTlGWk9zUERYMDdvQk1UNlQ0UFFIL2lyUmRHZXJxUVRNbndaZVp0RmlubA==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[31]</b> You Z H, Li S, Gao X, et al. Large-Scale Protein-Protein Interactions Detection by Integrating Big Biosensing Data with Computational Model[J]. Biomed Research International, 2014, 2014 (2) :598129.
                            </a>
                        </p>
                        <p id="241">
                            <a id="bibliography_32" target="_blank" href="http://scholar.cnki.net/result.aspx?q=PRED_PPI: a server for predicting protein-protein interactions based on sequence data with probability assignment">

                                <b>[32]</b> Guo Y, Li M, Pu X, et al. PRED_PPI: A Server for Predicting Protein-Protein Interactions Based on Sequence Data with Probability Assignment[J]. Bmc Research Notes, 2010, 3 (1) : 145-152.
                            </a>
                        </p>
                        <p id="243">
                            <a id="bibliography_33" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Large-Scale Prediction of Human Protein-Protein Interactions from Amino Acid Sequence Based on Latent Topic Features">

                                <b>[33]</b> Pan X Y, Zhang Y N, Shen H B. Large-scale Prediction of Human Protein-Protein Interactions from Amino Acid Sequence Based on Latent Topic Features[J]. Journal of Proteome Research 2010, 9 (10) :4992-5001.
                            </a>
                        </p>
                        <p id="245">
                            <a id="bibliography_34" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJDD8E623236FB51C19591C69B5C2F3C3F0&amp;v=MTc4ODFuYVFxV0UzRDdIblJzeWZDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0dGh3TDIvd2FFPU5qN0Jhc2V3YTlmT3JJMUdZcDE5Q1gxS3poOFc0ejRPVA==&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[34]</b> Sun T, Zhou B, Lai H, et al. Sequence-Based Prediction of Protein Protein Interaction Using a Deep-Learning Algorithm[J]. BMC Bioinformatics, 2017, 18 (1) : 277-285.
                            </a>
                        </p>
                        <p id="247">
                            <a id="bibliography_35" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prediction\sof protein-protein interactions from amino acid sequences\swith ensemble extreme learning machines and principal component analysis">

                                <b>[35]</b> You Z H, Lei Y K, Zhu L, et al. Prediction of Protein-Protein Interactions from Amino Acid Sequences with Ensemble Extreme Learning Machines and Principal Component Analysis[J]. BMC Bioinformatics, 2013, 14 (S8) : S10.
                            </a>
                        </p>
                        <p id="249">
                            <a id="bibliography_36" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prediction of protein-protein interactions using local description of amino acid sequence">

                                <b>[36]</b> Zhou Y Z, Gao Y, Zheng Y Y. Prediction of Protein-Protein Interactions Using Local Description of Amino Acid Sequence[J]. Communications in Computer &amp; Information Science, 2011, 202:254-262.
                            </a>
                        </p>
                        <p id="251">
                            <a id="bibliography_37" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Prediction of protein-protein interactions from protein sequence using local descriptors">

                                <b>[37]</b> Yang L, Xia J F, Gui J. Prediction of Protein-Protein Interactions from Protein Sequence Using Local Descriptors[J]. Protein &amp; Peptide Letters, 2010, 17 (9) :1085-1090.
                            </a>
                        </p>
                        <p id="253">
                            <a id="bibliography_38" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Protein-Protein Interactions Prediction Using a Novel Local Conjoint Triad Descriptor of Amino Acid Sequences">

                                <b>[38]</b> Wang J, Zhang L, Jia L Y, et al. Protein-Protein Interactions Prediction Using a Novel Local Conjoint Triad Descriptor of Amino Acid Sequences[J]. International Journal of Molecular Sciences, 2017, 18 (11) :2373.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JYRJ201904045" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYRJ201904045&amp;v=MDIzMDdUNmo1NE8zenFxQnRHRnJDVVI3cWZadVp0Rnl6Z1U3N0JMelRaWkxHNEg5ak1xNDlCWVlRS0RIODR2UjQ=&amp;uid=WEEvREcwSlJHSldRa1FhcTdnTnhXM3ZKQ291S0FIT2hubWNrWXI0QkNHMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="1" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
