<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637136421704033750%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJYRJ201903052%26RESULT%3d1%26SIGN%3dIycDl96ACEDAbKpB70HkAjB%252f%252btA%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JYRJ201903052&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JYRJ201903052&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYRJ201903052&amp;v=MTg5NzNJOUFab1FLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnNGaURoVkw3UEx6VFpaTEc0SDlqTXI=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#23" data-title="&lt;b&gt;0 引 言&lt;/b&gt; "><b>0 引 言</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#28" data-title="&lt;b&gt;1 K-means聚类算法简介&lt;/b&gt; "><b>1 K-means聚类算法简介</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#34" data-title="&lt;b&gt;2 算法设计&lt;/b&gt; "><b>2 算法设计</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#35" data-title="&lt;b&gt;2.1 工作流程&lt;/b&gt;"><b>2.1 工作流程</b></a></li>
                                                <li><a href="#38" data-title="&lt;b&gt;2.2 使用堆砌算法生成候选变异集&lt;/b&gt;"><b>2.2 使用堆砌算法生成候选变异集</b></a></li>
                                                <li><a href="#42" data-title="&lt;b&gt;2.3 变异特征&lt;/b&gt;"><b>2.3 变异特征</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#54" data-title="&lt;b&gt;3 实验分析&lt;/b&gt; "><b>3 实验分析</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#55" data-title="&lt;b&gt;3.1 实验数据集&lt;/b&gt;"><b>3.1 实验数据集</b></a></li>
                                                <li><a href="#58" data-title="&lt;b&gt;3.2 评价标准&lt;/b&gt;"><b>3.2 评价标准</b></a></li>
                                                <li><a href="#67" data-title="&lt;b&gt;3.3 实验结果&lt;/b&gt;"><b>3.3 实验结果</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#77" data-title="&lt;b&gt;4 结 语&lt;/b&gt; "><b>4 结 语</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#37" data-title="图1 变异识别算法的工作流程">图1 变异识别算法的工作流程</a></li>
                                                <li><a href="#41" data-title="图2 堆叠的可视化展示">图2 堆叠的可视化展示</a></li>
                                                <li><a href="#51" data-title="&lt;b&gt;表1 Fisher精确检验的列联表&lt;/b&gt;"><b>表1 Fisher精确检验的列联表</b></a></li>
                                                <li><a href="#69" data-title="图3 模拟数据PR曲线">图3 模拟数据PR曲线</a></li>
                                                <li><a href="#71" data-title="图4 IlluminaB17数据算法正例与真实正例对比">图4 IlluminaB17数据算法正例与真实正例对比</a></li>
                                                <li><a href="#72" data-title="图5 IlluminaB17数据PR曲线">图5 IlluminaB17数据PR曲线</a></li>
                                                <li><a href="#73" data-title="&lt;b&gt;表2 IlluminaB17数据算法性能对比&lt;/b&gt;"><b>表2 IlluminaB17数据算法性能对比</b></a></li>
                                                <li><a href="#75" data-title="图6 肺癌数据上算法交集数量对比">图6 肺癌数据上算法交集数量对比</a></li>
                                                <li><a href="#76" data-title="&lt;b&gt;表3 肺癌数据算法2-cross数量对比&lt;/b&gt;"><b>表3 肺癌数据算法2-cross数量对比</b></a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="3">


                                    <a id="bibliography_1" title=" 孟倩.基于高通量测序的短序列生物数据压缩研究[J].计算机应用与软件, 2017, 34 (4) :22-27, 98." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYRJ201704005&amp;v=MTk1Mzh0R0ZyQ1VSN3FmWnVac0ZpRGhWTDdQTHpUWlpMRzRIOWJNcTQ5RllZUUtESDg0dlI0VDZqNTRPM3pxcUI=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[1]</b>
                                         孟倩.基于高通量测序的短序列生物数据压缩研究[J].计算机应用与软件, 2017, 34 (4) :22-27, 98.
                                    </a>
                                </li>
                                <li id="5">


                                    <a id="bibliography_2" title=" Ng P C, Kirkness E F. Whole genome sequencing[J]. Methods in Molecular Biology, 2010, 628:215-226." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Whole genome sequencing">
                                        <b>[2]</b>
                                         Ng P C, Kirkness E F. Whole genome sequencing[J]. Methods in Molecular Biology, 2010, 628:215-226.
                                    </a>
                                </li>
                                <li id="7">


                                    <a id="bibliography_3" title=" Hudson T J, Anderson W, Artez A, et al. International network of cancer genome projects[J]. Nature 2010, 464:993-998." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=International network of cancer genome projects">
                                        <b>[3]</b>
                                         Hudson T J, Anderson W, Artez A, et al. International network of cancer genome projects[J]. Nature 2010, 464:993-998.
                                    </a>
                                </li>
                                <li id="9">


                                    <a id="bibliography_4" title=" Prieto T , Alves J M , Posada D . NGS Analysis of Somatic Mutations in Cancer Genomes[M]//Big Data Analytics in Genomics. Springer International Publishing, 2016." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=NGS Analysis of Somatic Mutations in Cancer Genomes">
                                        <b>[4]</b>
                                         Prieto T , Alves J M , Posada D . NGS Analysis of Somatic Mutations in Cancer Genomes[M]//Big Data Analytics in Genomics. Springer International Publishing, 2016.
                                    </a>
                                </li>
                                <li id="11">


                                    <a id="bibliography_5" title=" Larson D E , Harris C C , Chen K , et al. SomaticSniper: identification of somatic point mutations in whole genome sequencing data[J].Bioinformatics, 2012, 28 (3) :311-317." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=SomaticSniper: identification of somatic point mutations in whole genome sequencing data">
                                        <b>[5]</b>
                                         Larson D E , Harris C C , Chen K , et al. SomaticSniper: identification of somatic point mutations in whole genome sequencing data[J].Bioinformatics, 2012, 28 (3) :311-317.
                                    </a>
                                </li>
                                <li id="13">


                                    <a id="bibliography_6" title=" Cibulskis K , Lawrence M S , Carter S L , et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples[J]. Nature Biotechnology, 2013, 31 (3) :213-219." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples">
                                        <b>[6]</b>
                                         Cibulskis K , Lawrence M S , Carter S L , et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples[J]. Nature Biotechnology, 2013, 31 (3) :213-219.
                                    </a>
                                </li>
                                <li id="15">


                                    <a id="bibliography_7" title=" 郭占元, 林涛.面向大规模数据快速聚类K-means算法的研究[J].计算机应用与软件, 2017, 34 (5) :43-7, 53." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYRJ201705008&amp;v=Mjk2NjlqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVac0ZpRGhWTDdQTHpUWlpMRzRIOWJNcW85RmJJUUtESDg0dlI0VDY=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[7]</b>
                                         郭占元, 林涛.面向大规模数据快速聚类K-means算法的研究[J].计算机应用与软件, 2017, 34 (5) :43-7, 53.
                                    </a>
                                </li>
                                <li id="17">


                                    <a id="bibliography_8" title=" Li H, Handsaker B, Wysoker A, et al.The Sequence Alignment/Map format and SAMtools[J] Bioinformatics, 2009, 25 (16) :2078-2079." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The Sequence Alignment/Map format and SAMtools">
                                        <b>[8]</b>
                                         Li H, Handsaker B, Wysoker A, et al.The Sequence Alignment/Map format and SAMtools[J] Bioinformatics, 2009, 25 (16) :2078-2079.
                                    </a>
                                </li>
                                <li id="19">


                                    <a id="bibliography_9" title=" Mckenna A , Hanna M , Banks E , et al. The Genome Analysis Toolkit : A MapReduce framework for analyzing next-generation DNA sequencing data[J].Genome Research, 2010, 20 (9) :1297-1303." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data">
                                        <b>[9]</b>
                                         Mckenna A , Hanna M , Banks E , et al. The Genome Analysis Toolkit : A MapReduce framework for analyzing next-generation DNA sequencing data[J].Genome Research, 2010, 20 (9) :1297-1303.
                                    </a>
                                </li>
                                <li id="21">


                                    <a id="bibliography_10" title=" Roth A , Ding J , Morin R , et al. JointSNVMix: a probabilistic model for accurate detection of somatic mutations in normal/tumour paired next-generation sequencing data[J]. Bioinformatics, 2012, 28 (7) :907-913." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=JointSNVMix: a probabilistic model for accurate detection of somatic mutations in normal/tumour paired next-generation sequencing data">
                                        <b>[10]</b>
                                         Roth A , Ding J , Morin R , et al. JointSNVMix: a probabilistic model for accurate detection of somatic mutations in normal/tumour paired next-generation sequencing data[J]. Bioinformatics, 2012, 28 (7) :907-913.
                                    </a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JYRJ" target="_blank">计算机应用与软件</a>
                2019,36(03),287-290+333 DOI:10.3969/j.issn.1000-386x.2019.03.051            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm">K-means聚类算法在肿瘤基因变异识别中的应用</span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E5%8F%B6%E9%AA%81&amp;code=41287812&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">叶骁</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E5%A4%8D%E6%97%A6%E5%A4%A7%E5%AD%A6%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%A7%91%E5%AD%A6%E6%8A%80%E6%9C%AF%E5%AD%A6%E9%99%A2%E6%99%BA%E8%83%BD%E4%BF%A1%E6%81%AF%E5%A4%84%E7%90%86%E9%87%8D%E7%82%B9%E5%AE%9E%E9%AA%8C%E5%AE%A4&amp;code=0075855&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">复旦大学计算机科学技术学院智能信息处理重点实验室</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>二代测序NGS (Next-generation sequencing) 数据的迅速发展加快人们对于基因的探索, 同时也给测序数据分析任务带来更大的挑战。癌细胞特异变异的识别是测序数据分析的一项重要基础性工作。当前的变异识别工具大多采用贝叶斯模型方法, 特异度、灵敏度和速度都远远满足不了需求。K-means是一种简洁高效的无监督聚类算法, 基于此将位点信息映射成多维的特征, 再进行类别个数为2的聚类过程。该算法明显提高了准确度和召回率, 实验结果验证了算法的有效性。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=K-means&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">K-means;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%8F%98%E5%BC%82%E8%AF%86%E5%88%AB&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">变异识别;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E4%BA%8C%E4%BB%A3%E6%B5%8B%E5%BA%8F&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">二代测序;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    叶骁, 硕士, 主研领域:生物信息。;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-09-26</p>

            </div>
                    <h1><b>USING K-MEANS CLUSTERING ALGORITHM FOR CANCER GENE VARIANT DETECTING</b></h1>
                    <h2>
                    <span>Ye Xiao</span>
            </h2>
                    <h2>
                    <span>Shanghai Key Lab of Intelligent Information Processing, School of Computer Science, Fudan University</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>The rapid development of next-generation sequencing data has accelerated the exploration of genes, and has also brought greater challenges to sequencing data analysis tasks. The identification of cancer-specific mutations is an important basic task in sequencing data analysis. Most of the current mutation identification tools use Bayesian model methods, but the specificity, sensitivity, and speed are far from meeting the needs. K-means is a concise and efficient unsupervised clustering algorithm. The algorithm mapped the site information into multidimensional features, and then carried out the clustering process with two classes. The algorithm improved the accuracy and recall rate obviously. Experimental results verify the effectiveness of the algorithm.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=K-means&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">K-means;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Variant%20calling&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Variant calling;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Next-generation%20sequencing&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Next-generation sequencing;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                                            </p>
                                    <p><b>Received：</b> 2018-09-26</p>
                            </div>


        <!--brief start-->
                        <h3 id="23" name="23" class="anchor-tag"><b>0 引 言</b></h3>
                <div class="p1">
                    <p id="24">二代测序技术 (NGS) , 又名高通量测序技术, 是一项通过随机打断人体DNA样本, 再进行短片段测序, 从而推测出样本基因的实验技术手段<citation id="80" type="reference"><link href="3" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>。由二代测序数据产生的读段再经过测序数据分析流程, 包括读段匹配、重复片段检测、读段重新匹配以及碱基质量修正等操作后进入下游的一系列基因突变分析, 如单核苷酸多态性 (SNP) 、结构变异 (SV) 、拷贝数变异 (CNV) 和其他变异分析<citation id="81" type="reference"><link href="5" rel="bibliography" /><sup>[<a class="sup">2</a>]</sup></citation>。</p>
                </div>
                <div class="p1">
                    <p id="25">之前的研究主要集中在单样本正常细胞的测序数据分析, 近年来, 随着二代测序技术的不断发展和完善, 检测成本和错误率不断下降, 人们把更多的目光投向了致癌基因研究这一领域, 如前身为癌症基因图谱 (The Cancer Genome Atlas) 的国际癌症基因协会 (International Cancer Genome Consortium) <citation id="82" type="reference"><link href="7" rel="bibliography" /><sup>[<a class="sup">3</a>]</sup></citation>。对于癌细胞基因突变的检测有利于人们更多地了解癌症基因型与表型之间的关联, 从而提高临床诊断的效率, 加速新型药物的研发。无论是系统性错误如二代测序技术本身测序片段长度短、检测碱基不精确、PCR扩增时引入重复序列, 还是随机错误如下游数据分析中存在多个匹配位置以及INEDL附近难以匹配精确等问题都使得突变检测困难重重。加之癌症样本不同于单样本, 其中存在着癌细胞样本纯度、癌细胞倍性及多种癌细胞杂合等问题<citation id="83" type="reference"><link href="9" rel="bibliography" /><sup>[<a class="sup">4</a>]</sup></citation>。癌细胞样本纯度表现为癌细胞群中混杂有正常细胞, 这使得基因型不再是传统的AA/BB/AB三类, 频率也不再是0/0.5/1这三个离散的数值。研究表明在极低纯度的癌细胞中, 变异等位基因的频率可能只有5%, 在深度较低的测序数据中将无法被观测到。癌细胞倍性是指癌细胞中有可能出现单倍体或者多倍, 从而引发杂合性丢失。多种癌细胞杂合指出现多种癌细胞的亚种, 亚种之间基因型存在差别等问题。上述三类问题使得各种用于正常细胞突变检测的工具在面对癌细胞测序数据时表现得不是很好。</p>
                </div>
                <div class="p1">
                    <p id="26">目前, 在癌细胞突变检测方面已经有不少用于检测SNVs和INDEL的工具。如比较主流的应用等位基因频率建立贝叶斯模型的Somatic-Sniper<citation id="84" type="reference"><link href="11" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>和计算各种基因型后验概率来得到某一位点最可能基因型进而判断是否发生变异的Mutect2<citation id="85" type="reference"><link href="13" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>。但这些模型普遍包含大量参数需要使用者调整, 因而对于各类情景适应性不好。</p>
                </div>
                <div class="p1">
                    <p id="27">本文旨在设计一种利用无监督聚类方法检测配对样本基因突变的算法, 摒弃以往算法中各类参数在不同场景中调节困难的缺点, 同时仍能达到较高的精确率和召回率。</p>
                </div>
                <h3 id="28" name="28" class="anchor-tag"><b>1 K-means聚类算法简介</b></h3>
                <div class="p1">
                    <p id="29">K-means算法的输入是聚类个数<i>k</i>, 大小为<i>n</i>的样本集, 输出为满足方差最小的<i>k</i>个样本集中点聚成的类, 满足同一聚类中的样本相似度较高, 而不同聚类中的样本相似度较小<citation id="86" type="reference"><link href="15" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>。算法步骤为:</p>
                </div>
                <div class="p1">
                    <p id="30"> (1) 从维度空间任意选择<i>k</i>个点作为初始聚类中心。</p>
                </div>
                <div class="p1">
                    <p id="31"> (2) 根据每个聚类样本的特征值, 计算每个样本与这些中心点的距离, 并根据最小距离重新划分样本点所属类。</p>
                </div>
                <div class="p1">
                    <p id="32"> (3) 重新计算每个类的均值作为新的类中心。</p>
                </div>
                <div class="p1">
                    <p id="33"> (4) 计算测度函数, 当满足一定条件, 如函数收敛时, 则算法终止;如果条件不满足则回到步骤 (2) 。</p>
                </div>
                <h3 id="34" name="34" class="anchor-tag"><b>2 算法设计</b></h3>
                <h4 class="anchor-tag" id="35" name="35"><b>2.1 工作流程</b></h4>
                <div class="p1">
                    <p id="36">保存着读段信息的fastq文件 (fastq是一种存储了生物序列以及相应的质量评价的文本格式) 经二代测序标准流程处理后得到bam格式的文件。我们利用samtools<citation id="87" type="reference"><link href="17" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>提供的读段堆砌功能, 分别将tumor.bam、normal.bam转化为pileup格式的文件 (pileup格式文件相当于把每条染色体都竖起来, 将每条reads也竖起来平行的匹配到基因组上) tumor.pu、normal.pu。pileup文件格式提供了单个位点所处染色体、染色体上位置、参考基因上的碱基型、堆砌到此位点上碱基的集合、堆砌碱基质量的集合、堆砌碱基所处读段质量的集合等信息。我们先根据同时出现在癌细胞与正常细胞中这一规则筛选出候选位点, 再利用pileup文件信息抽取7个特征, 并进行标准化处理。由于正负样本数量相差巨大, 在数据过滤步骤中我们选择两个最为重要的特征FISHER_PVAL_T_VS_N和VAF_T_VS_N根据分位数过滤掉大部分数据, 再将剩下的位点用K-means模型聚类。关于FISHER_PVAL_T_VS_N和VAF_T_VS_N我们会在2.3节中详细说明。通过对比聚类结果中两个类中心的特征值选择正负样本。变异识别算法的工作流程如图1所示。</p>
                </div>
                <div class="area_img" id="37">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201903052_037.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 变异识别算法的工作流程" src="Detail/GetImg?filename=images/JYRJ201903052_037.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 变异识别算法的工作流程  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201903052_037.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="38" name="38"><b>2.2 使用堆砌算法生成候选变异集</b></h4>
                <div class="p1">
                    <p id="39">读段堆砌 (pileup) 的意思是, 对于基因组上某一个位点, 将所有覆盖了该位点的读段在该点处进行对齐, 有利于我们的算法计算模型所需要的相关信息, 更进一步地判断该点是否发生了变异, 发生的是何种变异。</p>
                </div>
                <div class="p1">
                    <p id="40">堆砌过程中将逐个遍历全基因组或者基因组上指定区域的所有位点, 在对于单核苷酸突变的研究中, 每个位点上可能出现4种变异。举例来说, 以‘A’腺嘌呤脱氧核糖核苷酸作为参考碱基, 则其可能突变为‘C’胞嘧啶脱氧核糖核苷酸, ‘G’鸟嘌呤脱氧核糖核苷酸或‘T’胸腺嘧啶脱氧核糖核苷酸3中。理论上还有可能发生INDEL——短基因片段的删除或插入, CNV基因片段拷贝数变异等变异, 由于该模型仅针对单碱基突变, 故不做研究。对于每一个同时出现在癌细胞和正常细胞中的基因位点, 我们都将利用读段堆砌的信息进行特征计算。堆叠的可视化展示如图2所示。</p>
                </div>
                <div class="area_img" id="41">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201903052_041.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 堆叠的可视化展示" src="Detail/GetImg?filename=images/JYRJ201903052_041.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 堆叠的可视化展示  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201903052_041.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="42" name="42"><b>2.3 变异特征</b></h4>
                <div class="p1">
                    <p id="43">在查阅相关文献和实验验证后, 我们总结出7个在检测肿瘤突变上有意义的特征, 对于所有位点, 分别提取以下7个特征:</p>
                </div>
                <div class="p1">
                    <p id="44"> (1) VAF_T癌细胞中等位基因频率。癌细胞等位基因频率是衡量该点碱基是否发生突变的一项重要指标, 同正常细胞中发生变异的基因频率不同 (通常只有0、0.5、1三个数值) , 癌细胞中的基因频率可能为0～1间的任意值。但是其分布仍然呈现一定的规律, 因此仅考虑癌细胞中的等位基因频率也是有意义的。</p>
                </div>
                <div class="p1">
                    <p id="45"> (2) VAF_T_VS_N癌细胞和正常细胞等位基因频率之比。前文中提到, 配对样本突变检测目的就是要找出只出现在癌细胞而不出现在正常细胞中的突变, 将AGCT 4种碱基在癌和正常细胞中的频率相除可以很明显地看出频率上的差别, 我们将它列为一项重要的特征。</p>
                </div>
                <div class="p1">
                    <p id="46"> (3) MEAN_MQVQ_T癌细胞位点上读段匹配质量均值。读段匹配质量均值代表该碱基所在读段 (READS) 的匹配好坏。如果匹配质量较差, 则很可能是一个误匹配位点, 那么后续的计算过程也是无效的。</p>
                </div>
                <div class="p1">
                    <p id="47"> (4) MEAN_BQVQ_T癌细胞位点上碱基质量均值。碱基质量代表从测序仪上读到该碱基的正确率, 和匹配质量相同, 我们需要把这一项也纳入聚类模型的特征中。</p>
                </div>
                <div class="p1">
                    <p id="48"> (5) FR_RATIO_T癌细胞中变异基因正反链数目之比。之前的研究表明, 如果变异基因只位于一条链 (正链或反链) 上, 那么很有可能是实验带来的误差。反之, 如果该基因在正反链上分布较均等, 那么就很有可能是变异基因。</p>
                </div>
                <div class="p1">
                    <p id="49"> (6) FISHER_PVAL_T_VS_N癌细胞和正常细胞中的变异。碱基数的Fisher精确检验P值, Fisher精确检验是一种假设检验, 通常用来检验一次随机实验的结果是否支持对于某个随机实验的假设。</p>
                </div>
                <div class="p1">
                    <p id="50">将某一位点上正常细胞和癌细胞里出现的参考碱基数和变异碱基数列成2×2的列联表, 如表1所示, 进行Fisher精确检验, 来判定两类细胞中的变异碱基数是否存在显著差异。这是许多传统算法采用的一个重要标准, 我们也将它列为一项特征。</p>
                </div>
                <div class="area_img" id="51">
                    <p class="img_tit"><b>表1 Fisher精确检验的列联表</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="51" border="1"><tr><td><br />Gene Cell</td><td>Ref</td><td>Alt</td></tr><tr><td><br />T</td><td>Ref_Count_T</td><td>Alt_Count_T</td></tr><tr><td><br />N</td><td>Ref_Count_N</td><td>Alt_Count_T</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="52"> (7) BQP癌细胞中变异等位基因与参考基因碱基质量的秩和检验P值。P值越低代表了alt基因和ref基因的质量差异越大, alt也就越有可能是测序误读。</p>
                </div>
                <div class="p1">
                    <p id="53">上述特征在以往传统的贝叶斯模型算法中也常常作为重要的参数, 因此我们有理由认为这些特征是有意义的。</p>
                </div>
                <h3 id="54" name="54" class="anchor-tag"><b>3 实验分析</b></h3>
                <h4 class="anchor-tag" id="55" name="55"><b>3.1 实验数据集</b></h4>
                <div class="p1">
                    <p id="56">模拟实验数据中, 肿瘤样本通过随机采样人类样本基因NA12981的序列替换NA12878的序列片段生成, 被替换的碱基也就是发生了体细胞突变的碱基, 而原来NA12878中的突变则为生殖细胞突变。由此我们得到了一批可以用于评估算法性能的模拟数据。</p>
                </div>
                <div class="p1">
                    <p id="57">真实实验数据来源于2017年全国肿瘤体细胞基因突变高通量测序检测生物信息学分析室间质量评价活动所提供Illumina HiSeq2500高通量测序平台的原始靶向测序数据。原始数据为fastq文件格式, 分为肿瘤细胞和正常细胞两组实验数据。经标准二代测序分析流程处理后得到bam格式的文件。读段匹配过程使用bwa进行匹配, 随后对产生的bam文件应用GATK<citation id="88" type="reference"><link href="17" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>工具包做了标准的预处理, 包括标记重复读段、局部重匹配和碱基质量再校正等。将本文算法与目前主流的3款肿瘤突变检测工具进行了对比, 包括Mutect2、Somatic-Sniper和JointSNVMix<citation id="89" type="reference"><link href="21" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>。三者使用的都是官方默认的参数配置或是官方推荐的“最佳实践”方案。</p>
                </div>
                <h4 class="anchor-tag" id="58" name="58"><b>3.2 评价标准</b></h4>
                <div class="p1">
                    <p id="59">分别选择Mutect2的Tlod, Somatic-Sniper的SSC, JointSNVMix的Prob和K-means中距离类中心的距离作为算法的输出分数, 据此生成PR曲线, 并在Tlod&gt;30, SSC&gt;50, Prob&gt;0.9时计算出各算法的<i>Precision</i>、<i>Recall</i>和<i>F</i>-<i>Score</i>。</p>
                </div>
                <div class="p1">
                    <p id="60"><mathml id="61"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Ρ</mi><mi>r</mi><mi>e</mi><mi>c</mi><mi>i</mi><mi>s</mi><mi>i</mi><mi>o</mi><mi>n</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ρ</mi></mrow></mfrac></mrow></math></mathml>      (1) </p>
                </div>
                <div class="p1">
                    <p id="62"><mathml id="63"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>R</mi><mi>e</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>l</mi><mo>=</mo><mfrac><mrow><mi>Τ</mi><mi>Ρ</mi></mrow><mrow><mi>Τ</mi><mi>Ρ</mi><mo>+</mo><mi>F</mi><mi>Ν</mi></mrow></mfrac></mrow></math></mathml>      (2) </p>
                </div>
                <div class="p1">
                    <p id="64"><mathml id="65"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>F</mi><mo>-</mo><mi>S</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi><mo>=</mo><mfrac><mrow><mn>2</mn><mo>×</mo><mi>Ρ</mi><mi>r</mi><mi>e</mi><mi>c</mi><mi>i</mi><mi>s</mi><mi>i</mi><mi>o</mi><mi>n</mi><mo>×</mo><mi>R</mi><mi>e</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>l</mi></mrow><mrow><mi>Ρ</mi><mi>r</mi><mi>e</mi><mi>c</mi><mi>i</mi><mi>s</mi><mi>i</mi><mi>o</mi><mi>n</mi><mo>+</mo><mi>R</mi><mi>e</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>l</mi></mrow></mfrac></mrow></math></mathml>      (3) </p>
                </div>
                <div class="p1">
                    <p id="66">PR曲线以<i>Precison</i>作为纵轴, 以<i>Recall</i>作为横轴, 设定不同的阈值将算法输出分数映射为正负样本, 整张图上曲线越向右上方凸往往算法性能越优。</p>
                </div>
                <h4 class="anchor-tag" id="67" name="67"><b>3.3 实验结果</b></h4>
                <div class="p1">
                    <p id="68">4种算法在模拟数据上的结果如图3所示。可以看出, 在模拟数据上本文算法在准确率和召回率上均占优势。</p>
                </div>
                <div class="area_img" id="69">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201903052_069.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 模拟数据PR曲线" src="Detail/GetImg?filename=images/JYRJ201903052_069.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 模拟数据PR曲线  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201903052_069.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="70">在有标签的IlluminaB17数据上的变异识别结果如图4、图5和表2所示。可以看出, 采用无监督聚类的算法较以往传统算法可以检测出更多的变异位点, 同时保证较高的查准率。</p>
                </div>
                <div class="area_img" id="71">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201903052_071.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 IlluminaB17数据算法正例与真实正例对比" src="Detail/GetImg?filename=images/JYRJ201903052_071.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 IlluminaB17数据算法正例与真实正例对比  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201903052_071.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="72">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201903052_072.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图5 IlluminaB17数据PR曲线" src="Detail/GetImg?filename=images/JYRJ201903052_072.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图5 IlluminaB17数据PR曲线  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201903052_072.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="73">
                    <p class="img_tit"><b>表2 IlluminaB17数据算法性能对比</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="73" border="1"><tr><td rowspan="2"><br />指标</td><td colspan="4"><br />工具</td></tr><tr><td><br />Mutect2</td><td>Somatic-Sniper</td><td>JointSNVMix</td><td>K-means</td></tr><tr><td>Precision</td><td>0.469 4</td><td>0.732 1</td><td>0.666 7</td><td>0.890 4</td></tr><tr><td><br />Recall</td><td>0.302 6</td><td>0.539 5</td><td>0.184 2</td><td>0.855 3</td></tr><tr><td><br />F-Score</td><td>0.368 0</td><td>0.621 2</td><td>0.288 6</td><td>0.872 4</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="74">在没有标签的肺癌患者肺部细胞数据上, 如图6和表3所示。可以发现, 用本文方法找出的位点与其他3种方法的二者交集2-cross、三者交集3-cross均表现较好。K-means与其他3种方法的交集数量都比较大, 说明该算法可以找出以往算法所认同的大部分变异。</p>
                </div>
                <div class="area_img" id="75">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201903052_075.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图6 肺癌数据上算法交集数量对比" src="Detail/GetImg?filename=images/JYRJ201903052_075.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图6 肺癌数据上算法交集数量对比  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201903052_075.jpg&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="76">
                    <p class="img_tit"><b>表3 肺癌数据算法2-cross数量对比</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="76" border="1"><tr><td>工具</td><td>Mutect2</td><td>Somatic-Sniper</td><td>JointSNVMix</td><td>K-means</td></tr><tr><td><br />Mutect2</td><td>——</td><td>10</td><td>7</td><td>56</td></tr><tr><td><br />Somatic-Sniper</td><td>10</td><td>——</td><td>109</td><td>74</td></tr><tr><td><br />JointSNVMix</td><td>7</td><td>109</td><td>——</td><td>82</td></tr><tr><td><br />K-means</td><td>56</td><td>74</td><td>82</td><td></td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <h3 id="77" name="77" class="anchor-tag"><b>4 结 语</b></h3>
                <div class="p1">
                    <p id="78">本文提出了一种利用无监督聚类来识别单核苷酸变异的肿瘤突变识别算法。不同于以往算法一贯采用贝叶斯模型, 我们通过结合高效产生候选变异集合的读段堆砌算法与无监督的聚类算法, 自动而准确地识别变异位点, 得以同时实现召回率、精度和速度三大指标上的良好表现。将其及另外三个变异识别器应用于来深度测序样本数据, 其结果证明了本文算法具有良好的效果。</p>
                </div>
                <div class="p1">
                    <p id="79">本文算法中使用的特征均为标准化后的特征, 在实际使用中由于数据集的差异, 不同特征的重要性存在差异, 为每个特征选择适当的权重或可明显提升算法的性能。权重的选择既可以基于经验估计, 也可以迭代取值进行实验求得最优的参数。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="3">
                            <a id="bibliography_1" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYRJ201704005&amp;v=MTM2OTZZWVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnNGaURoVkw3UEx6VFpaTEc0SDliTXE0OUY=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[1]</b> 孟倩.基于高通量测序的短序列生物数据压缩研究[J].计算机应用与软件, 2017, 34 (4) :22-27, 98.
                            </a>
                        </p>
                        <p id="5">
                            <a id="bibliography_2" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Whole genome sequencing">

                                <b>[2]</b> Ng P C, Kirkness E F. Whole genome sequencing[J]. Methods in Molecular Biology, 2010, 628:215-226.
                            </a>
                        </p>
                        <p id="7">
                            <a id="bibliography_3" target="_blank" href="http://scholar.cnki.net/result.aspx?q=International network of cancer genome projects">

                                <b>[3]</b> Hudson T J, Anderson W, Artez A, et al. International network of cancer genome projects[J]. Nature 2010, 464:993-998.
                            </a>
                        </p>
                        <p id="9">
                            <a id="bibliography_4" target="_blank" href="http://scholar.cnki.net/result.aspx?q=NGS Analysis of Somatic Mutations in Cancer Genomes">

                                <b>[4]</b> Prieto T , Alves J M , Posada D . NGS Analysis of Somatic Mutations in Cancer Genomes[M]//Big Data Analytics in Genomics. Springer International Publishing, 2016.
                            </a>
                        </p>
                        <p id="11">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=SomaticSniper: identification of somatic point mutations in whole genome sequencing data">

                                <b>[5]</b> Larson D E , Harris C C , Chen K , et al. SomaticSniper: identification of somatic point mutations in whole genome sequencing data[J].Bioinformatics, 2012, 28 (3) :311-317.
                            </a>
                        </p>
                        <p id="13">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples">

                                <b>[6]</b> Cibulskis K , Lawrence M S , Carter S L , et al. Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples[J]. Nature Biotechnology, 2013, 31 (3) :213-219.
                            </a>
                        </p>
                        <p id="15">
                            <a id="bibliography_7" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYRJ201705008&amp;v=MTYxMDBESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVac0ZpRGhWTDdQTHpUWlpMRzRIOWJNcW85RmJJUUs=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[7]</b> 郭占元, 林涛.面向大规模数据快速聚类K-means算法的研究[J].计算机应用与软件, 2017, 34 (5) :43-7, 53.
                            </a>
                        </p>
                        <p id="17">
                            <a id="bibliography_8" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The Sequence Alignment/Map format and SAMtools">

                                <b>[8]</b> Li H, Handsaker B, Wysoker A, et al.The Sequence Alignment/Map format and SAMtools[J] Bioinformatics, 2009, 25 (16) :2078-2079.
                            </a>
                        </p>
                        <p id="19">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data">

                                <b>[9]</b> Mckenna A , Hanna M , Banks E , et al. The Genome Analysis Toolkit : A MapReduce framework for analyzing next-generation DNA sequencing data[J].Genome Research, 2010, 20 (9) :1297-1303.
                            </a>
                        </p>
                        <p id="21">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=JointSNVMix: a probabilistic model for accurate detection of somatic mutations in normal/tumour paired next-generation sequencing data">

                                <b>[10]</b> Roth A , Ding J , Morin R , et al. JointSNVMix: a probabilistic model for accurate detection of somatic mutations in normal/tumour paired next-generation sequencing data[J]. Bioinformatics, 2012, 28 (7) :907-913.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JYRJ201903052" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYRJ201903052&amp;v=MTg5NzNJOUFab1FLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnNGaURoVkw3UEx6VFpaTEc0SDlqTXI=&amp;uid=WEEvREcwSlJHSldRa1FhcEFLUmVhZDN5c2Q1UnJpL2tUTHpCSWVzVW9XYz0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
