<!DOCTYPE html>
<html>
<head>
    <title>全文阅读--XML全文阅读--中国知网</title>
    <link rel="icon" href="/kxreader/favicon.ico" />
    <link rel="shortcut Icon" href="/kxreader/favicon.ico" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="keywords" content="文献 XML KBASE CNKI 中国知网" />
    <meta name="description" content="XML文献检索" />
    <link href="/kxreader/Content/css/detail?v=qX2z2KjRAEyQiNfAbKtl7dLnsqFoQ5Jdw3TZfDf0n1k1" rel="stylesheet"/>

    <script type="text/javascript">
        var APPPATH = '/kxreader';
    </script>
</head>

<body>
    
<script type="text/javascript" src="//login.cnki.net/TopLogin/api/loginapi/get?type=top&amp;localCSS=&amp;returnurl=%2f%2fkns.cnki.net%2f%2fKXReader%2fDetail%3fTIMESTAMP%3d637135613222346250%26DBCODE%3dCJFD%26TABLEName%3dCJFDLAST2019%26FileName%3dJYRJ201908049%26RESULT%3d1%26SIGN%3dtfo2LudXi%252bXTk3vASHNqRzh7X%252fk%253d"></script>

<div id="headerBox" class="header">
    <div class="topbar">
        <div class="textalign">
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JYRJ201908049&amp;align=md">
                <i class="icon-cen active" title="居中对齐"></i>
            </a>
            <a href="/kxreader/Detail?dbcode=CJFD&amp;filename=JYRJ201908049&amp;align=lt">
                <i class="icon-left " title="左对齐"></i>
            </a>
        </div>
        <h6 class="free-tip"><i class="icon"></i>HTML阅读开放试用阶段，欢迎体验！</h6>
    </div>
</div>

    



<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYRJ201908049&amp;v=MTU2ODlqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVadEZ5am1XcnJMTHpUWlpMRzRIOWpNcDQ5QmJZUUtESDg0dlI0VDY=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>

    <div class="main">

        

    <div class="sidebar-a">
        <!--sidebar start-->
        <div class="sidenav">
            <div class="arrow"><span></span></div>
            <!--sidebar_list start-->
            <dl class="sidenav-list">
                    <dt class="tit">目录结构</dt>
                            <dd class="guide">
                                    <p><a href="#41" data-title="&lt;b&gt;0 引 言&lt;/b&gt; "><b>0 引 言</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#45" data-title="&lt;b&gt;1 问题描述&lt;/b&gt; "><b>1 问题描述</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#50" data-title="&lt;b&gt;2 SVM和LS-SVM&lt;/b&gt; "><b>2 SVM和LS-SVM</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#85" data-title="&lt;b&gt;3 算法实现&lt;/b&gt; "><b>3 算法实现</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#86" data-title="&lt;b&gt;3.1 数据预处理&lt;/b&gt;"><b>3.1 数据预处理</b></a></li>
                                                <li><a href="#91" data-title="&lt;b&gt;3.2 相关系数矩阵&lt;/b&gt;"><b>3.2 相关系数矩阵</b></a></li>
                                                <li><a href="#95" data-title="&lt;b&gt;3.3 提取主成分分量&lt;/b&gt;"><b>3.3 提取主成分分量</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#104" data-title="&lt;b&gt;4 仿真实验&lt;/b&gt; "><b>4 仿真实验</b></a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#105" data-title="&lt;b&gt;4.1 实验数据及开发环境&lt;/b&gt;"><b>4.1 实验数据及开发环境</b></a></li>
                                                <li><a href="#109" data-title="&lt;b&gt;4.2 实验结果分析&lt;/b&gt;"><b>4.2 实验结果分析</b></a></li>
                                    </ul>
                            </dd>
                            <dd class="guide">
                                    <p><a href="#129" data-title="&lt;b&gt;5 结 语&lt;/b&gt; "><b>5 结 语</b></a><i></i></p>
                                                            </dd>
                            <dd class="guide">
                                    <p><a href="#" data-title="文内图表 ">文内图表</a><i></i></p>
                                                                    <ul class="contentbox">
                                                <li><a href="#107" data-title="&lt;b&gt;表1 实验数据及描述&lt;/b&gt;"><b>表1 实验数据及描述</b></a></li>
                                                <li><a href="#113" data-title="&lt;b&gt;表2 结肠癌数据集中选取的特征基因&lt;/b&gt;"><b>表2 结肠癌数据集中选取的特征基因</b></a></li>
                                                <li><a href="#116" data-title="&lt;b&gt;表3 结肠癌数据集选取的特征基因集实验结果&lt;/b&gt;"><b>表3 结肠癌数据集选取的特征基因集实验结果</b></a></li>
                                                <li><a href="#117" data-title="图1 结肠癌数据LS-SVM训练模型准确率">图1 结肠癌数据LS-SVM训练模型准确率</a></li>
                                                <li><a href="#118" data-title="图2 结肠癌数据LS-SVM独立测试准确率">图2 结肠癌数据LS-SVM独立测试准确率</a></li>
                                                <li><a href="#121" data-title="&lt;b&gt;表4 白血病数据集中选取的特征基因&lt;/b&gt;"><b>表4 白血病数据集中选取的特征基因</b></a></li>
                                                <li><a href="#125" data-title="&lt;b&gt;表5 白血病数据集选取的特征基因集实验结果&lt;/b&gt;"><b>表5 白血病数据集选取的特征基因集实验结果</b></a></li>
                                                <li><a href="#126" data-title="图3 白血病数据LS-SVM留一交叉检验测试准确率">图3 白血病数据LS-SVM留一交叉检验测试准确率</a></li>
                                                <li><a href="#127" data-title="图4 白血病数据集LS-SVM独立测试准确率">图4 白血病数据集LS-SVM独立测试准确率</a></li>
                                    </ul>
                            </dd>
                                    <dd class="guide">
                                        <h6>
                                            <p><a href="#a_bibliography">参考文献</a> </p>
                                        </h6>
                                    </dd>

            </dl>
        </div>
        <!--sidebar end-->
        &nbsp;
        <!--此处有一空格符 勿删-->
    </div>

                <div class="sidebar-b three-collumn" style="width:0;">
            <div class="refer" style="width: 0;">
                <div class="arrow off" title="参考文献"><span></span></div>
                <div class="js-scrollbox" >
                    
                    <div class="subbox active">
                        <h4>
                            <span class="tit">参考文献</span>
                            <a class="close" href="javascript:void(0)">x</a>
                        </h4>
                        <div class="side-scroller">
                            <ul class="refer-list">
                                <li id="3">


                                    <a id="bibliography_1" title=" Alon U, Barkai N, Notterman D A, et al.Broad Patterns of Gene Expression Revealed by Clustering Analysis of Tumor and Normal Colon Tissues Probed by Oligonucleotide Arrays[J].Proceedings of the National Academy of Sciences of the United States of America, 1999, 96 (12) :6745-6750." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Broad patterns of gene expression revealed by clustering analysis of tumor and normal colon tissues probed by oligonucleotide arrays">
                                        <b>[1]</b>
                                         Alon U, Barkai N, Notterman D A, et al.Broad Patterns of Gene Expression Revealed by Clustering Analysis of Tumor and Normal Colon Tissues Probed by Oligonucleotide Arrays[J].Proceedings of the National Academy of Sciences of the United States of America, 1999, 96 (12) :6745-6750.
                                    </a>
                                </li>
                                <li id="5">


                                    <a id="bibliography_2" title=" Furey T S, Cristianini N, Duffy N, et al.Support vector machine classification and validation of cancer tissue samples using microarray expression data[J].Bioinformatics, 2000, 16 (10) :906-914." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJOX&amp;filename=SJOX14010900013873&amp;v=MDE3ODdNQkhzNm9CTVQ2VDRQUUgvaXJSZEdlcnFRVE1ud1plWnRGaW5sVXJ6SUlGMGRieEU9TmlmRWRySzhIdERNcG85RlpPbw==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[2]</b>
                                         Furey T S, Cristianini N, Duffy N, et al.Support vector machine classification and validation of cancer tissue samples using microarray expression data[J].Bioinformatics, 2000, 16 (10) :906-914.
                                    </a>
                                </li>
                                <li id="7">


                                    <a id="bibliography_3" >
                                        <b>[3]</b>
                                     Golub T R, Slonim D K, Tamayo P, et al.Molecular classification of cancer:class discovery and class prediction by gene expression monitoring[J].Science, 1999, 286:531-537.</a>
                                </li>
                                <li id="9">


                                    <a id="bibliography_4" title=" Guyon I, Weston J, Bamhill S, et al.Gene selection for cancer classification using support vector machines[J].Machine Learning, 2002, 46:389-422." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00001340375&amp;v=Mjk2MDJxUnJ4b3hjTUg3UjdxZForWnVGaXZsVTd6QkpWMD1OajdCYXJPNEh0SE5ySXRGWit3S1kzazV6QmRoNGo5OVNY&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[4]</b>
                                         Guyon I, Weston J, Bamhill S, et al.Gene selection for cancer classification using support vector machines[J].Machine Learning, 2002, 46:389-422.
                                    </a>
                                </li>
                                <li id="11">


                                    <a id="bibliography_5" title=" Chiaretti S, Li X C, Gentleman R, et al.Gene expression profile of adult T-cell acute lymphocytic leukemia identifies distinct subsets of patients with different response to therapy and survival[J].Blood, 2004, 103 (7) :2771-2778." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Gene expression profile of adult T-cell acute lymphocytic leukemia identifies distinct subsets of patients with different response to therapy and survival">
                                        <b>[5]</b>
                                         Chiaretti S, Li X C, Gentleman R, et al.Gene expression profile of adult T-cell acute lymphocytic leukemia identifies distinct subsets of patients with different response to therapy and survival[J].Blood, 2004, 103 (7) :2771-2778.
                                    </a>
                                </li>
                                <li id="13">


                                    <a id="bibliography_6" title=" Sun Z F, Yang P.Gene expression profiling on lung cancer outcome prediction:present clinical value and future premise[J].Cancer Epidemiology, Biomarkers &amp;amp; Prevention, 2006, 15 (11) :2063-2068." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=Gene expression profiling on lung cancer outcome prediction: present clinical value and future premise">
                                        <b>[6]</b>
                                         Sun Z F, Yang P.Gene expression profiling on lung cancer outcome prediction:present clinical value and future premise[J].Cancer Epidemiology, Biomarkers &amp;amp; Prevention, 2006, 15 (11) :2063-2068.
                                    </a>
                                </li>
                                <li id="15">


                                    <a id="bibliography_7" title=" Devi A V C, Devaraj D, Venkatesulu M.Gene Expression data classification using support vector machine and mutual information-based gene selection[J].Procedia Computer Science, 2015, 47:13-21." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES53EA201C30891CA4B88FC003F8BF26A8&amp;v=MjUxOTJyTFUwNXR0aHhMdTJ4YXM9TmlmT2ZiYTdhNkRPcjQ0Mlorc0hCWDFLdmhKaDRqY0xPMy9pcjJROUM4U1dROHVYQ09OdkZTaVdXcjdKSUZwbWFCdUhZZk9HUWxmQg==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[7]</b>
                                         Devi A V C, Devaraj D, Venkatesulu M.Gene Expression data classification using support vector machine and mutual information-based gene selection[J].Procedia Computer Science, 2015, 47:13-21.
                                    </a>
                                </li>
                                <li id="17">


                                    <a id="bibliography_8" title=" Wang A G, An N, Chen G L, et al.Improving PLS-RFE based gene selection for microarray[J].Computers in Biology and Medicine, 2015, 62:14-24." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES15120600576292&amp;v=MDA1MjhGaW5sVXJ6SUlGMGRieEU9TmlmT2ZiSzlIOVBNcVk5Rllld0pEblU3b0JNVDZUNFBRSC9pclJkR2VycVFUTW53WmVadA==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[8]</b>
                                         Wang A G, An N, Chen G L, et al.Improving PLS-RFE based gene selection for microarray[J].Computers in Biology and Medicine, 2015, 62:14-24.
                                    </a>
                                </li>
                                <li id="19">


                                    <a id="bibliography_9" title=" Sharbaf F V, Mosafer S, Moattar M H.A hybrid gene selection approach for microarray data classification using cellular learning automata and colony optimization[J].Genomics, 2016, 107:231-238." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A hybrid gene selection approach for microarray data classi?cation using cellular learning automata and ant colony optimization">
                                        <b>[9]</b>
                                         Sharbaf F V, Mosafer S, Moattar M H.A hybrid gene selection approach for microarray data classification using cellular learning automata and colony optimization[J].Genomics, 2016, 107:231-238.
                                    </a>
                                </li>
                                <li id="21">


                                    <a id="bibliography_10" title=" Khan S, Naseem I, Togneri R, et al.A Novel Adaptive Kernel for the RBF Neural Networks[J].Circuits, Systems, and Signal Processing, 2017, 36 (4) :1639-1653." target="_blank"
                                       href="http://scholar.cnki.net/result.aspx?q=A novel adaptive kernel for the RBF neural networks">
                                        <b>[10]</b>
                                         Khan S, Naseem I, Togneri R, et al.A Novel Adaptive Kernel for the RBF Neural Networks[J].Circuits, Systems, and Signal Processing, 2017, 36 (4) :1639-1653.
                                    </a>
                                </li>
                                <li id="23">


                                    <a id="bibliography_11" title=" Xiao Y W, Wu J, Lin Z L, et al.A deep learning-based multi-model ensemble method for cancer prediction[J].Computer Methods and Programs in Biomedicine, 2018, 153:1-9." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES5279C9563C2CED2868FE75A97A20A739&amp;v=MDAxNTdCdUhZZk9HUWxmQnJMVTA1dHRoeEx1Mnhhcz1OaWZPZmJhNkdkaS9wb3BEWjVnTmZ3bE56UjRWNGtrSVQzcVRwUlZFZTdMbFFybVdDT052RlNpV1dyN0pJRnBtYQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[11]</b>
                                         Xiao Y W, Wu J, Lin Z L, et al.A deep learning-based multi-model ensemble method for cancer prediction[J].Computer Methods and Programs in Biomedicine, 2018, 153:1-9.
                                    </a>
                                </li>
                                <li id="25">


                                    <a id="bibliography_12" title=" 李颖新, 刘全金, 阮晓钢.急性白血病的基因表达谱分析与亚型分类特征的鉴别[J].中国生物医学工程学报, 2005, 24 (2) :240-244." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZSWY200502023&amp;v=MTg0NjBGckNVUjdxZlp1WnRGeWptV3JyTFB6N2NkN0c0SHRUTXJZOUhaNFFLREg4NHZSNFQ2ajU0TzN6cXFCdEc=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[12]</b>
                                         李颖新, 刘全金, 阮晓钢.急性白血病的基因表达谱分析与亚型分类特征的鉴别[J].中国生物医学工程学报, 2005, 24 (2) :240-244.
                                    </a>
                                </li>
                                <li id="27">


                                    <a id="bibliography_13" title=" 马煜, 陈莉, 欧立奇.基因微阵列数据的聚类分析算法研究[J].计算机工程与应用, 2006 (5) :176-178." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSGG200605054&amp;v=MjE3ODhIdGZNcW85QVlJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVadEZ5am1XcnJMTHo3TWFiRzQ=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[13]</b>
                                         马煜, 陈莉, 欧立奇.基因微阵列数据的聚类分析算法研究[J].计算机工程与应用, 2006 (5) :176-178.
                                    </a>
                                </li>
                                <li id="29">


                                    <a id="bibliography_14" title=" 韩利, 祁云嵩, 王俊.基于粗糙集的支持向量机微阵列数据分类方法[J].科学技术与工程, 2009, 9 (1) :152-155." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=KXJS200901036&amp;v=MTU1NzMzenFxQnRHRnJDVVI3cWZadVp0RnlqbVdyckxMalhCZmJHNEh0ak1ybzlHWW9RS0RIODR2UjRUNmo1NE8=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[14]</b>
                                         韩利, 祁云嵩, 王俊.基于粗糙集的支持向量机微阵列数据分类方法[J].科学技术与工程, 2009, 9 (1) :152-155.
                                    </a>
                                </li>
                                <li id="31">


                                    <a id="bibliography_15" title=" 朱钦平, 胡晓涵, 祁云嵩.基于非线性回归分析的差异基因选择方法[J].科学技术与工程, 2010, 10 (27) :6675-6678." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=KXJS201027014&amp;v=MjgxMzhyTExqWEJmYkc0SDlIT3FJOUVZSVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnRGeWptV3I=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[15]</b>
                                         朱钦平, 胡晓涵, 祁云嵩.基于非线性回归分析的差异基因选择方法[J].科学技术与工程, 2010, 10 (27) :6675-6678.
                                    </a>
                                </li>
                                <li id="33">


                                    <a id="bibliography_16" title=" 姚全珠, 蔡婕.基于PSO的LS-SVM特征选择与参数优化算法[J].计算机工程与应用, 2010, 46 (1) :134-136, 229." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSGG201001042&amp;v=MDkzNjk5SE1ybzlCWm9RS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVp0RnlqbVdyckxMejdNYWJHNEg=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[16]</b>
                                         姚全珠, 蔡婕.基于PSO的LS-SVM特征选择与参数优化算法[J].计算机工程与应用, 2010, 46 (1) :134-136, 229.
                                    </a>
                                </li>
                                <li id="35">


                                    <a id="bibliography_17" title=" 孙刚, 张靖.面向高维微阵列数据的混合特征选择算法[J].小型微型计算机系统, 2015, 36 (6) :1209-1213." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=XXWX201506011&amp;v=MjMyMjFUWGNkckc0SDlUTXFZOUVaWVFLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnRGeWptV3JyTFA=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[17]</b>
                                         孙刚, 张靖.面向高维微阵列数据的混合特征选择算法[J].小型微型计算机系统, 2015, 36 (6) :1209-1213.
                                    </a>
                                </li>
                                <li id="37">


                                    <a id="bibliography_18" title=" 杨勤, 董洪伟, 薛燕娜.核多元基因选择和极限学习机在微阵列分析中的应用[J].传感器与微系统, 2016, 35 (5) :146-148, 153." target="_blank"
                                       href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=CGQJ201605042&amp;v=MTY5MjFpcmFaTEc0SDlmTXFvOUJab1FLREg4NHZSNFQ2ajU0TzN6cXFCdEdGckNVUjdxZlp1WnRGeWptV3JyTEo=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                        <b>[18]</b>
                                         杨勤, 董洪伟, 薛燕娜.核多元基因选择和极限学习机在微阵列分析中的应用[J].传感器与微系统, 2016, 35 (5) :146-148, 153.
                                    </a>
                                </li>
                                <li id="39">


                                    <a id="bibliography_19" >
                                        <b>[19]</b>
                                     Torgo L.数据挖掘与R语言[M].李洪成, 陈道轮, 吴立明, 译.北京:机械工业出版社, 2016.</a>
                                </li>
                            </ul>
                            <div style='display: none;' class="zqscroller" >
                                <h4 class="">附加材料</h4>
                                <ul></ul>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            &nbsp;
            <!--此处有一空格符 勿删-->
        </div>

        
    <div class="content">



        <!--tips start-->
                            <div class="tips">
                    <a href="http://navi.cnki.net/KNavi/JournalDetail?pcode=CJFD&amp;pykm=JYRJ" target="_blank">计算机应用与软件</a>
                2019,36(08),288-292 DOI:10.3969/j.issn.1000-386x.2019.08.048            </div>
        <!--tips end-->
            <div class="top-title">
                <h1 class="title">
                    <span class="vm"><b>基于最小二乘支持向量机微阵列基因特征分类</b></span>
                                    </h1>

            </div>
                        <h2>
                                <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=au&amp;skey=%E9%AB%98%E6%8C%AF%E6%96%8C&amp;code=42375715&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">高振斌</a>
                </h2>
                    <h2>
                    <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=in&amp;skey=%E8%A5%BF%E5%AE%89%E8%B4%A2%E7%BB%8F%E5%A4%A7%E5%AD%A6%E7%BB%9F%E8%AE%A1%E5%AD%A6%E9%99%A2%E6%95%B0%E5%AD%A6%E4%B8%8E%E5%BA%94%E7%94%A8%E6%95%B0%E5%AD%A6%E7%A0%94%E7%A9%B6%E6%89%80&amp;code=1704597&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">西安财经大学统计学院数学与应用数学研究所</a>
            </h2>

        
<div class="link">
    <a id="aexport" class="icon icon-output"  onclick="" href="javascript:void(0);"><i></i>导出/参考文献</a>
    
    <span class="shareBoard" onmouseover="$('#sharedet').show();$('#this').addClass('shareBoardCUR')" onmouseout="$('#sharedet').hide();$('#this').removeClass('shareBoardCUR')">
        <a class="icon icon-share" href="#"><i></i>分享<em></em></a>
        <ul class="shareHide" id="sharedet" style="display: none;">
            <li><a title="复制链接" class="copy" onclick="" href="#"><i></i>复制链接</a></li>
            <li><a title="分享到新浪微博" class="xl" onclick="" href="javascript:common.ShareAction('xl');"><i></i>新浪微博</a></li>
            <li>
                <a title="分享到微信" class="wx" onclick="" href="#"><i></i>微信扫一扫</a>
                <div class="qrcode"><img src='' alt='' /></div>
            </li>
        </ul>

    </span>
    
    <a id="RefTrack" title="创建引文跟踪" class="icon icon-track" onclick="" href="javascript:void(0);"> <i></i>创建引文跟踪 </a>
    <a id="ashoucang" title="收藏" class="icon icon-favor" onclick="" href="javascript:void(0);"><i></i>收藏</a>
    <a class="icon icon-print" onclick="window.print();" href="javascript:void(0);"><i></i>打印</a>
    
    <!--版本切换 end-->
</div>
                            <div class="data" id="a_abstract">
                <span class="keys">摘<span style="font-family: 'Times New Roman';">&nbsp;&nbsp;&nbsp;&nbsp;</span>要：</span>
                <p>基因表达分析中的微阵列数据具有高维、高冗余的特点, 给基因表达数据分类带来很大的困难。机器学习中的最小二乘支持向量机算法具有计算效率高的优势, 从而为数据挖掘提供了一条有效途径。针对两类典型的癌症微阵列数据集 (结肠癌集和白血病集) , 进行归一化预处理并且计算其相关系数矩阵;使用主成分分析法进行降维处理, 得到用于特征选取和分类的信息基因集 (各取10个基因) ;采用最小二乘支持向量机分类器对信息基因集进行分类。实验结果表明, 该算法在两类癌症数据集上的留一交叉检验的准确率分别为97.5%和100%, 具有比其他分类器都高的测试准确率, 为进一步医学临床应用提供可靠的诊断依据。</p>
            </div>
                    <div class="data" id="a_keywords">
                <span class="keys">关键词：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E5%BE%AE%E9%98%B5%E5%88%97&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">微阵列;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E7%89%B9%E5%BE%81%E5%88%86%E7%B1%BB&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">特征分类;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E9%99%8D%E7%BB%B4&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">降维;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=%E6%9C%80%E5%B0%8F%E4%BA%8C%E4%B9%98%E6%94%AF%E6%8C%81%E5%90%91%E9%87%8F%E6%9C%BA&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">最小二乘支持向量机;</a>
                </p>
            </div>
        
        <!--brief start-->
        
            <div class="brief">
                    <p>
                            <b>作者简介：</b>
                                                        <span>
                                    高振斌, 副教授, 主研领域:复杂系统的控制与优化。;
                                </span>
                    </p>
                                    <p><b>收稿日期：</b>2018-11-02</p>

            </div>
                    <h1><b>MICROARRAY GENE FEATURE CLASSIFICATION BASED ON LS-SVM</b></h1>
                    <h2>
                    <span>Gao Zhenbin</span>
            </h2>
                    <h2>
                    <span>Institute of Mathematics and Applied Mathematics, School of Statistics, Xi'an University of Finance and Economics</span>
            </h2>
                            <div class="data" id="a_abstractEN">
                <span class="keys">Abstract：</span>
                <p>Microarray data in gene expression analysis is characterized by high dimensionality and redundancy, which makes it difficult to classify gene expression data. The least-squares support vector machine (LS-SVM) algorithm in machine learning has the advantage of high computational efficiency, which provides an effective way for data mining. For two types of typical cancer microarray data sets (colon cancer set and leukemia set) , we normalized the data and calculated the correlation coefficient matrix. The dimensionality reduction was carried out by principal component analysis, and the information gene sets (10 genes each) for feature selection and classification were obtained. Then, we used LS-SVM classifier to classify information gene sets. The experimental results show that the accuracy of this algorithm is 97.5% and 100% respectively, which is higher than other classifiers. It provides reliable diagnostic basis for further clinical application.</p>
            </div>
                    <div class="data" id="a_keywordsEN">
                <span class="keys">Keyword：</span>
                <p>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Microarray&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Microarray;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Feature%20classification&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Feature classification;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Reducing%20dimension&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Reducing dimension;</a>
                        <a href="/kcms/detail/knetsearch.aspx?dbcode=CJFD&amp;sfield=kw&amp;skey=Least-square%20support%20vector%20machine%20(LS-SVM)%20&amp;code=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" target="_blank">Least-square support vector machine (LS-SVM) ;</a>
                </p>
            </div>
                    <div class="brief">
                
                    <p>
                                            </p>
                                    <p><b>Received：</b> 2018-11-02</p>
                            </div>


        <!--brief start-->
                        <h3 id="41" name="41" class="anchor-tag"><b>0 引 言</b></h3>
                <div class="p1">
                    <p id="42">随着大规模基因表达谱技术的发展, 人类各种组织的正常基因表达已经获得, 各类病人的基因表达谱都有了参考的基准, 因此基因表达数据的分析与建模已经成为生物信息学研究领域中的重要课题。</p>
                </div>
                <div class="p1">
                    <p id="43">众多的研究者在此方向上进行了卓有成效的研究<citation id="145" type="reference"><link href="3" rel="bibliography" /><link href="5" rel="bibliography" /><link href="7" rel="bibliography" /><link href="9" rel="bibliography" /><sup>[<a class="sup">1</a>,<a class="sup">2</a>,<a class="sup">3</a>,<a class="sup">4</a>]</sup></citation>。Chiaretti等<citation id="131" type="reference"><link href="11" rel="bibliography" /><sup>[<a class="sup">5</a>]</sup></citation>对T细胞急性白血病的基因表达谱的分类进行了研究, 并应用到临床治疗和预测之中;Sun等<citation id="132" type="reference"><link href="13" rel="bibliography" /><sup>[<a class="sup">6</a>]</sup></citation>在肺癌临床治疗中通过对脱氧核糖核酸 (DNA) 微阵列数据特征分类从而作出预判;Devi等<citation id="133" type="reference"><link href="15" rel="bibliography" /><sup>[<a class="sup">7</a>]</sup></citation>基于互信息选择信息基因, 进而使用支持向量机 (SVM) 分类器对微阵列数据集进行分类评价;Wang等<citation id="134" type="reference"><link href="17" rel="bibliography" /><sup>[<a class="sup">8</a>]</sup></citation>采用改进的偏最小二乘递归式特征消除 (PLS-RFE) 算法对多个微阵列数据集进行特征分类和选择, 计算效率得到提高;Sharbaf等<citation id="135" type="reference"><link href="19" rel="bibliography" /><sup>[<a class="sup">9</a>]</sup></citation>先采用Fisher指标进行滤波, 降低数据集维数, 然后将元胞学习自动机 (CLA) 与蚁群算法 (ACO) 相结合, 提高了基因特征分类精度;Khan等<citation id="136" type="reference"><link href="21" rel="bibliography" /><sup>[<a class="sup">10</a>]</sup></citation>提出了一种新的自适应径向基核函数, 并在非线性系统辨识、微阵列数据分类以及函数近似计算中做了仿真研究;Xiao等<citation id="137" type="reference"><link href="23" rel="bibliography" /><sup>[<a class="sup">11</a>]</sup></citation>提出一种基于多模型集成的深度学习算法, 并对三种癌症数据集进行验证;李颖新等<citation id="138" type="reference"><link href="25" rel="bibliography" /><sup>[<a class="sup">12</a>]</sup></citation>研究了急性白血病的分类信息基因选取, 并以SVM作为分类器进行亚型识别;马煜等<citation id="139" type="reference"><link href="27" rel="bibliography" /><sup>[<a class="sup">13</a>]</sup></citation>将密度聚类与共享近邻法相结合, 对微阵列数据进行聚类分析;韩利等<citation id="140" type="reference"><link href="29" rel="bibliography" /><sup>[<a class="sup">14</a>]</sup></citation>将粗糙集与SVM结合, 通过粗糙集进行基因特征约简, 然后用SVM进行数据分类;朱钦平等<citation id="141" type="reference"><link href="31" rel="bibliography" /><sup>[<a class="sup">15</a>]</sup></citation>提出了一种微阵列基因差异表达的多重假设检验方法, 有效地减弱了数据噪声带来的假阳性结果;姚全珠等<citation id="142" type="reference"><link href="33" rel="bibliography" /><sup>[<a class="sup">16</a>]</sup></citation>研究了最小二乘支持向量机 (LS-SVM) 特征选择时参数优化算法;孙刚等<citation id="143" type="reference"><link href="35" rel="bibliography" /><sup>[<a class="sup">17</a>]</sup></citation>采用改进的LASSO算法对信息基因进行特征选择, 剔除冗余基因;杨勤等<citation id="144" type="reference"><link href="37" rel="bibliography" /><sup>[<a class="sup">18</a>]</sup></citation>提出一种核最小二乘特征基因选择方法, 对微阵列数据进行降维, 然后用极限学习机进行训练和预测。</p>
                </div>
                <div class="p1">
                    <p id="44">本文采用两种典型的肿瘤微阵列数据集 (结肠癌数据集、白血病数据集) , 对数据进行归一化处理, 计算其相关系数矩阵;使用主成分分析 (PCA) 法进行降维;使用LS-SVM对降维后的特征信息基因进行分类, 并与其他几种分类方法进行了比较。</p>
                </div>
                <h3 id="45" name="45" class="anchor-tag"><b>1 问题描述</b></h3>
                <div class="p1">
                    <p id="46">假设微阵列特征分类问题可表示为集合<i>O</i>= (<i>X</i>, <i>Y</i>, <i>F</i>) , 其中, <i>X</i>={<b><i>x</i></b><sub>1</sub>, <b><i>x</i></b><sub>2</sub>, …, <b><i>x</i></b><sub><i>N</i></sub>}为样本集, 共有<i>N</i>个样本;<i>Y</i>={<i>y</i><sub>1</sub>, <i>y</i><sub>2</sub>, …, <i>y</i><sub><i>N</i></sub>}为信息标签集;<i>F</i>={<b><i>f</i></b><sub>1</sub>, <b><i>f</i></b><sub>2</sub>, …, <b><i>f</i></b><sub><i>N</i></sub>}为特征集;并且, <b><i>x</i></b><sub><i>k</i></sub>∈<i>X</i>是一个包含<i>m</i>个元素 (基因表达水平) 的向量, 可表示为<b><i>x</i></b><sub><i>k</i></sub>=[<i>x</i><sub>1<i>k</i></sub>, <i>x</i><sub>2<i>k</i></sub>, …, <i>x</i><sub><i>mk</i></sub>]<sup>T</sup>∈<b><i>R</i></b><sup><i>m</i></sup>;<i>y</i><sub><i>k</i></sub>∈<i>Y</i>是与<b><i>x</i></b><sub><i>k</i></sub>相对应的标量;假设为两分类 (<i>ω</i><sub>1</sub>和<i>ω</i><sub>2</sub>) 问题, 则有:</p>
                </div>
                <div class="area_img" id="47">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JYRJ201908049_04700.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="49">目的是寻找一组特征信息基因向量<b><i>f</i></b><sub><i>k</i></sub>=[<i>f</i><sub>1<i>k</i></sub>, <i>f</i><sub>2<i>k</i></sub>, …, <i>f</i><sub><i>pk</i></sub>]<sup>T</sup>∈<i>F</i> (<i>p</i>≤<i>m</i>) , 使之能够精确区分样本的基因表达数据。假定所选的特征子集的数目<i>p</i>尽可能小。</p>
                </div>
                <h3 id="50" name="50" class="anchor-tag"><b>2 SVM和LS-SVM</b></h3>
                <div class="p1">
                    <p id="51">SVM是一种基于统计学习理论, 采用结构风险最小化原理的机器学习算法, 可以有效地处理高维样本的分类问题, 计算复杂度受样本维数的影响较小, 适合处理小样本、高维数的基因表达数据的样本分类问题。</p>
                </div>
                <div class="p1">
                    <p id="52">SVM模型的目的是构造一个如下形式的最优分类函数:</p>
                </div>
                <div class="p1">
                    <p id="53"><i>f</i> (<i>x</i>) =sgn[<b><i>w</i></b><sup>T</sup><i>φ</i> (<i>x</i>) +<i>b</i>]      (2) </p>
                </div>
                <div class="p1">
                    <p id="54">式中:<i>φ</i> (<i>x</i>) 为将输入数据映射到高维特征空间的非线性映射;<b><i>w</i></b>为超平面权值系数向量;<i>b</i>为偏置项。标准支持向量机分类问题可描述为如下优化问题:</p>
                </div>
                <div class="p1">
                    <p id="55"><mathml id="56"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><munder><mstyle mathsize="140%" displaystyle="true"><mrow><mi>min</mi></mrow></mstyle><mrow><mi>w</mi><mo>, </mo><mi>b</mi><mo>, </mo><mi>e</mi></mrow></munder><mi>J</mi><mo stretchy="false"> (</mo><mi mathvariant="bold-italic">w</mi><mo>, </mo><mi mathvariant="bold-italic">e</mi><mo stretchy="false">) </mo><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mi mathvariant="bold-italic">w</mi><msup><mrow></mrow><mtext>Τ</mtext></msup><mi mathvariant="bold-italic">w</mi><mo>+</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mi>γ</mi><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mi>e</mi></mstyle><msub><mrow></mrow><mi>i</mi></msub></mrow></math></mathml>      (3) </p>
                </div>
                <div class="p1">
                    <p id="57">s.t. <i>y</i><sub><i>k</i></sub>[<b><i>w</i></b><sup>T</sup><i>φ</i> (<b><i>x</i></b><sub><i>k</i></sub>) +<i>b</i>]≥1-<i>e</i><sub><i>k</i></sub></p>
                </div>
                <div class="p1">
                    <p id="58"><i>e</i><sub><i>k</i></sub>≥0, <i>k</i>=1, 2, …, <i>N</i></p>
                </div>
                <div class="p1">
                    <p id="59">式中:<i>e</i><sub><i>k</i></sub>为误差;<b><i>w</i></b>=[<i>w</i><sub>1</sub>, <i>w</i><sub>2</sub>, …, <i>w</i><sub><i>N</i></sub>]<sup>T</sup>为权值系数向量;<i>γ</i>&gt;0为惩罚系数, 它控制对超出误差样本的惩罚程度。</p>
                </div>
                <div class="p1">
                    <p id="60">LS-SVM算法是在SVM的基础上通过最小二乘法利用误差平方和选择超平面, 然后引进平方损失函数, 将不等式约束转换为线性等式条件, 将二次规划问题转化为线性求解问题。LS-SVM分类问题可描述为求解下面的等式约束优化问题:</p>
                </div>
                <div class="p1">
                    <p id="61"><mathml id="62"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><munder><mstyle mathsize="140%" displaystyle="true"><mrow><mi>min</mi></mrow></mstyle><mrow><mi>w</mi><mo>, </mo><mi>b</mi><mo>, </mo><mi>e</mi></mrow></munder><mi>J</mi><mo stretchy="false"> (</mo><mi mathvariant="bold-italic">w</mi><mo>, </mo><mi mathvariant="bold-italic">e</mi><mo stretchy="false">) </mo><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mi mathvariant="bold-italic">w</mi><msup><mrow></mrow><mtext>Τ</mtext></msup><mi mathvariant="bold-italic">w</mi><mo>+</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mi>γ</mi><mi mathvariant="bold-italic">e</mi><msup><mrow></mrow><mtext>Τ</mtext></msup><mi mathvariant="bold-italic">e</mi></mrow></math></mathml>      (4) </p>
                </div>
                <div class="p1">
                    <p id="63">s.t. <i>y</i><sub><i>k</i></sub>[<b><i>w</i></b><sup>T</sup><i>φ</i> (<b><i>x</i></b><sub><i>k</i></sub>) +<i>b</i>]=1-<i>e</i><sub><i>k</i></sub></p>
                </div>
                <div class="p1">
                    <p id="64"><i>k</i>=1, 2, …, <i>N</i></p>
                </div>
                <div class="p1">
                    <p id="65">式中:<b><i>e</i></b>=[<i>e</i><sub>1</sub>, <i>e</i><sub>2</sub>, …, <i>e</i><sub><i>N</i></sub>]<sup>T</sup>。</p>
                </div>
                <div class="p1">
                    <p id="66">构造Lagrange函数如下:</p>
                </div>
                <div class="p1">
                    <p id="67" class="code-formula">
                        <mathml id="67"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left"><mtr><mtd><mtext> </mtext><mi>L</mi><mo stretchy="false"> (</mo><mi mathvariant="bold-italic">w</mi><mo>, </mo><mi>b</mi><mo>, </mo><mi mathvariant="bold-italic">e</mi><mo>;</mo><mi mathvariant="bold-italic">α</mi><mo stretchy="false">) </mo><mo>=</mo><mi>J</mi><mo stretchy="false"> (</mo><mi mathvariant="bold-italic">w</mi><mo>, </mo><mi mathvariant="bold-italic">e</mi><mo stretchy="false">) </mo><mo>-</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mi>α</mi></mstyle><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">{</mo><mi>y</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">[</mo><mi mathvariant="bold-italic">w</mi><msup><mrow></mrow><mtext>Τ</mtext></msup><mi>φ</mi><mo stretchy="false"> (</mo><mi mathvariant="bold-italic">x</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">) </mo><mo>+</mo></mtd></mtr><mtr><mtd><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext></mrow><mi>b</mi><mo stretchy="false">]</mo><mo>-</mo><mn>1</mn><mo>+</mo><mi>e</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">}</mo><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>5</mn><mo stretchy="false">) </mo></mtd></mtr></mtable></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="68">式中:<i>α</i><sub><i>k</i></sub>≥0 (<i>k</i>=1, 2…, <i>N</i>) 为Lagrange乘子。对上式进行优化, 即求<i>L</i>对<b><i>w</i></b>、<i>b</i>、<i>e</i><sub><i>k</i></sub>、<i>α</i><sub><i>k</i></sub>的偏导数为零, 经过化简, 可得到如下的线性方程组:</p>
                </div>
                <div class="area_img" id="69">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JYRJ201908049_06900.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="71">式中:</p>
                </div>
                <div class="p1">
                    <p id="72"><b><i>Z</i></b>=[<i>φ</i> (<b><i>x</i></b><sub>1</sub>) <i>y</i><sub>1</sub>, <i>φ</i> (<b><i>x</i></b><sub>2</sub>) <i>y</i><sub>2</sub>, …, <i>φ</i> (<b><i>x</i></b><sub><i>N</i></sub>) <i>y</i><sub><i>N</i></sub>]<sup>T</sup>, </p>
                </div>
                <div class="p1">
                    <p id="73"><b><i>y</i></b>=[<i>y</i><sub>1</sub>, <i>y</i><sub>2</sub>, …, <i>y</i><sub><i>N</i></sub>]<sup>T</sup>, 1<sub><i>N</i></sub>=[1, 1, …, 1]<sup>T</sup>, </p>
                </div>
                <div class="p1">
                    <p id="74"><i>α</i>=[<i>α</i><sub>1</sub>, <i>α</i><sub>2</sub>, …, <i>α</i><sub><i>N</i></sub>]<sup>T</sup>, <b><i>I</i></b>为单位矩阵。消除变量<b><i>w</i></b>、<b><i>e</i></b>, 再利用Mercer条件:</p>
                </div>
                <div class="p1">
                    <p id="75"><i>Ω</i><sub><i>sl</i></sub>=<i>y</i><sub><i>s</i></sub><i>y</i><sub><i>l</i></sub><i>φ</i><sup>T</sup> (<b><i>x</i></b><sub><i>s</i></sub>) <i>φ</i> (<b><i>x</i></b><sub><i>l</i></sub>) =<i>y</i><sub><i>s</i></sub><i>y</i><sub><i>l</i></sub><i>K</i> (<b><i>x</i></b><sub><i>s</i></sub>, <b><i>x</i></b><sub><i>l</i></sub>) </p>
                </div>
                <div class="p1">
                    <p id="76"><i>s</i>, <i>l</i>=1, 2, …, <i>N</i>      (7) </p>
                </div>
                <div class="p1">
                    <p id="77">可得矩阵方程:</p>
                </div>
                <div class="area_img" id="78">
                            <div class="imgformula">
                                <img class="pFormula" alt="" src="Detail/GetImg?filename=images/JYRJ201908049_07800.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                <p class="formula_seq"></p>
                            </div>

                </div>
                <div class="p1">
                    <p id="80">式中:<i>Ω</i>=[<i>Ω</i><sub><i>sl</i></sub>]<sub><i>N</i>×<i>N</i></sub>。假设<b><i>A</i></b>=<i>Ω</i>+<i>γ</i><sup>-1</sup><b><i>I</i></b>, 由于<b><i>A</i></b>为对称半正定矩阵, 因而<b><i>A</i></b><sup>-1</sup>存在, 上式有解。得到LS-SVM分类器为:</p>
                </div>
                <div class="p1">
                    <p id="81"><i>f</i> (<i>x</i>) =sgn[<i>α</i><sub><i>k</i></sub><i>y</i><sub><i>k</i></sub><i>K</i> (<i>x</i>, <b><i>x</i></b><sub><i>k</i></sub>) +<i>b</i>] <i>k</i>=1, 2, …, <i>N</i>      (9) </p>
                </div>
                <div class="p1">
                    <p id="82">式中:<i>α</i><sub><i>k</i></sub>、<i>b</i>为式 (8) 的解。</p>
                </div>
                <div class="p1">
                    <p id="83">取径向基核函数为:</p>
                </div>
                <div class="p1">
                    <p id="84" class="code-formula">
                        <mathml id="84"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>Κ</mi><mo stretchy="false"> (</mo><mi>x</mi><mo>, </mo><mi mathvariant="bold-italic">x</mi><msub><mrow></mrow><mi>k</mi></msub><mo stretchy="false">) </mo><mo>=</mo><mrow><mi>exp</mi></mrow><mrow><mo>{</mo><mrow><mo>-</mo><mfrac><mrow><mrow><mo>|</mo><mrow><mi>x</mi><mo>-</mo><mi mathvariant="bold-italic">x</mi><msub><mrow></mrow><mi>k</mi></msub></mrow><mo>|</mo></mrow><msup><mrow></mrow><mn>2</mn></msup></mrow><mrow><mi>σ</mi><msup><mrow></mrow><mn>2</mn></msup></mrow></mfrac></mrow><mo>}</mo></mrow><mtext> </mtext><mi>k</mi><mo>=</mo><mn>1</mn><mo>, </mo><mn>2</mn><mo>, </mo><mo>⋯</mo><mo>, </mo><mi>Ν</mi><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>0</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <h3 id="85" name="85" class="anchor-tag"><b>3 算法实现</b></h3>
                <h4 class="anchor-tag" id="86" name="86"><b>3.1 数据预处理</b></h4>
                <div class="p1">
                    <p id="87">通常, 原始数据集在特征选择之前应该被标准化。对于微阵列数据上一个基因中每个样本的表达值, 减去该基因所有样本的平均值, 再除以该基因所有样本的标准差。经过标准化之后, 一个基因在所有样本上的表达值满足均值为0, 标准差为1的正态分布。</p>
                </div>
                <div class="p1">
                    <p id="88">针对微阵列样本集<i>X</i>={<b><i>x</i></b><sub>1</sub>, <b><i>x</i></b><sub>2</sub>, …, <b><i>x</i></b><sub><i>N</i></sub>}, 且<b><i>x</i></b><sub><i>k</i></sub>=[<i>x</i><sub>1<i>k</i></sub>, <i>x</i><sub>2<i>k</i></sub>, …, <i>x</i><sub><i>mk</i></sub>]<sup>T</sup>, 数据归一化计算如下:</p>
                </div>
                <div class="p1">
                    <p id="89" class="code-formula">
                        <mathml id="89"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>z</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>k</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>x</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>k</mi></mrow></msub><mo>-</mo><mi>μ</mi><msub><mrow></mrow><mi>i</mi></msub></mrow><mrow><mi>σ</mi><msub><mrow></mrow><mi>i</mi></msub></mrow></mfrac><mtext> </mtext><mi>i</mi><mo>=</mo><mn>1</mn><mo>, </mo><mn>2</mn><mo>, </mo><mo>⋯</mo><mo>, </mo><mi>m</mi><mo>, </mo><mi>k</mi><mo>=</mo><mn>1</mn><mo>, </mo><mn>2</mn><mo>, </mo><mo>⋯</mo><mo>, </mo><mi>Ν</mi><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mtext> </mtext><mrow><mtext> </mtext><mtext> </mtext><mtext> </mtext></mrow><mo stretchy="false"> (</mo><mn>1</mn><mn>1</mn><mo stretchy="false">) </mo></mrow></math></mathml>
                    </p>
                </div>
                <div class="p1">
                    <p id="90">式中:<i>μ</i><sub><i>i</i></sub>和<i>σ</i><sub><i>i</i></sub>分别是为第<i>i</i>个基因表达值的均值和标准差。</p>
                </div>
                <h4 class="anchor-tag" id="91" name="91"><b>3.2 相关系数矩阵</b></h4>
                <div class="p1">
                    <p id="92">对归一化数据求相关系数为:</p>
                </div>
                <div class="p1">
                    <p id="93"><mathml id="94"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi>r</mi><msub><mrow></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mfrac><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mi>z</mi></mstyle><msub><mrow></mrow><mrow><mi>i</mi><mi>k</mi></mrow></msub><mi>z</mi><msub><mrow></mrow><mrow><mi>j</mi><mi>k</mi></mrow></msub></mrow><mrow><msqrt><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mi>z</mi></mstyle><msubsup><mrow></mrow><mrow><mi>i</mi><mi>k</mi></mrow><mn>2</mn></msubsup><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>Ν</mi></munderover><mi>z</mi></mstyle><msubsup><mrow></mrow><mrow><mi>j</mi><mi>k</mi></mrow><mn>2</mn></msubsup></mrow></msqrt></mrow></mfrac><mtext> </mtext><mi>i</mi><mo>, </mo><mi>j</mi><mo>=</mo><mn>1</mn><mo>, </mo><mn>2</mn><mo>⋯</mo><mo>, </mo><mi>m</mi></mrow></math></mathml>      (12) </p>
                </div>
                <h4 class="anchor-tag" id="95" name="95"><b>3.3 提取主成分分量</b></h4>
                <div class="p1">
                    <p id="96">主成分分析法的基本思想是在保留尽可能多的原始信息的前提下达到降维的目的。</p>
                </div>
                <div class="p1">
                    <p id="97">求解特征方程:<mathml id="98"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mrow><mo>|</mo><mrow><mi>λ</mi><mi mathvariant="bold-italic">Ι</mi><mo>-</mo><mi mathvariant="bold-italic">R</mi></mrow><mo>|</mo></mrow><mo>=</mo><mn>0</mn></mrow></math></mathml>, 其中, <b><i>R</i></b>=[<i>r</i><sub><i>ij</i></sub>]∈<b><i>R</i></b><sup><i>m</i>×<i>m</i></sup>为相关系数矩阵;求出<i>m</i>个特征值<i>λ</i><sub>1</sub>≥<i>λ</i><sub>2</sub>≥…≥<i>λ</i><sub><i>m</i></sub>≥0。确定主成分数量<i>p</i> (&lt;<i>m</i>) 的值, 使信息的利用率达到85%以上。</p>
                </div>
                <div class="p1">
                    <p id="99">实现微阵列数据的降维和特征分类步骤如下:</p>
                </div>
                <div class="p1">
                    <p id="100"><b>Step 1</b> 数据预处理。对微阵列数据集进行归一化处理。</p>
                </div>
                <div class="p1">
                    <p id="101"><b>Step 2</b> 提取主成分分量。计算相关系数矩阵;采用PCA法, 对所选择的基因样本数据进行降维处理, 得到样本的特征信息基因集。</p>
                </div>
                <div class="p1">
                    <p id="102"><b>Step 3</b> 分类模型训练。对特征提取后的信息基因数据分别采用LS-SVM等分类器进行训练, 得到分类模型。</p>
                </div>
                <div class="p1">
                    <p id="103"><b>Step 4</b> 测试分类模型。将测试样本代入分类模型中, 分别采用留一检测法和独立测试法评估各种分类器的性能。</p>
                </div>
                <h3 id="104" name="104" class="anchor-tag"><b>4 仿真实验</b></h3>
                <h4 class="anchor-tag" id="105" name="105"><b>4.1 实验数据及开发环境</b></h4>
                <div class="p1">
                    <p id="106">实验采用两个公开的微阵列数据集来评估本文算法的性能。数据集的详细描述见表1。结肠癌数据集包括62个样本, 且分成两类:正常样本和结肠癌样本。其中, 正常样本22个, 结肠癌样本40个和2 000个基因<citation id="146" type="reference"><link href="3" rel="bibliography" /><sup>[<a class="sup">1</a>]</sup></citation>;白血病数据集包括128个样本, 分属于两类不同类型的肿瘤:T细胞ALL (共33例) 样本和B细胞ALL (共95例) 样本和12 625个基因<citation id="147" type="reference"><link href="11" rel="bibliography" /><link href="39" rel="bibliography" /><sup>[<a class="sup">5</a>,<a class="sup">19</a>]</sup></citation>。</p>
                </div>
                <div class="area_img" id="107">
                    <p class="img_tit"><b>表1 实验数据及描述</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="107" border="1"><tr><td>数据集</td><td>基因数量</td><td>样本数量</td><td>类别数</td></tr><tr><td><br />结肠癌</td><td>2 000</td><td>62 (22/40) </td><td>2</td></tr><tr><td><br />白血病</td><td>12 625</td><td>128 (33/95) </td><td>2</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="108">本文的实验环境:Intel CPU 2.53 GHz处理器, 2 GB内存的PC机, Windows XP操作系统, MATLAB 2014b开发环境。</p>
                </div>
                <h4 class="anchor-tag" id="109" name="109"><b>4.2 实验结果分析</b></h4>
                <h4 class="anchor-tag" id="110" name="110"><b>实验1</b> 结肠癌数据集分类</h4>
                <div class="p1">
                    <p id="111">针对结肠癌数据集, 实验首先经过数据预处理, 然后, 将正常样本和肿瘤样本按接近2∶1的比例随机地分配到训练集和测试集中。训练集有40个样本 (其中正常样本14, 肿瘤样本26) , 测试集有22个样本 (正常样本8个, 肿瘤样本14个) 。</p>
                </div>
                <div class="p1">
                    <p id="112">然后通过PCA降维方法, 提取主成分前十的特征信息基因如表2所示。</p>
                </div>
                <div class="area_img" id="113">
                    <p class="img_tit"><b>表2 结肠癌数据集中选取的特征基因</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="113" border="1"><tr><td>序号</td><td>基因名称</td><td>序号</td><td>基因名称</td></tr><tr><td><br />1</td><td>X53799</td><td>6</td><td>R80427</td></tr><tr><td><br />2</td><td>M29273</td><td>7</td><td>X75208</td></tr><tr><td><br />3</td><td>U21914</td><td>8</td><td>D29808</td></tr><tr><td><br />4</td><td>L00352</td><td>9</td><td>M59807</td></tr><tr><td><br />5</td><td>X90858</td><td>10</td><td>D13627</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="114">分别采用LS-SVM等分类器对选取的特征基因进行分类。各分类器分类准确率结果见表3。图1、图2为LS-SVM分类器训练模型准确率 (100%) 以及独立测试实验准确率 (68.18%) 结果图。</p>
                </div>
                <div class="area_img" id="116">
                    <p class="img_tit"><b>表3 结肠癌数据集选取的特征基因集实验结果</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="116" border="1"><tr><td>分类器</td><td>基因数</td><td>留一交叉<br />检验准确率</td><td>独立测试<br />准确率</td></tr><tr><td><br />PNN</td><td>10</td><td>0.625</td><td>0.682</td></tr><tr><td><br />RBF</td><td>10</td><td>0.675</td><td>0.682</td></tr><tr><td><br />BP</td><td>10</td><td>0.650</td><td>0.636</td></tr><tr><td><br />SVM RBF_ker</td><td>10</td><td>0.675</td><td>0.727</td></tr><tr><td><br />SVM Lin_ker</td><td>10</td><td>0.600</td><td>0.636</td></tr><tr><td><br />SVM poly_ker</td><td>10</td><td>0.625</td><td>0.727</td></tr><tr><td><br />LS-SVM RBF_ker</td><td>10</td><td>0.975</td><td>0.682</td></tr><tr><td colspan="4"><br />注:PNN表示概率神经网络;RBF为径向基神经网络;BP为反向传播神经网络;RBF_ker、Lin_ker、poly_ker分别为径向基核函数、线性核函数和多项式核函数</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="area_img" id="117">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201908049_117.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图1 结肠癌数据LS-SVM训练模型准确率" src="Detail/GetImg?filename=images/JYRJ201908049_117.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图1 结肠癌数据LS-SVM训练模型准确率  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201908049_117.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="118">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201908049_118.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图2 结肠癌数据LS-SVM独立测试准确率" src="Detail/GetImg?filename=images/JYRJ201908049_118.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图2 结肠癌数据LS-SVM独立测试准确率  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201908049_118.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <h4 class="anchor-tag" id="119" name="119"><b>实验2</b> 白血病数据集分类</h4>
                <div class="p1">
                    <p id="120">白血病数据集经过数据预处理, 由PCA法求得主成分前十的特征信息基因如表4所示。</p>
                </div>
                <div class="area_img" id="121">
                    <p class="img_tit"><b>表4 白血病数据集中选取的特征基因</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="121" border="1"><tr><td>序号</td><td>基因名称</td><td>序号</td><td>基因名称</td></tr><tr><td><br />1</td><td>X110_at</td><td>6</td><td>X1221_at</td></tr><tr><td><br />2</td><td>X1098_at</td><td>7</td><td>X1210_s_at</td></tr><tr><td><br />3</td><td>X1189_at</td><td>8</td><td>X1086_at</td></tr><tr><td><br />4</td><td>X1106_at</td><td>9</td><td>X1173_g_at</td></tr><tr><td><br />5</td><td>X1178_at</td><td>10</td><td>X1199_at</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="p1">
                    <p id="122">将数据集中的两类样本分配到训练集和测试集中。训练集有65个样本 (T细胞样本有48个, B细胞样本有17个) , 测试集有63个样本 (T细胞样本有47个, B细胞样本有16个) 。</p>
                </div>
                <div class="p1">
                    <p id="123">对选取的特征基因进行分类, 分别采用留一交叉检验和独立测试实验, 结果见表5。图3、图4分别为LSSVM分类器留一交叉检验准确率 (100%) 和独立测试实验准确率 (93.65%) 结果图。</p>
                </div>
                <div class="area_img" id="125">
                    <p class="img_tit"><b>表5 白血病数据集选取的特征基因集实验结果</b> <a class="downexcel" onclick="DownLoadReportExcel(this)">导出到EXCEL</a></p>
                    <p class="img_tit"></p>
                    <p class="img_note"></p>
                    <table id="125" border="1"><tr><td>分类器</td><td>基因数</td><td>留一交叉<br />检验准确率</td><td>独立测试<br />准确率</td></tr><tr><td><br />PNN</td><td>10</td><td>0.969</td><td>0.746</td></tr><tr><td><br />RBF</td><td>10</td><td>0.892</td><td>0.825</td></tr><tr><td><br />BP</td><td>10</td><td>0.738</td><td>0.746</td></tr><tr><td><br />SVM RBF_ker</td><td>10</td><td>0.738</td><td>0.825</td></tr><tr><td><br />SVM Lin_ker</td><td>10</td><td>0.738</td><td>0.746</td></tr><tr><td><br />LS-SVM RBF_ker</td><td>10</td><td>1.000</td><td>0.937</td></tr></table>
                    <form name="form" action="/kxreader/Detail/DownloadReportExcel" method="POST" style="display:inline">
                        <input type="hidden" name="hidTable" value="" />
                        <input type="hidden" name="hidFileName" value="" />
                    </form>
                    <p class="img_note"></p>
                    <p class="img_note"></p>
                </div>
                <div class="area_img" id="126">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201908049_126.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图3 白血病数据LS-SVM留一交叉检验测试准确率" src="Detail/GetImg?filename=images/JYRJ201908049_126.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图3 白血病数据LS-SVM留一交叉检验测试准确率  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201908049_126.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="area_img" id="127">
                                <a class="zoom-in" href="Detail/GetImg?filename=images/JYRJ201908049_127.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">
                                    <img alt="图4 白血病数据集LS-SVM独立测试准确率" src="Detail/GetImg?filename=images/JYRJ201908049_127.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
                                </a>
                                <p class="img_tit">图4 白血病数据集LS-SVM独立测试准确率  <a class="btn-zoomin" href="Detail/GetImg?filename=images/JYRJ201908049_127.jpg&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!"></a><a class="downimg">&nbsp;&nbsp;下载原图</a></p>
                            <p class="img_tit"></p>

                </div>
                <div class="p1">
                    <p id="128">从表3、表5中看出, 对于两个数据集的留一交叉检验结果, LS-SVM分类器的准确率最高, 分别为97.5%和100%, 其次是PNN分类器和RBF分类器;独立测试实验结果中, 白血病特征基因集的LS-SVM分类器的准确率最高, 为93.65%, 而结肠癌数据集LS-SVM分类结果与其他分类器的结果差别不大。</p>
                </div>
                <h3 id="129" name="129" class="anchor-tag"><b>5 结 语</b></h3>
                <div class="p1">
                    <p id="130">微阵列数据对疾病的诊断有很重要的参考价值, 但是, 微阵列数据的高维和高冗余给进一步挖掘其中蕴含的知识带来极大困难, 其中一个关键任务就是信息基因的选择。LS-SVM分类器将SVM优化问题的不等式约束转换为线性等式条件, 将二次规划问题转化为线性求解问题, 避免了求解耗时, 提高了运行效率。本文基于LS-SVM分类器对两类癌症微阵列数据集的基因分别进行提取和分类。首先, 对微阵列数据进行归一化预处理, 计算其相关系数矩阵, 并运用PCA法进行降维。提取特征信息基因集 (各取10个基因) , 运用不同的分类器 (包括LS-SVM、PNN、RBF、BP及SVM) 进行分类。从留一交叉检验和独立测试两种实验结果可以看出, 运用LS-SVM分类器, 结肠癌集准确率分别达到97.5%和68.2%;白血病集准确率分别达到100%和93.7%, 从而证明了本文提出的算法比运用其他分类器计算准确率相对较高, 能够为医学临床实践提供较为可靠的判断依据。</p>
                </div>

        <!--brief end-->
        
        <!--conten left  end-->
        <!--增强附件-->
        

        <!--reference start-->
            <div class="reference anchor-tag" id="a_bibliography">
                    <h3>参考文献</h3>
                                        <p id="3">
                            <a id="bibliography_1" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Broad patterns of gene expression revealed by clustering analysis of tumor and normal colon tissues probed by oligonucleotide arrays">

                                <b>[1]</b> Alon U, Barkai N, Notterman D A, et al.Broad Patterns of Gene Expression Revealed by Clustering Analysis of Tumor and Normal Colon Tissues Probed by Oligonucleotide Arrays[J].Proceedings of the National Academy of Sciences of the United States of America, 1999, 96 (12) :6745-6750.
                            </a>
                        </p>
                        <p id="5">
                            <a id="bibliography_2" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJOX&amp;filename=SJOX14010900013873&amp;v=MjYxNjZubFVyeklJRjBkYnhFPU5pZkVkcks4SHRETXBvOUZaT29NQkhzNm9CTVQ2VDRQUUgvaXJSZEdlcnFRVE1ud1plWnRGaQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[2]</b> Furey T S, Cristianini N, Duffy N, et al.Support vector machine classification and validation of cancer tissue samples using microarray expression data[J].Bioinformatics, 2000, 16 (10) :906-914.
                            </a>
                        </p>
                        <p id="7">
                            <a id="bibliography_3" >
                                    <b>[3]</b>
                                 Golub T R, Slonim D K, Tamayo P, et al.Molecular classification of cancer:class discovery and class prediction by gene expression monitoring[J].Science, 1999, 286:531-537.
                            </a>
                        </p>
                        <p id="9">
                            <a id="bibliography_4" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SSJD&amp;filename=SSJD00001340375&amp;v=MTc1ODUrWnVGaXZsVTd6QkpWMD1OajdCYXJPNEh0SE5ySXRGWit3S1kzazV6QmRoNGo5OVNYcVJyeG94Y01IN1I3cWRa&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[4]</b> Guyon I, Weston J, Bamhill S, et al.Gene selection for cancer classification using support vector machines[J].Machine Learning, 2002, 46:389-422.
                            </a>
                        </p>
                        <p id="11">
                            <a id="bibliography_5" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Gene expression profile of adult T-cell acute lymphocytic leukemia identifies distinct subsets of patients with different response to therapy and survival">

                                <b>[5]</b> Chiaretti S, Li X C, Gentleman R, et al.Gene expression profile of adult T-cell acute lymphocytic leukemia identifies distinct subsets of patients with different response to therapy and survival[J].Blood, 2004, 103 (7) :2771-2778.
                            </a>
                        </p>
                        <p id="13">
                            <a id="bibliography_6" target="_blank" href="http://scholar.cnki.net/result.aspx?q=Gene expression profiling on lung cancer outcome prediction: present clinical value and future premise">

                                <b>[6]</b> Sun Z F, Yang P.Gene expression profiling on lung cancer outcome prediction:present clinical value and future premise[J].Cancer Epidemiology, Biomarkers &amp; Prevention, 2006, 15 (11) :2063-2068.
                            </a>
                        </p>
                        <p id="15">
                            <a id="bibliography_7" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES53EA201C30891CA4B88FC003F8BF26A8&amp;v=MzE2NzhjTE8zL2lyMlE5QzhTV1E4dVhDT052RlNpV1dyN0pJRnBtYUJ1SFlmT0dRbGZCckxVMDV0dGh4THUyeGFzPU5pZk9mYmE3YTZET3I0NDJaK3NIQlgxS3ZoSmg0ag==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[7]</b> Devi A V C, Devaraj D, Venkatesulu M.Gene Expression data classification using support vector machine and mutual information-based gene selection[J].Procedia Computer Science, 2015, 47:13-21.
                            </a>
                        </p>
                        <p id="17">
                            <a id="bibliography_8" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES15120600576292&amp;v=MDE2MDdRVE1ud1plWnRGaW5sVXJ6SUlGMGRieEU9TmlmT2ZiSzlIOVBNcVk5Rllld0pEblU3b0JNVDZUNFBRSC9pclJkR2VycQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[8]</b> Wang A G, An N, Chen G L, et al.Improving PLS-RFE based gene selection for microarray[J].Computers in Biology and Medicine, 2015, 62:14-24.
                            </a>
                        </p>
                        <p id="19">
                            <a id="bibliography_9" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A hybrid gene selection approach for microarray data classi?cation using cellular learning automata and ant colony optimization">

                                <b>[9]</b> Sharbaf F V, Mosafer S, Moattar M H.A hybrid gene selection approach for microarray data classification using cellular learning automata and colony optimization[J].Genomics, 2016, 107:231-238.
                            </a>
                        </p>
                        <p id="21">
                            <a id="bibliography_10" target="_blank" href="http://scholar.cnki.net/result.aspx?q=A novel adaptive kernel for the RBF neural networks">

                                <b>[10]</b> Khan S, Naseem I, Togneri R, et al.A Novel Adaptive Kernel for the RBF Neural Networks[J].Circuits, Systems, and Signal Processing, 2017, 36 (4) :1639-1653.
                            </a>
                        </p>
                        <p id="23">
                            <a id="bibliography_11" target="_blank" href="/kcms/detail/detail.aspx?dbcode=SJES&amp;filename=SJES5279C9563C2CED2868FE75A97A20A739&amp;v=MDg3MTdmT0dRbGZCckxVMDV0dGh4THUyeGFzPU5pZk9mYmE2R2RpL3BvcERaNWdOZndsTnpSNFY0a2tJVDNxVHBSVkVlN0xsUXJtV0NPTnZGU2lXV3I3SklGcG1hQnVIWQ==&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[11]</b> Xiao Y W, Wu J, Lin Z L, et al.A deep learning-based multi-model ensemble method for cancer prediction[J].Computer Methods and Programs in Biomedicine, 2018, 153:1-9.
                            </a>
                        </p>
                        <p id="25">
                            <a id="bibliography_12" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=ZSWY200502023&amp;v=MzIwNzMzenFxQnRHRnJDVVI3cWZadVp0RnlqbVdyckxQejdjZDdHNEh0VE1yWTlIWjRRS0RIODR2UjRUNmo1NE8=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[12]</b> 李颖新, 刘全金, 阮晓钢.急性白血病的基因表达谱分析与亚型分类特征的鉴别[J].中国生物医学工程学报, 2005, 24 (2) :240-244.
                            </a>
                        </p>
                        <p id="27">
                            <a id="bibliography_13" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSGG200605054&amp;v=MzIyMTlMTHo3TWFiRzRIdGZNcW85QVlJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVadEZ5am1XcnI=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[13]</b> 马煜, 陈莉, 欧立奇.基因微阵列数据的聚类分析算法研究[J].计算机工程与应用, 2006 (5) :176-178.
                            </a>
                        </p>
                        <p id="29">
                            <a id="bibliography_14" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=KXJS200901036&amp;v=MTExNDU3cWZadVp0RnlqbVdyckxMalhCZmJHNEh0ak1ybzlHWW9RS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[14]</b> 韩利, 祁云嵩, 王俊.基于粗糙集的支持向量机微阵列数据分类方法[J].科学技术与工程, 2009, 9 (1) :152-155.
                            </a>
                        </p>
                        <p id="31">
                            <a id="bibliography_15" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=KXJS201027014&amp;v=MDg2MzlMTGpYQmZiRzRIOUhPcUk5RVlJUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVadEZ5am1XcnI=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[15]</b> 朱钦平, 胡晓涵, 祁云嵩.基于非线性回归分析的差异基因选择方法[J].科学技术与工程, 2010, 10 (27) :6675-6678.
                            </a>
                        </p>
                        <p id="33">
                            <a id="bibliography_16" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JSGG201001042&amp;v=MjI3OTF0RnlqbVdyckxMejdNYWJHNEg5SE1ybzlCWm9RS0RIODR2UjRUNmo1NE8zenFxQnRHRnJDVVI3cWZadVo=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[16]</b> 姚全珠, 蔡婕.基于PSO的LS-SVM特征选择与参数优化算法[J].计算机工程与应用, 2010, 46 (1) :134-136, 229.
                            </a>
                        </p>
                        <p id="35">
                            <a id="bibliography_17" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=XXWX201506011&amp;v=MjQxNjRSN3FmWnVadEZ5am1XcnJMUFRYY2RyRzRIOVRNcVk5RVpZUUtESDg0dlI0VDZqNTRPM3pxcUJ0R0ZyQ1U=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[17]</b> 孙刚, 张靖.面向高维微阵列数据的混合特征选择算法[J].小型微型计算机系统, 2015, 36 (6) :1209-1213.
                            </a>
                        </p>
                        <p id="37">
                            <a id="bibliography_18" target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=CGQJ201605042&amp;v=MDg2NzJPM3pxcUJ0R0ZyQ1VSN3FmWnVadEZ5am1XcnJMSmlyYVpMRzRIOWZNcW85QlpvUUtESDg0dlI0VDZqNTQ=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">

                                <b>[18]</b> 杨勤, 董洪伟, 薛燕娜.核多元基因选择和极限学习机在微阵列分析中的应用[J].传感器与微系统, 2016, 35 (5) :146-148, 153.
                            </a>
                        </p>
                        <p id="39">
                            <a id="bibliography_19" >
                                    <b>[19]</b>
                                 Torgo L.数据挖掘与R语言[M].李洪成, 陈道轮, 吴立明, 译.北京:机械工业出版社, 2016.
                            </a>
                        </p>
            </div>
        <!--reference end-->
        <!--footnote start-->
        <!--footnote end-->



    </div>

        <input id="fileid" type="hidden" value="JYRJ201908049" />
        <input id="dpi" type="hidden" value="300" />
    </div>

<script>
var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6e967eb120601ea41b9d312166416aa6";
  var s = document.getElementsByTagName("script")[0];
  s.parentNode.insertBefore(hm, s);
})();
</script>

    


<input id="hid_uid" name="hid_uid" type="hidden" value="WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!" />
<input id="hid_kLogin_headerUrl" name="hid_kLogin_headerUrl" type="hidden" value="/KLogin/Request/GetKHeader.ashx%3Fcallback%3D%3F" />
<input id="hid_kLogin_footerUrl" name="hid_kLogin_footerUrl" type="hidden" value="/KLogin/Request/GetKFooter.ashx%3Fcallback%3D%3F" />
<div class="btn-link" style="display: none"><a target="_blank" href="/kcms/detail/detail.aspx?dbcode=CJFD&amp;filename=JYRJ201908049&amp;v=MTU2ODlqNTRPM3pxcUJ0R0ZyQ1VSN3FmWnVadEZ5am1XcnJMTHpUWlpMRzRIOWpNcDQ5QmJZUUtESDg0dlI0VDY=&amp;uid=WEEvREcwSlJHSldRa1FhdkJtNEYxa3hKZG9XL1JrZXNhdGVWVEdvM0tSMD0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4IQMovwHtwkF4VYPoHbKxJw!!">知网节</a></div>
<div class="popflow" id="popupTips" style="display: none;">
    <div class="popflowArr"></div>
    <div class="popflowCot">
        <div class="hd"><a href="javascript:void(0);" onclick="$('#popupTips').hide();$('#popupmsg').html('')" class="close">X</a></div>
        <div class="bd">
            <p class="mes" id="popupmsg" name="popupmsg"></p>
          
        </div>
    </div>
</div>
<input type="hidden" id="myexport" value="//kns.cnki.net" />

<input type="hidden" id="KPCAPIPATH" value="//ishufang.cnki.net" />
<input type="hidden" id="CitedTimes" value="" />
<div class="link" id="GLSearch" style="display: none;">
    <i class="icon-trangle"></i>
    <div class="inner">
        <a class="icon" id="copytext">复制</a>
        <a class="icon" target="_blank" onclick="searchCRFD(this)">工具书搜索</a>
    </div>
</div>




<input id="hidVirtualPath" name="hidVirtualPath" type="hidden" value="/kxreader" />
<script src="/kxreader/bundles/detail?v=-ULdk-c6FkZHtJA2KAXPgHnyA8mtgyPnBde_C2VZ2BY1"></script>

<script src="/kxreader/Scripts/layer.min.js" type="text/javascript"></script>

<div id="footerBox" class="rootw footer">
</div>
<script>
    if (typeof FlushLogin == 'function') {
        FlushLogin();
    }
    modifyEcpHeader(true);
</script>

<!--图片放大功能 start-->
<script src="/kxreader/bundles/imagebox?v=W4phPu9SNkGcuPeJclikuVE3PpRyIW_gnfjm_19nynI1"></script>

<script type="text/javascript">
    $(function () {
        var j = $.noConflict();
        j(function () {
            j(".zoom-in,.btn-zoomin").imgbox({
                'alignment': 'center',
                'allowMultiple': false,
                'overlayShow': true
            });
        })
    });
</script>
<!--图片放大功能 end-->
<div class="fixedbar">
    <div class="backtop hiddenV" id="backtop">
        <a id="backTopSide" href="javascript:scroll(0,0);" title=""></a>
    </div>
</div>
<script type="text/javascript" src="/kxreader/Scripts/MathJax-2.6-latest/MathJax.js?config=MML_HTMLorMML-full"></script>

</body>
</html>
